<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Society of Mathematical Biology | SMB 2020</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Page 34 of 34 for Society of Mathematical Biology | SMB 2020</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Society of Mathematical Biology" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/page34/" />
<meta property="og:url" content="http://localhost:4000/page34/" />
<meta property="og:site_name" content="SMB 2020" />
<link rel="prev" href="http://localhost:4000/page33" />
<script type="application/ld+json">
{"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Society of Mathematical Biology","description":"The Society for Mathematical Biology - e-Conference 2020.","url":"http://localhost:4000/page34/","@type":"WebPage","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


    <!-- This goes before </head> closing tag, Google Analytics can be placed here --> 






		<meta name="twitter:card" content="summary_large_image">
		<meta name="twitter:site" content="smb2020.org">
		<meta name="twitter:creator" content="@ESmb2020">
		<meta name="twitter:title" content="Society of Mathematical Biology">
		<meta name="twitter:description" content="Society of Mathematical Biology">
		<meta name="twitter:image" content="https://raw.githubusercontent.com/JeffreyWest/esmb/master/assets/images/preview.png">

		<meta name="title" content="Society of Mathematical Biology" />
		
		<meta property="og:title" content="Society of Mathematical Biology" />
		
		
			<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
			<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
		
		
		
		<meta property="og:image" content="https://raw.githubusercontent.com/JeffreyWest/esmb/master/assets/images/preview.png"/>

		
	


</head>

<body class="">

    <!-- Navbar -->
    <nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
    <div class="container">
        <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarColor02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
        </button>
        <div class="navbar-collapse collapse" id="navbarColor02" style="">
            <ul class="navbar-nav mr-auto d-flex align-items-center">
               <!--  Replace menu links here -->

<li class="nav-item">
	
		<img src="../assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2">
	
</li>


<a class="navbar-brand" href="../"><strong>SMB 2020</strong></a>



<li class="nav-item">
	
	<a class="nav-link" href="../">Home</a>
	
</li>


<li class="nav-item">
	
	<a class="nav-link" href="../conference-announcement">Announcement</a>
	
</li>


<li class="nav-item">
	
		<a class="nav-link" href="../registration">Registration</a>
	
</li>

<li class="nav-item">
	
		<a class="nav-link" href="../store">Store</a>
	
</li>

<li class="nav-item">
	
		<a class="nav-link" id="strong-nav" href="../schedule"><strong>Schedule</strong></a>
	
</li>

<li class="nav-item">
	
	<a class="nav-link" href="../subgroups">Subgroups</a>
	
</li>

<li class="nav-item">
	
	<a class="nav-link" href="../faq">FAQ</a>
	
</li>

<li class="nav-item">
	
	<a class="nav-link" href="../logistics">Logistics</a>
	
</li>

            </ul>
<!--
            <ul class="navbar-nav ml-auto d-flex align-items-center">
                
<script src="../assets/js/lunr.js"></script>



<script>
$(function() {
    $("#lunrsearchresults").on('click', '#btnx', function () {
        $('#lunrsearchresults').hide( 1000 );
        $( "body" ).removeClass( "modal-open" );
    });
});
    

var documents = [{
    "id": 0,
    "url": "http://localhost:4000/404/",
    "title": "",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": " 404 Page not found :(  The requested page could not be found. "
    }, {
    "id": 1,
    "url": "http://localhost:4000/CDEV-posters.html",
    "title": "CDEV Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 2,
    "url": "http://localhost:4000/EDUC-posters.html",
    "title": "EDUC Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 3,
    "url": "http://localhost:4000/IMMU-posters.html",
    "title": "IMMU Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 4,
    "url": "http://localhost:4000/MEPI-posters.html",
    "title": "MEPI Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 5,
    "url": "http://localhost:4000/MFBM-posters.html",
    "title": "MFBM Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 6,
    "url": "http://localhost:4000/NEUR-posters.html",
    "title": "NEUR Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 7,
    "url": "http://localhost:4000/ONCO-posters.html",
    "title": "ONCO Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 8,
    "url": "http://localhost:4000/OTHE-posters.html",
    "title": "OTHE Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 9,
    "url": "http://localhost:4000/POPD-posters.html",
    "title": "POPD Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 10,
    "url": "http://localhost:4000/all-posters.html",
    "title": "All Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 11,
    "url": "http://localhost:4000/author-jane.html",
    "title": "Jane",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "                        Jane Follow:                                                    Posts by Jane:               "
    }, {
    "id": 12,
    "url": "http://localhost:4000/author-sal.html",
    "title": "Sal",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "                        Sal Follow:                                                    Posts by Sal:                   		Linnea Franssen	: 		  layout: single-posterauthor: esmbday: “All”group: “poster”title: “A Mathematical Framework for Modelling the Metastatic Spread of Cancer”subgroup: “ONCO”presenter: “Linnea Franssen”i. . . 	 			In 								Aug 02, 2020	        "
    }, {
    "id": 13,
    "url": "http://localhost:4000/authors-list.html",
    "title": "Authors",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "Authors:                                             Sandy Anderson :       (View Posts)      President of Society for Mathematical Biology                          &nbsp;       &nbsp;                                                                             eSMB2020 :       (View Posts)      Virtual conference of the Society for Mathematical Biology, 2020.                           &nbsp;       &nbsp;                                      "
    }, {
    "id": 14,
    "url": "http://localhost:4000/categories.html",
    "title": "Categories",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "          Categories               SMB:                                  		Conference Announcement	: 		  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. We will probably remember these . . . 	 			In 				SMB, 				eSMB, 								Jun 08, 2020						                              eSMB:                                  		Conference Announcement	: 		  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. We will probably remember these . . . 	 			In 				SMB, 				eSMB, 								Jun 08, 2020						                                             Featured:    				                  "
    }, {
    "id": 15,
    "url": "http://localhost:4000/faq/",
    "title": "Frequently Asked Questions",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Frequently Asked Questions		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt; 	  								* For other questions, please contact Heiko Enderling at Heiko. Enderling@moffitt. org. 									1. Where will eSMB2020 take place?: 	  	   				The eSMB2020 conference will be held virtually. Through partnership with MathDept. org, we will utilize Sococo, Zoom, and Slack to create an immersive conference experience. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  2. What are the registration costs?: 	  	   				The registration fee is $25 for our first virtual annual meeting. You must register for the meeting through the registration website. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  3. Do I need to register if I am not presenting a poster or abstract?: 	  	   				Yes, registration is required for participation in eSMB2020. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;									4. What the poster requirements?: 	  	   				Registration is required to present a poster. Please register at your earliest convenience at smb2020. org/registration.  A tutorial video of the sococo platform is online for you to get a glimpse into what the virtual poster session will look and feel like ( http://smb2020. org/logistics/ ). Please follow the following guidelines: 													Poster dimensions: Max of 3' x 4/ (91cm x 122cm). 					Format: PDF only					File Size: Not to exceed 25MB. 					If necessary, here are instructions to reduce file size: https://www. wikihow. com/Reduce-PDF-File-Size. 					Please save your poster with the following File Name: Lastname_Firstname_XXXX. pdf, where XXXX is to be replaced by the identifier of the SMB subgroup the poster belongs to. XXXX = { CDEV, EDUC, IMMU, MEPI, NEUR, ONCO, MFBM, POPD, OTHE }. All subgroups (and corresponding codes) are listed on the subgroup page. 					Please upload your poster pdf to the Subgroup Google Drive folder that your subgroup chair emailed you. 															   	  	  	  &lt;/br&gt;			&lt;/br&gt;																															  	  5. If I submitted my abstract to the Heidelberg ESMTB meeting, is it too late to transfer it to SMB2020?: 	  	   				If you have not yet transferred your abstract, please submit your abstract as a new abstract during eSMB2020 registration.  	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  6. Will the Heidelberg ESMTB meeting still take place?: 	  	   				ESMTB organizers plan to hold ESMTB in 2021 in Heidelberg. Unfortunately, this won’t be a joint meeting with the SMB. The SMB 2021 annual meeting will be held at UC Riverside. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  7. Will my talk be recorded or live-streamed?: 	  	   				The eSMB2020 conference offers the unprecedented opportunity to record talks for dissemination after the meeting. Live-streaming is currently not planned. During abstract submission, authors can select to opt in/out of recordings.  	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  8. Can I re-submit an updated abstract?: 	  	   				Yes. To submit an updated abstract, please coordinate with the SMB subgroup lead that your abstract belongs to. If your abstract does not belong to an SMB subgroup, please contact Torbjörn Lundh. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  9. Can I submit an abstract that is not affiliated with any subgroup?: 	  	   				Yes, during abstract submission you choose the option ‘none of the above’ for subgroup affiliation. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  10. Can we still organize minisymposia?: 	  	   				Please reach out to the SMB subgroup leaders that your minisymposium thematically belongs to to confirm if accommodations can be made for additional minisymopisia. Note: the mathematical oncology section is currently at capacity and cannot accept further MS submissions. Otherwise, if the MS thematically does not belong to a subgroup, please contact Torbjörn Lundh. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  11. When will I be notified about abstract selection?: 	  	   				All submitted abstract will be evaluated by the SMB Subgroups and conference scientific committee after the abstract submission deadline on July 15. Notification of abstract selection for poster or oral presentation will be sent before the end of July.  	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  12. I've completed the registration form, but did not submit an abstract yet. How do I submit my abstract post-registration?: 	  	   				You can send your title and abstract to the SMB Membership Team at membership@smb. org, and they will add it to your registration. 	   	  	  	  13. How do I format my virtual poster?: 	  	   				Posters should be landscape format and saved as a single page pdf, to ensure that everybody will be able to view it on their personal computer or tablet.  When designing your poster, please chose content and font size carefully to clearly communicate your research. All accepted posters will be uploaded to a shared drive for access through Sococo during the poster session. Details will be communicated in the next couple of weeks. 	   	  	  	  	  	  	  	  	  	  	  			"
    }, {
    "id": 16,
    "url": "http://localhost:4000/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		                       SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                                  	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 17,
    "url": "http://localhost:4000/logistics/",
    "title": "Conference Logistics",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Conference Logistics		    			    									    			    		    	    	  			&lt;/br&gt;	&lt;/br&gt;	&lt;/br&gt;				  								 						Sococo Orientation: 	  	  	       	        	  		  						  											August 15 Update: 	  	   																				Happy Sunday #SMB2020 participants!					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins tomorrow! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					Lost on Monday morning? Don’t miss the opening remarks (8am ET) and the Opening Plenary by Dr. Shayn Peirce-Cottler. The session will be linked in all mini-symposia and overflow rooms on all Subgroup floors. In case you have trouble connecting through Sococo, here is the direct Zoom link: https://zoom. us/j/97466727056 for this session. Zoom links are for registered participants only – please do not share or post on social media. 					&lt;/br&gt;&lt;/br&gt;					Official eSMB2020 Zoom background: For anyone interested, feel free to use our eSMB2020 Zoom background. Find it here in both mirrored &amp; unmirrored settings. 					&lt;/br&gt;&lt;/br&gt;					Help – judges needed! We still need a few volunteers to judge posters and contributed talks!&lt;/br&gt;					POPD contributed talks judges: contact Christina. Cobbold@glasgow. ac. uk&lt;/br&gt;					OTHE posters or contributed talks judges: contact torbjorn. lundh@chalmers. se&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					All things eSMB2020:&lt;/br&gt;					Schedule: All last-minute changes will be posted online and in Slack&lt;/br&gt;					SMB Business Meeting(Wed): come learn about our Society and welcome our new President-elect and Board of Directors members&lt;/br&gt;					Career Fair(Thurs): ‘SMB Social/Career Fair Floor’ – there’s still time to upload job announcements and CVs&lt;/br&gt;					Conference t-shirt: Also linked from ‘SMB Welcome/Industry Floor’&lt;/br&gt;					Slack space: New to Slack? Check out our FAQ page&lt;/br&gt;					Twitter: @ESmb2020 #SMB2020&lt;/br&gt;					Sponsors – Industry, Publishers, &amp; Software: ‘SMB Welcome/Industry Floor’&lt;/br&gt;					'Zoom-only’ participants: Zoom links have been sent!&lt;/br&gt;										&lt;/br&gt;&lt;/br&gt;					Tips &amp; Tricks:&lt;/br&gt;					Have your passwords handy for Zoom, Sococo, and Slack&lt;/br&gt;					Initialize yourself in Sococo on the Subgroup Floor that most aligns with your research area&lt;/br&gt;					Issues with Sococo? Try refreshing, then ask #help-desk or use ‘find’ feature in Sococo to locate ‘HELP’&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					All things SMB: &lt;/br&gt;					&lt;/br&gt; 					Society Leadership and Committees&lt;/br&gt;					Membership: ‘SMB Welcome/Industry Floor’&lt;/br&gt;					Subgroups: Not a member? Account -&gt; ‘edit profile’&lt;/br&gt;					Bulletin of Mathematical Biology: Editors available on ‘SMB Welcome/Industry Floor’&lt;/br&gt;					Merchandise: ‘SMB Welcome/Industry Floor’&lt;/br&gt;					SMB Newsletter&lt;/br&gt;					Academic Year Mentoring Program&lt;/br&gt;					SMB Digest Community Forum&lt;/br&gt;					Society Prizes&lt;/br&gt;					Travel and Meeting Grants&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					Looking forward to seeing all 1,732 of you at the best and largest SMB Annual Meeting yet!					Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye					&lt;/br&gt;&lt;/br&gt;					And, all other Subgroup leaders and eSMB2020 track coordinators,					Andreas Buttenschoen, Alex Fletcher, Linus Schumacher, Holly Gaff, Shelby Scott, Carrie Diaz Eaton, Angela Reynolds, Jonathan Forde, Stanca Ciupe, Jessica Conway, Jane Heffernan, John Glasser, Zhilan Feng, Cheng Ly, Wilten Nicola, Erica Rutter, John Nardini, Kevin Flores, Christina Cobbold, James Watmough, Torbjörn Lundh									 	   	  	  				  											August 15 Update: 	  	   															Happy Saturday #SMB2020 participants!					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins in just 2 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					Getting excited for our eSMB2020 scientific program? We know we are! And, we owe a BIG ‘thank you’ to our amazing Subgroup/track Chairs and organizers that put together a state-of-the-art scientific program:					&lt;/br&gt;&lt;/br&gt;											Cell and Developmental Biology (CDEV,@SMBdevBio ): Andreas Buttenschoen, Alex Fletcher, Linus Schumacher						Education (EDUC, @Edu2Smb): Holly Gaff, Shelby Scott, Carrie Diaz Eaton						Immunobiology and Infection (IMMU, @smb_imin): Angela Reynolds, Jonathan Forde, Stanca Ciupe, &amp; Jessica Conway						Mathematical Epidemiology (MEPI, #SMBMathEpi):Jane Heffernan, John Glasser, Zhilan Feng						Mathematical Neuroscience (NEUR, @MathNeuro_): Cheng Ly, Wilten Nicola						Mathematical Oncology (ONCO, @mathonco): Heiko Enderling, Sandy Anderson						Methods for Biological Modeling (MFBM, @SMBmfbm): Erica Rutter, John Nardini, Kevin Flores						Population Dynamics, Ecology &amp; Evolution (POPD, @smbpdee): Christina Cobbold, James Watmough						General/other track (OTHE): Torbjörn Lundh										&lt;/br&gt;&lt;/br&gt;					Jump on each group’s Slack channel in smb2020. slack. com, ‘get’ them in Sococo, attend the Subgroup business meeting, or drop them an email to thank them!					&lt;/br&gt;&lt;/br&gt;					Want to get involved or find out more about our Subgroups? https://www. smb. org/subgroups/					&lt;/br&gt;&lt;/br&gt;					Not a member of a Subgroup? https://www. smb. org/my-account/  ‘edit profile’					&lt;/br&gt;&lt;/br&gt;					eSMB2020 mentoring program: Our incredible mentoring committee, Elissa Swartz, Robert Smith?, and John Nardini, has outdone themselves organizing the largest mentoring program in our history!					&lt;/br&gt;&lt;/br&gt;					Pre-conference Mentoring workshop with 200+ attendees&lt;/br&gt;					eSMB2020 mentoring program with 550+ mentees and 175+ mentors&lt;/br&gt;					Mentoring luncheon on the ‘SMB Mentor Session Floor’ on Monday&lt;/br&gt;					Want to be a SMB mentor or mentee after eSMB2020 ends? Check out our Academic Year Mentoring Program&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					‘Zoom-only’ participants: Zoom links will be sent tomorrow! They are for registered participants only - please do not share! Want to upgrade to use Sococo? Contact us at membership@smb. org					&lt;/br&gt;					Mandatory Sococo orientation (full conference participants only): Please attend one of the remaining 3 orientation sessions (https://zoom. us/j/94700659356)					&lt;/br&gt;					Saturday, August 15th at 9am ET&lt;/br&gt;					Saturday, August 15th at 1pm ET&lt;/br&gt;					Sunday, August 16th at 9am ET&lt;/br&gt;					eSMB2020 Sponsors – Industry, Publishers, &amp; Software: Visit the ‘SMB Welcome/Industry Floor’ to meet with our sponsors and get information about the companies, products, available jobs, and options for publishing your manuscripts. 					&lt;/br&gt;&lt;/br&gt;					eSMB2020 conference t-shirt: Visit the ‘SMB Welcome/Industry Floor’ or get yours today					&lt;/br&gt;&lt;/br&gt;					Slack: Join our Slack channel! New to Slack? Check out our FAQ page					&lt;/br&gt;&lt;/br&gt;					Twitter: Follow us at @ESmb2020 and @SMB_MathBiology use #SMB2020 as the hashtag					&lt;/br&gt;&lt;/br&gt;					SMB membership: Visit the ‘SMB Welcome/Industry Floor’ or join today!					&lt;/br&gt;&lt;/br&gt;					SMB Digest community forum: Visit https://www. smb. org/forums/					&lt;/br&gt;&lt;/br&gt;										Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye																						 	   	  	  					  											August 14 Update: 	  	   																										Happy Friday #SMB2020 participants!						&lt;/br&gt;&lt;/br&gt;												The 2020 SMB Annual Meeting (eSMB2020) begins in just 3 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 						&lt;/br&gt;&lt;/br&gt;						eSMB2020 Sponsors – Industry, Publishers, &amp; Software: We are fortunate to have 9 companies that have graciously sponsored eSMB2020 and funded special poster and talk prizes. Representatives will be present and available on our ‘SMB Welcome/Industry Floor’ with information about their companies, available jobs, and options for publishing your manuscripts. Check out the PDF flyers linked at the bottom and their websites (hyperlinked) as a preview of the amazing things they have to offer. 						&lt;/br&gt;&lt;/br&gt;						Amgen&lt;/br&gt;						Applied BioMath&lt;/br&gt;						Berkeley-Madonna&lt;/br&gt;						DILIsym, a Simulations Plus Co. &lt;/br&gt;						Elsevier&lt;/br&gt;						Mathdept. org&lt;/br&gt;						Merck&lt;/br&gt;						Springer Nature&lt;/br&gt;						Wiley&lt;/br&gt;												&lt;/br&gt;&lt;/br&gt;						‘Zoom-only’ participants: If you selected ‘I only want to attend specific talks, and do not need access to the full conference’ upon registering, you will not receive a Sococo invitation. We will be sending you the Zoom links prior to the start of the conference. Zoom links are for registered participants only and should not be shared with friends and colleagues or on social media.   						Changed your mind and want to upgrade to use Sococo and have access to the full eSMB2020 experience? Contact us at membership@smb. org Poster presenters are highly encouraged to present their poster in Sococo for the most beneficial experience. 						Mandatory Sococo orientation (full conference participants only): All registrants who selected ‘full conference’ upon registering should have received their Sococo invitations! Please do not try to create your own account!						&lt;/br&gt;&lt;/br&gt;						Please attend one of the remaining 5 orientation sessions (https://zoom. us/j/94700659356)						&lt;/br&gt;&lt;/br&gt;						Friday, August 14th at 5pm ET (U. S. Eastern Time)&lt;/br&gt;						Friday, August 14th at 9pm ET&lt;/br&gt;						Saturday, August 15th at 9am ET&lt;/br&gt;						Saturday, August 15th at 1pm ET&lt;/br&gt;						Sunday, August 16th at 9am ET&lt;/br&gt;												&lt;/br&gt;&lt;/br&gt;						eSMB2020 virtual conference t-shirt: Yesterday, we unveiled our eSMB2020 t-shirt. Go get yours today: http://smb2020. org/store or visit our SMB merchandise store on the ‘SMB Welcome/Industry Floor’ in Sococo. We hope to see these …virtually everywhere…						&lt;/br&gt;&lt;/br&gt;						Slack: We’ll only post updates, changes, and be available for help over Slack. Join our Slack channel (smb2020. slack. com) by clicking here: https://bit. ly/31BG64m						&lt;/br&gt;&lt;/br&gt;						Twitter: Follow us at https://twitter. com/ESmb2020 and https://twitter. com/SMB_MathBiology. Use #SMB2020 as the hashtag						&lt;/br&gt;&lt;/br&gt;						SMB membership: Visit the SMB membership team on the ‘SMB Welcome/Industry Floor’ or join today! https://www. smb. org/application/						&lt;/br&gt;&lt;/br&gt;						SMB Digest community forum: Visit https://www. smb. org/forums/						&lt;/br&gt;&lt;/br&gt;										Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye					&lt;/br&gt;&lt;/br&gt;																		AppliedBioMath_CareersFlyer_August2020. pdf &lt;/br&gt;					BerkeleyMadonna_SMB2020. pdf&lt;/br&gt;					Elsevier. pdf&lt;/br&gt;					eSMB_DSS_Flyer. pdf&lt;/br&gt;					Springer_MathBio. pdf&lt;/br&gt;					Wiley_MeetingFlyer. pdf&lt;/br&gt;					WMODatSMB2020. pdf&lt;/br&gt;									 	   	  	  					  											August 12 Update: 	  	   															Happy Wednesday #SMB2020 participants!					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins in just 5 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					eSMB2020 virtual conference t-shirt: …drumroll, please…					&lt;/br&gt;&lt;/br&gt;					The 2020 eSMB2020 t-shirt features the conference logo on the front, and the map of all registered participants on the back. The Society for Mathematical Biology is …virtually everywhere…					&lt;/br&gt;&lt;/br&gt;					The t-shirt is available as ladies trim or unisex in various sizes and colors. Visit our new online store to order your shirt today at http://smb2020. org/store. Members receive a 22% discount (!) so be sure to log into your SMB account first (located at the top of your screen). You can also order your shirt at the SMB merchandise store on the ‘SMB Welcome/Industry Floor’. 					&lt;/br&gt;&lt;/br&gt;											&lt;/br&gt;&lt;/br&gt;					SMB membership information at eSMB: Want to learn more about the many benefits of SMB membership? Visit the SMB membership team on the ‘SMB Welcome/Industry Floor’ during the conference coffee breaks and social gatherings to discuss in person with our awesome Membership Committee Chairs, Marissa Renardy and Sam Erwin, and SMB leadership. You can also join today! https://www. smb. org/application/ 					&lt;/br&gt;&lt;/br&gt;					Mandatory Sococo orientation: All registrants will be receiving their Sococo invitations by email from MathDept. org tomorrow, Thursday, August 13th. The full Sococo experience is only available on a desktop or laptop using Google Chrome or the desktop app (https://app. sococo. com/a/download). Please attend one of the 6 orientation sessions (https://zoom. us/j/94700659356)					&lt;/br&gt;&lt;/br&gt;											Thursday, August 13th at 9pm ET (U. S. Eastern Time)						Friday, August 14th at 5pm ET						Friday, August 14th at 9pm ET						Saturday, August 15th at 9am ET						Saturday, August 15th at 1pm ET						Sunday, August 16th at 9am ET										&lt;/br&gt;&lt;/br&gt;					Twitter: Follow us at https://twitter. com/ESmb2020 and use #SMB2020 as the hashtag					&lt;/br&gt;&lt;/br&gt;					Slack: Join our Slack channel (smb2020. slack. com) by clicking here: https://bit. ly/31BG64m					&lt;/br&gt;&lt;/br&gt;					SMB Digest community forum: Visit https://www. smb. org/forums/					&lt;/br&gt;&lt;/br&gt;					Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye																			 	   	  	  						  											August 11 Update: 	  	   																				Dear friends and colleagues,					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins in just 6 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					Sococo: Through our partnership with MathDept. org, we are creating an immersive conference experience. To get us prepped for the upcoming week:					&lt;/br&gt;&lt;/br&gt;					Orientations: Mathdept. org will host mandatory orientation sessions. All orientations will use the same Zoom link: https://zoom. us/j/94700659356					&lt;/br&gt;&lt;/br&gt;											Thursday, August 13th at 9pm ET (U. S. Eastern Time)						Friday, August 14th at 5pm ET						Friday, August 14th at 9pm ET						Saturday, August 15th at 9am ET						Saturday, August 15th at 1pm ET						Sunday, August 16th at 9am ET										&lt;/br&gt;&lt;/br&gt;					Accounts: These are being made for all participants by MathDept. org. All registrants will be receiving their Sococo email invitations on Thursday, August 13th. 					&lt;/br&gt;&lt;/br&gt; 					Browser/App: Sococo is only supported in the Sococo app or in Google Chrome, so please download one of these ahead of time. 					&lt;/br&gt;&lt;/br&gt;					Zoom: All oral presentations will be streamed as a Zoom video-conference or webinar. Plenaries and Keynotes will be linked to all Sococo ‘floors’, and mini-symposia and contributed talks will be linked in their respective track ‘floors’. For those who selected ‘only Zoom’ in the registration, you will receive the links prior to the conference. 					&lt;/br&gt;&lt;/br&gt;					Schedule: The schedule of all presentations is on our website at http://smb2020. org/schedule. The program is clickable, sharable, and can be filtered by track (fancy, right?!) Can’t find your abstract or have any questions? Please contact your Subgroup or track Chair or the conference Co-Chairs (Amber Smith, amber. smith@uthsc. edu, or Heiko Enderling, heiko. enderling@moffitt. org). 					&lt;/br&gt;&lt;/br&gt;					Twitter: Follow us at https://twitter. com/ESmb2020 and use #SMB2020 as the hashtag. 					&lt;/br&gt;&lt;/br&gt;					Slack: Join our Slack channel (smb2020. slack. com) by clicking here: https://bit. ly/31BG64m!					&lt;/br&gt;&lt;/br&gt;					SMB membership: Join today! https://www. smb. org/application/. 					&lt;/br&gt;&lt;/br&gt;					SMB Digest community forum: Want to receive and post news on job postings, conferences, and other exciting things happening in Mathematical Biology? Visit https://www. smb. org/forums/ and sign up at https://www. smb. org/smb-digest-community-forum-how-to/. 					&lt;/br&gt;&lt;/br&gt;					Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling					&lt;/br&gt;&lt;/br&gt;					eSMB2020 Co-Chairs					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee:					&lt;/br&gt;&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye																   &lt;/div&gt;	  &lt;/div&gt;	  		&lt;/div&gt;	&lt;/div&gt;		  								 						A virtual tour of the Sococo platform: 	  	   				The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 	   	  	  	  	       	    					 					 Your browser does not support HTML video. 						        	  		  	      	    Supported Web Browsers	    						Always use the latest version of Google Chrome. Sococo supports the current version and the previous major release of Google Chrome. Other browsers are not supported. 					 					Network Connection with Adequate Bandwidth 											At a minimum, Sococo requires a download speed of 5 Mbps and an upload speed of 2 Mbps.  					 										Supported Operating Systems 											Sococo supports the current version and the two previous major releases for the following operating systems:													Mac OS X							Windows (administrator right required for the desktop app)							Linux (any modern distribution that is supported by Google Chrome)							Note: The desktop app for Linux is not supported. 											 										Computer Hardware 																			Any dual-core processor								Integrated camera or USB web camera											 	    	        	    	  	  	  	  	  	  	  					&lt;/div&gt; "
    }, {
    "id": 18,
    "url": "http://localhost:4000/monday/",
    "title": "SMB2020 Monday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Monday, August 17		    		    US Eastern Time Zone (EDT) 	    	&lt;/br&gt;		    		    			    				        								&lt;/br&gt;								&lt;/br&gt;															  Overview Sunday Monday Tuesday Wednesday Thursday				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 				                    Opening &amp; Welcome        8:00am:                   		  	                    Opening Plenary        8:30am:         Shayn Peirce-Cottler, &lt;/br&gt;University of Virginia, @PeirceLab             												     		       			       			                                  		   		    Agent-based Modeling of Multi-Cellular Systems for Designing Better Therapies 		     			    Abstract: The most prevalent, devastating, and complex diseases of our time, such as diabetes, cardiovascular disease, and infectious diseases, involve the interactions of heterogeneous cells with one another and with their changing environment. However, the drugs we typically use to treat these diseases target a single protein (e. g. anti-vascular endothelial growth factor (VEGF) for treating diabetic retinopathy) and disregard the fact that cells within tissues are highly heterogeneous and have individualized responses that contribute to the tissue-level outcomes. To bridge the gap between protein and multi-cell/tissue-levels, my lab develops agent-based computational models and uses them in combination with experimental approaches, to predict how individual cell behaviors give rise to tissue-level adaptations. We have used agent-based modeling to simulate the structural adaptations of large and small blood vessels, skeletal muscle regeneration following injury, and immune cell trafficking and differentiation during inflammation and infection. Our studies have suggested new mechanistic hypotheses and provided suggestions for the design of novel therapies that account for the dynamic and heterogeneous interactions between different cell types within a diseased tissue. 		     		    		     			    Bio: Shayn Peirce-Cottler, Ph. D. is Professor of Biomedical Engineering with secondary appointments in the Department of Ophthalmology and Department of Plastic Surgery at the University of Virginia. Dr. Peirce-Cottler received Bachelor’s of Science degrees in Biomedical Engineering and Engineering Mechanics from The Johns Hopkins University in 1997. She earned her Ph. D. in the Department of Biomedical Engineering at the University of Virginia in 2002. Dr. Peirce-Cottler develops and uses computational models in conjunction with experiments to study structural and functional adaptations of tissues, in both health and disease, in order to develop new therapies for inducing regeneration in injured tissues or restoring homeostasis to diseased tissues. Her lab’s core expertise is in combining agent-based computational modeling with in vivo imaging of murine experimental models to examine and control the multi-cell interactions involved in angiogenesis. Current projects in her lab seek to exploit perivascular support cells, immune cells, and stem cells to invoke tissue regeneration and curb fibrosis during acute and chronic inflammation. Dr. Peirce-Cottler has published over 100 peer reviewed papers and book chapters, and she is the inventor on two issued U. S. Patents. She is a fellow in the American Institute for Medical and Biological Engineering College of Fellows and a Fellow of the Biomedical Engineering Society. She is also Past-President of The Microcirculatory Society. At UVA she teaches courses on computational modeling and cell and molecular physiology, and she is the Director of the Graduate Program in Biomedical Engineering. 		     		    	    	             			  										  		    		      		        Sub-group minisymposia (9:30-11:00am)		        			  				  				  				  				  				 		  				  				  				  				  				  																																		 	CDEV: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Andrew Krause								  	 								  		University of Oxford								  									  									  										  			&lt;/br&gt; Matching Theory to Real Biology: Recent Progress and Open Questions in Turing's Theory of Morphogenesis 								  										  									  									  										  										  											  			Turing's reaction-diffusion theory of morphogenesis has been enormously well-studied from a variety of perspectives. While incredibly successful in motivating enormous theoretical and experimental work, there are many open questions in elucidating specific aspects of Turing-type morphogenesis in real developmental settings. I will present recent work on developing new tools and perspectives on matching Turing's idealized theory with the complexity of real biological development. This includes recent extensions to the classical theory of linear instability analysis to account for heterogeneity in space and time, curvature, growth, as well as open boundary conditions combining Turing's theory with other ideas in understanding spatial pattern formation, such as positional information. While this extends and confirms intuitive insights gained from experiments and simulations over the past few decades, it raises an enormous number of questions regarding how far we can push such extensions. I will briefly mention some of these, hopefully stimulating broader perspectives on how to develop simple yet physically interpretable theories of pattern formation. 									  											  									  									  									  								  								  								  	Daniel Lobo								  	 								  		University of Maryland, Baltimore County								  									  									  										  			&lt;/br&gt; A Turing system explains regeneration patterning and fission behavior in planaria 								  										  									  									  										  										  											  			Planarian worms have the extraordinary ability to regenerate any body part after an amputation. This ability allows them to reproduce asexually by fission, cutting themselves to produce two separated pieces each repatterning and regenerating a complete animal. The induction of this process is known to be dependent on the size of the worm as well as on environmental factors such as population density, temperature, and light intensity. Models based on Turing systems can explain the self-regulation of many biological mechanisms, from skin patterns to digit formation. Here, we combine experimental evidence with a modeling approach to show how a cross-inhibited Turing system can explain at once both the signaling mechanism of regeneration and fission in planaria. The model explains in a growing domain the precise signals that control the regeneration of the different body parts after amputations as well as when and where planaria fission, and its dependence on the worm length. We provide molecular implementations of the proposed model, which also explains the effects of environmental factors in the signaling of fission. In summary, the proposed controlled cross-inhibited Turing system represents a completely self-regulated model of the whole-body regeneration and fission signaling in planaria. 									  											  									  									  									  								  								  								  	Jason Ko								  	 								  		University of Maryland, Baltimore County								  									  									  										  			&lt;/br&gt; Regulated Cell Adhesion Dynamics in a Continuous Model: Sorting, Intercalation, and Involution 								  										  									  									  										  										  											  			Cell-cell adhesion can dictate tissue growth and multicellular pattern formation and it is crucial for the cellular dynamics during embryogenesis and cancer progression. While it is known that these adhesive forces are generated by cell adhesion molecules (CAMs), the regulation of CAMs is not well understood due to complex nonlinear interactions that span multiple levels of biological organization–from genetic regulation to whole-organism shape formation. We present a novel continuous model using partial differential equations that can explain the dynamic relationships between genetic regulation, CAM expression, and differential adhesion. This approach can demonstrate the mechanisms responsible for cell-sorting behaviors, cell intercalation in proliferating populations, and the involution of zebrafish germ layer cells during gastrulation. The model can predict the physical parameters controlling the amplitude and wavelength of a cellular intercalation interface as shown in vitro. We demonstrate the crucial role of N-cadherin regulation for the involution and migration of cells beyond the gradient of the morphogen Nodal during zebrafish gastrulation. Integrating the emergent spatial tissue behaviors with the regulation of genes responsible for essential cellular properties such as adhesion will pave the way toward understanding the genetic regulation of large-scale complex patterns and shapes formation in developmental, regenerative, and cancer biology. 									  											  									  									  									  								  								  								  	Timothy Ostler								  	 								  		Cardiff University								  									  									  										  			&lt;/br&gt; Choosing the best embryo in In-Vitro Fertilization 								  										  									  									  										  										  											  			We aim to characterise the shape and size of thawing embryos after they have been cryopreserved during In-Vitro Fertilization (IVF). Through image segmentation techniques and data analysis, we seek to determine appropriate metrics that can predict pregnancy. We also model the increasing temperature of thawing embryos, determining the conditions that will prevent damage. This is work undertaken through an academia-industry grant at Cardiff University in collaboration with the London Women's Clinic. Joint work with: Katerina Kaouri, Thomas Woolley, Karl Swann (Cardiff University), Andrew Thompson, Giles Palmer (London Women’s Clinic), with funding from the KESS2 Scholarship. Knowledge Economy Skills Scholarships (KESS 2) is a pan-Wales higher level skills initiative led by Bangor University on behalf of the HE sector in Wales. It is part funded by the Welsh Government’s European Social Fund (ESF) convergence programme for West Wales and the Valleys. sector in Wales. It is part funded by the Welsh Government’s European Social Fund (ESF) convergence programme for West Wales and the Valleys. 									  											  									  									  								  								  								  									  	Maria Abou Chakra								  	 								  		University of Toronto								  									  									  										  			&lt;/br&gt; Control of tissue development by cell-cycle dependent transcriptional filtering 								  										  									  									  										  										  											  			A fundamental question in biology is how a single eukaryotic cell produces the complexity required to develop into an organism. Cell cycle duration changes dramatically during development, starting out fast to generate cells quickly and slowing down over time as the organism matures. The cell cycle may also act as a transcriptional filter to control the expression of long genes which can’t be completely transcribed in short cycles. Using mathematical simulations, we discovered an inherent trade-off where fast cycling cells serve to increase cell number while slower cycling cells contribute to cell diversity by introducing genes in a controlled manner. Simulations show that cell-cycle duration can fine tune cell number, cell diversity and cell proportions in a tissue. Our predictions are supported by comparison to single-cell RNA-seq data captured over embryonic development. Our results support the idea that cell-cycle dynamics may be important for controlling gene expression and cell fate. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  																																		 	MEPI: Modeling COVID-19 to inform control efforts, Part I (9:30-11:00am)													 						  			  				  							  							  										  									  	Jonathan Read								  	 								  		Lancatser University								  									  									  										  			&lt;/br&gt; Modelling early transmission of Covid-19 within China 								  										  									  									  										  										  											  			In this talk, I will describe rapid modelling work conducted between 20-27 January 2020 to estimate key epidemiological parameters during the early stages of the Covid-19 outbreak in China. Key uncertain- ties at the time were the transmission potential of the new virus as well as the case ascertainment ratio and likely full size of the epidemic. We fitted a deterministic metapopulation SEIR model of transmission to reported case information across Chinese cities as well as in other locations around the world, up until large-scale movement restrictions were imposed on 23 January. We estimated that the R0 in China was 3. 11 (95%CI, 2. 39-4. 13) and the case ascertainment ratio in Wuhan, the center of the outbreak at the time, was only 5. 0% (3. 6-7. 4), demonstrating the potential for sizable epidemics that may be difficult to control. 									  											  									  									  									  								  								  								  	Yanni Xiao								  	 								  		Xi'an Jiaotang Univeresity								  									  									  										  			&lt;/br&gt; Modeling COVID-19 epidemic in mainland China based on multi-source data 								  										  									  									  										  										  											  			Since December 2019, the outbreak of new coronavirus in Wuhan has continued to spread, which has attracted worldwide attention. It is essential to effectively predict the development trend of the epidemic, including when the 2019-nCoV infection will peak, what are the specific peak value and the final size, etc. How do the large-scale directional movement of the national population and random movement of individuals influence on the national epidemic during the Spring Festival, and what is the role of the control strategies on the epidemic? Address these questions fall within the the scope of this talk. We develop a novel modeling approach with multiple control measures and parameterize the model based on a small amount of constantly updated data to quick predict the development trend and transmission risk of the disease, to reveal the development trend of the new coronavirus. We develop spatial network model to study the influences of population movement, information transmission, enhanced prevention and control measures on epidemic transmission, to identify the key factors that significantly affect the spread of disease, and to analyze the effectiveness of prevention and control measures of 2019-nCoV in- fection. The findings provide quantitative decision basis for national epidemic prevention and control. 									  											  									  									  									  								  								  								  	Julien Riou								  	 								  		University of Bern								  									  									  										  			&lt;/br&gt; Early transmission pattern and severity of COVID-19 in China 								  										  									  									  										  										  											  			The coronavirus disease 2019 (COVID-19) epidemic that originated in Wuhan, China, has spread globally. Early in the epidemic, we estimated the basic reproduction number R0 of 2019-nCoV to be around 2. 2 (1. 43. 8), indicating the potential for sustained human-to-human transmission. As more data was becoming available, we estimated the age-specific case fatality ratio (CFR) by fitting a transmis- sion model to data from China, accounting for underreporting and the time delay to death. Overall CFR among all infections was 1. 6% (1. 4-1. 8) and increased considerably for the elderly, highlighting the expected burden for healthcare systems with further expansion of the COVID-19 epidemic around the globe. This presentation aims to highlight methods that can be used to inform public health authorities in real time in situations of disease emergence, with a particular focus on how to handle uncertainty. 									  											  									  									  									  								  								  								  	Zhilan Feng								  	 								  		Purdue University								  									  									  										  			&lt;/br&gt; Staggered Release Policies for COVID-19 Control: Costs and Benefits of Relaxing Restrictions by Age and Risk 								  										  									  									  										  										  											  			Lockdown and social distancing restrictions have been widely used as part of policy efforts aimed at controlling the ongoing COVID-19 pandemic. Since these restrictions have a negative impact on the economy, there exists a strong incentive to relax these policies while protecting public health. Using a multigroup SEIR epidemiological model, we explore the costs and benefits associated with the sequential release of specific groups based on age and risk from isolation. The results suggest that properly designed staggered-release policies can do better than simultaneous-release policies in terms of protecting the most vulnerable members of a population, reducing health risks overall, and increasing economic activity. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				 		  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	MFBM: Simulations and Experiments in Small Scale Bio-Locomotion (9:30-11:00am)													 						  			  				  							  							  										  									  	Orrin		Shindell								  	 								  		Trinity University,	United States,	oshindel@trinity. edu								  									  									  										  			&lt;/br&gt; Rigid Body Dynamics of Motile Bacteria near Surfaces 								  										  									  									  										  										  											  			Bacteria in their natural environment switch between living as free-swimming individual cells and living as members of surface-aggregated communities. To perform this transition, individual cells must contend with the hydrodynamic force interactions between them and the surface. In our work, we determine these interactions by combining experiments and numerical simulations. Using total internal reflection microscopy, we acquire time lapse images of fluorescent bacteria swimming near a surface. By analyzing the intensity profiles, we reconstruct the three-dimensional trajectories of the bacteria. We then input the measured trajectories into a computational fluid dynamics model – the method of images for regularized Stokeslets – and calculate the force and torque exerted on the bacteria when they swim near the surface. In this talk, we will present the technical details of the experiment, show the resulting measurements, and discuss the computational approach. 									  											  									  									  									  								  								  								  	Meuriq		Galagher								  	 								  		University of Birmingham, United Kingdom, m. t. gallagher@bham. ac. uk								  									  									  										  			&lt;/br&gt; FAST: Automated Flagellar Capture as a Research and Clinical Tool 								  										  									  									  										  										  											  			In an age where huge amounts of imaging data can be readily produced it is increasingly important to be able to accurately and efficiently this information, and to be able to use these analyses as a marker for clinical outcome. However, semen analysis in the human is currently limited to methods such as sperm counting and analysis of fixed cells. To address this, we have developed and released FAST, a free-to-use package for the high-throughput detection and tracking of large numbers of beating flagella in experimental microscopy videos. In this talk we will discuss how the combination of experimental data analysis, integrated with mathematical and numerical modelling of the elastohydrodynamic environment, can be utilised to understand the characteristics of flagellar motility in a semen sample. We will focus on what this means in the context of using clinical data from Birmingham Women’s Hospital, and international partners, in order to improve outcomes from assisted reproductive technologies. 									  											  									  									  									  								  								  								  	Suzanne		Jacobs								  	 								  		UT Austin, United States,	sjacobs@chaos. utexas. edu									  									  									  										  			&lt;/br&gt; To Swarm or to Slide? Understanding the Mechanics of Bacterial Colony Expansion 								  										  									  									  										  										  											  			Whether in our guts or on our skin, on our medical devices or along the roots of our crops, microbes have always been an invisible part of human life. Now, thanks to advances in genomics technology, we can identify which microbes are where and understand how they influence human and environmental health. But to ever effectively manipulate these microbial ecosystems to our benefit, we must understand not only their composition, but also the physics of microbial surface colonization. In this talk, I will describe how colonies of the common soil bacteria Bacillus subtilis swarm and slide along surfaces and what we know about the forces that individual bacteria experience during these processes. Understanding such forces will be crucial to identifying the physical triggers that promote or inhibit various modes of surface colonization. 									  											  									  									  									  								  								  								  	Bruce		Rodenborn								  	 								  		Centre College, United States,	bruce. rodenborn@centre. edu									  									  									  										  			&lt;/br&gt; Life at Low Reynolds Number in a Macroscopic Lab 								  										  									  									  										  										  											  			The swimming of microorganisms is typically analyzed using biological experiments or numerical simulations because of the difficulty of making microscopic measurements of forces and torques. Our research group uses model macroscopic experiments with typical length scales of ≈ 10 cm, but match the low Reynolds number of microoganisms by using a highly viscous silicone oil that is 105 more viscous than water but with approximately the same density. We can build laboratory scale robotic swimmers and model microorganisms that are typically ≈ 10 µm, while keeping the Reynolds much less than unity. We also explore fundamental theories such as building a laboratory scale three-link swimmer (Purcell 1977). We compare our laboratory experiments of helical flagella with complimentary numerical simulations and find good agreement. We also compare our results with theory from geometric mechanics, which predicts the translation and rotation of a three-link swimmer for a given gate. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  																																		 	ONCO: Frontiers in MathOnco, Part 1 (9:30-11:00am)													 						  			  				  							  							  										  									  	Morgan	L. 	Craig								  	 								  		Université de Montréal, Canada								  									  									  										  			&lt;/br&gt; Leveraging patient-specific heterogeneity to establish effective immunotherapeutic protocols 								  										  									  									  										  										  											  			Cancer is an intrinsically heterogeneous disease distinguished by disparate outcomes based on cancer types, patient-specific characteristics, and treatment modalities. Further complicating this picture, therapeutic resistance poses a major challenge to the design and implementation of effective cancer treatments. To overcome these hurdles, it is crucial to characterize heterogeneity within and around the tumour and to quantify the effects that neighbouring tumour and immune cells have on therapeutic success. In contrast to generalized and cytotoxic chemotherapies, immunotherapies aim to harness an individual’s immune system to elicit a targeted immune response and hopefully provide durable therapeutic benefits. Unfortunately, recent disappointing trial results for a variety of immunotherapies stress the need for a more tailored approach to immunotherapeutic scheduling that takes into account patient-specific heterogeneity and the potential for developing resistance. In response, quantitative approaches provide a way to test therapeutic protocols before they are used in patients, ultimately reducing bottlenecks along the drug development pipeline, rationalizing therapeutic scheduling, and improving patient outcomes. Here I will discuss two recent projects focused on establishing effective therapeutic protocols for mono- and poly-oncolytic virus treatments. Using mathematical and computational modelling, we constructed models that recapitulated realistic patient cohorts to study combined vaccinia and vesicular stomatitis oncolytic viruses, and to understand the impact of the tumour microenvironment in glioblastoma multiforme (a deadly central nervous system tumour) on oncolytic virotherapy penetration and efficacy. In both cases, we showed that therapeutic success was principally determined by tailoring treatment to underlying patient characteristics, including tumour aggressivity and spatial structure. Our results highlight the relevance of quantitative approaches to pre-clinical development and therapeutic design, and underline the impact of inter- and intra-individual variability on treatment outcomes. 									  											  									  									  									  								  								  								  	David Basanta								  	 								  		Moffitt Cancer Center, Tampa, USA								  									  									  										  			&lt;/br&gt; Innocent bystander? The role of stromal cells in cancer evolution and treatment resistance 								  										  									  									  										  										  											  			Much work on cancer’s evolutionary dynamics is focused on the genetic mutations that characterize the different stages of cancer progression or the competition between different clones as the tumor grows and copes with different treatment options and schedules. Those evolutionary dynamics are shaped by the tumor ecosystem that, in the context of skeletal cancers, include cells such as osteoclasts, osteoblasts, macrophages and mesenchymal stem cells. Those cells perform a variety of roles in a normal bone and also have an impact on cancer. In this talk I will describe mathematical models that allow us to understand the role of those cells in the normal bone, which is a key step to uncover how they can be co-opted by a tumor and what role do they play as the tumor grows and undergoes treatment. 									  											  									  									  									  								  								  								  	Kaitlyn Johnson								  	 								  		University of Texas at Austin, USA								  									  									  										  			&lt;/br&gt; Towards an integrated framework for incorporating multimodal data sets into mechanistic models of treatment response dynamics in cancer 								  										  									  									  										  										  											  			In the field of mathematical oncology, we commonly look to longitudinal data to calibrate and validate models of tumor progression. Longitudinal data allow for precise model fitting and parameter estimation which can be used to predict tumor behavior. However, molecular level data, although often available at few snapshots in time, is what biologists and clinicians typically use to better understand underlying disease biology in both experimental and clinical settings. While the quantitative nature of these snapshot data sets has vastly improved with technologies such as single cell RNA sequencing (scRNAseq), there exists a need for integrating snapshot and longitudinal data into mathematical frameworks in order to develop the most informed models to describe and predict cancer progression. In this work, we integrate longitudinal drug-response data with snapshot scRNAseq data at just three times points, thus calibrating model outputs to experimental data for two distinct modes of data. We demonstrate that direct incorporation of high-resolution scRNAseq snapshot data into the parameter estimation improves the identifiability of the mathematical model and its predictive power. We present this work as an example of how mathematical oncology can develop novel workflows for incorporating the available biological data to better understand cancer treatment response. 									  											  									  									  									  								  								  								  	Khaphetsi	Joseph	Mahasa								  	 								  		Lesotho								  									  									  										  			&lt;/br&gt; Mesenchymal stem cells used as carrier cells of oncolytic adenovirus results in enhanced oncolytic virotherapy 								  										  									  									  										  										  											  			Mesenchymal stem cells (MSCs) loaded with oncolytic viruses are presently being investigated as a new modality of advanced/metastatic tumors treatment and enhancement of virotherapy. MSCs can, however, either promote or suppress tumor growth. To address the critical question of how MSCs loaded with oncolytic viruses affect virotherapy outcomes and tumor growth patterns in a tumor microenvironment, we developed and analyzed an integrated mathematical-experimental model. We used the model to describe both the growth dynamics in our experiments of firefly luciferase-expressing Hep3B tumor xenografts and the effects of the immune response during the MSCs-based virotherapy. We further employed it to explore the conceptual clinical feasibility, particularly, in evaluating the relative significance of potential immune promotive/suppressive mechanisms induced by MSCs loaded with oncolytic viruses. We were able to delineate conditions which may significantly contribute to the success or failure of MSC-based virotherapy as well as generate new hypotheses. In fact, one of the most impactful outcomes shown by this investigation, not inferred from the experiments alone, was the initially counter-intuitive fact that using tumor-promoting MSCs as carriers is not only helpful but necessary in achieving tumor control. Considering the fact that it is still currently a controversial debate whether MSCs exert a pro- or anti-tumor action, mathematical models such as this one help to quantitatively predict the consequences of using MSCs for delivering virotherapeutic agents in vivo. Taken together, our results show that MSC-mediated systemic delivery of oncolytic viruses is a promising strategy for achieving synergistic anti-tumor efficacy with improved safety profiles. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	POPD: Stochastic Population Dynamics (9:30-11:00am)													 						  			  				  							  							  										  									  	Fima C. Klebaner								  	 								  		Monash University								  									  									  										  			&lt;/br&gt; Appearance of random initial conditions in the Wright-Fisher model 								  										  									  									  										  										  											  			We consider the Wright-Fisher model with a positive selection parameter and a small di↵usion co- e cient. When the starting point is near zero, and the time goes to infinity appropriately as the noise goes to zero, we show that the approximating solution follows the logistic equation with a random initial condition. This condition has the form H(W), where H(x) is the scaled limit of the deterministic flow, and W is the long time limit of the approximating Feller di↵usion. This is joint work with J. Baker and K. Hamza (Monash University) and P. Chigansky (Hebrew University). More general results appeared in the paper “Persistance of small noise and random initial conditions” Adv. Appl. Probab. 2018, 50(A). 									  											  									  									  									  								  								  								  	Sophie Hautphenne								  	 								  		Univeristy of Melbourne								  									  									  										  			&lt;/br&gt; Inference in population-size-dependent branching processes 								  										  									  									  										  										  											  			Population-size-dependent branching processes (PSDBPs) are models which describe the evolution of populations where individuals in the same generation give birth independently according to a probability distribution which depends on the current population size. One important class of PSDBPs are branching processes with a carrying capacity; these are appropriate for modelling populations that exhibit logistic growth, where the population size tends to fluctuate, for a long period of time, around a threshold value corresponding to the maximum number of individuals that an ecosystem can support. We propose an estimator for the mean of the o↵spring distribution at each population size in a discrete- time PSDBP, based on the observation of the total population sizes up to some generation. Our main challenge is the fact that branching processes with a carrying capacity eventually become extinct with probability one (after a long time). We propose a way to derive asymptotic properties of the estimator in this setting. This leads to a number of questions about desired properties of estimators in branching processes that almost surely become extinct. (Joint work with Peter Braunsteins and Carmen Minuesa Abril. )									  											  									  									  									  								  								  								  	Carmen Minuesa Abril								  	 								  		Autonomous University of Madrid								  									  									  										  			&lt;/br&gt; Estimation of the carrying capacity in a population-size-dependent branching process 								  										  									  									  										  										  											  			In this talk we deal with the estimation of the parameters of branching processes with a carrying capacity, which is the maximum population size of the species that the environment can sustain given the resources available in the environment. In these processes, the population size grows logistically and lingers for a long period of time around the carrying capacity before eventually becoming extinct. We consider discrete-time branching processes with a carrying capacity and assume that the offspring distributions belong to some parametric family with unknown parameter. Based on the observation of the population sizes, we propose several estimators for the target parameters, and in particular, for the carrying capacity. Finally, we illustrate the properties of these estimators via some examples. (Joint work with Peter Braunsteins and Sophie Hautphenne. )									  											  									  									  									  								  								  								  	Göran Högnäs								  	 								  		Åbo Akademi University								  									  									  										  			&lt;/br&gt; Exit times of some nonlinear autoregressive processes 								  										  									  									  										  										  											  			We know from Klebaner and Nerman that certain size-dependent branching processes are well approximated by autoregressive processes, at least in the short run. Our aim is to estimate the exit time of some nonlinear autoregressive processes and hereby gain insight into the problem of determining the expected life time of the corresponding size-dependent branching process. Let {Xn}, n = 0, 1, 2, . . . , be a stochastic process defined by the recursion formula Xn+1 = f(Xn)+ e C n+1, X0 = 0 in Rd where f is a mapping from Rd to itself and the C’s are a sequence of i. i. d. normal random variables. e is a small nonnegative parameter. We assume that f(0) = 0 and that f is in some sense contracting so that the resulting Markov chain is positive recurrent. In the one-dimensional case we take f to be, for example, a piecewise linear function or a piecewise polynomial. The aim is to investigate, asymptotically as e goes to 0, the expected time until the process exits from the interval [-1,1]. In the case of linear autoregressive processes the asymptotics of the expected exit times was treated in Jung. (Joint work with Brita Jung. )									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  			             		    		  									                15 minute break (11:00-11:15am)                								  		    		      		        Sub-group minisymposia (11:15am)		        			  				  				  				  				  				 		  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	CDEV: Modeling dynamics in single-cell biology (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Laleh Haghverdi								  	 								  		Max Delbruck Center for Molecular Medicine, Berlin								  									  									  										  			&lt;/br&gt; Cell lineage trajectories and pseudotime reconstruction from single-cell transcriptomics 								  										  									  									  										  										  											  			The reconstruction of cell lineage trajectories from single-cell transcriptomics allows us to resolve temporal expression dynamics of several genes from snapshot collected data. The temporal order of gene activities in return, can provide new insights into the gene regulatory networks governing cell differentiation. We adapted a dimension-reduction technique called “diffusion maps” for the analysis and reconstruction of cell differentiation trajectories. We perceived cell differentiation as a diffusion-like process, where cells are gradually and to some extent stochastically changing their gene expression profiles as they proceed to more differentiated molecular states. Thus, cell differentiation dynamics can (in discrete form) be described by: p(t) = p(t − 1) * T.  That is, the probability distribution at the position of each sampled cell at time t, p(t) is given by the probability distributions at time (t − 1) multiplied by the cells’ pairwise transition probabilities matrix T. Thus, p(t) and p(t − 1) are vectors of length N (number of cells) and T is an N by N matrix which accommodates: 1) transition probabilities which are purely based on geometrical distances between the cells, 2) directional transition probabilities towards more differentiated cell states and 3) source/sink probabilities for accounting for cell’s prolifereation/death rates. Considering purely geometrical transition probabilities, Tgeom is a row-normalised positive-definite matrix which characterises the manifold on which cell differentiation is taking place in the high-dimensional gene expression space. The complete process (i. e. , including directional and source/sink diffusion terms) thus, describes different dynamics taking place on that same manifold. Because of this resemblance to a diffusion process, we introduced an adaptation of diffusion maps for data embedding and differentiation trajectories reconstruction from single-cell data. In a simple form, data embedding by diffusion maps is given by the first eigenvectors of Tgeom corresponding to it largest eigenvalues. We extended the diffusion map’s framework to define “pseudotime” quantifying the differentiation stage of each cell based on on-manifold diffusion distances from the root (pluripotent) cell state. Fur- thermore, we showed that differentiation branching events can be identified by application of the triangle inequality to the on-manifold distances. The diffusion metrics with new approaches for inclusion of directional and source/sink probabilities (infered from biologically known differentiation terminal points, spliced/unspliced mRNA comparisons, time-lapse measurements for a few genes) has been used in several trajectory reconstruction methods on exceedingly complex manifolds which I will briefly discuss in my presentation. 									  											  									  									  									  								  								  								  	Adam MacLean								  	 								  		University of Southern California								  									  									  										  			&lt;/br&gt; Inferring the structure and parameters of cell lineage models from single-cell data 								  										  									  									  										  										  											  			Since single-cell RNA sequencing technologies have become widespread, great efforts have been made to develop appropriate statistical methods to learn biological features from high dimensional data. Lag- ging behind are efforts to combine dynamical systems models with single-cell data. The existence of transient or intermediate cell states – corresponding to shallow attractor states on a potential landscape – further confounds the development of simple models. Here we present new approaches to address these challenges. We propose methods to infer both model structure and model parameters of cell lineage models using single-cell RNA-sequencing data. We fit model structure by assuming a core set of cell population dynamic processes, i. e. a general system described by ẋ(t) = (αf (x) − δ)x(t), where x(t) is a vector of cell states, α and δ are cell state-specific proliferation and death rates, and f (x) is a feedback function. We use methods for trajectory and cell-cell communication inference to infer lineage relationships and feedback interactions. Taking advantage of the information present in the pseudotemporal ordering of cells, we perform Bayesian parameter inference: that is we fit the model to “pseudo-dynamics” to constrain the parameters. With application to hematopoiesis, we demonstrate the ability to select between models and predict future cell differentiation dynamics. We will discuss how this model inference framework can also be applied to gene regulatory network dynamics, enabling not only the simulation of single-cell data-derived models, but also the characterization of their stable steady states and bifurcations. 									  											  									  									  									  								  								  								  	Caleb Weinreb								  	 								  		Harvard University								  									  									  										  			&lt;/br&gt; Limits on dynamic inference from single-cell RNA-seq and the case for additional measurements 								  										  									  									  										  										  											  			For decades, biologists have dreamed of building dynamical systems models of gene expression that could be used to understand and predict cell state. High-throughput single-cell analysis has brought the dream closer. It is now possible to sample cell states in high-dimension and to map trajectories through continuous gene expression space. Using prior knowledge or measurements of RNA maturity, the general direction of cell progression along these trajectories can be inferred. Yet it remains unclear to what extent single-cell RNA-seq encodes a unique dynamical system over cell state. Here, I will discuss two key obstacles to inferring cell state dynamics from single-cell RNA seq data, and how additional types single-cell measurements might overcome them. First, applying the principle of population reveals the importance of measuring cell proliferation and death rates for accurately inferring cell dynamics. Second, clonal tracing using DNA barcoding reveals that RNA-seq alone cannot distinguish cells with different cell-autonomous fate biases, emphasizing the need to measure other variables such as chromatin state. 									  											  									  									  									  								  								  								  	Heyrim Cho								  	 								  		University of California at Riverside								  									  									  										  			&lt;/br&gt; Modeling cell state dynamics of Hematopoiesis from single-cell gene sequencing data 								  										  									  									  										  										  											  			Recent advances in single-cell gene sequencing data and high-dimensional data analysis techniques are bringing new opportunities in modeling biological systems. In this talk, I will discuss different approaches to develop mathematical models from high-dimensional single-cell data. In particular, single-cell RNA sequencing data is challenging due to its high-dimensionality, and dimension reduction techniques are essential in finding the trajectories of cell states in the reduced differentiation space. We develop models using differential equations that describe the cell differentiation as directed and random movement on the abstracted graph or on the multi-dimensional reduced space. Normal hematopoiesis differentiation and abnormal processes of acute myeloid leukemia (AML) progression are simulated. We show that the model can predict the emergence of cells in novel intermediate states of differentiation consistent with immunophenotypic characterizations of AML, and compare the pros and cons of the models on the graph and on the multi-dimensional reduced space. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				 		  																																		 	MEPI: COVID-19 Contributed Talks I (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Alun Lloyd								  	 								  		North Carolina State University								  									  									  										  			&lt;/br&gt; Bidirectional contact tracing dramatically improves COVID-19 control 								  										  									  									  										  										  											  			Contact tracing is critical to controlling COVID-19, but most protocols only “forward-trace” to notify people who were recently exposed. Using a stochastic branching-process model, we show that “bidirectional” tracing to identify infector individuals and their other infectees robustly improves outbreak control, reducing the effective reproduction number (Reff) by at least ~0. 3 while dramatically increasing resilience to low case ascertainment and test sensitivity. Adding smartphone-based exposure notification can further reduce Reff by 0. 25, but only if nearly all smartphones can detect exposure events. Our results suggest that with or without digital approaches, implementing bidirectional tracing will enable health agencies to control COVID-19 more effectively without requiring high-cost interventions. 									  											  									  									  									  								  								  								  	Grzegorz Rempala								  	 								  		Ohio State University								  									  									  										  			&lt;/br&gt; Dynamical Survival Analysis for COVID-19 Predictions in Ohio 								  										  									  									  										  										  											  			Over the last several weeks many mathematicians, statisticians and data scientists have found themselves involved with various efforts in response to the public health crisis caused by the COVID-19 pandemic. Did predictive modeling really help with COVID preparedness and decision making? Following up on my earlier lectures on the topic over the summer, I will try to give a perspective of how various mathematical methods turned out to work (or not) in practical settings of the daily predictions of the pandemic size in Ohio. In particular, I will briefly outline some new ideas and possible improvements in the methodology of 'dynamic survival analysis' developed by the OSU COVID response team to help predict COVID hospital burden. 									  											  									  									  									  								  								  								  	Adeshina Adekunle								  	 								  		James Cook University								  									  									  										  			&lt;/br&gt; Delaying the COVID-19 epidemic in Australia: Evaluating the effectiveness of international travel bans 								  										  									  									  										  										  											  			Following the outbreak of novel Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2), and the disease named COVID-19, in Wuhan, China in late 2019, countries have implemented different interventions such as travel bans to slow the spread of this novel virus. This brief report evaluates the effect of travel bans imposed to prevent COVID-19 importation in the Australian context. We developed a stochastic meta-population model to capture the global dynamics and spread of COVID-19. By adjusting our model to capture the travel bans imposed globally and in Australia, the predicted COVID-19 cases imported to Australia were evaluated in comparison to observed imported cases. Our modelling results closely aligned with observed cases in Australia and elsewhere. We observed a 79% reduction in COVID-19 importation and a delay of the COVID-19 outbreak in Australia by approximately one month. Further projection of COVID-19 to May 2020 showed spread patterns depending on the basic reproduction number. Imposing the travel ban was effective in delaying widespread transmission of COVID-19. However, strengthening of the domestic control measures is needed to prevent Australia from becoming another epicentre. This report has shown the importance of border closure to pandemic control. 									  											  									  									  									  								  								  								  	David JD Earn								  	 								  		McMaster University								  									  									  										  			&lt;/br&gt; 1918 vs 2020: Influenza vs COVID-19 								  										  									  									  										  										  											  			Comparisons are constantly being made between the 1918 influenza pandemic and the present COVID-19 pandemic.  I will discuss our previous work on influenza pandemics, and and the tools we have used to understand the temporal patterns of those outbreaks.  Applying similar tools to the COVID-19 pandemic is easier in some respects and harder in others.  I will describe our current approach to modelling the spread of COVID-19, and some of the challenges and limitations of epidemic forecasting. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				 		  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	MFBM: Simulations and Experiments in Small Scale Bio-Locomotion (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Madeleine		Hall								  	 								  		Imperial College London,	United Kingdom,	madeleine. hall17@imperial. ac. uk								  									  									  										  			&lt;/br&gt; Optimal Turning Gaits for Undulators 								  										  									  									  										  										  											  			An organism’s ability to efficiently traverse and search their surroundings can be important to its survival. This has inspired the study of optimal gaits and locomotion strategies, in particular for the case of undulatory movement of slender bodies. The primary focus has been on finding optimal waveforms for moving forwards along straight paths. However, the ability to turn and manoeuvre is also relevant to survival. We revisit this problem in the context of low Reynolds number hydrodynamics and obtain the optimal waveforms for undulators along curved trajectories. For shallow turning angles, we obtain small perturbations of a travelling wave as optimal. For larger turning angles, however, the optimal gait can be radically different, with the undulator abruptly curling and uncurling itself. We believe that these results can lend insight into the search behaviours of simple organisms, such as C. elegans, as well as be a tool for phenotyping their behaviour across mutant strains and under different environmental conditions. 									  											  									  									  									  								  								  								  	David		Smith								  	 								  		University of Birmingham,	United Kingdom, 	D. J. Smith@bham. ac. uk								  									  									  										  			&lt;/br&gt; Algorithmic Developments in the Method of Regularized Stokeslets for the Study of Micro-Locomotion 								  										  									  									  										  										  											  			Microscopic swimming and propulsion due to cilia and flagella are characterised by complex shaped moving boundaries, and for Newtonian fluids, the linear Stokes flow equations. These features of the problem have motivated the development of methods based on singularity distributions (Stokeslets/Oseen tensors). In recent years the methods of regularized Stokeslets has become widely used in biological fluid dynamics; this technique avoids the need for generation of a connected surface mesh, removes the need to compute singular integrals, and therefore is very convenient and accessible. This talk focuses on some algorithmic refinements to the method of regularized Stokeslets aimed at improving the efficiency of the method while retaining the convenience and accessibility of the method. We discuss (1) the use of meshless interpolation, in particular the nearest-neighbour discretisation, (2) exploitation of graphical processing unit computing to handle relatively large problems through built-in linear algebra routines in Matlab, (3) the NEAREST software package in Matlab, and benchmarking against a compiled Fortran code. The focus will be on providing practical and reliable tools for the quantitative biology of microscopic motility. 									  											  									  									  									  								  								  								  	Hoa		Nguyen								  	 								  		Trinity University,	United States, hnguyen5@trinity. edu									  									  									  										  			&lt;/br&gt; Effects of Cell Morphology, Attachment to a Surface, and Colony Formation on the Hydrodynamic Performance of Choanoflagellates 								  										  									  									  										  										  											  			Choanoflagellates, eukaryotes that are important predators on bacteria in aquatic ecosystems, share a common ancestor with sponges and are used as a model system to study the evolution of animals from protozoan ancestors. The choanoflagellate Salpingoeca rosetta, which has a complex life cycle that includes unicellular and multicellular stages, provides a model system to study within one species the functional consequences of: 1) different cell morphologies (swimming cell with a collar of prey- capturing microvilli surrounding a single flagellum; dispersal-stage cell with a slender body, long flagellum, and short collar), 2) being free-swimming vs. sessile (thecate cell attached to a surface), and 3) being a single cell vs. a multicellular colony. For model organisms of different life stages, we use the method of regularized Stokeslets to examine the effects of different morphological features on swimming and feeding performance. We compare the model predictions with lab experiments that used high-speed microvideography to measure swimming and feeding currents produced by different life stages. We find that a longer flagellum increases swimming speed, longer microvilli reduce speed, and cell shape only affects speed when the collar is very short. The flux of prey-carrying water into the collar capture zone is greater for swimming than sessile cells, but this advantage decreases with collar size. Stalk length has little effect on flux for sessile cells. 									  											  									  									  									  								  								  								  	Genevieve		O' Brien								  	 								  		Johns Hopkins University,		United States,	gsteinobrien@jhmi. edu								  									  									  										  			&lt;/br&gt; Decomposing cell identity for transfer learning across cellular measurements, platforms, tissues, and species 								  										  									  									  										  										  											  			Single cell RNA sequencing (scRNAseq) has lead to the discovery of new cell types. Thus, it is important to have methods to determine the identifying and consequential features of cells. To this end, we developed scCoGAPS to learn robust feature dimensions from scRNAseq and projectR to investigate these, or other continuous valued, feature dimensions in biologically related data across technologies, omics, and species. We demonstrate how this implementation of transfer learning via dimension reduction represent a platform for in silico experimentation and hypothesis generation where knowledge from multiple data sets is leveraged for selection of meaningful feature dimensions for validation. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				 		  																																		 	ONCO: Frontiers in MathOnco, Part 2 (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Mohit Kumar Jolly								  	 								  		Indian Institute of Science, India								  									  									  										  			&lt;/br&gt; Integrating mechanism-based and data-based approaches to identify hybrid epithelial/mesenchymal phenotypes 								  										  									  									  										  										  											  			Epithelial–mesenchymal transition (EMT) is a key driver of metastasis and therapy resistance. During EMT, cells lose their epithelial traits and acquire mesenchymal ones to varying degrees. Recent evidence has suggested that cells need not necessarily display 'pure' epithelial or mesenchymal states, but can stably acquire one or more hybrid epithelial/mesenchymal (E/M) ones. In silico, in vitro and in vivo analysis indicates that these hybrid E/M states can be more aggressive and perhaps the 'fittest' for metastasis, thus identifying mechanisms enabling these hybrid E/M states is crucial for decoding tumor aggressiveness. Using an integrative approach involving mechanism-based models to identify master regulators of such hybrid E/M phenotypes, data-based models to decipher transcriptomics signatures specific to the hybrid E/M phenotypes, and experiments to test our predictions, we have identified how these hybrid E/M cell states may be maintained via cell-autonomous and non-cell autonomous mechanisms. Our work also highlights that the hybrid E/M specific signatures associate with worse clinicopathological traits, thus offering a mechanistic basis for its aggressiveness and pinpointing novel putative targets. 									  											  									  									  									  								  								  								  	Pamela Jackson								  	 								  		Mayo Clinic, Phoenix, USA								  									  									  										  			&lt;/br&gt; Parameterizing a Brain Tumor Growth Model Using Noisy Simulated MRIs 								  										  									  									  										  										  											  			Mathematical models of brain tumor growth are commonly parameterized using volumes of abnormal regions segmented from magnetic resonance imaging (MRI). For imaging-level brain tumor growth modeling, such as the Proliferation-Invasion (PI) model and its more complex variants, the abnormality region requires an assumption for the tumor cell density. This assumption is necessary to make estimates regarding the shape of the leading edge, however the use of an assumption can be limiting. Searching across MRI images that have been simulated across the model parameter space could eliminate the need for a cell density assumption, however noisy MRIs will reduce the reliability of such a technique. Our objective is to determine if matching the noise between simulations and MRIs can increase our ability to predict model parameters based on an image's characteristics. We generated phantom brain tumors using the Proliferation-Invasion-Hypoxia-Necrosis-Angiogenesis- Edema (PIHNA-E) model. One hundred unique phantoms were created assuming 10 different rates of migration (D [mm2/year]) and 10 different rates of proliferation ( [1/year])]. PIHNA-E simulations were then passed into a MRI signal model for generating simulated T2-weighted MRIs. We created two independent runs of six sets of 100 simulated MRIs, with each set incorporating a different level noise. The first run represented simulated images with known D and or 'ground truth', while the second run represented candidate 'real-world' noisy images for which we would have not normally known D and . We then compared each noise level from the second 'real-world' run to each noise level from the first 'ground truth' run. Within each set comparison, we calculated the L2-norm of twelve statistical features for each image in the second 'real-world' set relative to the 'ground truth' set of images. The D and of the 'ground truth' image with the lowest L2-norm relative to the 'real-world' image were selected as the predicted parameters. For each noise level, a prediction rate is calculated by assessing the percentage rate that the L2-norm-based selection was correct. As expected, the prediction rates decreased overall as the image noise level increased. For 'real- world' images with limited noise, creating a similar noise level in the 'ground truth' images did enable higher prediction rates of D and . Future directions of this work include repeating these methods across additional replicates and exploring processing steps for reducing MRI noise, which could increase our prediction rate of D. 									  											  									  									  									  								  								  								  	Jeffrey West								  	 								  		Moffitt Cancer Center, Tampa, USA								  									  									  										  			&lt;/br&gt; Anti-fragile Cancer Therapy 								  										  									  									  										  										  											  			Herein we present a novel paradigm of evolutionary cancer therapy based on the 'anti-fragility' of the drug dose-response function. Anti-fragility is a situation where the curvature of the dose-response function is convex, mathematically defined as a positive second derivative. This positive curvature is associated with benefits from increased variance or unevenness of a treatment schedule. For example, if the curvature is positive near a dose of ‘x’, continuous administration of x may have a less efficacious response compared to a regimen that switches equally between 120% of x and 80% of x, even though both regimens use the same total drug. Although dose-response data is readily available, such second-order effects are typically ignored. Nonlinear sigmoidal dose-response curves are ubiquitous in medicine and have both convex and concave regions. Selection pressure due to treatment selects for resistant phenotypes over time. In response, the magnitude of dose response curvature and the  convex-concave inflection point both decrease in value. The key insight is that dose-response convexity ('anti-fragility') decreases in proportion to the amount of resistance in the tumor population. This provides a time-dependent metric which 1) predicts the emergence of resistance and 2) determines the optimal subsequent dosing strategy.  We demonstrate this paradigm using a Hill function model parameterized by in vitro experimental data of H3122 non-small cell lung cancer cell lines confronted to 10 different drugs. Through this dataset, we present antifragility applied to 1) treatment resistance, 2) collateral sensitivity, and 3) combination therapy. 									  											  									  									  									  								  								  								  	Anna Marciniak-Czochra								  	 								  		University of Heidelberg, Germany								  									  									  										  			&lt;/br&gt; Stem cell niche dynamics in acute leukemia: Insights from mathematical modeling 								  										  									  									  										  										  											  			Acute myeloid leukemia (AML) is one of the most aggressive blood cancers. The cancer originates from a small population of so called leukemia stem cells (LSC) that survive treatment and trigger re- lapse. During the course of the disease leukemic cells accumulate in the bone marrow and impair healthy blood cell formation. The impact of this interaction on the clinical course of the disease remains not well understood. We develop and validate a mathematical model of stem cell competition in the human hematopoietic stem cell niche. Model simulations predict how processes in the stem cell niche affect the speed of disease progression. Combining the mathematical model with data of individual patients, we quantify the selective pressure LSC exert on HSC and demonstrate model's prognostic significance. We develop a novel model-based risk-stratification approach. This approach allows extracting prognostic information from counts of healthy and malignant cells at the time of diagnosis. We demonstrate its feasibility based on a cohort of ALDH-rare AML patients and show that the model-based risk strati- fication is an independent predictor of disease free and overall survival. This proof of concept study shows how model-based interpretation of patient data can improve prognostic scoring and contribute to personalization of treatment. The talk is based on joint works with Thomas Stiehl (Institute of Applied Mathematics, Heidelberg University), Wenwen Wang and Christoph Lutz (Heidelberg Medical Clinic). 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	POPD: (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Swarnendu Banerjee								  	 								  		Indian Statistical Institute, Kolkata								  									  									  										  			&lt;/br&gt; Chemical contamination mediated regime shifts in planktonic systems 								  										  									  									  										  										  											  			Increasing chemical contamination is a growing concern worldwide. Although, regime shifts leading to algal blooms are quite well known in aquatic ecosystems, the effect of contamination on such regime shift is not particularly well understood. Motivated by this, we studied the effect of copper enrichment on planktonic system using a minimal phytoplankton-zooplankton model. Interestingly, our results suggest that both the toxic and deficient concentration of copper in water bodies can lead to catastrophic transition of the ecosystem to an alternative stable state. Further, on adding stochasticity to the system the region of bistability is diminished and the system switches from zooplankton dominated state to the phytoplankton dominated state much prior to the tipping point. The bistability is further weakened on increasing noise intensity and redness. However, in case of systems with high nutrient enrichment, the bimodality in the probability density can be very prominent. Nevertheless, generic early warning signals may fails to predict an impending state shift due to contamination. Our study provides important perspective to regime shifts in the context of eco-toxicology. 									  											  									  									  									  								  								  								  	Jody Reimer								  	 								  		University of Utah								  									  									  										  			&lt;/br&gt; Long transient dynamics in the presence of noise 								  										  									  									  										  										  											  			Recent theoretical work has highlighted several mechanisms giving rise to so-called ``long transient'' dynamics. These long transients tantalizingly appear to replicate dynamics seen in real systems-with one critical difference: ecological data is noisy, a reality theoretical work often ignores. In general, stochasticity is known to have important consequences: it can qualitatively alter model dynamics as well as impact our ability to infer underlying processes through statistical analysis. To explore the effect of stochasticity on qualitative model behavior and the implications for our ability to infer underlying mechanisms, we generated time series from a simple model of long transient behavior with additive noise. We then examined if noise qualitatively changes the expected dynamics of the system and how well phenomenological and mechanistic statistical models could recover the underlying model. We found that long transients such as those generated by even the simplest stochastic model of a ghost attractor are highly sensitive to noise, and that the mean behavior of the stochastic model differs substantially from that of the deterministic model. In spite of this, we illustrate that statistical inference on a single realization may still provide insight into model parameters, and highlight that inference improves for an increasing number of realizations of the process. All approaches saw improved results with increasing data realizations. We suggest methods to increase our ability to draw inference from real ecological time series with suspected long transient dynamics. 									  											  									  									  									  								  								  								  	Carlos A.  Braumann								  	 								  		Universidade de Evora								  									  									  										  			&lt;/br&gt; General autonomous fishing models with Allee effects in a randomly varying environment 								  										  									  									  										  										  											  			In a randomly varying environment, a general fishing model is the stochastic differential equation dX(t) = f(X(t))X(t)dt+σX(t)dW(t)−qE(t,X(t))X(t)dt, where X(t) is the fished population size at time t, f (of class C1) is the per capita arithmetic average natural growth rate, σdW(t)/dt describes the effect of environmental fluctuations on the growth rate (with W(t) a standard Wiener process and σ &gt; 0), E(t,X(t)) is the harvesting effort applied and q &gt; 0 is the catchability. Here, we will consider autonomous models for which E(t, X(t)) ≡ E(X(t)) non-negative of class C1. The usual density-dependence case with f strictly decreasing and f(+∞) &lt; 0 was studied in [1] w. r. t. conditions for population extinction or for existence of a stationary density. In [5, 6], for the particular cases of f being logistic or Gompertz, profit optimization was studied comparing variable effort E(t, X(t)) fishing policies with constant effort E(t, X(t)) ≡ E policies. Sometimes, however, the population under fishing is affected by Allee effects, with an unexpected depression (accompanied by growth) of f(x) for small values of x, due, for instance, to the difficulty of finding mating partners or of putting together an effective collective defence from predators. [4] made a comparative study between variable and constant effort policies for the particular case of f being logistic-like with Allee effects. In [2, 3], for populations not subjected to fishing and for general growth models f with Allee effects, conditions for extinction or for existence of a stationary density were studied. We are now generalizing this study, under appopriate conditions, to fished populations with general growth models f(·) with Allee effects and with also general autonomous harvesting efforts E(·). Again, we found out that the deciding factor between extinction or existence of a stationary density is the sign, when the population size is small, of the net (i. e. , discounting the mortality rate due to fishing) geometric average per capita growth rate. The cases previously studied, as well as the gear saturation phenomenon, can be treated as particular cases. 									  											  									  									  									  								  								  								  	Toni Klauschies								  	 								  		Potsdam University								  									  									  										  			&lt;/br&gt; Ecological and evolutionary causes of intermittent predator-prey cycles 								  										  									  									  										  										  											  			The presence of trait variation in prey or predator populations may affect the stability and the shape, i. e. amplitude and phase, of predator-prey dynamics. However, while previous studies have shown how trait variation can alter the overall amplitude of the predator-prey oscillations, this altered amplitude remained constant over time. This strongly contrasts with empirically observed predator-prey dynamics and recent theoretical work, showing that several mechanisms may lead to so called intermittent predator- prey cycles where the amplitude of the predator-prey dynamics varies temporally. For instance, trait differences that determine the functional responses of two predators may provoke temporal fluctuations in the amplitudes of the predator-prey cycles due to recurrent changes in the relative abundance of the two predator types: a predator with a relatively linear functional response promotes small-amplitude oscillations whereas a predator with a more strongly non-linear functional response stimulates larger amplitudes. We analysed various models that incorporate trait variation within prey, predators, or both, and identified three general conditions that are necessary for intermittent cycles to occur. First, the predator-prey system comprises at least two subsystems that exhibit substantial differences in the amplitude of their population dynamics and thus tendency to promote stable or unstable population dynamics. Second, these subsystems recurrently alternate in their dominance, leading to a second “trait” cycle superimposed on the population dynamics. Finally, the time scale of the trait dynamics must be significantly slower than that of the population dynamics. For instance, co-evolution may promote intermittent cycles in predator-prey dynamics by inducing a lag in the predators’ trait adjustment in response to altered trait values of the prey. The resulting temporal variation in the interaction strength between predator and prey is associated with temporal changes in the amplitudes of the population dynamics since e. g. , a dominance of defended prey dampens the population dynamics whereas a high abundance of undefended prey enhances it. Our results highlight that intermittent cycles may frequently occur in simple predator-prey systems allowing for trait variation. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  			             		    		  											  		    		      		        Mentoring Luncheon		             		    		  											  		    		      		        Sub-group contributed talks (1:30-2:30pm)		        			  				  				  				  				  																																		 	CDEV: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Nathan Weinstein								  	 								  		UNAM								  									  									  										  			&lt;/br&gt; A computational model of the differentiation of endothelial cells into macrophages 								  										  									  									  										  										  											  			Endothelial cells (ECs), macrophages (MPs), and pericytes are the main cellular components of microvascular networks. The principal process that allows microvascular network remodeling and adaptation is angiogenesis. Angiogenesis begins when hypoxia or lack of nutrients triggers the release of a vascular endothelial growth factor (VEGF), which is typically VEGFA. The VEGF forms a gradient, and when the concentration of VEGF reaches a threshold around a segment of the blood vessel, it causes the segment to destabilize and to lose its mural cells. During the subsequent stage of sprouting angiogenesis, some of the ECs become tip cells that extend filipodia and migrate following the VEGF gradient. The ECs adjacent to the tip cells become stalk cells that elongate, proliferate, and trail the tip cell. Once the tip cell reaches another tip cell or blood vessel segment, an anastomosis forms establishing a new connection in the microvascular network. During anastomosis formation macrophages act as chaperones for the tip cell. Additionally, macrophages phagocytize the ECs that undergo apoptosis during microvascular pruning. Definitive hematopoiesis occurs first during early embryogenesis when a group of endothelial cells undergo the endothelial-to-hematopoietic transition (EHT) and become hematopoietic stem cells (HSC). Some of the HSCs become granulocyte monocyte precursor cells (GMPs). Certain GMPs differentiate into monocytes, and macrophages differentiate from monocytes.  In adult humans, ECs retain the capability to undergo EHT and we aim to study the potential of ECs to differentiate into macrophages and to elucidate the extracellular microenvironmental conditions that promote EC-to-macrophage differentiation. Our approach is to first assemble a molecular regulatory network based on the information available in the literature about the main molecules involved in the regulation of the process and the interactions between them. Then we will transform the model into a dynamic system in the form of a Boolean Network. Then, we simulate and analyze the dynamic behavior of the model. Afterward, we will validate the model by comparing the observed effect of the available relevant mutations with their simulated effect. 									  											  									  									  									  								  								  								  	Aden Forrow								  	 								  		University of Oxford								  									  									  										  			&lt;/br&gt; Learning developmental trajectories from CRISPR lineage tracing and single-cell gene expression 								  										  									  									  										  										  											  			Recent research has shown that the mathematical theory of optimal transport can effectively reconstruct developmental trajectories from time courses of single cell gene expression; however, this approach requires expensive experiments with fine time resolution. In this work, we present a novel framework that leverages new types of lineage-tracing information measured simultaneously with gene expression. These experimental techniques use heritable CRISPR-induced genetic barcodes to trace the history of cell divisions over the course of development. Crucially, these barcodes can be measured in the same cells as gene expression. Our method, designed for lineage-tracing time courses, learns from both kinds of information together using mathematical tools from graphical models, structural equation models, and optimal transport. We find that lineage data improves optimal transport’s effectiveness in disentangling complex state transitions with lower temporal resolution, thereby reducing experimental cost. Joint work with Geoffrey Schiebinger (University of British Columbia)									  											  									  									  									  								  								  								  	Claus Kadelka								  	 								  		Iowa State								  									  									  										  			&lt;/br&gt; Unraveling the Design Principles of Gene Regulatory Networks: a Meta-Analysis 								  										  									  									  										  										  											  			Gene regulatory networks (GRNs), frequently modeled using Boolean networks, describe how a collection of genes governs the processes within a cell. Boolean networks are intuitive, simple to describe, and yield qualitative results even when data is limited. This talk will outline a research program aimed at harnessing the collective knowledge contained in hundreds of diverse Boolean GRN models in order to understand the impact of network structure and topology on network dynamics and stability, and in particular the role of canalization in gene regulation. The biological term canalization reflects a cell’s ability to maintain a stable phenotype despite ongoing environmental perturbations. Accordingly, Boolean canalizing functions are functions where the output is already determined if a certain, canalizing variable takes on its canalizing input, regardless of all other inputs. Using text- and data-mining techniques and the PubMed search engine, we generated an expandable database of published, expert-curated Boolean GRN models, and extracted more than 5, 000 rules governing these networks. A meta-analysis of all these rules confirmed a strong overrepresentation of certain types of canalizing functions. We also studied the abundance of small structures within the networks, called network motifs, and found significant differences between published GRNs but also when comparing to random networks. Furthermore, we analyzed the dynamical robustness of GRNs and found most of them to operate at a “critical threshold” between order and chaos. These findings highlight how our continuously-expanding database provides a versatile tool to identify the overarching design principles underlying gene regulation. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	MEPI: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Vijay Pal Bajiya								  	 								  		Rajasthan University								  									  									  										  			&lt;/br&gt; The Impacts of Awareness and Medical Resources on Multi-group SIR Epidemic Model 								  										  									  									  										  										  											  			When an infectious disease occurs and spreads in a community, then the government department of disease control tries every conceivable afford to avoid and prevent disease transmission. The treatment of infective individuals is a signficant method to reduce infection risk and control the spreads of infectious diseases. In this work, we propose an SIR multi-group epidemic model with the awareness of susceptible population and saturated treatment function, which describes the effects of the availability of medical resources for treatment. We assume that treatment of the infected individuals of a group is affected by the medical resources for the treatment of each group. The basic reproduction number R0 is computed. The local and global stability of equilibrium has been determined in the term of R0 and availability of medical resources for treatment. The occurrence of backward bifurcation have been discussed at R0 = 1 and the existence of multiple endemic equilibria when R0 &lt; 1: The global stability of unique endemic equilibrium has been showed with help of some graph-theoretic techniques when R0 &gt; 1: Finally, numerical simulations of the system are also presented in various cases to support and counterpart the obtained theoretical results. We discussed the impacts of the awareness of the susceptible population and the availability of medical resources for treatment in every group on the epidemic size of each group. 									  											  									  									  									  								  								  								  	Evan Mitchell								  	 								  		Western University								  									  									  										  			&lt;/br&gt; The Effect of Host Prophylactic Behaviour on Pathogen Evolution 								  										  									  									  										  										  											  			Much work has considered the evolution of pathogens, but little is known about how they respond to changes in host behaviour. We build a model of sublethal disease effects where hosts are able to choose to engage in prophylactic measures that reduce the likelihood of disease transmission (e. g. , hand washing, social distancing). This choice is mediated by utility costs and benefits associated with prophylaxis, and the fraction of hosts engaged in prophylaxis is also affected by population dynamics. When prophylactic host behaviour occurs, we find that the level of pathogen host exploitation is reduced, by the action of selection, relative to the level that would otherwise be predicted in the absence of prophylaxis. Our work emphasizes the significance of the transmission-recovery trade-off faced by the pathogen and the ability of the pathogen to influence host prophylactic behaviour. 									  											  									  									  									  								  								  								  	Jeremy D'Silva								  	 								  		University of Michigan / Stanford University								  									  									  										  			&lt;/br&gt; Identifiability of linear compartmental models of infectious disease transmission 								  										  									  									  										  										  											  			Successful parameter estimation for mathematical models generally requires that the estimated parameters are unique. The property of unique estimates for a given output trajectory is called structural identifiability: a system is globally (resp. locally) structurally identifiable if the map from parameters to output trajectories is injective (resp. has finite fibers). Identifiability is especially important for infectious diesease models, since the parameter estimates are used for generating predictions and comparing outbreak intervention strategies. In this paper, we apply differential algebra techniques to study the structural identifiability of multistage infectious disease models (generalizations of the $SEIR$ model with multiple $I$ stages). We prove that these models are not structurally identifiable from incidence data, and we determine the rational functions of parameters that can be estimated from incidence data, called identifiable combinations, for the case where each $I$ stage has equal duration. We conjecture the identifiable combinations of the general model; we provide computational evidence for the conjecture and verify it for models with 2 infectious stages. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				 		  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	MFBM: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Jinsu Kim								  	 								  		University of California, Irvine								  									  									  										  			&lt;/br&gt; Stochastic epigenome systems under inflammatory signals and its application to study vivo nucleosome accessibility 								  										  									  									  										  										  											  			In cellular immune responses, inflammatory ligands activate signal-dependent transcription factors (SDTFs), which can display complex temporal profiles. SDTFs are central effectors for inflammatory gene expression, and hence they serve a critical role in immune system. The information contained in SDTF signals must also be decoded by the epigenome to allow controlled plasticity in cellular epigenetic states in response to environmental encounters. The mechanisms and biophysical principles that generate distinct epigenomes in response to different SDTF signaling remain unclear. Here, we develop and analyze stochastic models of nucleosome accessibility to study how SDTF signals alter the epigenome dynamics. Interestingly the response of our epigenome model to SDTF signals helps us to predict the system parameters of genome-scale nucleosome in vivo. On the genome-scale, the relation between the SDTF binding location and nucleosome accessibility plays a role of a parameter predictor since the epigenome dynamics depends on SDTF binding sites differently under various parameter regimes. We could compare our numerical results to experimental measurements to test our prediction. Our work proposes a systematic framework that allows a predictive understanding in vivo nucleosome accessibility. 									  											  									  									  									  								  								  								  	Alan Rendall								  	 								  		Johannes Gutenberg Universitat Mainz								  									  									  										  			&lt;/br&gt; Autophosphorylation as a source of multistability.  								  										  									  									  										  										  											  			Src kinases, which have many functions in cell biology, are known to undergo autophosphorylation in trans. This means that the phosphorylation of a site on the kinase is catalysed by another molecule of the same enzyme. This type of phosphorylation can lead to new phenomena. The usual model of the phosphorylation of a protein on a single site is known to admit only one steady state. In the work reported on here it is proved that in the case of a site which is subject to autophosphorylation there can be more than one stable steady state. Thus a substance of this type can serve as a switch. Src kinases such as Lck, which is involved in many processes in lymphocytes, have two tyrosine phosphorylation sites but a form of Lck in which one of these sites (Y394) has been knocked out has been studied experimentally. It represents a known substance to which the model we have studied is applicable. This work is intended as a step towards a better understanding of the regulation of the activation of Lck, and thus of the function of lymphocytes. This type of insight has the potential to improve cancer immunotherapy. 									  											  									  									  									  								  								  								  	Thomas Hillen								  	 								  		University of Alberta								  									  									  										  			&lt;/br&gt; Non-local models for cellular adhesion 								  										  									  									  										  										  											  			Cellular adhesion is one of the most important interaction forces between cells and other tissue components. In 2006, Armstrong, Painter and Sherratt introduced a non-local PDE model for cellular adhesion, which was able to describe known experimental results on cell sorting and cancer growth. Since then, this model has been the focus of applications and analysis. The analysis becomes challenging through non-local cell-cell interaction and interactions with boundaries. In this talk I will present theoretical results of the adhesion model, such as a random walk derivation, biologically realistic boundary conditions, pattern formation and results on local and global existence of solutions. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	ONCO: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Adam Malik								  	 								  		Chalmers University of Technology, University of Gothenburg								  									  									  										  			&lt;/br&gt; Modelling glioblastoma growth and invasion using Diffusion Tensor Imaging-data 								  										  									  									  										  										  											  			Many human cancers are studied using xenograft mouse models, in which human cancer cells are transplanted into mice. This constitutes a versatile tool, where various imaging modalities can reveal the dynamics of tumour progression, and novel drug targets can be investigated. However, this research occurs at expense of animal su ering. With the advent of high-performance computing and increasing knowledge of cellular and physiological processes the opportunity arises to replace in vivo mouse models with computational models that describe the growth of the tumour and its interactions with the host animal. We have developed a computational model of glioblastoma, a malignant form of brain tumour, which utilises data from Di usion Tensor Imaging (DTI) to represent anatomical structures within the model. The model takes into account a DTI-map obtained from adult normal mouse brains. The corresponding di usion tensor encodes information about nerve bre direction and density, and is therefore assumed to in uence cancer cell migration. In contrast to previous DTI-models we consider a cell-based model, where each cancer cell resides within a voxel (of volume 40 x 40 x 40 m3), which, due to volumetric constraints can contain a maximum of K cells. If the local density is below K each cell divides with at a rate α placing the daughter cell in the same voxel. Cell migration occurs with a rate Davg, which is the average value of the diffusion tensor, into one of the 6 neighbouring voxels, where the probability of moving to each neighbouring cell depends on the di usion tensor. A parameter, which ranges from 0 to 1, controls the impact of the di usion tensor, such that q = 0 corresponds to random migration and cells ignoring the anatomical structure, whereas q = 1 results in cells always moving along the direction of largest di usion (i. e. along bres). Simulations are initiated with 10,000 cells at the site of injection and contain approximately 1 million cells upon termination. Our results show that the model can recapitulate both nodal and di use tumours seen in the mouse model depending on the model parameters. Currently we are estimating model parameters from xenograft tumours obtained from di erent patients with the aim of identifying patient-speci c di erences, which, in the future, could inform personalised treatments. 									  											  									  									  									  								  								  								  	Emma Carrick Smith								  	 								  		Wolfson Centre for Mathematical Biology, Mathematical Institute, University of Oxford								  									  									  										  			&lt;/br&gt; Modelling the Clonal Evolution of Drug-Resistant Cells in Non-Small Cell Lung Carcinoma 								  										  									  									  										  										  											  			High levels of clonal heterogeneity in non-small cell lung carcinoma (NSCLC) and the selection of drug-resistant cells during treatment drive the evolution to unresponsive tumours to tyrosine kinase inhibitors (TKIs). Cell-autonomous effects such as genetic instability and non-cell-autonomous effects such as cancer therapies and their ability to induce resistance and the local cellular composition of the surrounding microenvironment are key elements promoting high levels of clonal heterogeneity in these evolving ecosystems. To understand the effects such features have on the prevalence of resistance at diagnosis and to predict the impact two-drug treatment protocols have on the level of resistance within the tumour during treatment, we have developed a stochastic Lotka-Volterra model of tumour growth incorporating four cell populations with varying levels of resistance to two drugs. The model incorporates both spontaneous and drug-induced resistance via mutations and employs population-dependent birth rates derived from a recent game-theory study to account for the effects of cellular interactions. The effect of the fitness of drug-resistant cells relative to drug-sensitive cells is also considered by incorporating both advantageous and deleterious mutation cases. We calculated the proportion of resistance in a tumour at diagnosis. The proportion of resistance is an increasing function of the mutation probability and cell turnover ratio with significantly larger proportions when mutations are advantageous as opposed to deleterious. At diagnosis, we introduced four treatment protocols applying either two TKIs with inhibitory effects or two cytotoxic drugs and observed the proportion of resistance in the tumour once it had reached a clinically relevant size. Our model demonstrates the relative fitness of drug-resistant cells to drug-sensitive cells, the ability for the drugs to induce resistance and sequential application of two TKIs results in a notable difference in the level of resistance in the tumour during treatment as opposed to when the drugs are applied simultaneously. These estimates of the size of the drug-resistant tumour cell population prior to and during two-drug treatment protocols can inform treatment strategies when combining two TKIs. 									  											  									  									  									  								  								  								  	Arturo Araujo								  	 								  		University of Roehampton								  									  									  										  			&lt;/br&gt; Multi-Scale Model of Colorectal Cancer Initiation 								  										  									  									  										  										  											  			Colorectal cancer (CRC) is currently the fourth leading cause of cancer-related deaths in the world. Development of more effective treatments is hindered by gaps in our understanding of CRC initiation and evolution, as well as the limitations of in vitro and in vivo experimental techniques. Specifically, in CRC we don’t fully understand the role of initiation mutations on the subsequent evolution of the disease, making it unique to each patient and therefore limiting the efficacy of general treatments. To bridge this gap in our knowledge, we use computational techniques such as agent-based modelling, gene regulatory network and continuous mathematical techniques to integrate molecular and cellular scale information to explain tumour growth and evolution. One of the biggest challenges we face is that collecting biological data is not enough. We need to figure out which are the right methods to analyse and utilise the data. Specially in cancer research, there is an abundance of data in many scales, from molecular and cellular to epidemiological; but without the right tools, such as multi-scale models, most of it remains under-utilised. To tackle CRC initiation, where colon epithelium loses its homeostasis, it was important to first have a clear understanding of the normal case. We constructed a cell-based model of a healthy colon crypt, incorporating different biological data and choosing the appropriate modelling abstraction for each one. We decided that agent-based modelling would provide us with the emergent property of homeostasis, common throughout biology. Through the interaction of the elements, cells in this case, a global balance arises which was not programmed into the equation and which was then validated by biological data. We extended this model with gene-regulatory network techniques to create an in silico experimental environment in which the effects of oncogenic mutations can be investigated and analysed with unique granularity. We further incorporated molecular data within the agent-based model to suggest novel therapies that consider not only the tumour, but the complex cross-talk between cancer and the rest of the healthy colon. We are currently working on incorporating data from the impact of current treatments, with the goal of tailoring it to individual patients to help control their specific disease and prolong their life. We envision for multi-scale models in the computational sciences to enable a greater understanding of the dynamics and evolution of diseases such as CRC, helping us explore, understand and harness the complex biological landscape, as well as supporting the development of new clinical prevention and treatment interventions. Further, these techniques can be readily deployed in hospitals, and some of them have already, as an aid for the clinician to help make better decisions, minimise costs and maximise the patients’ quality of life. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	OTHE: Other Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Gemma_Massonis								  	 								  		CSIS								  									  									  										  			&lt;/br&gt; Structural Identifiability and Observability of Compartmental Models of the COVID-19 Pandemic 								  										  									  									  										  										  											  			The recent coronavirus disease (COVID-19) outbreak has dramatically increased the public awareness and appreciation of the utility of dynamic models. At the same time, the dissemination of contradictory model predictions has highlighted their limitations.  If some parameters and/or state variables of a model cannot be determined from out-put measurements, its ability to yield correct insights – as well as the possibility of controlling the system – maybe compromised.  Epidemic dynamics are commonly analysed using compartmental models, and many variations of such models have been used for analysing and predicting the evolution of the COVID-19 pandemic.  In this paper we survey the different models proposed in the literature, assembling a list of 36 model structures and assessing their ability to provide reliable information. We address the problem using the control theoretic concepts of structural identifiability and observability. Since some parameters can vary during the course of an epidemic, we consider boththe constant and time-varying parameter assumptions.  We analyse the structural identifiability and observability ofall of the models, considering all plausible choices of outputs and time-varying parameters, which leads us to analyse 255 different model versions. We classify the models according to their structural identifiability and observability under the different assumptions and discuss the implications of the results. We also illustrate with an example several alternative ways of remedying the lack of observability of a model. Our analyses provide guidelines for choosing the most informative model for each purpose, taking into account the available knowledge and measurements. 									  											  									  									  									  								  								  								  	Cole Zmurchok								  	 								  		Vanderbilt								  									  									  										  			&lt;/br&gt; Mechanosensing can enhance adaptation to maintain polarity of migrating cells 								  										  									  									  										  										  											  			Migratory cells are known to adapt to environments that contain wide-ranging levels of chemoattractant. While biochemical models of adaptation have been previously proposed, here we discuss a different mechanism based on mechanosensing, where the interaction between biochemical signaling and cell tension facilitates adaptation. In this talk, we develop and analyze a model of mechanochemical-based adaptation consisting of a mechanics-based physical model coupled with the wave-pinning reaction-diffusion model for Rac GTPase activity. We use Local Perturbation Analysis to predict how cells adapt signaling parameters via feedback from mechanics to maintain polarity in response to chemoattractant levels. To confirm this prediction, we simulate the mechanochemical model in moving cells, demonstrating how mechanosensing results in persistent cell polarity when cells are stimulated with wide-ranging levels of chemoattractant in silico. These results demonstrate how mechanosensing may help cells adapt to maintain polarity in variable environments. 									  											  									  									  									  								  								  								  	Thomas Fai								  	 								  		Brandeis University								  									  									  										  			&lt;/br&gt; Length regulation of multiple flagella that self-assemble from a shared pool of components 								  										  									  									  										  										  											  			The single cell biflagellate Chlamydomonas reinhardtii has proven to be a very useful model organism for studies of size control. The lengths of its two flagella are tightly regulated. We study a model of flagellar length control whose key assumption is that proteins responsible for the intraflagellar transport (IFT) of tubulin are present in limiting amounts. In the case of two simultaneously assembling flagella, regardless of the details of how the flagella are coupled, we find that the widely-used assumption of a constant disassembly rate is inconsistent with experimental results. We therefore propose a model in which diffusion gives rise to a length-dependent concentration of depolymerizer at the flagellar tip. This model is found to be consistent with experimental results and generalizes to other situations such as arbitrary flagellar number. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	POPD: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Meike Wittmann								  	 								  		Bielefeld University								  									  									  										  			&lt;/br&gt; A new maximum-likelihood method to infer factors influencing establishment success of introduced species 								  										  									  									  										  										  											  			One of the most important goals in conservation biology and in the biology of introduced and invasive species is to understand why some small populations persist while others go extinct. Several factors play a potentially important role: 1) demographic and environmental stochasticity, 2) Allee effects, i. e. a reduction in the per-capita growth rate in small populations, for example due to mate-finding difficulties, and 3) propagule size, i. e. the initial number of individuals. It is known that both Allee effects and environmental stochasticity affect the relationship between propagule size and persistence probability in specific ways. Here I propose a new approach for the joint inference of the contributions of these two factors. The approach is based on a Markov chain model for population size with environmental stochasticity and Allee effects. The models without Allee effects or without environmental stochasticity are special cases of the general model. Given a data set for the persistence or extinction of populations of various sizes, the model parameters are estimated using a maximum-likelihood approach and then model choice is performed based on Akaike's information criterion. Using simulation studies, I explore the strengths and weaknesses of this approach. Finally, I apply the approach to published data sets on experimental introductions in the field or laboratory. 									  											  									  									  									  								  								  								  	Candy Abboud								  	 								  		University of Glasgow								  									  									  										  			&lt;/br&gt; Dating, localizing and predicting invasive-pathogen dynamics 								  										  									  									  										  										  											  			Prediction of invasive-pathogen dynamics is an essential step towards the assessment of eradication and containment strategies. Such predictions are performed using surveillance data and models grounded on partial differential equations (PDE), which form a framework often exploited to design invasion models. The framework allows the construction of phenomenological but concise models relying on mechanistic hypotheses. However, this may lead to models with overly rigid behaviour, in particular for describing phenomena in population biology. Hence, to avoid drawing a prediction relying on a single PDE-based model that would be prone to errors because of potential data-model mismatch, we propose to apply Bayesian model-averaging (BMA) for handling parameter and model uncertainties. Hence, we combine several competing spatio-temporal models of propagation for inferring parameters and drawing a consensual prediction of certain quantities of interest. This study is applied (i) to date and localize the invasion of Xylella fastidiosa, bacterium detected in Southern Corsica in 2015, France using post-introduction data, and (ii) to predict its future extent. 									  											  									  									  									  								  								  								  	Bo Zhang								  	 								  		Oklahoma State University								  									  									  										  			&lt;/br&gt; Species competition in heterogeneous environments with directed movement 								  										  									  									  										  										  											  			Understanding the mechanisms that promote species coexistence is a central topic in ecology.  Predicting coexistence in heterogeneous environments where populations are linked by dispersal is a challenge that has attracted attention of ecologists. A particular body of theory, based on Lotka-Volterra-like equations, has focused on the effects of different relative dispersal rates in the absence of other differences in competing species, and has predicted that the slower disperser always outcompetes the faster one in environments where the limiting resources are heterogeneously distributed. However, this theory has never been rigorously tested empirically, and has generally only considered random diffusion. Here, we extended previous theory to include exploitable resources and an additional component of directed movement, proving qualitatively novel results, which we tested experimentally using laboratory populations of C. elegans. We revealed, both theoretically and emperically, that stable coexistence can occur when two competing species have identical directed components but different diffusive components to their movement. Our results advance understanding of coexistence theory and has important ecological implications, such as the essential of individuals obtaining clues of neighboring environments, to determine where to disperse in changing environments. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  			             		    		  											  		    		      		        Poster Session (2:30-3:30pm)		        		        	Click to view posters for each subgroup:  	 All	 CDEV	 EDUC	 IMMU	 MEPI	 MFBM	 NEUR	 POPD	 ONCO	 OTHE			        		             		    		  								  		    		      		        Sub-group Keynote		        3:30pm: 		        Priyanga Amarasekare, &lt;/br&gt;University of California Los Angeles 		        Population Dynamics, Ecology, &amp; Evolution Subgroup 		      		      																     				       					       					     		             		    		    		    				   				    Effects of climate warming on consumer-resource interactions 				     					    There is increasing evidence that climate warming is impacting biodiversity by disrupting species interactions.  Trophic (consumer-resource) interactions, which comprise the fundamental units (modules) of food webs, are of particular importance because they have an intrinsic tendency to fluctuate in abundance, thus running to risk of stochastic extinction during periods of low abundances. Here I present a mathematical framework for predicting warming effects on consumer-resource interactions.  This work differs from previous theory in two important ways. First, it uses delay differential equations to realistically depict the developmental delays inherent in ectotherm life cycles, and incorporates mechanistic descriptions of phenotypic trait responses, derived from first principles of thermodynamics, into the dynamical delay model. Second, it investigates the recent IPCC predictions on the increase in the number of hotter-than-average days. I report three key results. First, across latitudes (tropical vs. temperate) and feeding strategies (juvenile vs. adult attacked), a greater increase in the maximum temperature compared to the minimum (hotter-than-average summers) is more detrimental to consumer-resource interactions than a greater increase in the minimum temperature (warmer-than-average winters).  Second, across latitude and warming scenarios, effects of warming are more detrimental when the consumer attacks the adult stage of the resource.  Third, across warming scenarios and feeding strategies, consumer-resource interactions in the tropics are more at risk of species losses due to warming while those in the temperate zone are more at risk of extreme fluctuations in species' abundances. I discuss implications of these results for biodiversity and biological pest control. 				     			    			   		    		    		  									                Happy hour with friends and colleagues (4:30)                						&lt;/br&gt;			&lt;/br&gt;									 				  Overview Sunday Monday Tuesday Wednesday Thursday			 																				      	      		      														Click to show abstracts											      	            "
    }, {
    "id": 19,
    "url": "http://localhost:4000/registration/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    August 17 - 20, 2020		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;   	  					Registration &amp; Abstract Submission: 	  	    		    To register for the SMB Conference, click below to be redirected to the registration form.  	  	    															    				    								 								  Register for SMB 2020								  $25 fee for registration. 								  &lt;/br&gt;										Register. . . 									 																	    						&lt;/br&gt;	    	  			"
    }, {
    "id": 20,
    "url": "http://localhost:4000/schedule/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    										Conference Schedule															Pre-conference Mentoring Program: 					The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone (EDT). If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 							    		    			    															 							  Pre-conference Mentoring Program							 			    			    		    		    						&lt;/br&gt;		    &lt;/br&gt;						Conference Overview: 						* all times are listed in US Eastern Time Zone (EDT). This schedule is tentative and subject to change. Click to view more details. 						        	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;		    			    &lt;/br&gt;		    &lt;/br&gt;		    		    			    															  Overview Sunday Monday Tuesday Wednesday Thursday								The conference schedule is posted by day, above. Poster abstracts will be posted soon. 				    			    		    		    		    		    Industry Sponsors: 														Thinking about a job in industry, or looking to get your work published? Don't forget to visit our industry sponsors in the SMB Welcome/Industry space! See the flyers below for their schedules. 						 														DILIsym (click here)&lt;/br&gt;								Merck (click here)&lt;/br&gt;								Elsevier (click here)&lt;/br&gt;								Wiley (click here)&lt;/br&gt;								Springer (click here)&lt;/br&gt;								In addition, Applied BioMath will be staffing their room 1pm-3pm Mon-Wed. &lt;/br&gt;													 		    		    	    &lt;/div&gt;	  &lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 21,
    "url": "http://localhost:4000/store/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    August 17 - 20, 2020		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;   	  					Society for Mathematical Biology - Store: 	  	    		    To purchase SMB merchandise, click below to be redirected to the SMB store.  	  	    															    				    								 								  SMB 2020 								  For sale: SMB 2020 conference shirt, and more. 								  								  								  &lt;/br&gt;								  &lt;/br&gt;								  &lt;/br&gt;										View merchandise. . . 									 																	    						&lt;/br&gt;	    	  			"
    }, {
    "id": 22,
    "url": "http://localhost:4000/subgroups/",
    "title": "SMB Subgroups",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Society for Mathematical Biology - Subgroups		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;   	  					Cell and Developmental Biology - CDEV: 	  	   		    	The Subgroup on Cell and Developmental Biology brings together researchers who are interested in understanding cell and developmental biology phenomena using quantitative approaches. Cell and developmental biology has long proven a fertile ground for mathematical modelling and, with recent advances in microscopy, image analysis, and fluorescent reporter methods, it is becoming an increasingly quantitative discipline. We are now able to constrain mathematical models of developmental processes, for example through parameter estimation, increasing the potential of such models to be used in a predictive way. The objective of this Subgroup is to accelerate these efforts, by supporting and advancing the use of mathematical modelling as part of the ‘toolkit’ of cell and developmental biology. Chair: Andreas Buttenschoen, abuttens@math. ubc. ca Secretary: Alexander Fletcher, a. g. fletcher@sheffield. ac. uk Committee Members: Ruth Baker, Linus Schumacher, and Michael Watson 	   	  	  &lt;/br&gt;			&lt;/br&gt;	  Education - EDUC: 	  	    	The Education Subgroup of SMB is comprised of members who desire to improve the teaching and mentoring of future scientists, and in particular undergraduate and graduate mathematical and computational biology researchers. As new research frontiers emerge, our members seek to understand the best practices to help future researchers of biological questions be successful in this space. Content areas include, but are not limited to the teaching and learning of modeling, computing, data science, and quantitative reasoning. Challenge spaces include, but are not limited to designing learning experiences across disciplines, defining and assessing outcomes for new interdisciplinary and technological spaces, integrating data with mathematical modeling, educational outreach initiatives, and providing an inclusive and diverse environment for learning and for the profession. Chair: Dr. Holly Gaff, HGafff@odu. eduPast Chair: Carrie Diaz Eaton, cdeaton@bates. eduTwitter: @Edu4SMB 					  	  &lt;/br&gt;			&lt;/br&gt;	  Immunobiology and Infection - IMMU: 	  	    		    		    The Immunobiology and Infection Subgroup brings together researchers who are interested in the modeling and analysis of immune processes in human disease and of host-pathogen interactions. Our members study various topics, including within-host infectious disease dynamics, host immune responses, causes and effects of inflammation, disease progression and outcome, integration of experimental and clinical data into models, and model-driven experimental design. Chair: Angela Reynolds, areynolds2@vcu. edu Secretary/Treasurer: Jonathan Forde Advisory Committee Members: Jessica Conway and Stanca Ciupe Twitter: @smb_imin 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Mathematical Epidemiology - MEPI: 	  	    		    		    The Mathematical Epidemiology Subgroup brings together scholars interested in dynamical modeling and statistical analysis of the spread of infectious diseases. Topics of interest to the group include: deterministic and stochastic modelling of population dynamics of infectious diseases; statistical analysis of population-level infectious disease data; describing and modelling human behavioral decisions in relation to infectious diseases; control and eradication strategies for endemic infections; prevention and response strategies for emerging and re-emerging infections; effects of antimicrobial resistance; policy implications. Chairs: Zhilan Feng fengz@purdue. edu Twitter: #SMBMathEpi 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Mathematical Neuroscience - NEUR: 	  	    		    		    The Mathematical Neuroscience Subgroup provides a forum for the use and development of advanced mathematical techniques in the modelling and analysis in all areas of neuroscience – from cells to networks. The frameworks of Dynamical Systems, Network Science, Scientific Computation, Information theory, and Stochastic processes are just a few areas that we are using to advance knowledge on synaptic plasticity, neural rhythm generation, memory and learning, decision making, and natural computation, hand-in-hand with colleagues from experimental and clinical neuroscience. Chairs: Cheng Ly, cly@vcu. edu; Wilten Nicola, wilten. nicola@ucalgary. ca Email: mathneurocollective@gmail. com Twitter: @MathNeuro_ 		    		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Mathematical Oncology - ONCO: 	  	    		    		    The Mathematical Oncology Subgroup is focused on integrating and applying mathematical and computational models to better understand and predict cancer initiation, progression, and treatment. Chairs: Alexander R. A. Anderson, Alexander. Anderson@moffitt. org; Heiko Enderling, Heiko. Enderling@moffitt. org Twitter: @mathonco Website: http://mathematical-oncology. org 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Methods for Biological Modeling - MFBM: 	  	    		    		    The Methods for Biological Modeling Subgroup brings together those interested in the development of novel application of methods (e. g. , numerical, analytical/theoretical, agent-based modeling, statistical, parameter estimation, identifiability, and machine learning) towards modeling biological systems. In applying standard methodologies, mathematical modelers encounter many challenges with simulating and analyzing biological systems, such as high dimensionality, computational complexity, observation or process error, model bias, and intra- and inter-individual heterogeneity. The objective of the Subgroup is to support and disseminate new advances in tackling these challenges to the Mathematical Biology community. Chairs: Kevin Flores, kbflores@ncsu. edu; Erica Rutter, erutter2@ucmerced. edu; John Nardini, jtnardin@ncsu. edu Twitter: @SMBmfbm 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Population Dynamics, Ecology &amp; Evolution - POPD: 	  	    						The Population Dynamics, Ecology and Evolution Subgroup aims to foster a discussion on theoretical, mathematical and empirical approaches to core topics in population dynamics, ecology and evolution. Chair: R. Christina Cobbold, Christina. Cobbold@glasgow. ac. uk Secretary: James Watmough, watmough@unb. ca Twitter: @smbpdee Google Group: smbpdee 	    																	&lt;/div&gt;	  &lt;/div&gt;		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt; "
    }, {
    "id": 23,
    "url": "http://localhost:4000/sunday/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Sunday, August 16		    		    US Eastern Time Zone (EDT) 	    	&lt;/br&gt;		    		    			    				        								&lt;/br&gt;								&lt;/br&gt;															  Overview Sunday Monday Tuesday Wednesday Thursday				    			    		    	    	  			                    Pre-conference Mentoring Program        1:00-5:00pm:         																										 Pre-conference Mentoring Program Schedule									 									  			  							  										  			The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 							  										  			* all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 																											 												  												   Time												   Speaker												   Scheduled Event												  												 												 												  												   1:00pm												   Robert Smith?												   Introduction												  												  												   1:00–1:30pm												   Alexandra Hogan												   How to Attend a Virtual Conference												  												  												   1:30–2:00pm												   Rebecca Everett												   Things I Wish I'd Known as a Postdoc												  												  												   2:00–3:30pm												   Robert Smith?												   How to Get a Tenure-Track Job												  												  												   3:30–4:00pm												   Meghan Burke												   The Tenure Process												  												  												   4:00–5:00pm												   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   												   												   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)												  												 																						  																	 									 																						                                                                                                                                                                                                   					 		  Overview Sunday Monday Tuesday Wednesday Thursday	 	"
    }, {
    "id": 24,
    "url": "http://localhost:4000/tags.html",
    "title": "Tags",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "          Tags               SMB:                                  		Conference Announcement	: 		  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. We will probably remember these . . . 	 			In 				SMB, 				eSMB, 								Jun 08, 2020						                              eSMB:                                  		Conference Announcement	: 		  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. We will probably remember these . . . 	 			In 				SMB, 				eSMB, 								Jun 08, 2020						                                             Featured:    				                  "
    }, {
    "id": 25,
    "url": "http://localhost:4000/thursday/",
    "title": "SMB2020 Thursday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Thursday, August 20		    		    US Eastern Time Zone (EDT) 	    	&lt;/br&gt;		    		    			    				        																&lt;/br&gt;								&lt;/br&gt;															  Overview Sunday Monday Tuesday Wednesday Thursday				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 				                Coffee with friends &amp; colleagues       8:00am:                  	                Subgroup Keynote              8:30am:       Sue Ann Campbell, &lt;/br&gt;University of Waterloo       Neuroscience Subgroup           								            	       	                           	   	    Modulation of synchronization by a slowly varying M-current 	     The neurotransmitter acetylcholine has been shown to modulate the firing properties of several types of neurons through the down-regulation of voltage dependent potassium currents such as the muscarine-sensitive M-current. In the brain, levels of acetylcholine change with activity. For example, acetylcholine is higher during waking and REM sleep and low during slow wave sleep.  In this talk we show how the M-current affects the bifurcation structure of a generic conductance-based neural model and how this determines synchronization properties of the model. We then use phase-model analysis to study the effect of a slowly varying M-current on synchronization. This is joint work with Victoria Booth, Xueying Wang and Isam Al-Darbasah. 	                    			  													  		    		      		        Sub-group contributed talks (9:30-11:00am)		        			  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	CDEV: The interplay of intracellular pattern formation, geometry, polarisation and mechanics (9:30-11:00am)													 						  			  				  							  							  										  									  	Erwin Frey								  	 								  		Ludwig-Maximilians University Munich								  									  									  										  			&lt;/br&gt; Self-organization principles of intracellular pattern formation 								  										  									  									  										  										  											  			Dynamic patterning of specific proteins is essential for the spatiotemporal regulation of many important intracellular processes in procaryotes, eucaryotes, and multicellular organisms. The emergence of patterns generated by interactions of diffusing proteins is a paradigmatic example for self-organization. We will discuss quantitative models for intracellular Min protein patterns in E. coli, Cdc42 polarization in S. cerevisiae, and the bipolar PAR protein patterns found in C. elegans. By analyzing the molecular processes driving these systems we will show how to derive a theoretical perspective on general principles underlying self-organized pattern formation of proteins in cells. 									  											  									  									  									  								  								  								  	Veronica Grieneisen								  	 								  		Cardiff University								  									  									  										  			&lt;/br&gt; Self-organization through the mesoscale, lessons from plant patterning and adaptive responses 								  										  									  									  										  										  											  			This core ability of a cell to polarize in isolation from others then leads to the collective emergence of tissue polarity, which will be dependent on the mode of cell-cell interaction. In plants, such tissue polarity will allow for information flow over large distances, be it to guide development or to interface the environment, such as in plant-nutrient uptake. We will show how for such organism-level coordination of signals, sub-cellular components turn out to be extremely important. Also, we will discuss temporal-spatial dynamical constraints operating on such polarized tissues which are likely to have a evolutionary-developmental role in the systems we encounter in nature. 									  											  									  									  									  								  								  								  	Stan Maree								  	 								  		Cardiff University								  									  									  										  			&lt;/br&gt; Intracellular patterning coupled to cell shape can lead to sensitization of signal detection 								  										  									  									  										  										  											  			A lot of animal and plant cell polarity can be understood by simple biochemical processes. Exploring their dynamics shows that simple circuits are able to generate a plethora of diverse cellular patterns. Firstly, I will discuss how we can explore the potential for break-of-symmetry within cells. I will then explore, how through modelling we can better understand the feedbacks between these patterns and biophysical triggers that cause cell shape changes. Finally, I will show how polarity formation can also be used as an integrator for sensing external cues, and discuss how alterations of this could cause tissue-level disruption. To extend these insights directly to experimental data, I will then show how our modelling framework can also be used for segmentation of imaging data, showing examples that range from complex epithelia to organoids. 									  											  									  									  									  								  								  								  	Padmini Rangamani								  	 								  		University of California at San Diego								  									  									  										  			&lt;/br&gt; Stability analysis of a bulk-surface model for membrane-protein clustering 								  										  									  									  										  										  											  			Protein aggregation on the plasma membrane (PM) is of critical importance on many cellular processes such as cell adhesion, endocytosis, fibrillar conformation, and vesicle transport. Lateral diffusion of protein aggregates or clusters on the surface of the PM plays an important role in governing their heterogeneous surface distribution. However, the stability behavior of the surface distribution of protein aggregates remains poorly understood. Therefore, understanding the spatial patterns that can emerge on the PM uniquely through protein-protein interaction and diffusion is an important step towards a more complete description of the mechanisms behind protein clustering on the cell surface. In this work, we investigate the pattern formation of a reaction-diffusion model that describes the dynamics of a system of ligand-receptor complexes. The purely diffusive ligand in the cytosol can bind receptors in the PM and the resultant ligand-receptor complexes not only diffuse laterally but can also form clusters resulting in different oligomers. From a methodological viewpoint, we provide theoretical estimates for diffusion-driven instabilities of the protein aggregates based on the Turing mechanism. We also obtain the distribution of the size of the protein aggregates and their spatial locations depending on both initial conditions and kinetic parameters using computational methods. Our results suggest that spatial heterogeneity emerges only when the cluster diffusion rates decay as a function of cluster size. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	IMMU: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Shilian Xu								  	 								  		Monash University								  									  									  										  			&lt;/br&gt; Saturation of influenza virus neutralization and antibody consumption can both lead to bistable growth kinetics 								  										  									  									  										  										  											  			Influenza virus is a major human health threat. Neutralizing antibodies elicited through prior infection or vaccination play an irreplaceable role in protection from subsequent infection. The efficacy of antibody-dependent vaccines relies on both virus replication and neutralisation, but their quantitative relationship was unknown. Here we use mathematical models to quantitatively investigate viral survivability determined by antibody concentration and inocula size. We performed focus reduction assays for 49 seasonal influenza A/H3N2 viruses circulating during 2017–2019 against influenza antisera raised in ferrets, and find that the antibody consumption rates of individual reactions were either small or large, and this was strongly positively correlated with virus saturation. Regardless of antibody consumption rate, virus-antibody interactions always lead to antibody-induced bistable viral kinetics. As a result, at a specific interval of antibody concentration, small viral inocula are eliminated but not large virus inocula, which is triggered by saturated virus neutralization or antibody consumption. Our finding highlights virus-antibody interaction with different antigenic properties, thereby explaining commonly observed influenza re-infection and enhancing vaccine efficiency. 									  											  									  									  									  								  								  								  	Jacob Summers								  	 								  		University of Tennessee								  									  									  										  			&lt;/br&gt; Mathematical modeling of Mycobacterium tuberculosis dynamics in macaques 								  										  									  									  										  										  											  			Mycobacterium tuberculosis (Mtb) is the causative agent of tuberculosis (TB), infecting a large proportion (~30%) of the world’s population. Only a small proportion of infected individuals develop clinical disease, and factors determining why some individuals remain asymptomatic and some become sick remain unknown. Unfortunately, understanding disease progression in humans remains challenging because most people do not know they are infected. Instead, several different animal species such as mice and monkeys can be infected with Mtb, thus, providing potential explanations for progression of humans to TB. Several recent studies provided interesting insights into early Mtb dynamics in monkeys. In particular, it was found that infection of monkeys with a set of individually barcoded Mtb strains resulted in most local foci of infection (called granulomas) to contain a single barcode. This suggested that individual granulomas were started by a single bacterium. We used stochastic mathematical models to understand whether this observation is also consistent with the hypothesis that multiple strains cause granuloma formation but most die during early dynamics. Our model included the simplest possible way to describe early stochastic dynamics (linear birth-death model); interestingly, we could not find one set of model parameters that allowed us to accurately describe both mean number of bacteria per granuloma and distribution of founder strains in different granulomas. This suggests early Mtb dynamics in monkeys are unlikely to be described by a simple birth-death model, and other biological aspects must be included in the model, in particular, impact of the immune response on rates of Mtb replication and death. 									  											  									  									  									  								  								  								  	Suneet Singh Jhutty								  	 								  		Frankfurt Institue for Advanced Studies (FIAS); Goethe University Frankfurt, Germany								  									  									  										  			&lt;/br&gt; Mapping Influenza from blood data using deep learning 								  										  									  									  										  										  											  			Seasonal and pandemic influenza causes enormous economic loss and leads to health complications and death. A better understanding of the role of different blood constituents during infection is necessary. Mathematical analysis of data can help us to better understand the link between blood properties and the influenza virus. Furthermore, the measurement of influenza viral load in a person is laborious and time-consuming. Therefore, it is crucial to have a reliable and fast method to determine the viral load in a patient. Here, we analyze blood data from mice and explore the different correlations between the key players during an infection. We test successfully a novel approach to use deep learning to infer viral load from this blood data. Hence, the viral load is directly inferred from a blood test. Using a simple multilayer perceptron, we train the algorithm with a comparatively small data set, to map blood data to the viral load. This shows the general possibility to use blood constituents measured in every routine blood count (like lymphocytes and erythrocytes) to infer the viral load in the body. Even with high variability in the data, the model prediction is reasonably accurate. Our results may lead the way to allow the measurement of the viral load from already collected blood data in the future. Hence, it would not only reduce the workload but be probably also faster. Lastly, our results suggest that platelets and granulocytes play an essential role during influenza infection. 									  											  									  									  									  								  								  								  	Yuhuang Wu								  	 								  		University of New South Wales, Sydney								  									  									  										  			&lt;/br&gt; Impact of fluctuation in frequency of HIV reactivation during antiretroviral therapy interruption 								  										  									  									  										  										  											  			Antiretroviral Therapy (ART) provides effective control of human immunodeficiency virus (HIV) replication and maintains the viral loads of HIV at undetectable levels. Interruption of ART causes recrudescence of HIV plasma viremia due to the reactivation of latently HIV-infected cells, generally within weeks of discontinuation of ART. Here we characterize the timing of both the initial and subsequent successful viral reactivations following ART interruption in macaques infected with simian immunodeficiency virus (SIV). We compare these to previous results from human patients infected with HIV. We find that on average the time until the first successful viral reactivation event is longer than the time between subsequent successful reactivations. Based on this result, we hypothesise that the reactivation frequency of both HIV and SIV may fluctuate over time and that this may have implications for treatment of HIV. We develop a stochastic model to simulate the behaviour of viral reactivation following ART interruption that incorporates fluctuations in the frequency of reactivation. Our model is able to explain the difference in timing between the initial and subsequent successful reactivation events. Furthermore, we show that one of the impacts of a fluctuating reactivation frequency would be to significantly reduce the efficacy of “anti-latency” interventions for HIV that aim to reduce the frequency of reactivation. It is therefore essential to consider the possibility of a fluctuating reactivation frequency when assessing the impact of such intervention strategies. 									  											  									  									  								  								  								  									  	Vitaly Ganusov								  	 								  		University of Tennessee								  									  									  										  			&lt;/br&gt; Structure-imposed constrains make Brownian walkers efficient searchers 								  										  									  									  										  										  											  			Pathogen-specific CD8 T cells face the problem of finding rare cells that present their cognate antigen either in the lymph node or infected tissue. To optimize the search for rare targets it has been proposed that T cells might perform a random walk with long displacements called Levy walks enabling superdiffusive behavior and shorter search times. Many agents ranging from molecules to large animals have been found to perform Levy walks suggesting that Levy walk-based search may be evolutionary selected. However, whether random walk patterns are driven by agent-intrinsic programs or being shaped by environmental factors remains largely unknown. We examined the behavior of activated CD8 T cells in the liver where both the movement of the cells and the underlying structural constrains can be clearly defined. We show that Plasmodium-specific liver-localized CD8 T cells perform Brownian, short displacement walks and yet display superdiffusive overall displacement, the cardinal feature of efficient Levy walks. Because liver-localized CD8 T cells are mainly associated with liver sinusoids, we show that linear structure of the sinusoids is sufficient to cause T cells to superdiffuse even when movement lengths are Brownian. Simulations of Brownian or Levy walkers in structures derived from the liver sinusoids illustrate that structure alone can enforce superdiffusive movement. Moreover, Brownian walkers require less time and thus are more efficient than Levy walkers at finding a rare target when T cells search for the infection in physiologically-derived liver structures. Importantly, analysis of fibroblastic reticular cell networks on which CD8 T cells move in lymph nodes also allows for superdiffusion in simulations which is not observed experimentally suggesting that structure is not the only factor determining movement patterns of T cells. Our results strongly suggest that observed patterns of movement of CD8 T cells are likely to result as a combination of a cell-intrinsic movement program, physical constrains imposed by the environmental structures, and other environmental cues. Future work needs to focus on quantifying relative contributions of these factors to the overall observed movement patterns of agents. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	MEPI: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Samson Ogunlade								  	 								  		James Cook University								  									  									  										  			&lt;/br&gt; Modeling the potential of wAu-Wolbachia strain invasion in mosquitoes to control Aedes-borne arboviral infections 								  										  									  									  										  										  											  			Arboviral infections such as dengue, Zika and chikungunya are fast spreading diseases that pose significant health problems globally. In order to control these infections, an intracellular bacterium called Wolbachia has been introduced into wild-type mosquito populations in the hopes of replacing the vector transmitting agent, Aedes aegypti with one that is incapable of transmission. In this study, we developed a Wolbachia transmission model for the novel wAu strain which possesses several favourable traits (e. g. , enhanced viral blockage and maintenance at higher temperature) but not cyctoplasmic incompatibility (CI) - when a Wolbachia-infected male mosquito mates with an uninfected female mosquito, producing no viable offspring. This model describes the competitive dynamics between wAu-Wolbachia-infected and uninfected mosquitoes and the role of imperfect maternal transmission. By analysing the system via computing the basic reproduction number(s) and stability properties, the potential of the wAu strain as a viable strategy to control arboviral infections is established. The results of this work show that enhanced maintenance of Wolbachia infection at higher temperatures can overcome the lack of CI induction to support wAu-Wolbachia infected mosquito invasion. This study will support future arboviral control programs, that rely on the introduction of new Wolbachia variants. 									  											  									  									  									  								  								  								  	Maryam Aliee								  	 								  		University of Warwick								  									  									  										  			&lt;/br&gt; Estimating the distribution of extinction times of infectious diseases in deterministic models 								  										  									  									  										  										  											  			For many infectious diseases the eventual aim of control measures is eradication - completely removing the pathogen from host populations and the environment. Theoretical models can be used to predict the time to extinction under specific interventions. In general, this question requires the use of stochastic models which recognise the inherent individual-based, chance-driven nature of the dynamics; yet stochastic models are inherently computationally expensive, especially when considering parameter uncertainty. On the other side, deterministic models are practical and tractable, however, the endpoint of an infection is by definition ambiguous in these models since the populations are represented by continuous variables that never reach zero. We study the extinction problem in deterministic models with the help of an effective ``birth-death'' description of infection and recovery processes. We present a practical method to estimate the distribution, and therefore robust means and prediction intervals, of extinction times by calculating their different moments within the birth-death framework. We compare these predictions with the solutions of the corresponding forward Kolmogorov equations. We then extend this framework to estimate the extinction time of more complex and realistic infection dynamics, African sleeping sickness, gHAT, which is a vector-borne disease transmitted to humans by tsetse. This method will enable us to improve predictions of the timing of elimination of transmission for gHAT using our existing deterministic framework. 									  											  									  									  									  								  								  								  	Sung-mok Jung								  	 								  		Hokkaido University								  									  									  										  			&lt;/br&gt; Reconstruction and analysis of the transmission network of African swine fever in People’s Republic of China, August 2018–September 2019 								  										  									  									  										  										  											  			Introduction of African swine fever (ASF) to China in mid-2018 and subsequent transboundary spread across Asia devastated regional swine production, affecting live pig and pork product-related markets worldwide. In order to explore the spatiotemporal spread of ASF in China, we reconstructed possible ASF transmission networks using nearest neighbour, exponential function, equal probability, and spatiotemporal case-distribution algorithms. From these networks we estimated the reproduction numbers, serial intervals, and transmission distances of the outbreak. The mean serial interval between paired units was around days for all algorithms, while the mean transmission distance ranged from 332–456 kilometers. The reproduction numbers for each algorithm peaked during the first two weeks and steadily declined through the end of 2018 before hovering around the epidemic threshold value of one with sporadic increases during 2019. These results suggest that: 1) swine husbandry practices and production systems that lend themselves to long-range transmission drove ASF spread, and 2) outbreaks went undetected by the surveillance system. China and other affected countries have stepped up efforts to control ASF within their jurisdictions, and continued support for strict implementation of biosecurity standards and improvements to ASF surveillance are essential for halting transmission in China and further spread across Asia. 									  											  									  									  									  								  								  								  	Usman Sanusi								  	 								  		Technical University of Munich								  									  									  										  			&lt;/br&gt; Influence of quiescence on host-parasite coevolutionary dynamics 								  										  									  									  										  										  											  			Mathematical Modelling is widely being used as a tool to predict and understand the spread of infectious disease such as HIV, tuberculosis, Measles, Malaria, corona virus,. . . . However, most diseases have an intra-host quiescent stage defined sometimes as covert infection (malaria for example), while other parasites have dormant stages in the environment. The influence of these two life-history traits seems to be neglected by mathematicians when developing their models [1]. In this research, we develop a coevolutionary model similar to [2,3] to predict and understand the spread of disease considering the effect of intra-host quiescence. Analytical results are obtained for the stability of the system. We especially derived a stability conditions for a five by five system with quiescent stage. Numerical simulations were also performed and we show that the period of oscillations and the time to damping off is almost double under the quiescence compared to the classic epidemiological model. We finally extended the model to include the effect of stochasticity on disease transmission and study analytically the outcome using a Markov-Chain model. The model dynamics with stochasticity follow the deterministic one. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	MFBM: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Fiona		Macfarlane								  	 								  			University of St. Andrews, Scotland, frm3@st-andrews. ac. uk								  									  									  										  			&lt;/br&gt; Bridging the gap between individual-based and continuum models of growing cell populations 								  										  									  									  										  										  											  			Stochastic individual-based modelling approaches allow for the description of single cells in a biological system. These models generally include rules that each cell follows independently of other cells in the population. Various mechanisms and biological phenomena can be described using these simplistic mathematical models. However, these models cannot be analysed mathematically. Therefore, it can be beneficial to derive the corresponding deterministic model from the underlying random walk of the stochastic model. The resulting deterministic models, usually partial differential equations (PDEs), can then be analysed to provide further information about the biological systems studied. We have developed a range of simple IB models that describe biological systems with various properties of interest, such as, volume-filling and chemotaxis and pressure-dependent growth and proliferation. For each model, we were able to derive PDEs from the underlying random walk. We carried out comparisons between the stochastic and deterministic model highlighting situations where there is agreement in the models, and situations where they do not agree. Ultimately, the results illustrate how the simple rules governing the dynamics of single cells in our individual-based model can lead to the emergence of complex spatial patterns of population growth observed in continuum models. These models can be applied to a variety of biological situations such as tumour growth, tumour invasion and wound healing. 									  											  									  									  									  								  								  								  	Alexey		Penenko								  	 								  		ICM&amp;MG SB RAS,	Russian Federation,	a. penenko@yandex. ru									  									  									  										  			&lt;/br&gt; Inverse Modeling of Biological Processes with Adjoint Ensemble Methods 								  										  									  									  										  										  											  			In order to study complicated natural processes through mathematical modeling, it is necessary to take into account both mathematical models of the processes and the available measurement data collected about these processes. We refer to this modeling approach as inverse modeling. In biological studies, the models can change very rapidly, so the unified algorithms that do not take much effort to apply them to different models are preferable. The general inverse modeling framework based on the ensembles of the adjoint equation solutions is considered in the context of biomedical applications with the advection-diffusion-reaction models. The sensitivity relations and adjoin equations allow constructing the approximation of the inverse modeling problem stated a system of ordinary or partial differential equations in the form of a parametric family of quasi-linear operator equations with the sensitivity operators. This approximation allows both solving, analyzing, and comparing a wide range of biomedical inverse problems in a unified fashion. 									  											  									  									  									  								  								  								  	Karina		Islas Rios									  	 								  		Monash University	Australia	karina. islasrios@monash. edu									  									  									  										  			&lt;/br&gt; NetScan: a computational tool for discovering and visualizing biochemical networks with defined topological structures.  								  										  									  									  										  										  											  			Our quantitative understanding of biochemical networks empowered by computational modelling have shown that the topology (or structure) of a network often have determining roles in shaping the network’s dynamic and steady state behaviours. For examples, negative feedback can give rise to oscillation while positive feedback can bring about bistability to the host network. Thus, being able to systematically identify sub-networks with defined topological structures within the human protein interactome is critical for the discovery of biochemical networks with desired behavioural properties. However, this is a non-trivial task given the enormous complexity of the human protein-protein interaction network. Structural principles of biochemical networks can be discovered by focusing on small sub-networks. Finding those sub-networks in the assembly of complex biochemical networks can be achieved by implementing graph theory-based algorithms. Here, we develop NetScan, an open source web-based application capable of ‘scanning’ the large and complex human signalling interactome within the Signor 2. 0 and STRING databases to identify all sub-networks with given structural topologies, e. g. those with a specific negative feedback, positive feedback or feed-forward loop wiring. NetScan allows users to specify the specific input topologies and the interactome network within which it will explore, and return all the smallest sub-networks with the desired topologies. The resulting sub-networks are displayed in two forms: a detailed version which includes all interaction links, and a simplified version presenting the net effects between the nodes. In summary, NetScan is a web application that provides unprecedented ability to systematically identify and visualise sub-networks within the human protein-protein interaction network with specific topological wiring. 									  											  									  									  									  								  								  								  	Hyukpyo		Hong								  	 								  		Korea Advanced Institute of Science and Technology,	Republic of Korea,	hphong@kaist. ac. kr								  									  									  										  			&lt;/br&gt; Derivation of stationary distributions of biochemical reaction networks via structure transformation 								  										  									  									  										  										  											  			Long-term behaviors of biochemical reaction networks are described by steady states in deterministic models and stationary distributions in stochastic models. Unlike deterministic steady states, stationary distributions capturing inherent fluctuations of reactions are extremely difficult to derive analytically due to the curse of dimensionality. Here, we develop a method for deriving stationary distributions from deterministic steady states by transforming a network to have a special dynamic property. Specifically, we merge nodes and edges of a network to make the steady states complex balanced, i. e. , the in- and out- flows of each node are equal. By applying our approach to networks that model autophosphorylation of EGFR, PAK1, and Aurora B kinase and a multi-timescale toggle switch, we identify robustness, sensitivity, and multi-modality of their stationary distributions. Our method provides an effective tool to understand long-term behaviors of stochastic biochemical systems. 									  											  									  									  								  								  								  									  	Linard		Hoessly								  	 								  		University of Copenhagen,	Denmark,	hoessly@math. ku. dk									  									  									  										  			&lt;/br&gt; Stationary distributions of stochastic reaction networks via decomposition 								  										  									  									  										  										  											  			Stochastic reaction networks (CRNs) are often used to describe systems with small molecular counts, which applies to many processes in living systems. They are usually modelled through continuous-time Markov processes. Studying dynamics of stochastic CRNs is in general hard, both analytically and by simulation. In particular stationary distributions of stochastic reaction networks are only known in some cases like, e. g. , complex balanced or autocatalytic CRNs. I will review some results on form of stationary distribution and convergence to stationary distribution. Then, I am going to analyse CRNs under the operation of join and examine conditions such that the form of the stationary distributions of a CRN is derived from the parts of the decomposed CRNs, which allows recursive application. To illustrate the theory I present examples of stochastic reaction networks of interest in order to highlight the decomposition. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  																																		 	NEUR: Rhythms, Sleep, Reward, Vision (9:30-11:00am)													 						  			  				  							  							  										  									  	Kevin Hannay								  	 								  		University of Michigan								  									  									  										  			&lt;/br&gt; Circadian State Estimation using Wearable Data 								  										  									  									  										  										  											  			Almost every living thing exhibits daily cycles in behavior and physiology known as circadian rhythms. In humans, disrupted circadian rhythms have been implicated in a spectrum of both mental and physical health maladies including cancer, diabetes, addiction, depression and sleep disorders. Therefore, it is a matter of vital importance to understand and predict human circadian rhythms. In previous work we have derived and fit a low dimensional dynamical model for human circadian rhythms. However, the heterogeneity of circadian behaviors (early birds/night owls) in the human population must be accounted for in the model. In this work we investigate how data collected by wearable devices (apple watch, fitbit, etc) can be used to personalize the circadian parameters and improve forecasting accuracy. 									  											  									  									  									  								  								  								  	Maia Angelova								  	 								  		Deakin University								  									  									  										  			&lt;/br&gt; Data driven model for detecting insomnia from multi-night actigraphy time series data 								  										  									  									  										  										  											  			Sleep is an important part of human existence as we spend 1/3 of our lives sleeping. It is a complex multi-dimensional cycle that reflects developmental changes in mental and physical health, along with the day-to-day state fluctuations. Insomnia is characterised by the inability to fall asleep or stay asleep and/or waking too early and being unable to fall back asleep.  Insomnia is a sleep disorder that remains under-diagnosed.  We propose a new data driven model for classification of nocturnal awakenings in acute and chronic insomnia and normal sleep from nocturnal actigraphy collected from pre-medicated individuals with insomnia and normal sleep controls. Our model does not require sleep diaries or any other subjective information from the individuals. We derive dynamical and statistical features from the actigraphy time series data. These features are then combined in machine learning model to classify individuals with insomnia from healthy sleepers. The model includes a classifier followed by optimization algorithm that incorporates the predicted quality of each night of sleep for an individual to classify into acute/chronic insomnia or healthy group. The developed model provides a signature of acute/chronic insomnia obtained from actigraphy only and is very promising as a pre-screening tool to detect the condition in home environment. M Angelova, C Karmakar, Y Zhu, SP Drummond, J Ellis. (2020). Automated Method for Detecting Acute Insomnia Using Multi-Night Actigraphy Data. IEEE Access, 8, 74413-74422. 									  											  									  									  									  								  								  								  	Davide Maestrini								  	 								  		UCLA								  									  									  										  			&lt;/br&gt; A mathematical model of 'wanting', 'liking, and brain reward circuitry in drug addiction 								  										  									  									  										  										  											  			We propose a mathematical model combining the so-called opponent process theory and the reward prediction error (RPE) in the context of drug addiction. Using this model, we investigate the different dynamics towards addiction and provide a possible description of the process of detoxification. The opponent processes are modeled by introducing a response kernel that integrates dopamine-induced neuronal activity to form a sense of reward. The shape of the kernel, which might be associated with physical and biological characteristics of neurocircuits of the brain reward system, plays a key role in determining the overall experience of consumption of addictive substances and the mismatch between the expected reward and the actual eward, the reward prediction error (RPE). With time and repeated exposures to drugs of abuse, the response kernel will change based on the value of the RPE and on the process of neuroadaptation. The dynamics of this change represents the evolution towards addiction and is mathematically described as a trajectory in a three-dimensional parameter space representing the RPE. In our framework, the surface representing the RPE is divided in two regions representing positive and negative values of value of the RPE, respectively.  We show that the dynamics associated with naïve drug users is represented by a trajectory lying in the first, positive-RPE region but, with time and repeated exposure to drugs of abuse, the trajectory will enter into the second, negative-RPE region. After transition to addiction, the subsequent dynamics is largely confined within the negative-RPE region. We finally propose models for exiting the negative-RPE parameter regions and connect it to a description of a detoxification protocols, such as the use of methadone to address heroin addiction. 									  											  									  									  									  								  								  								  	Paul Roberts								  	 								  		University of Sussex								  									  									  										  			&lt;/br&gt; Using mathematics to investigate the mechanisms behind vision loss 								  										  									  									  										  										  											  			The retina is a tissue layer at the back of the eye that uses photoreceptor cells to detect light. Photoreceptors can be characterised as either rods or cones. Rods provide achromatic vision under low light conditions, while cones provide high-acuity colour vision under well-lit conditions. The term Retinitis Pigmentosa (RP) refers to a range of genetically mediated retinal diseases that cause the loss of photoreceptors and hence visual function. RP leads to a patchy degeneration of photoreceptors and typically directly affects either rods or cones, but not both. During the course of the disease, degenerate patches spread and the photoreceptor type unaffected by the mutation also begins to degenerate. The cause underlying these phenomena is currently unknown; however, several key mechanisms have been hypothesised: oxygen toxicity, trophic factor depletion and the release of toxic substances by dying cells. Here we present mathematical models, formulated as systems of PDEs, to investigate the trophic factor hypothesis. Using a combination of numerical simulations and mathematical analysis, we determine the geographic variation in retinal susceptibility to degeneration, evaluate the degree to which in vivo spatio-temporal patterns of degeneration can be replicated by our models and predict the effects of various clinically-relevant treatment strategies. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  																																		 	ONCO: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Ryan John Murphy								  	 								  		Queensland University of Technology, Australia								  									  									  										  			&lt;/br&gt; Mechanical cell competition in heterogeneous epithelial tissues 								  										  									  									  										  										  											  			Mechanical cell competition is important during tissue development, cancer invasion, and tissue ageing. To study this phenomenon, we propose a one-dimensional mechanical model of cell migration in heterogeneous epithelial tissues that includes cell-length-dependent proliferation and death mechanisms. Proliferation and death are modelled in the discrete model stochastically and arise as source/sink terms in the corresponding continuum model that we derive. Applications we discuss include the evolution of homogeneous tissues experiencing proliferation and death, and cancer invasion with a cancerous tissue competing for space with an adjacent normal tissue. This framework allows us to postulate new mecha- nisms that explain the ability of cancer cells to outcompete healthy cells through mechanical differences rather than by having some intrinsic proliferative advantage. 									  											  									  									  									  								  								  								  	Aleksandra Ardaseva								  	 								  		University of Oxford, Oxford, UK								  									  									  										  			&lt;/br&gt; Modelling evolutionary adaptation of cancer cells to fluctuating oxygen levels 								  										  									  									  										  										  											  			A major challenge in malignant tumours is cell heterogeneity, which has been proposed to arise due to temporal variations in nutrient supply caused by highly irregular vasculature. Such variability requires cells to adapt to potentially lethal variations in environmental conditions. Risk spreading (“bet-hedging”) through spontaneous phenotypic variations is an evolutionary strategy that allows species to survive in temporally varying environments. Individuals within a species diversify their phenotypes ensuring that at least some of them can survive in the face of sudden environmental change. We aim to investigate whether cancer cells may adopt this strategy when dealing with rapidly changing levels of nutrient due to temporally-varying blood flow. Here, we present a system of nonlocal partial differential equations modelling the evolutionary dynamics of phenotype-structured cancer cell populations exposed to fluctuating oxygen levels. In this model, the phenotypic state of every cell is described by a continuous variable that provides a simple representation of its metabolic phenotype, ranging from fully oxidative to fully glycolytic. The cells are grouped into two competing populations that undergo heritable, spontaneous, phenotypic variations at different rates. A combination of analysis and numerical simulations indicates that under certain conditions the cell-oxygen dynamics can lead to regions of chronic hypoxia (low oxygen level) and cycling hypoxia. Moreover, the model shows that under chronic-hypoxic conditions lower rates of phenotypic variation lead to a competitive advantage, whereas higher rates of phenotypic variation can confer a competitive advantage under cycling-hypoxic conditions. In the latter case, bet-hedging evolutionary strategies, whereby cells switch between oxidative and glycolytic phenotypes, can spontaneously emerge. These results shed light on the evolutionary processes that may underpin the emergence of phenotypic heterogeneity in vascularised tumours, and suggest potential therapeutic strategies. 									  											  									  									  									  								  								  								  	Phillip J. Brown								  	 								  		The University of Adelaide, Australia								  									  									  										  			&lt;/br&gt; Modelling colon cancer: Investigating serrated crypts using a new model for deformable membranes 								  										  									  									  										  										  											  			Serrated sessile polyps (SSPs) are a type of lesion found in the colon that are known to lead to colorectal cancer. They develop when there are disruptions in the processes controlling the function of colonic crypts - the test-tube shaped structures that make up the lining of colon. It is currently not clear what will cause a healthy crypt to become serrated. The crypt has been extensively modelled, owing to its relatively simple composition. However, little modelling work has focused on the formation of serrated crypts, perhaps because of the relatively difficult task of modelling the epithelial monolayer on a deformable supporting structure. In this talk, we will introduce new a modelling approach that allows us to build a deformable membrane, sidestepping the issues that made the predominant off lattice models less suitable. We will then present some preliminary findings on the potential causes behind the characteristic appearance of serrated crypts elucidated by this model. 									  											  									  									  									  								  								  								  	Lisa C. Tucker-Kellog								  	 								  		Singapore								  									  									  										  			&lt;/br&gt; During multi-drug combination therapy, the speed of evolving drug-resistance is affected by the uniformity of pairwise synergism, additivity, or antagonism between the drugs 								  										  									  									  										  										  											  			Introduction: The search for combination therapies against cancer has focused on studying synergistic combinations (drug combinations with greater-than-additive efficacy) because they exhibit enhanced therapeutic efficacy at lower doses. Although synergistic combinations are intuitively attractive, therapeutic success often depends on whether drug resistance develops.  In computational modeling of drug-resistance evolution, our recent work (Saputra et. al, Cancer Res, 2018) delineated conditions under which synergistic pairs of drugs would have worse long-term performance than non-synergistic pairs of drugs, due to faster evolution of drug-resistance. In this work, we extend our modeling to three-drug combinations. In a multi-drug cocktail, some pairs of drugs may be synergistic while other pairs of drugs in the same cocktail may be additive or antagonistic (quantified using the combination index). Does this matter for evolution of drug-resistance? We develop theoretical generalizations about relative performance, for winning the race between cancer-killing efficacy versus drug-resistance evolution, using multi-drug cocktails with equal efficacy but different distributions of combination index (CI). Methods: We performed mathematical modeling of tumor cells evolving under selective pressure from synergistic, additive, and antagonistic three-drug combinations. Starting with small populations of drug-sensitive cells, we allowed rare mutational events to change a cell’s phenotype toward any single drug, which over time created potential for the presence or absence of drug-resistance against any or all drugs in the multi-drug therapy. Meanwhile, proliferation and death were simulated according to the combined cocktail efficacy toward each phenotype of cell. Results and Discussion: Longer duration of cancer control was achieved by multi-drug combinations having higher uniformity of pairwise CI (i. e. , all pairs of drugs within the cocktail having similar levels of synergism, or all pairs having similar levels of antagonism), compared with multi-drug cocktails having equal initial efficacy and equal overall CI, but greater differences in the pairwise CI’s. In other words, treatment was more likely to fail sooner if the three drugs had non-uniform amounts of pairwise synergism, compared with cocktails that had more uniform CI between the drug-pairs within the cocktail. The difference in outcomes was due to partially resistant phenotypes that achieved greater competitive advantage (meaning greater clonal expansion and greater sabotage of therapy) by resisting the most synergistic aspects of the cocktail.  This is because resisting one part of a synergistic group destroys not on									  											  									  									  								  								  								  									  	Yangjin Kim								  	 								  		Konkuk University, Republic of Korea								  									  									  										  			&lt;/br&gt; How the surgery-induced transition of reactive astrocytes to stem cell-like phenotypes leads to recurrence of GBM by Cxcl5: hybrid multi-scale approaches 								  										  									  									  										  										  											  			Glioblastoma multiforme (GBM) is the most aggressive form of brain cancer with a short median survival time. GBM is characterized by the hallmarks of aggressive proliferation and cellular infiltration of normal brain tissue. Tumor cells also interact with many cells including astrocytes and immune cells, and extracellular matrix (ECM) in a tumor microenvironment (TME) via exchanging molecular signals in order to increase survival rates in response to biochemical and biomechanical challenges. miRNAs and their downstream molecules are known to play a pivotal role in regulation of the balance of proliferation and aggressive invasion in response to metabolic stress in the tumour microenvironment (TME). Surgery-induced transition in reactive astrocyte populations can dramatically change the growth and invasion dynamics of GBM cells. In this work, we develop a multi-scale mathematical model of the tumor astrocytes dynamics in response to surgical resection of the primary tumor in TME. The hybrid model takes into account miR-451-LKB1-AMPK-OCT1-mTOR pathway signalling (ODEs), individual cell dynamics of the tumour, reactive astrocytes, stem cell-like astrocytes (lattice-free individual model), and signal transport by diffusible molecules (PDEs). We show how the effects of fluctuating glucose on tumour cell dynamics need to be reprogrammed by taking into account the recent history of glucose variations and an LKB1/OCT1 reciprocal feedback loop, which then determines tumor cell proliferation and migration. The model shows that surgery-induced changes in TME are the important factors for inducing the critical transition from reactive astrocytes to stem cell-like phenotypes. The model illustrates how variations in glucose availability significantly affect the activity of signalling molecules and, in turn, lead to critical cell migration. The model also predicts that (i) microsurgery of a primary tumour induces phenotypical changes in reactive astrocytes and stem cell-like astrocytes promoting tumour cell proliferation and migration by Cxcl5, (ii) this critical transition essentially increases the recurrence potential of GBM and leads to the low survival rate of patients. Finally, we investigated a new anti-tumour strategy by Cxcl5-targeting drugs in order to prevent this critical recurrence of the tumor. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	OTHE: Other Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Josephine Solowiej-Wedderburn								  	 								  		Surrey								  									  									  										  			&lt;/br&gt; Sensing some resistance: A mathematical model for the contractile mechanosensory mechanism within cells 								  										  									  									  										  										  											  			It is becoming increasingly clear that physical force and the mechanical properties of their microenvironment play a crucial role in determining cellular behaviour and coordination. Un- derstanding these differences has significant implications for tissue engineering applications and to determine how the mechanical microenvironment may affect, for example, cancer growth and invasion. We use a continuum elasticity-based model with an active contractile component to describe the mechanosensory mechanism of a cell or cell layer adhered to a substrate. The model context focuses on the most common biophysical experimental set-ups investigating cel- lular contractility, which infer the cell-generated force from observed deformations of substrates with experimentally known mechanical properties. The mathematical model is analysed and solved using both analytical approaches (exploiting approximations and symmetry arguments) and Finite Element Methods. We use the model to explain observed cellular adaptations to changes in the mechanical properties of the underlying gel. In particular, we consider the dis- tribution of adhesion throughout a cell. For experimentally realistic distributions of adhesion points, the model is capable of recreating cell shapes and deformations that are consistent with those experimentally observed. Furthermore, energy considerations are shown to have significant implications for the optimisation of cell adhesion. Thus, we demonstrate the neces- sity of considering the whole cell-substrate system, including the patterning of adhesion, when investigating cell stiffness sensing. 									  											  									  									  									  								  								  								  	Ashlee N. Ford Versypt								  	 								  		Oklahoma State								  									  									  										  			&lt;/br&gt; Investigation of Short Chain Fatty Acids on the Gut-Bone Axis: From Mechanism to a Computational Systems Approach 								  										  									  									  										  										  											  			Short chain fatty acids produced by the microbial community in the gut play a role in regulating the immune system. Butyrate is a specific short chain fatty acid that can be modulated through diet. In this collaborative project, we seek to formulate a series of ordinary differential equations to provide a mechanistic description of the dynamic effects of butyrate on cells of the immune system locally in the intestine and on bone health through systemic consequences. The team involves three mathematical biologists with chemical engineering backgrounds and an experimental collaborator with expertise in the physiological interactions between the digestive, immune, and musculoskeletal systems. 									  											  									  									  									  								  								  								  	Shaza Alsibaai								  	 								  		McGill University								  									  									  										  			&lt;/br&gt; Mathematical Modeling of Human Erythropoiesis 								  										  									  									  										  										  											  			As red blood cells (RBCs), also called erythrocytes, are known for their vital function of transporting oxygen to tissues, their production rate unsurprisingly depends on the tissues need for oxygen (hypoxia). The hormone erythropoietin (Epo) is the main factor regulating erythropoiesis, the process by which erythrocytes are produced. In adult humans, Epo is produced principally in the kidney and released in response to hypoxia in order to stimulate the production of more erythrocytes. Patients suffering from severe kidney impairment therefore under-produce RBCs and subsequently suffer from severe anemia. Exogenous administration of synthetic Epo is required, but what is the optimal dose, and how frequently should it be given? To even begin to address such questions, and to understand the underlying process requires a mathematical model. We develop a model consisting of two delay differential equations that reflects the effect of Epo on increasing the production rate of erythrocytes by preventing the apoptosis of its precursor cells. The explicit modelling of the Epo concentration will allow for later use of the model in the case of exogenous dosing of Epo. We analyse our proposed model, and as expected, are able to prove the non-negativity of solutions and the existence of a unique steady-state solution for the system which represents the normal number of mature erythrocytes and the concentration of Epo for a healthy human. We find a simple condition on the other parameters under which the steady state is locally asymptotically stable for arbitrary large delays. When this parameter condition is violated we show that the steady state is stable for small delays and loses its stability through a Hopf bifurcation. 									  											  									  									  									  								  								  								  	Torbjörn Lundh								  	 								  		Gothenburg								  									  									  										  			&lt;/br&gt; Möbius invariances in biology 								  										  									  									  										  										  											  			We will discuss several examples where using the extension of our rigid transformations to the class of Möbius mappings will gain insights into both the morphogenesis and dynamic use of anatomical features in fungi, mammals and especially primates. The examples will range from 'Growth and Form' to data augmentation in AI. 									  											  									  									  								  								  								  									  	Jessica Crawshaw								  	 								  		U. Melbourne								  									  									  										  			&lt;/br&gt; Examining the mechanical forces driving vascular regression using a fluid-structure-growth model 								  										  									  									  										  										  											  			Vascular regression is a critical process concluding the maturation of developing capillary networks, in which redundant blood vessels are removed. Recent research suggests that forces from the local blood flow (haemodynamic forces) trigger polarized endothelial cell migration against the flow, resulting in capillary collapse and regression. However, vascular regression is also driven by several additional pathways including local adhesion forces and cellular signalling factors. Due to the delicate nature of these microvessels, it is difficult to experimentally untangle the roles of each pathway during vascular development. As such, the development of computational models to analyse the relationship between the local haemodynamic forces and the surrounding vasculature during regression are invaluable. In this talk, we will present a novel computational framework to mathematically study and isolate the role of haemodynamic and adhesive forces in vessel deformation and endothelial cell migration during vascular regression. To model regression, we describe the capillary wall as a discretised hyperelastic membrane, hosting a multicellular model of endothelial cells. The capillary wall and the endothelial cells interact with and respond to the local blood flow in an iterative manner, creating a coupled fluid-structure growth simulation. This discrete approach provides a natural framework to consider the relationship between the capillary wall and the local blood flow, and allows for the easy inclusion of structural heterogeneities across the capillary wall. Using this model we are able to examine the relative roles of the haemodynamic forces and the local adhesion forces in vascular regression, and the network level ramifications of local regression. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	POPD: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Sharon Bewick								  	 								  		Clemson University								  									  									  										  			&lt;/br&gt; Nested Metapopulations 								  										  									  									  										  										  											  			Metapopulation theory has a long history in ecology, where it is used to describe the dynamics of organisms that frequent patchy and ephemeral habitats.  Recently, metapopulation theory has been used to model host-associated microbiomes, with hosts acting as patches, and microbes as dispersing organisms. Extending this framework to scenarios where the hosts themselves exist as metapopulations gives what we term 'nested metapopulations'. Although similar models have been studied in epidemiology, the focus is typically the effect of limited dispersal among patches on disease spread.  What is not considered, however, is the other key premise of ecological metapopulations - frequent extinction events at the patch scale. We explore models of nested metapopulations, focusing on how host-scale metapopulation structure and microbe-scale metapopulation structure interact to govern the distribution of microbes across both hosts and the habitat patches in which they live. 									  											  									  									  									  								  								  								  	Lihong Zhao								  	 								  		University of Idaho								  									  									  										  			&lt;/br&gt; Eco-evolutionary Dynamics of Microbial Communities 								  										  									  									  										  										  											  			Microbes form complex communities which have profound effects on host health status. Understanding the evolutionary dynamics of microbial communities is a key step towards the goal of manipulating microbiomes to promote beneficial states. While interactions within a microbial community and between microbes and their environment collectively determine the community composition and population dynamics, we are often concerned with traits or functions of a microbiome that link more directly to host health. To study how traits of a microbiome are impacted by eco-evolutionary dynamics, we recast a classic resource-mediated population dynamic model into a population genetic framework which incorporates traits. Using simple communities as example, we illustrate how natural selection, mutation, and shifts in the environment work together to produce changes in trait values over time. 									  											  									  									  									  								  								  								  	Erida Gjini								  	 								  		Instituto Gulbenkian de Ciência								  									  									  										  			&lt;/br&gt; A critical transition in N-strain co-colonization dynamics 								  										  									  									  										  										  											  			Interacting systems with multiple strains generate epidemiological, ecological and evolutionary dynam- ics. These dynamics are typically hard to analyze, especially for high number of strains and population structure. Diversity in interaction traits enables the strains to create dynamically their niches for growth and persistence, ‘engineer’ and respond to their common environment. How such a network of interactions with others mediates collective coexistence remains hard to understand, and integrate with intervention effects such as drugs and vaccines. Furthermore, the gradients shaping stability and complexity in such systems remain poorly understood. In a mathematical study, we present a new analytic framework for an N-strain SIS epidemiological system with altered susceptibilities to co-colonization/co-infection between strains. We map the multi- strain SIS dynamics to a replicator equation for N frequencies using separation of timescales. This framework enables explicit examination of the key drivers of competition and coexistence regimes in such a system. We find the ratio of single to co-colonization μ critically determines the type of equilibrium and number of coexisting strains. This key quantity in the model encodes a trade-off between overall transmission intensity R0 and mean interaction coefficient in strain space k, and links our model with the stress-gradient hypothesis (SGH) in ecology. I will show how this co-colonization model provides fresh insights for understanding critical transitions in community dynamics potentiated by mean-field and environmental gradients. 									  											  									  									  									  								  								  								  	Christel Kamp								  	 								  		Paul-Ehrlich-Institut								  									  									  										  			&lt;/br&gt; Kinetic fingerprint of bacterial depletion indicates phage synergy 								  										  									  									  										  										  											  			Phages are viruses that infect bacteria which leads to complex dynamics depending on their specific, potentially changing life styles: Lysogenic phages integrate into bacterial genomes and propagate through bacterial replication. Lytic phages replicate within their host cells and destroy them during phage release making them highly specific anti-microbial agents. Bacterial and phage population dynamics do not only depend on life styles but also on the details of the infection process of a specific bacterium and corresponding phage: Some phages can enter a bacterium at multiple sites whereas others are restricted to a single or few entry points. Bacteria further defend against phage induced lysis by various mechanisms including adaptive CRISPR-Cas immunity. Generally, the interaction between lytic phages with their corresponding susceptible bacteria is followed by rapid emergence of bacterial resistance against these phages. The time scales for the emergence (and sustainment) of resistance as well as the specific temporal evolution of bacterial and phage population sizes depend on the characteristics both of the specific phage and corresponding bacterial strain. This pattern can be seen as a kinetic fingerprint which gives insights into the underlying dynamics including binding kinetics and mechanisms of bacterial resistance and phage evasion. In studying the interaction between Klebsiella pneumoniae and its corresponding phage we see enhanced inter-phage synergy leading to faster depletion of bacterial populations than expected from simple mass action kinetics. Within our modelling framework we discuss potential underlying mechanisms of phage binding and synergy as well as the relevance of the kinetic fingerprint in characterizing the interaction between bacteria and phages. 									  											  									  									  								  								  								  									  	David Versluis								  	 								  		Leiden University								  									  									  										  			&lt;/br&gt; Multiscale Modelling of the Colonic Microbiota in Infants 								  										  									  									  										  										  											  			Nearly immediately after birth, a complex and dynamic ecosystem forms in the human gastrointestinal tract. The characteristics of this system influence the infants health in both the short- and long-term. It differs in striking ways from that found in adults,both in composition and in dynamics. The first few days generally feature an initial dominance of facultative anaerobic species such as Enterobacteriaceae, most often followed by a dominance of anaerobic species, particularly Bifidobacteriaceae. While the influence of oxygen in this succession is often hypothesised, there is no clear view of the impact or mechanism of this influence. We use a multi-scale spatiotemporal model of the infant colon to simulate the effects of variations in initial oxygen concentration on the composition and metabolic activity of the microbiota from birth to three weeks of age. Using flux balance analysis with molecular crowding on a consortium of genome-scale metabolic models from the AGORA project, we calculate species-specific bacterial fluxes for different locations and time points at a high resolution. The resulting fluxes are integrated together into a model of the ecosystem that feeds back into the flux calculations [3]. The model takes into account the nutrition and development of the infant, and can so give insight and produce predictions for the composition and metabolite formation of the infant microbiota over time and under different conditions. We find that the initial presence of oxygen can explain the specific early dominance of Enterobacteriaceae and the succession by Bifidobacteriaceae out of a broad consortium of infant microbiota species. This is derived solely from the genome-derived differences in oxygen metabolism between species, without having to take into account oxygen toxicity. We also show a complex network of spatial separation and metabolic interactions emerging within the infant gut. Our general aim is to reach a deeper understanding of the major metabolic influences, such as prebiotics and nutrition as a whole, on the development of the infant microbiota. This in turn is the first step towards a more comprehensive understanding of the formation of a steady state adult GI-tract microbiota. This research was financially supported by Friesland Campina. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  			             		    		  									                15 minute break (11:00-11:15am)                					  		    		      		        Subgroup Keynote		        11:15am: 		        David Ho, &lt;/br&gt;Columbia University 		        Immunobiology &amp; Infection Subgroup 				      		      																     				       					       					     		             		    		    				   				    Potent Neutralizing Monoclonal Antibodies Directed to Multiple Epitopes on the SARS-CoV-2 Spike 				     					    The SARS-CoV-2 pandemic rages on with devasting consequences on human lives and the global economy.  The discovery and development of virus-neutralizing monoclonal antibodies could be one approach to treat or prevent infection by this novel coronavirus.  Here we report the isolation of 61 SARS-CoV-2-neutralizing monoclonal antibodies from 5 infected patients hospitalized with severe disease.  Among these are 19 antibodies that potently neutralized the authentic SARS-CoV-2 in vitro, 9 of which exhibited exquisite potency, with 50% virus-inhibitory concentrations of 0. 7 to 9 ng/mL.  Epitope mapping showed this collection of 19 antibodies to be about equally divided between those directed to the receptor-binding domain (RBD) and those to the N-terminal domain (NTD), indicating that both of these regions at the top of the viral spike are quite immunogenic.  In addition, two other powerful neutralizing antibodies recognized quaternary epitopes that are overlapping with the domains at the top of the spike.  Cyro-electron microscopy structures of one antibody targeting RBD, a second targeting NTD, and a third bridging two RBDs revealed recognition of the closed, “all RBD-down” conformation of the spike.  Several of these monoclonal antibodies are promising candidates for clinical development as potential therapeutic and/or prophylactic agents against SARS-CoV-2. 				     			    			   		    		    		  											  		    		      		        Subgroup Keynote		        		        12:15pm: 		        Adam Martin, &lt;/br&gt;Massachusetts Institute of Technology, @martinlabmit 		        Cell &amp; Developmental Biology Subgroup 				      		      																     				       					       					     		             		    		    		    		    				   				    Folding tissues across length scales: Cell-based origami 				     					    Throughout the lifespan of an organism, tissues are remodeled to shape organs and organisms and to maintain tissue integrity and homeostasis. Apical constriction is a ubiquitous cell shape change of epithelial tissues that promotes epithelia folding and cell/tissue invagination in a variety of contexts.  Apical constriction promotes tissue bending by changing the shape of constituent cells from a columnar-shape to a wedge-shape.  Drosophila gastrulation is one of the classic examples of apical constriction, where cells constrict to fold the primitive epithelial sheet and internalize cells that will give rise to internal organs. We have used a combination of imaging, experimental perturbation, and modeling, to determine how actomyosin organization promotes tissue folding.  The actin cytoskeleton is organized in both time and space to facilitate apical constriction.  We found that actomyosin contraction is pulsatile and requires dynamic regulation of upstream signaling processes. In addition, we found that actomyosin becomes organized into oriented fibers, which generates anisotropic tension that is critical for tissue shape.  Furthermore, connectivity within the network of actomyosin fibers is highly redundant, promoting the robustness of folding. 				     			    			   		    		    		  											  		    		      		        Career Fair		             		    		  											  		    		      		        Sub-group contributed talks (1:30-3:30pm)		        			  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	CDEV: Integrating cell mechanics and cell mechanosensing (1:30-3:30pm)													 						  			  				  							  							  										  									  	Ulrich Schwarz								  	 								  		University of Heidelberg								  									  									  										  			&lt;/br&gt; Emergence of elasticity in cell mechanics 								  										  									  									  										  										  											  			Biological cells adapt shape and behavior to their mechanical environment through an intricate system of mechanotransduction processes. While in highly dynamic situations like development, cell migration or wound healing, cells need to be soft and viscous, in homeostatic situations like maintenance of connective or epithelial tissues, they have to keep stresses and strains for long times, like an elastic material. Because the actomyosin cytoskeleton determining cell mechanics is in a state of constant turnover, achieving effectively elastic behavior is a real challenge for cells. In this talk, I will first discuss the experimental evidence for the elastic behavior of cells, including the invaginated shapes of single cells in connective tissue and the elastic nature of epithelial monolayers as revealed by traction force and monolayer stress microscopy. I then will discuss different mechanisms which allow this elastic behavior to emerge on cellular scales, including the generation of stress fibers, the exchange dynamics of fast and slow myosin motors in myosin minifilaments and their regulation through the Rho-pathway, which can be controlled e. g. by optogenetics. In each case, I will address the fundamental question how multi-scale modeling and homogenization techniques can be used to mathematically connect the microscopic and macroscopic scales. 									  											  									  									  									  								  								  								  	Fabian Spill								  	 								  		University of Birmingham								  									  									  										  			&lt;/br&gt; Role of physical and geometrical drivers of tumour metastasis 								  										  									  									  										  										  											  			Cancer mostly kills through metastasis - the process where cancer cells leave the primary tumour and colonialize distant organs. Such movement of cells naturally requires forces. How cells generate forces through molecular pathways is thus an intense field of study. Moreover, it is becoming increasingly appreciated that forces, and other physical properties, not only arise from intracellular pathways, but can also reprogram pathways, for instance, through molecular mechanosensors. I will present recent results obtained from mathematical modelling and experimental work aimed at investigating the interplay of physical, geometrical and molecular drivers of tumour progression. At the example of an endothelial cell monolayer, I will show how forces alter the chemical binding rates of cell-cell adhesions, which consequently can lead to gaps in the monolayer. Experiments show that these gaps can be exploited by cancer cells when transmigrating through the endothelial monolayer - a crucial process during metastasis. I will then present some work on modelling mechano-chemical pathways in cells. Pathways including the YAP/TAZ and Rho signalling pathways are sensitive e. g. to extrinsic factors such as extracellular stiffness, or intrinsic factors such as cellular geometry. The complex interplay of such factors can lead to reprogramming of cells. Consequently, a physically altered tumour microenvironment can activate intracellular pathways and, independent or complementary to genetic changes, alter tumour cells towards more aggressive behaviour. 									  											  									  									  									  								  								  								  	Carina Dunlop								  	 								  		University of Surrey								  									  									  										  			&lt;/br&gt; Cytoskeletal contractility in mechanosensing 								  										  									  									  										  										  											  			Cells have been demonstrated to be extremely sensitive to the physical properties of their external environments, changing behaviours as diverse as proliferation, differentiation and migration in response. The mechanism by which this mechanosensing is achieved is broadly understood. Molecular motors em- bedded within the cellular cytoskeletal network bring the network into tension - this contraction is resisted by the cellular adhesion to the external environment with the degree of resistance thus broadly signalling information about the local external stiffness. Great strides have been made in understanding the molec- ular signalling mechanisms mechanisms with research focusing on the focal adhesions, supramolecular adhesive patches, and on pathways such as YAP/TAZ. However, there is emerging evidence of signalling away from focal adhesions including at the nuclear envelope (nuclear mechanotransduction). This necessitates a spatially resolved model of cell stiffness sensing that incorporates intracellular mechani- cal interactions. Here I will present a continuum elasticity model of cellular contractility and stiffness sensing that accounts for the experimentally observed variations in contractile activity within cells. This model demonstrates how on stiff substrates non-uniform contraction will lead to localised regions of internal cell stretch well away from focal adhesions. Additionally where the cell shape is elongated (as is often observed on stiff substrates) this effect is enhanced with increased internal stretch as compared with isotropic cells. These internal strains offer a potential physical mechanism linking physical exter- nal substrate stiffness to stretch activated molecular signalling within the cell and at the nuclear envelope. 									  											  									  									  									  								  								  								  	Benedikt Sabass								  	 								  		Forschungszentrum Juelich								  									  									  										  			&lt;/br&gt; Substrate durokinesis of the bacterium P. aeruginosa 								  										  									  									  										  										  											  			Bacteria can generate mechanical forces that are important for the colonization of surfaces, forma- tion of biofilms, and infection of host cells. In Gram-negative bacterial pathogens, such as Pseudomonas aeruginosa, forces result from ATP-hydrolysis-driven extension-retraction cycles of extracellular filaments called type-IV pili. How bacteria adapt their pilus-based behavior to the mechanical environment is not known. Here, we show that the early stage of surface colonization by P. aeruginosa is modulated by substrate-dependent pilus activity. Our experimental data reveals a complex response of the bacterial migration machinery to substrate properties, including adaptation of the dynamics of pili, their spatial arrangement, and their number. The combination of the experimental data with mathematical modeling allows us to construct a comprehensive picture of the stochastic dynamics of pilus-based motion of P. aeruginosa on different substrates. Overall, our findings reveal how mechanoregulation is involved in essential aspects of the migratory behavior of a paradigmatic bacterial pathogen. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	IMMU: Subgroup Contributed Talks (1:30-3:30pm)													 						  			  				  							  							  										  									  	Sahak Makaryan								  	 								  		University of Southern California								  									  									  										  			&lt;/br&gt; In Silico Control and Optimization of Granzyme B and Perforin-1 Secretion in Natural Killer Cells 								  										  									  									  										  										  											  			Natural killer (NK) cells are immune effector cells that can detect and lyse cancer cells. NK cell exhaustion, a phenotype characterized by reduced effector functionality, can limit the NK cell’s capacity for cell lysis. The processes mediating NK cell exhaustion are many, unfortunately, rendering a single preventative approach unlikely to be optimal in all cases. In lieu of prevention, we investigated in silico whether the effects of exhaustion can be nullified by strategies that maximize the continuous secretion of effector molecules. Here, we constructed a system of nonlinear ordinary differential equations (ODEs) that describes the dynamics of the cytolytic molecules granzyme B (GZMB) and perforin-1 (PRF1). The model predictions were calibrated to published, experimental data where the model parameters were estimated via the Metropolis-Hastings algorithm. Furthermore, the model was interrogated using an information-theoretic global sensitivity analysis, to determine which model parameters (i. e. , inputs) shared a significant degree of mutual information with the secreted amount of effector molecules (i. e. , outputs). Interestingly, we found the inhibition of phosphatase activity maximizes the secretion of GZMB and PRF1. We appended the baseline model with a system of ODEs describing a synthetic Notch (synNotch) signaling circuit as a method for controlling the production and secretion of the cytolytic molecules. Briefly, the synNotch receptor is chimeric molecule consisting of a target-ligand specific single-chain variable fragment (scFv) in its extracellular domain, a string of cleavable amino acid sequences in its transmembrane domain and a transcription factor in its intracellular domain. Once bound to a target ligand, the synNotch receptor is cleaved by membrane proteases. This unchains the transcription factor from the cell membrane, and thereby freeing the molecule to initiate gene expression by binding to a plasmid. We included two separate plasmids in the model: (1) a multi-cistronic plasmid coding for the effector molecules and (2) a plasmid coding for a long-noncoding RNA (lncRNA) molecule that binds and sequesters the phosphatase from inhibiting signal transduction. The synNotch model was optimized by determining the optimal quantity of plasmids and synNotch receptor to maximize the secretion of the effector molecules while using the minimum amount of material. We found the optimal synNotch system depends on the frequency of NK cell stimulation: for fewer rounds of stimulation, both plasmids should be given at maximal dose; for many rounds of stimulation, the model predicts only the cytolytic molecule-coding plasmid should be given and at maximal value. This suggests that inhibition of phosphatase activity, while beneficial in the short-term, is not optimal for multiple rounds of stimulation. In total, we developed a theoretical framework that provides actionable insight into engineering robust NK cells for clinical applications. 									  											  									  									  									  								  								  								  	Naveen Vaidya								  	 								  		San Diego State University								  									  									  										  			&lt;/br&gt; HIV Infection and Antiretroviral Therapy: the Brain as a Reservoir 								  										  									  									  										  										  											  			It is not fully understood whether the brain acts as an HIV reservoir causing obstacle to cure through treatment. In this talk, I will present a novel mathematical model describing virus dynamics under antiretroviral therapy to study the role of the brain in virus persistence. Using experimental data from SIV infected macaques, we identify key parameters related to the brain infection, including virus-transfer across blood-brain barrier. Our model predicts that the brain can be an important reservoir causing long-term virus persistence in the brain, despite successful control of viral load in the plasma by antiretroviral drugs. 									  											  									  									  									  								  								  								  	Cristina Leon								  	 								  		RUDN University, Russia								  									  									  										  			&lt;/br&gt; Reaction-diffusion model of viruses coexistence in the space of genotypes 								  										  									  									  										  										  											  			We propose a mathematical model describing the competition of two viruses, in the host organism, taking into account virus mutation, reproduction, and genotype dependent mortality, either natural or determined by an antiviral treatment. The model describes the virus density distribution u(x; t) for the first virus and v(y; t) for the second one as functions of genotypes x and y considered as continuous variables and of time t. The model consists of a system of reaction-diffusion equations with integral terms characterizing virus competition for host cells. The analysis of the model shows the conditions of virus coexistence or elimination in the host organism. This study continues the cycle of works devoted to reaction-diffusion models of virus mutation and evolution. 									  											  									  									  									  								  								  								  	Mohammad Aminul Islam								  	 								  		Oklahoma State University								  									  									  										  			&lt;/br&gt; Computational modeling of the gut-bone axis and implications of butyrate treatment on osteoimmunology 								  										  									  									  										  										  											  			The interplay between gut microbiota and the immune system has a pivotal role in the maintenance of bone health.  Recently, short-chain fatty acids (SCFAs) produced by gut microbiota have emerged as key regulatory participants in shaping the immune system.  Butyrate, the most versatile among SCFAs, has been observed to have local and systemic effects including inducing the differentiation of peripheral regulatory T cells (Tregs) in the intestine, blood, and bone marrow [1].  Tregs are the central actors of the negative feedback component of the immune system.  The interaction between Tregs and cytotoxic CD8+ T cells suppress the inflammatory status and promote the production of Wnt10b to increase bone anabolism [2].  Studies in the mouse models show that the ablation of butyrate in the intestine alters the bone marrow density.  However, the therapeutic benefit of butyrate in bone anabolism remains poorly understood. We developed a multi-compartment physiologically based pharmacokinetic model to track and quantify the effects of butyrate on Tregs in the gut, blood, and bone marrow.  The model consists of five species butyrate, naive CD4+ T cells, Tregs, TGF-β, and Wnt10b distributed across three compartments intestine, blood, and bone.  We consider an open system with the processes of formation, excretion, differentiation, cell death, and migration to another compartment.  The variation of butyrate concentration from homeostasis value changes the percentage of Tregs, and the production of TGF-β, and Wnt10b.  Using the model, we analyze experimental data reported in [2] to evaluate the expansion of Tregs, TGF-β, and Wnt10b in the bone marrow.  The probiotic LGG increases butyrate concentration in the intestine and serum blood by 0. 18 μM and 0. 29 μM respectively.  Our simulation result shows 5% increase of Tregs, 3. 4-fold increase of TGF-β, and 3-fold increase of Wnt10b in the bone marrow consistent with the net change information due to stimulus of a probiotic microbiota in the gut.  The computational approach described here gives insight into the pharmacokinetics of butyrate, biodistribution of Tregs in the gut-bone axis, fold changes of Wnt10b in bone marrow, and their contributions to modulating bone formation. Research reported in this abstract was supported by NIH NIGMS R35GM133763. 									  											  									  									  								  								  								  									  	Georges Ferdinand Randriafanomezantsoa Radohery								  	 								  		Kirby Institute, University of New South Wales, Sydney								  									  									  										  			&lt;/br&gt; Incorporating parasite viability data into PK/PD modelling of artemisinin treatment of human malaria 								  										  									  									  										  										  											  			Malaria is a global health threat killing one child every two minutes. The rise of artemisinin resistance, which is the main component of most recommended anti-malarial regimens, has prompted the need to develop better alternative antimalarial treatments. Modelling plays an important role in the development of these new antimalarial drugs. Pharmacokinetic/Pharmacodynamic (PK/PD) models of antimalarial treatment, which relate drug concentration to parasite killing, are central in assessing and optimizing drug therapy effectiveness. A common assumption in these models is that the parasites that remain in circulation after treatment are all viable. Recently a method was developed to estimate the concentration of viable parasites after artesunate treatment, rather than simply total parasite numbers. Here, we aimed to include this additional parasite viability information into an adapted PK/PD model to more accurately estimate the drug killing rate in sensitive and resistant infections. We use data from volunteers infected with artemisinin-sensitive or resistant P. falciparum blood-stage parasites and treated with a single dose of oral artesunate monotherapy during a volunteer infection study. Parasite qPCR counts before and after treatment, the proportion of viable parasites after treatment and dihydroartemisinin plasma concentration were obtained and modelled using age-structured P. falciparum parasite population growth and a PK/PD model. We use these models for separately measuring parasite killing rate and parasite clearance rate. The addition of viability data into PK/PD model of artesunate monotherapy provided a good fit of the data. The model estimated an in vivo DHA’s EC50 of 1. 42 μg/L (95% CI: 0. 07, 2. 78). In artemisinin-sensitive infections, the parasites mean killing half-life is 0. 20 h (95% CI: 0. 16 h,0. 26 h) and the mean removal half-life of dead parasites was 2. 77 h (95% CI: 2. 4 h, 3. 4 h). Further, we showed that the reduced killing observed in resistant infections were consistent with a 9-fold reduction in the sensitivity of parasites in the first 12 h of the lifecycle, and drug-killed resistant parasites were estimated to be removed slower than drug-killed sensitive parasites. Incorporating parasite viability data into PK/PD models allows refining the estimation of parasite killing rates and provides information on the stage-specific activity of antimalarial drugs in vivo. Killing half-life is faster than previously thought. These differences are likely to have important implications for optimal dosing strategies and predicting overall drug efficacy. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	MEPI: Subgroup Contributed Talks (1:30-3:30pm)													 						  			  				  							  							  										  									  	(Speaker cancellation)								  	 								  		N/A								  									  									  										  			&lt;/br&gt; (Speaker cancellation) 								  										  									  									  										  										  											  			(Speaker cancellation)									  											  									  									  									  								  								  								  	Folashade Agusto								  	 								  		University of Kansas								  									  									  										  			&lt;/br&gt; To isolate or not to isolate: The impact of changing behavior on COVID-19 transmission 								  										  									  									  										  										  											  			In this talk I will present a model developed for COVID-19 using a system of ordinary differential equation following the natural history of the infection. Using appropriate payoff functions relating to the perception of risk measured using disease incidence and severity of infection the model is coupled to a series of human behaviors including ignoring social distancing, and breaking out of isolation and quarantine. Analysis and simulations of the model show the possibility of multiple waves of infection. Discouraging the population from disease magnifying behavior such as escaping from isolation and quarantine eliminates the multiple waves of infection and greatly flattens the curves. 									  											  									  									  									  								  								  								  	Alexander Beams								  	 								  		University of Utah								  									  									  										  			&lt;/br&gt; Mask, or JASC? What are the conditions under which SARS-CoV-2 becomes Just Another Seasonal Coronavirus? 								  										  									  									  										  										  											  			Background: Will SARS-CoV-2 become Just Another Seasonal Coronavirus? It bears some important resemblances with its more benign cold-causing relatives, after all. For example, Coronavirus NL63 uses the same ACE2 receptor to gain entry into cells, and circumstantial evidence suggests Coronavirus OC43 may have caused a pandemic in the late 1800's. Both are now just 'common colds'. Is it possible that SARS-CoV-2 will take the same path? Hypotheses: We are writing models to address how three factors might push SARS-CoV-2 towards JASC. First, evidence suggests asymptomatic cases tend to shed less virus. If viral dose affects disease severity, and vice versa, selection might act to alleviate virulence. Second, we know that kids are far less likely to experience more severe forms of COVID-19, and different disease outcomes in kids vs adults will influence the epidemiology in important ways. Third, the JASC hypothesis might be sensitive to the duration of immunity to SARS-CoV-2. Eventually we hope to consider cross-immunity or interaction with other viruses, including seasonal coronaviruses, with a view to understand the COVID-19 pandemic in the context of the pre-existing respiratory virome. Results: Our models show that JASC is possible if viral dose correlates with disease severity and if immunity is sufficiently strong and long-lasting. We will present analyses of how our three factors influence the long-term outcomes of the pandemic. Conclusions: Although it is too early to say for certain, it is possible that SARS-CoV-2 could persist in humans as another cold-causing virus. 									  											  									  									  									  								  								  								  	Juan Gutierrez								  	 								  		University of Texas at San Antonio								  									  									  										  			&lt;/br&gt; Modeling COVID-19 Under Lockdown Conditions 								  										  									  									  										  										  											  			Coronavirus disease 2019 (COVID-19) is a novel human respiratory disease caused by the SARS-CoV-2 virus. Asymptomatic carriers of the virus display no clinical symptoms but are known to be contagious. Recent evidence reveals that this sub-population, as well as persons with mild symptoms, are major contributors in the propagation of COVID-19, a first for a respiratory virus. In another first, COVID-19 caused generalized restrictions to human movement and interactions. In this talk, we will discuss a traditional compartmentalized mathematical model taking into account asymptomatic carriers and lock-down conditions. The theoretical model is used to calibrate a model for the City of San Antonio, Texas. 									  											  									  									  								  								  								  									  	Diego Volpatto								  	 								  		Laboratorio Nacional de Computacao Cientifica								  									  									  										  			&lt;/br&gt; A Bayesian approach to assess the spread of COVID-19 using an extended SEIRD model with implicit quarantine mechanism: applications in Brazilian locations 								  										  									  									  										  										  											  			In this work, we develop a generalized SEIRD model that implicitly takes into account the quarantine mechanism to describe the spread of COVID-19 with applications in Brazil. We assume uncertain scenarios with limited testing capacity, lack of reliable data, under-reporting of cases, and restricted testing policy. To deal with data and model uncertainties, we developed a Bayesian framework for the identification of model parameters. A global sensitivity analysis is performed beforehand to identify the most significant parameters on either the cumulative numbers of confirmed and death cases, or the effective reproduction number. Less important parameter values are set according to the current knowledge on the disease in order to overcome the bottleneck of parameter identifiability. We show that the model parameter related to social distancing measures is one of the most influential along all stages of the disease spread and the most influential after the infection peak. Different relaxation strategies of social distancing measures are then investigated in order to determine which strategies are viable and less hazardous to the population. The results highlight the need of keeping social distancing policies to control the disease spread. Specifically, the considered scenario of abrupt social distancing relaxation implemented after the occurrence of the peak of positively diagnosed cases can prolong the epidemic, with a significant increase of the projected numbers of confirmed and death cases. An even worse scenario could occur if the quarantine relaxation policy is implemented before evidence of the epidemiological control, indicating the importance of the proper choice of when to start relaxing social distancing measures. The employed approach and subsequent analysis applied over the Brazilian scenarios may be used to other locations. 									  											  									  									  								  								  								  									  	Julie Spencer								  	 								  		Los Alamos National Lab								  									  									  										  			&lt;/br&gt; Prioritizing Mitigation Strategies for COVID-19 in New Mexico 								  										  									  									  										  										  											  			As of July 12, 2020, the novel zoonotic virus SARS-CoV-2 caused 15,028 confirmed infections and 545 deaths in the State of New Mexico. New Mexico ranks as the third highest state in the United States for per capita testing, and the time from identification of a traced contact to quarantine is two days; however, during the past seven days, cases have been increasing by 1. 8% per day, leaving open the question of how best to intervene in the epidemic, given limited resources. Recent modeling studies have addressed mitigation strategies for COVID-19; however, there is a need for an age-structured mitigation model that provides pre-symptomatic, asymptomatic, symptomatic, testing, and quarantine compartments. We developed a deterministic, Susceptible-Exposed-Infected-Recovered (SEIR) model to assess the merits of a range of non-pharmaceutical intervention measures. We simulated all combinations of three social distancing levels, six testing levels, three testing turnaround speeds, and four testing accuracy levels, in order to evaluate 216 mitigation scenarios. We found that social distancing and testing are both necessary for decreasing total infections and for delaying the peak of the epidemic. We additionally found that increasing the turnaround speed of test results and decreasing the proportion of false negative tests has the potential to result in 27% fewer infections and 33% fewer deaths over the course of a two-year simulated epidemic. The epidemic outcomes are mitigated more effectively when school-aged individuals have less contact, as when schools are closed or operating virtually, than when working-aged individuals have less contact, as when businesses are closed or operating virtually. These are difficult but important prioritizations. UNCLASSIFIED LA-UR-20-25199									  											  									  									  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	MFBM: Subgroup Contributed Talks (1:30-3:30pm)													 						  			  				  							  							  										  									  	David		Hardman								  	 								  		University of Edinburgh,	Scotland,	david. hardman@ed. ac. uk									  									  									  										  			&lt;/br&gt; Optimising muscle cell co-culturing protocol through a combined in vitro-agent based modelling approach 								  										  									  									  										  										  											  			The Myochip project is a prospective organ-on-a-chip platform co-culturing neurons, muscle cells and endothelial cells to create functional skeletal muscle. Cell culturing involves a wide range of variables and optimal protocols are currently undefined when co-culturing different cell types in vitro. A ‘trial and error’ based experimental approach to optimisation is inefficient and costly and relies on animal-derived in vitro models.  We propose a combined and iterative in silico} / in vitro approach to optimising experimental cell culturing protocols using a small number of experiments to validate parametric mathematical models which can in turn be used to predict optimum conditions. Here, we focus on the trade-off between the cell differentiation media required to allow co-culturing of muscle fibres with neurons and optimal muscle cell growth. During myogenesis, myoblast cells fuse together to create multinucleated myotubes which elongate and mature into muscle fibres.  Fixed and time-lapse images of muscle cells were acquired throughout the differentiation and early myotube formation phases for cells cultured in muscle differentiation medium, neuron differentiation medium and a 1:1 mixture of both media. Metrics of myoblast migration speed, migration direction and rate of fusion were quantified from time-lapse imaging and used to inform a mathematical agent-based model (ABM) of myoblast motion and fusion. Metrics of myotube growth and alignment over time were quantified and used to validate the ABM output. The validated ABM can then be used to conduct virtual trials in different media conditions in order to ascertain the optimum balance between muscle growth and specialisation of differentiation medium. 									  											  									  									  									  								  								  								  	Doris		Schittenhelm								  	 								  		Friedrich-Alexander-Universität Erlangen-Nürnberg	Germany	doris. ds. schittenhelm@fau. de									  									  									  										  			&lt;/br&gt; A Bayesian framework for parameter estimation from fluorescence data 								  										  									  									  										  										  											  			Measurements of fluorescence intensity are a good way of monitoring time-dependent cellular processes, and can be used for estimating parameter of such models. In addition to uncertainty from modelling, experimental set-up and measurement, there is another source of error that needs to be factored in: Crosstalk denotes the interaction of neighboured samples during measurement and cannot be ignored in fluorescence intensity measurement scenarios. In order to quantify the indispensability of crosstalk, we formulate two models for the measurement process: one without crosstalk and one with crosstalk. These models contain a high number of unknowns due to properties of the measurement instrument and of the experimental set-up. Our goal is to identify the most plausible model from given data and assumed measurement uncertainties and use it for parameter estimation afterwards. For this we employ a Bayesian model selection approach where we compute the evidence for either model with the nested sampling algorithm, which is suited for high-dimensional problems. This allows us to simultaneously pick the most favourable model and obtain parameter samples from the posterior. 									  											  									  									  									  								  								  								  	Endre	T	Somogyi								  	 								  		Indiana University Bloomington,	United States,	andy. somogyi@gmail. com									  									  									  										  			&lt;/br&gt; Real-Time Interactive, Scriptable 3D Simulation of Cell / Virus-Like-Particle Endocytosis With Tellurium / Mechanica 								  										  									  									  										  										  											  			Endocytosis / Exocytosis is one of the more common mechanisms a biological cell uses to transport materials through the cell membrane. Understanding biological responses to cell-membrane / virus-like-particles is a key to developing potential therapeutics and vaccines to viruses. Endocytosis is a cellular mechanism where a cell internalizes substances from the external environment. In endocytosis, external objects such as nanoparticles, virus-like particles (VLP), or chemicals adhere to the cell surface. The cell then envelopes these adsorbed particles typically by wrapping a portion of its own cell membrane around these particles. The adsorbed material then becomes trapped inside the vesicle, effectively becoming a ‘cargo’ or ‘payload’, and the cell transports this vesicle into the cell. In endocytosis, external objects can either adhere directly to the cell membrane with a certain affinity, or they can bind to explicit membrane receptors. There are many unanswered scientific questions when we are trying to better understand the endocytosis process.  The realism, the amount of physical detail required in a ‘useful’ model, can differ widely and very much depends on what kinds of questions we ask. For example, do we need to include individual lipids?, explicit individual atoms? Or is it sufficient to have a more coarse-grained model that treats individual VLPs as discrete particles? We are developing tools that will give users the flexibility they need to create multi-scale biophysics models that include different (and therefore more appropriate) levels of physical detail and realism.  We present a new particle-based simulation environment, Mechanica, that enables users to interactively create, manipulate and simulate models of biological cells and tissues using physics-motivated python API. Using Mechanica, researchers can explore the kinds of information and ask the questions required to model accurately, say, at a level that sits between coarse-grained molecular dynamics and cell-center models. We present a composite particle based simulation approach where we augment the traditional cell-center-model type cells with explicit surface receptor binding sites, and model explicit external VLP particles, where we have explicit surface receptors that can diffuse about the cell surface, and explicitly bind to external VLPs. We allow VLPs to explicitly bind to surface receptors, and we demonstrate our event model, where users can bind cellular responses such as particle adsorption to events.  Our user model description API is based on physical and chemical abstractions which enables a great deal of freedom for our users in the kinds of semantic meaning they ascribe to physical objects. For example, a user could choose to allow one of our ‘particles’ to represent a molecule, nano-particle, cellular organelle, or even a complete cell. 									  											  									  									  									  								  								  								  	Patrick	S. 	Eastham									  	 								  		Florida State University,	United States,	peastham@math. fsu. edu									  									  									  										  			&lt;/br&gt; A framework for simulating precipitate reactions in microfluidic devices 								  										  									  									  										  										  											  			Chemical processes within flows are ubiquitous. There exists an important class of reactions that result in a phase change from liquid to solid: precipitation reactions. Inspired by recent microfluidic experiments, this talk describes a novel mathematical framework for handling such reactions occurring within a slow-moving fluid flow. A key challenge for precipitate reactions is that, in general, the location of the developed solid is unknown a priori. To model this situation, we use a multiphase framework with fluid and solid phases; the aqueous chemicals exist as scalar fields that react within the fluid to induce phase change. To demonstrate the functionality of this framework, we conduct full-scale simulations in a realistic microfluidic geometry. The framework can be applied to precipitate reactions where the precipitate greatly affects the surrounding flow, a situation appearing in many laboratory and geophysical contexts including the hydrothermal vent theory for the origin of life. More generally, this model can be used to address low Reynolds number fluid–structure interaction problems that feature the dynamic generation of solids. 									  											  									  									  								  								  								  									  	Anastasios		Siokis								  	 								  		Helmholtz Centre for Infection Research,	Germany,	anastasiossio@gmail. com								  									  									  										  			&lt;/br&gt; An agent-based simulation platform studying the immunological synapse dynamics 								  										  									  									  										  										  											  			During immunological synapse (IS) formation, T cell receptor (TCR) signaling complexes, integrins, as well as costimulatory and inhibitory molecules exhibit characteristic spatial localization. The IS is built around a TCR-peptide-major histocompatibility complex (pMHC) core, and is surrounded by an integrin ring (Monks, et al. , 1998). Small immunoglobulin superfamily (sIGSF) adhesion complexes form a corolla of microdomains outside the integrin ring, which is shown to recruit and retain the major costimulatory and checkpoint complexes that regulate the responses to TCR engagement (Demetriou, et al. , 2019). The positioning of these molecules drives T cell signaling and fate decision, making forces that govern IS formation of particular interest. To gain insights into the mechanisms underlying molecular reorganization and characteristic pattern formation during IS formation, we developed a general agent-based simulation platform able to test different hypotheses. The simulations suggest the following: 1. A radial gradient of integrin complexes (LFA-1-ICAM-1) in the peripheral supramolecular activation cluster (pSMAC) toward the central SMAC (cSMAC) emerged as a combined consequence of actin binding and diffusion and modified the positioning of other molecules (Siokis, et al. , 2018). 2. The costimulatory complexes CD28-CD80 passively follow the TCR-pMHC microcluster centripetal movement, however their characteristic localization in a ring-like pattern around the cSMAC only emerges with a particular CD28-actin coupling strength that induces a centripetal motion (Yokosuka, et al. , 2008; Siokis, et al. , 2018). 3. sIGSF complexes are passively excluded to the distal aspect of the IS provided their interactions with the ramified F-actin transport network are sufficiently weak (Siokis, et al. , 2018; Siokis, et al. , 2020). 4. Attractive forces between sIGSF adhesion (such as CD2-CD58) and costimulatory complexes (such as CD28-CD80) relocate the latter from the IS-centre to the corolla (Siokis, et al. , 2020). 5. Size-based sorting interactions with large glycocalyx components, such as CD45, explain the sIGSF adhesion corolla `petals' (Siokis, et al. , 2020). 6. A short-range self-attraction between sIGSF complexes explain the corolla `petals' (Siokis, et al. , 2020). This establishes a general simulation framework that can recapitulate complex pattern formation processes observed in cell-bilayer and cell-cell interfaces. The presented results have implications for the understanding of T cell activation and fate decision. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				 		  																																		 	ONCO: Subgroup Contributed Talks (1:30-3:30pm)													 						  			  				  							  							  										  									  	Noemi Andor								  	 								  		Moffitt Cancer Center								  									  									  										  			&lt;/br&gt; Invasion of homogeneous and polyploid populations in nutrient-limiting environments? 								  										  									  									  										  										  											  			Breast cancer progresses in a multistep process from primary tumor growth and stroma invasion to metastasis. Progression is accompanied by a switch to an invasive cell phenotype. Nutrient-limiting environments promote chemotaxis with aggressive morphologies characteristic of invasion. It is unknown how co-existing cells differ in their response to nutrient limitations and how this impacts invasion of the metapopulation as a whole. We integrate mathematical modeling with microenvironmental perturbation-data to investigate invasion in nutrient-limiting environments inhabited by one or two cancer cell subpopulations. Hereby, subpopulations are defined by their energy efficiency and chemotactic ability. We estimate the invasion-distance traveled by a homogeneous population. For heterogeneous populations, our results suggest that an imbalance between nutrient efficacy and chemotactic superiority accelerates invasion. Such imbalance will spatially segregate the two populations and only one type will dominate at the invasion front. Only if these two phenotypes are balanced do the two subpopulations compete for the same space, which decelerates invasion. We investigate ploidy as a candidate biomarker of this phenotypic heterogeneity to discern circumstances when inhibiting chemotaxis amplifies internal competition and decelerates tumor progression, from circumstances that render clinical consequences of chemotactic inhibition unfavorable. 									  											  									  									  									  								  								  								  	Parthasakha Das								  	 								  		IIEST, SHIBPUR, India								  									  									  										  			&lt;/br&gt; Optimal treatment strategies with multiple therapeutic approach in Cancer remission: A model based study 								  										  									  									  										  										  											  			In this talk, a delayed tumor-immune model is proposed and analyzed in the presence of a multi-therapeutic drug. Local dynamics of drug-free steady states are studied and Hopf-bifurcation is observed with delay bifurcation parameter. By formulating a quadratic control based functional, an optimal control problem is constructed with treatments as control variables. The formulation of the functional is aimed at minimizing the proliferation rate of tumor cells and the detrimental effects of injected drugs. Additionally, maximizing the effector cells and maintaining an attributed level of normal cells are also a priority. By applying Pontryagin’s maximum principle, the sufficient and necessary conditions of optimality system are established. The sensitivity analysis of cost functional is performed with different combinations of control variables. The cost-effectiveness analysis is carried out to determine the most cost-effective strategy. The numerical results verify analytical findings and demonstrate that a multi-therapeutic treatment protocol can alleviate tumor burden within a few months of drug administration. 									  											  									  									  									  								  								  								  	Anna K. Miller								  	 								  		H. Lee Moffitt Cancer Center, Tampa USA								  									  									  										  			&lt;/br&gt; Modeling the spatiotemporal dynamics of the vicious cycle in multiple myeloma 								  										  									  									  										  										  											  			Multiple myeloma is a cancer characterized by the expansion of plasma cells in the bone marrow and causes bone pain in over 80% of patients. Bone pain occurs as a result of the interaction between myeloma cells and the trabecular bone microenvironment. Bone is a highly dynamic tissue that is maintained in homeostasis through a balance between bone resorption and bone formation. Multiple myeloma tips the balance in favor of bone resorption, creating a “vicious cycle” in which growth factors released by bone resorption leads to increased survival of multiple myeloma cells, which in turn results in more bone destruction. Multiple myeloma is treatable but largely incurable due to the failure of treatment to completely eradicate the disease. Because the interactions between multiple myeloma and the bone microenvironment contribute to the progression of the disease, it is essential to understand the spatiotemporal dynamics of the vicious cycle to improve treatment response. To explore these dynamics, we developed a hybrid agent-based model that is integrated with published data and data generated by the Lynch lab.  We simulate the progression of myeloma growth and bone disease, starting from normal bone remodeling dynamics. To test the assumptions of the normal bone model, we perturb model parameters and show that the model is consistent with data from similar in-vivo experiments. We discuss which model assumptions and parameters are necessary to drive the vicious cycle and capture the data from an in-vivo model of multiple myeloma. This computational model provides a foundation to explore how spatiotemporal dynamics between multiple myeloma and bone microenvironment contribute to drug resistance and tumor growth which ultimately has the potential to help improve treatment strategies. 									  											  									  									  									  								  								  								  	Fabio	A. 	Milner								  	 								  		Arizona State University								  									  									  										  			&lt;/br&gt; A model for acute myeloid leukemia (AML) 								  										  									  									  										  										  											  			Idasanutlin (RG7388) is a selective MDM2 antagonist showing promising responses in phase 1 studies of relapsed AML. The drug is presently undergoing Phase I and II clinical trials. RG7388 was generally well tolerated, with GI toxicity being the most commonly reported adverse event. In laboratory cultures of MOLM-13 wild type cells it was observed that using increasing dosages of RG7388 led, within 3 months, to a complete replacement of wild type (drug-sensitive) cells by mutant (drug-resistant) cells. We propose a model for the growth of the two strains of cells in such cultures that is designed to elucidate whether the replacement is due to the RG7388 selecting for mutant type cells or rather generating the mutants the TP53 mutant clones. 									  											  									  									  								  								  								  									  	Inmaculada C. Sorribes								  	 								  		Duke University								  									  									  										  			&lt;/br&gt; Detailed quantitative framework of in silico xenografts implanted with high-grade gliomas reveals novel dosage schedule of several chemotherapeutic agents 								  										  									  									  										  										  											  			Glioblastoma multiforme (GBM) is one of the most lethal cancers, with a 5-year survival rate below 25%. GBM-associated death rates remain high, in part because the last few decades have produced only modest advances in treatment. Consequently, the standard therapy for GBM remains palliative, rather than curative, and patients ultimately die from this disease. Chemotherapy has proven to be effective against cancers in general; however, in the case of brain tumors, it has failed to produce sustained remission. Developed in collaboration with experimentalists we present a quantitative framework that captures the specific pharmacokinetics and pharmacodynamics of three different chemotherapeutic agents: temozolomide, Avastin, and vincristine. The effect of these drugs is incorporated into a simple tumor growth model and parametrized using clinical data. The resulting models are used to virtually simulate novel dosage schedules for these drugs and predict their potential in improving survival times. 									  											  									  									  								  								  								  									  	Sara Hamis								  	 								  		School of Mathematics and Statistics, University of St Andrews, St Andrews KY16 9SS, Scotland								  									  									  										  			&lt;/br&gt; Bridging in vitro and in vivo research via an agent-based modelling approach 								  										  									  									  										  										  											  			Translating quantitative information between in vitro and in vivo research remains a scientifically and financially challenging step in preclinical drug development processes. However, well-developed in silico tools can be used to facilitate this in vitro to in vivo translation, and we here propose using an agent-based model to bridge the gap between in vitro and in vivo research. In order to capture the multi-scale nature of cancer, when simulating cancer growth and treatment re- sponses, we use a multi-scale, agent-based model that links individual cell behaviour with the macroscopic behaviour of cell organisation and the microenvironment. We highlight how agent-based models, that are currently underutilised in pharmaceutical contexts, can be used in preclinical drug development and in finding optimal treatment schedules. In pursuit of hindering the onset of drug resistance in melanoma, we investigate various targeted drug dosing regimens in silico. 									  											  									  									  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	OTHE: Other Contributed Talks (1:30-3:30pm)													 						  			  				  							  							  										  									  	Eleanor Doman								  	 								  		UCL, London								  									  									  										  			&lt;/br&gt; Investigating the mechanical properties of peripheral nerve tissue using new biomechanical models 								  										  									  									  										  										  											  			It is estimated that 2. 8% of trauma patients suffer damage to the peripheral nerve system leading to potentially permanent disability. The current gold standard treatment for peripheral nerve injury, the nerve autograph, is estimated to have a success rate of 40 to 50% depending on the scale of the injury. New nerve repair options are currently being developed including the use of tissue engineered implants to replace the injured nerve. However for this to be an effective treatment, the implant must match the mechanical properties of the native tissue to prevent additional damage due to the build-up of localised stress. Peripheral nerves are complex structures with mechanical properties that vary with nerve type and along the length of each nerve. Past experimental work has inferred a link between the mechanical properties of nerve and the nano-structural make-up of the nerve. Data on the mechanical properties of living tissue is difficult to obtain, however, the data on structure of the nerve can be obtained from cadavers. Our work therefore focuses on constructing new biomechanical models that predict the mechanical properties of a nerve given structural data. To do this we use asymptotic homogenisation - an analytical technique that exploits differences in spatial scales to solve PDE systems on spatially complex domains. We will discuss our work on constructing general biomechanical models and their specific application to peripheral nerves. 									  											  									  									  									  								  								  								  	Julia Arciero								  	 								  		IUPUI								  									  									  										  			&lt;/br&gt; Modeling blood flow regulation and tissue oxygenation in the retina 								  										  									  									  										  										  											  			Glaucoma is a serious ocular disease characterized by damage to retinal ganglion cells that results in irreversible vision loss.  Impaired tissue perfusion has been identified as a significant contributing factor to glaucoma. Theoretical modeling provides a useful tool for predicting the effect of several hemodynamic factors on retinal oxygenation.  In this study, a heterogeneous model of the retinal arteriolar vascular network is used to show the impact of flow regulation on tissue oxygenation as oxygen demand is varied.  The metabolic signal (Si) is implemented as a wall-derived signal that reflects the oxygen deficit along the network, and three cases of conduction are considered: no conduction, a constant signal, and a flow-weighted signal.  The model shows that the increases in average tissue PO2 due to a flow-weighted signal are often more significant than if the entire level of signal is increased. This indicates that the heterogeneity of the downstream conducted responses serves to regulate flow better than a constant conducted response.  A hybrid model is also presented that combines a heterogeneous arteriolar network with a compartmental vascular network model for the capillaries and veins. This hybrid model combines a wall-derived conducted metabolic response with spatial data from the retinal arterial network to yield improved predictions of retinal tissue oxygenation. 									  											  									  									  									  								  								  								  	Peter Mortensen								  	 								  		Glasgow								  									  									  										  			&lt;/br&gt; Action potential propagation in a myocyte-fibroblast model of cardiac tissue 								  										  									  									  										  										  											  			Coupling electrophysiological models of myocytes and fibroblasts is key to understanding the electrical response of fibrotic regions and associated arrhythmias. There has been extensive work on the analysis of these cell-level models which have been upscaled to organ level. However, there is a lack of understanding of the effects of fibroblast coupling has on the spatial propagation of the action potentials. We identify two properties of the degree of fibrosis. Firstly, the number of fibroblasts in relation to the number of myocytes and secondly the geometry of the fibrotic region. Using direct numerical simulations of a monodomain model of fibrous cardiac tissue, we demonstrate that action potentials (APs) slow down as the severity of fibrosis is increased until eventually excitation is fully blocked. We also identify two cases of non-uniform fibroblast distribution. Here direct numerical simulations show that successful AP propagation is dependent on the geometry of the fibrous region and the number of fibroblasts per myocyte. 									  											  									  									  									  								  								  								  	Katarzyna Rejniak								  	 								  		Moffitt Cancer Center								  									  									  										  			&lt;/br&gt; In silico tools for deconvolution of complex tumor microenvironments: Organoid3D and silicoDCIS 								  										  									  									  										  										  											  			Malignant tumors are highly heterogeneous in terms of their cellular composition, varying levels of oxygenation, acidity, and nutrients, as well as local changes in the extracellular matrix. Furthermore, tumor tissue and tumor microenvironment properties can dynamically evolve not only during tumor growth but also when anticancer treatments are administered. We developed a suite of computational models to recapitulate the complexity of cancers, especially their physical and chemical microenvironments. These simulation tools were used to: derive hypotheses about the development of ductal microinvasions; formulate hypotheses on relative importance of microenvironmental factors and chemotherapeutic treatments on the growth of organoids derived from a non-tumorigenic breast cell line and its mutants; examine the interactions of tumor cell and stromal cells in co-cultures. The developed in silico tools are versatile enough to be adjusted to other organoid cultures, other tumor tissues and other components of the tumor microenvironment to generate testable hypotheses about tumor progression and response to treatments. 									  											  									  									  								  								  								  									  	Ruben Perez-Carrasco								  	 								  		Imperial								  									  									  										  			&lt;/br&gt; Effects of cell cycle variability on stochastic gene expression 								  										  									  									  										  										  											  			Many models of stochastic gene expression do not incorporate a cell cycle description. I will show how this can be tackled mathematically studying how mRNA fluctuations are influenced by DNA replication and cell cycle duration stochasticity. Results show that omitting cell cycle details can introduce significant errors in the predicted mean and variance of gene expression for prokaryotic and eukaryotic organisms, reaching 25% error in the variance for mouse fibroblasts. Furthermore, we derive a negative binomial approximation to the mRNA distribution, indicating that cell cycle stochasticity introduces similar fluctuations to bursty transcription. Finally, I will show how disregarding cell cycle stochasticity can introduce inference errors in transcription rates bigger than 10%. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  																																		 	POPD: Subgroup Contributed Talks (1:30-3:30pm)													 						  			  				  							  							  										  									  	Eva Kisdi								  	 								  		University of Helsinki								  									  									  										  			&lt;/br&gt; The evolution of habitat choice facilitates niche expansion 								  										  									  									  										  										  											  			Matching habitat choice and local adaptation are two key factors that control the distribution and diversification of species. We study their joint evolution in a structured metapopulation model with a continuous distribution of habitats. Habitat choice follows as the outcome of dispersal with non-random immigration, a process always acknowledged yet rarely incorporated into theoretical models. For fixed local adaptation, we find the evolutionarily stable habitat choice as a function linking the probability of settlement to the local environment. When the local adaptation trait co-evolves, the metapopulation can become polymorphic. Our main result shows that coexisting strains with only slightly different local adaptation traits evolve substantially different habitat choice. In turn, different habitat use selects for divergent local adaptations. We thus propose that the joint evolution of habitat choice and local adaptation can facilitate niche expansion via diversification under wide conditions, also when the local adaptation trait evolving alone would attain an ESS restricted to a narrower niche. 									  											  									  									  									  								  								  								  	Enpei Zhang								  	 								  		University of Helsinki								  									  									  										  			&lt;/br&gt; Evolution of maturation time in a stage structure prey-predator model 								  										  									  									  										  										  											  			We study the evolution of predator's maturation time. Longer maturation time resulting in stronger adult predator allows the predator to improve predation ability, reduce handling time and increase survival probability. To understand whether two predators can attain a stable coexistence via evolutionary branching of maturation time, we apply the method of critical function analysis to construct three different types of trade-off functions between maturation time with predator's capture rate, handling time and death rate, respectively. A new method to calculate the invasion fitness is applied in non-equilibrium dynamics. Evolutionary branching of maturation time is possible in the cases of capture rate and handling time. The coexistence of two predators induced by branching is evolutionarily stable. In the case of death rate, the monomorphic predator population can evolve to an evolutionarily stable strategy. 									  											  									  									  									  								  								  								  	Daniel Cooney								  	 								  		Princeton University								  									  									  										  			&lt;/br&gt; PDE Models of Multilevel Selection: The Evolution of Cooperation and the Shadow of Individual Selection 								  										  									  									  										  										  											  			Here we consider a game theoretic model of multilevel selection in which individuals compete based on their payoff and groups also compete based on the average payoff of group members. Our focus is on the Prisoners’ Dilemma: a game in which individuals are best off cheating, while groups of individuals do best when composed of many cooperators. We analyze the dynamics of the two-level replicator dynamics, a nonlocal hyperbolic PDE describing deterministic birth-death dynamics for both individuals and groups. Comparison principles and an invariant property of the tail of the population distribution are used to characterize the threshold level of between-group selection dividing a regime in which the population converges to a delta function at the equilibrium of the within-group dynamics from a regime in which between-group competition facilitates the existence of steady-state densities supporting greater levels of cooperation. In particular, we see that the threshold selection strength and average payoff at steady state depend on a tug-of-war between the individual-level incentive to be a defector in a many-cooperator group and the group-level incentive to have many cooperators over many defectors. We also find that lower-level selection casts a long shadow: if groups are best off with a mix of cooperators and defectors, then there will always be fewer cooperators than optimal at steady state, even in the limit of infinitely strong competition between groups. 									  											  									  									  									  								  								  								  	Cecilia Berardo								  	 								  		University of Helsinki								  									  									  										  			&lt;/br&gt; Evolution of density-dependent handling times in a predator-prey model.  								  										  									  									  										  										  											  			The competitive exclusion principle states that in a constant population two species competing for the same limited resource cannot coexist. This cannot be generalised to non-constant populations. In particular, it has been shown that two predator species competing for the same niche can coexist if the population exhibits non-equilibrium dynamics such as limit cycles. In addition to the ecological question, there emerges the problem whether the coexistence of different predator types competing for a single prey is evolutionarily robust and attainable. Geritz et al. [1] used the theory of adaptive dynamics to study the evolution of the handling time in a model with Holling type II functional response. They found that under certain conditions the handling time undergoes evolutionary branching and leads to the establishment of the evolutionarily robust coexistence of two predator types. Essential is the assumption of a trade-off between the handling time and the conversion factor connecting the predator's birth rate to its capture rate. It was found that the predator type with the short handling time dominates in the part of the population cycle where the prey is abundant, while the other type prevails when the prey is rare. Ecologically this makes sense: when the prey density is low, prospects of capturing new prey are diminished and it becomes worthwhile to cling to the prey one has already got despite of its gradually decreasing returns. When the prey is common, however, it is easy to replace the partially spent prey by a new one. In this presentation we derive a Holling type II functional response with handling time that depends on the prey density. The ecological setting raises some considerations on the importance of the mechanistic approach when we look at the functions which model the population dynamics as well as leads to new interesting evolutionary questions. Using the theory of adaptive dynamics, we investigate if at least some level of density dependence is favoured whether or not the population is cycling. A further question is if the density dependence can eliminate the possibility of coexistence of different predator types. This makes sense when a single predator with density dependent handling time dynamically shifts its niche between low and high prey densities depending on the phase of the population cycle in a way that a predator with a fixed handling time cannot. 									  											  									  									  								  								  								  									  	Maksim Mazuryn								  	 								  		Technical University of Denmark								  									  									  										  			&lt;/br&gt; Diel Vertical Migration as a Mean Field Game 								  										  									  									  										  										  											  			The phenomenon of diel vertical migration is one of the largest daily movements of marine species where animals remain in deep, dark water during daylight hours to avoid visual predation and migrate to upper levels at dusk to feed. The migration of each organism can be rationalized as a trade-off between growth and survival with strategies as spatial distributions of the populations. The dynamics driving vertical migration have broad implications for fluxes through the food-web predator-pray interactions [2, 4]; for vertical transport of carbon from upper to deeper layers (i. e. the so-called 'biological carbon pump') with implications for global climate. Here, we present progress of ongoing work on a framework for expressing diel vertical migration as a 'vertical game' in terms of partial differential equations. In the model setup we consider a population of animals distributed in the water column. It is assumed that each animal in this game moves optimally, seeking regions with high growth rate and small mortality, avoiding regions with high population density. The Nash equilibrium for this mean field game is characterized by a system of partial differential equations, which governs the population distribution and migration velocities of animals. The derived system of PDEs has similarities to equations that appear in the fluid dynamics, specifically the Euler equations for compressible inviscid fluids. For the established framework we derive a discretized system of equations to solve the PDEs governing this game theoretical model and present results of numerical simulations. The discretization process is based on the spectral collocation method which has exponential convergence rate for smooth enough functions compared to finite difference schemes. This allows us to reduce computational complexity of the discrete version of PDEs.  Recent results either doesn't take into account cost of movement [4] or doesn't resolve time continuously [2]. We formulate the equations in continuous time, incorporate costs on excessive movements in our model and illustrate the theory with numerical examples. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  			             		    		  											  		    		      				      	Closing Plenary			      		        3:30pm: 		        Carl Bergstrom, &lt;/br&gt;University of Washington, @CT_Bergstrom 		      		      																     				       					       					     		             		    		    		    				   				    Misinfodemic 2020: How quantitative misinformation misleads the public about COVID-19, and what mathematical biologists can do about it. 				     					    Most people think they have pretty good bullshit detectors, at least when it comes to advertising hyperbole, weasel words, and politicians’ promises. Quantitative claims are harder: Numbers have the sense of objectivity and precision, and people feel less confident in challenging them. Yet with COVID-19, quantitative epidemiology is on the front page of every newspaper on every day of the week. Shortly before the pandemic broke out, I finished a book on how to spot and refute quantitative misinformation. Every lesson in the book has proven useful during the current pandemic. In this talk, I will present a tour of how misleading numbers, statistics, mathematics, and data graphics have muddied the social and traditional media streams that we all rely upon during COVID. I will give examples of deliberate disinformation, and examples of unintended misinformation around the pandemic. And I will explore how as citizens and as educators we all can promote data reasoning and quantitative literacy. 				     			    			   		    		    		    		  									&lt;/br&gt;			&lt;/br&gt;									 				  Overview Sunday Monday Tuesday Wednesday Thursday			 																	      		      									Click to show abstracts						            "
    }, {
    "id": 26,
    "url": "http://localhost:4000/tuesday/",
    "title": "SMB2020 Tuesday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Tuesday, August 18		    		    US Eastern Time Zone (EDT) 	    	&lt;/br&gt;		    		    			    				        																&lt;/br&gt;								&lt;/br&gt;															  Overview Sunday Monday Tuesday Wednesday Thursday				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 			                Coffee with friends &amp; colleagues       8:00am:                  	                    Subgroup Keynote        8:30am:         Deirdre Hollingsworth, &lt;/br&gt;Oxford Big Data Institute, @DeirdreHoll         Mathematical Epidemiology Subgroup             												     		       			       			                        			  													  		    		      		        Sub-group minisymposia (9:30-11:00am)		        			  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	CDEV: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Leonie van Steijn								  	 								  		Leiden University								  									  									  										  			&lt;/br&gt; Regulation of persistent cell migration by the extracellular matrix 								  										  									  									  										  										  											  			Amoeboid cell motility is an important process in many eukaryotic cell types such as immune cells. Adequate immune cell motility is necessary to clear infections. The motility of cells is affected by interaction with the extracellular matrix (ECM) as well as by chemoattractants and other molecular signals. To study cell-ECM interactions, we consider two situations that we study using a combination of experiments and mathematical modeling. These two environmental cues are also present for in vivo immune cells as they move through tissue filled with other cells and ECM. First is topotaxis where cell motion is guided by the topography of the environment. As a model of amoeboid cell motility through pores in the ECM, we study the motility of Dictyostelium discoideum cells on a substrate covered with microscopic pillars. The pillars are spaced widely enough to let the cells through and there is a gradient from densely packed pillars to more widely spaced pillars. Here the cells perform a random walk with a slight drift to the more widely spaced area. Using a Cellular Potts model, we study how cell persistence mode affect topotaxis using the actin-derived persistence of the Act-model and an active Brownian particle-based persistence. We find that both modes result in topotactic drift comparable to the experimentally found drift, but the actin-based persistent cells show a more efficient drift. Next we study how ECMs of more dense structure affect the motility of B-lymphocytes. B-lymphocytes show different motitility modes on different matrices: a slow but persistent random walk on collagen IV with low cell-ECM contact area and a faster Brownian walk on fibronectin with high cell-ECM contact area. Expanding the Act-model with cell-ECM bonds that can form, grow and shrink, and rupture, we can obtain different cell motion with by using different rupture energies and new bond formation. Simulated cells mostly show a persistent random walk, with low persistence and low diffusivity for poorly attached cells, and high persistence and high diffusivity for dynamically attached cells. Cells with sustained attachement show pivoting behaviour which was persistent in short time-scales, but subdiffusive on long time scales. We conclude that cell-ECM interactions can affect cell motility in multiple ways. ECM pores can steer cells from denser to looser ECM areas through topotaxis, whereas attachment to the ECM can alter the motility type of cells. Combined, these cues could lead to a range of different possible motility types. How in vivo cells integrate these cues together with other cues such as chemokine signalling is subject for further studies. 									  											  									  									  									  								  								  								  	Emine Atici Endes								  	 								  		Heriot-Watt University								  									  									  										  			&lt;/br&gt; Modelling Scratch Wound Healing Assay using an Improved Non-local Equation 								  										  									  									  										  										  											  			Wound healing assays, in the other words scratch assays, are based on observing cells migrate into a wound or open space created an artificial scratch on a monolayer of cells. The assays are commonly used to quantify the rate of gap closure, which is a measure of the speed of the collective motion of cells and they are able to evaluate cell migration usefully in vitro wound healing. Obviously, the actual wound is more complex than the wound is done by making a scratch on a cell monolayer, however; the scratch wound assay is a technically simple, inexpensive, and fast method for analysis of cell migration and does allow modeling and testing of cell migration under well-defined conditions. We introduce a novel continuum model that extended the derived continuous model of a single population of cells. To derive our continuum model, we consider an integro-advection-diffusion-reaction equation for the adhesion of the single-cell motility in one dimension.  And in this specific study, we analyse the applicability of our model to scratch-wound healing assay based on some experimented cases. 									  											  									  									  									  								  								  								  	Bradford Peercy								  	 								  		University of Maryland, Baltimore County								  									  									  										  			&lt;/br&gt; A Minimal Model for STAT regulation in Initiation of Clustered Border Cell Migration 								  										  									  									  										  										  											  			Cell migration is pivotal in development as well as homeostasis, immune function, and pathology.  It is important to understand the molecular activity that allows some cells to assume the migratory cell fate.  The critical interaction we consider, in Drosophila melanogaster, is between the well-conserved Signal Transducer and Activator of Transcription (STAT) and downstream transcription factors Apontic (APT) and Slow Border Cells (SLBO).  We derive a detailed mechanistic mathematical model and then reduce it to the three main transcription factors.  The reduction maintains the steady state behavior including a bistable switch between stationary and migratory states.  However, the basins of attraction vary, and the manifolds separating the basins can be associated with delays in cell fate decisions.  Experiments with miRNA disruption of cell migration compare well with the equivalent model manipulation. 									  											  									  									  									  								  								  								  	Ulrich Dobramysl								  	 								  		University of Cambridge								  									  									  										  			&lt;/br&gt; Sensing and triangulation of chemical gradients 								  										  									  									  										  										  											  			In many biological processes, in particular embryonic and brain development, cells need to follow chemical gradients to arrive at a precise location. They need to determine the direction and position of sources releasing diffusing molecular guidance cues from information gathered by receptors located on the cell membrane. Using matched asymptotics, we developed a model that relates the chemical fluxes to receptors to the gradient source position. We learned that simple direction sensing using comparison of fluxes is strongly limited. In contrast, full recovery of the gradient source position from receptor fluxes is possible even over relatively large distances. We quantify the uncertainty associated with location triangulation and show how the accuracy depends on the number and distribution of receptors. 									  											  									  									  								  								  								  									  	Dimitris Goussis								  	 								  		Khalifa University, UAE								  									  									  										  			&lt;/br&gt; Endogenous and exogenous IgG competing for FcRn receptors: multi-scale analysis 								  										  									  									  										  										  											  			In many cases of IgG subclass deficiencies or FcRn malfunction, it is desired to elevate the levels of IgG, in order to strengthen the immune system. Conversely, in cases in which pathogenic or excess IgG antibodies are the aetiological agents, it is desirable to lower the IgG levels, in order to alleviate the symptoms; as in autoimmune diseases. One of the most efficient approaches to decrease the pathogenic IgG levels is to enhance its catabolism by administering recombinant IgG, which competes with the endogenous for the binding to the FcRn receptor. It was shown experimentally that the administration of exogenous IgG with high affinity, delivers better results in enhancing the degradation of the endogenous pathogenic IgG, than the classical intravenous immunoglobulin (IVIG) treatment, which is only effective in high doses. In this study, the competing interactions of the exogenous IgG and endogenous (pathogenic) IgG, when binding with the FcRn receptor are analyzed, on the basis of the model proposed in. A multi- scale analysis is carried out by employing the Computational Singular Perturbation (CSP) algorithmic methodology. With this algorithm, the constraints that develop progressively, form the start of the process to the fixed point, are identified, along with the reduced model that governs the evolution of the system within these constraints. CSP provides the tools for system-level understanding, by identifying the physical processes that (i) contribute to the emergence of the constraints (equilibria), (ii) drive the slow evolution of the system within these constraints and (iii) are responsible for the development of the fast and slow timescales in the dynamics of the model. The objective of this manuscript is to provide meaningful insights regarding the dynamical properties of the competitive binding of the endogenous and exogenous IgG with the FcRn receptor. Given that the modulation of the IgG-FcRn interaction allows for the control of the IgG half- life, the analysis provides a guideline to engineer effective recombinant IgG antibodies, in order to reduce the endogenous IgG levels. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	IMMU: Immunobiology and Infection Minisymposium (9:30-11:00am)													 						  			  				  							  							  										  									  	Rob de Boer								  	 								  		Universiteit Utrecht, Utrecht, The Netherlands								  									  									  										  			&lt;/br&gt; Why do some populations of cells accumulate deuterium faster than lose it? 								  										  									  									  										  										  											  			Deuterium labelling experiments are often used to infer the kinetic properties (i. e. , turnover rates, maintenance mechanisms) of various cell populations in vivo. For a homogeneous population that is at steady state, it is natural to expect that the population gains and loses labelled cells at the same rate. However, if the measured population is kinetically heterogeneous, it is natural to expect that the rate at which the labeling curve increases (i. e. , the up-slope) is slower than the rate at which it decreases (i. e. , the down-slope). Surprisingly, recent data from multiple deuterium-labelling experiments have the opposite property, and predict a gain of label that is faster than the rate of loss. Using various mathematical models, we search for mechanisms that can account for such unexpected labelling data. We show that the short-term labeling data can be explained when the deuterium remains available for a longer duration than it was administered. For the long-term labeling data we study models where (a) the population was considered to behave like a stem-cell population, or (b) the phases of the cell cycle were modeled with delay equations (i. e. , with the Smith-Martin model). Both provide scenarios where the gain of label can be faster than its loss. However, the effect is small and these two models fail to give a good description of the experimental data. When we finally drop the assumption that the population remains at steady state, we readily explain the experimental data with a simple source, division and death model. This mechanism however requires that these populations are largely expanding by a source from a precursor compartment, and not by cell division (i. e. , self-renewal). 									  											  									  									  									  								  								  								  	Katia Koelle								  	 								  		Emory University, Atlanta, GA, USA								  									  									  										  			&lt;/br&gt; Considering the consequences of cellular coinfection in within-host viral dynamics and modeling 								  										  									  									  										  										  											  			Within-host viral dynamic models often times categorize cells as infected or uninfected, similar to epidemiological microparasite models. However, in vitro and in vivo studies indicate that cellular coin- fection occurs frequently in influenza, HIV, and coronaviruses, among other viral pathogens. Here, I first discuss work from my group that develops simple models that allow for cellular coinfection in a scalable manner, with cellular multiplicity of infection affecting the phenotypes of infected cells, such as their death rate, viral production rate, and interferon induction rate. I then present work focusing on quantitatively characterizing the evolutionary consequences of cellular coinfection for newly arising mutations (both beneficial and deleterious). Our findings indicate that cellular coinfection decreases the ability of selection to act on individual mutations and results in genetic drift playing a larger role in modulating allele frequencies in a within-host viral population. 									  											  									  									  									  								  								  								  	Carmen Molina Paris								  	 								  		University of Leeds, Leeds, UK								  									  									  										  			&lt;/br&gt; A stochastic Model of Infection: Francisella T ularensis 								  										  									  									  										  										  											  			With a mouse infection model, agent-based computation and mathematical analysis, we study the pathogenesis of Francisella tularensis infection. A small initial number of bacteria enter host cells and proliferate inside them, eventually destroying the host cell and releasing numerous copies that infect other cells. Our analysis of disease progression is based on a stochastic model of a population of infectious agents inside one host cell, extending the birth-and-death process by the occurrence of catastrophes: cell rupture events that affect all bacteria in a cell simultaneously. Closed expressions are obtained for the survival function of an infected macrophage, the number of bacteria released as a function of time after infection, and total bacterial load. We compare our analysis with the results of agent-based computation and, via Approximate Bayesian Computation, with experimental measurements carried out after of murine aerosol infection with the virulent SCHU S4 strain of the bacterium. The posterior distribution is consistent with the estimate that the time between rounds of bacterial division is less than 6 hours in vivo. 									  											  									  									  									  								  								  								  	Catherine Weathered								  	 								  		Purdue University, West Lafayette								  									  									  										  			&lt;/br&gt; Mycobacterium Avium infection in the lungs: effects of bacterial phenotype and biofilm 								  										  									  									  										  										  											  			Mycobacterium avium complex (MAC), a type of nontuberculous mycobacteria, are environmental mi- crobes, capable of colonizing and infecting humans following inhalation of the bacteria. MAC-pulmonary disease is difficult to treat and prone to recurrence, and both incidence and prevalence are increasing. MAC form biofilms and diverse colonies in the environment. These biofilms can aid in epithelial cell invasion, cause premature apoptosis in macrophages, and inhibit antibiotic efficacy [4]. We hypothesize a balance of bacterial factors (phenotypic diversity and biofilm formation) and host immune factors (speed and magnitude of response) is key to establishing and prolonging infections in the lung. To test these hypotheses, we developed a 3D agent-based model (ABM) that incorporates known interactions between bacteria, biofilm and immune cells in virtual lung tissue. We implement our model in Repast Simphony. The simulation grid represents a length of lung airway with a layer of mucus. Bacterial agents are classified as either sessile or planktonic phenotypes that determine their behavior: biofilm formation, macrophage phagocytosis and replication rate. All bacterial agents and infected macrophages release a generic chemoattractant representing pathogen associated molecular patterns and chemokines respectively. These chemoattractants diffuse through the grid and are treated as continuous variables. Macrophages probabilistically follow the chemoattractant gradient, phagocytose bacteria, and accumulate apoptotic signals (representing hyperstimulation in the TNF-α pathway) through exposure to biofilm and internal bacteria. Model results show an early relationship between the initial number of macrophages or distance that chemoattractants diffuse, and the ratio of planktonic to sessile bacteria. Larger initial macrophage numbers result in a stronger and more sustained reduction in planktonic bacteria early after infection. However, as the infection progresses, the bacterial population is sustained by the sessile bacteria that are protected in biofilms or inside infected macrophages, allowing the planktonic population to recover. This effect is offset with further chemoattract diffusion, as the macrophages can clear the infection early or recruit more macrophages. Thus, the model predicts that both bacterial phenotypes and a suppressed immune responses affect the bacterial ability to survive, propagate, and eventually establish infection. Future directions of this work include exploring the continued role of phenotypes later in infection and treatment, and adding drug pharmacokinetics and cell-level pharmacodynamics to better understand the role of biofilm in treatment efficacy. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	MEPI: Modeling COVID-19 to inform control efforts, Part II (9:30-11:00am)													 						  			  				  							  							  										  									  	Annelies Wilder-Smith								  	 								  		Umea University								  									  									  										  			&lt;/br&gt; COVID-19 Outbreak on the Diamond Princess Cruise Ship: Estimating the Epidemic Potential and Effectiveness of Public Health Countermeasures 								  										  									  									  										  										  											  			Background: Cruise ships carry a large number of people in confined spaces with relative homogeneous mixing. On 3 February, 2020, an outbreak of COVID-19 on cruise ship Diamond Princess was reported with 10 initial cases, following an index case on board around 21-25th January. By 4th February, public health measures such as removal and isolation of ill passengers and quarantine of non-ill passengers were implemented. By 20th February, 619 of 3700 passengers and crew (17%) were tested positive. Methods: We estimated the basic reproduction number from the initial period of the outbreak using SEIR models. We calibrated the models with transient functions of countermeasures to incidence data. We additionally estimated a counterfactual scenario in absence of countermeasures, and established a model stratified by crew and guests to study the impact of differential contact rates among the groups. We also compared scenarios of an earlier versus later evacuation of the ship. Results: The basic reproduction rate was initially 4 times higher on-board compared to the R0 in the epicentre in Wuhan, but the countermeasures lowered it substantially. Based on the modeled initial R0 of 14. 8, we estimated that without any interventions within the time period of 21 January to 19 February, 2920 out of the 3700 (79%) would have been infected. Isolation and quarantine therefore prevented 2307 cases, and lowered the ${R}_0$ to 1. 78. We showed that an early evacuation of all passengers on 3 February would have been associated with 76 infected persons in their incubation time. Conclusions: The cruise ship conditions clearly amplified an already highly transmissible disease. The public health measures prevented more than 2000 additional cases compared to no interventions. However, evacuating all passengers and crew early on in the outbreak would have prevented many more passengers and crew from infection. 									  											  									  									  									  								  								  								  	Samuel Clifford								  	 								  		LSHTM								  									  									  										  			&lt;/br&gt; Can traveller interventions delay a local outbreak? 								  										  									  									  										  										  											  			Interventions aimed at travellers, such as syndromic screening, sensitisation to symptoms, and contact tracing on onset of symptoms may help delay the establishment of a SARS-CoV-2 outbreak in a previously unaffected country. Here we consider how the probability of detecting an infected traveller varies with the sensitivity of screening and duration of travel relative to the incubation period. We also show how a combination of these traveller interventions may be effective at delaying the establishment early on in a global outbreak but become less effective as the rate at which infected travellers arrive increases. 									  											  									  									  									  								  								  								  	Jonathan Dushoff								  	 								  		McMaster University								  									  									  										  			&lt;/br&gt; Time distributions and coronavirus control 								  										  									  									  										  										  											  			Early investigations of coronavirus epidemiology have highlighted a number of practical (and interesting) questions about time distributions, including generation and serial intervals; and latent, incubation, and infectious periods. I will discussing different ways of measuring (and defining) these distributions, and implications for disease prediction and control. 									  											  									  									  									  								  								  								  	Caroline Colijn								  	 								  		Simon Fraser University								  									  									  										  			&lt;/br&gt; Modelling and estimation for COVID19: classic estimates of key parameters and the role of genomic data 								  										  									  									  										  										  											  			The novel coronavirus that was identified in Wuhan, China in December 2019 spread widely following reports of initial cases who were likely exposed at the Huanan seafood market in the city of Wuhan. The subsequent spread, rapid control measures, and reported international cases caused global public health concern. The scientific community responded rapidly, sharing data for modelling and estimation, and sharing viral sequences. These strongly suggested that there was a single introduction event from an animal reservoir to humans in approximately November 2019. However, key epidemiological parameters remained uncertain even in late February, as reporting fractions and social distancing measures varied between places, and because the time at which an individual is infected is of course unknown. We present early estimates of the incubation period and serial interval for several reported clusters, using contact tracing data. We describe the differences in model predictions based on these estimates and early models based on SARS parameters. We then discuss how early viral sequences and classic epidemiological estimates of key parameters can be integrated to refine estimates and inform transmission models. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				 		  				  				  				 		  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	MFBM: Topological and network analyses for data (9:30-11:00am)													 						  			  				  							  							  										  									  	Maria-Veronica Ciocanel								  	 								  		Duke University,	United States,	ciocanel@math. duke. edu								  									  									  										  			&lt;/br&gt; Ring Channel Dynamics using Topological Data Analysis 								  										  									  									  										  										  											  			Contractile rings are structures made of actin filaments that are important in development, wound healing, and cell division. In many model organisms, ring channels allow nutrient exchange to developing egg cells and are regulated by forces exerted by myosin motor proteins. I will present an agent-based modeling and data analysis framework for the interactions between filaments and motor proteins inside cells. This approach may provide key insights for the mechanistic differences between two motors that are believed to maintain the rings at a constant diameter. In particular, we develop methods leveraging topological data analysis techniques to investigate time-series data of filament interactions. Our proposed methods clearly reveal the impact of kinetic parameters on significant topological hole formation, thus giving insight into ring channel formation and maintenance. I will also discuss methods available for distinguishing between noise and signal in these topological summaries of the filament organization through time. 									  											  									  									  									  								  								  								  	John Lagergren								  	 								  		NC State,	United States,	jhlagerg@ncsu. edu								  									  									  										  			&lt;/br&gt; Biologically-informed neural networks guide mechanistic modeling from sparse experimental data 								  										  									  									  										  										  											  			Biologically-informed neural networks (BINNs), an extension of physics-informed neural networks, are introduced and used to discover the underlying dynamics of biological systems from sparse experimental data. In the present work, BINNs are trained in a supervised learning framework to approximate in vitro cell biology assay experiments while respecting a generalized form of the governing reaction-diffusion partial differential equation (PDE). By allowing the diffusion and reaction terms to be multilayer perceptrons (MLPs), the nonlinear forms of these terms can be learned while simultaneously converging to the solution of the governing PDE. Further, the trained MLPs are used to guide the selection of biologically interpretable mechanistic forms of the PDE terms which provides new insights into the biological and physical mechanisms that govern the dynamics of the observed system. The method is evaluated on sparse real-world data from wound healing assays with varying initial cell densities. 									  											  									  									  									  								  								  								  	John Nardini								  	 								  		NC State,	United States,	jtnardin@ncsu. edu								  									  									  										  			&lt;/br&gt; Analyzing Collective Motion with Machine Learning and Topology 								  										  									  									  										  										  											  			We use topological data analysis and machine learning to study a seminal model of collective motion in biology. This model describes agents interacting nonlinearly via attractive-repulsive social forces and gives rise to collective behaviors such as flocking and milling. To classify the emergent collective motion in a large library of numerical simulations and to recover model parameters from the simulation data, we apply machine learning techniques to two different types of input. First, we input time series of order parameters traditionally used in studies of collective motion. Second, we input measures based on topology that summarize the time-varying persistent homology of simulation data over multiple scales. This topological approach does not require prior knowledge of the expected patterns. For supervised machine learning methods for classification of mechanistic model parameters, the topological approach outperforms the one that is based on traditional order parameters. 									  											  									  									  									  								  								  								  	Adelie Garin								  	 								  		EPFL, Switzerland, adelie. garin@epfl. ch								  									  									  										  			&lt;/br&gt; Topological Data Analysis for (biological) image analysis 								  										  									  									  										  										  											  			Topological data analysis (TDA) methods extract features representing the number of connected components, loops and cavities of shapes at different scales through an iterative process called a filtration. In this talk, we describe several TDA tools to improve image processing (classification, comparison and reconstruction). These methods provide a global view of images, which takes into account intrinsic geometric and topological properties of the images and complement standard methods very well. They can be used for several purposes in image analysis: classification (of leaves, cells, neuron shapes,…), comparison (to differentiate a healthy tissue from a tumour for instance) or to help optimisation algorithms to reconstruct deteriorated images (for example to reconstruct 3D neuron images from scanned sliced brains). 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	ONCO: Multiscale models of cancer heterogeneity, with applications in drug development and precision medicine (9:30-11:00am)													 						  			  				  							  							  										  									  	Matthias Reuss								  	 								  		Stuttgart Research Center Systems Biology, University of Stuttgart								  									  									  										  			&lt;/br&gt; Spatial-temporal multiscale modelling and simulation of vascular tumour growth - development of new multicellular simulators based on structural consistency between model and computer architecture 								  										  									  									  										  										  											  			Multiscale modelling and simulation in systems medicine is an emerging methodology and discipline to tackle the challenges posed by complex disease, including cancer. We will present applications of a 3D multiscale hybrid discrete-continuum model to simulate angiogenesis and vascular tumour growth. The model uses a cellular automaton approach and couples intracellular dynamics, active cell movement, cell- cell interaction, extracellular diffusion, and a dynamically evolving vascular network. The model accounts for the interplay between subcellular events and the macroscopic properties of the tumour. Simulation of larger tumours taking into account the structure of the tissue and blood supply consisting of millions of interacting cells, and the architectures of real tissues, e. g. liver lobules, or validation of theoretical findings with aid of imaging and simulations of therapies for larger patient communities, are both compu- tationally challenging. Therefore, we will introduce new hardware and software developments applicable to multiscale modeling based on structural consistency between the multiscale model and the architecture of the computer hardware. These efficiently solve the problems of interactions between subcellular and tissue levels. I will summarize multicellular simulators based on hybrid structures of parallelised graphic and central processors. I will present applications of these hybrid parallelised computer systems to integrate simulation of 3D-growth of larger vascularized tumours, dynamic models of intracellular metabolism of hepatocytes, and the the 3D architecture of liver lobules. The coupled modeling of blood perfusion between intravascular and interstitial spaces in the microvasculature permits the simulation of the perfusion CT to create model-based images which can be compared with clinical observations from radiology. 									  											  									  									  									  								  								  								  	Holger Perfahl								  	 								  		Stuttgart Research Center Systems Biology, University of Stuttgart								  									  									  										  			&lt;/br&gt; Hybrid Modelling of Transarterial Chemoembolisation Therapies (TACE) for Hepatocellular Carcinoma (HCC) 								  										  									  									  										  										  											  			We will present an agent-based multiscale model of vascular tumour growth and angiogenesis to describe transarterial chemoembolisation (TACE) therapies. The model describes tumour and normal cells nested in a vascular system that changes structure in response to tumour-related growth factors and also interacts with oxygen that influences cell viability. Within the extended model TACE is included as a two-step process. First, the purely mechanical influence of the embolisation therapy is modelled by a local occlusion of the tumour vasculature. We distinguish between partial and complete responders, where parts of the vascular system are occluded for the former and the whole tumour vasculature is destroyed for the latter. In the second part of the model, drug eluting beads release the chemotherapeutic drug doxorubicin at destroyed vascular locations. Simulation results are parameterised to qualitatively reproduce clinical observations. Our simulations reveal that directly after a TACE treatment an unstable tumour state can be observed, where regrowth and total tumour death are equally likely. This short time-window is favorable for another therapeutic intervention with a less radical therapy. This procedure produces a more favorable outcome. Simulation results with an oxygen therapy within the unstable time- window demonstrate a potentially positive manipulated outcome. Finally, we conclude that our TACE model motivates new therapeutic strategies and can help clinicians to understand intertwined relations and crosstalk in tumours. 									  											  									  									  									  								  								  								  	Samuel Handelman								  	 								  		Internal Medicine, University of Michigan								  									  									  										  			&lt;/br&gt; Incorporation of morphological features to inverse problems in high-content screening of anti-cancer therapies 								  										  									  									  										  										  											  			Cell-based assays are a mainstay of drug development, including especially for anti-cancer drugs and anti-cancer drug combinations. However, these assays generally use cell-death as an endpoint, which is more suited to simple cytotoxicity than to anti-cancer efficacy. This is especially a challenge in the context of combination therapies, where the correct linkage function in a statistical model of two co-administered cytotoxic compounds is not obvious. Therefore, we propose imaging-based and molecular markers of cancer cell phenotype, as an alternative efficacy endpoint in cancer drug screening. We will combine and compare results in combination therapy screens with different efficacy measures and a range of linkage functions corresponding to different assumptions on the nature of cancer drug synergy. These approaches have the added benefit of better addressing inter-cellular heterogeneity, which is evident even in clonal cell lines. Finally. we will review relevant biological background helpful to a quantitative audience in contextualizing the other talks in this session. 									  											  									  									  									  								  								  								  	Harsh Jain								  	 								  		Mathematics, Florida state University								  									  									  										  			&lt;/br&gt; A Standing Variation Model of Prostate Cancer Response to Live Cell Vaccination 								  										  									  									  										  										  											  			Making quantitative predictions with data-driven models, the core approach of applied mathematical biology, requires parameter estimation from imperfect measurements. Therefore, parameter identifiability and estimability become a major concern. In this talk, I will present a model of immunotherapy in the treatment of prostate cancer. I introduce our novel approach, standing variation modeling, which exploits practical unidentifiability in model parameters to capture individual heterogeneity. In particular, we use experimental data to infer distributions on parameters that are critical to tumor growth and to the resultant immune response of the body. Sampling model parameters from these distributions allows us to simulate heterogeneity, both, at the level of the tumor cells, and the individual being treated. Model simulations offer an explanation for the very limited success of this prostate cancer immunotherapy that has been observed in practice. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	OTHE: Eco-evolutionary dynamics across scales of organisation (9:30-11:00am)													 						  			  				  							  							  										  									  	Paula Vasconcelos								  	 								  		Uppsala								  									  									  										  			&lt;/br&gt; How does joint evolution of consumer traits affect resource specialization? 								  										  									  									  										  										  											  			The origin and maintenance of diversity in nature are central themes in evolutionary biology and ecology. Recently, the framework of adaptive dynamics has been applied to try and shed light onto these questions, with interesting results. The phenomenon of disruptive selection when extreme phenotypes have a fitness advantage over more intermediate phenotypes is particularly interesting because, under this regime, selection favors the evolution of adaptive phenotypic diversity. More specifically, it can drive speciation as well as the evolution and maintenance of polymorphism within a species. The conditions that result in this specific selective regime are well understood at the theoretical level, but with one fun- damental restriction: the majority of research is based on the assumption of a single evolving quantitative trait. However, biological organisms are complex and initial results from the few models that incorporate multidimensional trait evolution indicate that increasing the number of co-evolving traits facilitates the emergence of disruptive selection. To systematically extend the existing theory, we study the conditions for the emergence of disruptive selection based on models with several co-evolving traits. These models are (i) characterized by a set of evolving traits that determines the fitness of individuals in a population due to interactions with a complex environment (prey, predators, pathogens), and (ii) mechanistic in the sense that each trait has an interpretation at the level of the organism. This approach thus allows us to better understand when and under what circumstances multiple coevolving traits facilitate or hinder the emergence of biological diversity. In this work, we analyze the evolutionary dynamics of consumer traits in a consumer-resource model. Consumer growth depends on search efficiency, handling time and conversion efficiency for two resources. Feeding on these alternative resources is subject to trade-offs such that, for instance, increasing search efficiency for one resource can only be achieved by decreasing search efficiency for the other. We investigate the evolution of these traits in isolation and various combinations. Our results show how moving from one to two, and then three coevolving traits affects the conditions un- der which resource polymorphism arises through evolutionary branching. We also show that the critical trade-off curvatures that lead to the different evolutionary outcomes that is, whether evolution leads to one generalist, one specialist or two specialists depend decisively on the specific combination of coevolv- ing traits. Finally, we show that, with multidimensional trait spaces, the parameter range can be split into one region in which evolutionary branching is independent of the mutational variance-covariance matrix, and another in which branching depends on it. 									  											  									  									  									  								  								  								  	Lynn Govaert								  	 								  		Eawag								  									  									  										  			&lt;/br&gt; Towards an integrated theory of eco-evolutionary communities 								  										  									  									  										  										  											  			The increasing amount of studies showing evidence of rapid evolution occurring on similar timescales as ecological processes, have demonstrated the importance of including eco-evolutionary dynamics to further our understanding on population, community and ecosystem processes. Most empirical studies still focus on the ecological consequences of evolutionary change within a single species. However, in natural systems species co-occur together, comprising a community. Hence, all member species may display an evolutionary response. Thus focusing on evolution of a single species may lead to an over- or underestimation of evolution on ecological processes. To understand the role of eco-evolutionary dynamics within communities of multiple coexisting species, there is a need for a formal theory of eco-evolutionary communities integrating processes of evolutionary biology and community ecology operating at different temporal and spatial scales. Based on previous frameworks of evolutionary biology and community ecology, I here present an integrated framework for eco-evolutionary communities, bringing together theoretical, conceptual and technical approaches of these two fields. Integrating fundamental processes of evolutionary biology and community ecology improves our understanding of eco-evolutionary dynamics within multi-species communities and allows the design of new experimental approaches and testing for new hypotheses. 									  											  									  									  									  								  								  								  	Charles Mullon								  	 								  		Lausanne								  									  									  										  			&lt;/br&gt; Eco-evolutionary dynamics under non-random interactions 								  										  									  									  										  										  											  			Organisms continuously modify their living conditions, transforming their environment, microbiome, and sometimes culture. Where these modifications influence the fitness of conspecifics, a feedback emerges between the evolution of traits and the environment in which they are expressed. To investigate such feedback, it is typically assumed that individuals interact at random. In this case, one can study the invasion of a rare mutant trait in an environment set by a common resident ignoring mutant-mutant interactions. However, non-random interactions are common in nature. In this talk, I will report some of my results on the effect of non-random interactions on eco-evolutionary dynamics, focusing on two mechanisms that lead to such non-random interactions: spatial structure and biased behaviours between parents and their offspring. In both cases, selection depends on complex feedbacks between individuals of the same mutant lineage. By disentangling and quantifying these feedbacks, this research can help understand the nature of adaptation via non-genetic modifications, with implications for how organisms evolve to transform their environments, microbiome, or culture. 									  											  									  									  									  								  								  								  	Josep Sardanyés								  	 								  		CRM Barcelona								  									  									  										  			&lt;/br&gt; Dynamics of cooperation: from origins of life to ecosystems 								  										  									  									  										  										  											  			In this talk we will introduce the hypercycle model, originally conceived by Manfred Eigen and Peter Schuster to study the dynamics of prebiotic replicators. Hypercycles are dynamical systems formed by replicators with catalytic activity, thus they have been also employed to investigate cooperation in complex ecosystems at different levels. Following this mathematical model, we will show the dynamics and bifurcations tied to cooperation, from origins of life to models of facilitation in metapopulations and dynamics of semiarid ecosystems. We will emphasise on the evidences we have of such dynamics in real biological systems (both at the experimental or field levels), thus showing that dynamical systems theory offers us a unique framework to understand the basic mechanims behind the dynamics and the transitions in systems with strong nonlinearities. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	POPD: Mathematical models of evolutionary rescue (9:30-11:00am)													 						  			  				  							  							  										  									  	Stephan Peischl								  	 								  		University of Berne								  									  									  										  			&lt;/br&gt; The effect of gene flow on evolutionary rescue 								  										  									  									  										  										  											  			It seems certain that a substantial fraction of our planet’s current biodiversity will be lost to extinction as species’ habitats change at an accelerating rate. Some species, however, may be able to escape that fate by adapting, shifting their geographical ranges, or both. This leads to the questions of when, where and how might adaptation allow species to survive, leading to ‘evolutionary rescue’. Some basic answers to those questions come from theory. Experimental and theoretical studies have highlighted the impact of gene flow on the probability of evolutionary rescue. Mathematical modelling and simulations of evolutionary rescue in spatially or otherwise structured populations showed that intermediate migration rates can often maximise the probability of rescue in gradually or abruptly deteriorating habitats. In this talk, I present several mathematical approaches to studying evolutionary rescue in spatial or otherwise structured populations with gene flow between sub-populations, using discrete or continuous space models. I present simple conditions for when gene flow facilitates evolutionary rescue as compared to isolated populations, investigate the role of long-distance dispersal, as well as the role of phenotypic variation in dispersal traits. 									  											  									  									  									  								  								  								  	Robert Noble								  	 								  		ETH Zurich								  									  									  										  			&lt;/br&gt; The logic of containing tumours 								  										  									  									  										  										  											  			Challenging the paradigm of maximum tolerated dose, evolutionary theory suggests that the emer- gence of resistance to cancer therapy may be prevented or delayed by exploiting competitive ecological interactions between drug-sensitive and resistant tumour sub-clones. Recent studies have shown that a treatment strategy aiming for containment, not elimination, can control tumour burden more effectively than more aggressive approaches in vitro, in mouse models, and in the clinic, but theoretical understand- ing of these outcomes is underdeveloped. I will present a new, mathematically rigorous framework for understanding tumour containment that unifies and generalizes previous formulations. Results obtained within this framework provide timely guidance for empirical research including the design of clinical trials. 									  											  									  									  									  								  								  								  	Mario Santer								  	 								  		Max Planck Institute for Evolutionary Biology								  									  									  										  			&lt;/br&gt; Evolutionary Rescue and Drug Resistance on Multicopy Plasmids 								  										  									  									  										  										  											  			Bacteria often carry 'extra DNA' in form of plasmids in addition to their chromosome. Many plasmids have a copy number greater than one such that the genes encoded on these plasmids are present in multiple copies per cell. This has evolutionary consequences by increasing the mutational target size, by prompting the (transitory) co-occurrence of mutant and wild-type alleles within the same cell, and by allowing for gene dosage effects. We present a mathematical model for bacterial adaptation to harsh environmental change if adaptation is driven by beneficial alleles on multicopy plasmids. Successful adaptation depends on the availability of advantageous alleles and on their establishment probability. The establishment process involves the segregation of mutant and wild-type plasmids to the two daughter cells, allowing for the emergence of mutant homozygous cells over the course of several generations. To model this process, we use the theory of multi-type branching processes, where a type is defined by the genetic composition of the cell. Both factors – the availability of advantageous alleles and their establishment probability – depend on the plasmid copy number, and they often do so antagonistically. We find that in the interplay of various effects, a lower or higher copy number may maximize the probability of evolutionary rescue. The decisive factor is the dominance relationship between mutant and wild-type plasmids and potential gene dosage effects. Results from a simple model of antibiotic degradation indicate that the optimal plasmid copy number may depend on the specific environment encountered by the population. 									  											  									  									  									  								  								  								  	Jacek Miękisz								  	 								  		University of Warsaw								  									  									  										  			&lt;/br&gt; Evolution of populations with strategy- dependent time delays 								  										  									  									  										  										  											  			We address the issue of the stability of coexistence of two strategies with respect to time delays in evolving populations. It is well known that time delays may cause oscillations. Here we report a novel behavior. We show that a microscopic model of evolutionary games with a unique mixed evolutionarily stable strategy (a globally asymptotically stable interior stationary state in the standard replicator dynamics) and with strategy-dependent time delays leads to a new type of replicator dynamics. It describes the time evolution of fractions of the population playing given strategies and the size of the population. Unlike in all previous models, stationary states of such dynamics depend on time delays. Moreover, at certain time delays, an interior stationary state may disappear or there may appear another interior stationary state. This shows that effects of time delays are much more complex then it was previously thought. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  			             		    		  									                15 minute break (11:00-11:15am)                								  		    		      		        Sub-group minisymposia (11:15am)		        			  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	CDEV: Shapes, patterns, and forces in development biology (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Steffen Rulands								  	 								  		Max Planck Institute								  									  									  										  			&lt;/br&gt; Setting up the epigenome: a collective phenomenon 								  										  									  									  										  										  											  			Methods from single-cell multi-omics allow measuring several layers of regulation along the one- dimensional sequence of the DNA. The biological function of these processes relies, however, on emergent processes in the three-dimensional space of the nucleus, such as droplet formation through phase sepa- ration. How can measurements along the sequence of the DNA be translated into an understanding of emergent dynamics in nuclear space? Here, we combine single-cell NMT-sequencing experiments with a theoretical and computational approach to rigorously map measurements along the DNA sequence to a description of the emergent spatial dynamics in the nucleus. Drawing on scNMT-seq experiments in vitro and in vivo we demonstrate our approach in the context of early development. We show how epigenetic modifications of the DNA, DNA methylation, are established through the interplay between chemical and topological modifications of the DNA, leading to the formation of condensates of methylated DNA in the nucleus. Using this theoretical framework, we finally identify epigenetic processes that precede lineage decisions in the early embryo. Our work sheds new light on epigenetic mechanisms involved in cellular decision making. It also provides a general framework of how mechanistic insights into the spatio-temporal processes governing cell-fate decisions can be gained by the combination of methods from single-cell multi-genomics, computational biology and theoretical physics. 									  											  									  									  									  								  								  								  	Alessandra Bonfanti								  	 								  		University of Cambridge								  									  									  										  			&lt;/br&gt; Characterising the rheology of soft tissues using Fractional Viscoelastic models 								  										  									  									  										  										  											  			When subjected to external mechanical loading, many biological materials, such as ligaments, lung tissue, endothelial cells, or collagen fibrils, exhibit a viscoelastic power-law behaviour. Using standard viscoelastic models involving combinations of spring and dashpot elements to capture this behaviour oversimplifies the response; this limits our ability to adequately quantify the characteristics of these materials. Alternatively, empirical expressions designed to fit the power-law behaviour may provide effective means to describe experimental measurements, but the use of such ad-hoc models without a proper constitutive relationship limits the scope of the measurements and restricts their predictive capability. Fractional calculus provides a convenient framework to accurately capture power-law behaviours. Various empirical expressions introduced to fit experimental data can be derived or approximated with simple fractional models, often using fewer parameters. Furthermore, this approach seems to be well suited to extract material properties. As we demonstrated in the context of single cells and simple tissues, fitting a model on one set of experiments, can be used to predict the response of the same material to a broad range of external stimuli. Furthermore, using consistent models accross various experimental measurement methods enables us to compare material parameters that could not be easily compared otherwise. This allows us to shed new light on behaviour previously reported in the literature. Fractional calculus is a niche area of mathematics that has been available for a long time in the literature. It has to date attracted limited attention in the biological field, or any other area due to the mathematical complexity. To promote the use of such generalised fractional viscoelastic models, we provide an open source library RHEOS for numerical analysis of experimental data. The occurrence of such power-law behaviour also in non-living tissues, e. g. gels, casein, plants; implies that fractional viscoelastic models can have a great impact on. 									  											  									  									  									  								  								  								  	Mathias Sonja								  	 								  		Uppsala University								  									  									  										  			&lt;/br&gt; Impact of force function formulations on the numerical simulation of center-based models 								  										  									  									  										  										  											  			Center-based models are a framework for the computational study of multicellular systems with widespread use in cancer modeling and computational developmental biology. At the core of these models are the numerical scheme used to update cell positions and the force functions that encode the pairwise mechanical interactions of cells. For the latter there are multiple choices that could potentially affect both the biological behavior captured, and the robustness and efficiency of simulation. For example, available open-source software implementations of center-based models rely on different force functions for their default behavior and it is not straightforward for a modeler to know if these are interchangeable. Our study addresses this problem and contributes to the understanding of the potential and limitations of three popular force functions from a numerical perspective. We show empirically that choosing the force parameters such that the relaxation time for two cells after cell division is consistent between the different force functions results in good agreement of the population radius of a growing monolayer. Furthermore, we report that numerical stability is not sufficient to prevent unphysical cell trajectories following cell division, and consequently, that too large time steps can cause geometrical differences at the population level. We illustrate that the different force functions show varying sensitivity to this issue. 									  											  									  									  									  								  								  								  	Clinton Durney								  	 								  		UBC								  									  									  										  			&lt;/br&gt; Quantifying cellular contributions to salivary gland tubulogenesis 								  										  									  									  										  										  											  			Epithelial cells organize themselves into tubes for the necessary functions of gas and nutrient transport, and the production and secretion of hormones and enzymes. The tubes of the Drosophila salivary gland result from the organization and collective motion of a flat sheet of polarized epithelial cells through a budding process. Through orchestrated cell movements and cell rearrangements, the nascent tube begins to form. In this talk, we develop a novel 3D-vertex model that allows for the investigation and quantification of the role that cellular mechanics and cellular rearrangements play during this vital morphogenetic process. The novel 3D model is able to quantify cell mechanical behavior and analyze the effect of different forces during invagination. Using this biophysical model, we investigate how patterning of forces on the apical surface cause cell shape changes that lead to invagination.  Specifically, we investigate the roles of apicomedial induced cellular constriction, junctional actomyosin, a supracellular actomyosin cable and cellular intercalations have during gland morphogenesis. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  				  																																		 	IMMU: Immunobiology and Infection Minisymposium (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Cristian van Dorp								  	 								  		Los Alamos National Laboratory, Los Alamos, NM, USA								  									  									  										  			&lt;/br&gt; Stochastic viral dynamics modeling of time series from HIV-1 cure experiments in macaque and mouse models 								  										  									  									  										  										  											  			Conventional wisdom has it that the only way to control HIV-1 infection is lifelong antiretroviral drug therapy (ART). However recent observations indicate that functional cure, i. e. , control of HIV infection in absence of ART, is possible. Treatment strategies to achieve functional cure have therefore become a very active area of research. The efforts to develop a functional cure for HIV-1 must overcome the persistence of the latent reservoir of infected CD4+ T cells. These cells contain integrated HIV-1 DNA and are both long-lived and can re-active producing new viral particles that re-establish infection after ART interruption. All progress towards a HIV-1 cure must contend with this problem. Proposed strategies for curing HIV-1 include methods to reduce the size of the reservoir, or induce immune responses that can prevent viral rebound after ART interruption. Such novel treatments are typically tested in the macaque model or in humanized mice. We analyze data from two such studies: a macaque model to assess the effect of early ART initiation, which may limit latent reservoir size, and a CD4+ T-cell xenograft mouse model to assess the effect of immuno-therapies targeted at HIV-specific CD8+ T cells (HSTs). Stochasticity plays a major role in both these experiments. The formation of the latent reservoir and reactivation from this reservoir leading to viral rebound are both thought to be highly stochastic processes. In the mouse model, the virus escapes from the CD8+ T-cell immune response in a largely unpredictable manner, despite the fact that biological variation between repeated experiments is reduced to a minimum. We are therefore motivated to develop stochastic viral dynamics models to describe the data, rather than the standard deterministic viral dynamics model, and further, we estimate parameters using Sequential Monte Carlo methods for panel data. This allows us to integrate data from repeated experiments in our inference. In our investigation of early SIV infection with macaque data, we compare different models of reservoir formation. We show that a model in which reservoir establishment saturates at high viral loads, can reconcile early establishment with the observed distribution of rebound times. In our investigation of immuno-therapies with mouse data, we succeed in disentangling viral rebound due to escape from the HST response, from rebound due to typical expansion and contraction dynamics of the HSTs. 									  											  									  									  									  								  								  								  	Stanca Ciupe								  	 								  		Virginia Tech, Department of Mathematics, Blacksburg, VA, USA								  									  									  										  			&lt;/br&gt; Understanding the antiviral effects of RNAi-based therapy on chronic hepatitis B infection 								  										  									  									  										  										  											  			Reaching functional cure following chronic hepatitis B virus infections is hindered difficult by the presence of large numbers of HBsAg in the blood of infected patients. Therapies with the RNA interfer- ence drug ARC-520, which silence viral translation, together with daily administration of the nucleoside analogue drug entecavir have showed reduction in the overall levels of serum HBsAg in HBeAg-positive, treatment naive patients. Understanding the relative effects of ARC-520 alone, and in combination with entecavir, is particularly important in informing the development of new generation antiHBsAg drugs. A mathematical model describing the mechanistic interactions between HBV DNA, HBsAg, and HBeAg in the presence of ARC-520 and entecavir has been developed. We fitted the model to patient data and investigated the long term dynamics of the virus and viral protein titers under entecavir alone and under combination therapy. We run in silico boosting experiments and used them to determine the tradeoff between viral protein decay and drug induced toxicity. Such results can inform policy. 									  											  									  									  									  								  								  								  	Ruy Ribeiro								  	 								  		Faculdade de Medicina da Universidade de Lisboa, Lisbon, Portugal								  									  									  										  			&lt;/br&gt; How do CD8+ T cells control HIV infection? 								  										  									  									  										  										  											  			Human immunodeficiency virus (HIV) infection is still one of the most important causes of morbidity and mortality in the world, with a disproportionate human and economic burden especially in poorer countries. Despite many years of intense research, an aspect that still is not well understood is what immune mechanisms control the viral load during the prolonged asymptomatic stage of infection. Because CD8+ T cells have been implicated in this control by multiple lines of evidence, there has been a focus on understanding the potential mechanisms of action of this immune effector population. One type of experiment used to this end has been depleting these cells with monoclonal antibodies in the SIV-macaque model and then studying the effect of that depletion on the viral dynamics. These experiments generated controversial results, with dynamical models developed to help interpret these data leading to conflicting conclusions. We propose a new explanation for these results and provide both new experimental data and modeling evidence that helps to reconcile previous observations. In this hypothesis the main effect of CD8+ T cells occurs before viral integration. 									  											  									  									  									  								  								  								  	Eva Stadler								  	 								  		University of New South Wales, Sydney								  									  									  										  			&lt;/br&gt; Heterogeneity in the risk of latent malaria parasite reactivation explains the timing and pattern of infection recurrences in (Plasmodium vivax) malaria endemic settings 								  										  									  									  										  										  											  			The parasite Plasmodium vivax causes both blood-stage malaria infection and the formation of latent liver-stage parasites called hypnozoites. These recurrence of infection through hypnozoite activation is a major contributor to the total new infections in Plasmodium vivax endemic regions. After being treated for a single infection it is well known that some individuals will experience a second recurrence very rapidly while others will not experience a recurrence for some time. The mechanisms governing the ‘schedule’ of reactivation are not completely understood. A variety of conceptual models have been proposed, including a ‘biological clock’ mechanism, induction by external factors such as fever, or simply random reactivation of hypnozoites. In addition to these models, we propose an alternative explanation that there is heterogeneity in the risk of malaria relapse within the population. To explore the mechanisms governing P. vivax recurrence, we constructed differential equation models of each of the above conceptual models of hypnozoite reactivation. The models were compared, through fitting and simulation, to previously published time-to-infection data from a malaria endemic region following 1299 people for about one year. The data used in our study provided a powerful opportunity to study the mechanisms underlying P. vivax relapse because rather than including only a single infection event for each individual, multiple occurrences were recorded in many individuals over a one-year follow-up. However, the multiple measurements within each individual added complexity to fitting our custom time-to-infection model and required us to build these models within a mixed-effects type model framework. Our results show that the models with population heterogeneity in the reactivation rate provided the simplest and best explanation of the data and unlike the other conceptual models, heterogeneity could explain the observed patterns of P. vivax recurrences between individuals. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	MEPI: COVID-19 Contributed Talks II (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Nicholas Steyn								  	 								  		University of Auckland								  									  									  										  			&lt;/br&gt; The effect of border controls on the risk of COVID-19 re-incursion in New Zealand 								  										  									  									  										  										  											  			As of mid-July, New Zealand appears to have eliminated community transmission of COVID-19, allowing for almost no domestic restrictions on activity. The risk of re-incursion is mitigated by strict quarantine requirements at the border. These measures include a mandatory 14-day stay in a government managed facility, multiple RT-PCR tests, and regular symptom checkups. We use a simple individual based model to investigate the risk that international arrivals pose. Arriving individuals are randomly assigned an infection status and, if relevant, an exposure date. False negative testing rates and infectiousness vary over time; while the asymptomatic status, symptom onset date, and daily contacts are assigned according to estimated distributions. Results suggest that minimising mixing in the facilities should be the primary focus of risk reduction efforts. We also propose a measure that can be used to estimate the level of transmission occurring within the facilities: the ratio of cases detected in their second week of stay to cases detected in their first week. 									  											  									  									  									  								  								  								  	Lin Wang								  	 								  		University of Cambridge								  									  									  										  			&lt;/br&gt; Serial interval of SARS-CoV-2 was substantially shortened over time by non-pharmaceutical interventions 								  										  									  									  										  										  											  			Studies of novel coronavirus disease (COVID-19) have reported varying estimates of epidemiological parameters including serial interval distributions, i. e. the time between illness onset in successive cases in a transmission chain, and reproduction numbers. By compiling a line-list database of transmission pairs in mainland China, we show that mean serial intervals of COVID-19 have shortened substantially from 7. 8 days to 2. 6 days within a month (January 9 to February 13, 2020). This change is driven by enhanced non-pharmaceutical interventions, in particular case isolation. We also show that using real-time estimation of serial intervals allowing for variation over time, provides more accurate estimates of reproduction numbers than using conventionally fixed serial interval distributions. These findings would improve assessment of transmission dynamics, forecasting future incidence, and estimating the impact of control measures. 									  											  									  									  									  								  								  								  	Louise Dyson								  	 								  		University of Warwick								  									  									  										  			&lt;/br&gt; The impact of contact networks upon SARS-CoV-2 transmission in workplaces and universities 								  										  									  									  										  										  											  			Following the first cases of COVID-19 being reported in the UK in late January 2020, by early March it was evident that sustained community transmission was occurring. As part of social distancing measures enforced to tackle the epidemic, non-key workers were not allowed into the workplace and universities moved to online teaching and examination for the remainder of the 2019/2020 academic year. As steps are taken to relax social distancing measures, questions surround the ramifications on community disease spread of workers returning to the workplace and students returning to university. To study these aspects, we present a network model to capture the transmission of SARS-CoV-2 over four overlapping sets of networks: (i) fixed workplace contacts; (ii) social contacts; (iii) contacts at home; and (iv) dynamic workplace contacts (for workers who see people from multiple places, such as in the service sector). Additionally, we showcase the flexibility offered by the framework by describing its use in a university setting. We assess the impact of contact tracing adherence upon the spread of infection and the total number of individuals in isolation. We also consider the impact of backwards tracing, whereby resources are focused upon identifying the source of any reported infected cases, and larger scale isolation of portions of the population if case level alerts are triggered. Our results suggest that high adherence with contact tracing can result in a significant reduction in the number of infected individuals and the total number of people who would be required to isolate over the duration of the epidemic. We observe only a weak effect of backwards contact tracing - there is a slight reduction in epidemic size as the probability of successfully tracing the source of infection increases. Finally, we observe that, in order for reactive closures to be effective, such a policy needs to be enacted when only a small proportion of those that interact with that setting have recently begun to display symptomatic infection. We conclude that ensuring high adherence to contact tracing should be prioritised in order to reduce future infection levels. 									  											  									  									  									  								  								  								  	Robin Thompson								  	 								  		University of Oxford								  									  									  										  			&lt;/br&gt; Mathematical modelling in the earliest stages of the COVID-19 pandemic 								  										  									  									  										  										  											  			In the early stages of the COVID-19 pandemic, when cases had only been reported in China, it was important to assess the risk that cases exported elsewhere would lead on to local epidemics. In this talk, I will show how the epidemic risk was assessed in countries worldwide. I will also present a simple approach for extending this analysis for models in which additional epidemiological complexity is included (e. g. presymptomatic transmission, age-structure, time-varying infection rates). This shows how epidemic risk estimates can be generated, informed using outbreak data, and then adjusted in real-time as more information becomes available about any newly invading pathogen. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				 		  				  				  				 		  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  																																		 	MFBM: Topological and network analyses for data (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Samuel		Heroy								  	 								  		University of Oxford,	United Kingdom,	samuel. heroy@maths. ox. ac. uk									  									  									  										  			&lt;/br&gt; Rigidity percolation in random rod networks 								  										  									  									  										  										  											  			In certain classes of both biological (e. g. actin) and material-based (e. g. nanocomposites) networks, the underlying system undergoes a transition in the mechanical strength at a critical system density. For instance, in a composite material composed of rigid interacting monodisperse particles randomly dispersed in a soft polymer matrix, the system experiences a phase transition at a critical particle density, whereas this transition may depend for instance on the mean number of filaments per contact in an actin network. This experimental phenomenon, termed rheological percolation, has been shown to occur in many systems at a density that is beyond the contact percolation threshold, demonstrating that a more complex mechanism is responsible for the observed mechanical gains. In this study, we construct a network model in which sphereocylinders are randomly dispersed in a medium and contact at intersection points (supposing penetrability). Idealizing these sphereocylinders (rods) as attractive particles that stay fixed at but can rotate about their points of contact (hinge-like connections), we posit that the rheological transition occurs when the rods form a spanning component that is not only connected, but connected in such a way as to remove all nontrivial degrees of freedom in the component (rigidity percolation). We build on results from two dimensions (see the paper https://epubs. siam. org/doi/abs/10. 1137/17M1157271) to develop an approximate algorithm that identiﬁes such spanning components through hierarchically identifying and compressing provably rigid motifs—contact patterns by which rigid components interact to form larger rigid components. We apply this algorithm to networks we generate at various density/system size, using a ﬁnite size scaling approach to rigorously estimate a rigidity percolation transition point bound, which we show agrees fairly well with a simple mean ﬁeld estimation. We also estimate the transition point (and critical exponents) for networks with different rod aspect ratios, and find that the transition point scales with the square of the aspect ratio. In this study, we construct a network model in which sphereocylinders are randomly dispersed in a medium and contact at intersection points (supposing penetrability). Idealizing these sphereocylinders (rods) as attractive particles that stay fixed at but can rotate about their points of contact (hinge-like connections), we posit that the rheological transition occurs when the rods form a spanning component that is not only connected, but connected in such a way as to remove all nontrivial degrees of freedom in the component (rigidity percolation). We build on results from two dimensions (see the paper https://epubs. siam. org/doi/abs/10. 1137/17M1157271) to develop an approximate algorithm that identiﬁes such spanning components through hierarchically identifying and compressing provably rigid motifs—contact patterns by which rigid components interact to form larger rigid components. We apply this algorithm to networks we generate at various density/system size, using a ﬁnite size scaling approach to rigorously estimate a rigidity percolation transition point bound, which we show agrees fairly well with a simple mean ﬁeld estimation. We also estimate the transition point (and critical exponents) for networks with different rod aspect ratios, and find that the transition point scales with the square of the aspect ratio. 									  											  									  									  									  								  								  								  	Yu-Min		Chung								  	 								  		UNC Greensboro,	United States,	y_chung2@uncg. edu									  									  									  										  			&lt;/br&gt; On the morphology of mitochondria via a multi-parameter persistent homology approach 								  										  									  									  										  										  											  			Mutations in autophagy-gene Optineurin (OPTN) are associated with Primary Open Angle Glaucoma (POAG) and amyotrophic lateral sclerosis, but the pathophysiological mechanism is unclear.  The E50K OPTN mutation is associated with glaucoma.  Recent studies have shown that OPTN may play an important role in regulating mitochondrial networks and interacting with parkin as part of the mitophagy pathway.  We hypothesized that loss of normal OPTN function disrupts mitochondrial morphology.  To investigate and quantify the phenomena, we use multi-parameter persistent homology on confocal images of cells from transgenic mice with the E50K mutation and genetic knockout of optineurin.  In particular, we combine methods in mathematical morphology to form a multi-parameter filtration.  We will show that such filtration contains both topological and geometric information about the mitochondria, and will demonstrate ways to extract meaningful features from it.  Preliminary results support the hypothesis.  This is a joint work with Chuan-Shen Hu at National Taiwan Normal University, Emily Sun, and Dr. Henry C. Tseng at Duke Eye Center. 									  											  									  									  									  								  								  								  	Alexandria		Volkenning								  	 								  		Northwestern University,	United States,	alexandria. volkening@northwestern. edu									  									  									  										  			&lt;/br&gt; Topological data analysis of zebrafish skin patterns 								  										  									  									  										  										  											  			Wild-type zebrafish feature black and yellow stripes across their body and fins, but mutants display a range of altered patterns, including spots and labyrinth curves. All these patterns form due to the interactions of pigment cells, which sort out through movement, birth, and competition during development. Using an agent-based approach, we have coupled deterministic cell migration by ODEs with stochastic rules for updating population size to reproduce stripe pattern development and predict cell interactions that may be altered in mutant patterns. Within a single zebrafish mutant, however, there is a lot of variability, and this makes it challenging to first identify the features of a pattern that we are trying to reproduce and then judge model success. Moreover, agent-based models have many parameters, and empirical descriptions of zebrafish patterns are largely qualitative. To help address these challenges, we draw on topological data analysis to develop a set of methods for automatically quantifying pattern features in a fully interpretable, cell-based way. We apply our techniques to both simulated data and real fish images, and we show how to quantitatively distinguish between and characterize different patterns. 									  											  									  									  									  								  								  								  	Ashish		Raj								  	 								  		UCSF,	United States, ashish. raj@ucsf. edu								  									  									  										  			&lt;/br&gt; Inference on models of network spread and protein aggregation in Alzheimer’s and dementia 								  										  									  									  										  										  											  			Alzheimer’s disease, Parkinson’s and other related dementias involve widespread, stereotyped and progressive deposition of misfolded proteins. There is mounting evidence for “prion-like” trans-neuronal transmission, whereby proteins misfold, trigger misfolding of adjacent same-species proteins, and there- upon cascade along neuronal pathways, giving rise to networked spread along white matter projections. The question of how protein aggregation and subsequent spread lead to stereotyped progression in the brain remains unresolved. We present here mathematically precise parsimonious modeling of these patho- physiological processes, extrapolated to the whole brain. We model monomer seeding and production at specific seed regions, aggregation using Smoluchowski equations; and networked spread using our prior Network-Diffusion model, whereby anatomic connections govern the rate at which two distant but con- nected brain regions can transfer pathologic proteins. These models involve several unknown parameters, some or all of which might be individual-specific. Hence parameter inference is a challenging problem, without which downstream applications of such models in disease diagnosis, prognosis and therapy will not be possible. We present several inference strategies we have explored in our lab, including nonlinear cost function minimization, variational Bayesian inference, and purely machine learning methods like support vectors and neural nets. Unlike most previous theoretical studies of protein aggregation, using these techniques our theoretical models are able to be fitted to and validated by experimental in vivo imaging and fluid protein measurements from large datasets. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	ONCO: Data-based modeling in cancer research with focus on clinical applications (11:15am-12:45pm)													 						  			  				  							  							  										  									  	David Cheek								  	 								  		Program for Evolutionary Dynamics, Harvard University								  									  									  										  			&lt;/br&gt; DNA sequence evolution in the Yule process 								  										  									  									  										  										  											  			We study a fundamental model of DNA evolution in a growing population of cells: cell divisions follow the Yule process, and each cell contains a sequence of nucleotides which can mutate at cell division. Following typical parameter values in bacteria and cancer cell populations, we take the mutation rate to zero and the final number of cells to infinity. We prove that almost every site (entry of the sequence) is mutated in only a finite number of cells, and these numbers are independent across sites. However independence breaks down for the rare sites which are mutated in a positive fraction of the population. The model is free from the popular but disputed infinite sites assumption. Violations of the infinite sites assumption are widespread while their impact on mutation frequencies is negligible at the scale of population fractions. Some results are generalised to allow for cell death, selection, and site- specific mutation rates. To illustrate our results we estimate mutation rates in a lung adenocarcinoma. 									  											  									  									  									  								  								  								  	Christoph Engel								  	 								  		Institute for Medical Informatics, Statistics and Epidemiology (IMISE), University of Leipzig								  									  									  										  			&lt;/br&gt; Utility of specialized clinical registries for knowledge-generating care in oncology: Results from two large German consortia on hereditary cancer predisposition syndromes 								  										  									  									  										  										  											  			Colon and breast cancer are among the most common cancers. An estimated 5% of these cancers are due to a hereditary cancer disposition caused by germline mutations in DNA repair genes. Mutation carriers have a greatly increased risk of developing cancer and therefore require intensified early detection measures. In order to precisely quantify the underlying cancer risks, to identify risk factors and to evaluate the benefit of early detection measures, two multi-centre interdisciplinary clinical registry studies have been established in Germany, which collect and analyse quality-controlled care data in a standardised manner. I will present selected results from the “German Consortium for Familial Intestinal Cancer” and the “German Consortium for Hereditary Breast and Ovarian Cancer” which had a direct impact on future patient care. Example 1: Individuals with Lynch syndrome (LS) are at highly increased risk for colorectal cancer (CRC). Regular colonoscopic surveillance is recommended, but there is no international consensus on the appropriate interval. Comparing prospective data from three countries with different surveillance policies (annually, 1–2-yearly, 2–3-yearly), we found that a policy of strict annual colonoscopies was not associated with lower CRC incidence or stage. However, we could identify several factors suitable for risk stratification. This study led us to change our recommendations for colonoscopic surveillance of LS patients in Germany. Example 2: Individuals with high breast cancer risk are recommended to participate in an intensified multimodal breast imaging program. Using cohort data from 10 years of prospective surveillance we could confirm the importance of MRI in high-risk screening, compared with mammography and ultrasound. However, the efficacy of the program was limited with regard to high-risk patients without a predisposing germline mutation. Moreover, both from retrospective and prospective registry data we determined age-dependent breast cancer risks in different risk groups. As in example 1, these results also led to modifications of future clinical decision strategies. In conclusion, these examples demonstrate that specialized clinical- epidemiological registries provide an important means to generate evidence for the further development of risk-adapted early cancer de- tection strategies. The registries are also a valuable basis for planning and conducting controlled clinical trials. 									  											  									  									  									  								  								  								  	Saskia Haupt 								  	 								  		Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University								  									  									  										  			&lt;/br&gt; Modeling multiple pathways in hereditary colorectal cancer development 								  										  									  									  										  										  											  			Lynch syndrome is the most common inherited colorectal cancer syndrome and accounts for 5–10% of the overall colorectal cancer burden. Like many other tumors, Lynch syndrome cancers develop through multiple pathways incorporating different driver mutations. However, a comprehensive understanding of LS tumor evolution which allows for tailored clinical interventions for treatment and even prevention is still lacking. We suggest a system of coupled ordinary differential equations modeling the evolution of the different pathways in order to address some of the most relevant unanswered questions in LS management. It is based on existing data on Lynch syndrome cancer incidence as well as mutational and molecular data for the individual pathways. The ansatz strikes a balance between being expressible on the one hand and not being too complex on the other hand. This yields an explainable and predictive behavior and makes the model amenable to a thorough mathematical analysis. It can be extended in a straightforward manner to include more mutated genes or to take new and improved measurements of mutation probabilities into account. 									  											  									  									  									  								  								  								  	Matthias Kloor								  	 								  		Department of Applied Tumor Biology, Heidelberg University Hospital								  									  									  										  			&lt;/br&gt; From disease models to clinical applications — lessons from Lynch syndrome colorectal cancer 								  										  									  									  										  										  											  			Lynch syndrome is caused by heterozygous germ line mutations of the DNA mismatch repair (MMR) genes. During life, somatic mutation events (second hits) lead to loss of MMR function in multiple crypts within the colonic mucosa. From thousands of such MMR-deficient crypt foci, however, only a very small part develops into clinically manifest cancers. These cancers are mostly diploid, but characterized by the microsatellite instability (MSI) phenotype, i. e. the accumulation of numerous insertion/deletion muta- tions at repetitive microsatellite sequences. Mutations affecting microsatellites in genes coding for tumor suppressor genes promote MSI tumor development in Lynch syndrome. Using a bioinformatics-based model, we have predicted a set of coding microsatellite mutations with likely driver function in Lynch syndrome. These mutations also lead to shifts of the translational reading frame and to the generation of MMR deficiency-related frameshift peptides (FSPs). The well-defined pattern of MMR deficiency- induced mutations and neoantigens has wide-ranging implications on the clinical course of the disease: as exactly the same mutations recurrently occur in exactly the same tumor suppressor genes, Lynch syndrome cancers share a small and predictable set of highly immunogenic FSP neoantigens. Immune responses against these FSP neoantigens can already be detected in tumor-free Lynch syndrome mutation carriers, suggesting that there is a lifelong interaction between the immune system and emerging precan- cerous cell clones. This is also reflected by the fact that immune-mediated elimination of immunogenic tumor cells leaves traces in manifest MSI cancers, a process termed immunoediting. We will discuss how bioinformatics approaches and mathematical modeling can help unraveling fundamental processes of cancer evolution, and how this information can be used to design novel, innovative approaches for cancer therapy and even prevention. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	POPD: Subgroup Contributed Talks (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Joany Mariño								  	 								  		Memorial University								  									  									  										  			&lt;/br&gt; Symbiosis increases population size and mitigates environmental fluctuations in a physiologically-structured model parameterized for bivalves 								  										  									  									  										  										  											  			As a nutritional strategy, symbiosis increases the metabolic capabilities of the host. In thyasirid clams, it has been shown that trophic symbiosis can alter the energy allocation pattern of a host. However, the possible role of symbiosis as a life history strategy that can shape population dynamics remains unknown. Here, we show how nutritional symbiosis and the abundance of and dependence on symbionts can modulate the host's population dynamics and buffer resource limitation. We used Dynamic Energy Budget (DEB) theory to build a physiologically-structured population model that explicitly includes energy acquisition and allocation at different stages of an organisms' life cycle. We formulated the model deriving the demographic rates from a DEB model and assuming equal mortality rates in both populations. We parameterized the model for two cohabiting clam species: asymbiotic (specialists that feed on free-living bacteria) and symbiotic thyasirids (generalists that gain nutrients from both free-living and symbiotic bacteria). We demonstrate that, without seasonal fluctuations, symbiotic thyasirids have higher abundances than asymbiotic thyasirids since the symbiotic bacteria act as an energy reserve allowing for higher energy allocation to reproduction. In a seasonal environment, when temperatures are low and resource is limiting, symbiotic and asymbiotic thyasirids have similar population sizes; nonetheless, the symbiotic population is less prone to extinction. Our findings suggest different adaptations to resource fluctuation: asymbiotic thyasirids depend on a larger energy reserve, while symbiotic thyasirids rely on symbiont assimilation. Our results highlight the relevance of linking individual energetics and life-history traits to population dynamics and are the first step towards a general understanding of the role of symbioses in populations' resilience. 									  											  									  									  									  								  								  								  	Catherine Wangen								  	 								  		Utah State University								  									  									  										  			&lt;/br&gt; Modeling phenological consequences of warming climate for a southern population of mountain pine beetle 								  										  									  									  										  										  											  			The mountain pine beetle (MPB, Dendroctonus ponderosae Hopkins) attacks living Pinus trees, and reproduces in the phloem. Adults must attack a host simultaneously to overwhelm host defenses and successfully colonize. Temperatures directly but non-linearly affect MPB progress through life stages and the phenology of adult emergence. MPB are successful in a thermal niche where they are univoltine and synchronize emergence. Changing temperatures have broadened that niche geographically, leading to tree mortality of over 5. 2 Mha in the western US. Successful bivoltine MPB have not been observed in the field, although a phenology model parameterized for northern US MPB populations suggests bivoltinism is possible in the southern MPB range under future warming scenarios. Bivoltinism could have devastating impacts on pine forests. However, northern and southern MPB are genetically different in response to temperature, requiring geographic-specific model parameters. Using rate curves parameterized with developmental observations from MPB in Arizona we have constructed a predictive cohort model for a southern MPB population. Initiating the model with field attack data and using temperature data recorded under the bark of attacked hosts, we simulated warming scenarios by adding to the yearly mean to test thermal regimes that would result in a bivoltine MPB population. We successfully constructed a predictive cohort model for a southern MPB population. A key result is a new method for projecting observed variability in oviposition, through multiple larval instars, into emergence distributions. Comparison of the cohort model with field emergence data allows us to infer developmental rates for unwitnessed pre-ovipositional adults and also validate model predictions. Model responses to simulated temperatures highlight thermal regimes that promote bivoltinism for the southern MPB population. 									  											  									  									  									  								  								  								  	Maria Martignoni								  	 								  		University of British Columbia Okanagan								  									  									  										  			&lt;/br&gt; Mathematical insights into mechanisms leading to coexistence and competitive exclusion among mutualist guilds 								  										  									  									  										  										  											  			Microbial inoculants have been used as organic fertilizers worldwide. One of the most widely used commercial products are arbuscular mycorrhizal (AM) fungi, as these fungi can associate with a vast variety of crops. Despite the potential benefits for soil quality and crop yield associated with the use of AM fungi, experiments assessing the effective establishment of the fungi in the field have given inconsistent results, where some observations show field establishment and improved crop yield, while other studies show poor establishment of the inoculated species.  Additionally, it is not yet clear whether the introduction of commercial inoculants could lead to a biodiversity loss in the native fungal community, and ultimately have a negative impact on plant growth. Here we develop a series of ordinary and partial differential equation models to study the spatio-temporal dynamics of a guild of mutualists (the fungal species) sharing a resource provided by the same partner (the host plant), constituing a shared resource for all fungi. Our results allow to assess the risks and benefit of inoculation, by identifying under which conditions inoculation can effectively boost productivity, when it has no significant effect on plant growth and on the native fungal community, and when it represents an invasion risk.  More generally, our models provide important ecological insights into the mechanisms responsible for coexistence and competitive exclusion among mutualist guilds, and constitute a framework to predict the consequences of species manipulation in mutualist communities.  Indeed, the models are simple enough to apply to a broad range of mutualisms found in nature, such as pollination or seed dispersal. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  			             		    		  											  		    		      		        Subgroup Business Meeting		             		    		  											  		    		      		        Sub-group contributed talks (1:30-2:30pm)		        			  				  				  				  				  				  				  				  				  				  																																		 	CDEV: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Jonathan Harrison								  	 								  		University of Warwick								  									  									  										  			&lt;/br&gt; Hierarchical Bayesian modelling of chromosome segregation allows characterisation of a distinct dynamic signature of errors in cell division 								  										  									  									  										  										  											  			Cells divide via a self-organising process known as mitosis where a crucial step is the high fidelity separation of duplicated chromosomes to daughter cells. Errors in segregating chromosomes during cell division are a hallmark of cancer and are associated with developmental syndromes. How cell division achieves high fidelity remains an outstanding question, in particular how errors are detected and corrected. Through automated tracking of chromosomes at fine spatio-temporal resolution over long timescales, we can produce detailed quantification of the behaviour of human cells during mitosis. We propose a force-based stochastic differential equation model, dependent on hidden states governed by a Markov process, to describe the oscillations and segregation of chromosomes in mitosis. By fitting this dynamic model to experimental data in a Bayesian framework, we can infer the timing of the metaphase-anaphase transition (chromosome separation) for each duplicated chromosome pair. By extending this to a hierarchical Bayesian framework, we are able to capture rare reversal events during anaphase in the model. Model comparison provides evidence that the hierarchical model with reversals is preferred over a model without reversals. Application of this computational modelling pipeline to experimental data allows characterisation of a distinct signature of model parameters related to lagging chromosomes, and subsequent correction of these errors by the cell. 									  											  									  									  									  								  								  								  	Marcin Zagorski								  	 								  		Jagiellonian University								  									  									  										  			&lt;/br&gt; How is information decoded in developmental systems? 								  										  									  									  										  										  											  			The development of multicellular organisms is a dynamic process in which cells divide, rearrange, and interpret molecular signals to adopt specific cell fates. Despite the intrinsic stochasticity of cellular events, the cells identify their position within the tissue with striking precision of one cell diameter in fruit fly or three cell diameters in vertebrate spinal cord. How do cells acquire this positional information? How is this information encoded and how do cells decode it to achieve the observed level of cell fate reproducibility? These are fundamental questions in biology that are still poorly understood. In this talk, I will combine both information theory methods and mechanistic models to address these questions in the context of spinal cord development. I will consider the two opposing morphogen signals that are integrated to specify the arrayed pattern of neural progenitor domains that later on give raise to different type of neurons. Based on the maximum likelihood estimation rule I will define decoding map that provides predictions for shifts in the target gene domains in mutants. The predictions will be validated using experimental data obtained from naïve chick neural plate explants and from embryos with altered ventral morphogen signaling. I will present a simple model of a gene regulatory network that integrates the two morphogen signals and is sufficient to recapitulate the observed shifts in the target domains. I will investigate to what extent the level of noise in the input signals affects precision of the resulting gene expression pattern. Interestingly, the observed precision of gene expression pattern is close to the theoretical limit of precision of decoding of noisy signals. 									  											  									  									  									  								  								  								  	Tim Liebisch								  	 								  		Frankfurt Inst. of Advanced Study								  									  									  										  			&lt;/br&gt; Cell Fate Clusters in ICM Organoids Arise from Cell Fate Heredity &amp; Division – a Modelling Approach 								  										  									  									  										  										  											  			Background: During the mammalian preimplantation phase, cells undergo two subsequent cell fate decisions. During the first cell fate decision, cells become either part of an outer trophectoderm or part of the inner cell mass. Subsequently, the inner cell mass (ICM) segregates into the epiblast and the primitive endoderm, giving rise to the embryo and the placenta respectively. Recently, ICM organoids have been published as an in vitro model system towards preimplantational development. ICM organoids mimic the second cell fate decision taking place in the in vivo mouse embryos. In a previous study, the spatial pattern of the different cell lineage types was investigated. The study revealed that cells of the same fate tend to cluster stronger than expected for the currently hypothesised purely random cell fate distribution. Three major processes are hypothesised to contribute to the final cell fate arrangements at the mid and late blastocysts or 24 h old and 48 h old ICM organoids, respectively: 1) intra- and intercellular chemical signalling; 2) a cell sorting process; 3) cell proliferation.  Methods &amp; Results: In order to quantify the influence of cell proliferation on the emergence of the observed cell lineage type clustering behaviour, an agent-based model was developed. The model accounts for mechanical cell-cell interactions, cell growth and cell division and was applied to compare several current assumptions of how ICM neighbourhood structures are generated. The model supports the hypothesis that initial cell fate acquisition is a stochastically driven process, taking place in the early development of inner cell mass organoids. The model further shows that the observed neighbourhood structures can emerge due to cell fate heredity during cell division and allows the inference of a time point for the cell fate decision.  Discussion: Simulations based on the model show that cell divisions involving cell fate heredity seem sufficient to lead to the local clustering observed in 24 h old ICM organoids, and that the initial cell differentiation process takes place only during a small time window, during or prior to ICM organoid composition. Our results leave little room for extracellular signaling believed to be important in cell fate decision, therefore we are discussing an alternative role of chemical signaling in this process. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	IMMU: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Esteban Abelardo Hernandez Vargas								  	 								  		Universidad Autonoma de Mexico (UNAM)								  									  									  										  			&lt;/br&gt; In-host Mathematical Modelling of COVID-19 in Humans 								  										  									  									  										  										  											  			COVID-19 pandemic has underlined the impact of emergent pathogens as a major threat for human health. The development of quantitative approaches to advance comprehension of the current outbreak is urgently needed to tackle this severe disease.  In this talk, mathematical models will be introduced to represent SARS-CoV-2 dynamics in infected patients. Considering different starting times of infection, parameters sets that represent infectivity of SARS-CoV-2 are computed for the target cell limited model and compared with other viral infections that can also cause pandemics. The best model to fit the data was including immune cell response, which suggests a slow immune response peaking between 5 to 10 days post onset of symptoms. The model with eclipse phase, time in a latent phase before becoming productively infected cells, was not supported. Interestingly, both, the target cell limited model and the model including immune responses, predict that SARS-CoV-2 may replicate very slowly in the first days after infection, and it could be below detection levels during the first 4 days post infection.  These models can serve for future evaluation of control theoretical approaches to tailor new potential drugs against COVID-19. 									  											  									  									  									  								  								  								  	(CANCELLED) Christopher Rowlatt								  	 								  		University of St Andrews								  									  									  										  			&lt;/br&gt; (CANCELLED) Modelling the within-host spread of SARS-CoV-2 infection, and subsequent immune response, using a hybrid multi-scale individual-based model 								  										  									  									  										  										  											  			The coronavirus 2019 (COVID-19) pandemic, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has affected millions of people worldwide. A dysfunctional immune response, and the interaction with secreted cytokines (cytokine storm), has been observed to correlate with disease severity. However, the precise mechanisms which lead to disease severity remain unclear. In this talk, we employ a hybrid multi-scale individual-based model to study the spread of SARS-CoV-2 on an epithelial monolayer, its interaction with the host immune response and the immune cell cross-talk, as well as the interaction with secreted cytokines. 									  											  									  									  									  								  								  								  	Rajat Desikan								  	 								  		Indian Institute of Science Bangalore								  									  									  										  			&lt;/br&gt; Targeting TMPRSS2 and Cathepsin B/L together may be synergistic against SARS-CoV-2 infection 								  										  									  									  										  										  											  			The entry of SARS-CoV-2 into target cells requires the activation of its surface spike protein, S, by host proteases. The host serine protease TMPRSS2 and cysteine proteases Cathepsin B/L can activate S, making two independent entry pathways accessible to SARS-CoV-2. Blocking the proteases prevents SARS-CoV-2 entry in vitro. This blockade may be achieved in vivo through ‘repurposing’ drugs, a potential treatment option for COVID-19 that is now in clinical trials. Here, we found, surprisingly, that drugs targeting the two pathways, although independent, could display strong synergy in blocking virus entry. We predicted this synergy first using a mathematical model of SARS-CoV-2 entry and dynamics in vitro. The model considered the two pathways explicitly, let the entry efficiency through a pathway depend on the corresponding protease expression level, which varied across cells, and let inhibitors compromise the efficiency in a dose-dependent manner. The synergy predicted was novel and arose from effects of the drugs at both the single cell and the cell population levels. Validating our predictions, available in vitro data on SARS-CoV-2 and SARS-CoV entry displayed this synergy. Further, analysing the data using our model, we estimated the relative usage of the two pathways and found it to vary widely across cell lines, suggesting that targeting both pathways in vivo may be important and synergistic given the broad tissue tropism of SARS-CoV-2. Our findings provide insights into SARS-CoV-2 entry into target cells and may help improve the deployability of drug combinations targeting host proteases required for the entry. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	MEPI: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Laura Strube								  	 								  		Virginia Tech								  									  									  										  			&lt;/br&gt; The role of repeat infection in the dynamics of a simple model of waning and boosting immunity 								  										  									  									  										  										  											  			Some infectious diseases produce lifelong immunity while others only produce temporary immunity. In the case of short-lived immunity, the level of protection wanes over time and may be boosted upon re-exposure, via infection or vaccination. Previous work developed a simple model capturing waning and boosting immunity, known as the Susceptible-Infectious-Recovered-Waned-Susceptible (SIRWS) model, which exhibits rich dynamical behavior including supercritical and subcritical Hopf bifurcations among other structures. Here, we extend the bifurcation analyses of the SIRWS model to examine the influence of all parameters on these bifurcation structures. We show that the bistable region, involving both a stable fixed point and a stable limit cycle, exists only for a small region of biologically realistic parameter space. Furthermore, we contrast the SIRWS model with a modified version, where immune boosting depends on the occurrence of a secondary infection. Analysis of this extended model shows that oscillations and bistability, as found in the SIRWS model, depend on strong assumptions about infectivity and recovery rate of secondary infection. Understanding the dynamics of models of waning and boosting immunity is important for accurately assessing epidemiological data. 									  											  									  									  									  								  								  								  	Miranda Teboh-Ewungkem								  	 								  		Lehigh University								  									  									  										  			&lt;/br&gt; Mathematical assessment of the impact of human-antibodies on sporogony during the within-mosquito dynamics of Plasmodium falciparum parasites 								  										  									  									  										  										  											  			We develop and analyze a deterministic ordinary differential equation mathematical model for the within-mosquito dynamics of the Plasmodium falciparum malaria parasite. Our model takes into account the action and effect of blood resident human-antibodies, ingested by the mosquito during a blood meal from humans, in inhibiting gamete fertilization. The model also captures subsequent developmental processes that lead to the different forms of the parasite within the mosquito. Continuous functions that model the switching transition from oocyst to sporozoites as well as human antibody density variations within the mosquito gut are proposed and used. In sum, our model integrates the developmental stages of the parasite within the mosquito such as gametogenesis, fertilization and sporogenesis culminating in the formation of sporozoites. Quantitative and qualitative analyses including a sensitivity analysis for influential parameters are performed. We quantify the average sporozoite load produced at the end of the within-mosquito malaria parasite's developmental stages. Our analysis shows that an increase in the efficiency of the ingested human antibodies in inhibiting fertilization within the mosquito's gut results in lowering the density of oocysts and hence sporozoites that are eventually produced by each mosquito vector. So, it is possible to control and limit oocysts development and hence sporozoites development within the mosquito by boosting the efficiency of antibodies as a pathway to the development of transmission-blocking vaccines. 									  											  									  									  									  								  								  								  	Maia Martcheva								  	 								  		University of Florida								  									  									  										  			&lt;/br&gt; Coinfection Dynamics of Heroin Transmission and HIV Infection in a Single Population 								  										  									  									  										  										  											  			We propose a model of a joint spread of heroin use and HIV infection. The unique disease-free equilibrium always exists and it is stable if the basic reproduction numbers of heroin use and HIV infection are both less than 1. The semi-trivial equilibrium of HIV infection (heroin use) exists if the basic reproduction number of HIV infection (heroin use) is larger than 1 and it is locally stable if and only if the invasion number of heroin use (HIV infection) is less than 1. When both semi-trivial equilibria lose their stability, a coexistence equilibrium occurs, which may not be unique. We compare the model to US data on heroin use and HIV transmission. We conclude that the two diseases in the US are in a coexistence regime. Elasticities of the invasion numbers suggest two foci for control measures: targeting the drug abuse epidemic and reducing HIV risk in drug-users. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				 		  				  				  				 		  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	MFBM: Subgroup contributed talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Rui Borges								  	 								  		Vetmeduni Vienna, Austria, ruiborges23@gmail. com								  									  									  										  			&lt;/br&gt; Consistency and identifiability of the polymorphism-aware phylogenetic models 								  										  									  									  										  										  											  			Polymorphism-aware phylogenetic models (PoMo) constitute an alternative approach for species tree estimation from genome-wide data. PoMo builds on the standard substitution models of DNA evolution but expands the classic alphabet of the four nucleotide bases to include polymorphic states. By doing so, PoMo accounts for ancestral and current intra-population variation, while also accommodating population-level processes ruling the substitution process (e. g. genetic drift, mutations, allelic selection). PoMo has shown to be a valuable tool in several phylogenetic applications but a proof of statistical consistency (and identifiability, a necessary condition for consistency) is lacking. Here, we prove that PoMo is identifiable and, using this result, we further show that the maximum a posteriori (MAP) tree estimator of PoMo is a consistent estimator of the species tree. We complement our theoretical results with a simulated data set mimicking the diversity observed in natural populations exhibiting incomplete lineage sorting. We implemented PoMo in a Bayesian framework and show that the MAP tree easily recovers the true tree for typical numbers of sites that are sampled in genome-wide analyses. 									  											  									  									  									  								  								  								  	Zoe Lange								  	 								  		Frankfurt Institute for Advanced Studie, Germany,	zlange@fias. uni-frankfurt. de								  									  									  										  			&lt;/br&gt; Force Inference – Estimating the dynamics of mechanical forces in epithelial tissues from time-lapse images 								  										  									  									  										  										  											  			Embryonic development, wound repair and cancer growth are complex results of collective cell migration. Collective cell migration is based on the biochemical and mechanical interactions between cells with individual cells regulating their mechanical state and exerting forces on the surrounding tissue. To gain further insights into the multi-scale effects of force propagation from cells to tissues and the shaping properties on whole organisms, it is of great interest to study spatial and temporal dynamics of forces. Force inference is a computational approach to estimate tissue stress from images using a biomechanical model and a mathematical inverse method. It is a good supplement to experimental force measurement techniques like laser ablation. As a non-invasive observation technique, it requires no mechanical probing of the tissue and yields results for the whole system at once. The force inference method proposed by Ishihara and Sugimura (2012) applies a Bayesian framework to handle the indefiniteness of a system of linear force-balance equations. In this study, we use cell-vertex simulation generated data to validate our implementation of Bayesian force inference and apply it to investigate force dynamics in a classical wound healing assay. We discuss our workflow including image segmentation, graph construction and calibration based on force geometry relation, as well as robustness for segmentation errors and theoretical limits of force inference. We find that the method is challenged by ragged boundaries produced with a scratching technique. We show that pressure gradients flatten with closing of the wound. 									  											  									  									  									  								  								  								  	Anastasios Matzavinos								  	 								  		Brown University,	United States,	tasos@brown. edu								  									  									  										  			&lt;/br&gt; Bayesian uncertainty quantification for particle-based simulation of lipid bilayer membranes 								  										  									  									  										  										  											  			A number of problems of interest in applied mathematics and biology involve the quantification of uncertainty in computational and real-world models. A recent approach to Bayesian uncertainty quantification using transitional Markov chain Monte Carlo (TMCMC) is extremely parallelizable and has opened the door to a variety of applications which were previously too computationally intensive to be practical. In this talk, we first explore the machinery required to understand and implement Bayesian uncertainty quantification using TMCMC. We then describe dissipative particle dynamics, a computational particle simulation method which is suitable for modeling biological structures on the subcellular level, and develop an example simulation of a lipid membrane in fluid. Finally, we apply the algorithm to a basic model of uncertainty in our lipid simulation, effectively recovering a target set of parameters (along with distributions corresponding to the uncertainty) and demonstrating the practicality of Bayesian uncertainty quantification for complex particle simulations. This work was partially supported by the NSF through grant no. DMS-1552903. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	ONCO: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Pirmin Schlicke								  	 								  		Chair for Mathematical Modeling at the Center of Mathematics, Technical University of Munich, Germany								  									  									  										  			&lt;/br&gt; How Mathematical Modeling Could Contribute to the Quantification of Metastatic Tumor Burden Under Therapy 								  										  									  									  										  										  											  			Cancer is one of the leading death causes globally with about 8. 2 million deaths per year with rising numbers in recent years. About 90% of cancer deaths do not occur due to primary tumors but to metastases, of which most are not clinically identifiable due to their relatively small size at primary diagnosis and limited technical possibilities. However, as therapeutic decisions are formed depending on the existence of metastases and their properties - non-identified metastases might have huge influence in the treatment outcome. It is therefore of clinical interest to give an estimation of the metastatic burden to assist in planning optimal treatments accordingly for individual cancer patients. A mathematical model addressing this problem has been developed based on a transport equation introduced by (Iwata et. al. ) and extended by currently available systemic treatment options such as chemo- and immunotherapy. The model is defined in a continuous setting which allows it to also model the transition of a single primary tumor towards a metastatic disease, therefore indicating the metastatic cascade necessary to develop multiple metastatic tumors. Numerical implementation of the model framework allows for parameter estimation from clinical data, in our case we gathered parameter values from systemically treated lung cancer patients. We successfully quantified the total metastatic burden retrospectively for those patients over time given systemic treatment. In silico experiments allow for insights in differing therapeutic schedules, different medications and the further development of the metastatic burden. A sensitivity analysis on the model framework gave valuable insights in the behavior of model parameters and the clinical outcome. 									  											  									  									  									  								  								  								  	Maximilian Strobl								  	 								  		Department of Integrated Mathematical Oncology, H. Lee Moffitt Cancer Center Tampa, USA								  									  									  										  			&lt;/br&gt; Personalising adaptive cancer therapy in theory and in practice: the role of resistance costs and cellular competition 								  										  									  									  										  										  											  			Control and conquer - this is the philosophy behind adaptive therapy, which seeks to exploit intra- tumoral competition to avoid, or at least, delay the emergence of therapy resistance in cancer. Motivated by promising results from theoretical, experimental and, most recently, clinical studies, there is an in- creasing interest in extending this approach to other cancers. As such, it is urgent to understand the characteristics of a cancer which determine whether it will respond well to adaptive therapy, or not. One plausible such candidate is the “cost of resistance” in which acquisition of the resistance mechanism decreases a cell’s fitness in the absence of drug. To investigate the role of fitness costs, we initially study a simple 2-population ODE model in which we assume tumour cells are either drug-sensitive or resistant and compete in a Lotka-Volterra fashion. We identify the initial fraction of resistance, the proximity of the tumour to carrying capacity, resistance costs and turnover as important determinants of the benefit of adaptive therapy over standard-of-care continuous therapy. Moreover, we show that a resistance cost is neither a necessary nor a sufficient criterion for the success of adaptive therapy, but that the effect of a cost is dependent on the tumour’s proximity to carrying capacity and the rate of cellular turnover. Subsequently, we test whether our conclusions extend into space by considering a 2-D on-lattice cellular automaton model. While all the aforementioned factors remain important, we show that they interact in a non-linear fashion with the spatial architecture of the tumour. To conclude, we will show applications of our insights to the development of an adaptive therapy trial for the treatment of ovarian cancer with PARP inhibitors. This illustrates some of our theoretical predictions and raises new questions about when to adapt therapy and when not to. Overall, our work helps to clarify under which circumstances adaptive therapy may be beneficial and suggests that turnover may play an unexpectedly important role. 									  											  									  									  									  								  								  								  	Johannes Reiter								  	 								  		Stanford University School of Medicine, Stanford, USA								  									  									  										  			&lt;/br&gt; A mathematical model of ctDNA shedding predicts tumor detection size 								  										  									  									  										  										  											  			Early cancer detection aims to find tumors before they progress to an uncurable stage.  Prospective studies with tens of thousands of healthy participants are ongoing to determine whether asymptomatic cancers can be accurately detected by analyzing circulating tumor DNA (ctDNA) from blood samples. We developed a stochastic mathematical model of tumor evolution and ctDNA shedding to investigate the potential and the limitations of ctDNA-based cancer early detection tests.  We inferred ctDNA shedding rates of early stage lung cancers and calculated that a 15 mL blood sample contains on average only 1. 5 genome equivalents of ctDNA for lung tumors with 1 billion cells (size of  1 cm3).  We considered two clinically different scenarios: cancer screening and cancer relapse detection.  For monthly relapse testing with a sequencing panel covering 20tumor-specific mutations, we found a median detection size of 0. 24 cm3 corresponding to a lead time of 160 days compared to imaging-based relapse detection.  For annual screening, we found a median detection size of 2. 8-4. 8 cm3 depending on the sequencing panel size and on the mutation frequency.  The expected detection sizes correspond to lead times of 390-520 days compared to current median lung tumor sizes at diagnosis.  This quantitative framework provides a mechanistic interpretation of ctDNA-based cancer detection approaches and helps to optimize cancer early detection strategies. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	OTHE: Other Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Anne Talkington								  	 								  		Chapel Hill								  									  									  										  			&lt;/br&gt; A PBPK model for clearance of PEGylated nanomedicines 								  										  									  									  										  										  											  			Physiologically based pharmacokinetic (PBPK) models are a means of conducting virtual experiments on a large scale as an alternative to extensive trials that would be prohibitively time-consuming, unethical, or otherwise costly.  PBPK can be used to compute and test optimal dosing strategies, among other features, of proposed treatments using known or learned kinetics of the system mimicking complex human physiology.  In turn, PBPK modeling can enable more efficient design and optimization of in vivo experiments, and consequently accelerate pre-clinical screening and development.  I will discuss the application of PBPK modeling to an important problem in the medical community – the accelerated clearance of PEGylated drugs in the presence of anti-PEG antibodies (APA).  While this phenomenon renders an entire class of drugs (i. e. , PEGylated drugs) ineffective in many patients, the medical community is largely unaware of how drastically this can alter prognosis or how to mitigate this effect.  I will describe a multi-compartment PBPK model to accurately capture and ultimately predict clearance behavior, with the goal of validating results against drug biodistribution data obtained via PET/CT technology.  Specifically, I will focus on the initial transient dynamics as nanomedicines are cleared from the circulation in patients with high APA titers.  I will then discuss further applications for this model in the context of targeted therapeutics. 									  											  									  									  									  								  								  								  	Tatiana Marquez-Lago								  	 								  		UAB								  									  									  										  			&lt;/br&gt; Mathematical methods for microbiome research 								  										  									  									  										  										  											  			The complex community of the human microbiota and its specific role in health maintenance and disease has become an intense topic of study –and debate- over the last years. We do know that microorganisms colonizing human bodies exceed the total number of human cells, and that the number of microbial genes inside our bodies is roughly 100 times higher than the number of genes contained in the human genome, impacting human biology in various ways. For instance, the human immune system is in great part composed of and trained by resident microorganisms, and different microbiome compositions associate with the onset and progression of a large variety of human diseases, including diseases typically considered as non-communicable. Functional, causal links remain largely unexplored in many cases, however, in great part due to sampling limitations, data volume and integration complexity. Due to this gap and the importance of studying microbiome interactions, we have developed methods and tools toward multi-scale analysis and predictive (mechanistic) modeling, as well as integration of multi ‘omics’ data. On the one hand, feature selection and machine learning allows identification of patterns and relationships in large data collections, such as those in human microbiota studies. On the other, mathematical modeling and simulations provide a comprehensive framework to identify connections and key (onset) mechanisms in disease models. Both approaches are essential for analysis and forward engineering personalized therapeutics. In this talk, I will discuss available tools and mathematical methods in this area of research, and what is still needed toward integrative models of host-microbiota dynamics. 									  											  									  									  									  								  								  								  	Camile Kunz								  	 								  		Goethe U. 								  									  									  										  			&lt;/br&gt; Chemotaxis impact on pattern formationChemotaxis impact on pattern formation 								  										  									  									  										  										  											  			During embryo development there is a rapid growth in cell numbers that forms complex structures. Skin pattern formation is an early process during the embryogenesis and happens before the cells fully differentiate. In the present project we consider skin patterning in mouse embryos, where cell aggregates form based on a hierarchical process, involving interactions between the epidermal cell populations. The reaction-diffusion pre-pattern is driven by fibroblast growth factor (FGF20), bone morphogenic protein (BMP) and WNT. Considering mathematical models, there are two main processes involved in the pattern formation: Turing reaction-diffusion systems and chemotaxis. The Turing system models the concentration of two interacting chemicals, and the patterns arises from an instability driven by a difference between their diffusion coefficients. Some previous studies show that this behavior is essential for self-organization in the mouse hair follicle and chicken feather pre-pattern formation. Another key mechanism is chemotaxis, where the cells move in the direction of a chemical attractant, where patterns can also be observed. Experimental data indicates a hierarchical system, where cell chemotaxis is guided by a Turing system. We aim at developing mathematical models to describe the underlying biological processes leading to skin patterning, especially the interaction of chemotaxis with reaction-diffusion (Turing) systems. A mathematical model using partial differential equations is solved numerically, and some results are presented and compared to the experimental data. We study the parameter-dependence of the model and different model structures, and their impact on the pattern forming process. According to the experimental data the Turing system and the chemotaxis seems to be intrinsically related on the mouse skin patterning. Using a numerical approach for the PDE system, we develop a framework to study quantitatively how chemotaxis and Turing systems are related and their impact on the patterning process. During embryo development there is a rapid growth in cell numbers that forms complex structures. Skin pattern formation is an early process during the embryogenesis and happens before the cells fully differentiate. In the present project we consider skin patterning in mouse embryos, where cell aggregates form based on a hierarchical process, involving interactions between the epidermal cell populations. The reaction-diffusion pre-pattern is driven by fibroblast growth factor (FGF20), bone morphogenic protein (BMP) and WNT. Considering mathematical models, there are two main processes involved in the pattern formation: Turing reaction-diffusion systems and chemotaxis. The Turing system models the concentration of two interacting chemicals, and the patterns arises from an instability driven by a difference between their diffusion coefficients. Some previous studies show that this behavior is essential for self-organization in the mouse hair follicle and chicken feather pre-pattern formation. Another key mechanism is chemotaxis, where the cells move in the direction of a chemical attractant, where patterns can also be observed. Experimental data indicates a hierarchical system, where cell chemotaxis is guided by a Turing system. We aim at developing mathematical models to describe the underlying biological processes leading to skin patterning, especially the interaction of chemotaxis with reaction-diffusion (Turing) systems. A mathematical model using partial differential equations is solved numerically, and some results are presented and compared to the experimental data. We study the parameter-dependence of the model and different model structures, and their impact on the pattern forming process. According to the experimental data the Turing system and the chemotaxis seems to be intrinsically related on the mouse skin patterning. Using a numerical approach for the PDE system, we develop a framework to study quantitatively how chemotaxis and Turing systems are related and their impact on the patterning process. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	POPD: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Peter Jagers								  	 								  		Chalmers &amp; University of Gothenburg								  									  									  										  			&lt;/br&gt; Galton and Watson were (almost) right: virtually all populations are eventually extinct 								  										  									  									  										  										  											  			145 years have passed since the publication of Galton’s and Watson’s famous paper where they claimed that “all surnames (and by analogy all populations) tend to extinction”. Strangely, their theorem was largely accepted for more than half a century, until Haldane and Steffensen established the true dichotomy between subcritical populations always dying out and supercritical, which either die out or else grow exponentially. But this is under stable conditions. For population-size-dependent reproduction, we show the following very general extinction theorem: Consider a process giving the size of a population in discrete time. Asume that reproductive events occur one-by-one so that at each step the population either decreases by one (an individual dies) or increases by a random number (the number of children born at the event). Note that theses changes are neither assumed independent nor identically distributed. On the contrary, there is a carrying capacity K, such that the process constitutes a supermartingale when larger than K and a submartingale otherwise. Further, zero is assumed to be the only absorbing state. Then the process dies out almost surely. 									  											  									  									  									  								  								  								  	Max Souza								  	 								  		Universidade Federal Fluminense								  									  									  										  			&lt;/br&gt; Fitness potentials and qualitative properties of the Wright-Fisher dynamics 								  										  									  									  										  										  											  			We present a mechanistic formalism for the study of evolutionary dynamics models based on the diffusion approximation described by the 1-D Kimura Equation (2 type and no mutation). In this formalism, the central component is the fitness potential, from which we obtain an expression for the amount of work necessary for a given type to reach fixation. In particular, within this interpretation, we develop a graphical analysis — similar to the one used in classical mechanics — providing the basic tool for a simple heuristic that describes both the short and long term dynamics. Using this toolkit, we propose a definition of an evolutionary stable state in finite population that includes the case of mixed populations. We also discuss extensions to more than two types and weak mutation. This is joint work with Fabio A. C. C. Chalub. 									  											  									  									  									  								  								  								  	Matthew Nitschke								  	 								  		University of Adelaide								  									  									  										  			&lt;/br&gt; The effect of bottleneck size on evolution in nested darwinian populations 								  										  									  									  										  										  											  			Recent theories about the transition from unicellular life have introduced the idea of ecological scaffolding as a potential explanation for how early groups of cells would have gained the properties necessary to participate in evolution by natural selection.  This is the idea that particular ecologies and environments can scaffold Darwinian properties onto groups of cells.  The scaffolding allows cells to directly participate in the process of evolution by natural selection as if they were members of multicellular collectives, with groups participating in a birth-death process.  The ingredients for this process to operate are only patchily distributed resources and a regularly occurring dispersal process that also creates a bottleneck.  In this talk, I will discuss the effect of bottleneck size on this process and how this alters the evolutionary dynamics at both levels of the system. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  			             		    		  											  		    		      		        Poster Session (2:30-3:30pm)		        	Click to view posters for each subgroup:  	 All	 CDEV	 EDUC	 IMMU	 MEPI	 MFBM	 NEUR	 POPD	 ONCO	 OTHE			             		    		  								  		    		      		        Sub-group Keynote		        3:30pm: 		        Andrea Bild, &lt;/br&gt;City of Hope, @AndreaBildLab 		        Mathematical Oncology Subgroup 		      		      																     				       					       					     		             		    		    		    				   				    Competitive and cooperative interactions between resistant and sensitive breast cancer cells during therapy 				     					    Our research focuses on estrogen receptor positive breast (ER+) cancer, the most commonly diagnosed and metastatic subtype. In a tumor, neighboring subclonal cancer cell populations interact both through competition for resources and through cooperation, by the production of growth promoting signals. Promoting competition can maintain drug sensitivity when resistant cells get outcompeted. Cooperation can promote cell survival under selective pressure, as well as potentially protect cellular populations that may otherwise be susceptible to treatment. Our research program integrates three key components to quantify cancer cell interactions: serial patient tumor samples molecularly profiled at the single cell level, a parallel in vitro model system where mechanisms can be tested through experimental manipulation and measured over time, and integrative dynamic mathematical models of the growth and interaction of subclonal cancer populations. By embedding the time course data and differential equation models into a Bayesian inference framework, we can estimate the strength of positive and negative interactions between subclonal populations during treatment and identify key mechanisms driving cell communication. Our analyses of patient derived samples further indicates that heterogeneous multi-clonal ER+ breast cell populations evolve to acquire alternative signaling states that drive cell cycle progression independent of estrogen availability, and that cells can cooperate to survive during therapy through the increased production and local sharing of growth promoting factors. 					    				     			    			   		    		    		  									                Happy hour with friends and colleagues (4:30)                						&lt;/br&gt;			&lt;/br&gt;									 				  Overview Sunday Monday Tuesday Wednesday Thursday			 											      		      									Click to show abstracts						            "
    }, {
    "id": 27,
    "url": "http://localhost:4000/wednesday/",
    "title": "SMB2020 Wednesday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Wednesday, August 19		    		    US Eastern Time Zone (EDT) 	    	&lt;/br&gt;		    		    			    				    								    																&lt;/br&gt;								&lt;/br&gt;															  Overview Sunday Monday Tuesday Wednesday Thursday				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 			                Coffee with friends &amp; colleagues       8:00am:                		                    Subgroup Keynote        8:30am:         Alicia Prieto Langrica, &lt;/br&gt;Youngstown State University, @AliciaPL25         Education Subgroup             												     		       			       			                             	   	    Lessons from the Pandemic: Why and how mathematical biologist educators are morally obligated to respond 	     The COVID-19 pandemic has made painfully clear the scientific illiteracy of the general population. Many are confused, rely on less than reliable sources for information, and reject scientific authorities equating the evolution of knowledge to lack of expertise. Given the current situation: what is our role and responsibility as educators to help our students be more informed, critical thinkers? How must we empower them through education to identify reliable sources and to better understand the scientific process? In this talk, we will reflect on ways in which we can respond to the current knowledge deficit, and why it is our duty to do so. 	                  	  		  													  		    		      		        Sub-group minisymposia (9:30-11:00am)		        			  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				 		  																																		 	CDEV: Subgroup Contributed Talks (9:30-11:00am)													 						  			  				  							  							  										  									  	Galane J. Luo								  	 								  		University of Birmingham								  									  									  										  			&lt;/br&gt; A fluid mechanical model of the plant cell wall reveals underlying mechanism for helical organ morphology 								  										  									  									  										  										  											  			Plant morphology emerges from cellular growth. The turgor-driven diffuse growth of a cell can be highly anisotropic: significant in the longitudinal direction and negligible in the radial direction. Such anisotropy is ensured by cellulose microfibrils (CMF) reinforcing the cell wall in the hoop direction. To maintain the cell's integrity during growth, new wall material including CMF must be continually deposited. In this talk, I will present a mathematical model which describes the cell as a cylindrical pressure vessel and the cell wall as a fibre-reinforced viscous sheet, explicitly including the mechano-sensitive angle of CMF deposition. The model incorporates interactions between turgor, external forces, CMF reorientation during wall extension, and matrix stiffening. I will explain the general formulation of the model from a fluid mechanics perspective, and summarise the technical steps required in order to obtain evolution equations for growth variables such as cell length and twist. I will discuss how the handedness of helical cell growth depends on external torque and intrinsic wall properties, and interpret numerical simulation results in light of recent experimental findings. Overall, the model provides a meaningful step towards a unified mechanical framework for understanding left- and right-handed growth as seen in many plants. Such a framework can help us harness the potential of plants in our effort to address society's sustainable development needs. 									  											  									  									  									  								  								  								  	Euan Smithers								  	 								  		University of Birmingham								  									  									  										  			&lt;/br&gt; How do plant leaf pavement cells form puzzle piece like shapes? Using a multi-model approach to simulate chemical and visco-elastic mechanical processes and experimental methods to discover their secrets.  								  										  									  									  										  										  											  			Pavement cells in the plant leaf epidermis form interesting and intriguing interlocking puzzle like shapes with undulations of lobes and indents. However, no one has been able to fully explain how these shapes form. There are two possible pathways of pavement cell development, one involving a combination of plant Rho like GTPases signalling proteins and cytoskeleton components, specifically microtubules which could provide a feedback loop and the second being possible mechanical effects from the tissue. As a result, we have developed three models, one to model microtubule behaviour, the second to model the protein signalling dynamics and the third to model the mechanics of the cells, using a stochastic network, reaction diffusion equations solved via the finite element method and a visco-elastic vertex element model. I shall also outline some of the experimental procedures we have carried out to test how pavement cells develop. We can demonstrate that the signalling pathways provide a feedback loop to sustain pavement cell shape, but don’t initiate the shape, while the mechanical effects from the tissue can initiate pavement cell lobes. 									  											  									  									  									  								  								  								  	Ulyana Zubairova								  	 								  		Russian Academy of Science								  									  									  										  			&lt;/br&gt; Leaf epidermal pattern development in the cereals: lessons from LSM-image analysis and computer simulations 								  										  									  									  										  										  											  			The leaf epidermis of a monocotyledonous plant gives a unique model system for studying morphogenesis due to diverse cell types and constant growth direction. For such leaves, a unidirectional growth occurring for a long time enables us to observe a series of successive morphogenetic stages at one snapshot. In this work, we propose the concept of using a growing wheat leaf to study dynamical changes in morphogenesis, including stress-induced changes. Linear leaf of wheat, during its formation for a long time, maintains a phase of steady growth. Therefore, it is possible to observe a series of successive events of morphogenesis fixed in the cellular architecture of a mature leaf. High-resolution 3D LSM-images allow extracting quantitative characteristics describing the cellular structure of leaf epidermis. However, to obtain a large number of statistical data methods of high throughput, computer-based image segmentation should be used. We developed a workflow for the detection of structural properties of leaf epidermis from 3D images obtained from confocal LSM-images (Zubairova, U. S. , Verman, P. Y. , Oshchepkova, P. A. , Elsukova, A. S. , &amp; Doroshkov, A. V. (2019). LSM-W 2: laser scanning microscopy worker for wheat leaf surface morphology. BMC systems biology, 13(1), 22. ). The workflow includes the protocol of sample preparation, image processing ImageJ-plugin, and data extraction algorithms. The data on the cellular architecture further acts as a basis for the elaboration and verification of spatial models accounting for structural features of leaves. For the leaf epidermis of cereals, a brickwork-like pattern combined with unidirectional growth allows to reduce the dimension and use a quasi-one-dimensional representation of the cellular ensemble in the model. This idea was realized in the model (Zubairova, U. , Nikolaev, S. , Penenko, A. , Podkolodnyy, N. , Golushko, S. , Afonnikov, D. , &amp; Kolchanov, N. (2016). Mechanical behavior of cells within a cell-based model of wheat leaf growth. Frontiers in plant science, 7, 1878. ) growth of a linear leaf blade. The model allows for fitting of the visible cell length using the experimental cell length distribution along the longitudinal axis of leaf epidermis. In this work, we assume a unidirectional growing cell ensemble starting from a meristem-like layer of generative cells and then generating parallel cell rows from every cell of the initial layer. We considered the growth zone of the leaf includes division and elongation zones; also, the division zone consists of a region of asymmetric divisions forming specialized cells (trichomes and stomata). The model was verified on qualitative and quantitative data on stress-induced disturbances of morphogenesis in the epidermis of a wheat leaf. The study was carried out with a grant from the Russian Science Foundation (project No. 19-74-10037). 									  											  									  									  									  								  								  								  	Tamsin Spelman								  	 								  		University of Cambridge								  									  									  										  			&lt;/br&gt; Nucleus shape in plant root hair cells 								  										  									  									  										  										  											  			A plant root hair is a single long thin cell, which in Arabidopsis is ≈ 10 μm in diameter and reaches ≈ 1 mm in length, growing rapidly at speeds of ≈ 1 μm/min. The nucleus tracks the growing tip remaining ≈ 75 μm back from the growing front [1]. While the nucleus motion has been studied, less work has considered the nucleus shape during growth and how this impacts cell growth. Using a numerical model, we analyse the forces imposed on the nucleus by the cytoskeleton (the internal fibre network of the cell), for a range of nucleus shape, size and position enclosed within a cuboidal domain. We compare our results with experimental data from root hairs grown within microchannels. We segment the experimental images to reconstruct the 3D nuclei shapes, from which we can also estimate the spacial distribution of forces being imposed. 									  											  									  									  								  								  								  									  	David Holloway								  	 								  		BCIT								  									  									  										  			&lt;/br&gt; Polar auxin transport dynamics of primary and secondary vein patterning in dicot leaves 								  										  									  									  										  										  											  			The growth regulator auxin plays a central role in development across plants. Auxin spatial patterning is critical in the phyllotactic arrangement of leaves along a stem, the shapes of the leaves themselves, and venation within leaves. These patterns depend on polar auxin transport (PAT) at the cellular level, particularly the preferential allocation of PIN efflux proteins to certain areas of the plasma membrane. Two general mechanisms have been studied: an up-the-gradient (UTG) allocation dependent on neighbouring-cell auxin concentrations, and a with-the-flux (WTF) allocation dependent on the flow of auxin across walls. Auxin appears to flow both towards auxin maxima (associated with UTG) and away from auxin maxima (associated with WTF), depending on the developmental phenomena. Both types of flow are implicated in vein patterning in leaves. We have developed a UTG+WTF model to quantify these combined dynamics. The model simulates intracellular and membrane kinetics and intercellular transport, and is solved for a 2D leaf of several hundred cells. Building upon earlier models for primary vein (mid-vein) formation, we developed a model for the formation of the secondary vein pattern. These arise from the margin of the leaf, in a distal to proximal sequence, and connect with the mid-vein to form the main vascular network of the leaf. These networks can be characteristic of species, and associated with species-specific leaf morphologies. The model responds to decreasing PAT, as in experiments increasing the PAT inhibitor NPA, with: a switch from several distinct vein initiation sites to many less-distinct sites; a delay in vein canalization; inhibited connection of new veins to old; and finally loss of patterning in the margin, loss of vein extension, and confinement of auxin to the margin. We have removed assumptions of long range attraction factors from earlier work. Simulations of vein patterning and leaf growth indicate that growth itself may help bridge the scale from the cell-cell resolution of the PIN-auxin dynamics to vein patterns on the whole-leaf scale. 									  											  									  									  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				 		  																																		 	EDUC: Education Minisymposium II (9:30-11:00am)													 						  			  				  							  							  										  									  	Dmitry Kondrashov								  	 								  		University of Chicago								  									  									  										  			&lt;/br&gt; Quantitative modeling remotely 								  										  									  									  										  										  											  			I teach the course Introduction to Quantitative Modeling for Biology, which is integrated into the biological sciences curriculum at University of Chicago and serves around two hundred students every year. I will discuss the adaptation of this course to online learning, present student assessments results from spring 2020, and share materials that can be incorporated into other courses. 									  											  									  									  									  								  								  								  	Glenn Ledder								  	 								  		University of Nebraska-Lincoln								  									  									  										  			&lt;/br&gt; Teaching Mathematical Epidemiology at Different Mathematical Levels Using a Multiple Representation Theory of Mathematical Modeling 								  										  									  									  										  										  											  			The COVID-19 pandemic has made mathematical epidemiology a topic of critical importance, providing mathematics educators with an unparalled opportunity.  This opportunity is accompanied by a challenge: how do mathematics educators, some of whom have little personal experience with mathematical modeling, teach mathematical epidemiology to their students in courses ranging from precalculus to differential equations, and do so in a way that builds understanding of epidemic disease dynamics as well as mathematical methods? We address this issue by presenting a framework based on a multiple representation theory of mathematical modeling and using that framework to offer examples of building blocks that include a physical simulation activity, model development, parameterization, various methods for analysis and visualization of results, and guidelines for how to get students to use writing to facilitate their understanding. 									  											  									  									  									  								  								  								  	Heiko Enderling								  	 								  		Moffitt Cancer Center								  									  									  										  			&lt;/br&gt; High school internship program in integrated mathematical oncology 								  										  									  									  										  										  											  			Modern cancer research, and the wealth of data across multiple spatial and temporal scales, has created the need for researchers that are well-versed in the life sciences (cancer biology, developmental biology, immunology), medical sciences (oncology) and natural sciences (mathematics, physics, engineering, computer sciences). College undergraduate education traditionally occurs in disciplinary silos, which creates a steep learning curve at the graduate and postdoctoral levels that increasingly bridge multiple disciplines. Numerous colleges have begun to embrace interdisciplinary curricula, but students who double-major in mathematics (or other quantitative sciences) and biology (or medicine) remain scarce. We identified the need to educate junior and senior high school students about integrating mathematical and biological skills, through the lens of mathematical oncology, to better prepare students for future careers at the interdisciplinary interface. The High school Internship Program in Integrated Mathematical Oncology (HIP IMO) at Moffitt Cancer Center has so far trained 59 students between 2015 and 2019. We report here on the program structure, training deliverables, curriculum, and outcomes. We hope to promote interdisciplinary educational activities early in a student's career. 									  											  									  									  									  								  								  								  	Brian Winkel								  	 								  		SIMIODE								  									  									  										  			&lt;/br&gt; SIMIODE - Systemic Initiative for Modeling Investigations and Opportunities with Differential Equations: Biological SIMIODE - Systemic Initiative for Modeling Investigations and Opportunities with Differential Equations: Biological Efforts 								  										  									  									  										  										  											  			SIMIODE is an NSF funded Community of Practice based at www. simiode. org which supports faculty and students in efforts to motivate and teach differential equations in context through modeling. While differential equations is a widely applicable branch of mathematics we share a number of efforts in biological applications which are featured in SIMIODE: population growth, death, and immigration; ant tunnel building; LSD and problem solving; malaria and Ebola; acorns, rodents, and snakes; crop harvesting; intraocular gas bubbles; tumor growth; drug administration; inner ear drug delivery; epidemics; and dialysis. We discuss examples of modeling in the life sciences using data and the full modeling cycle, while introducing mathematical content and supporting students in learning the underlying mathematics. Further, we share news of SCUDEM - SIMIODE Challenge Using Differential Equations Modeling, now in its fifth year is an annual international team challenge, in which students engage in model building using differential equations and share their results for faculty judging and commentary and peer review. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	IMMU: Immunobiology and Infection Minisymposium (9:30-11:00am)													 						  			  				  							  							  										  									  	Lubna Pinky								  	 								  		University of Tennessee Health Science Center								  									  									  										  			&lt;/br&gt; Quantifying the Effects of Dose, Strain and Tissue Tropism on Parainfluenza Virus Infection Kinetics 								  										  									  									  										  										  											  			Human parainfluenza viruses (HPIVs) are a leading cause of acute respiratory infection hospitalization in children yet little is known about the growth and clearance kinetics. In mice, longitudinal measurements are possible by using reporter sendai viruses that express luciferase, where the insertion location yields wild-type (rSeV-luc(M-F)) or attenuated (rSeV-luc(P-M)) phenotypes. Bioluminescence from individual animals suggests that there is a rapid increase in expression within the first 1-2 days followed by a peak, biphasic clearance, and resolution. However, these kinetics vary with dose, strain, individuals, and the upper and lower respiratory tracts. To quantify the differences, we first translated the bioluminescence measurements taken from nasopharynx, trachea, and lungs into viral loads. We then fit a mathematical model to the estimated viral load data for each scenario using nonlinear mixed effects modeling. The results confirmed a higher rate of virus production with the rSeV-luc(M-F) virus compared to its attenuated counterpart, and suggested that the infected cell clearance was expedited when infected with rSeV-luc(M-F) in lungs with high dose. Our analysis suggested that the number of infected cells scales with dose, and that distinct infected cell clearance rates are associated with each dose, strain, individual and respiratory tract compartment. This analysis provides important insight into parainfluenza infection kinetics and the dynamical differences based on dose, viral attenuation, individual heterogeneity, and tissue tropism. 									  											  									  									  									  								  								  								  	Maria Rodriguez Martinez								  	 								  		IBM Research Europe, Zurich (Switzerland)								  									  									  										  			&lt;/br&gt; Multiscale clonal model of Germinal Center B cell differentiation 								  										  									  									  										  										  											  			Germinal Centers (GCs) are B cell follicles in the secondary lymphoid organs where B cells proliferate, mature their B cell receptors (BCRs) following exposure to antigen and interaction with other GCs cells, and eventually differentiate as plasma cells or memory B cells. We have recently developed a stochastic hybrid model of the GC reaction that combines: i) an intra-cellular gene circuit that captures the regulatory interplay of a few key master regulators of the differentiation process; and ii) an extra-cellular component that accounts for the stochastic events that take place in the GC, such as antigen acquisition and competition for T cell help (https://www. mdpi. com/2073-4409/9/6/1448). Mimicking the evolutionary processes undergone by B cells, in our model new B cells constantly emerge exhibiting mutated BCRs that present different affinity to the existing pool of antigens, which in turn probabilistically bias their evolution to certain fates. We faithfully recapitulate the process of BCR maturation by explicitly modelling the hypermutation process that takes place in the BCR genes. We compare the model predictions to experimental data of matched single-cell transcriptional high-resolution maps and BCR repertoire sequencing of GC B cells.  Our explicit modeling of B cell maturation enables us to characterise the evolutionary processes and competition at the heart of the GC dynamics, and explains the emergence of clonal dominance as a result of initially small stochastic advantages in the affinity to antigen. Interestingly, a subset of the GC undergoes massive expansion of higher-affinity B cell variants (clonal bursts), leading to a loss of clonal diversity at a significantly faster rate than in GCs that do not exhibit clonal dominance. Our work contributes towards an in silico vaccine design, and has implications for the better understanding of the mechanisms underlying autoimmune disease and GC-derived lymphomas. 									  											  									  									  									  								  								  								  	Gustavo Hernandez Mejia								  	 								  		Frankfurt Institue for Advanced Studies (FIAS); Goethe University Frankfurt, Germany								  									  									  										  			&lt;/br&gt; Antibodies cross-reaction in influenza A infections: a modeling approach 								  										  									  									  										  										  											  			Disclosing key phenomena of how antibodies (Abs) induced by one influenza strain are effective against another, the so-called cross-reaction, is central for the design of universal flu vaccines. Here, using data of mice infected with influenza, we develop a stochastic mathematical modeling scheme to explore the impact of consecutive influenza A infection in the cross-reactome. The model successfully recapitulates the antibody cross-reactive data from mice infected with different H3N2 influenza strains. Of note, without framework modifications, the model can also represent the Abs response in mice to diverse H1N1 strains. Furthermore, we found that while the antigen differences, time of infection, and the B-cells population shape between infections directly influence the Abs outcome, the naive B-cells repertoire has minor effects on Abs behavior. Importantly, we found that affinity changes in immunity between infections satisfy necessary conditions for a successful Abs cross-reaction. We envisage this work will add to the forces among public makers, virologists, biologists, and theoreticians, bringing clarity of mind when experimental and clinical evidence is fragmented. 									  											  									  									  									  								  								  								  	Paul Macklin								  	 								  		Indiana University								  									  									  										  			&lt;/br&gt; Community-driven multiscale modeling of SARS-CoV-2 								  										  									  									  										  										  											  			The 2019 novel coronavirus (SARS-CoV-2) is a pathogen of critical significance to international public health. Knowledge of the interplay between molecular-, cellular-, and multicellular-scale processes that drive disease dynamics is limited. Multiscale simulation models can shed light on these dynamics, identify actionable 'choke points' for intervention, screen potential therapies, and identify potential biomarkers that differentiate patient outcomes. In this talk, we present progress by a multi-institution, multi-disciplinary coalition of over 40 mathematical biologists, immunologists, virologists, pharmacologists, and others to build a comprehensive multiscale model of SARS-CoV-2 infection dynamics and immune response in lung tissue. We will demonstrate and explore the current spatio-temporal agent-based model prototype, which includes intracellular virus and chemokine transport, virus-ACE2 receptor binding, receptor trafficking, viral replication dynamics (and subsequent viral release), infected cell phenotypic responses, cell-cell communication, immune cell recruitment, chemotactic exploration, T cell attacks on infected cells, and phagocytosis. This coalition-based approach is developing submodels in parallel and coordination, allowing us to rapidly advance towards a framework that can drive many independent investigations on COVID-19. Moreover, the novel mix of domain experts is fueling creative advances and new technical capabilities in multiscale tissue modeling. We anticipate that this progress will drive advances in immunology, inflammation, CAR T cell therapy, and virus-driven carcinogenesis for years to come. Interested members of the audience can try this open source framework live in a web browser at https://nanohub. org/tools/pc4covid19. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	MEPI: Models of Infectious Diseases and Control (9:30-11:00am)													 						  			  				  							  							  										  									  	Hyojung Lee								  	 								  		National Institute for Mathematical Sciences, Korea								  									  									  										  			&lt;/br&gt; Transmission dynamics of Coronavirus disease (COVID19) in Korea 								  										  									  									  										  										  											  			The novel coronavirus outbreak has rapidly spread out from Wuhan, Hubei Province, China to other countries since December, 2019. More than 2,000 cases have confirmed since the first case was reported on 20 January, 2020 in South Korea. . The aim of this research is to analyze the transmission dynamics of COVID-19 in South Korea during the early phase of the outbreak. To do so, first, using the 30 confirmed cases, we estimated the basic epidemiological parameters from the data of the symptom onset dates by employing maximum likelihood estimation. Second, super-spreading events were occurred in Daegu, Korea from 31 January, 2020. Accounting for the heterogeneous transmission dynamics, we construct the mathematical model to estimate the geographic reproduction number. Finally, we assess the effect of the control interventions by simulating the various control scenarios to suggest the most effective intervention to halt the transmission									  											  									  									  									  								  								  								  	Robert Smith?								  	 								  		University of Ottawa								  									  									  										  			&lt;/br&gt; Assessing potential COVID-19 outcomes for a university campus with and without physical distancing 								  										  									  									  										  										  											  			By early March of 2020, it became apparent that COVID-19 was going to have a significant impact on institutions throughout society. On March 12, the University of Ottawa commissioned an informal modeling study to aid in their decision-making process. Here we present the results from that study: a differential-equation model was created in order to describe the states of susceptibility, exposure, infection, asymptomatic individuals, recovery and death. Our results shows that, in the worst-case scenario, the epidemic would peak at 7000 cases on campus with 63 dead, starting 115 days after the first case. Reducing contacts by 50% could lower the number of cases and fatalities, but it would expand the timeframe by several years. One day after this modeling study was completed, the University of Ottawa closed campus entirely. It follows that modeling and simulation in the midst of a fast-moving pandemic can be valuable tools for decision-makers. 									  											  									  									  									  								  								  								  	Aurelio A de los Reyes								  	 								  		Institute of Mathematics, University of the Philippines Diliman								  									  									  										  			&lt;/br&gt; Intervention strategies to mitigate HIV/AIDS transmission in the Philippines 								  										  									  									  										  										  											  			Human Immunodeficiency Virus (HIV) impairs a person’s immune system leading to Acquired Immunodeficiency Syndrome (AIDS) – a condition characterized by severe illnesses. The number of HIV infections has more than doubled in the Philippines within the last decade, prompting the need to develop a model of disease transmission in the country. In this study, disease-free and endemic equilibria are obtained, stability analysis is performed, and the basic reproduction number is computed. Available data is utilized to identify model parameters giving insights on the trend of the disease in the country. Furthermore, effectiveness of control measures including precaution, HIV screening, Anti-Retroviral Treatment, and Pre-Exposure Prophylaxis (PrEP) are investigated in the framework of optimal control theory. These various control efforts are compared with regard to cost efficiency and effectivity in minimizing the number of infected individuals. Given limited available control measures, PrEP-only scenario is shown to be most cost effective. 									  											  									  									  									  								  								  								  	Jonggul Lee								  	 								  		National Institute of Mathematical Sciences, South Korea								  									  									  										  			&lt;/br&gt; Spatial heterogeneity and control measures during avian influenza epidemic 2014-2015 in Korea 								  										  									  									  										  										  											  			During the winter of 2014-2015, an epidemic of highly pathogenic avian influenza (HPAI) led to high mortality in poultry and put a serious burden on the poultry industry of the Republic of Korea. Effective control measures considering spatial heterogeneity to mitigate the HPAI epidemic is still a challenging issue. Here we develop a farm-based HPAI model to analyze the spatiotemporal evolution of the epidemic and assess the impact of control strategies. The epidemiological and geographical data of the domestic poultry farms in South Korea are used to find the best-fitted parameters of the model. We investigate potential for two control measures against HPAI: preemptive culling and farm rest. The best culling radius to maximize the final size of the susceptible farms and minimize the total number of culled farms is calculated from the model. The reproductive number of a farm is calculated as an indicator of virus transmission in a given area. Simulation results indicate that this parameter is strongly influenced not only by epidemiological factors such as transmissibility and/or susceptibility of poultry species but also by geographical and demographical factors such as the distribution of poultry farms (or density) and connectivity (or distance) between farms. Based on this result, we suggest the optimal culling radius and number of resting farms with respect to the reproductive number in a targeted area. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				 		  																																		 	MFBM: Stochastic methods for epidemiology and biochemical reaction networks (9:30-11:00am)													 						  			  				  							  							  										  									  	Eben		Kenah								  	 								  		The Ohio State University,	United States,	kenah. 1@osu. edu									  									  									  										  			&lt;/br&gt; Pairwise survival analysis for measuring and controlling risk in epidemics 								  										  									  									  										  										  											  			When a disease is transmitted from person to person, infections in different individuals are not independent. These “dependent happenings” cause fundamental problems for standard principles of epidemiologic study design and data analysis in infectious disease epidemiology. Pairwise survival analysis is an extension of standard survival analysis to the transmission of infectious diseases through households, hospitals, or other situations where there is a clearly-defined population at risk of infection. We show how this framework can be used to assess and control for confounding and selection bias. We then show how these methods can be used to extend traditional derivations of case-control and case-cohort designs to obtain novel study designs for outbreak investigations and public health intervention trials for emerging infections. Finally, we consider the possibilities for incorporating pathogen genetic sequences. 									  											  									  									  									  								  								  								  	Jessica		Stockdale								  	 								  		Simon Fraser University,	Canada,	jessica stockdale@sfu. ca									  									  									  										  			&lt;/br&gt; How long does it take to detect a change in COVID-19 control measures? 								  										  									  									  										  										  											  			Countries around the world have implemented population-wide interventions in efforts to control COVID-19, with varying extent and success. Many jurisdictions are moving to relax measures, while others are re-intensifying them to curb growing spread. But uncertainty remains around the length of time between a population-level change in control measures and its observable impact on detected cases. I will describe our recent work in estimating the time frame for a substantial difference between the cases that occur following a change in control and those that would have occurred under continued strategy, under a compartmental model for disease transmission incorporating physical distancing. Using a likelihood-based approach and data from British Columbia, Canada, we examine how long it takes to detect such a difference given delays and noise in reported cases. We find that these time frames are long: longer than the mean incubation period and the often-used 14 days. 									  											  									  									  									  								  								  								  	Forrest		Crawford								  	 								  		Yale University,	United States,	forrest. crawford@yale. edu									  									  									  										  			&lt;/br&gt; Causal evaluation of infectious disease interventions using stochastic transmission models 								  										  									  									  										  										  											  			Deterministic and stochastic models of infectious disease transmission are widely used to understand the dynamics of epidemics, and project the impact of control measures, in human populations. However, most clinical evaluations of vaccines and other interventions designed to prevent infection do not use these models. Instead, clinical infectious disease epidemiologists use randomized trials and statistical regression models to evaluate interventions. Recent work has shown that these approaches may deliver erroneous estimates of the susceptibility effect of the vaccine, even when treatment is randomized or all baseline confounders are measured. In this presentation, we develop approaches to causal evaluation of interventions in networked populations in randomized and observational trials using a flexible semi-parametric class of stochastic transmission models. We show analytically and by simulation that causal susceptibility and infectiousness effects are identified, and that researchers do not need to specify the functional form of some model components in order to make useful inferences. The approach illustrated in an application to evaluation of risk factors for tuberculosis in a large-scale cluster cohort study. 									  											  									  									  									  								  								  								  	Boseung		Choi								  	 								  		Korea University,	South Korea,	cbskust@korea. ac. kr								  									  									  										  			&lt;/br&gt; Statistical inference for epidemic models using the Survival dynamical system based on the Bayesian approach 								  										  									  									  										  										  											  			I introduce new methods for Bayesian Markov Chain Monte Carlo-based in references in certain of a stochastic model for epidemic data. SIR (Susceptible-Infected-Removed) model is the classical method for modeling infectious disease spread. In this research, we applied solutions of ordinary differential equations describing the large-population limits of Markovian stochastic epidemic models to individual-level SIR model by introducing survival or cumulative hazard functions derived from population-level equations. We call the method as survival dynamical system (SDS). In this research, we also construct an additional estimation step for initial number of susceptible by utilizing a hierarchical Bayesian approach for inference of the number of trials in the Binomial distribution. we applied the SDS approach to data from a 2009 influenza A(H1N1) outbreak at Washington State University. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				 		  																																		 	NEUR: Dynamics and Noise in Neural Networks (9:30-11:00am)													 						  			  				  							  							  										  									  	Cheng Ly								  	 								  		Virginia Commonwealth								  									  									  										  			&lt;/br&gt; The Circuit Mechanisms that alter Spiking Statistics in Mammalian Olfactory Bulb Cells 								  										  									  									  										  										  											  			The olfactory bulb is one of the primary stages of odor processing, possessing a unique architecture hallmarked by fast dendro-dendritic synapses between all inhibitory cells (perioglomerular cells (PGC) and granule cells (GC)) connected to excitatory cells (mitral/tufted cells (MC)).  Although there has been ample theoretical and experimental studies of the olfactory bulb (OB), dissecting the circuit mechanisms of modulation of the first and second order (MC) spiking statistics of populations is lacking.  In particular, we analyze data from our (Shew) lab in spontaneous and odor-evoked states n vivo, with multi-electrode arrays that enable studying second order spiking statistics with simultaneous recordings. Based on a multicompartment large-scale biophysical model, we develop a reduced firing rate model that enables us to efficiently and accurately capture our data.  We show that granule cell inhibition in particular helps decorrelate in the spontaneous state but is unlocked so that there is stimulus-induced correlation in the evoked state; this is in contrast to many cortical sensory systems where there is often stimulus-induced decorrelation. We also consider pharmacological drug applications to manipulate inhibitory synaptic strengths (both PGC and GC) that alter the spiking statistics. Our model qualitatively captures the statistically significant changes. 									  											  									  									  									  								  								  								  	Kelsey Gasior								  	 								  		Florida State								  									  									  										  			&lt;/br&gt; Mathematically Modeling Neuron Biophysics in Response to Ramped Input Current 								  										  									  									  										  										  											  			In the nervous system, olfactory bulb dopamine-secreting neurons (OBDA neurons) process different odors by inhibiting other downstream neurons. Recently, novel experiments were performed wherein current applied to individual neurons was continuously ramped to mimic biologically realistic neuronal input (at Florida State University, the Trombley lab performed) in comparison to previous standard protocols where current is applied in steps. However, this new stimulus protocol raises the questions of what is the proper way to interpret these data and how can mathematical analysis help? In this project, we have developed an integrated experimental-mathematical methodology to study transient dynamics in the electrical activity of single neurons while maintaining a close interdisciplinary collaboration with the biologists who carry out these electrophysiological experiments. One of the aims of our work is to create a positive feedback loop wherein experimental data informs the mathematical model, and in turn, the model directs future experiments. In particular, we are using bifurcation analysis to characterize the onset and offset of tonic spiking as well as the frequency of spiking in a neuron stimulated with both slow and fast applied current ramps. This work allows us to understand how different ion channels shape the transient response dynamics in OBDA neurons. Importantly, we have also developed mathematical tools that can be used to explore the behavior of other cell types as it is our belief that the ramping technique could be extended to study the single-neuron dynamics of all neuron types. The use of fast-slow analysis helps us to understand how the ramped applied current and the slow M-type Potassium channel influence tonic spiking behavior and frequency. Ultimately, this work helps close the gap between mathematical modeling and biological data in computational neuroscience. 									  											  									  									  									  								  								  								  	Jan Kirchner								  	 								  		Max Planck Institute for Brain Research								  									  									  										  			&lt;/br&gt; Local and global organization of synaptic inputs on cortical dendrites 								  										  									  									  										  										  											  			Synaptic inputs on cortical dendrites are organized with remarkable subcellular precision at the micron level. This organization emerges during early postnatal development through patterned spontaneous activity and manifests both locally where nearby synapses tend to share functional properties, and globally with distance to the soma. Recent experimental studies reveal species-specific differences in this organization between mouse, ferret and macaque visual cortex: While in the mouse, synapses are retinotopically organized along the proximal-distal axis of the dendrite, no such organization is present in the ferret or macaque. Instead, here synapses are organized into local clusters according to orientation preference. We propose a computational framework that combines activity derived from retinal waves with functional and structural plasticity to generate these different types of organization across species, as well as across scales by including attenuating backpropagating action potentials. Within this framework, a single anatomical factor -- the size of the visual cortex and the resulting magnification of visual space -- can explain the observed differences. This allows us to make predictions about the organization of synapses also in other species and indicates that the proximal-distal axis of a dendrite might be central in endowing a neuron with its powerful computational capacities. 									  											  									  									  									  								  								  								  	Fereshteh Lagzi								  	 								  		University of Chicago								  									  									  										  			&lt;/br&gt; Cell-type specific inhibitory plasticity can speed up assembly formation and slow down assembly degradation 								  										  									  									  										  										  											  			It is popular to ascribe distinct network functions to the different inhibitory neuron subtypes that makeup cortical circuits. Carving out functionally determined, cell-type specific circuit wiring is an essential component of this hypothesis. However, the plasticity rules for different interneuron subtypes, and how their interactions shape network dynamics are largely unexplored. We use in vitro patch clamp techniques paired with cell-specific optogenetic stimulation to measure the spike timing dependent plasticity (STDP) rules of the inhibitory inputs onto excitatory (E) pyramidal neurons from both Parvalbumin (PV) and Somatostatin (SOM) interneurons in mouse orbital frontal cortex (OFC). Consistent with past studies, PV inhibition shows a symmetric STDP that is often associated with the homeostatic control of excitatory firing rates. By contrast, SOM inhibition shows an asymmetric Hebbian STDP rule, so that recurrent SOM inhibition onto driving E neurons will ultimately depress. To understand the role of these different plasticity mechanisms in network dynamics we exploit large-scale network simulations of networks of spiking neuron model with plastic synapses, along with associated mean field theories of synaptic dynamics. We show that the asymmetric SOM plasticity rule promotes cross-inhibition between distinct E neuron assemblies, effectively providing a mechanism for competition between functionally grouped principle neurons. This competition will enhance computations where input comparisons must be made, as is often the case in decision task where the OFC is known to be essential. However, strong cross inhibition could lead to extreme (and unstable) winner-take-all assembly dynamics. Fortunately, the symmetric PV plasticity rule provides stability for the circuit, ensuring rich network dynamics. Finally, the increased correlations due to the asymmetric Hebbian learning for SOM connections to pyramidal cells can speed up assembly formation during training and slow assembly degradation post training. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	ONCO: Differentiation and stemmness, in cell migration, cancer invasion, and development (9:30-11:00am)													 						  			  				  							  							  										  									  	Luke Tweedy								  	 								  		Beaton Institute of Cancer Research, Glasgow								  									  									  										  			&lt;/br&gt; Seeing around corners: Cell migration is determined by the complex interaction of environmental topology and attractant degradation 								  										  									  									  										  										  											  			Cell migration is often guided by gradients of attractants. Many cells are known to degrade the molecules that attract them, creating dynamic gradients that evolve and change as the cells migrate up them. In unrestricted environments, this enables more robust directed migration over much greater distances than can be explained by chemotaxis to an externally imposed gradient. However, its effects in a complex topology remain unclear. This is important to understand, because the in-vivo topologies in which cells migrate are almost invariably complex. We therefore modelled the behaviour of cells solving a variety of mazes, varying dead end lengths and complexities. We then tested each design experimentally. We found specific rules governing the collective decisions of cells connecting cell speed, attractant diffusivity and dead-end length and complexity. We even found topologies in which a majority of cells would favour a dead end over a path to a large attractant reservoir. This self-generated view of chemotaxis in complex environments will help us better understand immune responses and the patterns of metastasis for some cancers. 									  											  									  									  									  								  								  								  	Niklas Kolbe								  	 								  		Faculty of Mathematics and Physics, Kanazawa University, Japan								  									  									  										  			&lt;/br&gt; Stochastic modelling of TGF-β signalling in single cells 								  										  									  									  										  										  											  			The cytokine TGFb plays an important role in cancer progression as it can both prevent uncontrolled tissue growth and trigger epithelial-to-mesenchymal transition. To better understand the intracellular responses of the cells to the cytokine we have developed a stochastic model that we present in this talk. This model explains heterogeneous signaling dynamics between the cells found in experimental data where time-resolved measurements at the single-cell level were taken. We elaborate on our parameter estimation technique considering the distribution of features in the time paths and demonstrate the accordance of model simulation and measurement data. Joint work with Lukas-Malte Bammert (JGU Mainz), Stefan Legewie (IMB Mainz), Maria Lukacova (JGU Mainz), Lorenz Ripka (IMB Mainz)									  											  									  									  									  								  								  								  	Cicely Macnamara								  	 								  		School of Mathematics and Statistics, University of St. Andrews, UK								  									  									  										  			&lt;/br&gt; Computational modelling and simulation of cancer growth and migration within a 3D heterogeneous tissue 								  										  									  									  										  										  											  			The term cancer covers a multitude of bodily diseases, broadly categorised by having cells which do not behave normally. Since cancer cells can arise from any type of cell in the body, cancers can grow in or around any tissue or organ making the disease highly complex. Our research is focused on understanding the specific mechanisms that occur in the tumour microenvironment via mathematical and computational modeling. We present a 3D individual-based model which allows one to simulate the behaviour of, and spatio-temporal interactions between, cells, extracellular matrix fibres and blood vessels. Each agent (a single cell, for example) is fully realised within the model and interactions are primarily governed by mechanical forces between elements. However, as well as the mechanical interactions we also consider chemical interactions, for example, by coupling the code to a finite element solver to model the diffusion of oxygen from blood vessels to cells. The current state of the art of the model allows us to simulate tumour growth around an arbitrary blood-vessel network or along the striations of fibrous tissue. 									  											  									  									  									  								  								  								  	Filip Klawe								  	 								  		Institute of Applied Mathematics, Heidelberg University, Germany								  									  									  										  			&lt;/br&gt; Mathematical model of stem cell specification in a growing domain 								  										  									  									  										  										  											  			We will study a mathematical framework for analysis and simulation of development of stem cell based, growing organs with cell self-renewal and differentiation regulated by signalling factors. Considered model consists of PDEs which describe concentrations of signals in moving domain Ω(t) and ODEs. One of the ODEs describes evolution of domain Ω(t) The main novelty of our work is a coupling between of PDEs solutions and deformation of the domain. There is no general approach which allow us to obtain mathematical results for such phenomena. However, assuming that Ω(t) is a circle and it is changing uniformly in all directions we are able to prove existence and uniqueness of solution. The considered model may be used to describe the signal concentration (and domain evolution) of shoot apical meristem of arabidopsis thaliana. We present numerical simulations which show that model fits to its biological origin. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	OTHE: Flow and Transport in Complex Tissues, Part I (9:30-11:00am)													 						  			  				  							  							  										  									  	Alys Clark								  	 								  		Auckland								  									  									  										  			&lt;/br&gt; Emerging organ-scale function via large-scale network models of blood flow and exchange 								  										  									  									  										  										  											  			For healthy development and aging we must acquire sufficient oxygen from our environment to supply our metabolic demands. Before we are born we get oxygen from our mother’s blood through the pla- centa, and after birth the lungs take over the placenta’s role exchanging oxygen from the air. These two exchange organs have complex vascular branching structures with multiple generations of asymmetrically branching blood vessels which provide a large capillary surface area for exchange. Disruptions to this vascular branching and heterogeneity in blood delivery (perfusion) have been implicated in a number of diseases. Often, micro-vascular dysfunction contributes to an organ scale pathology, which can be difficult to detect in clinical imaging. We present organ-scale computational models vascular networks in lung and placenta, which take as inputs vascular anatomy derived from imaging, and simulate both haemodynamic and exchange function. This allows functional prediction of how anatomical perturbations in vascular structures contribute to organ function in health and disease, and provides steps toward determining what constitutes normal and abnormal levels of vascular heterogeneity in representative populations. 									  											  									  									  									  								  								  								  	Igor Chernyavsky								  	 								  		Manchester								  									  									  										  			&lt;/br&gt; Structural and physical determinants of transport in complex microvascular networks 								  										  									  									  										  										  											  			Across mammalian species, solute transport takes place in complex microvascular networks. How- ever, despite recent advances in three-dimensional (3D) imaging, there has been poor understanding of geometric and physical factors that determine solute exchange and link the structure and function. Here, we use an example of the human placenta, a fetal life-support system, where the primary exchange units, terminal villi, contain disordered networks of fetal capillaries and are surrounded externally by maternal blood. We show how the irregular internal structure of a terminal villus determines its exchange capacity for a wide range of solutes, integrating 3D image-based properties into new non-dimensional parameters. We characterise the structure-function relationship of terminal villi via a simple and robust algebraic approximation, revealing transitions between flow- and diffusion-limited transport at vessel and network levels. The developed theory accommodates for nonlinear blood rheology and tissue metabolism and offers an efficient method for multi-scale modelling [2]. Our results show how physical estimates of trans- port, based on scaling arguments and carefully defined geometric statistics, provide a useful tool for understanding solute exchange in placental and other complex microvascular systems. 									  											  									  									  									  								  								  								  	Felix Meigel								  	 								  		Max Planck Institue								  									  									  										  			&lt;/br&gt; Robust increase in supply by local vessel dilation in globally coupled microvasculature 								  										  									  									  										  										  											  			Vascular networks pervade all organs of animals and are the paradigm of adaptive transport net- works. In the brain, neural activity induces changes in blood flow by locally dilating vessels in the brain microvasculature. How can the local dilation of a single vessel increase flow-based metabolite supply, given that flows are globally coupled within the microvasculature? Here, we build a theoretical model for flow-based transport and absorption of nutrients and determine how capillary geometry and network topology affect the control by active adaptation. On the level of an individual capillary, we derive ana- lytically how vessel parameters affect the change in supply due to dilation. Solving the supply dynamics for a rat brain microvasculature, we find one parameter regime to dominate physiologically. This regime allows for robust increase in supply independent of the position in the network, which we explain ana- lytically. We show that local coupling of vessels promotes spatially correlated increased supply by dilation. 									  											  									  									  									  								  								  								  	Edwina Yeo								  	 								  		Oxford								  									  									  										  			&lt;/br&gt; Magnetically-driven Cell Aggregation in Blood 								  										  									  									  										  										  											  			In regenerative medicine magnetic cell targeting aims to deliver stem cells precisely to an injury site. For a safe and effective therapy, the stem cells must be delivered in large numbers under physiological flow conditions, but critically, the aggregation of cells at the target site must be controlled. Mathematical modelling offers insight into the dynamics of the system and allows efficient examination of physiological and therapeutic parameters. We adapt existing continuum models for the delivery of magnetic nanoparticles [2] to magnetic cell delivery. Cells are captured on the vessel wall closest to the magnet, this leads to the growth of a solid cell aggregate which obstructs the flow. We determine how the interplay between aggregate growth af- fects stem cell capture and identify parameter regimes in which potentially dangerous vessel blockage is predicted. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	POPD: Emergence and Stability of Population Structure and Biological Aggregates Across Scales (9:30-11:00am)													 						  			  				  							  							  										  									  	Olivia Chu								  	 								  		Princeton University								  									  									  										  			&lt;/br&gt; An Adaptive Voter Model in Heterogeneous Environments 								  										  									  									  										  										  											  			In human social systems, it is natural to assume that individuals’ opinions influence and are influenced by their interactions. Mathematically, it is common to represent such systems as networks, where nodes are individuals and edges denote connections. Adaptive network models explore the dynamic relationship between node properties and network topology. For opinion dynamics, adaptive voter models provide two mechanisms through which changes can occur within the network. First, through homophily, an edge forms between two individuals who already agree; second, through social learning, an individual adopts one of their neighbor’s opinions. Central to these models is assortative mixing: individuals more frequently attach to those who are similar to them, which facilitates the formation of sub-communities of like-minded individuals. However, it is not always the case that individuals want to cluster into homogeneous groups. Instead, they might attempt to surround themselves with individuals who both agree and disagree with them, in an effort to attain a balance of inclusion and distinctiveness in their social environments. In this work, we explore the effects that such preferences for heterogeneous environments have on the dynamics of the adaptive voter model. 									  											  									  									  									  								  								  								  	Feng Fu								  	 								  		Dartmouth College								  									  									  										  			&lt;/br&gt; How phenotypic similarity begets cooperation 								  										  									  									  										  										  											  			Tag-based cooperation, or cooperation based on phenotypic similarity, has long been seen as a potent mechanism of cooperation. The evolutionary origin and variability of tag-based cooperation has yet to be fully answered. Here we show analytically and by means of simulations that tag-based cooperation can always evolve by natural selection in the presence of sufficient tag diversity. Our work provides fundamental insights into understanding the widespread of tag-based cooperation in the real world from microbial populations to complex human societies. 									  											  									  									  									  								  								  								  	Yuriy Pichugin								  	 								  		Max Planck Institute for Evolutionary Biology								  									  									  										  			&lt;/br&gt; Evolution of clonal life cycles: recipes for multicellularity, equal split, and single-cell bottleneck 								  										  									  									  										  										  											  			There is a huge variety in reproduction modes observed even among the simplest organisms. Many species are unicellular but some form simple colonies. Some of the colonies reproduce by splitting into equal parts others produce unicellular propagules. What is the driving force, which shapes the evolution of life cycles? What are the conditions promoting uni- or multi-cellular life cycles? We developed the stage-structured matrix population model of the growth and reproduction of unstructured multicellular organisms. Using this model, we investigated the conditions favoring the evolution of diverse life cycles: unicellular, with an equal split of a colony, and with the reproduction via single-cell bottleneck. We identified the set of profiles of size-dependent growth and death rates promoting each of these life cycles. We found that the conditions promoting a single-cell bottleneck are the steady improvement in the performance of the colony with its size. At the same time, the equal splits require the sudden rise in growth rate (drop in the death rate) at the size of the newborn offspring. Altogether, our findings demonstrate the patterns behind the evolution of multicellular life cycles. 									  											  									  									  									  								  								  								  	Pawel Romanczuk								  	 								  		Hombolt University of Berlin								  									  									  										  			&lt;/br&gt; Flocking in complex environments – attention trade-offs in collective information processing 								  										  									  									  										  										  											  			The ability of biological and artificial collectives to outperform solitary individuals in a wide variety of tasks depends crucially on the efficient processing of social and environmental information at the level of the collective. Here, we model collective behavior in complex environments with many potentially distracting cues. Counter-intuitively, large-scale coordination in such environments can be maximized by strongly limiting the cognitive capacity of individuals, where due to self-organized dynamics the collective self-isolates from disrupting information. We observe a fundamental trade-off between coordination and collective responsiveness to environmental cues. Our results offer important insights into possible evo- lutionary trade-offs in collective behavior in biology and suggests novel principles for design of artificial swarms exploiting attentional bottlenecks. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  			             		    		  									                15 minute break (11:00-11:15am)                								  		    		      		        Sub-group minisymposia (11:15am)		        			  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	CDEV: Understanding development through theory and experiment (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Shane Hutson								  	 								  		Vanderbilt University								  									  									  										  			&lt;/br&gt; Something's wrong in the (cellular) neighborhood: Mechanisms of wound detection in epithelia 								  										  									  									  										  										  											  			When a sheet of epithelial cells is wounded, cells around the wound are recruited from their near quiescent state to reactivate motility and proliferation behaviors similar to early development. As the first stage of that recruitment, surrounding epithelial cells undergo a dramatic increase in cytosolic calcium. This increase occurs quickly: calcium floods into damaged cells within 0. 1 s, moves into adjacent cells over ~20 s, and appears in a much larger set of surrounding cells via a delayed second expansion over 40-300 s. Nonetheless, increased calcium is a reporter; cells must detect wounds even earlier. Using the calcium response as a proxy for wound detection, we identify an upstream G-protein-coupled-receptor (GPCR) signaling pathway, including the receptor and its chemokine ligand. We present experimental and computational evidence that the pathway involves the chemokine acting as a protease bait. Its pro-peptide form is already present extracellularly and is converted to an active form by multiple proteases released during cell lysis/wounding. We will discuss the experimental evidence and corresponding computational models developed to test the plausibility of these hypothesized mechanisms. 									  											  									  									  									  								  								  								  	Stanislav	Y	Shvartsman								  	 								  		Princeton University								  									  									  										  			&lt;/br&gt; From data to knowledge in studies of developmental patterning 								  										  									  									  										  										  											  			I will present our ongoing work on model-based data integration in studies of pattern formation in the early Drosophila embryo, an experimental system that offers unrivaled opportunities for combining cutting edge molecular genetics, live imaging, and omics-based techniques. 									  											  									  									  									  								  								  								  	Celeste Nelson								  	 								  		Princeton University								  									  									  										  			&lt;/br&gt; How to fold a tube into a lung 								  										  									  									  										  										  											  			'Our real teacher has been and still is the embryo, who is, incidentally, the only teacher who is always right. ' – Viktor Hamburger Evolution has generated an enormous diversity of biological form. Given this diversity, it is highly likely that every tissue structure that one can imagine has been built by the embryo of one species or another. We are interested in uncovering the physical (mechanical) mechanisms by which epithelial sheets fold themselves into branching tubes in the embryo, and using those mechanisms to engineer tissues in culture. Over the past half century, developmental biologists have identified several biochemical signaling pathways and genetic control mechanisms necessary for tissue morphogenesis. In parallel, biological systems must obey Newton’s laws of motion, and physical forces need to be generated in order to sculpt simple populations of cells into complex tissue forms. Inspired by the evolutionary diversity of embryonic forms, we have created microfabrication- and lithographic tissue engineering-based approaches to investigate the mechanical forces and downstream signaling pathways that are responsible for generating the airways of the lung. I will discuss how we combine these experimental techniques with computational models to uncover the physical forces that drive morphogenesis. I will also describe efforts to uncover and actuate the different physical mechanisms used to build the airways in lungs from birds, mammals, and reptiles. 									  											  									  									  									  								  								  								  	Roeland Merks								  	 								  		Leiden University								  									  									  										  			&lt;/br&gt; Mathematical Modeling of Cell-Extracellular Matrix Interactions to Explain Collective Cell Behavior and Cell Migration 								  										  									  									  										  										  											  			During embryonic development, the behavior of individual cells must be coordinated to create the large scale patterns and tissue movements that shape the whole embryo. Apart from chemical signals exchanged between cells, a prominent role is played by the extracellular matrix (ECM); these are the hard or jelly materials (e. g. collagens, fibronectin) that form the micro-environment of many cells in tissues. To get a better grip on the role of the extracellular matrix in determining the behavior of cells, we are developing mathematical and computational approaches to analyse the interactions of the mechanics of cells and the extracellular matrix (ECM). The cell models are usually based on the Cellular Potts model, whereas the ECM is model is based on a variety of approaches, including the finite-element model and molecular dynamics. I will discuss how these mathematical approaches help to elucidate the regulation of cell migration, collective cell behavior during angiogenesis and other mechanisms, including epithelial branching and the evolution of multicellularity. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				 		  				 		  				  				  				  				  				  				  				  				  				  																																		 	EDUC: Education Minisymposium II (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Chonilo	S. 	Saldon								  	 								  		Zamboanga del Norte National High School								  									  									  										  			&lt;/br&gt; High School Students' Interest in Applied Mathematics 								  										  									  									  										  										  											  			With the the implementation of the Enhanced Basic Education Curriculum in the Philippines doubled with the students'access to college scholarship,more students are considering Mathematics and Science-related courses. Using a mixed-method approach, this study looks into the beliefs, interest and motivation in pursuing applied mathematics in college. Implications shall be discussed in terms of career advising on secondary level. 									  											  									  									  									  								  								  								  	Widodo Samyono								  	 								  		Jarvis Christian College								  									  									  										  			&lt;/br&gt; Jarvis Summer Undergraduate Research Experience 2020 								  										  									  									  										  										  											  			Jarvis Christian College is a small liberal arts HBCU at Hawkins, Texas. With the NSF Funding through Targeting Infusion Project (TIP) Historically Black Colleges and Universities Undergraduate Programs (HBCU-UP) we were able to have a summer undergraduate research in computational and mathematical biology. In this presentation we would like to share our challenges and opportunities for doing undergraduate research in computational and mathematical biology with underrepresented minority students in STEM. We will explain how we prepare ourselves and our students and how we change the challenges to opportunities. 									  											  									  									  									  								  								  								  	Winfried Just								  	 								  		Ohio University								  									  									  										  			&lt;/br&gt; COVID-19 modeling for quantitative literacy courses 								  										  									  									  										  										  											  			There is a deluge of information about the COVID-19 pandemic. Many of our students are struggling with making sense of it all, especially the science and math behind the various models. This talk will present a collection of materials that can be used in quantitative literacy courses to sort out some of the confusion and demonstrate how the math that is usually taught in such courses helps in understanding the real world and making good decisions. In these materials, the models are explained by Alice, a student of epidemiology, to three of her peers from other disciplines. The protagonists bring a variety of attitudes and concerns to the discussions. Together, they sort things out and gain remarkable insights with using very simple mathematics. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	IMMU: Increasingly biologically accurate models of influenza A virus infection spread in vivo &amp; in vitro (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Christian Quirouette								  	 								  		Ryerson University								  									  									  										  			&lt;/br&gt; A mathematical model describing the localization and spread of influenza A virus infection within the human respiratory tract 								  										  									  									  										  										  											  			Within the human respiratory tract (HRT), virus diffuses through the periciliary fluid (PCF) bathing the epithelium. But it also undergoes advection: as the mucus layer sitting atop the PCF is pushed along by the ciliated cell’s beating cilia, the PCF and its virus contents are also pushed along, upwards towards the nose and mouth. Many mathematical models (MMs) describe the course of influenza virus infections in vivo, but none consider the impact of both diffusion and advection on the infection’s kinetics and localization. Our MM represents the HRT as a one-dimensional track extending from the nose down to the lower HRT, wherein stationary cells interact with virus which moves within (diffusion) and along with (advection) the PCF. Diffusion was found to be negligible in the presence of advection which effectively sweeps away virus, preventing infection from disseminating below the depth of deposition. Higher virus production rates (10-fold) are required at higher advection speeds (40 micron/s) to maintain equivalent infection severity and timing. Because virus is entrained upwards, upper parts of the HRT located downstream of the advection flow see more virus than lower parts, and so infection grows, peaks, and resolves later in the lower HRT. Clinically, the infection would appear to progress from the upper towards the lower HRT, as reported in mice, even when the lower HRT infection precedes, and indeed causes, that in the upper HRT. When the spatial MM is expanded to include cellular regeneration and an immune response, it reproduces tissue damage levels reported in patients. It can also captures the kinetics of both seasonal and avian strain infections via parameter changes consistent with reported differences between these strains. This new MM offers a convenient and unique platform from which to study the localization and spread of respiratory viruses (flu, RSV, COVID-19) within the HRT during an infection. 									  											  									  									  									  								  								  								  	Amber Smith								  	 								  		University of Tennessee Health Science Center								  									  									  										  			&lt;/br&gt; Modeling Influenza–Mediated Acute Lung Injury 								  										  									  									  										  										  											  			Influenza viruses infect millions of individuals each year and cause a significant amount of morbidity and mortality. Understanding how the virus spreads within the lung, how efficacious host immune control is, and how each influences acute lung injury and disease severity is critical to combat the infection. Thus, we used an integrative model-experiment exchange to establish the dynamical connections between infection, lung injury, and disease kinetics. Examining these connections during neuraminidase inhibitor (NAI) therapy further validated the model and analysis, and suggested that profound effects on lung injury are possible with minimal changes to host-pathogen kinetics. This work provides important biological and mathematical insight and enhances our ability to effectively forecast the disease and antiviral efficacy. 									  											  									  									  									  								  								  								  	Daniel Rüdiger								  	 								  		Max Planck Institute for Dynamics of Complex Technical Systems, Magdeburg, Germany								  									  									  										  			&lt;/br&gt; Multiscale model of DIP replication and its effects on influenza A virus infection in animal cell culture 								  										  									  									  										  										  											  			Defective interfering particles (DIPs) that lack part of the viral genome are considered for use as antivirals against flu, because they can strongly impede wild type influenza A virus (IAV) replication. Multiple mathematical modeling approaches have been applied to examine the mechanisms of DIP inter- ference. However, these models focused on either the intra- or the extracellular level of virus replication. In this work, we extend a recently published multiscale model to describe DIP propagation in animal cell cultures infected by IAV. This new model covers fundamental steps during the intracellular replica- tion and the spread of DIPs on the population level. Particularly, the model incorporates the infection conditions, i. e. the multiplicity of infection (MOI), which may change drastically during an infection and represent a crucial factor for IAV replication and DIP interference. To elucidate DIP infection dynamics we conducted a set of cultivations observing the effects of DIPs on wild type virus replication using various IAV and DIP seed virus concentrations. Based on these experimental data, we calibrated our multiscale model to enable the prediction of DIP-induced infection dynamics for a wide range of MOI conditions. Furthermore, we used the model to elucidate options for antiviral therapy, i. e. the DIP to IAV ratio required to inhibit the progression of an influenza infection in animals or humans. In summary, we established a mathematical model that provides a comprehensive description of DIP replication on the intra- and extracellular level to facilitate the development of antiviral therapies, and to describe DIP production in animal cell culture for therapeutic use. 									  											  									  									  									  								  								  								  	João Rodrigues Correia Ramos								  	 								  		Max Planck Institute for Dynamics of Complex Technical Systems, Magdeburg, Germany								  									  									  										  			&lt;/br&gt; A dynamic model for cell growth, metabolism and virus production of MDCK suspension cells 								  										  									  									  										  										  											  			Cell culture-based influenza virus production is well established and different virus replication stages have been studied experimentally in detail. In addition, quantitative mathematical models have been derived that describe dynamics of viral replication at the intracellular and the cell population level. However, to better understand the complex interplay between the virus and its host cell, metabolism during virus replication should be considered in more detail. In this context, MDCK suspension cells were cultivated in shaker flasks and concentrations of external and internal metabolites monitored during cell growth and influenza A virus infection. To characterize the impact of virus infection on host cell metabolism, we have formulated a dynamic mathematical model combining a segregated cell growth model with a structured model of intracellular metabolism in a similar manner as models established for other cell lines. Overall, it considers the dynamics of cell growth, virus production, substrates, metabolic by-products and concentration of key intracellular metabolites from glycolysis, citric acid cycle (TCA), glutaminolysis and pentose phosphate pathway. Model parameters were fitted using mock-infections, and simulations describe well the time courses of the viable cell concentration, mean cell diameter, external substrates, metabolic by-products, and key intracellular metabolites. After virus infection, using the same set of parameters, the model also describes well the dynamics of the viable cell concentration, mean cell diameter changes, substrates, and the virus titer. Only minor differences were found for infected versus mock cultures in the glycolytic pathway. Nev- ertheless, differences between experimental data and model simulations regarding metabolites of TCA and metabolic by-products suggest changes in metabolism, which might be virus-induced. As virus in- fection also affects apoptosis and cell lysis, the interpretation of such differences is not straightforward and experimental findings as well as parameters fitted need to be evaluated in more detail to confirm a virus-related impact on the TCA. Overall, this work will contribute to a better understanding of the complex interplay between cell growth, changes in cell size, virus production and metabolism and support the identification of parameters rele- vant for increasing specific viral productivity of MDCK suspension cells. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	MEPI: COVID-19 Contributed Talks III (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Edward Hill								  	 								  		University of Warwick								  									  									  										  			&lt;/br&gt; Predictions of COVID-19 dynamics in the UK: short-term forecasting, analysis of potential exit strategies and impact of reopening schools 								  										  									  									  										  										  											  			Efforts to suppress transmission of SARS-CoV-2 in the UK saw non-pharmaceutical interventions being invoked throughout March 2020, culminating in the application of lockdown measures. From mid-April, COVID-19 cases were declining and there was good evidence to suggest that the effective reproduction number had dropped below 1. A multi-phase relaxation plan to emerge from lockdown was put in place, including primary schools being scheduled to partially reopen in England on 1st June. Regarding the future course of the COVID-19 outbreak in the UK, mathematical models have provided, and continue to provide, short and long term forecasts to support evidence-based policymaking. We present a deterministic, age-structured transmission model for SARS-CoV-2 that uses real-time data on confirmed cases requiring hospital care and mortality to provide predictions on epidemic spread in ten regions of the UK. The model captures a range of age-dependent heterogeneities, reduced transmission from asymptomatic infections and produces a good fit to the key epidemic features over time. We illustrate how the model has been used to generate short-term predictions, assess potential lockdown exit strategies, and the impact of children returning to school. We found that significant relaxation of social distancing measures from 7th May could lead to a rapid resurgence of COVID-19 disease and the health system being quickly overwhelmed by a second epidemic wave. On reopening schools, whilst children returning to educational establishments results in more mixing between children and an increase in transmission of the disease, the magnitude of that increase can be low dependent upon the age-groups that return to school and the behaviour of the remaining population. Our work confirmed the effectiveness of stringent non-pharmaceutical measures in March 2020 to suppress the epidemic. It also provided support for the need for a cautious, measured approach to relaxation of lockdown measures, to support the health service through subduing demand on hospital beds. Finally, it indicated that any reopening of schools would result in increased mixing and infection amongst children and the wider population, although the opening of schools alone at that time was unlikely to push the value of R above one. 									  											  									  									  									  								  								  								  	Ruian Ke								  	 								  		Los Alamos National Lab								  									  									  										  			&lt;/br&gt; Estimating the epidemic growth rate and the reproductive number R0 of SARS-CoV-2 and implications for vaccination 								  										  									  									  										  										  											  			SARS-CoV-2 is a novel pathogen causes the COVID-19 pandemic. Some of the basic epidemiological parameters, such as the exponential epidemic growth rate and R0 are debated.  We collected and analyzed data from China, eight European countries and the US using a variety of inference approaches.  In all countries, the early epidemic grew exponentially at rates between 0. 19-0. 29/day (epidemic doubling times between 2. 4-3. 7 days).  I will discuss the appropriate serial intervals to estimate the basic reproductive number R0 and argue that existing evidence suggests a highly infectious virus with an R0 likely between 4. 0 and 7. 1.  Further, we found that similar levels of intervention efforts are needed, no matter the goal is mitigation or containment. We further show that regular repeated vaccinations will be required to maintain herd immunity if the duration of protective immunity is consistent with other known coronaviruses and that individual-level heterogeneity in protective immunity can significantly affect vaccination policy. 									  											  									  									  									  								  								  								  	Thi Mui Pham								  	 								  		UMCUtretcht								  									  									  										  			&lt;/br&gt; Impact of self-imposed prevention measures and short-term government intervention on mitigating and delaying a COVID-19 epidemic 								  										  									  									  										  										  											  			Background: With new cases of COVID-19 surging around the world, many countries have to prepare for moving beyond the containment phase. Prediction of the effectiveness of non-case-based interventions for mitigating, delaying or preventing the epidemic is urgent, especially for countries affected by the ongoing seasonal influenza activity. Methods: We developed a deterministic transmission model to evaluate the impact of self-imposed pre- vention measures (handwashing, mask-wearing, and social distancing) due to the spread of COVID-19 awareness and of short-term government-imposed social distancing on the peak number of diagnoses, attack rate and time until the peak number of diagnoses. Findings: For fast awareness spread in the population, self-imposed measures can significantly reduce the attack rate, diminish and postpone the peak number of diagnoses. A large epidemic can be prevented if the efficacy of these measures exceeds 50%. For slow awareness spread, self-imposed measures reduce the peak number of diagnoses and attack rate but do not affect the timing of the peak. Short-term government policies on social distancing (e. g. community-wide quarantine) that are initiated early can only postpone the epidemic peak whereas later implementation can lead to a reduction of the attack rate and a flatter peak. Interpretation: Handwashing, mask-wearing and social distancing as a reaction to information dissem- ination about COVID-19 can be effective strategies to mitigate and delay the epidemic. We stress the importance of rapidly spreading awareness on the use of these self-imposed prevention measures in the population. Early-initiated short-term government-imposed social distancing can buy time for healthcare systems to prepare for an increasing COVID-19 burden. 									  											  									  									  									  								  								  								  	Jasmina Panovska-Griffiths								  	 								  		University College London								  									  									  										  			&lt;/br&gt; Determining the optimal strategy for reopening schools, workplaces and society in the UK: modelling patterns of reopening, the impact of test and trace strategies and risk of occurrence of a secondary COVID-19 pandemic wave 								  										  									  									  										  										  											  			Background: As evidence is emerging that the UK lockdown has slowed the spread of the pandemic, it is important to assess the impact of any changes in strategy, including school reopening and broader relaxation of physical distancing measures moving forward. This work uses an individual-based model to predict the impact of two possible strategies for reopening schools to all students (full-time versus part-time rotas) in the UK from September 2020, in combination with different assumptions about the scale-up of testing. Methods: We use Covasim, a stochastic agent-based model for transmission of COVID-19, calibrated to the UK epidemic. The model describes individuals’ contact networks stratified into household, school, workplace and community layers, and uses demographic and epidemiological data from the UK. We simulate six different scenarios, representing the combination of two school reopening strategies and three testing scenarios, and estimate the number of new infections, cases and deaths, as well as the effective reproduction number (R) under different strategies. To account for uncertainties within the stochastic simulation, we also simulated different levels of infectiousness of children and young adults under 20 years old compared to older ages. Findings: We found that with increased levels of testing (between 59% and 87% of symptomatic people tested at some point during an active COVID-19 infection, depending on the scenario), and effective contact tracing and isolation, an epidemic rebound may be prevented. Assuming 68% of contacts could be traced, we estimate that 75% of those with symptomatic infection would need to be diagnosed and isolated if schools return full-time in September, or 65% if a part-time rota system were used. If only 40% of contacts could be traced, these figures would increase to 87% and 75%, respectively. However, without such measures, reopening of schools together with gradual relaxing of the lockdown measures are likely to induce a secondary wave that would peak in December 2020 if schools open full-time in September, and in February 2021 if a part-time rota system were adopted. In either case, the secondary wave would result in R rising above 1 and a resulting secondary wave of infections 2-2. 3 times the size of the original COVID-19 wave. When infectiousness of &lt;20 year olds was varied from 100% to 50% of that of older ages, we still find that comprehensive and effective TTI would be required to avoid a secondary COVID-19 wave. Interpretation: To prevent a secondary COVID-19 wave, relaxation of physical distancing including reopening schools in the UK must be accompanied by large-scale testing of symptomatic individuals and effective tracing of their contacts, followed by isolation of diagnosed individuals. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  																																		 	MFBM: Stochastic methods for epidemiology and biochemical reaction networks (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Ankit		Gupta								  	 								  		ETH Zurich,	Switzerland,	ankit. gupta@bsse. ethz. ch									  									  									  										  			&lt;/br&gt; The probability distribution of the reconstructed phylogenetic tree with occurrence data 								  										  									  									  										  										  											  			Stochastic birth-death processes are extensively used in epidemiology to model the underlying population dynamics of infected individuals. In such models the infection history of extant population naturally gives rise to a phylogenetic tree which can be used to study the evolution of the epidemiological process in the past. In this talk we study the problem of computing the probability distribution of such phylogenetic trees arising from partially sampled birth death processes. We consider observations from three distinct sampling schemes. First, individuals can be sampled and removed, through time, and included in the tree. Second, they can be occurrences which are sampled and removed through time and not included in the tree. Third, extant individuals can be sampled and included in the tree. The outcome of the process is thus composed of the reconstructed phylogenetic tree spanning all individuals sampled and included in the tree, and a timeline of occurrence events which are not placed along the tree. We derive a formula for computing the joint probability density of this outcome, which can readily be used to perform maximum likelihood or Bayesian estimation of the parameters of the birth-death model. In the context of epidemiology, our probability density enables the estimation of transmission rates through a joint analysis of epidemiological case count data and phylogenetic trees reconstructed from pathogen sequences. 									  											  									  									  									  								  								  								  	Grzegorz		Rempala								  	 								  		The Ohio State University,	United States,	rempala. 3@osu. edu									  									  									  										  			&lt;/br&gt; Mathematical Model of a Pandemic: 2019-20 Coronavirus Analysis 								  										  									  									  										  										  											  			The modeling of a pandemic may be typically divided into three time phases: the early stochastic one, the mid-course deterministic one and the final, also stochastic. I will show on the example of Corona virus pandemic of 2019 how such model may be effectively used for predictions about disease dynamics applying both multiscale approximation and the idea of survival dynamical system obtained from the aggregate network model. 									  											  									  									  									  								  								  								  	Hye-Won		Kang								  	 								  		University of Maryland at Baltimore County,	United States	hwkang@umbc. edu									  									  									  										  			&lt;/br&gt; A stochastic model for enzyme clustering in glucose metabolism 								  										  									  									  										  										  											  			A sequence of metabolic enzymes tightly regulates glycolysis and gluconeogenesis. It has been hypothesized that these enzymes form multienzyme complexes and regulate glucose flux. In the previous work, it was identified that several rate-limiting enzymes form multienzyme complexes and control the direction of glucose flux between energy metabolism and building block biosynthesis. A recent study introduced a mathematical model to support this finding, in which the association of the rate-limiting enzymes into multienzyme complexes in included. However, this model did not fully account for dynamic and random movement of the enzyme clusters, as observed in the experiment. In this talk, I will introduce a stochastic model for enzyme clustering in glucose metabolism. The model will describe both the enzyme kinetics and the spatial organization of metabolic enzyme complexes. Then, I will discuss underlying model assumptions and approximation methods									  											  									  									  									  								  								  								  	Wasiur		KhudaBukhsh								  	 								  		The Ohio State University,	United States,	khudabukhsh. 2@osu. edu									  									  									  										  			&lt;/br&gt; Incorporating delays and non-Markovian dynamics into biochemical reaction networks 								  										  									  									  										  										  											  			Markov models for many biophysical systems are often found to be unrealistic because of the assumption that the interactions occur instantaneously or that the inter-reaction times follow an exponential distribution. In this talk, we consider relaxing those assumptions by incorporating delays into the system’s dynamics. We show that this modification leads to approximations by means of Partial Differential Equation (PDE) limits instead of the classical Ordinary Differential Equation (ODE) ones. Describing the dynamics by means of measure-valued processes is at the heart of such approximations. While the theory is developed for a general class of chemical reaction networks, we will also discuss some concrete examples. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	NEUR: Stochastic and Mean-Field Analysis in Neuroscience (11:15am-12:45pm)													 						  			  				  							  							  										  									  	James MacLaurin								  	 								  		NJIT								  									  									  										  			&lt;/br&gt; Coarse-Graining of the Primary Visual Cortex 								  										  									  									  										  										  											  			Experimental evidence indicates that the dynamic response of the primary visual cortex to stimulii continues over relatively slow timescales of O(100ms). Typically, the initial 'bump' of activity, that is centered at a particular orientation, slowly softens and dissipates after the stimulus is removed. To explain this phenomenon, we develop a realistic microscopic neural field model, and perform a coarse-graining to obtain a macroscopic neural field equation. The microscopic model contains slow synaptic dynamics, and stochasticity resulting from synaptic transmission failure. At the macroscopic level, after a coarse-graining procedure, this stochasticity results in a neural field equation with an integral convolution (as is standard in neural field equations) but also a spatial Laplacian, and with the diffusion coefficient proportional to the population activity.  Our macroscopic equation can be thought of as a spatially-extended population density equation: it combines the strengths of neural-field equations and population density equations into a single formalism. It allows a deeper understanding of how changes in the average synchronization of neurons affects the macroscopic dynamics. It partly parallels the efforts of Coombes et al in recent years to derive such 'next-generation neural field equations'. We next perform a Large Deviations analysis to study the typical stochastic fluctuations about the limiting equation. This allows us to determine the most likely abnormal behavior that could be induced in the system by finite size effects. This work is based on a preprint (joint with Bart Krekelberg) entitled 'Coarse-Graining of Neural Networks with Stochastic Dynamic Connections. '									  											  									  									  									  								  								  								  	Youngmin Park								  	 								  		Brandies U								  									  									  										  			&lt;/br&gt; Dynamics of Vesicles Driven into Closed Constrictions by Molecular Motors 								  										  									  									  										  										  											  			We study the dynamics of a model of membrane vesicle transport into dendritic spines, which are bulbous intracellular compartments in neurons driven by molecular motors. We explore the effects of noise on the reduced lubrication model proposed in (Fai et al, Active elastohydrodynamics of vesicles in narrow, blind constrictions. Phys. Rev. Fluids, 2 (2017), 113601). The Fokker-Planck approximation fails to capture mean first passage times of velocity switching (tug-of-war effect), and the agent-based model is computationally expensive. For relatively efficient computations, we turn to the master equation and find that it requires an additional calculation to account for non-equilibrium dynamics in the underlying myosin motor population. We discuss remaining questions and future directions in this ongoing work. 									  											  									  									  									  								  								  								  	Victor Matveev								  	 								  		NJIT								  									  									  										  			&lt;/br&gt; Mass-Action vs Stochastic Modeling of First Passage Time to Ca2+-Triggered Vesicle Release 								  										  									  									  										  										  											  			Like most physiological cell mechanisms, synaptic neurotransmitter vesicle release (exocytosis) is characterized by a high degree of variability in all steps of the process, from Ca2+ channel gating to the final triggering of membrane fusion by the SNARE machinery. The associated fluctuations can be quite large since only a small number of Ca2+ ions enter the cell through a single channel during an action potential, and further increased by the stochasticity in the Ca2+ binding to Ca2+ buffers and sensors. This leads to a widely-held assumption that solving mass-action reaction-diffusion equations for buffered Ca2+ diffusion does not provide sufficient insight into the underlying Ca2+-dependent cell processes. However, several comparative studies showed a surprisingly close agreement between deterministic and trial-averaged stochastic simulations of Ca2+ diffusion, buffering and binding, as long as Ca2+ channel gating is not Ca2+ dependent. We present further comparison of stochastic and mass-action simulations, focusing on Ca2+ dynamics downstream of Ca2+ channel gating and considering spatially-resolved reaction-diffusion modeling in 3D. Namely, we compare the distributions of first-passage-times (FPT) to full binding of the model Ca2+ sensor for vesicle fusion obtained using stochastic and deterministic approaches. We note that in the deterministic formulation, FPT density is equivalent to the time-dependent rate of the final irreversible transition to the fusion-ready sensor state. We show that the discrepancy between deterministic and stochastic approaches in simulating the FPT density can be surprisingly small even when only as few as 40 ions enter the cell per single channel-vesicle complex, despite the fluctuations caused by the Ca2+ binding and unbinding. Further, we demonstrate this close agreement between stochastic and deterministic FPT computation using a highly simplified two-compartment model, whereby the FPT density can be computed exactly using either of the two approaches. The reason for the close agreement between the two methods is that in the absence of Ca2+-induced Ca2+-release, the non-linearities in the exocytosis process involve only bi-molecular reactions. Therefore, the discrepancy between the two approaches is primarily determined by the size of correlations between reactant molecule number fluctuations rather than the fluctuation amplitudes. The small size of reactant correlations is in turn determined by the relationship between the rate of diffusion relative the rate of Ca2+ buffering and binding, as suggested in prior studies. In most common parameter regimes, FPT density is not very sensitive to the fluctuations in the rates of Markovian transitions between distinct sensor states arising from the Ca2+ fluctuations, which in turn leads to small discrepancies between the two approaches. This work is supported by NSF grant DMS-1517085. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	ONCO: Applications and challenges of using quantitative imaging data for biologically-based mathematical oncology (11:15am-12:45pm)													 						  			  				  							  							  										  									  	David A. Hormuth, II								  	 								  		The University of Texas at Austin, Austin, Texas USA								  									  									  										  			&lt;/br&gt; Translating image driven models of response to radiation therapy from the pre-clinical to clinical setting 								  										  									  									  										  										  											  			Magnetic resonance imaging (MRI) is able to provide quantitative, non-invasive measurements of tissue and tumor properties related to perfusion, vascularity, proliferation, and cellularity that can be used to observe tumor growth throughout the course of therapy. We and others have leveraged this type of quantitative data to initialize and calibrate biologically-based models of tumor growth and response. Here, we investigate the use of diffusion weighted (DW-) MRI and dynamic contrast-enhanced (DCE-) MRI to non-invasively estimate tumor cellularity and vascularity, respectively, in high grade gliomas at the pre-clinical and clinical levels. At the pre-clinical level, we have developed a 3D, two-species reaction diffusion-based model describing the spatial-temporal evolution of tumor and blood volume fractions [1] during the course of fractionated radiation therapy. Images collected during therapy are used to calibrate tumor-specific growth and response parameters. These calibrated parameters are then used in a forward evaluation of the model to predict response following therapy. We observed less than 12. 3% error in tumor volume predictions. At the voxel-level, we observed 6. 6% and 14. 1% in voxel-wise estimates of tumor and blood volume fraction, respectively. At the clinical level, we have developed a 3D reaction diffusion-based model describing the spatial-temporal evolution of tumor cellularity during and following chemoradiation. Using images collected at pre-treatment and at the 1-month post-chemoradiation visit, we calibrate for patient-specific model parameters. The calibrated parameters are then used to provide individualized predictions of tumor growth and treatment response. In a preliminary study with four patients, we observed less than 11. 1% error in tumor volume predictions and 9. 5% error at the voxel- level. These two studies demonstrate the utility of quantitative imaging data to initialize and calibrate mathematical models of tumor growth and response that can accurately predict both changes in tumor volume and intratumoral heterogeneity. 									  											  									  									  									  								  								  								  	Sarah Brüningk								  	 								  		Institute of Molecular Systems Biology, ETH Zurich, Zurich Switzerland								  									  									  										  			&lt;/br&gt; Intermittent radiotherapy as alternative treatment for recurrent high grade gliomas: A modelling study based on longitudinal tumour measurements 								  										  									  									  										  										  											  			Recurrent high grade glioma patients are faced with a poor prognosis for which there cur- rently exists no curative treatment option. In contrast to prescribing high dose hypofrac- tionated stereotactic radiosurgery (HFRS, 5x ≥ 6 Gy in daily intervals) with curative intent, we suggest a personalized, palliative treatment strategy aiming for tumour volume management by delivering intermittent high dose treatment every six weeks (iRT, ≥ 6 Gy per fraction). We performed a simulation analysis to compare HFRS, iRT and iRT plus boost (3x ≥ 6 Gy on consecutive days, delivered at time of progression) based on a simple mathematical model of tumour growth, radiation response and patient specific resistance to additional treatments (PD-L1 inhibitor and vascular disruptive agents). Our model uses only two patient specific parameters describing the surviving fraction fol- lowing each HFRS treatment fraction, and the rate of resistance evolution. Tumour growth rate and the efficacy of non-HFSR treatments were estimated for the patient population as a whole. Model parameters were fit from clinical tumour growth response curves of 16 patients of the Phase 1/2 clinical trial NCT02313272 that combined HFSR with beva- cizumab (10 mg/kg, every 2 weeks) and pembrolizumab (100 or 200 mg, every 3 weeks). Tumour volume was assessed by T1-weighted contrast enhanced magnetic resonance imag- ing at four to ten (median six) time points per patient. The obtained parameters were used to estimate the growth response of alternative iRT and iRT+boost treatments for up to 5-10 treatment fractions. Treatment efficacy was scored based on time to regrowth to the last recorded tumour volume per patient. The median coefficient of determination of the model fits was 0. 93(0. 67,0. 99). The model predictions indicated that iRT may delay time to progression only for a subset of eleven patients, whereas iRT+boost treatment was equal or superior to HFSR in 15 out of 16 cases. For up to ten intermittently delivered fractions, iRT+boost was predicted to be marginally significantly better (p = 0. 048) than HFRS. This simulation did not include other aspects of iRT, such as synergistic action with im- munotherapy through repeated antigen sampling, or the flexibility to treat both distal and primary lesions. Despite choosing this worst case estimate, our results suggest that iRT+boost may be a promising treatment alternative for recurrent high grade glioma patients. 									  											  									  									  									  								  								  								  	Andrea Hawkins-Daarud								  	 								  		Precision Neurotherapeutics Innovation Program, Mayo Clinic, USA								  									  									  										  			&lt;/br&gt; Assessing clinical utility of a model based patient-specific response metric for glioblastoma incorporating uncertainty quantification from image acquisition and segmentation 								  										  									  									  										  										  											  			Glioblastomas are lethal primary brain tumors known for their heterogeneity and invasiveness. A growing literature has been developed demonstrating the clinical relevance of a biomathematical model, the Proliferation- Invasion (PI) model, of glioblastoma growth. Of interest here is the development of a treatment response metric, Days Gained (DG). This metric is based on individual tumor kinetics of cellular diffusion and proliferation estimated through segmented volumes of hyperintense regions on T1-weighted gadolinium enhanced (T1Gd) and T2-weighted magnetic resonance images (MRIs). This metric was shown to be prognostic of time to progression and to be more prognostic of outcome than standard response metrics. While promising, the original paper did not account for uncertainty in the calculation of the DG metric leaving the robustness and the ultimate utility of this response metric in question. Using the Bayesian framework, we consider the impact of two sources of uncertainty: 1) image acquisition and 2) interobserver error in image segmentation. We first utilize synthetic data to characterize what non-error variants are influencing the final uncertainty in the DG metric. We then consider the original patient cohort along with additional cohorts from the recurrent setting to investigate clinical patterns of uncertainty and to determine how robust this metric is for predicting time to progression and overall survival in multiple scenarios. Our results indicate that the key clinical variants are the time between pre-treatment images and the underlying tumor growth kinetics, matching our observations in the clinical cohort. In the original cohort, we demonstrated that for this cohort there was a continuous range of cutoffs between 94 and 105 for which the prediction of the time to progression and was over 80% reliable [3]. While further validation must be done, this work represents a key step in ascertaining the clinical utility of this metric. 									  											  									  									  									  								  								  								  	Victor M. Perez-Garcia								  	 								  		Universidad de Castilla – La Mancha, Spain								  									  									  										  			&lt;/br&gt; If you have mathematical models then you have imaging biomarkers: Applications to gliomas, lung cancer, breast cancer and head &amp; neck cancer 								  										  									  									  										  										  											  			The ultimate goal of mathematical models fed with imaging and molecular/clinical data is to provide prediction machines allowing to get precise estimates for patient survival, time to relapse, and to set up personalized therapeutic schedules. However, this is a very complex task due to our limited knowledge of the many biological processes involved and the scarce amount of information available today for patients. A more accessible goal is to use mathematical models to obtain qualitative information allowing to classify patients in classes according to features found to be relevant in the modelling. When those features can be obtained from medical images, we speak of imaging biomarkers. In this talk I will present different types of mathematical models and discuss how each modelling approach leads to the finding of imaging biomarkers that turn out to be true when tested on real patients imaging data. Specifically I will present: MRI-based biomarkers in glioblastoma based on either partial differential equations or discrete mesoscopic simulators [1-3], prognostic PET-based biomarkers based on the peak activity validated for lung and breast cancer [4] and PET-based prognosis biomarkers based on scaling laws for lung cancer, breast cancer, gliomas, and head &amp; neck cancer [5]. These imaging biomarkers are all easy to implement in clinical practice since they do not require model simulations but just measurement of meaningful quantities on the images. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	OTHE: Flow and Transport in Bacterial Biofilms, Part II (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Mohit Dalwadi								  	 								  		Oxford								  									  									  										  			&lt;/br&gt; Using homogenization to determine the effective nutrient uptake in a biofilm from microscale bacterial properties 								  										  									  									  										  										  											  			In biofilm models that include nutrient delivery to bacteria, it can be computationally expensive to include many small bacterial regions acting as volumetric nutrient sinks. To combat this problem, such models often impose an effective uptake instead. However, it is not immediately clear how to relate properties on the bacterial scale with this effective result. For example, one may intuitively expect the effective uptake to scale with bacterial volume for weak uptake, and with bacterial surface area for strong uptake. I will present a general model for bacterial nutrient uptake, and upscale the system using homogenization theory to determine how the effective uptake depends on the microscale bacterial properties [1, 2]. This will show us when the intuitive volume and surface area scalings are each valid, as well as the correct form of the effective uptake when neither of these scalings is appropriate. 									  											  									  									  									  								  								  								  	Alexandre Persat								  	 								  		EFPL								  									  									  										  			&lt;/br&gt; Cellular advective-diffusion drives the emergence of bacterial surface colonization patterns 								  										  									  									  										  										  											  			In the wild, bacteria predominantly live as multicellular aggregates called biofilms. In contrast to lab- oratory settings, natural biofilms commonly comprise multiple strains or species that interact with each other. The nature of such interactions depends on the relative proximity of different strains and thus on the spatial organization of the biofilm. However, little is known about how environmental factors affect this organization. Here, we demonstrate that biofilms of the freshwater bacterium /emphCaulobacter crescentus form distinct patterns of surface colonization depending on local hydrodynamic conditions. By imaging C. crescentus biofilms grown in microfluidic chambers under controlled flow, we observed that surface colonization rate decreases with increasing flow velocity. We also probed the effect of flow intensity on spatial lineage structure using two C. crescentus clones. Surprisingly, segregation increased with flow velocity, in contrary to the assumption that flow induces lineage mixing. To understand how these colonization patterns arise, we developed a theoretical model based on the balance between ad- vection and diffusion: planktonic cells released from the biofilm effectively diffuse as they swim with random trajectories, while they are transported unidirectionally by flow. As flow intensity increases, the residence time of planktonic cells in the channel decreases, reducing their spreading across the channel and decreasing encounters with the surface. To sum up, our work demonstrates that hydrodynamic forces impact biofilm architecture and lineage distribution in multi-strain biofilms. This indicates that social interactions within biofilms are not only dictated by biological factors, but also by mechanical conditions imposed onto the community. 									  											  									  									  									  								  								  								  	Sara Jabbari								  	 								  		Birmingham								  									  									  										  			&lt;/br&gt; Targeting bacterial adhesion as a novel therapeutic for Pseudomonas aeruginosa infections 								  										  									  									  										  										  											  			The rise in antibiotic resistance, combined with the shortage in discovery of new classes of these drugs, has led to the pursuit of creative ways to tackle bacterial infections that do not necessarily kill bacteria in conventional ways. Many of these approaches, however, are only partially successful when tested in infection models. We focus here on an approach that targets the ability of bacteria to bind to host cells – the first stage in infection and of biofilm formation. Through differential equation modelling and com- parison against experimental data, we investigate why the treatment is not fully effective. Furthermore, we use the model to predict how to improve the treatment in a variety of scenarios, including through changes in drug design and/or combination with alternative therapies. We illustrate when the therapy could replace or reduce antibiotic use, ultimately suggesting experimental pathways that should aid in accelerating the development of this novel therapy. 									  											  									  									  									  								  								  								  	Philip Pearce								  	 								  		Harvard								  									  									  										  			&lt;/br&gt; The biofilm life cycle in high flow environments 								  										  									  									  										  										  											  			Bacterial biofilms represent a major form of microbial life on Earth. In their natural environments, ranging from human organs to industrial pipelines, biofilms have evolved to grow robustly under signifi- cant fluid shear. Despite intense practical and theoretical interest, it is unclear how strong fluid flow alters aspects of the biofilm life cycle including their formation, growth, and dispersal. In this talk, I will discuss how external flow affects biofilms across multiple scales at each of these stages of their life cycle, through transport of quorum sensing molecules, realignment of growing cells, and biofilm deformation and erosion. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  																																		 	POPD: Subgroup Contributed Talks (11:15am-12:45pm)													 						  			  				  							  							  										  									  	Annalisa Iuorio								  	 								  		University of Vienna								  									  									  										  			&lt;/br&gt; Modelling competitive interactions and plant-soil feedback in vegetation dynamics 								  										  									  									  										  										  											  			Plant-soil feedback has been proved to play an important role in the formation of vegetation patterns for a single species.  In real-life, however, plants rarely grow in monoculture; hence multi-species scenarios are more realistic. In these cases, additional effects between different species - such as competition and interaction - must be considered. Moreover, plant-soil feedback is recognised as a causal mechanism for plant-species coexistence. Using a mathematical model consisting of four PDEs, we investigate mechanisms of inter- and intra-specific plant-soil feedback on the coexistence of two competing plant species. In particular, the model takes into account both negative and positive feedback influencing the growth of the same and the other plant species. Both the coexistence of the plant species and the dominance of a particular plant species are examined with respect to all model parameters. Analytical and numerical results reveal the emergence of spatio-temporal patterns. 									  											  									  									  									  								  								  								  	Lukas Eigentler								  	 								  		University of Dundee								  									  									  										  			&lt;/br&gt; Spatial self-organisation enables species coexistence in a model for dryland vegetation patterns 								  										  									  									  										  										  											  			Vegetation patterns are a ubiquitous feature of drylands across the globe. Despite the competition for water, species coexistence of herbaceous and woody species is commonly observed. Thus, tree-grass coexistence in drylands provides an apparent contradiction to the principle of competitive exclusion.  In this talk, I propose that a pattern-inducing spatial self-organisation principle, caused by a positive feedback between local vegetation growth and water redistribution towards dense biomass patches, can also act as a coexistence mechanism for plant species in water-limited ecosystems. To this end, I present a bifurcation analysis of an ecohydrological PDE model for two plant species interacting with a sole limiting resource, based on the Klausmeier reaction-advection-diffusion system for vegetation patterns. Patterned solutions occur as periodic travelling waves and thus theory on limit cycles in dynamical systems can be utilised in the analysis. Firstly, a stability analysis of the system's single-species patterns, performed through a calculation of their essential spectra, provides an insight into the onset of coexistence states. I show that coexistence solution branches bifurcate off single-species solution branches as the single-species states lose their stability to the introduction of a second species. Secondly, I present a comprehensive existence and stability analysis to establish key conditions, including a balance between the species' local competitive abilities and their colonisation abilities, for species coexistence in the model. Finally, I show that the inclusion of intraspecific competition dynamics has a significant impact on the coexistence mechanism that significantly differs from results on classical, nonspatial competition models. (Joint work with Jonathan A. Sherratt)									  											  									  									  									  								  								  								  	Gabriel Maciel								  	 								  		ICTP - SAIFR &amp; IFT - UNESP								  									  									  										  			&lt;/br&gt; Spatial self-organization promotes coexistence between two species in nonlocal competition models 								  										  									  									  										  										  											  			Nonlocal interactions are a remarkable feature of several ecological systems ranging from microorganisms and coral reefs to plants and animals. The potential of spatially extended interactions to generate patterns of space distribution in single populations has been extensively explored in the past few decades. In systems of two competing species, nonlocal interactions and the self-organization of populations can have critical dynamical effects but our understanding in these situations is still very limited. In this talk, I will present a kernel-based model for the dynamics of two species competition with nonlocal interactions. Inspired by different biological examples I will show that we can have two distinct scenarios that differ in how intra and interspecific interaction ranges are determined based on each species competition characteristics. In both scenarios we will see that pattern formation offers a coexistence mechanism where the inferior competitor takes advantage of low density locations in the superior competitor spatial distribution. 									  											  									  									  									  								  								  								  	Nikunj Goel								  	 								  		University of Texas at Austin								  									  									  										  			&lt;/br&gt; Dispersal increases the resilience of tropical savanna and forest distribution 								  										  									  									  										  										  											  			Global change may induce changes in savanna and forest distributions, but the dynamics of these changes remain unclear. Classical biome theory suggests that climate is predictive of biome distributions, such that shifts will be continuous and reversible. This view, however, cannot explain the overlap in the climatic ranges of tropical biomes, which some argue may result from fire-vegetation feedbacks, maintaining savanna and forest as bistable states. Under this view, biome shifts are argued to be discontinuous and irreversible. Mean-field bistable models, however, are also limited, as they cannot reproduce the spatial aggregation of biomes. Here we suggest that both models ignore spatial processes, such as dispersal, which may be important when savanna and forest abut. Using a combination of spatial mathematical models and remote sensing data, we examine the contributions of dispersal to determining biome distributions. We find that including dispersal in biome dynamics resolves both the limitations mentioned above of biome models. First, local dispersive spatial interactions, with an underlying precipitation gradient, can reproduce the spatial aggregation of biomes with a stable savanna-forest boundary. Second, the boundary is determined not only by the amount of precipitation but also by the geometrical shape of the precipitation contours. These geometrical effects arise from continental-scale source-sink dynamics, which reproduce the mismatch between biome and climate. Dynamically, the spatial model predicts that dispersal may increase the resilience of tropical biome in response to global change: the boundary continuously tracks climate, recovering following disturbances, unless the remnant biome patches are too small. 									  											  									  									  								  								  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  			             		    		  											  		    		      		        SMB Business Meeting		             		    		  											  		    		      		        Sub-group contributed talks (1:30-2:30pm)		        			  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  																																		 	CDEV: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Adriana Dawes								  	 								  		Ohio State University								  									  									  										  			&lt;/br&gt; Dynein dynamics in the first cell cycle of the C. elegans embryo 								  										  									  									  										  										  											  			Asymmetric cell division, where daughter cells inherit unequal amounts of specific factors, is critical for development and cell fate specification. In polarized cells, where specific factors are segregated to opposite ends of the cell, asymmetric cell division occurs as a result of positioning the centrosomes along the polarity axis. In many systems, this positioning involves both translocation as well as rotation of the nucleus and its associated centrosomes. Using an individual-based stochastic model of centrosome-associated microtubule dynamics and experiments in early embryos of the nematode worm C. elegans, we explore the role of the motor protein dynein under both wild type and knockdown conditions. We show that dynein activity but not localization is implicated in specific centrosome movement defects in the first cell cycle. 									  											  									  									  									  								  								  								  	Tracy Stepien								  	 								  		University of Florida								  									  									  										  			&lt;/br&gt; Spreading Mechanics and Differentiation of Astrocytes During Retinal Development 								  										  									  									  										  										  											  			In embryonic development, formation of the retinal vasculature is critically dependent on prior establishment of a mesh of astrocytes. Astrocytes emerge from the optic nerve head and then migrate over the retinal surface in a radially symmetric manner and mature through differentiation. We develop a PDE model describing the migration and differentiation of astrocytes, and numerical simulations are compared to experimental data to assist in elucidating the mechanisms responsible for the distribution of astrocytes. 									  											  									  									  									  								  								  								  	Renske Vroomans								  	 								  		Origins Center								  									  									  										  			&lt;/br&gt; Conservative evolution of epithelial morphogenesis 								  										  									  									  										  										  											  			Morphogenesis is a complex process involving multiple levels of organisation. Cell differentiation within tissues is governed by extensive gene expression regulation within cells and communication via chemical signals between cells. Based on their gene expression, cells may divide and change their physical properties, leading to cell- and tissue-level physical forces which can in turn feed back on gene expression between cells. This developmental process may change over time due to Darwinian evolution, but development itself influences the course of evolution by determining the effect that mutations have on the final phenotype. Currently, it is not well-understood how the complex interactions involved in morphogenesis impact the kind of evolutionary changes that can occur.  Here, we investigate the evolution of developmental mechanisms that govern morphogenesis with Embryomaker, an in silico model of 3d epithelium development. We look at short-term evolutionary changes that occur under conservative selection, meaning that the final shape of the tissue is conserved while all else -- gene expression pattern, growth pattern, developmental trajectory -- is allowed to change. We find that substantial change can occur at all levels of organisation, from the structure of the gene regulatory network to gene expression pattern, the duration of the developmental process and the course of morphogenesis. In some cases we observe an increase in the reliability of the developmental process despite not explicitly selecting for it, and show how it is caused by a non-trivial combination of mutations which on their own do not improve -- or even diminish -- fitness. Finally, we show how the coupling of gene expression regulation to morphogenesis influences which mutations are accepted, and thereby the ancestry of genes in the genome. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	MEPI: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Matthew Jones								  	 								  		Dartmouth College								  									  									  										  			&lt;/br&gt; Spatial Games of Fake News 								  										  									  									  										  										  											  			When it becomes so quick and easy to freely 'share' another's understanding (via retweets/reposts), rather than researching and formulating one's own, social media platforms seem to facilitate the spread of fake news. A recent study used an online crowdsourcing fact-checking approach as one possible intervention to reduce misinformation. However, it remains largely unclear under what conditions crowdsourcing fact-checking efforts can actually deter the spread of misinformation. To address this issue, we model such distributed fact-checking efforts as 'peer policing' that will reduce the perceived payoff to share or disseminate false information (fake news) and also reward the spread of trustworthy information (real news). We use the diffusion approximation method and agent-based simulations to quantify the degree of penalty vs reward needed to make sharing fake news unfavorable in social networks. In the limit of weak selection, we obtain closed-form analytical conditions, which can be expressed as an inequality of these payoff values, with coefficients summarizing the effect of fact-checkers' presence. In reality, fact-checking is subject to human errors. Some fake news occasionally goes unnoticed and endorsed, and some real news is temporally labelled to be fake by fact-checkers. We also quantify the precision threshold required for fact-checkers to ensure fair and transparent policing of wrongdoers while in favor of real news spreaders. Our work has practical guide for developing model-based mitigation strategies for controlling the spread of misinformation that interferes with the political discourse. 									  											  									  									  									  								  								  								  	Emma Southall								  	 								  		Emma Southall								  									  									  										  			&lt;/br&gt; Identifying indicators of critical transitions in epidemiological data 								  										  									  									  										  										  											  			A challenging problem in infectious disease modelling is assessing when a disease has been eliminated. Control campaigns have substantial economic consequences; as such there are high demands to reduce costs and reallocate resources. However, if campaigns are stopped prematurely it can result in disease resurgence and subsequently put control efforts back by decades. Early-warning signals offer a computationally inexpensive technique to monitor the progress towards elimination, using statistical indicators calculated on time series data. Early-warning signals are widely used in many fields to anticipate a critical threshold prior to reaching it. A system undergoes the phenomenon known as critical slowing down as it crosses through a threshold. Theory predicts that fluctuations away from the mean will recover more slowly as the system approaches a critical transition (Scheffer et al. , 2009). This is key in infectious disease modelling to assess when the basic reproduction number is reduced below the threshold of one. Recent theoretical advances have shown indicators of critical transitions in epidemiology such as measuring the variance in synthetic disease data. Our work highlights several challenges when applying this theory in practice. One potential problem is known as 'detrending' the data, which can be difficult to achieve in a single time series (Dessavre &amp; Southall et al. , 2019). Accurately detrending the signal removes the mean to obtain the fluctuations, whilst preserving any statistical properties.  We present a novel approach using a metapopulation framework to successfully detrend data using the mean of different geographical subpopulations. A second limitation is that often only incidence-level data is available publicly. However, current theoretical analyses of statistical indicators concentrate on prevalence data, instead of new cases. We demonstrate that indicators calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data (Southall et al. , 2020). Inconsistencies in time series traits between different diseases systems and a variety of disease data types could lead to misleading results when applied to collected data.  In this talk we present methods for dealing with the typical data collected and our results show promising methods for calculating early-warning signals of elimination on real-world noisy data. 									  											  									  									  									  								  								  								  	Scott Greenhalgh								  	 								  		Siena College								  									  									  										  			&lt;/br&gt; A generalized differential equation compartmental model of infectious disease transmission 								  										  									  									  										  										  											  			For decades, mathematical models of disease transmission have provided researchers and public health officials with critical insights into the progression, control, and prevention of disease spread. Of these models, one of most fundamental is the SIR differential equation model. However, this ubiquitous model has one significant and rarely acknowledged shortcoming: it is unable to account for a disease’s true infectious period distribution.  As the misspecification of such a biological characteristic is known to significantly affect model behavior, there is a need to develop new modeling approaches that capture such information. In this talk, we illustrate an innovative take on compartmental models, derived from their general formulation as systems of nonlinear Volterra integral equations, to capture a broader range of infectious period distributions, yet maintain the desirable formulation as systems of differential equations. Our results include a compartmental model that captures any Erlang distributed duration of infection with only 3 differential equations, instead of the typical inflated model sizes required by differential equation compartmental models, and a compartmental model that captures any mean, standard deviation, skewness, and kurtosis of an infectious period distribution with 4 differential equations. The significance of our work is that it opens up a new class of easy-to-use compartmental models to predict disease outbreaks that does not require a complete overhaul of existing theory, and thus provides a starting point for multiple research avenues of investigation under the contexts of mathematics, public health, and evolutionary biology. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				  				  				  				  				  				 		  				  				  				  				  				  				  				  				 		  				  				  				  				  				 		  				  				 		  				 		  				 		  																																		 	MFBM: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Archana		Hari								  	 								  		UMBC, United States,		archh1@umbc. edu								  									  									  										  			&lt;/br&gt; A novel methodology and web application for computing, visualizing and analyzing genome-scale metabolic flux networks 								  										  									  									  										  										  											  			Genome-scale metabolic models not only represent the biochemical circuits within cells but can also be used to simulate and analyze cellular phenotypes. However, there is a lack of methodologies and user-friendly tools to visualize the complex reaction graphs within these genome-scale metabolic models and simulate their flux behaviors. For this, we have developed a novel methodology and implemented it in a freely-available web application called Fluxer ( https://fluxer. umbc. edu ) that streamlines the simulation and visualization of genome-scale metabolic flux networks with an easy-to-use interface. The application can take as input any metabolic model encoded with the Systems Biology Markup Language format, automatically perform Flux Balance Analysis using linear programming and apply different methods to compute flux graphs. The flux networks can be visualized as spanning trees and complete graphs with different layouts. The interactive graphs can be used to study major pathways contributing to any metabolic reaction or biosynthesis of any metabolite as well as to simulate reaction knockouts. In addition, Fluxer can compute the k-shortest paths between two reaction or metabolites within the model. Nodes can display detailed metabolic and reaction information, including molecular weights, reaction fluxes, and molecular structures. Over 80 whole-genome metabolic reconstructions are readily available for visualization and analysis. The proposed methodology enables efficient analysis and visualization of genome-scale metabolic models towards the discovery of key metabolic pathways. 									  											  									  									  									  								  								  								  	Peter		Stechlinski								  	 								  		University of Maine,	United States,	peter. stechlinski@maine. edu								  									  									  										  			&lt;/br&gt; Sensitivity analysis of nonsmooth biological models 								  										  									  									  										  										  											  			Nonsmooth dynamical systems are an appropriate modeling framework for a variety of problems in mathematical biology, ranging from glucose-insulin kinetics to rioting activity. The presence of nonsmoothness in these models arises from switching phenomena, such as a biochemical threshold signaling sudden insulin release, or a bandwagon effect in rioting behavior corresponding to an outburst of social activity. Nonsmooth modeling frameworks now possess a sensitivity theory that is relevant for nonsmooth ODEs, nonsmooth differential-algebraic equations, optimization-constrained ODEs, and complementarity systems, among others. In this talk, the nonsmooth sensitivity theory is presented, including the generalized derivatives theory upon which it is built. The theory yields an auxiliary, nonsmooth system whose unique solution characterizes (local) sensitivity information. The theory is highlighted using examples from mathematical biology, with numerical implementations that give nonsmooth sensitivity indices. The results can be used to help inform policy decisions (e. g. , in rioting) or for design purposes (e. g. , in the intravenous glucose tolerance test for type 2 diabetes) by uncovering the mechanisms driving the dynamics. 									  											  									  									  									  								  								  								  	Ivo		Siekmann									  	 								  		Liverpool John Moores University,	United Kingdom,	i. siekmann@ljmu. ac. uk								  									  									  										  			&lt;/br&gt; Data-driven modelling of ion channels incorporating uncertainty using hierarchical Markov models 								  										  									  									  										  										  											  			Ion channels are proteins that regulate the flow of ions across the cell membrane. Patch clamp recordings enable us to watch a single ion channel in action by detecting the electrical current flowing through the channel over time. At first glance we only see that the channel opens and closes stochastically but often a closer look reveals that it also alternates between two or more levels of activity – highly active modes where the channel opens and closes frequently, nearly inactive modes in which the channel is mostly closed, as well as intermediate levels of activity. The striking differences in the dynamics the channel exhibits in the different modes suggest that each mode is associated with a distinct three-dimensional configuration (conformation) of the channel protein. Applying statistical change point analysis to a large single channel data set collected from the inositol-triphosphate receptor (IP3R) highlights the importance of this observation: We find that the dynamics of the IP3R is entirely regulated by switching between two modes. In order to build a mathematical model that takes this into account we develop a novel model, the hierarchical Markov model, which enables us to separate modelling the slower dynamics of mode switching from the faster dynamics of opening and closing characteristic of each mode. We will illustrate this approach by fitting a hierarchical Markov model to type 1 and type 2 IP3R data for a wide range of concentrations of IP3R, Ca2+ and ATP. In contrast to most other ion channel models currently available we account for uncertainty by calculating probability distributions for the parameters of our model following a Bayesian Markov chain Monte Carlo (MCMC) approach. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  																																		 	NEUR: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Lucas Stolerman								  	 								  		UCSD								  									  									  										  			&lt;/br&gt; Stability Analysis of a Bulk–Surface Reaction Model for Membrane Protein Clustering 								  										  									  									  										  										  											  			Protein aggregation on the plasma membrane (PM) is of critical importance to many cellular processes such as cell adhesion, endocytosis, fibrillar conformation, and vesicle transport. Lateral diffusion of protein aggregates or clusters on the surface of the PM plays an important role in governing their heterogeneous surface distribution. However, the stability behavior of the surface distribution of protein aggregates remains poorly understood. Therefore, understanding the spatial patterns that can emerge on the PM solely through protein–protein interaction, lateral diffusion, and feedback is an important step toward a complete description of the mechanisms behind protein clustering on the cell surface. In this work, we investigate the pattern formation of a reaction–diffusion model that describes the dynamics of a system of ligand–receptor complexes. The purely diffusive ligand in the cytosol can bind receptors in the PM and the resultant ligand–receptor complexes not only diffuse laterally but can also form clusters resulting in different oligomers. Finally, the largest oligomers recruit ligands from the cytosol using positive feedback. From a methodological viewpoint, we provide theoretical estimates for diffusion-driven instabilities of the protein aggregates based on the Turing mechanism. Our main result is a threshold phenomenon, in which a sufficiently high recruitment of ligands promotes the input of new monomeric components and consequently drives the formation of a single-patch spatially heterogeneous steady state. 									  											  									  									  									  								  								  								  	Hammed Fatoyinbo								  	 								  		Massey U Palmerston North								  									  									  										  			&lt;/br&gt; Spatiotemporal dynamics in spontaneous excitable cells 								  										  									  									  										  										  											  			Pacemaker dynamics is the spontaneous excitation-contraction coupling in muscle cells. It may arise as a result of interaction between ion fluxes through the voltage-gated ion channels.  In this work, we consider a model of electrically coupled pacemaker smooth muscle cells to investigate the formation of spatiotemporal patterns. We analyse the behaviour of an isolated smooth muscle cell using numerical bifurcation analysis. By modulating model parameters, the result reveals transitions between Type I and II excitabilities in the parameter space. Numerical simulations of our model show that the pattern can bifurcate from been stable to spatiotemporal chaos. 									  											  									  									  									  								  								  								  	Chitaranjan Mahapatra								  	 								  		UCSF								  									  									  										  			&lt;/br&gt; Evaluation of a mathematical model for estradiol effect on membrane excitability of detrusor smooth muscle cell 								  										  									  									  										  										  											  			The urinary bladder is composed of detrusor smooth muscle (DSM) cell to perform contraction, triggered by the intracellular calcium concentration after the generation of the action potential (AP). The DSM cells display enhanced spontaneous APs during the overactive bladder state. Estradiol, which is a natural sex hormone, has been suggested to be beneficial in the treatment of overactive bladder.  This study aims in investigating the quantitative analysis of estradiol on membrane excitability of DSM cells. To simulate the estradiol effect, conductances of calcium- and voltage-dependent potassium channels (BK channels) were increased by 40% of its control value in a published DSM model cell. We found that the resting membrane potential (RMP) was more negative (─ 53 mV) than the control (─ 50 mV) value. The peak amplitude of the AP due to estradiol treatment was also significantly decreased. Similar to in the control condition, we have implemented the voltage clamp protocol to investigate the whole cell outward current. Under the effect of the estradiol, the amplitude of outward current was greatly increased due to BK channel.  These findings are consistent with the experiment in guinea pig and rat DSM cells. The future investigation would provide some insight towards the modulating role of voltage-gated Ca2+ current in DSM cells due to estradiol treatment. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				 		  				 		  				  				 		  				  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	ONCO: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Renee Brady-Nicholls								  	 								  		Department of Integrated Mathematical Oncology, H. Lee Moffitt Cancer Center Tampa, USA								  									  									  										  			&lt;/br&gt; Forecasting Individual Responses to Intermittent Androgen Deprivation Therapy Using PSA Dynamics 								  										  									  									  										  										  											  			Prostate cancer (PCa) is the second most common cancer in men and the fifth leading cause of death worldwide. Intermittent androgen deprivation therapy (IADT), whereby treatment is cycled on and off, is an attractive therapy option that can reduce cumulative dose and limit toxicities, when compared to continuous therapy. We simulate prostate-specific antigen (PSA) dynamics, with enrichment of PCa stem-like cell (PCaSC) during treatment as a plausible mechanism of resistance evolution. Simulated PCaSC proliferation patterns correlate with longitudinal serum PSA measurements in 70 biochemically recurrent PCa patients. Learning dynamics from each treatment cycle in a leave-one-out study, model simulations predict patient-specific evolution of resistance with an overall accuracy of 89% (sensitivity=73%, specificity=91%). Previous studies have shown a benefit of concurrent therapies with ADT in both low- and high-volume metastatic hormone-sensitive PCa. Model simulations based on response dynamics from the first IADT cycle identify patients who may benefit from concurrent docetaxel, demonstrating the feasibility and potential value of adaptive clinical trials guided by patient-specific mathematical models of intratumoral evolutionary dynamics. 									  											  									  									  									  								  								  								  	Balázs G. Madas								  	 								  		Centre for Energy Research, Hungary								  									  									  										  			&lt;/br&gt; Low dose hyper-radiosensitivity as a consequence of bystander signalling aiming to reduce the mutation load of the cell population 								  										  									  									  										  										  											  			Radiation therapy is frequently applied in cancer treatment. The fraction of surviving cells is a crucial quantity predicting both tumour control and normal tissue toxicity. The gold standard in vitro measurement for cell survival is the colony forming assay (CFA), which is based on the ability of a single cell to grow into a colony. The relationship between surviving fraction (SF) and absorbed dose (D) can be described by a linear-quadratic function: SF=exp(-aD-bD^2), where a and b depend on radiation type other cell properties. However, many cell lines show hyper-radiosensitivity (HRS) resulting in a local minimum in the surviving curve below 1 Gy. It is an interesting phenomenon and it may also have clinical implications related to the potential benefits of hyperfractionation. It is generally thought that HRS is a mechanism that protects against carcinogenesis following low dose ionizing radiation. Recently, it has been shown that minimizing the mutation load at the cell population level may result in local minimum in survival-dose relationships if both spontaneous and radiation induced mutations are considered. The model proposed, however, does not account for how cells receive the information required to reduce the mutation load. The aim of the present study was to test whether bystander signalling through the medium provides enough information for the cells to reduce the mutation load of the population with their individual survival or death decision. For this purpose, a mathematical model of bystander signalling was elaborated. Cells are located randomly in a disc shaped dish. Extracellular signals are produced by cells with mutagenic damages. The signal production is proportional to the number of mutagenic damages, while the signal strength decreases by the distance from the signal producing cell. A cell goes into apoptosis if the difference between the number of its mutagenic damages and the number of spontanenous mutations per cell division is high compared to the signal strength at its location. The threshold signal strength for this decision equals to the mean signal strength if all cells have one mutagenic damage. The number of mutagenic damages follows Poisson distribution, and its mean is proportional to absorbed dose. Simulations were performed to reproduce the deviation from the linear quadratic survival curve in case of 45 experimental datasets. During this fitting process, the spontaneous mutation rate and the mutagenic damage induction rate were varied. The model describes most experimental data very well, while the spontanenous mutation rate is close to the reasonable number of 1 mutation per cell division, and the mutagenic damage induction rate is in the order of magnitude of 1 per Gy. These results confirm that low dose hyper-radiosensitivity can be a consequence of bystander signalling aiming to reduce the mutation load of the cell population. 									  											  									  									  									  								  								  								  	Guillermo	Lorenzo								  	 								  		The University of Texas at Austin, USA								  									  									  										  			&lt;/br&gt; Image-based mechanistic modeling of prostate cancer for personalized forecasting of tumor growth 								  										  									  									  										  										  											  			The current clinical management of prostate cancer (PCa) enables its detection at early organ-confined stages by combining regular screening and patient classification in risk groups. Although these newly diagnosed tumors do not usually pose a threat to the patient, most PCa cases are prescribed a radical treatment immediately after diagnosis (e. g. , surgery or radiotherapy). However, the limited individualization of the clinical management beyond risk-group definition has led to significant overtreatment and undertreatment rates, which might adversely impact the patients' lives and life expectancy, respectively. Thus, PCa is a paradigmatic disease in which an individualized predictive technology could make a crucial difference in clinical practice, thereby separating less aggressive tumors that could be safely monitored from lethal tumors that require immediate treatment. To address this critical need, we propose to use routine clinical and imaging data to construct and parametrize personalized mathematical models of PCa growth including the key mechanisms involved in this pathology. We can then run computer simulations with these models to forecast the growth of a patient's tumor, which may assist physicians in clinical-decision making. Our models have been able to qualitatively reproduce tumor growth over the local anatomy of a patient's prostate extracted from imaging data. We have also been able to capture the dynamics of the Prostate Specific Antigen (PSA), which is a ubiquitous biomarker in PCa clinical management. We have also studied the inhibitive effect of growth-induced mechanical stress on PCa and how the compression exerted by concomitant benign prostatic hyperplasia dramatically impedes tumor growth. Our ongoing efforts aim at leveraging longitudinal clinically-available quantitative magnetic resonance data to initialize and parameterize our PCa growth models. We believe that our imaging-based models could constitute a promising computational technology to assist physicians to provide a personalized clinical management of PCa. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  				 		  																																		 	OTHE: Other Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Rosemary Dyson								  	 								  		Birmingham								  									  									  										  			&lt;/br&gt; Vesicle transport and cytoplasmic streaming in the pollen tube tip 								  										  									  									  										  										  											  			The rapid elongation of the pollen tube in seed plants cannot occur without the transport of sufficient cell wall and membrane material to the growing apex. The movement of this material, delivered via exocytic secretory vesicles, can be categorised under two regimes: 'long distance' movement in the shank (via active transport along actin ﬁlaments), and 'short distance' movement in the apex (where vesicles diffuse and advect freely). Many current models of vesicle transport focus on diffusion in the apical region alone, neglecting advective eﬀects as well as the resulting distribution proﬁle in the pollen tube shank. Using the method of regularised Stokeslets with an adjustment made for axisymmetry, we produce a complete advective velocity proﬁle for cytosolic ﬂow in the tube based on the drag induced by the active transport of vesicles along actin. We use this to calculate exocytic and endocytic vesicle motion in the tube, incorporating vesicle uptake and deposition at the wall, and generating insight into pollen tube growth dynamics. 									  											  									  									  									  								  								  								  	Omer Karin								  	 								  		Weizmann Inst. 								  									  									  										  			&lt;/br&gt; A new model for the HPA axis explains dysregulation of stress hormones on the timescale of weeks 								  										  									  									  										  										  											  			Stress activates a complex network of hormones known as the Hypothalamic-Pituitary-Adrenal (HPA) axis. The HPA axis is dysregulated in chronic stress and psychiatric disorders, but the origin of this dysregulation is unclear and cannot be explained by current HPA models. To address this, we developed a new mathematical model for the HPA axis that incorporates changes in the total functional mass of the HPA hormone-secreting glands. The mass changes are caused by the HPA hormones which act as growth factors for the glands in the axis. We find that the HPA axis shows the property of dynamical compensation, where gland masses adjust over weeks to buffer variation in physiological parameters. These mass changes explain the experimental findings on dysregulation of cortisol and ACTH dynamics in alcoholism, anorexia and postpartum. Dysregulation occurs for a wide range of parameters, and is exacerbated by impaired glucocorticoid receptor (GR) feedback, providing an explanation for the implication of GR in mood disorders. These findings suggest that gland-mass dynamics may play an important role in the pathophysiology of stress-related disorders. 									  											  									  									  									  								  								  								  	Preeti Dubey								  	 								  		Loyola								  									  									  										  			&lt;/br&gt; Modeling early hepatitis D virus kinetics in transgenic-hNTCP mice 								  										  									  									  										  										  											  			The hepatitis delta virus (HDV) is a dependent virus of hepatitis B virus that uses hepatitis B surface antigen to create its envelope and achieve secretion. HDV infection is the most severe form of chronic viral hepatitis. Currently, no therapies have been approved that can cure delta hepatitis. Understanding of early HDV-host dynamics post infection is lacking. I will present early serum HDV kinetics in transgenic mice expressing human NTCP (tg-hNTCP), the receptor for hepatitis B virus and HDV, and non-tg control mice after inoculation and provide insights into HDV-host interplay using mathematical modeling. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  																																		 	POPD: Subgroup Contributed Talks (1:30-2:30pm)													 						  			  				  							  							  										  									  	Wolfram Moebius								  	 								  		University of Exeter								  									  									  										  			&lt;/br&gt; Two layers of chance associated with spatially expanding populations: How demographic noise and environmental heterogeneity shape the evolutionary path of a population 								  										  									  									  										  										  											  			In nature, populations expand into new habitat at different spatial and temporal scales. The expansion process can thereby affect the evolutionary path of the growing population, a topic that has gathered much interest recently. The effects of environmental heterogeneity on the evolutionary dynamics of such range expansions remains poorly understood so far - not least due to the large variety of environmental heterogeneity found in nature. We investigate these effects in two different scenarios: neutral evolution of an expanding population and emergence of a new genotype that can spread exclusively in parts of the environment. Specifically, we first consider the effects of isolated obstacles and hotspots as well as bumps in an otherwise flat habitat. The former two are regions which hinder and accelerate the invasion, respectively. We find that those structures have characteristic consequences for neutral genetic diversity. We observe an additional layer of ‘survival of the luckiest’ – complementary to, yet qualitatively different from, founder effects occurring in the presence of 'spatial bottlenecks'. Second, we investigate the establishment of a new genotype that we allow to spread exclusively in some parts of the environment but constrain to have a selective disadvantage elsewhere. We describe the role that environmental structure has in shaping the fate of this new genotype. 									  											  									  									  									  								  								  								  	Philipp Altrock								  	 								  		H. Lee Moffitt Cancer Center								  									  									  										  			&lt;/br&gt; Time scales and wave formation in non-linear spatial public goods games 								  										  									  									  										  										  											  			Evolutionary public good (PG) games capture the essence of production of growth-beneficial factors that are vulnerable to exploitation by free-riders who do not carry the cost of production. PGs emerge in cellular populations, for example in growing bacteria and cancer cells. We study the eco-evolutionary dynamics of a PG in populations that grow in space. In our model, PG-producer cells and free-rider cells can grow according to their own birth and death rates. Co-evolution occurs due to public good-driven surplus in the intrinsic growth rates at a cost to producers. A net growth rate-benefit to free-riders leads to the well-known tragedy of the commons in which producers go extinct. What is often omitted from discussions is the time scale on which this extinction can occur, especially in spatial populations. Here, we derive analytical estimates of the ε-extinction time in differ- ent spatial settings. As we do not consider a stochastic process, the ε-extinction time captures the time needed to approach an extinction state. We identify spatial scenarios in which extinction takes long enough such that the tragedy of the commons never occurs within a meaningful lifetime of the system. Using numerical simulations we analyze the deviations from our analytical predictions. 									  											  									  									  									  								  								  								  	Debora Princepe								  	 								  		University of Campinas								  									  									  										  			&lt;/br&gt; Modeling Mito-nuclear Compatibility and Its Role in Species Identification 								  										  									  									  										  										  											  			Mitochondria play a key role in population genetics and evolutionary biology. Praised as a reliable genetic marker, the utility of mtDNA derives from its particular molecular properties, including high evolutionary rate, uniparental inheritance, and small size. Such properties make the mtDNA a powerful substrate for inferring geographic structure of populations and phylogenetic relationships. An important application is the use of a standardized segment of the mtDNA for species identification in animals, called the DNA barcode. The high rate of success to distinguish both phylogenetically close and distant species of vertebrates motivates a fundamental question: why does the barcode work and how does it relate to the nuclear DNA (nDNA) divergences during speciation? Nuclear and mitochondrial DNA’s interact during the respiration process, which depends on the coordination of genes from both sources. Recent observations of coevolution between these genomes suggest that this genetic interaction affects organism fitness; thus, mtDNA may play a fundamental role in speciation. Here we study how mito-nuclear interactions affect the speciation process and whether the accuracy of species identification by mtDNA is a consequence of the mito-nuclear coevolution. We investigate the validity of mtDNA-based barcoding in an evolving population, comparing the classification provided by mtDNA with the classification based on nuclear genetic content. Starting from an individual-based model for spatially distributed populations, we simulate the evolution of a population of individuals who carry a recombining nuclear genome and a mitochondrial genome inherited maternally. We compare a null model fitness landscape that lacks any mito-nuclear interaction against a scenario in which interactions influence fitness. Fitness is assigned to individuals according to their mito-nuclear compatibility, which drives the coevolution of the nuclear and mitochondrial genomes. Depending on the model parameters, the population breaks into distinct species and the model output then allows us to analyze the accuracy of mtDNA barcode for species identification depending on the selection strength over the mito-nuclear compatibility. We also register the signatures left in the genetic content and spatial distribution of the populations due to selection imposed on mito-nuclear compatibility. These analyses allow us to evaluate the effects of mito-nuclear interaction on the diversification process and examine to what extent mito-nuclear coevolution assists speciation. 									  											  									  									  									  								  								  							  								  								  								  								  						  										 						 																		 		  				 		  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				  				 		  			             		    		  											  		    		      		        Poster Session (2:30-3:30pm)		        	Click to view posters for each subgroup:  	 All	 CDEV	 EDUC	 IMMU	 MEPI	 MFBM	 NEUR	 POPD	 ONCO	 OTHE			             		    		  								  		    		      			      		        Sub-group Keynote		        3:30pm: 		        Robert Insall, &lt;/br&gt;University of Galsgow, @robinsall 		        Methods for Biological Modeling Subgroup 		      		      																     				       					       					     		             		    		    		    				   				    Cells, cancer and mazes: Understanding what steers cells, using cycles of mathematics, modelling and experiments. 				     					    Chemotaxis is important to many of the things cells do in vivo, particularly during development and disease. It is also a favourite topic for mathematical biologists. However, despite decades of studies of how cells can interpret chemotactic gradients, there has been very little attention to how gradients form, and where the information comes from. We have studied a particularly interesting aspect of this problem - self-generated gradients, in which cells form their own gradients at the same time as interpreting them and migrating up them. Self-generated gradients are fascinating, and ideal for mathematical biologists - they are so dynamic that they are nearly impossible to measure, and they incorporate multiple levels of feedback. I will describe how we use modelling, mathematics and experiments on Dictyostelium and cancer cells to understand different self-generated gradients. In particular I will show how cells can solve arbitrarily complex mazes, escape from tumours, and use the information in a gradient to move down-gradient as well as up-gradient. 					    				     			    			   		    		    		  									                Happy hour with friends and colleagues (4:30)                						&lt;/br&gt;			&lt;/br&gt;									 				  Overview Sunday Monday Tuesday Wednesday Thursday			 											      		      									Click to show abstracts						            "
    }, {
    "id": 28,
    "url": "http://localhost:4000/robots.txt",
    "title": "",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "Sitemap: http://localhost:4000/sitemap. xml "
    }, {
    "id": 29,
    "url": "http://localhost:4000/page2/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 30,
    "url": "http://localhost:4000/page3/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 31,
    "url": "http://localhost:4000/page4/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 32,
    "url": "http://localhost:4000/page5/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 33,
    "url": "http://localhost:4000/page6/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 34,
    "url": "http://localhost:4000/page7/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 35,
    "url": "http://localhost:4000/page8/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 36,
    "url": "http://localhost:4000/page9/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 37,
    "url": "http://localhost:4000/page10/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 38,
    "url": "http://localhost:4000/page11/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 39,
    "url": "http://localhost:4000/page12/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 40,
    "url": "http://localhost:4000/page13/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 41,
    "url": "http://localhost:4000/page14/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 42,
    "url": "http://localhost:4000/page15/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 43,
    "url": "http://localhost:4000/page16/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 44,
    "url": "http://localhost:4000/page17/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 45,
    "url": "http://localhost:4000/page18/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 46,
    "url": "http://localhost:4000/page19/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 47,
    "url": "http://localhost:4000/page20/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 48,
    "url": "http://localhost:4000/page21/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 49,
    "url": "http://localhost:4000/page22/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 50,
    "url": "http://localhost:4000/page23/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 51,
    "url": "http://localhost:4000/page24/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 52,
    "url": "http://localhost:4000/page25/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 53,
    "url": "http://localhost:4000/page26/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 54,
    "url": "http://localhost:4000/page27/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 55,
    "url": "http://localhost:4000/page28/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 56,
    "url": "http://localhost:4000/page29/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 57,
    "url": "http://localhost:4000/page30/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 58,
    "url": "http://localhost:4000/page31/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 59,
    "url": "http://localhost:4000/page32/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 60,
    "url": "http://localhost:4000/page33/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 61,
    "url": "http://localhost:4000/page34/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		      	  		  	    		    															    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	            	Click to filter by subgroup: &lt;/br&gt;	   All	   MEPI 	   CDEV	   EDUC	   IMMU	   NEUR	   MFBM	   POPD         	   ONCO 	   OTHE            &lt;/br&gt;&lt;/br&gt;       Time (EDT)   Monday   Tuesday   Wednesday   Thursday         8:00am   Opening &amp; Welcome   Coffee with friends and colleagues       8:30am   Opening Plenary&lt;/br&gt;Shayn Peirce-Cottler   Epidemiology Keynote&lt;/br&gt;Deirdre Hollingsworth&lt;/td&gt;   Education Keynote&lt;/br&gt;Alicia Prieto Langrica&lt;/td&gt;   Neuroscience Keynote&lt;/br&gt;Sue Ann Campbell&lt;/td&gt;  &lt;/tr&gt;     9:30am      	   		   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   EDUC		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	  		  			  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;		   ONCO		   MEPI		   POPD		   NEUR		   CDEV		   MFBM		   OTHE		   IMMU		   	   	  	  	         11:00am   15 minute break       11:15am   	   	   Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   	   	  	  	  		  		  Minisymposium, &lt;/br&gt; Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   EDUC	   NEUR	   CDEV	   MFBM	   OTHE	   IMMU	   	   	  	  	  		  			 	Immunobiology Keynote&lt;/br&gt;David Ho			 		             	Cell dev. biology Keynote&lt;/br&gt;Adam Martin            12:45pm   Mentoring luncheon   Subgroup business meeting   SMB business meeting   Career fair       1:30pm						   	   	   Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   IMMU	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   NEUR	   CDEV	   MFBM	   OTHE	   	   	  	  	  	  		  		  Contributed Session&lt;/br&gt;	   ONCO	   MEPI	   POPD	   CDEV	   MFBM	   OTHE	   IMMU	   	   	             2:30pm   Poster session   Poster session   Poster session             3:30pm   Population Dyn. Keynote&lt;/br&gt;Priyanga Amarasekare   Oncology Keynote&lt;/br&gt;Andrea Bild   Methods Keynote&lt;/br&gt;Robert Insall   Closing Plenary&lt;/br&gt;Carl Bergstrom       4:30pm   Happy hour with friends &amp; colleagues   Poster, Talk and Twitter awards will be announced.        &lt;/tbody&gt;&lt;/table&gt;&lt;/div&gt;	    						&lt;/div&gt;	  &lt;/div&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt;"
    }, {
    "id": 62,
    "url": "http://localhost:4000/Bejamin-J-Jessie/",
    "title": "Respiratory Syncytia Virus",
    "body": "2020/08/06 - "
    }, {
    "id": 63,
    "url": "http://localhost:4000/Michael-Pablo/",
    "title": "Early-phase decoupling between population mobility and death rates",
    "body": "2020/08/05 - "
    }, {
    "id": 64,
    "url": "http://localhost:4000/William-(Bill)-Sherwin/",
    "title": "Monitoring and Forecasting Diversity: Entropy Unifies Molecules and Ecosystems",
    "body": "2020/08/04 - "
    }, {
    "id": 65,
    "url": "http://localhost:4000/Vivian-Dornelas/",
    "title": "Impact of the landscape heterogeneity on the spatial organization of a single-species population",
    "body": "2020/08/04 - "
    }, {
    "id": 66,
    "url": "http://localhost:4000/Vandana-Revathi-Venkateswaran/",
    "title": "The effect of parental investment on immunocompetence and sexual immune dimorphism",
    "body": "2020/08/04 - "
    }, {
    "id": 67,
    "url": "http://localhost:4000/Tatiana-Yakushkina/",
    "title": "Modified Replicator Systems with Ecological DIversification",
    "body": "2020/08/04 - "
    }, {
    "id": 68,
    "url": "http://localhost:4000/Renato-Antunes-Costa-de-Andrade/",
    "title": "Persistence of intraguild predation in a 1D finite domain using variational approximations",
    "body": "2020/08/04 - "
    }, {
    "id": 69,
    "url": "http://localhost:4000/PIJUSH-PANDAY/",
    "title": "Dynamical behaviour of a stage-structured predator-prey model by incorporating cost and benefits of group defense",
    "body": "2020/08/04 - "
    }, {
    "id": 70,
    "url": "http://localhost:4000/Mauro-Mobilia/",
    "title": "Population Dynamics in a Changing Environment: Random versus Periodic Switching",
    "body": "2020/08/04 - "
    }, {
    "id": 71,
    "url": "http://localhost:4000/Luana-Tais-Bassani/",
    "title": "Basic sanitation's role over breeding sites maintenance for Aedes aegypti development during rain absence according to a fuzzy rule-based system",
    "body": "2020/08/04 - "
    }, {
    "id": 72,
    "url": "http://localhost:4000/Lee-Altenberg/",
    "title": "Spectral Graph Theory in the Analysis of Biological Evolution",
    "body": "2020/08/04 - "
    }, {
    "id": 73,
    "url": "http://localhost:4000/Jack-M-Hughes/",
    "title": "Thermodynamic Inhibition in a Biofilm Reactor with Suspended Bacteria",
    "body": "2020/08/04 - "
    }, {
    "id": 74,
    "url": "http://localhost:4000/Enrico-Sandro-Colizzi/",
    "title": "Evolution of multicellularity driven by emergent collective migration",
    "body": "2020/08/04 - "
    }, {
    "id": 75,
    "url": "http://localhost:4000/Bhawna-Malik/",
    "title": "Stochastic extinction of drug-resistant strains: Modelling the role of socio-economic factors in the pattern of extinction",
    "body": "2020/08/04 - "
    }, {
    "id": 76,
    "url": "http://localhost:4000/Anna-H-Sisk/",
    "title": "Linking Immuno-Epidemiology Principles to Violence",
    "body": "2020/08/04 - "
    }, {
    "id": 77,
    "url": "http://localhost:4000/Vitor-de-Oliveira-Sudbrack/",
    "title": "Population dynamics in highly fragmented landscapes",
    "body": "2020/08/03 - "
    }, {
    "id": 78,
    "url": "http://localhost:4000/Vahini-Reddy-Nareddy/",
    "title": "Representing spatial ecological oscillators by dynamical Ising model with memory",
    "body": "2020/08/03 - "
    }, {
    "id": 79,
    "url": "http://localhost:4000/So-Nakashima/",
    "title": "Lineage EM Algorithm for Inferring Latent States from Cellular Lineage Trees",
    "body": "2020/08/03 - "
    }, {
    "id": 80,
    "url": "http://localhost:4000/Silas-Poloni/",
    "title": "Intraguild Predation in Periodic Habitats",
    "body": "2020/08/03 - "
    }, {
    "id": 81,
    "url": "http://localhost:4000/Shota-Shibasaki/",
    "title": "Fluctuating environments affect the strength of species interactions and diversity in microbial communities similarly",
    "body": "2020/08/03 - "
    }, {
    "id": 82,
    "url": "http://localhost:4000/Russell-Milne/",
    "title": "Effects of Variation in Fishing Rate and Nutrient Loading on Coral Reef Health with Implications for Marine Protected Area Design",
    "body": "2020/08/03 - "
    }, {
    "id": 83,
    "url": "http://localhost:4000/Rom%C3%A1n-Zapi%C3%A9n-Campos/",
    "title": "When timescales meet: Microbiome dynamics is influenced by hosts’ life-history.",
    "body": "2020/08/03 - "
    }, {
    "id": 84,
    "url": "http://localhost:4000/Renier-G-Mendoza/",
    "title": "Explicit solution of an age-structured model using a generalized Lambert W function",
    "body": "2020/08/03 - "
    }, {
    "id": 85,
    "url": "http://localhost:4000/Pawe%C5%82-Klimasara/",
    "title": "A Savanna Dynamics Model",
    "body": "2020/08/03 - "
    }, {
    "id": 86,
    "url": "http://localhost:4000/Lucy-Lansch-Justen/",
    "title": "Evolutionary rescue from mutational meltdown",
    "body": "2020/08/03 - "
    }, {
    "id": 87,
    "url": "http://localhost:4000/Laurie-Balstad/",
    "title": "Parasite intensity influences evolution of migratory behavior via migratory escape",
    "body": "2020/08/03 - "
    }, {
    "id": 88,
    "url": "http://localhost:4000/Lauren-Mossman-Kylie-Landa/",
    "title": "Phage-antibiotic synergy inhibited by temperate and chronic virus competition",
    "body": "2020/08/03 - "
    }, {
    "id": 89,
    "url": "http://localhost:4000/Kathyrn-R-Fair/",
    "title": "Spatial Structure in Protected Forest-Grassland Mosaics: Exploring Futures Under Climate Change",
    "body": "2020/08/03 - "
    }, {
    "id": 90,
    "url": "http://localhost:4000/Jorge-Arroyo-Esquivel/",
    "title": "Spatial dynamics and spread of ecosystem engineers",
    "body": "2020/08/03 - "
    }, {
    "id": 91,
    "url": "http://localhost:4000/Fernando-Luiz-Pio-dos-Santos/",
    "title": "Investigation of the Aedes spread using a reaction-diffusion mathematical model",
    "body": "2020/08/03 - "
    }, {
    "id": 92,
    "url": "http://localhost:4000/Femke-Thon/",
    "title": "Modeling intraspecific chemodiversity - theory and first results",
    "body": "2020/08/03 - "
    }, {
    "id": 93,
    "url": "http://localhost:4000/Felipe-E-M-Campos/",
    "title": "A modelling approach to study landscape effects on abundance from patch demographic processes",
    "body": "2020/08/03 - "
    }, {
    "id": 94,
    "url": "http://localhost:4000/Easton-R.-White/",
    "title": "Catastrophes and socio-ecological systems",
    "body": "2020/08/03 - "
    }, {
    "id": 95,
    "url": "http://localhost:4000/Dominic-Brass/",
    "title": "Phenotypic plasticity as a cause and consequence of population dynamics",
    "body": "2020/08/03 - "
    }, {
    "id": 96,
    "url": "http://localhost:4000/Connah-Griffith-Michael-Johnson/",
    "title": "Developing a computational tool for multiscale simulations of chemically coupled cell populations",
    "body": "2020/08/03 - "
    }, {
    "id": 97,
    "url": "http://localhost:4000/Christopher-Carlson/",
    "title": "Partner Specificity in Mutualisms",
    "body": "2020/08/03 - "
    }, {
    "id": 98,
    "url": "http://localhost:4000/Caren-Barcel%C3%B3/",
    "title": "Projecting the time scale of initial increase in fishery yield after implementation of marine protected areas",
    "body": "2020/08/03 - "
    }, {
    "id": 99,
    "url": "http://localhost:4000/Bryce-Morsky/",
    "title": "Evolution of contribution timing in public goods games",
    "body": "2020/08/03 - "
    }, {
    "id": 100,
    "url": "http://localhost:4000/Bj%C3%B6rn-Vessman/",
    "title": "A theoretical investigation of artificial community selection methods",
    "body": "2020/08/03 - "
    }, {
    "id": 101,
    "url": "http://localhost:4000/Antonela-Marozzi/",
    "title": "An alternative method for calf density and recruitment estimation using pregnancy on wild guanacos (Lama guanicoe).",
    "body": "2020/08/03 - "
    }, {
    "id": 102,
    "url": "http://localhost:4000/Angela-Lynn-Peace/",
    "title": "A Simple model of pathogen-mediated nutrient dynamics",
    "body": "2020/08/03 - "
    }, {
    "id": 103,
    "url": "http://localhost:4000/Adelle-Coster/",
    "title": "The Distance Between: Stochastic Models of Cellular Protein Transport",
    "body": "2020/08/03 - "
    }, {
    "id": 104,
    "url": "http://localhost:4000/Zviiteyi-Chazuka/",
    "title": "A mathematical model for in host dynamics of an immune evading virus",
    "body": "2020/08/02 - "
    }, {
    "id": 105,
    "url": "http://localhost:4000/Zeinab-Tajik-Mansoury/",
    "title": "Opioid Addiction Affects Neuronal Synchronization in the Hippocampus: A Computational Model",
    "body": "2020/08/02 - "
    }, {
    "id": 106,
    "url": "http://localhost:4000/Yeeren-Low/",
    "title": "Data-driven modeling of T cell morphodynamics during migration",
    "body": "2020/08/02 - "
    }, {
    "id": 107,
    "url": "http://localhost:4000/Yahe-Yu/",
    "title": "Finding Optimal STI Strategies for HIV Using Fitted Q Algorithm with XGBoost Regression",
    "body": "2020/08/02 - "
    }, {
    "id": 108,
    "url": "http://localhost:4000/Yafei-Wang/",
    "title": "Multicellular simulation in cancer treatment with nanoparticles",
    "body": "2020/08/02 - "
    }, {
    "id": 109,
    "url": "http://localhost:4000/Xinzhe-Zuo/",
    "title": "Backtracking in RNA polymerization",
    "body": "2020/08/02 - "
    }, {
    "id": 110,
    "url": "http://localhost:4000/William-D-Martinson/",
    "title": "Comparative analysis of continuum angiogenesis models",
    "body": "2020/08/02 - "
    }, {
    "id": 111,
    "url": "http://localhost:4000/Will-Leone/",
    "title": "Understanding factors contributing to bacterial burden in granulomas of Mycobacterium tuberculosis-infected monkeys",
    "body": "2020/08/02 - "
    }, {
    "id": 112,
    "url": "http://localhost:4000/Vojtech-Kumpost/",
    "title": "Elucidating Mechanisms Underlying Experimentally Induced Changes in the Zebrafish Circadian Clock",
    "body": "2020/08/02 - "
    }, {
    "id": 113,
    "url": "http://localhost:4000/Viktor-Zenkov/",
    "title": "A minority of liver-resident CD8 T cells searching for Plasmodium-infected hepatocytes demonstrate difficult-to-detect attraction",
    "body": "2020/08/02 - "
    }, {
    "id": 114,
    "url": "http://localhost:4000/Vehpi-Yildirim/",
    "title": "Mathematical Modeling Postprandial Lipoprotein Metabolism and Investigating the Effects of the Bariatric Surgery",
    "body": "2020/08/02 - "
    }, {
    "id": 115,
    "url": "http://localhost:4000/Vedang-Narain/",
    "title": "A boids-based model of collective cell migration in wound healing",
    "body": "2020/08/02 - "
    }, {
    "id": 116,
    "url": "http://localhost:4000/Tiffany-Leung/",
    "title": "Evaluating the effectiveness of social distancing interventions: delaying the epidemic or flattening the curve?",
    "body": "2020/08/02 - "
    }, {
    "id": 117,
    "url": "http://localhost:4000/Thomas-N-Vilches/",
    "title": "Assessing the effects of the diagnostic methods on the schistosomiasis dynamics",
    "body": "2020/08/02 - "
    }, {
    "id": 118,
    "url": "http://localhost:4000/Thi-Minh-Thao-Le/",
    "title": "Quasi-neutral dynamics in a co-infection system with N strains",
    "body": "2020/08/02 - "
    }, {
    "id": 119,
    "url": "http://localhost:4000/Suzanne-Sindi/",
    "title": "Multiscale Modeling of Protein Aggregation",
    "body": "2020/08/02 - "
    }, {
    "id": 120,
    "url": "http://localhost:4000/Sungyoung-Shin/",
    "title": "A mathematical model of GβL (de)ubiquitination switch uncovers biphasic response within the PI3K/mTOR signalling network",
    "body": "2020/08/02 - "
    }, {
    "id": 121,
    "url": "http://localhost:4000/Subbalakshmi-A-R/",
    "title": "NFATc acts as a non-canonical phenotypic stability factor for a hybrid epithelial/mesenchymal phenotype",
    "body": "2020/08/02 - "
    }, {
    "id": 122,
    "url": "http://localhost:4000/Stephen-Y-Zhang/",
    "title": "Inference of stochastic cellular dynamics from time-series data using optimal transport",
    "body": "2020/08/02 - "
    }, {
    "id": 123,
    "url": "http://localhost:4000/Stefano-Pasetto/",
    "title": "Bayesian framework for tumor board decision making",
    "body": "2020/08/02 - "
    }, {
    "id": 124,
    "url": "http://localhost:4000/Soumen-Bera/",
    "title": "Biphasic Adaptive Activity of Plant Nitrate Transporter NRT1.1",
    "body": "2020/08/02 - "
    }, {
    "id": 125,
    "url": "http://localhost:4000/Simona-Catozzi/",
    "title": "Predicted ‘wiring landscape’ of Ras network in 29 human tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 126,
    "url": "http://localhost:4000/Simon-Mitchell/",
    "title": "A Systems Biology Approach Predicts Distinct Roles for NFkB Subunits cRel and RelA in DLBCL",
    "body": "2020/08/02 - "
    }, {
    "id": 127,
    "url": "http://localhost:4000/Shohel-Ahmed/",
    "title": "Global Asymptotic Stability for a Diffusive Opportunistic Diseases Model",
    "body": "2020/08/02 - "
    }, {
    "id": 128,
    "url": "http://localhost:4000/Shelby-M-Scott/",
    "title": "COVID and Crime: Analysis of Crime Dynamics Amidst Social Distancing Protocols",
    "body": "2020/08/02 - "
    }, {
    "id": 129,
    "url": "http://localhost:4000/Shawn-Ryan/",
    "title": "Role of Hydrodynamics in Chemotaxis of Bacterial Populations",
    "body": "2020/08/02 - "
    }, {
    "id": 130,
    "url": "http://localhost:4000/Shaon-C-Chakrabarti/",
    "title": "Why cousins are more similar than mother daughters: implications for cell cycle control",
    "body": "2020/08/02 - "
    }, {
    "id": 131,
    "url": "http://localhost:4000/Shadi-Esmaeili-Wellman/",
    "title": "Density Dependent Resource Budget Model for Alternate Bearing",
    "body": "2020/08/02 - "
    }, {
    "id": 132,
    "url": "http://localhost:4000/Seokjoo-Chae/",
    "title": "The data-based inference method reveals the network structure of the SCN",
    "body": "2020/08/02 - "
    }, {
    "id": 133,
    "url": "http://localhost:4000/Sebastian-Ruhle/",
    "title": "Analysis of cell contact inhibition during growth of epithelial tissue",
    "body": "2020/08/02 - "
    }, {
    "id": 134,
    "url": "http://localhost:4000/Sayan-Biswas/",
    "title": "Predicting mitochondrial functional state : An imaging based SVM tool",
    "body": "2020/08/02 - "
    }, {
    "id": 135,
    "url": "http://localhost:4000/Sarah-MacQueen/",
    "title": "Modelling the effects of site constancy in bumble bees",
    "body": "2020/08/02 - "
    }, {
    "id": 136,
    "url": "http://localhost:4000/Sara-Sommariva/",
    "title": "Mathematical model of loss and gain of function mutations in a chemical reaction network for colorectal cancer cells.",
    "body": "2020/08/02 - "
    }, {
    "id": 137,
    "url": "http://localhost:4000/Ryan-Schenck/",
    "title": "The Tick-Tock of the Molecular Clock: Random methylation state changes inform homeostasis in the intestinal crypt",
    "body": "2020/08/02 - "
    }, {
    "id": 138,
    "url": "http://localhost:4000/Ryan-Murphy/",
    "title": "Mechanical cell competition in heterogeneous epithelial tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 139,
    "url": "http://localhost:4000/Roxana-L%C3%B3pez-Cruz/",
    "title": "A Coupled Mathematical Model Introducing Biological Control of Dengue",
    "body": "2020/08/02 - "
    }, {
    "id": 140,
    "url": "http://localhost:4000/Rodrigo-Garcia/",
    "title": "Small-worldness favours network inference in synthetic neural networks",
    "body": "2020/08/02 - "
    }, {
    "id": 141,
    "url": "http://localhost:4000/Robyn-Shuttleworth/",
    "title": "Application of a cell-dense triphasic model to cryoprotectant equilibration",
    "body": "2020/08/02 - "
    }, {
    "id": 142,
    "url": "http://localhost:4000/Rifaldy-Fajar/",
    "title": "Analysis of Mathematical Model on the Development of Tumor Cells after Drug Therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 143,
    "url": "http://localhost:4000/Rhudaina-Z.-Mohammad/",
    "title": "Cellular patterning in sensory systems: An interface evolution problem",
    "body": "2020/08/02 - "
    }, {
    "id": 144,
    "url": "http://localhost:4000/Rey-Audie-Escosio/",
    "title": "Parameter Estimation of the Fitzhugh-Nagumo Model via a Perturbed Accelerated Gradient Descent Algorithm with an n-Dimensional Golden Section Search Method",
    "body": "2020/08/02 - "
    }, {
    "id": 145,
    "url": "http://localhost:4000/Renee-Dale/",
    "title": "Using a trait-based dynamic mathematical framework to investigate the relationship between phenotypic dynamical parameters and the plant genome",
    "body": "2020/08/02 - "
    }, {
    "id": 146,
    "url": "http://localhost:4000/Reginaldo-Jos%C3%A9-da-Silva/",
    "title": "Acoustic measures for voices in the classification of Parkinson's Disease",
    "body": "2020/08/02 - "
    }, {
    "id": 147,
    "url": "http://localhost:4000/Rebecca-Sanft/",
    "title": "Exploring Mathematical Modeling in Biology Through Case Studies and Experimental Activities",
    "body": "2020/08/02 - "
    }, {
    "id": 148,
    "url": "http://localhost:4000/Rebecca-H-Chisholm/",
    "title": "A model of population dynamics with complex household structure and mobility: implications for transmission and control of communicable diseases",
    "body": "2020/08/02 - "
    }, {
    "id": 149,
    "url": "http://localhost:4000/Rebecca-Bekker/",
    "title": "Immunological Consequences of Uniform vs Spatially Fractionated Radiotherapy",
    "body": "2020/08/02 - "
    }, {
    "id": 150,
    "url": "http://localhost:4000/Ranjini-Bhattacharya/",
    "title": "Interpreting the Evolutionary Games Played in the NSCLC Microenvironment",
    "body": "2020/08/02 - "
    }, {
    "id": 151,
    "url": "http://localhost:4000/Rahul-Bishnoi/",
    "title": "Eliminating the Need for User-Definition of Gap Penalties using Linear Programming",
    "body": "2020/08/02 - "
    }, {
    "id": 152,
    "url": "http://localhost:4000/Rahma-Jerbi/",
    "title": "modélisation mathématique",
    "body": "2020/08/02 - "
    }, {
    "id": 153,
    "url": "http://localhost:4000/Rafael-Ramon-Bravo/",
    "title": "Cancer Crusade: Crowdsourcing Adaptive Therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 154,
    "url": "http://localhost:4000/Quan-Anh-Hoang/",
    "title": "Analyzing Bacterial Motility Near a Smooth Surface",
    "body": "2020/08/02 - "
    }, {
    "id": 155,
    "url": "http://localhost:4000/Priyom-Adhyapok/",
    "title": "Modeling liver injury progression and repair",
    "body": "2020/08/02 - "
    }, {
    "id": 156,
    "url": "http://localhost:4000/Prashant-K-Srivastava/",
    "title": "The impact of information and saturated treatment with time delay in an Infectious disease model",
    "body": "2020/08/02 - "
    }, {
    "id": 157,
    "url": "http://localhost:4000/Praneeth-Reddy-Sudalagunta/",
    "title": "A pharmacodynamic model of clinical synergy in multiple myeloma",
    "body": "2020/08/02 - "
    }, {
    "id": 158,
    "url": "http://localhost:4000/Pranav-Khade/",
    "title": "Using alpha shapes to characterize protein flexibility",
    "body": "2020/08/02 - "
    }, {
    "id": 159,
    "url": "http://localhost:4000/Prachi-Bisht/",
    "title": "Interface growth driven by a single active particle",
    "body": "2020/08/02 - "
    }, {
    "id": 160,
    "url": "http://localhost:4000/Pooja-Dnyane/",
    "title": "Boolean Model for Melanogenesis",
    "body": "2020/08/02 - "
    }, {
    "id": 161,
    "url": "http://localhost:4000/Phillip-Rossbach/",
    "title": "Determination of the critical adhesion parameter for the sorting behavior of a cell system with several cell types using statistical learning methods",
    "body": "2020/08/02 - "
    }, {
    "id": 162,
    "url": "http://localhost:4000/Phebe-M.-A-Havor/",
    "title": "Dynamics of disease models with self-diffusion: Case study of Cholera in Ghana",
    "body": "2020/08/02 - "
    }, {
    "id": 163,
    "url": "http://localhost:4000/Perminov-Dmitrievich-Valeriy/",
    "title": "Agent-based models for influenza epidemic dynamics and its decision-making capability",
    "body": "2020/08/02 - "
    }, {
    "id": 164,
    "url": "http://localhost:4000/Pedro-Vilanova/",
    "title": "Synchronization in Stochastic Oscillators Subject to Common Extrinsic Noise",
    "body": "2020/08/02 - "
    }, {
    "id": 165,
    "url": "http://localhost:4000/Paul-J-Hurtado/",
    "title": "Reproduction Numbers for ODE Models of Arbitrary Finite Dimension: An Application of the Generalized Linear Chain Trick",
    "body": "2020/08/02 - "
    }, {
    "id": 166,
    "url": "http://localhost:4000/Paul-D-Alexander/",
    "title": "Treatment of Viral Co-Infections",
    "body": "2020/08/02 - "
    }, {
    "id": 167,
    "url": "http://localhost:4000/Paul-A-Roberts/",
    "title": "Using mathematics to investigate the mechanisms behind vision loss",
    "body": "2020/08/02 - "
    }, {
    "id": 168,
    "url": "http://localhost:4000/Pau-Capera-Aragones/",
    "title": "A mechanistic partial integro-differential equations model for bumblebee foraging behaviour",
    "body": "2020/08/02 - "
    }, {
    "id": 169,
    "url": "http://localhost:4000/Patrick-Gelbach/",
    "title": "Metabolomics and Mechanistic Kinetic Modeling Reveal Mechanisms Driving Intracellular Metabolism and Insulin Secretion in Pancreatic Beta Cells",
    "body": "2020/08/02 - "
    }, {
    "id": 170,
    "url": "http://localhost:4000/Pappu-Kumar/",
    "title": "Theoretical investigation of non-equilibrium bio-heat transfer during thermal therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 171,
    "url": "http://localhost:4000/Pamela-Kim-N-Salonga/",
    "title": "A mathematical model of the dynamics of lymphatic filariasis in Caraga Region, The Philippines",
    "body": "2020/08/02 - "
    }, {
    "id": 172,
    "url": "http://localhost:4000/Oke-I-Segun/",
    "title": "Mathematical model for the estrogen paradox in breast cancer treatment",
    "body": "2020/08/02 - "
    }, {
    "id": 173,
    "url": "http://localhost:4000/Noemi-Andor/",
    "title": "Invasion of homogeneous and polyploid populations in nutrient-limiting environments",
    "body": "2020/08/02 - "
    }, {
    "id": 174,
    "url": "http://localhost:4000/Nikolaos-Dimitriou/",
    "title": "Modelling geometric patterns of cancer progression",
    "body": "2020/08/02 - "
    }, {
    "id": 175,
    "url": "http://localhost:4000/Nicole-Althermeler/",
    "title": "Ancestral lines under selection for multiple sites: Pruning the Ancestral Selection Graph",
    "body": "2020/08/02 - "
    }, {
    "id": 176,
    "url": "http://localhost:4000/Nara-Yoon/",
    "title": "Modeling collaterally sensitive drug cycles: shaping heterogeneity to allow adaptive therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 177,
    "url": "http://localhost:4000/Muhammad-Said/",
    "title": "Sensitivity and stability analysis of an Ebola Virus disease and GB virus C co-infection.",
    "body": "2020/08/02 - "
    }, {
    "id": 178,
    "url": "http://localhost:4000/Muhammad-Humayun-Kabir/",
    "title": "Model-aided Understanding of the Outbreak of COVID-19 in Bangladesh",
    "body": "2020/08/02 - "
    }, {
    "id": 179,
    "url": "http://localhost:4000/Mubasher-Rashid-Rather/",
    "title": "Organization of biogeochemical nitrogen pathways with switch-like adjustment in fluctuating soil redox conditions",
    "body": "2020/08/02 - "
    }, {
    "id": 180,
    "url": "http://localhost:4000/Mohsin-Saleet-Jafri/",
    "title": "Predicting Drug Resistance by Applying Machine Learning to Molecular Simulation",
    "body": "2020/08/02 - "
    }, {
    "id": 181,
    "url": "http://localhost:4000/Mohammad-U-Zahid/",
    "title": "Forecasting Individual Patient Response to Radiotherapy with a Dynamic Carrying Capacity Model in Head and Neck Cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 182,
    "url": "http://localhost:4000/Moacyr-A-H-B-Silva/",
    "title": "A death-based mathematical model of the ongoing COVID-19 pandemic",
    "body": "2020/08/02 - "
    }, {
    "id": 183,
    "url": "http://localhost:4000/Min-Song/",
    "title": "Model Predicts Distinct Mechanisms of Endothelial Cell Growth Upon the Stimulation of FGF and VEGF",
    "body": "2020/08/02 - "
    }, {
    "id": 184,
    "url": "http://localhost:4000/Miller-Orlando-Cer%C3%B3n/",
    "title": "Mathematical model with carriers and general non-linear incidence rate",
    "body": "2020/08/02 - "
    }, {
    "id": 185,
    "url": "http://localhost:4000/Milad-Ghomlaghi/",
    "title": "Akt regulates PIP3 production by PI3K to form a potent negative feedback loop",
    "body": "2020/08/02 - "
    }, {
    "id": 186,
    "url": "http://localhost:4000/Mikahl-Banwarth-Kuhn/",
    "title": "Quantifying the Biophysical Impact of Budding Cell Division on the Spatial Organization of Growing Yeast Colonies",
    "body": "2020/08/02 - "
    }, {
    "id": 187,
    "url": "http://localhost:4000/Michelle-Przedborski/",
    "title": "A combined systems biology and machine learning approach to study patient response to anti-PD1 immunotherapy",
    "body": "2020/08/02 - "
    }, {
    "id": 188,
    "url": "http://localhost:4000/Masud-M-A/",
    "title": "",
    "body": "2020/08/02 - "
    }, {
    "id": 189,
    "url": "http://localhost:4000/Marvin-Fritz/",
    "title": "On the modelling and analysis of tumor growth with phase-field equations of Cahn-Hilliard type",
    "body": "2020/08/02 - "
    }, {
    "id": 190,
    "url": "http://localhost:4000/Martin-Lopez-Garcia/",
    "title": "Exact approaches for the analysis of stochastic epidemic processes on small networks",
    "body": "2020/08/02 - "
    }, {
    "id": 191,
    "url": "http://localhost:4000/Marta-Helena-Oliveira/",
    "title": "MATHEMATICAL MODELLING OF THE INFLAMMATORY PHASE OF SKIN WOUND HEALING IN RATS",
    "body": "2020/08/02 - "
    }, {
    "id": 192,
    "url": "http://localhost:4000/Mark-Robertson-Tessi/",
    "title": "Immune predation promotes aggressive metabolic phenotypes in a context-dependent manner",
    "body": "2020/08/02 - "
    }, {
    "id": 193,
    "url": "http://localhost:4000/Marilin-Nathalya-Guerrero-Laos/",
    "title": "Approach and analysis of a mathematical model in ordinary differential equations, applied to HIV transmission, considering protection strategies.",
    "body": "2020/08/02 - "
    }, {
    "id": 194,
    "url": "http://localhost:4000/Marc-Pereyra/",
    "title": "3D Particle image velocimetry (PIV) analysis of cellular migration during embryonic development",
    "body": "2020/08/02 - "
    }, {
    "id": 195,
    "url": "http://localhost:4000/Lutz-Brusch/",
    "title": "Morpheus: A user-friendly simulation framework for multi-cellular systems biology",
    "body": "2020/08/02 - "
    }, {
    "id": 196,
    "url": "http://localhost:4000/Luke-Andrejek/",
    "title": "A Robust Mathematical Model of Adaxial-Abaxial Patterning",
    "body": "2020/08/02 - "
    }, {
    "id": 197,
    "url": "http://localhost:4000/Lucas-Boettcher/",
    "title": "Unifying continuous, discrete, and hybrid susceptible-infected-recovered processes on networks",
    "body": "2020/08/02 - "
    }, {
    "id": 198,
    "url": "http://localhost:4000/Lucas-Barberis/",
    "title": "Modeling Helps Understand the Influence of Substrate on Tumorsphere Growth",
    "body": "2020/08/02 - "
    }, {
    "id": 199,
    "url": "http://localhost:4000/Linnea-Franssen/",
    "title": "Linnea Franssen",
    "body": "2020/08/02 - layout: single-posterauthor: esmbday: “All”group: “poster”title: “A Mathematical Framework for Modelling the Metastatic Spread of Cancer”subgroup: “ONCO”presenter: “Linnea Franssen”institution: “Roche Pharmaceutical Research and Early Development (Translational Modeling and Simulation)”abstract: “Cancer is a complex disease that starts with mutations of key genes in one cell or a small group of cells at a primary site in the body. If these cancer cells continue to grow successfully and, at some later stage, invade the surrounding tissue and acquire a vascular network, they can spread to distant secondary sites in the body. This process, known as metastatic spread, is responsible for around 90— "
    }, {
    "id": 200,
    "url": "http://localhost:4000/Leili-Shahriyari/",
    "title": "A path toward personalized cancer treatments",
    "body": "2020/08/02 - "
    }, {
    "id": 201,
    "url": "http://localhost:4000/Lee-Curtin/",
    "title": "Lacunarity and fractal dimension as prognostic biomarkers in glioblastoma",
    "body": "2020/08/02 - "
    }, {
    "id": 202,
    "url": "http://localhost:4000/Lan-K-Nguyen/",
    "title": "Integrative mathematical modelling unveils hidden mechanism of resistance to PI3K  inhibition and identifies new effective combination therapies for breast cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 203,
    "url": "http://localhost:4000/Koushik-Garain/",
    "title": "Treatment control is one of the most important factor to reduce the spread of COVID-19 epidemic",
    "body": "2020/08/02 - "
    }, {
    "id": 204,
    "url": "http://localhost:4000/Khem-Raj-Ghusinga/",
    "title": "Molecular switch architecture determines response properties of signaling pathways",
    "body": "2020/08/02 - "
    }, {
    "id": 205,
    "url": "http://localhost:4000/Kento-Nakamura/",
    "title": "Optimality of the sensory system of Escherichia coli",
    "body": "2020/08/02 - "
    }, {
    "id": 206,
    "url": "http://localhost:4000/Kelly-R-Buch/",
    "title": "Mathematical Model of Basal Sprout Production in Response to Laurel Wilt",
    "body": "2020/08/02 - "
    }, {
    "id": 207,
    "url": "http://localhost:4000/Kellen-Myers/",
    "title": "Resource Limitations and Household Economics in Outbreaks",
    "body": "2020/08/02 - "
    }, {
    "id": 208,
    "url": "http://localhost:4000/Keertana-Yalamanchili/",
    "title": "Assessing the Efficacy of Various NOX Enzyme Inhibitors as Potential Treatments for Ischemic Stroke in Silico",
    "body": "2020/08/02 - "
    }, {
    "id": 209,
    "url": "http://localhost:4000/Karina-Vilches/",
    "title": "A mathematical approach for visualizing cell migration during tumor progression",
    "body": "2020/08/02 - "
    }, {
    "id": 210,
    "url": "http://localhost:4000/Junho-Lee/",
    "title": "Synergistic Effects of Bortezomib-OV Therapy and  Anti-Invasive Strategies in Glioblastoma:  A Mathematical Model",
    "body": "2020/08/02 - "
    }, {
    "id": 211,
    "url": "http://localhost:4000/Jungmin-Han/",
    "title": "Missing Data Imputation and Gene Network Inference in Single Cell Analysis",
    "body": "2020/08/02 - "
    }, {
    "id": 212,
    "url": "http://localhost:4000/Juliano-Ferrari-Gianlupi/",
    "title": "Multiscale spatiotemporal modeling of acute primary viral infection and immune response in epithelial tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 213,
    "url": "http://localhost:4000/Juliane-Schr%C3%B6ter/",
    "title": "A theoretical model for the natural course of HIV infection in infants",
    "body": "2020/08/02 - "
    }, {
    "id": 214,
    "url": "http://localhost:4000/Juan-Calvo/",
    "title": "The initial-boundary value problem for the Lifshitz-Slyozov system with inflow boundary conditions: Analysis and applications",
    "body": "2020/08/02 - "
    }, {
    "id": 215,
    "url": "http://localhost:4000/Joshua-A-Bull/",
    "title": "Description of immune cell infiltration in solid tumours using spatial statistics and topological data analysis",
    "body": "2020/08/02 - "
    }, {
    "id": 216,
    "url": "http://localhost:4000/Josephine-Naa-Ayeley-Tetteh/",
    "title": "Switching strategy for mitigation against bacterial resistance",
    "body": "2020/08/02 - "
    }, {
    "id": 217,
    "url": "http://localhost:4000/Joseph-S-Abrams/",
    "title": "A CELL-BASED MODEL OF INTERCELLULAR MECHANICS DURING CPA EQUILIBRATION IN PREANTRAL OVARIAN FOLLICLES",
    "body": "2020/08/02 - "
    }, {
    "id": 218,
    "url": "http://localhost:4000/Jordy-Jose-Cevallos-Chavez/",
    "title": "Mobility impact in the spreading of COVID-19 in Ecuador ",
    "body": "2020/08/02 - "
    }, {
    "id": 219,
    "url": "http://localhost:4000/Jonggul-Lee/",
    "title": "Machine Learning for Risk Prediction of Highly Pathogenic Avian In uenza in the Republic of Korea",
    "body": "2020/08/02 - "
    }, {
    "id": 220,
    "url": "http://localhost:4000/Jonas-Knoch/",
    "title": "A mathematical model of HIF-1 regulated cellular energy metabolism",
    "body": "2020/08/02 - "
    }, {
    "id": 221,
    "url": "http://localhost:4000/John-R-Jungck/",
    "title": "Ethnomathematics: Art, Culture and Social Justice",
    "body": "2020/08/02 - "
    }, {
    "id": 222,
    "url": "http://localhost:4000/John-Metzcar/",
    "title": "Mathematical modeling of leader-follower cell invasion of tumor-associated stroma using a novel extracellular matrix model",
    "body": "2020/08/02 - "
    }, {
    "id": 223,
    "url": "http://localhost:4000/Jim-Greene/",
    "title": "A novel COVID-19 model reveals unexpected consequences of social distancing strategies",
    "body": "2020/08/02 - "
    }, {
    "id": 224,
    "url": "http://localhost:4000/Jessica-Wellington/",
    "title": "A Mathematical Model to Investigate Iron Allocation in Plants",
    "body": "2020/08/02 - "
    }, {
    "id": 225,
    "url": "http://localhost:4000/Jessica-Conway/",
    "title": "Stochastic time-inhomogeneous HIV dynamics following treatment suspension",
    "body": "2020/08/02 - "
    }, {
    "id": 226,
    "url": "http://localhost:4000/Jayrah-Bena-E-Ri%C3%B1on/",
    "title": "A Mathematical Model and Optimal Control of Schistosomiasis in Agusan del Sur, Philippines",
    "body": "2020/08/02 - "
    }, {
    "id": 227,
    "url": "http://localhost:4000/Javier-C-Urcuyo/",
    "title": "Understanding glioblastoma-macrophage interactions through radiomics, transcriptome sequencing, and mathematical modeling",
    "body": "2020/08/02 - "
    }, {
    "id": 228,
    "url": "http://localhost:4000/Jared-Barber/",
    "title": "Mathematical and mechanical models of cell motion",
    "body": "2020/08/02 - "
    }, {
    "id": 229,
    "url": "http://localhost:4000/Janina-Hesse/",
    "title": "Diurnal variation in gene expression and sports performance: a matter of timing?",
    "body": "2020/08/02 - "
    }, {
    "id": 230,
    "url": "http://localhost:4000/Jairo-G-Silva/",
    "title": "Mathematical Models About Radioactive Iodine-Refractory Differentiated Thyroid Cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 231,
    "url": "http://localhost:4000/Jahedi-Sana/",
    "title": "When the best pandemic models are the simplest",
    "body": "2020/08/02 - "
    }, {
    "id": 232,
    "url": "http://localhost:4000/Jackelyn-M-Kembro/",
    "title": "Accelerometers as a tool to characterize reproductive behavior within social groups in long term experiments: the case of the Japanese Quill",
    "body": "2020/08/02 - "
    }, {
    "id": 233,
    "url": "http://localhost:4000/Iulia-Martina-Bulai/",
    "title": "Modeling COVID-19 considering asymptomatic cases and avoid contacts",
    "body": "2020/08/02 - "
    }, {
    "id": 234,
    "url": "http://localhost:4000/Ielyaas-Cloete/",
    "title": "Not all hormone receptors are created equal",
    "body": "2020/08/02 - "
    }, {
    "id": 235,
    "url": "http://localhost:4000/Himanshu-Aggarwal/",
    "title": "Impact of temperature change on Malaria incidence in high altitude regions of India",
    "body": "2020/08/02 - "
    }, {
    "id": 236,
    "url": "http://localhost:4000/Heber-Rocha/",
    "title": "Qualitative study of cell migration associated with hypoxia in the dynamics of tumor growth",
    "body": "2020/08/02 - "
    }, {
    "id": 237,
    "url": "http://localhost:4000/Haryana-Thomas/",
    "title": "Modeling Cellular Signaling and Mesangial Fibrosis during Diabetic Kidney",
    "body": "2020/08/02 - "
    }, {
    "id": 238,
    "url": "http://localhost:4000/Harsimran-Kaur/",
    "title": "Computational modeling of mechano-metabolic adaptation to a stiff microenvironment in Cancer cells",
    "body": "2020/08/02 - "
    }, {
    "id": 239,
    "url": "http://localhost:4000/Hana-Dobrovolny/",
    "title": "SARS-CoV-2 coinfections: Implications for the second wave",
    "body": "2020/08/02 - "
    }, {
    "id": 240,
    "url": "http://localhost:4000/Gustavo-J-Sibona/",
    "title": "SIRS dynamics on a diffusive agent system",
    "body": "2020/08/02 - "
    }, {
    "id": 241,
    "url": "http://localhost:4000/Giulio-Bonifazi/",
    "title": "Predicting excitotoxicity at the onset of Alzheimer’s disease by a model of Aβ-dependent trafficking of astrocytic glutamate transporters",
    "body": "2020/08/02 - "
    }, {
    "id": 242,
    "url": "http://localhost:4000/Giulia-Laura-Celora/",
    "title": "Analysis of the dynamics of tumour cells along a stemness axis under different oxygen conditions",
    "body": "2020/08/02 - "
    }, {
    "id": 243,
    "url": "http://localhost:4000/Gilberto-C-Gonzalez-Parra/",
    "title": "Forecasting cases of RSV using artificial neural networks and mechanistic models",
    "body": "2020/08/02 - "
    }, {
    "id": 244,
    "url": "http://localhost:4000/Gess-Iraji/",
    "title": "Modeling Clogging of Red Blood Cells in Microfluidic Devices with Simple Geometry",
    "body": "2020/08/02 - "
    }, {
    "id": 245,
    "url": "http://localhost:4000/Gemma-Massonis/",
    "title": "Structural Identifiability and Observability of Compartmental Models of the COVID-19 Pandemic",
    "body": "2020/08/02 - "
    }, {
    "id": 246,
    "url": "http://localhost:4000/Furkan-Kurtoglu/",
    "title": "Temporospatial Modeling of CRC-CAF relation using molecular and tissue level",
    "body": "2020/08/02 - "
    }, {
    "id": 247,
    "url": "http://localhost:4000/Florian-Franke/",
    "title": "Is Cell segregation just like oil and water: A phase field approach",
    "body": "2020/08/02 - "
    }, {
    "id": 248,
    "url": "http://localhost:4000/Fillipe-H-Georgiou/",
    "title": "Modelling Locust Foraging And Its Effect On Swarm Formation",
    "body": "2020/08/02 - "
    }, {
    "id": 249,
    "url": "http://localhost:4000/Felipe-Rubio/",
    "title": "Evaluating the role of memory B and T cells during secondary dengue infection",
    "body": "2020/08/02 - "
    }, {
    "id": 250,
    "url": "http://localhost:4000/Fatima-Sulayman/",
    "title": "Dynamical systems analysis of tuberculosis with the impact of transmission rate and vaccination.",
    "body": "2020/08/02 - "
    }, {
    "id": 251,
    "url": "http://localhost:4000/Euan-Smithers/",
    "title": "How do plant leaf pavement cells form puzzle piece like shapes? Using a multi-model approach to simulate chemical and visco-elastic mechanical processes and experimental methods to discover their secrets.",
    "body": "2020/08/02 - "
    }, {
    "id": 252,
    "url": "http://localhost:4000/Estadilla-S-Carlo-Delfin/",
    "title": "Optimal Strategies for Mitigating the COVID-19 Epidemic in the Philippines",
    "body": "2020/08/02 - "
    }, {
    "id": 253,
    "url": "http://localhost:4000/Erika-Tsingos/",
    "title": "A computational tool to optimise experiments for estimating cell cycle parameters",
    "body": "2020/08/02 - "
    }, {
    "id": 254,
    "url": "http://localhost:4000/Erik-Amezquita/",
    "title": "Quantifying barley morphology using Euler characteristic curves",
    "body": "2020/08/02 - "
    }, {
    "id": 255,
    "url": "http://localhost:4000/Erica-M.-Rutter/",
    "title": "Predicting Bladder Pressure and Contractions with Dense Time-Series Data",
    "body": "2020/08/02 - "
    }, {
    "id": 256,
    "url": "http://localhost:4000/Emma-Southall/",
    "title": "Identifying indicators of critical transitions in epidemiological data",
    "body": "2020/08/02 - "
    }, {
    "id": 257,
    "url": "http://localhost:4000/Emma-L-Fairbanks/",
    "title": "Re-parameterisation of a mathematical model of AHSV using data from literature",
    "body": "2020/08/02 - "
    }, {
    "id": 258,
    "url": "http://localhost:4000/Emily-Zhang/",
    "title": "Personalized Time Series Forecasting of Blood Glucose Levels",
    "body": "2020/08/02 - "
    }, {
    "id": 259,
    "url": "http://localhost:4000/Emilia-Kozlowska/",
    "title": "Mathematical modeling of palliative treatment in non-small cell lung cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 260,
    "url": "http://localhost:4000/Eman-Alwani/",
    "title": "Mathematical Analysis of Feedback Requirements for Planar Polarisation in the Fly Wing",
    "body": "2020/08/02 - "
    }, {
    "id": 261,
    "url": "http://localhost:4000/Elizabeth-A-Fedak/",
    "title": "Getting mixed messages: How p53 controls its dynamics to interpret variable upstream signals",
    "body": "2020/08/02 - "
    }, {
    "id": 262,
    "url": "http://localhost:4000/Editha-Jose/",
    "title": "Complex Balanced Equilibria of Weakly Reversible Poly-PL Kinetic Systems and Evolutionary Games",
    "body": "2020/08/02 - "
    }, {
    "id": 263,
    "url": "http://localhost:4000/Donggu-Lee/",
    "title": "Role of OCT1 in regulation of miR-451-LKB1-AMPK-OCT1-mTOR core signaling network and cell invasion in glioblastoma.",
    "body": "2020/08/02 - "
    }, {
    "id": 264,
    "url": "http://localhost:4000/Dominic-Olver/",
    "title": "Time Dependent Osmotic Damage in Sea Urchin Oocytes",
    "body": "2020/08/02 - "
    }, {
    "id": 265,
    "url": "http://localhost:4000/Domenic-PJ-Germano/",
    "title": "Towards a realistic 3D deformable model of tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 266,
    "url": "http://localhost:4000/Divyoj-Singh/",
    "title": "Emergent Properties of the HNF4α-PPARγ Network May Drive Consequent Phenotypic Plasticity in NAFLD",
    "body": "2020/08/02 - "
    }, {
    "id": 267,
    "url": "http://localhost:4000/Diego-Samuel-Rodrigues/",
    "title": "Mathematical modeling of pharmacokinetic profiles of magnetic nanoparticles acquired by multichannel alternating current biosusceptrometry",
    "body": "2020/08/02 - "
    }, {
    "id": 268,
    "url": "http://localhost:4000/Dhiraj-Kumar-Das/",
    "title": "Influence of the smear-microscopy in global dynamics of tuberculosis transmission",
    "body": "2020/08/02 - "
    }, {
    "id": 269,
    "url": "http://localhost:4000/Dhananjay-Bhaskar/",
    "title": "Quantifying cellular (re)-organization in 3-D cancer models using persistent homology",
    "body": "2020/08/02 - "
    }, {
    "id": 270,
    "url": "http://localhost:4000/Derek-Park/",
    "title": "Synergizing chemotherapy with immune and evolutionary tradeoffs: Searching for Goldilocks",
    "body": "2020/08/02 - "
    }, {
    "id": 271,
    "url": "http://localhost:4000/Denis-H%C3%BCnniger/",
    "title": "Effects of tumor-originating niches on intra-tumor heterogeneity",
    "body": "2020/08/02 - "
    }, {
    "id": 272,
    "url": "http://localhost:4000/Deena-Schmidt/",
    "title": "Building age-structured network models from interaction data",
    "body": "2020/08/02 - "
    }, {
    "id": 273,
    "url": "http://localhost:4000/Debasmita-Mukherjee/",
    "title": "Atherosclerosis: A Mathematical Overview",
    "body": "2020/08/02 - "
    }, {
    "id": 274,
    "url": "http://localhost:4000/David-Gurarie/",
    "title": "Individual-based modeling of Covid-19 in local community settings",
    "body": "2020/08/02 - "
    }, {
    "id": 275,
    "url": "http://localhost:4000/David-Cheek/",
    "title": "Genetic composition of an exponentially growing cell population",
    "body": "2020/08/02 - "
    }, {
    "id": 276,
    "url": "http://localhost:4000/Darwin-Bandoy/",
    "title": "Pandemic dynamics of COVID-19 using epidemic stage, instantaneous reproductive number and pathogen genome identity (GENI) score: modeling molecular epidemiology",
    "body": "2020/08/02 - "
    }, {
    "id": 277,
    "url": "http://localhost:4000/Daniel-Tudor/",
    "title": "Probing immune cell wound recruitment signals using Bayesian inference and random walk models",
    "body": "2020/08/02 - "
    }, {
    "id": 278,
    "url": "http://localhost:4000/Daniel-Koch/",
    "title": "In signal transduction, all you need are oligomers!  Dynamic signal encoding, homeostasis, bistability and more",
    "body": "2020/08/02 - "
    }, {
    "id": 279,
    "url": "http://localhost:4000/Daniel-J-Glazar/",
    "title": "Tumor growth and inhibition model predicts progression in recurrent high-grade glioma",
    "body": "2020/08/02 - "
    }, {
    "id": 280,
    "url": "http://localhost:4000/Daniel-Abler/",
    "title": "Tumor Growth and Biomechanics – Challenges & Opportunities",
    "body": "2020/08/02 - "
    }, {
    "id": 281,
    "url": "http://localhost:4000/DaeWook-Kim/",
    "title": "Systems pharmacology model reveals the sources of the inter- and intraspecies variability in drug efficacy",
    "body": "2020/08/02 - "
    }, {
    "id": 282,
    "url": "http://localhost:4000/Cristian-C.-E.-Espitia/",
    "title": "A mathematical model of HIV/AIDS Spread in human population, the triangle transmission case.",
    "body": "2020/08/02 - "
    }, {
    "id": 283,
    "url": "http://localhost:4000/Cristeta-Jamilla/",
    "title": "Inverse Problems for Neutral Delay Differential Equations using Nature-Inspired Optimization Algorithms",
    "body": "2020/08/02 - "
    }, {
    "id": 284,
    "url": "http://localhost:4000/Cody-FitzGerald/",
    "title": "Red light and the dormancy-germination decision in Arabidopsis seeds",
    "body": "2020/08/02 - "
    }, {
    "id": 285,
    "url": "http://localhost:4000/Chiara-Villa/",
    "title": "Mechanical pattern formation in biological tissue: Relax and go with the (viscous) flow",
    "body": "2020/08/02 - "
    }, {
    "id": 286,
    "url": "http://localhost:4000/Chay-Paterson/",
    "title": "Cancer incidence as a form of convergent evolution",
    "body": "2020/08/02 - "
    }, {
    "id": 287,
    "url": "http://localhost:4000/Chandler-Gatenbee/",
    "title": "Immune escape at the onset of human colorectal cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 288,
    "url": "http://localhost:4000/Caroline-Franco/",
    "title": "Modelling non-pharmaceutical interventions to mitigate COVID-19 in Sao Paulo",
    "body": "2020/08/02 - "
    }, {
    "id": 289,
    "url": "http://localhost:4000/Carlos-Enrique-Bustamante-Orellana/",
    "title": "Modeling and Preparedness: The Transmission Dynamics of COVID19 Outbreak in Provinces of Ecuador",
    "body": "2020/08/02 - "
    }, {
    "id": 290,
    "url": "http://localhost:4000/Carley-Cook/",
    "title": "Mathematical modeling of the relationship between T cell produced Wnt10b and the bone remodeling cycle",
    "body": "2020/08/02 - "
    }, {
    "id": 291,
    "url": "http://localhost:4000/Cara-Sulyok/",
    "title": "A Mathematical Framework to Augment the Q-MARSH Score in the Diagnosis of Celiac Disease",
    "body": "2020/08/02 - "
    }, {
    "id": 292,
    "url": "http://localhost:4000/Camile-FD-Kunz/",
    "title": "Chemotaxis impact on pattern formation",
    "body": "2020/08/02 - "
    }, {
    "id": 293,
    "url": "http://localhost:4000/Cameline-Nafula/",
    "title": "Modelling the spatio-temporal risk of measles outbreaks and options for their control in Kenya",
    "body": "2020/08/02 - "
    }, {
    "id": 294,
    "url": "http://localhost:4000/Calmelet-Jeanne-Colette/",
    "title": "Surfactant Driven Flows in Respiratory and Ocular Regions",
    "body": "2020/08/02 - "
    }, {
    "id": 295,
    "url": "http://localhost:4000/Bin-Zhang/",
    "title": "The Ecology of Collective Cancer Invasion: An Evolutionary Game Theory Model",
    "body": "2020/08/02 - "
    }, {
    "id": 296,
    "url": "http://localhost:4000/Bhattacharyya-Samit/",
    "title": "Evolutionary game models to explain the potential interactions among Adverse events, Vaccine scares and Individual vaccination choice",
    "body": "2020/08/02 - "
    }, {
    "id": 297,
    "url": "http://localhost:4000/Beryl-O-Musundi/",
    "title": "An immuno-epidemiological model linking within-host and between-host dynamics of cholera",
    "body": "2020/08/02 - "
    }, {
    "id": 298,
    "url": "http://localhost:4000/Bertin-Hoffman/",
    "title": "The initial engraftment of tumor cells is critical for the future growth pattern",
    "body": "2020/08/02 - "
    }, {
    "id": 299,
    "url": "http://localhost:4000/Benedikt-Sabass/",
    "title": "Molecular mechanics of type-IV pili driven migration of P. aeruginosa",
    "body": "2020/08/02 - "
    }, {
    "id": 300,
    "url": "http://localhost:4000/Baylor-Fain/",
    "title": "Modeling the impact of inoculum dose and transmission mode on viral infection with an agent-based model",
    "body": "2020/08/02 - "
    }, {
    "id": 301,
    "url": "http://localhost:4000/Baeckkyoung-Sung/",
    "title": "Mathematical modeling of a temperature-sensitive and tissue-mimicking gel matrix",
    "body": "2020/08/02 - "
    }, {
    "id": 302,
    "url": "http://localhost:4000/Augustine-Okebunor-Okolie/",
    "title": "Exact and approximate formulas for contact tracing on random trees",
    "body": "2020/08/02 - "
    }, {
    "id": 303,
    "url": "http://localhost:4000/Atchuta-Srinivas-Duddu/",
    "title": "Multistability in cellular differentiation enabled by a mutually antagonistic triad",
    "body": "2020/08/02 - "
    }, {
    "id": 304,
    "url": "http://localhost:4000/Atanaska-Dobreva/",
    "title": "Mathematical modeling of photoreceptor metabolism",
    "body": "2020/08/02 - "
    }, {
    "id": 305,
    "url": "http://localhost:4000/Artur-C%C3%A9sar-Fassoni/",
    "title": "Mathematical modeling and methodology to identify patient-specific immunological landscapes in CML treatment using TKI cessation and dose reduction data",
    "body": "2020/08/02 - "
    }, {
    "id": 306,
    "url": "http://localhost:4000/Anuraag-Bukkuri/",
    "title": "GLUT1 Production in Cancer Cells: A Tragedy of the Commons",
    "body": "2020/08/02 - "
    }, {
    "id": 307,
    "url": "http://localhost:4000/Aniruddha-Deka/",
    "title": "Optimal public health strategy during an influenza outbreak",
    "body": "2020/08/02 - "
    }, {
    "id": 308,
    "url": "http://localhost:4000/Angelo-J-Zorn/",
    "title": "An Epidemic Mobility Model with Symptomatic and Asymptomatic Individuals Allowing Variation of Contact Rates Between Individuals and Across Regions",
    "body": "2020/08/02 - "
    }, {
    "id": 309,
    "url": "http://localhost:4000/Angela-Michelle-Jarrett/",
    "title": "Modeling of the spatio-temporal evolution of tumor vasculature to improve predictions of breast cancer response to neoadjuvant chemotherapy regimens",
    "body": "2020/08/02 - "
    }, {
    "id": 310,
    "url": "http://localhost:4000/Angela-L-Moreno/",
    "title": "ART Neural Networks in the Classification of Spinal Pathologies",
    "body": "2020/08/02 - "
    }, {
    "id": 311,
    "url": "http://localhost:4000/Andreas-Buttenschoen/",
    "title": "Spatio-Temporal Heterogeneities in a Mechano-Chemical Model of Collective Cell Migration",
    "body": "2020/08/02 - "
    }, {
    "id": 312,
    "url": "http://localhost:4000/Anca-Radulescu/",
    "title": "Effects of local mutations in quadratic iterations",
    "body": "2020/08/02 - "
    }, {
    "id": 313,
    "url": "http://localhost:4000/Ananta-Samrajya-Shri-Kishore-Hari/",
    "title": "Identifying inhibitors of epithelial–mesenchymal plasticity using a network topology-based approach",
    "body": "2020/08/02 - "
    }, {
    "id": 314,
    "url": "http://localhost:4000/Amanda-Alexander/",
    "title": "Mathematical modeling of regulatory T cell mechanisms in experimental autoimmune encephalomyelitis",
    "body": "2020/08/02 - "
    }, {
    "id": 315,
    "url": "http://localhost:4000/Alexis-Erich-S-Almocera/",
    "title": "Modeling the viral infectious disease from infection to epidemic",
    "body": "2020/08/02 - "
    }, {
    "id": 316,
    "url": "http://localhost:4000/Alexey-A.-Tokarev/",
    "title": "Velocity-Amplitude relationship in the Gray-Scott autowave model in isolated conditions",
    "body": "2020/08/02 - "
    }, {
    "id": 317,
    "url": "http://localhost:4000/Alexander-P-Hoover/",
    "title": "Exploring Neuromechanical Resonance in Jellyfish Locomotion",
    "body": "2020/08/02 - "
    }, {
    "id": 318,
    "url": "http://localhost:4000/Akhil-Kumar-Srivastav/",
    "title": "Mathematical Modeling of Malaria with Saturated Treatment-A Case Study of India",
    "body": "2020/08/02 - "
    }, {
    "id": 319,
    "url": "http://localhost:4000/Adrianne-Jenner/",
    "title": "Exploring the impact of intratumoral heterogeneity on oncolytic virotherapy using agent-based modelling",
    "body": "2020/08/02 - "
    }, {
    "id": 320,
    "url": "http://localhost:4000/Adriana-Zanca/",
    "title": "Multicellular model of collective cell migration with an irregular free boundary",
    "body": "2020/08/02 - "
    }, {
    "id": 321,
    "url": "http://localhost:4000/Adnan-A-Khan/",
    "title": "Understanding the COVID-19 Outbreak in Pakistan",
    "body": "2020/08/02 - "
    }, {
    "id": 322,
    "url": "http://localhost:4000/Abdel-H.-Halloway/",
    "title": "Non-Equilibrial Dynamics in Under-Saturated Communities",
    "body": "2020/08/02 - "
    }, {
    "id": 323,
    "url": "http://localhost:4000/POPD_Wednesday_MS2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 324,
    "url": "http://localhost:4000/POPD_Wednesday_MS1/",
    "title": "Emergence and Stability of Population Structure and Biological Aggregates Across Scales",
    "body": "2020/08/01 - "
    }, {
    "id": 325,
    "url": "http://localhost:4000/POPD_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 326,
    "url": "http://localhost:4000/POPD_Tuesday_MS2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 327,
    "url": "http://localhost:4000/POPD_Tuesday_MS1/",
    "title": "Mathematical models of evolutionary rescue",
    "body": "2020/08/01 - "
    }, {
    "id": 328,
    "url": "http://localhost:4000/POPD_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 329,
    "url": "http://localhost:4000/POPD_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 330,
    "url": "http://localhost:4000/POPD_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 331,
    "url": "http://localhost:4000/POPD_Monday_MS2/",
    "title": "",
    "body": "2020/08/01 - "
    }, {
    "id": 332,
    "url": "http://localhost:4000/POPD_Monday_MS1/",
    "title": "Stochastic Population Dynamics",
    "body": "2020/08/01 - "
    }, {
    "id": 333,
    "url": "http://localhost:4000/POPD_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 334,
    "url": "http://localhost:4000/OTHE_Wednesday_MS2/",
    "title": "Flow and Transport in Bacterial Biofilms, Part II",
    "body": "2020/08/01 - "
    }, {
    "id": 335,
    "url": "http://localhost:4000/OTHE_Wednesday_MS1/",
    "title": "Flow and Transport in Complex Tissues, Part I",
    "body": "2020/08/01 - "
    }, {
    "id": 336,
    "url": "http://localhost:4000/OTHE_Wednesday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 337,
    "url": "http://localhost:4000/OTHE_Tuesday_MS1/",
    "title": "Eco-evolutionary dynamics across scales of organisation",
    "body": "2020/08/01 - "
    }, {
    "id": 338,
    "url": "http://localhost:4000/OTHE_Tuesday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 339,
    "url": "http://localhost:4000/OTHE_Thursday_CT2/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 340,
    "url": "http://localhost:4000/OTHE_Thursday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 341,
    "url": "http://localhost:4000/OTHE_Monday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 342,
    "url": "http://localhost:4000/ONCO_Wednesday_MS2/",
    "title": "Applications and challenges of using quantitative imaging data for biologically-based mathematical oncology",
    "body": "2020/08/01 - "
    }, {
    "id": 343,
    "url": "http://localhost:4000/ONCO_Wednesday_MS1/",
    "title": "Differentiation and stemmness, in cell migration, cancer invasion, and development",
    "body": "2020/08/01 - "
    }, {
    "id": 344,
    "url": "http://localhost:4000/ONCO_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 345,
    "url": "http://localhost:4000/ONCO_Tuesday_MS2/",
    "title": "Data-based modeling in cancer research with focus on clinical applications",
    "body": "2020/08/01 - "
    }, {
    "id": 346,
    "url": "http://localhost:4000/ONCO_Tuesday_MS1/",
    "title": "Multiscale models of cancer heterogeneity, with applications in drug development and precision medicine",
    "body": "2020/08/01 - "
    }, {
    "id": 347,
    "url": "http://localhost:4000/ONCO_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 348,
    "url": "http://localhost:4000/ONCO_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 349,
    "url": "http://localhost:4000/ONCO_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 350,
    "url": "http://localhost:4000/ONCO_Monday_MS2/",
    "title": "Frontiers in MathOnco, Part 2",
    "body": "2020/08/01 - "
    }, {
    "id": 351,
    "url": "http://localhost:4000/ONCO_Monday_MS1/",
    "title": "Frontiers in MathOnco, Part 1",
    "body": "2020/08/01 - "
    }, {
    "id": 352,
    "url": "http://localhost:4000/ONCO_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 353,
    "url": "http://localhost:4000/NEUR_Wednesday_MS2/",
    "title": "Stochastic and Mean-Field Analysis in Neuroscience",
    "body": "2020/08/01 - "
    }, {
    "id": 354,
    "url": "http://localhost:4000/NEUR_Wednesday_MS1/",
    "title": "Dynamics and Noise in Neural Networks",
    "body": "2020/08/01 - "
    }, {
    "id": 355,
    "url": "http://localhost:4000/NEUR_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 356,
    "url": "http://localhost:4000/NEUR_Thursday_CT1/",
    "title": "Rhythms, Sleep, Reward, Vision",
    "body": "2020/08/01 - "
    }, {
    "id": 357,
    "url": "http://localhost:4000/MFBM_Wednesday_MS2/",
    "title": "Stochastic methods for epidemiology and biochemical reaction networks",
    "body": "2020/08/01 - "
    }, {
    "id": 358,
    "url": "http://localhost:4000/MFBM_Wednesday_MS1/",
    "title": "Stochastic methods for epidemiology and biochemical reaction networks",
    "body": "2020/08/01 - "
    }, {
    "id": 359,
    "url": "http://localhost:4000/MFBM_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 360,
    "url": "http://localhost:4000/MFBM_Tuesday_MS2/",
    "title": "Topological and network analyses for data",
    "body": "2020/08/01 - "
    }, {
    "id": 361,
    "url": "http://localhost:4000/MFBM_Tuesday_MS1/",
    "title": "Topological and network analyses for data",
    "body": "2020/08/01 - "
    }, {
    "id": 362,
    "url": "http://localhost:4000/MFBM_Tuesday_CT1/",
    "title": "Subgroup contributed talks",
    "body": "2020/08/01 - "
    }, {
    "id": 363,
    "url": "http://localhost:4000/MFBM_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 364,
    "url": "http://localhost:4000/MFBM_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 365,
    "url": "http://localhost:4000/MFBM_Monday_MS2/",
    "title": "Simulations and Experiments in Small Scale Bio-Locomotion",
    "body": "2020/08/01 - "
    }, {
    "id": 366,
    "url": "http://localhost:4000/MFBM_Monday_MS1/",
    "title": "Simulations and Experiments in Small Scale Bio-Locomotion",
    "body": "2020/08/01 - "
    }, {
    "id": 367,
    "url": "http://localhost:4000/MFBM_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 368,
    "url": "http://localhost:4000/MEPI_Wednesday_MS2/",
    "title": "COVID-19 Contributed Talks III",
    "body": "2020/08/01 - "
    }, {
    "id": 369,
    "url": "http://localhost:4000/MEPI_Wednesday_MS1/",
    "title": "Models of Infectious Diseases and Control",
    "body": "2020/08/01 - "
    }, {
    "id": 370,
    "url": "http://localhost:4000/MEPI_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 371,
    "url": "http://localhost:4000/MEPI_Tuesday_MS2/",
    "title": "COVID-19 Contributed Talks II",
    "body": "2020/08/01 - "
    }, {
    "id": 372,
    "url": "http://localhost:4000/MEPI_Tuesday_MS1/",
    "title": "Modeling COVID-19 to inform control efforts, Part II",
    "body": "2020/08/01 - "
    }, {
    "id": 373,
    "url": "http://localhost:4000/MEPI_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 374,
    "url": "http://localhost:4000/MEPI_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 375,
    "url": "http://localhost:4000/MEPI_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 376,
    "url": "http://localhost:4000/MEPI_Monday_MS2/",
    "title": "COVID-19 Contributed Talks I",
    "body": "2020/08/01 - "
    }, {
    "id": 377,
    "url": "http://localhost:4000/MEPI_Monday_MS1/",
    "title": "Modeling COVID-19 to inform control efforts, Part I",
    "body": "2020/08/01 - "
    }, {
    "id": 378,
    "url": "http://localhost:4000/MEPI_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 379,
    "url": "http://localhost:4000/IMMU_Wednesday_MS2/",
    "title": "Increasingly biologically accurate models of influenza A virus infection spread in vivo & in vitro",
    "body": "2020/08/01 - "
    }, {
    "id": 380,
    "url": "http://localhost:4000/IMMU_Wednesday_MS1/",
    "title": "Immunobiology and Infection Minisymposium",
    "body": "2020/08/01 - "
    }, {
    "id": 381,
    "url": "http://localhost:4000/IMMU_Tuesday_MS2/",
    "title": "Immunobiology and Infection Minisymposium",
    "body": "2020/08/01 - "
    }, {
    "id": 382,
    "url": "http://localhost:4000/IMMU_Tuesday_MS1/",
    "title": "Immunobiology and Infection Minisymposium",
    "body": "2020/08/01 - "
    }, {
    "id": 383,
    "url": "http://localhost:4000/IMMU_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 384,
    "url": "http://localhost:4000/IMMU_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 385,
    "url": "http://localhost:4000/IMMU_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 386,
    "url": "http://localhost:4000/EDUC_Wednesday_MS2/",
    "title": "Education Minisymposium II",
    "body": "2020/08/01 - "
    }, {
    "id": 387,
    "url": "http://localhost:4000/EDUC_Wednesday_MS1/",
    "title": "Education Minisymposium II",
    "body": "2020/08/01 - "
    }, {
    "id": 388,
    "url": "http://localhost:4000/CDEV_Wednesday_MS2/",
    "title": "Understanding development through theory and experiment",
    "body": "2020/08/01 - "
    }, {
    "id": 389,
    "url": "http://localhost:4000/CDEV_Wednesday_MS1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 390,
    "url": "http://localhost:4000/CDEV_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 391,
    "url": "http://localhost:4000/CDEV_Tuesday_MS2/",
    "title": "Shapes, patterns, and forces in development biology",
    "body": "2020/08/01 - "
    }, {
    "id": 392,
    "url": "http://localhost:4000/CDEV_Tuesday_MS1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 393,
    "url": "http://localhost:4000/CDEV_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 394,
    "url": "http://localhost:4000/CDEV_Thursday_CT2/",
    "title": "Integrating cell mechanics and cell mechanosensing",
    "body": "2020/08/01 - "
    }, {
    "id": 395,
    "url": "http://localhost:4000/CDEV_Thursday_CT1/",
    "title": "The interplay of intracellular pattern formation, geometry, polarisation and mechanics",
    "body": "2020/08/01 - "
    }, {
    "id": 396,
    "url": "http://localhost:4000/CDEV_Monday_MS2/",
    "title": "Modeling dynamics in single-cell biology",
    "body": "2020/08/01 - "
    }, {
    "id": 397,
    "url": "http://localhost:4000/CDEV_Monday_MS1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 398,
    "url": "http://localhost:4000/CDEV_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 399,
    "url": "http://localhost:4000/conference-announcement/",
    "title": "Conference Announcement",
    "body": "2020/06/08 - Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. We will probably remember these days, in years to come, as a true opportunity to contribute to the global fight against the pandemic. Predictive mathematical models that capture the underlying mechanistic biology of specific processes are at the heart of everything we do. The COVID-19 virus pandemic has brought into sharp focus the critical importance of mathematical models in both making predictions about current trends in the disease dynamics but also how specific interventions, such as social distancing, might modify them. Understanding these interventions theoretically becomes important not only for mathematical biologists, but also for politicians and other decision makers. Most of the our society members are currently doing some form of social distancing, as is a large proportion of the world. In these confined and difficult times, anxiety and stress are running high. But it is important to remember that we have the power of mathematical modelling behind us. Their predictive ability illustrates why we must slowdown the spread of this virus by staying home. Flattening the curve has become our new mantra. Importantly mathematical modelling will also help us decide how to best end this quarantine. We are at a key point in the history of our society where our models are helping to inform policy decisions that impact millions of lives. Therefore it is fundamental that we make every effort to articulate as clearly as possible what mathematical models are, what they can be used for, and what are their limitations. The integration of mathematics and biology may sound strange at first to the public. But if you find yourself in a situation where you need to explain why certain decisions are being made, please take the time to do so. Articulate the role of predictive mathematical models, but not as something magical, even though they might seem so at times. Rather that they are based on our understanding of the physical and biological processes that underly the spread of this disease. We are a diverse group of scientists and educators who work at the interface of many disciplines not just epidemiology. Our society does have a dedicated subgroup in Mathematical Epidemiology and I encourage you join them if you are interested in this area of research or if you want to learn more. We also have 7 other subgroups covering everything from development to oncology and more - I’m extremely proud of our research diversity. The purpose of this email was to shine a little light and say thank you, to celebrate the wonderful discipline we are part of and highlight the important role mathematical models are playing right now. We are fortunate in many ways, with our ability to more readily work from home than most disciplines. We can make predictions about disease dynamics without ever actually treating a patient or being forced to make difficult decisions in medical care. We can run thousands of realisations of an experiment in a fraction of the time it would take in real life. We can make rigorously constrained predictions about the spread of an epidemic and how to slow it down. Ultimately the countermeasures needed to control and end this pandemic will rely on biomedical research on drugs and vaccines. But again, their impact can and should be studied through the lens of mathematical modelling. I would be remiss if I didn’t give a special thanks to our colleagues, friends and family members in the medical profession. Without their frontline action flattening the curve becomes meaningless. Stay safe, stay home and celebrate mathematical biology! -Sandy Anderson, SMB President "
    }];

var idx = lunr(function () {
    this.ref('id')
    this.field('title')
    this.field('body')
    

    documents.forEach(function (doc) {
        this.add(doc)
    }, this)
});


    
function lunr_search(term) {
    $('#lunrsearchresults').show( 1000 );
    $( "body" ).addClass( "modal-open" );
    
    document.getElementById('lunrsearchresults').innerHTML = '<div id="resultsmodal" class="modal fade show d-block"  tabindex="-1" role="dialog" aria-labelledby="resultsmodal"> <div class="modal-dialog shadow-lg" role="document"> <div class="modal-content"> <div class="modal-header" id="modtit"> <button type="button" class="close" id="btnx" data-dismiss="modal" aria-label="Close"> &times; </button> </div> <div class="modal-body"> <ul class="mb-0"> </ul>    </div> <div class="modal-footer"><button id="btnx" type="button" class="btn btn-secondary btn-sm" data-dismiss="modal">Close</button></div></div> </div></div>';
    if(term) {
        document.getElementById('modtit').innerHTML = "<h5 class='modal-title'>Search results for '" + term + "'</h5>" + document.getElementById('modtit').innerHTML;
        //put results on the screen.
        var results = idx.search(term);
        if(results.length>0){
            //console.log(idx.search(term));
            //if results
            for (var i = 0; i < results.length; i++) {
                // more statements
                var ref = results[i]['ref'];
                var url = documents[ref]['url'];
                var title = documents[ref]['title'];
                var body = documents[ref]['body'].substring(0,160)+'...';
                document.querySelectorAll('#lunrsearchresults ul')[0].innerHTML = document.querySelectorAll('#lunrsearchresults ul')[0].innerHTML + "<li class='lunrsearchresult'><a href='" + url + "'><span class='title'>" + title + "</span><br /><small><span class='body'>"+ title +"</span><br /><span class='url'>"+ url +"</span></small></a></li>";
            }
        } else {
            document.querySelectorAll('#lunrsearchresults ul')[0].innerHTML = "<li class='lunrsearchresult'>Sorry, no results found. Close & try a different search!</li>";
        }
    }
    return false;
}
</script>
<style>
    .lunrsearchresult .title {color: #d9230f;}
    .lunrsearchresult .url {color: silver;}
    .lunrsearchresult a {display: block; color: #777;}
    .lunrsearchresult a:hover, .lunrsearchresult a:focus {text-decoration: none;}
    .lunrsearchresult a:hover .title {text-decoration: underline;}
</style>




<form class="bd-search hidden-sm-down" onSubmit="return lunr_search(document.getElementById('lunrsearch').value);">
<input type="text" class="form-control text-small"  id="lunrsearch" name="q" value="" placeholder="Type keyword and enter..."> 
</form>
            </ul>
-->
        </div>
    </div>
    </nav>

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content">
        <div class="container">
	
	<div class="col-md-12 main-loop">
	    <div class="row">
	        <div class="col-md-12  text-center">
		        <h1 class="mb-3">SMB 2020 Annual Meeting</h1>
		        <h2 class="mb-3">August 17 - 20, 2020</h2>
		        <div class="row">
			        <div class="col-md-4 offset-md-4">
								<a href="https://www.smb.org/"><img class="w-100" src="/assets/images/smb-motion-logo.gif" alt="SMB Logo"></a>        
			        </div>
		        </div>
		        <div class="row">
			        <div class="col-12 text-center">
			        	</br>
                <p class="mb-3 lead">The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020.</p>	 
                <p class="mb-3 lead">Are you a member of SMB? If not please support mathematical biology and your career by <a href="https://www.smb.org/application/" style="color: #3076ed;">joining the society</a> today!</p>	    	        
			        </div>
			      </div>
	        </div>
	    </div>
	</div>
	
    
 <!--endif page url is / -->

    <div class="row mt-3 d-md-none">  
	    <div class="col-md-12">
		    <div class="row">
	        <div class="col-12">
		        
							<a href="/logistics"><img class="w-100" src="../assets/images/sococo-classroom.png" alt="SMB Logo" style="margin-top: -50px;"></a>
						
		        
	        </div>
		    </div>
			</div>
		</div>
  
    
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			<h4 class="font-weight-bold spanborder"><span>Conference Information</span></h4>
	    <div class="row main-font">
        <div class="col-md-12">				        
					<p>The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between  August 17-20, 2020. The meeting will be held between 8.30 am and 5.00 pm US Eastern Time Zone to maximize the participation of people across the world. </p>
					<p>We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups.</p>		      		        
        
        <p>While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.</p>
        </div>
	    </div>
			</br>
			
			
			        
	    <h4 class="font-weight-bold spanborder"><span>Conference Schedule</span></h4>
	    <div class="row main-font">
        <div class="col-md-12">    
	        <p>* all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change.</p>
	        
        </br>
        </br>
	        

<div class="col-md-12 text-center rightborder pl-0">
     
     <strong>
     	Click to filter by subgroup: </br>
	     <span class="click-legend" onclick="myFunction('all')">All</span>
	     <span class="click-legend uMEPI" onclick="myFunction('text-MEPI')">MEPI</span> 
	     <span class="click-legend uCDEV" onclick="myFunction('text-CDEV')">CDEV</span>
	     <span class="click-legend uEDUC" onclick="myFunction('text-EDUC')">EDUC</span>
	     <span class="click-legend uIMMU" onclick="myFunction('text-IMMU')">IMMU</span>
	     <span class="click-legend uNEUR" onclick="myFunction('text-NEUR')">NEUR</span>
	     <span class="click-legend uMFBM" onclick="myFunction('text-MFBM')">MFBM</span>
	     <span class="click-legend uPOPD" onclick="myFunction('text-POPD')">POPD</span>                 
	     <span class="click-legend uONCO" onclick="myFunction('text-ONCO')">ONCO</span> 
	     <span class="click-legend uOTHE" onclick="myFunction('text-OTHE')">OTHE</span>   
     </strong>               
</div>
</br>
</br>

<div class="table-responsive text-left" id="full-table">
  <table class="table table-sm table-bordered table-hover text-left">
  <thead>
    <tr>
      <th scope="col">Time (EDT)</th>
      <th scope="col"><a href="/monday">Monday</a></th>
      <th scope="col"><a href="/tuesday">Tuesday</a></th>
      <th scope="col"><a href="/wednesday">Wednesday</a></th>
      <th scope="col"><a href="/thursday">Thursday</a></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">8:00am</th>
      <td><a href="/monday#welcome">Opening & Welcome</a></td>
      <td colspan="3">Coffee with friends and colleagues</td>
    </tr>
    <tr>
      <th scope="row">8:30am</th>
      <td class="concurrent"><a href="/monday#opening"><strong>Opening Plenary</strong></br>Shayn Peirce-Cottler</a></td>
      <td class="MEPI"><a href="/tuesday#opening"><strong>Epidemiology Keynote</strong></br>Deirdre Hollingsworth</td>
      <td class="EDUC"><a href="/wednesday#opening"><strong>Education Keynote</strong></br>Alicia Prieto Langrica</td>
      <td class="NEUR"><a href="/thursday#opening"><strong>Neuroscience Keynote</strong></br>Sue Ann Campbell</td>
    </tr>
    <tr>
      <th scope="row">9:30am</th>
      
      <td>
	      <a href="/monday#MS1">
		      <strong>Minisymposium, </br> Contributed Session</br>
		      <span class="text-ONCO">ONCO</span>
		      <span class="text-MEPI">MEPI</span>
		      <span class="text-POPD">POPD</span>
		      <span class="text-CDEV">CDEV</span>
		      <span class="text-MFBM">MFBM</span>
		      </strong>
	      </a>
	    </td>
	    
	    <td>
		    <a href="/tuesday#MS1">
			    <strong>Minisymposium, </br> Contributed Session</br>
		      <span class="text-ONCO">ONCO</span>
		      <span class="text-MEPI">MEPI</span>
		      <span class="text-POPD">POPD</span>
		      <span class="text-CDEV">CDEV</span>
		      <span class="text-MFBM">MFBM</span>
		      <span class="text-OTHE">OTHE</span>
		      <span class="text-IMMU">IMMU</span>
		      </strong>
	      </a>
	    </td>
	    
	    <td>
		    <a href="/wednesday#MS1">
			    <strong>Minisymposium, </br> Contributed Session</br>
		      <span class="text-ONCO">ONCO</span>
		      <span class="text-MEPI">MEPI</span>
		      <span class="text-POPD">POPD</span>
		      <span class="text-EDUC">EDUC</span>
		      <span class="text-NEUR">NEUR</span>
		      <span class="text-CDEV">CDEV</span>
		      <span class="text-MFBM">MFBM</span>
		      <span class="text-OTHE">OTHE</span>
		      <span class="text-IMMU">IMMU</span>
		      </strong>
	      </a>
	    </td>
	    
	    <td>
		    <a href="/thursday#MS1">
			    <strong>Minisymposium, </br> Contributed Session</br>
		      <span class="text-ONCO">ONCO</span>
		      <span class="text-MEPI">MEPI</span>
		      <span class="text-POPD">POPD</span>
		      <span class="text-NEUR">NEUR</span>
		      <span class="text-CDEV">CDEV</span>
		      <span class="text-MFBM">MFBM</span>
		      <span class="text-OTHE">OTHE</span>
		      <span class="text-IMMU">IMMU</span>
		      </strong>
	      </a>
	    </td>
	    
	    
    </tr>
    <tr>
      <th scope="row">11:00am</th>
      <td colspan="4"><strong>15 minute break</strong></td>
    </tr>
    <tr>
      <th scope="row" rowspan="2">11:15am</th>
      <td rowspan="2">
	      <a href="/monday#MS2">
	      <strong>Minisymposium, </br> Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      </strong>
	      </a>
	    </td>
	    
	    <td rowspan="2">
		    <a href="/tuesday#MS2">
		    <strong>Minisymposium, </br> Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      <span class="text-IMMU">IMMU</span>
	      </strong>
	      </a>
	    </td>
	    
	    <td rowspan="2">
		    <a href="/wednesday#MS2">
		    <strong>Minisymposium, </br> Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-EDUC">EDUC</span>
	      <span class="text-NEUR">NEUR</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      <span class="text-OTHE">OTHE</span>
	      <span class="text-IMMU">IMMU</span>
	      </strong>
	      </a>
	    </td>
	    
	    <td class="IMMU">
		    <a href="/thursday#keynote1">
			  	<strong>Immunobiology Keynote</strong></br>David Ho
			  </a>
		  </td>      
    </tr>
    
    <tr>
      	<td class="CDEV"><a href="/thursday#keynote2"><strong>Cell dev. biology Keynote</strong></br>Adam Martin</a></td>      
    </tr>
    
    <tr>
      <th scope="row">12:45pm</th>
      <td><a href="/monday#lunch"><strong>Mentoring luncheon</strong></a></td>
      <td><a href="/tuesday#lunch"><strong>Subgroup business meeting</strong></a></td>
      <td><a href="/wednesday#lunch"><strong>SMB business meeting</strong></a></td>
      <td><a href="/thursday#lunch"><strong>Career fair</strong></a></td>
    </tr>
    <tr>
      <th scope="row">1:30pm</th>
			
			<!--  Monday CT1 -->
      <td>
	      <a href="/monday#CT1">
	      <strong>Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      <span class="text-OTHE">OTHE</span>
	      </strong>
	      </a>
	    </td>
	    
	    <!--  Tuesday CT1 -->
	    <td>
		    <a href="/tuesday#CT1">
		    <strong>Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      <span class="text-IMMU">IMMU</span>
	      <span class="text-OTHE">OTHE</span>
	      </strong>
	      </a>
	    </td>
	    
	    <!--  Wednesday CT1 -->
	    <td>
		    <a href="/wednesday#CT1">
		    <strong>Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-NEUR">NEUR</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      <span class="text-OTHE">OTHE</span>
	      </strong>
	      </a>
	    </td>
	    
	    <!--  Thursday CT1 -->
	    <td rowspan="2">
		    <a href="/thursday#CT1">
		    <strong>Contributed Session</br>
	      <span class="text-ONCO">ONCO</span>
	      <span class="text-MEPI">MEPI</span>
	      <span class="text-POPD">POPD</span>
	      <span class="text-CDEV">CDEV</span>
	      <span class="text-MFBM">MFBM</span>
	      <span class="text-OTHE">OTHE</span>
	      <span class="text-IMMU">IMMU</span>
	      </strong>
	      </a>
	    </td>    
    </tr>
    
    <tr>
      <th scope="row">2:30pm</th>
      <td><a href="/monday#poster"><strong>Poster session</strong></a></td>
      <td><a href="/monday#poster"><strong>Poster session</strong></a></td>
      <td><a href="/monday#poster"><strong>Poster session</strong></a></td>
    </tr>
    
    
    
    <tr>
      <th scope="row">3:30pm</th>
      <td class="POPD"><a href="/monday#keynote"><strong>Population Dyn. Keynote</strong></br>Priyanga Amarasekare</a></td>
      <td class="ONCO"><a href="/tuesday#keynote"><strong>Oncology Keynote</strong></br>Andrea Bild</a></td>
      <td class="MFBM"><a href="/wednesday#keynote"><strong>Methods Keynote</strong></br>Robert Insall</a></td>
      <td class="concurrent"><a href="/thursday#closing"><strong>Closing Plenary</strong></br>Carl Bergstrom</a></td>
    </tr>
    <tr>
      <th scope="row">4:30pm</th>
      <td colspan="3"><strong>Happy hour with friends & colleagues</strong></td>
      <td colspan="3"><strong>Poster, Talk and Twitter awards will be announced.</strong></td>
    </tr>
        
  </tbody>
</table>
</div>



	        	
					</div>
	    </div>
	    </br>
	    </br>
	    </br>
	    
	    <h4 class="font-weight-bold spanborder"><span>Plenary Speakers</span></h4>
	    <div class="row main-font">
        <div class="col-md-12"> 
	        <ol class="list-featured">
		        <div class="row">
			        <div class="col-md-10 offset-1">
				        <div class="row">
					        <div class="col-md-6">				        
						        <li class="mb-4">
						          <span>
								          <a href="https://www.biology.washington.edu/people/profile/carl-bergstrom">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
								                  Carl Bergstrom
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">University of Washington, <a href="https://twitter.com/CT_Bergstrom">@CT_Bergstrom</a><span class="sep"></span></span>
						              </span>
													</br>
						              <img src="/assets/images/bergstrom.jpg" class="rounded-circle" alt="Carl Bergstrom" style="height: 200px;">
						          </span>
						        </li>
					        </div>
					        <div class="col-md-6">
								        <li class="mb-4">
						          <span>
								          <a href="https://www.peircecottlerlab.com/">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
								                  Shayn Peirce-Cottler
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">University of Virginia, <a href="https://twitter.com/PeirceLab">@PeirceLab</a><span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/peirce.png" class="rounded-circle" alt="Shayn Peirce-Cottler" style="height: 200px;">
						          </span>
						        </li>
					        </div>
					      </div>				        
			        </div> 
			      </div> <!-- end row of plenary speakers -->
			    </ol>
				</div>
			</div>
			
			</br>
			</br>
			</br>
			
			
			<h4 class="font-weight-bold spanborder"><span>Subgroup Keynote Speakers</span></h4>
	    <div class="row main-font">
        <div class="col-md-12"> 
				  <ol class="list-featured">
				    <!-- PLATFORM SPEAKERS --> 
				    <div class="row">
			        <div class="col-md-10 offset-1">
				        <div class="row">
					        <div class="col-md-6">
						        <li class="mb-4">
						          <span>
								          <a href="https://ysu.edu/faculty-experts/prieto-langarica-alicia">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
								                  Alicia Prieto Langarica
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">Youngstown State University, <a href="https://twitter.com/AliciaPL25">@AliciaPL25</a><span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/prieto.jpeg" class="rounded-circle" alt="Alicia Prieto Langarica" style="height: 200px;">
						          </span>
						        </li>
						      </div>
					        <div class="col-md-6">
						        <li class="mb-4">
						          <span>
								          <a href="https://en.wikipedia.org/wiki/David_Ho">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
								                  David Ho
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">Columbia University<span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/ho.jpg" class="rounded-circle" alt="David Ho" style="height: 200px;">
						          </span>
						        </li>
						        
						      </div>
					      </div>				        
			        </div> 
			      </div> <!-- end row of plenary speakers -->
				        
				    </br>
				    
				    <!-- PLATFORM SPEAKERS --> 
				    <div class="row">
			        <div class="col-md-10 offset-1">
				        <div class="row">
					        <div class="col-md-6">				        
						        <li class="mb-4">
						          <span>
								          <a href="https://www.gla.ac.uk/researchinstitutes/cancersciences/staff/robertinsall/">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
								                  Robert Insall
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">University of Galsgow, <a href="https://twitter.com/robinsall">@robinsall</a>
				<span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/insall.jpg" class="rounded-circle" alt="Robert Insall" style="height: 200px;">
						          </span>
						        </li>
						      
						      </div>
					        <div class="col-md-6">
						        
						        <li class="mb-4">
						          <span>
								          <a href="https://faculty.eeb.ucla.edu/amarasekare/index.html">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
																Priyanga Amarasekare
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">University of California Los Angeles<span class="sep"></span></span>
						              </span>
						              </br>
						              
						              <img src="/assets/images/ama.png" class="rounded-circle" alt="Priyanga Amarasekare" style="height: 200px;">
						          </span>
						        </li>
						      </div>
					      </div>				        
			        </div> 
			      </div> <!-- end row of plenary speakers -->
			      
			      </br>
				        
				    <!-- PLATFORM SPEAKERS --> 
				    <div class="row">
			        <div class="col-md-10 offset-1">
				        <div class="row">
					        <div class="col-md-6">
						        <li class="mb-4">
						          <span>
								          <a href="https://bildlab.org/">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
																Andrea Bild 
								              </h6>						              
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">City of Hope, <a href="https://twitter.com/AndreaBildLab">@AndreaBildLab</a><span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/bild.jpg" class="rounded-circle" alt="Andrea Bild" style="height: 200px;">
						          </span>
						        </li>
				        	</div>

					        <div class="col-md-6">
						        <li class="mb-4">
						          <span>
								          <a href="https://biology.mit.edu/profile/adam-c-martin/">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
																Adam Martin
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">Massachusetts Institute of Technology, <a href="https://twitter.com/martinlabmit">@martinlabmit</a><span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/martin.jpg" class="rounded-circle" alt="Adam Martin" style="height: 200px;">
						          </span>
						        </li>
						        
						      </div>
					      </div>				        
			        </div> 
			      </div> <!-- end row of plenary speakers -->
			      
			      
			      <!-- PLATFORM SPEAKERS --> 
				    <div class="row">
			        <div class="col-md-10 offset-1">
				        <div class="row">
					        <div class="col-md-6">
						        <li class="mb-4">
						          <span>
								          <a href="http://www.math.uwaterloo.ca/~sacampbe/">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
																Sue Ann Campbell
								              </h6>						              
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">University of Waterloo<span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/campbell.jpg" class="rounded-circle" alt="Andrea Bild" style="height: 200px;">
						          </span>
						        </li>
				        	</div>

					        <div class="col-md-6">
						        <li class="mb-4">
						          <span>
								          <a href="https://www.ntdmodelling.org/">
								              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
																Deirdre Hollingsworth
								              </h6>
								          </a>
						              <span class="d-block text-muted">
						              <span class="catlist text-capitalize text-muted smoothscroll">Oxford Big Data Institute, <a href="https://twitter.com/DeirdreHoll">@DeirdreHoll</a><span class="sep"></span></span>
						              </span>
						              </br>
						              <img src="/assets/images/deirdre.jpg" class="rounded-circle" alt="Adam Martin" style="height: 200px;">
						          </span>
						        </li>
						        
						      </div>
					      </div>				        
			        </div> 
			      </div> <!-- end row of plenary speakers -->
				        				    
		        
	        </ol>

					
        </div>
	    </div>
	    </br>

			<h4 class="font-weight-bold spanborder"><span>Logistics & Registration</span></h4>
	    <div class="row main-font">
        <div class="col-md-12">
				<p>At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (<a href="https://www.sococo.com/">Sococo</a> and <a href="https://www.mathdept.org/">MathDept.org</a>) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other.</p>


				<p>The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  <strong>You must still register for the meeting through the <a href="https://smb.wildapricot.org/event-3878590">registration website</a>, for a $25 fee.</strong> You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting.</p>



				<p>The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology.</p>



				<p>If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform.</p>



				<p>If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).</p>
        
        </div>
	    </div>
	    
	    </br>
	    
	    <div class="row main-font">
        <div class="col-md-5 offset-1">
	        <a href="./conference-announcement">
		        <div class="card" style="background-color: #e8f3ec; color: black;">
						  <div class="card-body">
						    <h5 class="card-title">Read the conference announcement</h5>
						    <h6 class="card-subtitle mb-2 text-muted">by Sandy Anderson, SMB President</h6>
						    <p class="card-text">Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye...</p>						   
								<span class="btn btn-my-dark">Read more...</span>	 
						  </div>
						</div>
					</a>
        </div>
        <div class="col-md-5">
	        <a href="./registration">
		        <div class="card" style="background-color: #e8f3ec; color: black;">
						  <div class="card-body">
						    <h5 class="card-title">Register for SMB 2020</h5>
						    <h6 class="card-subtitle mb-2 text-muted">$25 registration fee</h6>
						    <p class="card-text">To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25.</p>						   
								<span class="btn btn-my-dark">Registration</span>	 
						  </div>
						</div>
					</a>
        </div>
	    </div>
			
			</br>
	    </br>


	    
	    <h4 class="font-weight-bold spanborder"><span>Pre-conference Mentoring Program</span></h4>
	    <div class="row main-font">
        <div class="col-md-12">  
					          
          
          <p>The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at <a href="mailto:jtnardin@ncsu.edu">jtnardin@ncsu.edu</a>.  This workshop will consist of the following speakers and talks:</p>

	        <p>* all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change.</p>




					<table class="table table-striped">
					  <thead>
					    <tr>
					      <th scope="col">Time</th>
					      <th scope="col">Speaker</th>
					      <th scope="col">Scheduled Event</th>
					    </tr>
					  </thead>
					  <tbody>
					    <tr>
					      <th scope="row">1:00pm</th>
					      <td>Robert Smith?</td>
					      <td>Introduction</td>
					    </tr>
					    <tr>
					      <th scope="row">1:00–1:30pm</th>
					      <td>Alexandra Hogan</td>
					      <td>How to Attend a Virtual Conference</td>
					    </tr>
					    <tr>
					      <th scope="row">1:30–2:00pm</th>
					      <td>Rebecca Everett</td>
					      <td>Things I Wish I'd Known as a Postdoc</td>
					    </tr>
					    <tr>
					      <th scope="row">2:00–3:30pm</th>
					      <td>Robert Smith?</td>
					      <td>How to Get a Tenure-Track Job</td>
					    </tr>
					    <tr>
					      <th scope="row">3:30–4:00pm</th>
					      <td>Meghan Burke</td>
					      <td>The Tenure Process</td>
					    </tr>
					    <tr>
					      <th scope="row">4:00–5:00pm</th>
					      <td>Helen Moore <em>(Applied Biomath)</em> </br> Marc Strauss <em>(Springer-Nature)</em> </br> Susan Massey <em>(Systems Oncology)</em> </br>   Cheryl Zapata-Allegro <em>(BioFire Diagnostics)</em>				      
					      </td>
					      <td>Panel discussion </br> <em>(Moderated by Rebecca Everett)</em></td>
					    </tr>
					  </tbody>
					</table>

					
					
          
        </div>
	    </div>
		</div>
	</div>
</div>
    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    

    <!-- Footer <div class="row justify-content-between"> -->
    <footer class="bg-white border-top p-3 text-muted small">
      <div class="container">
        <div class="row">
          <div class="col-12">
	          	
	          	<span class="navbar-brand mr-2 mb-0"><strong>SMB 2020 Sponsors</strong></span></br>
	          	<img class="w-100" src="/assets/images/sponsors.png" alt="SMB Sponsors">
                
          </div>
        </div>
        
        </br>
        
        <div class="row">
          <div class="col-12"> 
						<span class="navbar-brand mr-2 mb-0"><strong>Organizing Committee</strong></span> 
          </div>
        </div>
        
        </br>
        <div class="row">
	        	        
          <div class="col-4"> 
	           
						<ol class="list-featured">
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Heiko Enderling (Co-Chair)
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">H. Lee Moffitt Cancer Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Amber Smith (Co-Chair)
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Tennessee Health Science Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Sandy Anderson
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">H. Lee Moffitt Cancer Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
				  	
				  								        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Ruth Baker
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Oxford, UK<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        
			        
						  </ol>
          </div>
            
          <div class="col-4">
            <ol class="list-featured-second">	
	            <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Denise Kirschner
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Michigan, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Torbjörn Lundh
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">Chalmers and University of Gothenburg, Sweden<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>					        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  John Nardini
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">NC State University, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Marissa Renardy
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Michigan, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			      </ol>
          </div>		
          
          <div class="col-4">
	          <ol class="list-featured-second">	
	          
	          	<li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Santiago Schnell
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Michigan, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Elissa Schwartz
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">Washington State University, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Jeffrey West
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">H. Lee Moffitt Cancer Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
	                  		Ping Ye 
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">South Dakota State University, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
	          </ol>
	          
	          <span>Copyright SMB © <script>document.write(new Date().getFullYear())</script>.</span>
          </div>
                
        </div> <! -- end row justify-content-between --> 
      </div>
    </footer>

    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
