<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>All Posters | SMB 2020</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>All Posters | SMB 2020</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="All Posters" />
<meta name="author" content="esmb" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/all-posters.html" />
<meta property="og:url" content="http://localhost:4000/all-posters.html" />
<meta property="og:site_name" content="SMB 2020" />
<script type="application/ld+json">
{"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"},"name":"esmb"},"author":{"@type":"Person","name":"esmb"},"headline":"All Posters","description":"The Society for Mathematical Biology - e-Conference 2020.","url":"http://localhost:4000/all-posters.html","@type":"WebPage","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


    <!-- This goes before </head> closing tag, Google Analytics can be placed here --> 






		<meta name="twitter:card" content="summary_large_image">
		<meta name="twitter:site" content="smb2020.org">
		<meta name="twitter:creator" content="@ESmb2020">
		<meta name="twitter:title" content="All Posters">
		<meta name="twitter:description" content="All Posters">
		<meta name="twitter:image" content="https://raw.githubusercontent.com/JeffreyWest/esmb/master/assets/images/preview.png">

		<meta name="title" content="All Posters" />
		
		<meta property="og:title" content="All Posters" />
		
		
			<meta name="description" content="/all-posters.html" />
			<meta property="og:description" content="/all-posters.html" />
		
		
		
		<meta property="og:image" content="https://raw.githubusercontent.com/JeffreyWest/esmb/master/assets/images/preview.png"/>

		
	


</head>

<body class="">

    <!-- Navbar -->
    <nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
    <div class="container">
        <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarColor02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
        </button>
        <div class="navbar-collapse collapse" id="navbarColor02" style="">
            <ul class="navbar-nav mr-auto d-flex align-items-center">
               <!--  Replace menu links here -->

<li class="nav-item">
	
		<img src="../assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2">
	
</li>


<a class="navbar-brand" href="../"><strong>SMB 2020</strong></a>



<li class="nav-item">
	
	<a class="nav-link" href="../">Home</a>
	
</li>


<li class="nav-item">
	
	<a class="nav-link" href="../conference-announcement">Announcement</a>
	
</li>


<li class="nav-item">
	
		<a class="nav-link" href="../registration">Registration</a>
	
</li>

<li class="nav-item">
	
		<a class="nav-link" href="../store">Store</a>
	
</li>

<li class="nav-item">
	
		<a class="nav-link" id="strong-nav" href="../schedule"><strong>Schedule</strong></a>
	
</li>

<li class="nav-item">
	
	<a class="nav-link" href="../subgroups">Subgroups</a>
	
</li>

<li class="nav-item">
	
	<a class="nav-link" href="../faq">FAQ</a>
	
</li>

<li class="nav-item">
	
	<a class="nav-link" href="../logistics">Logistics</a>
	
</li>

            </ul>
<!--
            <ul class="navbar-nav ml-auto d-flex align-items-center">
                
<script src="../assets/js/lunr.js"></script>



<script>
$(function() {
    $("#lunrsearchresults").on('click', '#btnx', function () {
        $('#lunrsearchresults').hide( 1000 );
        $( "body" ).removeClass( "modal-open" );
    });
});
    

var documents = [{
    "id": 0,
    "url": "http://localhost:4000/404/",
    "title": "",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": " 404 Page not found :(  The requested page could not be found. "
    }, {
    "id": 1,
    "url": "http://localhost:4000/CDEV-posters.html",
    "title": "CDEV Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 2,
    "url": "http://localhost:4000/EDUC-posters.html",
    "title": "EDUC Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 3,
    "url": "http://localhost:4000/IMMU-posters.html",
    "title": "IMMU Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 4,
    "url": "http://localhost:4000/MEPI-posters.html",
    "title": "MEPI Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 5,
    "url": "http://localhost:4000/MFBM-posters.html",
    "title": "MFBM Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 6,
    "url": "http://localhost:4000/NEUR-posters.html",
    "title": "NEUR Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 7,
    "url": "http://localhost:4000/ONCO-posters.html",
    "title": "ONCO Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 8,
    "url": "http://localhost:4000/OTHE-posters.html",
    "title": "OTHE Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 9,
    "url": "http://localhost:4000/POPD-posters.html",
    "title": "POPD Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 10,
    "url": "http://localhost:4000/all-posters.html",
    "title": "All Posters",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": ""
    }, {
    "id": 11,
    "url": "http://localhost:4000/author-jane.html",
    "title": "Jane",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "                        {{page. title}} Follow:         {{ site. authors. jane. site }}         {{ site. authors. jane. bio }}                                   Posts by {{page. title}}:       {% assign posts = site. posts | where: author , jane  %}      {% for post in posts %}      {% include main-loop-card. html %}      {% endfor %}  "
    }, {
    "id": 12,
    "url": "http://localhost:4000/author-sal.html",
    "title": "Sal",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "                        {{page. title}} Follow:         {{ site. authors. sal. site }}         {{ site. authors. sal. bio }}                                   Posts by {{page. title}}:       {% assign posts = site. posts | where: author , sal  %}      {% for post in posts %}      {% include main-loop-card. html %}      {% endfor %}  "
    }, {
    "id": 13,
    "url": "http://localhost:4000/authors-list.html",
    "title": "Authors",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "{{page. title}}:     {% for author in site. authors %}                                         {{ author[1]. name }} :       (View Posts)      {{ author[1]. bio }}                          &nbsp;       &nbsp;                                    {% endfor %}  "
    }, {
    "id": 14,
    "url": "http://localhost:4000/categories.html",
    "title": "Categories",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "          Categories          {% for category in site. categories %}     {{ category[0] }}:           {% assign pages_list = category[1] %}    {% for post in pages_list %}    {% if post. title != null %}     {% if group == null or group == post. group %}           {% include main-loop-card. html %}     {% endif %}    {% endif %}    {% endfor %}    {% assign pages_list = nil %}    {% assign group = nil %}    {% endfor %}                  {% include sidebar-featured. html %}          "
    }, {
    "id": 15,
    "url": "http://localhost:4000/faq/",
    "title": "Frequently Asked Questions",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Frequently Asked Questions		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt; 	  								* For other questions, please contact Heiko Enderling at Heiko. Enderling@moffitt. org. 									1. Where will eSMB2020 take place?: 	  	   				The eSMB2020 conference will be held virtually. Through partnership with MathDept. org, we will utilize Sococo, Zoom, and Slack to create an immersive conference experience. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  2. What are the registration costs?: 	  	   				The registration fee is $25 for our first virtual annual meeting. You must register for the meeting through the registration website. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  3. Do I need to register if I am not presenting a poster or abstract?: 	  	   				Yes, registration is required for participation in eSMB2020. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;									4. What the poster requirements?: 	  	   				Registration is required to present a poster. Please register at your earliest convenience at smb2020. org/registration.  A tutorial video of the sococo platform is online for you to get a glimpse into what the virtual poster session will look and feel like ( http://smb2020. org/logistics/ ). Please follow the following guidelines: 													Poster dimensions: Max of 3' x 4/ (91cm x 122cm). 					Format: PDF only					File Size: Not to exceed 25MB. 					If necessary, here are instructions to reduce file size: https://www. wikihow. com/Reduce-PDF-File-Size. 					Please save your poster with the following File Name: Lastname_Firstname_XXXX. pdf, where XXXX is to be replaced by the identifier of the SMB subgroup the poster belongs to. XXXX = { CDEV, EDUC, IMMU, MEPI, NEUR, ONCO, MFBM, POPD, OTHE }. All subgroups (and corresponding codes) are listed on the subgroup page. 					Please upload your poster pdf to the Subgroup Google Drive folder that your subgroup chair emailed you. 															   	  	  	  &lt;/br&gt;			&lt;/br&gt;																															  	  5. If I submitted my abstract to the Heidelberg ESMTB meeting, is it too late to transfer it to SMB2020?: 	  	   				If you have not yet transferred your abstract, please submit your abstract as a new abstract during eSMB2020 registration.  	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  6. Will the Heidelberg ESMTB meeting still take place?: 	  	   				ESMTB organizers plan to hold ESMTB in 2021 in Heidelberg. Unfortunately, this won’t be a joint meeting with the SMB. The SMB 2021 annual meeting will be held at UC Riverside. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  7. Will my talk be recorded or live-streamed?: 	  	   				The eSMB2020 conference offers the unprecedented opportunity to record talks for dissemination after the meeting. Live-streaming is currently not planned. During abstract submission, authors can select to opt in/out of recordings.  	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  8. Can I re-submit an updated abstract?: 	  	   				Yes. To submit an updated abstract, please coordinate with the SMB subgroup lead that your abstract belongs to. If your abstract does not belong to an SMB subgroup, please contact Torbjörn Lundh. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  9. Can I submit an abstract that is not affiliated with any subgroup?: 	  	   				Yes, during abstract submission you choose the option ‘none of the above’ for subgroup affiliation. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  10. Can we still organize minisymposia?: 	  	   				Please reach out to the SMB subgroup leaders that your minisymposium thematically belongs to to confirm if accommodations can be made for additional minisymopisia. Note: the mathematical oncology section is currently at capacity and cannot accept further MS submissions. Otherwise, if the MS thematically does not belong to a subgroup, please contact Torbjörn Lundh. 	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  11. When will I be notified about abstract selection?: 	  	   				All submitted abstract will be evaluated by the SMB Subgroups and conference scientific committee after the abstract submission deadline on July 15. Notification of abstract selection for poster or oral presentation will be sent before the end of July.  	   	  	  	  &lt;/br&gt;			&lt;/br&gt;	  	  12. I've completed the registration form, but did not submit an abstract yet. How do I submit my abstract post-registration?: 	  	   				You can send your title and abstract to the SMB Membership Team at membership@smb. org, and they will add it to your registration. 	   	  	  	  13. How do I format my virtual poster?: 	  	   				Posters should be landscape format and saved as a single page pdf, to ensure that everybody will be able to view it on their personal computer or tablet.  When designing your poster, please chose content and font size carefully to clearly communicate your research. All accepted posters will be uploaded to a shared drive for access through Sococo during the poster session. Details will be communicated in the next couple of weeks. 	   	  	  	  	  	  	  	  	  	  	  			"
    }, {
    "id": 16,
    "url": "http://localhost:4000/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 17,
    "url": "http://localhost:4000/logistics/",
    "title": "Conference Logistics",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Conference Logistics		    			    									    			    		    	    	  			&lt;/br&gt;	&lt;/br&gt;	&lt;/br&gt;				  								 						Sococo Orientation: 	  	  	       	        	  		  						  											August 15 Update: 	  	   																				Happy Sunday #SMB2020 participants!					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins tomorrow! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					Lost on Monday morning? Don’t miss the opening remarks (8am ET) and the Opening Plenary by Dr. Shayn Peirce-Cottler. The session will be linked in all mini-symposia and overflow rooms on all Subgroup floors. In case you have trouble connecting through Sococo, here is the direct Zoom link: https://zoom. us/j/97466727056 for this session. Zoom links are for registered participants only – please do not share or post on social media. 					&lt;/br&gt;&lt;/br&gt;					Official eSMB2020 Zoom background: For anyone interested, feel free to use our eSMB2020 Zoom background. Find it here in both mirrored &amp; unmirrored settings. 					&lt;/br&gt;&lt;/br&gt;					Help – judges needed! We still need a few volunteers to judge posters and contributed talks!&lt;/br&gt;					POPD contributed talks judges: contact Christina. Cobbold@glasgow. ac. uk&lt;/br&gt;					OTHE posters or contributed talks judges: contact torbjorn. lundh@chalmers. se&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					All things eSMB2020:&lt;/br&gt;					Schedule: All last-minute changes will be posted online and in Slack&lt;/br&gt;					SMB Business Meeting(Wed): come learn about our Society and welcome our new President-elect and Board of Directors members&lt;/br&gt;					Career Fair(Thurs): ‘SMB Social/Career Fair Floor’ – there’s still time to upload job announcements and CVs&lt;/br&gt;					Conference t-shirt: Also linked from ‘SMB Welcome/Industry Floor’&lt;/br&gt;					Slack space: New to Slack? Check out our FAQ page&lt;/br&gt;					Twitter: @ESmb2020 #SMB2020&lt;/br&gt;					Sponsors – Industry, Publishers, &amp; Software: ‘SMB Welcome/Industry Floor’&lt;/br&gt;					'Zoom-only’ participants: Zoom links have been sent!&lt;/br&gt;										&lt;/br&gt;&lt;/br&gt;					Tips &amp; Tricks:&lt;/br&gt;					Have your passwords handy for Zoom, Sococo, and Slack&lt;/br&gt;					Initialize yourself in Sococo on the Subgroup Floor that most aligns with your research area&lt;/br&gt;					Issues with Sococo? Try refreshing, then ask #help-desk or use ‘find’ feature in Sococo to locate ‘HELP’&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					All things SMB: &lt;/br&gt;					&lt;/br&gt; 					Society Leadership and Committees&lt;/br&gt;					Membership: ‘SMB Welcome/Industry Floor’&lt;/br&gt;					Subgroups: Not a member? Account -&gt; ‘edit profile’&lt;/br&gt;					Bulletin of Mathematical Biology: Editors available on ‘SMB Welcome/Industry Floor’&lt;/br&gt;					Merchandise: ‘SMB Welcome/Industry Floor’&lt;/br&gt;					SMB Newsletter&lt;/br&gt;					Academic Year Mentoring Program&lt;/br&gt;					SMB Digest Community Forum&lt;/br&gt;					Society Prizes&lt;/br&gt;					Travel and Meeting Grants&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					Looking forward to seeing all 1,732 of you at the best and largest SMB Annual Meeting yet!					Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye					&lt;/br&gt;&lt;/br&gt;					And, all other Subgroup leaders and eSMB2020 track coordinators,					Andreas Buttenschoen, Alex Fletcher, Linus Schumacher, Holly Gaff, Shelby Scott, Carrie Diaz Eaton, Angela Reynolds, Jonathan Forde, Stanca Ciupe, Jessica Conway, Jane Heffernan, John Glasser, Zhilan Feng, Cheng Ly, Wilten Nicola, Erica Rutter, John Nardini, Kevin Flores, Christina Cobbold, James Watmough, Torbjörn Lundh									 	   	  	  				  											August 15 Update: 	  	   															Happy Saturday #SMB2020 participants!					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins in just 2 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					Getting excited for our eSMB2020 scientific program? We know we are! And, we owe a BIG ‘thank you’ to our amazing Subgroup/track Chairs and organizers that put together a state-of-the-art scientific program:					&lt;/br&gt;&lt;/br&gt;											Cell and Developmental Biology (CDEV,@SMBdevBio ): Andreas Buttenschoen, Alex Fletcher, Linus Schumacher						Education (EDUC, @Edu2Smb): Holly Gaff, Shelby Scott, Carrie Diaz Eaton						Immunobiology and Infection (IMMU, @smb_imin): Angela Reynolds, Jonathan Forde, Stanca Ciupe, &amp; Jessica Conway						Mathematical Epidemiology (MEPI, #SMBMathEpi):Jane Heffernan, John Glasser, Zhilan Feng						Mathematical Neuroscience (NEUR, @MathNeuro_): Cheng Ly, Wilten Nicola						Mathematical Oncology (ONCO, @mathonco): Heiko Enderling, Sandy Anderson						Methods for Biological Modeling (MFBM, @SMBmfbm): Erica Rutter, John Nardini, Kevin Flores						Population Dynamics, Ecology &amp; Evolution (POPD, @smbpdee): Christina Cobbold, James Watmough						General/other track (OTHE): Torbjörn Lundh										&lt;/br&gt;&lt;/br&gt;					Jump on each group’s Slack channel in smb2020. slack. com, ‘get’ them in Sococo, attend the Subgroup business meeting, or drop them an email to thank them!					&lt;/br&gt;&lt;/br&gt;					Want to get involved or find out more about our Subgroups? https://www. smb. org/subgroups/					&lt;/br&gt;&lt;/br&gt;					Not a member of a Subgroup? https://www. smb. org/my-account/  ‘edit profile’					&lt;/br&gt;&lt;/br&gt;					eSMB2020 mentoring program: Our incredible mentoring committee, Elissa Swartz, Robert Smith?, and John Nardini, has outdone themselves organizing the largest mentoring program in our history!					&lt;/br&gt;&lt;/br&gt;					Pre-conference Mentoring workshop with 200+ attendees&lt;/br&gt;					eSMB2020 mentoring program with 550+ mentees and 175+ mentors&lt;/br&gt;					Mentoring luncheon on the ‘SMB Mentor Session Floor’ on Monday&lt;/br&gt;					Want to be a SMB mentor or mentee after eSMB2020 ends? Check out our Academic Year Mentoring Program&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					‘Zoom-only’ participants: Zoom links will be sent tomorrow! They are for registered participants only - please do not share! Want to upgrade to use Sococo? Contact us at membership@smb. org					&lt;/br&gt;					Mandatory Sococo orientation (full conference participants only): Please attend one of the remaining 3 orientation sessions (https://zoom. us/j/94700659356)					&lt;/br&gt;					Saturday, August 15th at 9am ET&lt;/br&gt;					Saturday, August 15th at 1pm ET&lt;/br&gt;					Sunday, August 16th at 9am ET&lt;/br&gt;					eSMB2020 Sponsors – Industry, Publishers, &amp; Software: Visit the ‘SMB Welcome/Industry Floor’ to meet with our sponsors and get information about the companies, products, available jobs, and options for publishing your manuscripts. 					&lt;/br&gt;&lt;/br&gt;					eSMB2020 conference t-shirt: Visit the ‘SMB Welcome/Industry Floor’ or get yours today					&lt;/br&gt;&lt;/br&gt;					Slack: Join our Slack channel! New to Slack? Check out our FAQ page					&lt;/br&gt;&lt;/br&gt;					Twitter: Follow us at @ESmb2020 and @SMB_MathBiology use #SMB2020 as the hashtag					&lt;/br&gt;&lt;/br&gt;					SMB membership: Visit the ‘SMB Welcome/Industry Floor’ or join today!					&lt;/br&gt;&lt;/br&gt;					SMB Digest community forum: Visit https://www. smb. org/forums/					&lt;/br&gt;&lt;/br&gt;										Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye																						 	   	  	  					  											August 14 Update: 	  	   																										Happy Friday #SMB2020 participants!						&lt;/br&gt;&lt;/br&gt;												The 2020 SMB Annual Meeting (eSMB2020) begins in just 3 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 						&lt;/br&gt;&lt;/br&gt;						eSMB2020 Sponsors – Industry, Publishers, &amp; Software: We are fortunate to have 9 companies that have graciously sponsored eSMB2020 and funded special poster and talk prizes. Representatives will be present and available on our ‘SMB Welcome/Industry Floor’ with information about their companies, available jobs, and options for publishing your manuscripts. Check out the PDF flyers linked at the bottom and their websites (hyperlinked) as a preview of the amazing things they have to offer. 						&lt;/br&gt;&lt;/br&gt;						Amgen&lt;/br&gt;						Applied BioMath&lt;/br&gt;						Berkeley-Madonna&lt;/br&gt;						DILIsym, a Simulations Plus Co. &lt;/br&gt;						Elsevier&lt;/br&gt;						Mathdept. org&lt;/br&gt;						Merck&lt;/br&gt;						Springer Nature&lt;/br&gt;						Wiley&lt;/br&gt;												&lt;/br&gt;&lt;/br&gt;						‘Zoom-only’ participants: If you selected ‘I only want to attend specific talks, and do not need access to the full conference’ upon registering, you will not receive a Sococo invitation. We will be sending you the Zoom links prior to the start of the conference. Zoom links are for registered participants only and should not be shared with friends and colleagues or on social media.   						Changed your mind and want to upgrade to use Sococo and have access to the full eSMB2020 experience? Contact us at membership@smb. org Poster presenters are highly encouraged to present their poster in Sococo for the most beneficial experience. 						Mandatory Sococo orientation (full conference participants only): All registrants who selected ‘full conference’ upon registering should have received their Sococo invitations! Please do not try to create your own account!						&lt;/br&gt;&lt;/br&gt;						Please attend one of the remaining 5 orientation sessions (https://zoom. us/j/94700659356)						&lt;/br&gt;&lt;/br&gt;						Friday, August 14th at 5pm ET (U. S. Eastern Time)&lt;/br&gt;						Friday, August 14th at 9pm ET&lt;/br&gt;						Saturday, August 15th at 9am ET&lt;/br&gt;						Saturday, August 15th at 1pm ET&lt;/br&gt;						Sunday, August 16th at 9am ET&lt;/br&gt;												&lt;/br&gt;&lt;/br&gt;						eSMB2020 virtual conference t-shirt: Yesterday, we unveiled our eSMB2020 t-shirt. Go get yours today: http://smb2020. org/store or visit our SMB merchandise store on the ‘SMB Welcome/Industry Floor’ in Sococo. We hope to see these …virtually everywhere…						&lt;/br&gt;&lt;/br&gt;						Slack: We’ll only post updates, changes, and be available for help over Slack. Join our Slack channel (smb2020. slack. com) by clicking here: https://bit. ly/31BG64m						&lt;/br&gt;&lt;/br&gt;						Twitter: Follow us at https://twitter. com/ESmb2020 and https://twitter. com/SMB_MathBiology. Use #SMB2020 as the hashtag						&lt;/br&gt;&lt;/br&gt;						SMB membership: Visit the SMB membership team on the ‘SMB Welcome/Industry Floor’ or join today! https://www. smb. org/application/						&lt;/br&gt;&lt;/br&gt;						SMB Digest community forum: Visit https://www. smb. org/forums/						&lt;/br&gt;&lt;/br&gt;										Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye					&lt;/br&gt;&lt;/br&gt;																		AppliedBioMath_CareersFlyer_August2020. pdf &lt;/br&gt;					BerkeleyMadonna_SMB2020. pdf&lt;/br&gt;					Elsevier. pdf&lt;/br&gt;					eSMB_DSS_Flyer. pdf&lt;/br&gt;					Springer_MathBio. pdf&lt;/br&gt;					Wiley_MeetingFlyer. pdf&lt;/br&gt;					WMODatSMB2020. pdf&lt;/br&gt;									 	   	  	  					  											August 12 Update: 	  	   															Happy Wednesday #SMB2020 participants!					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins in just 5 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					eSMB2020 virtual conference t-shirt: …drumroll, please…					&lt;/br&gt;&lt;/br&gt;					The 2020 eSMB2020 t-shirt features the conference logo on the front, and the map of all registered participants on the back. The Society for Mathematical Biology is …virtually everywhere…					&lt;/br&gt;&lt;/br&gt;					The t-shirt is available as ladies trim or unisex in various sizes and colors. Visit our new online store to order your shirt today at http://smb2020. org/store. Members receive a 22% discount (!) so be sure to log into your SMB account first (located at the top of your screen). You can also order your shirt at the SMB merchandise store on the ‘SMB Welcome/Industry Floor’. 					&lt;/br&gt;&lt;/br&gt;											&lt;/br&gt;&lt;/br&gt;					SMB membership information at eSMB: Want to learn more about the many benefits of SMB membership? Visit the SMB membership team on the ‘SMB Welcome/Industry Floor’ during the conference coffee breaks and social gatherings to discuss in person with our awesome Membership Committee Chairs, Marissa Renardy and Sam Erwin, and SMB leadership. You can also join today! https://www. smb. org/application/ 					&lt;/br&gt;&lt;/br&gt;					Mandatory Sococo orientation: All registrants will be receiving their Sococo invitations by email from MathDept. org tomorrow, Thursday, August 13th. The full Sococo experience is only available on a desktop or laptop using Google Chrome or the desktop app (https://app. sococo. com/a/download). Please attend one of the 6 orientation sessions (https://zoom. us/j/94700659356)					&lt;/br&gt;&lt;/br&gt;											Thursday, August 13th at 9pm ET (U. S. Eastern Time)						Friday, August 14th at 5pm ET						Friday, August 14th at 9pm ET						Saturday, August 15th at 9am ET						Saturday, August 15th at 1pm ET						Sunday, August 16th at 9am ET										&lt;/br&gt;&lt;/br&gt;					Twitter: Follow us at https://twitter. com/ESmb2020 and use #SMB2020 as the hashtag					&lt;/br&gt;&lt;/br&gt;					Slack: Join our Slack channel (smb2020. slack. com) by clicking here: https://bit. ly/31BG64m					&lt;/br&gt;&lt;/br&gt;					SMB Digest community forum: Visit https://www. smb. org/forums/					&lt;/br&gt;&lt;/br&gt;					Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling&lt;/br&gt;					eSMB2020 Co-Chairs&lt;/br&gt;					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee,&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye																			 	   	  	  						  											August 11 Update: 	  	   																				Dear friends and colleagues,					&lt;/br&gt;&lt;/br&gt;					The 2020 SMB Annual Meeting (eSMB2020) begins in just 6 days! Here is today’s installment of our daily countdown to #SMB2020 with important news and updates. 					&lt;/br&gt;&lt;/br&gt;					Sococo: Through our partnership with MathDept. org, we are creating an immersive conference experience. To get us prepped for the upcoming week:					&lt;/br&gt;&lt;/br&gt;					Orientations: Mathdept. org will host mandatory orientation sessions. All orientations will use the same Zoom link: https://zoom. us/j/94700659356					&lt;/br&gt;&lt;/br&gt;											Thursday, August 13th at 9pm ET (U. S. Eastern Time)						Friday, August 14th at 5pm ET						Friday, August 14th at 9pm ET						Saturday, August 15th at 9am ET						Saturday, August 15th at 1pm ET						Sunday, August 16th at 9am ET										&lt;/br&gt;&lt;/br&gt;					Accounts: These are being made for all participants by MathDept. org. All registrants will be receiving their Sococo email invitations on Thursday, August 13th. 					&lt;/br&gt;&lt;/br&gt; 					Browser/App: Sococo is only supported in the Sococo app or in Google Chrome, so please download one of these ahead of time. 					&lt;/br&gt;&lt;/br&gt;					Zoom: All oral presentations will be streamed as a Zoom video-conference or webinar. Plenaries and Keynotes will be linked to all Sococo ‘floors’, and mini-symposia and contributed talks will be linked in their respective track ‘floors’. For those who selected ‘only Zoom’ in the registration, you will receive the links prior to the conference. 					&lt;/br&gt;&lt;/br&gt;					Schedule: The schedule of all presentations is on our website at http://smb2020. org/schedule. The program is clickable, sharable, and can be filtered by track (fancy, right?!) Can’t find your abstract or have any questions? Please contact your Subgroup or track Chair or the conference Co-Chairs (Amber Smith, amber. smith@uthsc. edu, or Heiko Enderling, heiko. enderling@moffitt. org). 					&lt;/br&gt;&lt;/br&gt;					Twitter: Follow us at https://twitter. com/ESmb2020 and use #SMB2020 as the hashtag. 					&lt;/br&gt;&lt;/br&gt;					Slack: Join our Slack channel (smb2020. slack. com) by clicking here: https://bit. ly/31BG64m!					&lt;/br&gt;&lt;/br&gt;					SMB membership: Join today! https://www. smb. org/application/. 					&lt;/br&gt;&lt;/br&gt;					SMB Digest community forum: Want to receive and post news on job postings, conferences, and other exciting things happening in Mathematical Biology? Visit https://www. smb. org/forums/ and sign up at https://www. smb. org/smb-digest-community-forum-how-to/. 					&lt;/br&gt;&lt;/br&gt;					Sincerely,					&lt;/br&gt;&lt;/br&gt;					Amber Smith &amp; Heiko Enderling					&lt;/br&gt;&lt;/br&gt;					eSMB2020 Co-Chairs					&lt;/br&gt;&lt;/br&gt;					On behalf of the eSMB2020 Organizing Committee:					&lt;/br&gt;&lt;/br&gt;					Sandy Anderson, Ruth Baker, Denise Kirschner, Torbjörn Lundh, John Nardini, Marissa Renardy, Santiago Schnell, Elissa Schwartz, Jeffrey West, Ping Ye																   &lt;/div&gt;	  &lt;/div&gt;	  		&lt;/div&gt;	&lt;/div&gt;		  								 						A virtual tour of the Sococo platform: 	  	   				The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 	   	  	  	  	       	    					 					 Your browser does not support HTML video. 						        	  		  	      	    Supported Web Browsers	    						Always use the latest version of Google Chrome. Sococo supports the current version and the previous major release of Google Chrome. Other browsers are not supported. 					 					Network Connection with Adequate Bandwidth 											At a minimum, Sococo requires a download speed of 5 Mbps and an upload speed of 2 Mbps.  					 										Supported Operating Systems 											Sococo supports the current version and the two previous major releases for the following operating systems:													Mac OS X							Windows (administrator right required for the desktop app)							Linux (any modern distribution that is supported by Google Chrome)							Note: The desktop app for Linux is not supported. 											 										Computer Hardware 																			Any dual-core processor								Integrated camera or USB web camera											 	    	        	    	  	  	  	  	  	  	  					&lt;/div&gt; "
    }, {
    "id": 18,
    "url": "http://localhost:4000/monday/",
    "title": "SMB2020 Monday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Monday, August 17		    		    {% include clock. html %}		    		    			    				        								&lt;/br&gt;								&lt;/br&gt;															 {% include schedule_menu. html %}				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 				                    Opening &amp; Welcome        8:00am:                   		  	                    Opening Plenary        8:30am:         Shayn Peirce-Cottler, &lt;/br&gt;University of Virginia, @PeirceLab             												     		       			       			                                  		   		    Agent-based Modeling of Multi-Cellular Systems for Designing Better Therapies 		     			    Abstract: The most prevalent, devastating, and complex diseases of our time, such as diabetes, cardiovascular disease, and infectious diseases, involve the interactions of heterogeneous cells with one another and with their changing environment. However, the drugs we typically use to treat these diseases target a single protein (e. g. anti-vascular endothelial growth factor (VEGF) for treating diabetic retinopathy) and disregard the fact that cells within tissues are highly heterogeneous and have individualized responses that contribute to the tissue-level outcomes. To bridge the gap between protein and multi-cell/tissue-levels, my lab develops agent-based computational models and uses them in combination with experimental approaches, to predict how individual cell behaviors give rise to tissue-level adaptations. We have used agent-based modeling to simulate the structural adaptations of large and small blood vessels, skeletal muscle regeneration following injury, and immune cell trafficking and differentiation during inflammation and infection. Our studies have suggested new mechanistic hypotheses and provided suggestions for the design of novel therapies that account for the dynamic and heterogeneous interactions between different cell types within a diseased tissue. 		     		    		     			    Bio: Shayn Peirce-Cottler, Ph. D. is Professor of Biomedical Engineering with secondary appointments in the Department of Ophthalmology and Department of Plastic Surgery at the University of Virginia. Dr. Peirce-Cottler received Bachelor’s of Science degrees in Biomedical Engineering and Engineering Mechanics from The Johns Hopkins University in 1997. She earned her Ph. D. in the Department of Biomedical Engineering at the University of Virginia in 2002. Dr. Peirce-Cottler develops and uses computational models in conjunction with experiments to study structural and functional adaptations of tissues, in both health and disease, in order to develop new therapies for inducing regeneration in injured tissues or restoring homeostasis to diseased tissues. Her lab’s core expertise is in combining agent-based computational modeling with in vivo imaging of murine experimental models to examine and control the multi-cell interactions involved in angiogenesis. Current projects in her lab seek to exploit perivascular support cells, immune cells, and stem cells to invoke tissue regeneration and curb fibrosis during acute and chronic inflammation. Dr. Peirce-Cottler has published over 100 peer reviewed papers and book chapters, and she is the inventor on two issued U. S. Patents. She is a fellow in the American Institute for Medical and Biological Engineering College of Fellows and a Fellow of the Biomedical Engineering Society. She is also Past-President of The Microcirculatory Society. At UVA she teaches courses on computational modeling and cell and molecular physiology, and she is the Director of the Graduate Program in Biomedical Engineering. 		     		    	    	             			  										  		    		      		        Sub-group minisymposia (9:30-11:00am)		        {% include talks. html myday=page. day mygroup=page. ms1 mytime=page. ms1time %}		             		    		  									{% include morning_break. html %}								  		    		      		        Sub-group minisymposia (11:15am)		        {% include talks. html myday=page. day mygroup=page. ms2 mytime=page. ms2time %}		             		    		  											  		    		      		        Mentoring Luncheon		             		    		  											  		    		      		        Sub-group contributed talks (1:30-2:30pm)		        {% include talks. html myday=page. day mygroup=page. ct1 mytime=page. ct1time %}		             		    		  											  		    		      		        Poster Session (2:30-3:30pm)		        		        {% include poster_menu. html %}		        		             		    		  								  		    		      		        Sub-group Keynote		        3:30pm: 		        Priyanga Amarasekare, &lt;/br&gt;University of California Los Angeles 		        Population Dynamics, Ecology, &amp; Evolution Subgroup 		      		      																     				       					       					     		             		    		    		    				   				    Effects of climate warming on consumer-resource interactions 				     					    There is increasing evidence that climate warming is impacting biodiversity by disrupting species interactions.  Trophic (consumer-resource) interactions, which comprise the fundamental units (modules) of food webs, are of particular importance because they have an intrinsic tendency to fluctuate in abundance, thus running to risk of stochastic extinction during periods of low abundances. Here I present a mathematical framework for predicting warming effects on consumer-resource interactions.  This work differs from previous theory in two important ways. First, it uses delay differential equations to realistically depict the developmental delays inherent in ectotherm life cycles, and incorporates mechanistic descriptions of phenotypic trait responses, derived from first principles of thermodynamics, into the dynamical delay model. Second, it investigates the recent IPCC predictions on the increase in the number of hotter-than-average days. I report three key results. First, across latitudes (tropical vs. temperate) and feeding strategies (juvenile vs. adult attacked), a greater increase in the maximum temperature compared to the minimum (hotter-than-average summers) is more detrimental to consumer-resource interactions than a greater increase in the minimum temperature (warmer-than-average winters).  Second, across latitude and warming scenarios, effects of warming are more detrimental when the consumer attacks the adult stage of the resource.  Third, across warming scenarios and feeding strategies, consumer-resource interactions in the tropics are more at risk of species losses due to warming while those in the temperate zone are more at risk of extreme fluctuations in species' abundances. I discuss implications of these results for biodiversity and biological pest control. 				     			    			   		    		    		  									{% include happy_hour. html %}						&lt;/br&gt;			&lt;/br&gt;									 				 {% include schedule_menu. html %}			 																				      	      		      														Click to show abstracts											      	            "
    }, {
    "id": 19,
    "url": "http://localhost:4000/registration/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    August 17 - 20, 2020		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;   	  					Registration &amp; Abstract Submission: 	  	    		    To register for the SMB Conference, click below to be redirected to the registration form.  	  	    															    				    								 								  Register for SMB 2020								  $25 fee for registration. 								  &lt;/br&gt;										Register. . . 									 																	    						&lt;/br&gt;	    	  			"
    }, {
    "id": 20,
    "url": "http://localhost:4000/schedule/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    										Conference Schedule															Pre-conference Mentoring Program: 					The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone (EDT). If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 							    		    			    															 							  Pre-conference Mentoring Program							 			    			    		    		    						&lt;/br&gt;		    &lt;/br&gt;						Conference Overview: 						* all times are listed in US Eastern Time Zone (EDT). This schedule is tentative and subject to change. Click to view more details. 						{% include full_schedule. html %}		    			    &lt;/br&gt;		    &lt;/br&gt;		    		    			    															 {% include schedule_menu. html %}								The conference schedule is posted by day, above. Poster abstracts will be posted soon. 				    			    		    		    		    		    Industry Sponsors: 														Thinking about a job in industry, or looking to get your work published? Don't forget to visit our industry sponsors in the SMB Welcome/Industry space! See the flyers below for their schedules. 						 														DILIsym (click here)&lt;/br&gt;								Merck (click here)&lt;/br&gt;								Elsevier (click here)&lt;/br&gt;								Wiley (click here)&lt;/br&gt;								Springer (click here)&lt;/br&gt;								In addition, Applied BioMath will be staffing their room 1pm-3pm Mon-Wed. &lt;/br&gt;													 		    		    	    	  	"
    }, {
    "id": 21,
    "url": "http://localhost:4000/store/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    August 17 - 20, 2020		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;   	  					Society for Mathematical Biology - Store: 	  	    		    To purchase SMB merchandise, click below to be redirected to the SMB store.  	  	    															    				    								 								  SMB 2020 								  For sale: SMB 2020 conference shirt, and more. 								  								  								  &lt;/br&gt;								  &lt;/br&gt;								  &lt;/br&gt;										View merchandise. . . 									 																	    						&lt;/br&gt;	    	  			"
    }, {
    "id": 22,
    "url": "http://localhost:4000/subgroups/",
    "title": "SMB Subgroups",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Society for Mathematical Biology - Subgroups		    			    									    			    		    	    	  	&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;   	  					Cell and Developmental Biology - CDEV: 	  	   		    	The Subgroup on Cell and Developmental Biology brings together researchers who are interested in understanding cell and developmental biology phenomena using quantitative approaches. Cell and developmental biology has long proven a fertile ground for mathematical modelling and, with recent advances in microscopy, image analysis, and fluorescent reporter methods, it is becoming an increasingly quantitative discipline. We are now able to constrain mathematical models of developmental processes, for example through parameter estimation, increasing the potential of such models to be used in a predictive way. The objective of this Subgroup is to accelerate these efforts, by supporting and advancing the use of mathematical modelling as part of the ‘toolkit’ of cell and developmental biology. Chair: Andreas Buttenschoen, abuttens@math. ubc. ca Secretary: Alexander Fletcher, a. g. fletcher@sheffield. ac. uk Committee Members: Ruth Baker, Linus Schumacher, and Michael Watson 	   	  	  &lt;/br&gt;			&lt;/br&gt;	  Education - EDUC: 	  	    	The Education Subgroup of SMB is comprised of members who desire to improve the teaching and mentoring of future scientists, and in particular undergraduate and graduate mathematical and computational biology researchers. As new research frontiers emerge, our members seek to understand the best practices to help future researchers of biological questions be successful in this space. Content areas include, but are not limited to the teaching and learning of modeling, computing, data science, and quantitative reasoning. Challenge spaces include, but are not limited to designing learning experiences across disciplines, defining and assessing outcomes for new interdisciplinary and technological spaces, integrating data with mathematical modeling, educational outreach initiatives, and providing an inclusive and diverse environment for learning and for the profession. Chair: Dr. Holly Gaff, HGafff@odu. eduPast Chair: Carrie Diaz Eaton, cdeaton@bates. eduTwitter: @Edu4SMB 					  	  &lt;/br&gt;			&lt;/br&gt;	  Immunobiology and Infection - IMMU: 	  	    		    		    The Immunobiology and Infection Subgroup brings together researchers who are interested in the modeling and analysis of immune processes in human disease and of host-pathogen interactions. Our members study various topics, including within-host infectious disease dynamics, host immune responses, causes and effects of inflammation, disease progression and outcome, integration of experimental and clinical data into models, and model-driven experimental design. Chair: Angela Reynolds, areynolds2@vcu. edu Secretary/Treasurer: Jonathan Forde Advisory Committee Members: Jessica Conway and Stanca Ciupe Twitter: @smb_imin 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Mathematical Epidemiology - MEPI: 	  	    		    		    The Mathematical Epidemiology Subgroup brings together scholars interested in dynamical modeling and statistical analysis of the spread of infectious diseases. Topics of interest to the group include: deterministic and stochastic modelling of population dynamics of infectious diseases; statistical analysis of population-level infectious disease data; describing and modelling human behavioral decisions in relation to infectious diseases; control and eradication strategies for endemic infections; prevention and response strategies for emerging and re-emerging infections; effects of antimicrobial resistance; policy implications. Chairs: Zhilan Feng fengz@purdue. edu Twitter: #SMBMathEpi 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Mathematical Neuroscience - NEUR: 	  	    		    		    The Mathematical Neuroscience Subgroup provides a forum for the use and development of advanced mathematical techniques in the modelling and analysis in all areas of neuroscience – from cells to networks. The frameworks of Dynamical Systems, Network Science, Scientific Computation, Information theory, and Stochastic processes are just a few areas that we are using to advance knowledge on synaptic plasticity, neural rhythm generation, memory and learning, decision making, and natural computation, hand-in-hand with colleagues from experimental and clinical neuroscience. Chairs: Cheng Ly, cly@vcu. edu; Wilten Nicola, wilten. nicola@ucalgary. ca Email: mathneurocollective@gmail. com Twitter: @MathNeuro_ 		    		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Mathematical Oncology - ONCO: 	  	    		    		    The Mathematical Oncology Subgroup is focused on integrating and applying mathematical and computational models to better understand and predict cancer initiation, progression, and treatment. Chairs: Alexander R. A. Anderson, Alexander. Anderson@moffitt. org; Heiko Enderling, Heiko. Enderling@moffitt. org Twitter: @mathonco Website: http://mathematical-oncology. org 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Methods for Biological Modeling - MFBM: 	  	    		    		    The Methods for Biological Modeling Subgroup brings together those interested in the development of novel application of methods (e. g. , numerical, analytical/theoretical, agent-based modeling, statistical, parameter estimation, identifiability, and machine learning) towards modeling biological systems. In applying standard methodologies, mathematical modelers encounter many challenges with simulating and analyzing biological systems, such as high dimensionality, computational complexity, observation or process error, model bias, and intra- and inter-individual heterogeneity. The objective of the Subgroup is to support and disseminate new advances in tackling these challenges to the Mathematical Biology community. Chairs: Kevin Flores, kbflores@ncsu. edu; Erica Rutter, erutter2@ucmerced. edu; John Nardini, jtnardin@ncsu. edu Twitter: @SMBmfbm 		    		   	  	  &lt;/br&gt;			&lt;/br&gt;	  Population Dynamics, Ecology &amp; Evolution - POPD: 	  	    						The Population Dynamics, Ecology and Evolution Subgroup aims to foster a discussion on theoretical, mathematical and empirical approaches to core topics in population dynamics, ecology and evolution. Chair: R. Christina Cobbold, Christina. Cobbold@glasgow. ac. uk Secretary: James Watmough, watmough@unb. ca Twitter: @smbpdee Google Group: smbpdee 	    																	&lt;/div&gt;	  &lt;/div&gt;		&lt;/div&gt;	&lt;/div&gt;&lt;/div&gt; "
    }, {
    "id": 23,
    "url": "http://localhost:4000/sunday/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Sunday, August 16		    		    {% include clock. html %}		    		    			    				        								&lt;/br&gt;								&lt;/br&gt;															 {% include schedule_menu. html %}				    			    		    	    	  			                    Pre-conference Mentoring Program        1:00-5:00pm:         																										 Pre-conference Mentoring Program Schedule									 									  			  							  										  			The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 							  										  			* all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 																											 												  												   Time												   Speaker												   Scheduled Event												  												 												 												  												   1:00pm												   Robert Smith?												   Introduction												  												  												   1:00–1:30pm												   Alexandra Hogan												   How to Attend a Virtual Conference												  												  												   1:30–2:00pm												   Rebecca Everett												   Things I Wish I'd Known as a Postdoc												  												  												   2:00–3:30pm												   Robert Smith?												   How to Get a Tenure-Track Job												  												  												   3:30–4:00pm												   Meghan Burke												   The Tenure Process												  												  												   4:00–5:00pm												   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   												   												   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)												  												 																						  																	 									 																						                                                                                                                                                                                                   					 		 {% include schedule_menu. html %}	 	"
    }, {
    "id": 24,
    "url": "http://localhost:4000/tags.html",
    "title": "Tags",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "          Tags          {% for tag in site. tags %}     {{ tag[0] }}:           {% assign pages_list = tag[1] %}    {% for post in pages_list %}    {% if post. title != null %}     {% if group == null or group == post. group %}           {% include main-loop-card. html %}     {% endif %}    {% endif %}    {% endfor %}    {% assign pages_list = nil %}    {% assign group = nil %}    {% endfor %}                  {% include sidebar-featured. html %}          "
    }, {
    "id": 25,
    "url": "http://localhost:4000/thursday/",
    "title": "SMB2020 Thursday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Thursday, August 20		    		    {% include clock. html %}		    		    			    				        																&lt;/br&gt;								&lt;/br&gt;															 {% include schedule_menu. html %}				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 				{% include coffee. html %}  	                Subgroup Keynote              8:30am:       Sue Ann Campbell, &lt;/br&gt;University of Waterloo       Neuroscience Subgroup           								            	       	                           	   	    Modulation of synchronization by a slowly varying M-current 	     The neurotransmitter acetylcholine has been shown to modulate the firing properties of several types of neurons through the down-regulation of voltage dependent potassium currents such as the muscarine-sensitive M-current. In the brain, levels of acetylcholine change with activity. For example, acetylcholine is higher during waking and REM sleep and low during slow wave sleep.  In this talk we show how the M-current affects the bifurcation structure of a generic conductance-based neural model and how this determines synchronization properties of the model. We then use phase-model analysis to study the effect of a slowly varying M-current on synchronization. This is joint work with Victoria Booth, Xueying Wang and Isam Al-Darbasah. 	                    			  													  		    		      		        Sub-group contributed talks (9:30-11:00am)		        {% include talks. html myday=page. day mygroup=page. ms1 mytime=page. ms1time %}		             		    		  									{% include morning_break. html %}					  		    		      		        Subgroup Keynote		        11:15am: 		        David Ho, &lt;/br&gt;Columbia University 		        Immunobiology &amp; Infection Subgroup 				      		      																     				       					       					     		             		    		    				   				    Potent Neutralizing Monoclonal Antibodies Directed to Multiple Epitopes on the SARS-CoV-2 Spike 				     					    The SARS-CoV-2 pandemic rages on with devasting consequences on human lives and the global economy.  The discovery and development of virus-neutralizing monoclonal antibodies could be one approach to treat or prevent infection by this novel coronavirus.  Here we report the isolation of 61 SARS-CoV-2-neutralizing monoclonal antibodies from 5 infected patients hospitalized with severe disease.  Among these are 19 antibodies that potently neutralized the authentic SARS-CoV-2 in vitro, 9 of which exhibited exquisite potency, with 50% virus-inhibitory concentrations of 0. 7 to 9 ng/mL.  Epitope mapping showed this collection of 19 antibodies to be about equally divided between those directed to the receptor-binding domain (RBD) and those to the N-terminal domain (NTD), indicating that both of these regions at the top of the viral spike are quite immunogenic.  In addition, two other powerful neutralizing antibodies recognized quaternary epitopes that are overlapping with the domains at the top of the spike.  Cyro-electron microscopy structures of one antibody targeting RBD, a second targeting NTD, and a third bridging two RBDs revealed recognition of the closed, “all RBD-down” conformation of the spike.  Several of these monoclonal antibodies are promising candidates for clinical development as potential therapeutic and/or prophylactic agents against SARS-CoV-2. 				     			    			   		    		    		  											  		    		      		        Subgroup Keynote		        		        12:15pm: 		        Adam Martin, &lt;/br&gt;Massachusetts Institute of Technology, @martinlabmit 		        Cell &amp; Developmental Biology Subgroup 				      		      																     				       					       					     		             		    		    		    		    				   				    Folding tissues across length scales: Cell-based origami 				     					    Throughout the lifespan of an organism, tissues are remodeled to shape organs and organisms and to maintain tissue integrity and homeostasis. Apical constriction is a ubiquitous cell shape change of epithelial tissues that promotes epithelia folding and cell/tissue invagination in a variety of contexts.  Apical constriction promotes tissue bending by changing the shape of constituent cells from a columnar-shape to a wedge-shape.  Drosophila gastrulation is one of the classic examples of apical constriction, where cells constrict to fold the primitive epithelial sheet and internalize cells that will give rise to internal organs. We have used a combination of imaging, experimental perturbation, and modeling, to determine how actomyosin organization promotes tissue folding.  The actin cytoskeleton is organized in both time and space to facilitate apical constriction.  We found that actomyosin contraction is pulsatile and requires dynamic regulation of upstream signaling processes. In addition, we found that actomyosin becomes organized into oriented fibers, which generates anisotropic tension that is critical for tissue shape.  Furthermore, connectivity within the network of actomyosin fibers is highly redundant, promoting the robustness of folding. 				     			    			   		    		    		  											  		    		      		        Career Fair		             		    		  											  		    		      		        Sub-group contributed talks (1:30-3:30pm)		        {% include talks. html myday=page. day mygroup=page. ct1 mytime=page. ct1time %}		             		    		  											  		    		      				      	Closing Plenary			      		        3:30pm: 		        Carl Bergstrom, &lt;/br&gt;University of Washington, @CT_Bergstrom 		      		      																     				       					       					     		             		    		    		    				   				    Misinfodemic 2020: How quantitative misinformation misleads the public about COVID-19, and what mathematical biologists can do about it. 				     					    Most people think they have pretty good bullshit detectors, at least when it comes to advertising hyperbole, weasel words, and politicians’ promises. Quantitative claims are harder: Numbers have the sense of objectivity and precision, and people feel less confident in challenging them. Yet with COVID-19, quantitative epidemiology is on the front page of every newspaper on every day of the week. Shortly before the pandemic broke out, I finished a book on how to spot and refute quantitative misinformation. Every lesson in the book has proven useful during the current pandemic. In this talk, I will present a tour of how misleading numbers, statistics, mathematics, and data graphics have muddied the social and traditional media streams that we all rely upon during COVID. I will give examples of deliberate disinformation, and examples of unintended misinformation around the pandemic. And I will explore how as citizens and as educators we all can promote data reasoning and quantitative literacy. 				     			    			   		    		    		    		  									&lt;/br&gt;			&lt;/br&gt;									 				 {% include schedule_menu. html %}			 																	      		      									Click to show abstracts						            "
    }, {
    "id": 26,
    "url": "http://localhost:4000/tuesday/",
    "title": "SMB2020 Tuesday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Tuesday, August 18		    		    {% include clock. html %}		    		    			    				        																&lt;/br&gt;								&lt;/br&gt;															 {% include schedule_menu. html %}				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 			{% include coffee. html %}  	                    Subgroup Keynote        8:30am:         Deirdre Hollingsworth, &lt;/br&gt;Oxford Big Data Institute, @DeirdreHoll         Mathematical Epidemiology Subgroup             												     		       			       			                        			  													  		    		      		        Sub-group minisymposia (9:30-11:00am)		        {% include talks. html myday=page. day mygroup=page. ms1 mytime=page. ms1time %}		             		    		  									{% include morning_break. html %}								  		    		      		        Sub-group minisymposia (11:15am)		        {% include talks. html myday=page. day mygroup=page. ms2 mytime=page. ms2time %}		             		    		  											  		    		      		        Subgroup Business Meeting		             		    		  											  		    		      		        Sub-group contributed talks (1:30-2:30pm)		        {% include talks. html myday=page. day mygroup=page. ct1 mytime=page. ct1time %}		             		    		  											  		    		      		        Poster Session (2:30-3:30pm)		        {% include poster_menu. html %}		             		    		  								  		    		      		        Sub-group Keynote		        3:30pm: 		        Andrea Bild, &lt;/br&gt;City of Hope, @AndreaBildLab 		        Mathematical Oncology Subgroup 		      		      																     				       					       					     		             		    		    		    				   				    Competitive and cooperative interactions between resistant and sensitive breast cancer cells during therapy 				     					    Our research focuses on estrogen receptor positive breast (ER+) cancer, the most commonly diagnosed and metastatic subtype. In a tumor, neighboring subclonal cancer cell populations interact both through competition for resources and through cooperation, by the production of growth promoting signals. Promoting competition can maintain drug sensitivity when resistant cells get outcompeted. Cooperation can promote cell survival under selective pressure, as well as potentially protect cellular populations that may otherwise be susceptible to treatment. Our research program integrates three key components to quantify cancer cell interactions: serial patient tumor samples molecularly profiled at the single cell level, a parallel in vitro model system where mechanisms can be tested through experimental manipulation and measured over time, and integrative dynamic mathematical models of the growth and interaction of subclonal cancer populations. By embedding the time course data and differential equation models into a Bayesian inference framework, we can estimate the strength of positive and negative interactions between subclonal populations during treatment and identify key mechanisms driving cell communication. Our analyses of patient derived samples further indicates that heterogeneous multi-clonal ER+ breast cell populations evolve to acquire alternative signaling states that drive cell cycle progression independent of estrogen availability, and that cells can cooperate to survive during therapy through the increased production and local sharing of growth promoting factors. 					    				     			    			   		    		    		  									{% include happy_hour. html %}						&lt;/br&gt;			&lt;/br&gt;									 				 {% include schedule_menu. html %}			 											      		      									Click to show abstracts						            "
    }, {
    "id": 27,
    "url": "http://localhost:4000/wednesday/",
    "title": "SMB2020 Wednesday Schedule",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    Schedule for Wednesday, August 19		    		    {% include clock. html %}		    		    			    				    								    																&lt;/br&gt;								&lt;/br&gt;															 {% include schedule_menu. html %}				    			    		    	    	  								Legend / Color-coding: 							MEPI (Mathematical Epidemiology)				CDEV (Cell and Developmental Biology)				EDUC (Education)				IMMU (Immunobiology and Infection)				NEUR (Mathematical Neuroscience)				 	 							&lt;/br&gt;				MFBM (Methods for Biological Modeling)				POPD (Population Dynamics, Ecology &amp; Evolution)     				ONCO (Mathematical Oncology)				OTHE (Other)				 			{% include coffee. html %}		                    Subgroup Keynote        8:30am:         Alicia Prieto Langrica, &lt;/br&gt;Youngstown State University, @AliciaPL25         Education Subgroup             												     		       			       			                             	   	    Lessons from the Pandemic: Why and how mathematical biologist educators are morally obligated to respond 	     The COVID-19 pandemic has made painfully clear the scientific illiteracy of the general population. Many are confused, rely on less than reliable sources for information, and reject scientific authorities equating the evolution of knowledge to lack of expertise. Given the current situation: what is our role and responsibility as educators to help our students be more informed, critical thinkers? How must we empower them through education to identify reliable sources and to better understand the scientific process? In this talk, we will reflect on ways in which we can respond to the current knowledge deficit, and why it is our duty to do so. 	                  	  		  													  		    		      		        Sub-group minisymposia (9:30-11:00am)		        {% include talks. html myday=page. day mygroup=page. ms1 mytime=page. ms1time %}		             		    		  									{% include morning_break. html %}								  		    		      		        Sub-group minisymposia (11:15am)		        {% include talks. html myday=page. day mygroup=page. ms2 mytime=page. ms2time %}		             		    		  											  		    		      		        SMB Business Meeting		             		    		  											  		    		      		        Sub-group contributed talks (1:30-2:30pm)		        {% include talks. html myday=page. day mygroup=page. ct1 mytime=page. ct1time %}		             		    		  											  		    		      		        Poster Session (2:30-3:30pm)		        {% include poster_menu. html %}		             		    		  								  		    		      			      		        Sub-group Keynote		        3:30pm: 		        Robert Insall, &lt;/br&gt;University of Galsgow, @robinsall 		        Methods for Biological Modeling Subgroup 		      		      																     				       					       					     		             		    		    		    				   				    Cells, cancer and mazes: Understanding what steers cells, using cycles of mathematics, modelling and experiments. 				     					    Chemotaxis is important to many of the things cells do in vivo, particularly during development and disease. It is also a favourite topic for mathematical biologists. However, despite decades of studies of how cells can interpret chemotactic gradients, there has been very little attention to how gradients form, and where the information comes from. We have studied a particularly interesting aspect of this problem - self-generated gradients, in which cells form their own gradients at the same time as interpreting them and migrating up them. Self-generated gradients are fascinating, and ideal for mathematical biologists - they are so dynamic that they are nearly impossible to measure, and they incorporate multiple levels of feedback. I will describe how we use modelling, mathematics and experiments on Dictyostelium and cancer cells to understand different self-generated gradients. In particular I will show how cells can solve arbitrarily complex mazes, escape from tumours, and use the information in a gradient to move down-gradient as well as up-gradient. 					    				     			    			   		    		    		  									{% include happy_hour. html %}						&lt;/br&gt;			&lt;/br&gt;									 				 {% include schedule_menu. html %}			 											      		      									Click to show abstracts						            "
    }, {
    "id": 28,
    "url": "http://localhost:4000/robots.txt",
    "title": "",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "      Sitemap: {{ “sitemap. xml”   absolute_url }}   "
    }, {
    "id": 29,
    "url": "http://localhost:4000/page2/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 30,
    "url": "http://localhost:4000/page3/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 31,
    "url": "http://localhost:4000/page4/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 32,
    "url": "http://localhost:4000/page5/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 33,
    "url": "http://localhost:4000/page6/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 34,
    "url": "http://localhost:4000/page7/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 35,
    "url": "http://localhost:4000/page8/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 36,
    "url": "http://localhost:4000/page9/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 37,
    "url": "http://localhost:4000/page10/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 38,
    "url": "http://localhost:4000/page11/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 39,
    "url": "http://localhost:4000/page12/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 40,
    "url": "http://localhost:4000/page13/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 41,
    "url": "http://localhost:4000/page14/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 42,
    "url": "http://localhost:4000/page15/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 43,
    "url": "http://localhost:4000/page16/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 44,
    "url": "http://localhost:4000/page17/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 45,
    "url": "http://localhost:4000/page18/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 46,
    "url": "http://localhost:4000/page19/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 47,
    "url": "http://localhost:4000/page20/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 48,
    "url": "http://localhost:4000/page21/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 49,
    "url": "http://localhost:4000/page22/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 50,
    "url": "http://localhost:4000/page23/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 51,
    "url": "http://localhost:4000/page24/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 52,
    "url": "http://localhost:4000/page25/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 53,
    "url": "http://localhost:4000/page26/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 54,
    "url": "http://localhost:4000/page27/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 55,
    "url": "http://localhost:4000/page28/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 56,
    "url": "http://localhost:4000/page29/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 57,
    "url": "http://localhost:4000/page30/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 58,
    "url": "http://localhost:4000/page31/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 59,
    "url": "http://localhost:4000/page32/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 60,
    "url": "http://localhost:4000/page33/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 61,
    "url": "http://localhost:4000/page34/",
    "title": "Society of Mathematical Biology",
    
    "abstract1": "",
    "abstract2": "",
    "abstract3": "",
    "abstract4": "",
    "abstract5": "",
    "abstract6": "",
    "abstract": "",
    "presenter": "",
    
    "title1": "",
    "title2": "",
    "title3": "",
    "title4": "",
    "title5": "",
    "title6": "",
    
    
    
    "body": "			  	    		    SMB 2020 Annual Meeting		    August 17 - 20, 2020: 		    			    								    			    		    		    			    			    	&lt;/br&gt;        The Society of Mathematical Biology will host its annual meeting online during the week of August 17 through 20, 2020. 	         Are you a member of SMB? If not please support mathematical biology and your career by joining the society today! 	  	    			    			   	    	  		  {% if page. url ==  /  %}                     SMB goes virtual        Join us for a society first - hosting the Society of Mathematical Biology conference entirely online.                  Through partnership with MathDept. org, we will utilize the following platforms to create an immersive conference experience. 																	Sococo for poster sessions, networking, and socializing									Zoom for plenary and minisymposium talks									Slack for general announcements and messaging								        							                              {% endif %}    	  		  	    		    {% if page. url ==  /  %}													{% else %}													{% endif %}		    	    		  					   	  					Conference Information: 	      				    					The joint Annual Meeting between the Society for Mathematical Biology and the European Society for Mathematical Biology at Heidelberg has been postponed. However, the Board of Directors of the Society for Mathematical Biology has decided to organize a virtual meeting, eSMB 2020, between August 17-20, 2020. The meeting will be held between 8. 30 am and 5. 00 pm US Eastern Time Zone to maximize the participation of people across the world.  					We are inviting all the SMB Subgroup to organize a Mini-symposium, Poster Section and invite a Keynote Speaker. Independent submission for oral or poster presentations will also be accepted. The eSMB 2020 organizing committee will identify the plenary speakers for the annual meeting, and will organize a Mini-symposium and a Poster Session outside the scope of the SMB Subgroups. 		   		            While COVID-19 continues spreading across the world, the Society for Mathematical Biology is keeping the health and safety of our members as our top priority. To reduce or slow the spread of infection, we will hold our 2020 Annual Meeting virtually between August 17-20, 2020. We acknowledge that many are facing challenging personal and/or professional circumstances, but we hope that our Virtual Annual Meeting will allow our community to stay connected, share research, and support each other.     	  			&lt;/br&gt;									    	  Conference Schedule: 	        	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 	        &lt;/br&gt;    &lt;/br&gt;	    {% include full_schedule. html %}	    							  	  &lt;/br&gt;	  &lt;/br&gt;	  &lt;/br&gt;	  	  Plenary Speakers: 	       	    		    			    				    					    				    						    						     								     								       								         Carl Bergstrom								       								     						       						       University of Washington, @CT_Bergstrom						       													&lt;/br&gt;						       						     						    					    					    								    						     								     								       								         Shayn Peirce-Cottler								       								     						       						       University of Virginia, @PeirceLab						       						       &lt;/br&gt;						       						     						    					    					   				    			     			    			  													&lt;/br&gt;			&lt;/br&gt;			&lt;/br&gt;									Subgroup Keynote Speakers: 	       				 				   				  			    				    					    						    						     								     								       								         Alicia Prieto Langarica								       								     						       						       Youngstown State University, @AliciaPL25						       						       &lt;/br&gt;						       						     						    						   					    						    						     								     								       								         David Ho								       								     						       						       Columbia University						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  &lt;/br&gt;				  				   				  			    				    					    				    						    						     								     								       								         Robert Insall								       								     						       						       University of Galsgow, @robinsall										       						       &lt;/br&gt;						       						     						    						   						   					    						    						    						     								     								       																Priyanga Amarasekare								       								     						       						       University of California Los Angeles						       						       &lt;/br&gt;						       						       						     						    						   					   				    			     			    			   			   &lt;/br&gt;				    				   				  			    				    					    						    						     								     								       																Andrea Bild 								       						       								     						       						       City of Hope, @AndreaBildLab						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Adam Martin								       								     						       						       Massachusetts Institute of Technology, @martinlabmit						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    			   			   			    				  			    				    					    						    						     								     								       																Sue Ann Campbell								       						       								     						       						       University of Waterloo						       						       &lt;/br&gt;						       						     						    				    						    						    						     								     								       																Deirdre Hollingsworth								       								     						       						       Oxford Big Data Institute, @DeirdreHoll						       						       &lt;/br&gt;						       						     						    						    						   					   				    			     			    				    				  		    	    					    	  	  &lt;/br&gt;			Logistics &amp; Registration: 	      				At the same time, we are excited that the 2020 Annual Meeting will be the inaugural SMB Virtual Meeting. Our communities have long been leaders in sharing research online, and this opportunity is another significant step in that direction.  The Society has selected a third-party virtual meeting platform (Sococo and MathDept. org) that seamlessly supports live video presentations of sessions, recording of and technical support for those sessions, pre-recorded talks, poster sessions, networking and socializing. The site also will allow small groups of participants to interact in real-time directly with each other. 				The Society for Mathematical Biology is waiving registration fees for our first virtual annual meeting.  You must still register for the meeting through the registration website, for a $25 fee. You can support the dissemination of research and education at the interface between the mathematical and biological sciences by a small gift of $25 to partially cover the third-party virtual platform cost for your participation. You can make a bigger impact for young mathematical biologists with two other registration options. You can give $50 to support your participation and the participation of a young mathematical biologist at this meeting. Alternatively, you can give $100 to partially support a society travel award to help a student participate at our 2021 Annual Meeting. 				The COVID-19 pandemic is seriously impacting the finances of research and higher education, and academic societies. We need your help, because every young mathematical biologist requires our support and the opportunity to participate in our annual meetings. Thank you for considering joining us in this Virtual Annual Meeting, and your support for mathematical biology. 				If you are interested in attending certain talks, but do not want to participate in the full conference, please indicate so in the registration form. We will ensure that you receive Zoom links for conference talks, but will not register you in the Sococo platform. Since SMB will be paying approximately $25 for each registrant's Sococo access, it is helpful for us to know ahead of time if you do not intend to use the platform. 				If you are submitting an abstract that is not affiliated with an SMB subgroup, please do so in the registration form. If you would like to submit an abstract affiliated with an SMB subgroup, please contact your subgroup chair(s).         	  	  	  &lt;/br&gt;	  	      	    		    						 						  Read the conference announcement						  by Sandy Anderson, SMB President						  Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. . . 						  								Read more. . . 	 						 											        	    		    						 						  Register for SMB 2020						  $25 registration fee						  To register for the SMB Conference, please fill out the form below. Registrants can be 1) presenters or 2) participants. Registration fees are $25. 						  								Registration	 						 											    	  						&lt;/br&gt;	  &lt;/br&gt;	  	  Pre-conference Mentoring Program: 	       					               The mentoring workshop will be held the day before the e-conference on Sunday August 16, 2020 from 1:00 - 5:00 PM for the US Eastern Time Zone. If you have not yet registered for the mentoring workshop, you may do so by contacting John Nardini at jtnardin@ncsu. edu.  This workshop will consist of the following speakers and talks: 	    * all times are listed in US Eastern Time Zone. This schedule is tentative and subject to change. 										 					  					   Time					   Speaker					   Scheduled Event					  					 					 					  					   1:00pm					   Robert Smith?					   Introduction					  					  					   1:00–1:30pm					   Alexandra Hogan					   How to Attend a Virtual Conference					  					  					   1:30–2:00pm					   Rebecca Everett					   Things I Wish I'd Known as a Postdoc					  					  					   2:00–3:30pm					   Robert Smith?					   How to Get a Tenure-Track Job					  					  					   3:30–4:00pm					   Meghan Burke					   The Tenure Process					  					  					   4:00–5:00pm					   Helen Moore (Applied Biomath) &lt;/br&gt; Marc Strauss (Springer-Nature) &lt;/br&gt; Susan Massey (Systems Oncology) &lt;/br&gt;  Cheryl Zapata-Allegro (BioFire Diagnostics)				   					   					   Panel discussion &lt;/br&gt; (Moderated by Rebecca Everett)					  					 															         	  			"
    }, {
    "id": 62,
    "url": "http://localhost:4000/Bejamin-J-Jessie/",
    "title": "Respiratory Syncytia Virus",
    "body": "2020/08/06 - "
    }, {
    "id": 63,
    "url": "http://localhost:4000/Michael-Pablo/",
    "title": "Early-phase decoupling between population mobility and death rates",
    "body": "2020/08/05 - "
    }, {
    "id": 64,
    "url": "http://localhost:4000/William-(Bill)-Sherwin/",
    "title": "Monitoring and Forecasting Diversity: Entropy Unifies Molecules and Ecosystems",
    "body": "2020/08/04 - "
    }, {
    "id": 65,
    "url": "http://localhost:4000/Vivian-Dornelas/",
    "title": "Impact of the landscape heterogeneity on the spatial organization of a single-species population",
    "body": "2020/08/04 - "
    }, {
    "id": 66,
    "url": "http://localhost:4000/Vandana-Revathi-Venkateswaran/",
    "title": "The effect of parental investment on immunocompetence and sexual immune dimorphism",
    "body": "2020/08/04 - "
    }, {
    "id": 67,
    "url": "http://localhost:4000/Tatiana-Yakushkina/",
    "title": "Modified Replicator Systems with Ecological DIversification",
    "body": "2020/08/04 - "
    }, {
    "id": 68,
    "url": "http://localhost:4000/Renato-Antunes-Costa-de-Andrade/",
    "title": "Persistence of intraguild predation in a 1D finite domain using variational approximations",
    "body": "2020/08/04 - "
    }, {
    "id": 69,
    "url": "http://localhost:4000/PIJUSH-PANDAY/",
    "title": "Dynamical behaviour of a stage-structured predator-prey model by incorporating cost and benefits of group defense",
    "body": "2020/08/04 - "
    }, {
    "id": 70,
    "url": "http://localhost:4000/Mauro-Mobilia/",
    "title": "Population Dynamics in a Changing Environment: Random versus Periodic Switching",
    "body": "2020/08/04 - "
    }, {
    "id": 71,
    "url": "http://localhost:4000/Luana-Tais-Bassani/",
    "title": "Basic sanitation's role over breeding sites maintenance for Aedes aegypti development during rain absence according to a fuzzy rule-based system",
    "body": "2020/08/04 - "
    }, {
    "id": 72,
    "url": "http://localhost:4000/Lee-Altenberg/",
    "title": "Spectral Graph Theory in the Analysis of Biological Evolution",
    "body": "2020/08/04 - "
    }, {
    "id": 73,
    "url": "http://localhost:4000/Jack-M-Hughes/",
    "title": "Thermodynamic Inhibition in a Biofilm Reactor with Suspended Bacteria",
    "body": "2020/08/04 - "
    }, {
    "id": 74,
    "url": "http://localhost:4000/Enrico-Sandro-Colizzi/",
    "title": "Evolution of multicellularity driven by emergent collective migration",
    "body": "2020/08/04 - "
    }, {
    "id": 75,
    "url": "http://localhost:4000/Bhawna-Malik/",
    "title": "Stochastic extinction of drug-resistant strains: Modelling the role of socio-economic factors in the pattern of extinction",
    "body": "2020/08/04 - "
    }, {
    "id": 76,
    "url": "http://localhost:4000/Anna-H-Sisk/",
    "title": "Linking Immuno-Epidemiology Principles to Violence",
    "body": "2020/08/04 - "
    }, {
    "id": 77,
    "url": "http://localhost:4000/Vitor-de-Oliveira-Sudbrack/",
    "title": "Population dynamics in highly fragmented landscapes",
    "body": "2020/08/03 - "
    }, {
    "id": 78,
    "url": "http://localhost:4000/Vahini-Reddy-Nareddy/",
    "title": "Representing spatial ecological oscillators by dynamical Ising model with memory",
    "body": "2020/08/03 - "
    }, {
    "id": 79,
    "url": "http://localhost:4000/So-Nakashima/",
    "title": "Lineage EM Algorithm for Inferring Latent States from Cellular Lineage Trees",
    "body": "2020/08/03 - "
    }, {
    "id": 80,
    "url": "http://localhost:4000/Silas-Poloni/",
    "title": "Intraguild Predation in Periodic Habitats",
    "body": "2020/08/03 - "
    }, {
    "id": 81,
    "url": "http://localhost:4000/Shota-Shibasaki/",
    "title": "Fluctuating environments affect the strength of species interactions and diversity in microbial communities similarly",
    "body": "2020/08/03 - "
    }, {
    "id": 82,
    "url": "http://localhost:4000/Russell-Milne/",
    "title": "Effects of Variation in Fishing Rate and Nutrient Loading on Coral Reef Health with Implications for Marine Protected Area Design",
    "body": "2020/08/03 - "
    }, {
    "id": 83,
    "url": "http://localhost:4000/Rom%C3%A1n-Zapi%C3%A9n-Campos/",
    "title": "When timescales meet: Microbiome dynamics is influenced by hosts’ life-history.",
    "body": "2020/08/03 - "
    }, {
    "id": 84,
    "url": "http://localhost:4000/Renier-G-Mendoza/",
    "title": "Explicit solution of an age-structured model using a generalized Lambert W function",
    "body": "2020/08/03 - "
    }, {
    "id": 85,
    "url": "http://localhost:4000/Pawe%C5%82-Klimasara/",
    "title": "A Savanna Dynamics Model",
    "body": "2020/08/03 - "
    }, {
    "id": 86,
    "url": "http://localhost:4000/Lucy-Lansch-Justen/",
    "title": "Evolutionary rescue from mutational meltdown",
    "body": "2020/08/03 - "
    }, {
    "id": 87,
    "url": "http://localhost:4000/Laurie-Balstad/",
    "title": "Parasite intensity influences evolution of migratory behavior via migratory escape",
    "body": "2020/08/03 - "
    }, {
    "id": 88,
    "url": "http://localhost:4000/Lauren-Mossman-Kylie-Landa/",
    "title": "Phage-antibiotic synergy inhibited by temperate and chronic virus competition",
    "body": "2020/08/03 - "
    }, {
    "id": 89,
    "url": "http://localhost:4000/Kathyrn-R-Fair/",
    "title": "Spatial Structure in Protected Forest-Grassland Mosaics: Exploring Futures Under Climate Change",
    "body": "2020/08/03 - "
    }, {
    "id": 90,
    "url": "http://localhost:4000/Jorge-Arroyo-Esquivel/",
    "title": "Spatial dynamics and spread of ecosystem engineers",
    "body": "2020/08/03 - "
    }, {
    "id": 91,
    "url": "http://localhost:4000/Fernando-Luiz-Pio-dos-Santos/",
    "title": "Investigation of the Aedes spread using a reaction-diffusion mathematical model",
    "body": "2020/08/03 - "
    }, {
    "id": 92,
    "url": "http://localhost:4000/Femke-Thon/",
    "title": "Modeling intraspecific chemodiversity - theory and first results",
    "body": "2020/08/03 - "
    }, {
    "id": 93,
    "url": "http://localhost:4000/Felipe-E-M-Campos/",
    "title": "A modelling approach to study landscape effects on abundance from patch demographic processes",
    "body": "2020/08/03 - "
    }, {
    "id": 94,
    "url": "http://localhost:4000/Easton-R.-White/",
    "title": "Catastrophes and socio-ecological systems",
    "body": "2020/08/03 - "
    }, {
    "id": 95,
    "url": "http://localhost:4000/Dominic-Brass/",
    "title": "Phenotypic plasticity as a cause and consequence of population dynamics",
    "body": "2020/08/03 - "
    }, {
    "id": 96,
    "url": "http://localhost:4000/Connah-Griffith-Michael-Johnson/",
    "title": "Developing a computational tool for multiscale simulations of chemically coupled cell populations",
    "body": "2020/08/03 - "
    }, {
    "id": 97,
    "url": "http://localhost:4000/Christopher-Carlson/",
    "title": "Partner Specificity in Mutualisms",
    "body": "2020/08/03 - "
    }, {
    "id": 98,
    "url": "http://localhost:4000/Caren-Barcel%C3%B3/",
    "title": "Projecting the time scale of initial increase in fishery yield after implementation of marine protected areas",
    "body": "2020/08/03 - "
    }, {
    "id": 99,
    "url": "http://localhost:4000/Bryce-Morsky/",
    "title": "Evolution of contribution timing in public goods games",
    "body": "2020/08/03 - "
    }, {
    "id": 100,
    "url": "http://localhost:4000/Bj%C3%B6rn-Vessman/",
    "title": "A theoretical investigation of artificial community selection methods",
    "body": "2020/08/03 - "
    }, {
    "id": 101,
    "url": "http://localhost:4000/Antonela-Marozzi/",
    "title": "An alternative method for calf density and recruitment estimation using pregnancy on wild guanacos (Lama guanicoe).",
    "body": "2020/08/03 - "
    }, {
    "id": 102,
    "url": "http://localhost:4000/Angela-Lynn-Peace/",
    "title": "A Simple model of pathogen-mediated nutrient dynamics",
    "body": "2020/08/03 - "
    }, {
    "id": 103,
    "url": "http://localhost:4000/Adelle-Coster/",
    "title": "The Distance Between: Stochastic Models of Cellular Protein Transport",
    "body": "2020/08/03 - "
    }, {
    "id": 104,
    "url": "http://localhost:4000/Zviiteyi-Chazuka/",
    "title": "A mathematical model for in host dynamics of an immune evading virus",
    "body": "2020/08/02 - "
    }, {
    "id": 105,
    "url": "http://localhost:4000/Zeinab-Tajik-Mansoury/",
    "title": "Opioid Addiction Affects Neuronal Synchronization in the Hippocampus: A Computational Model",
    "body": "2020/08/02 - "
    }, {
    "id": 106,
    "url": "http://localhost:4000/Yeeren-Low/",
    "title": "Data-driven modeling of T cell morphodynamics during migration",
    "body": "2020/08/02 - "
    }, {
    "id": 107,
    "url": "http://localhost:4000/Yahe-Yu/",
    "title": "Finding Optimal STI Strategies for HIV Using Fitted Q Algorithm with XGBoost Regression",
    "body": "2020/08/02 - "
    }, {
    "id": 108,
    "url": "http://localhost:4000/Yafei-Wang/",
    "title": "Multicellular simulation in cancer treatment with nanoparticles",
    "body": "2020/08/02 - "
    }, {
    "id": 109,
    "url": "http://localhost:4000/Xinzhe-Zuo/",
    "title": "Backtracking in RNA polymerization",
    "body": "2020/08/02 - "
    }, {
    "id": 110,
    "url": "http://localhost:4000/William-D-Martinson/",
    "title": "Comparative analysis of continuum angiogenesis models",
    "body": "2020/08/02 - "
    }, {
    "id": 111,
    "url": "http://localhost:4000/Will-Leone/",
    "title": "Understanding factors contributing to bacterial burden in granulomas of Mycobacterium tuberculosis-infected monkeys",
    "body": "2020/08/02 - "
    }, {
    "id": 112,
    "url": "http://localhost:4000/Vojtech-Kumpost/",
    "title": "Elucidating Mechanisms Underlying Experimentally Induced Changes in the Zebrafish Circadian Clock",
    "body": "2020/08/02 - "
    }, {
    "id": 113,
    "url": "http://localhost:4000/Viktor-Zenkov/",
    "title": "A minority of liver-resident CD8 T cells searching for Plasmodium-infected hepatocytes demonstrate difficult-to-detect attraction",
    "body": "2020/08/02 - "
    }, {
    "id": 114,
    "url": "http://localhost:4000/Vehpi-Yildirim/",
    "title": "Mathematical Modeling Postprandial Lipoprotein Metabolism and Investigating the Effects of the Bariatric Surgery",
    "body": "2020/08/02 - "
    }, {
    "id": 115,
    "url": "http://localhost:4000/Vedang-Narain/",
    "title": "A boids-based model of collective cell migration in wound healing",
    "body": "2020/08/02 - "
    }, {
    "id": 116,
    "url": "http://localhost:4000/Tiffany-Leung/",
    "title": "Evaluating the effectiveness of social distancing interventions: delaying the epidemic or flattening the curve?",
    "body": "2020/08/02 - "
    }, {
    "id": 117,
    "url": "http://localhost:4000/Thomas-N-Vilches/",
    "title": "Assessing the effects of the diagnostic methods on the schistosomiasis dynamics",
    "body": "2020/08/02 - "
    }, {
    "id": 118,
    "url": "http://localhost:4000/Thi-Minh-Thao-Le/",
    "title": "Quasi-neutral dynamics in a co-infection system with N strains",
    "body": "2020/08/02 - "
    }, {
    "id": 119,
    "url": "http://localhost:4000/Suzanne-Sindi/",
    "title": "Multiscale Modeling of Protein Aggregation",
    "body": "2020/08/02 - "
    }, {
    "id": 120,
    "url": "http://localhost:4000/Sungyoung-Shin/",
    "title": "A mathematical model of GβL (de)ubiquitination switch uncovers biphasic response within the PI3K/mTOR signalling network",
    "body": "2020/08/02 - "
    }, {
    "id": 121,
    "url": "http://localhost:4000/Subbalakshmi-A-R/",
    "title": "NFATc acts as a non-canonical phenotypic stability factor for a hybrid epithelial/mesenchymal phenotype",
    "body": "2020/08/02 - "
    }, {
    "id": 122,
    "url": "http://localhost:4000/Stephen-Y-Zhang/",
    "title": "Inference of stochastic cellular dynamics from time-series data using optimal transport",
    "body": "2020/08/02 - "
    }, {
    "id": 123,
    "url": "http://localhost:4000/Stefano-Pasetto/",
    "title": "Bayesian framework for tumor board decision making",
    "body": "2020/08/02 - "
    }, {
    "id": 124,
    "url": "http://localhost:4000/Soumen-Bera/",
    "title": "Biphasic Adaptive Activity of Plant Nitrate Transporter NRT1.1",
    "body": "2020/08/02 - "
    }, {
    "id": 125,
    "url": "http://localhost:4000/Simona-Catozzi/",
    "title": "Predicted ‘wiring landscape’ of Ras network in 29 human tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 126,
    "url": "http://localhost:4000/Simon-Mitchell/",
    "title": "A Systems Biology Approach Predicts Distinct Roles for NFkB Subunits cRel and RelA in DLBCL",
    "body": "2020/08/02 - "
    }, {
    "id": 127,
    "url": "http://localhost:4000/Shohel-Ahmed/",
    "title": "Global Asymptotic Stability for a Diffusive Opportunistic Diseases Model",
    "body": "2020/08/02 - "
    }, {
    "id": 128,
    "url": "http://localhost:4000/Shelby-M-Scott/",
    "title": "COVID and Crime: Analysis of Crime Dynamics Amidst Social Distancing Protocols",
    "body": "2020/08/02 - "
    }, {
    "id": 129,
    "url": "http://localhost:4000/Shawn-Ryan/",
    "title": "Role of Hydrodynamics in Chemotaxis of Bacterial Populations",
    "body": "2020/08/02 - "
    }, {
    "id": 130,
    "url": "http://localhost:4000/Shaon-C-Chakrabarti/",
    "title": "Why cousins are more similar than mother daughters: implications for cell cycle control",
    "body": "2020/08/02 - "
    }, {
    "id": 131,
    "url": "http://localhost:4000/Shadi-Esmaeili-Wellman/",
    "title": "Density Dependent Resource Budget Model for Alternate Bearing",
    "body": "2020/08/02 - "
    }, {
    "id": 132,
    "url": "http://localhost:4000/Seokjoo-Chae/",
    "title": "The data-based inference method reveals the network structure of the SCN",
    "body": "2020/08/02 - "
    }, {
    "id": 133,
    "url": "http://localhost:4000/Sebastian-Ruhle/",
    "title": "Analysis of cell contact inhibition during growth of epithelial tissue",
    "body": "2020/08/02 - "
    }, {
    "id": 134,
    "url": "http://localhost:4000/Sayan-Biswas/",
    "title": "Predicting mitochondrial functional state : An imaging based SVM tool",
    "body": "2020/08/02 - "
    }, {
    "id": 135,
    "url": "http://localhost:4000/Sarah-MacQueen/",
    "title": "Modelling the effects of site constancy in bumble bees",
    "body": "2020/08/02 - "
    }, {
    "id": 136,
    "url": "http://localhost:4000/Sara-Sommariva/",
    "title": "Mathematical model of loss and gain of function mutations in a chemical reaction network for colorectal cancer cells.",
    "body": "2020/08/02 - "
    }, {
    "id": 137,
    "url": "http://localhost:4000/Ryan-Schenck/",
    "title": "The Tick-Tock of the Molecular Clock: Random methylation state changes inform homeostasis in the intestinal crypt",
    "body": "2020/08/02 - "
    }, {
    "id": 138,
    "url": "http://localhost:4000/Ryan-Murphy/",
    "title": "Mechanical cell competition in heterogeneous epithelial tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 139,
    "url": "http://localhost:4000/Roxana-L%C3%B3pez-Cruz/",
    "title": "A Coupled Mathematical Model Introducing Biological Control of Dengue",
    "body": "2020/08/02 - "
    }, {
    "id": 140,
    "url": "http://localhost:4000/Rodrigo-Garcia/",
    "title": "Small-worldness favours network inference in synthetic neural networks",
    "body": "2020/08/02 - "
    }, {
    "id": 141,
    "url": "http://localhost:4000/Robyn-Shuttleworth/",
    "title": "Application of a cell-dense triphasic model to cryoprotectant equilibration",
    "body": "2020/08/02 - "
    }, {
    "id": 142,
    "url": "http://localhost:4000/Rifaldy-Fajar/",
    "title": "Analysis of Mathematical Model on the Development of Tumor Cells after Drug Therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 143,
    "url": "http://localhost:4000/Rhudaina-Z.-Mohammad/",
    "title": "Cellular patterning in sensory systems: An interface evolution problem",
    "body": "2020/08/02 - "
    }, {
    "id": 144,
    "url": "http://localhost:4000/Rey-Audie-Escosio/",
    "title": "Parameter Estimation of the Fitzhugh-Nagumo Model via a Perturbed Accelerated Gradient Descent Algorithm with an n-Dimensional Golden Section Search Method",
    "body": "2020/08/02 - "
    }, {
    "id": 145,
    "url": "http://localhost:4000/Renee-Dale/",
    "title": "Using a trait-based dynamic mathematical framework to investigate the relationship between phenotypic dynamical parameters and the plant genome",
    "body": "2020/08/02 - "
    }, {
    "id": 146,
    "url": "http://localhost:4000/Reginaldo-Jos%C3%A9-da-Silva/",
    "title": "Acoustic measures for voices in the classification of Parkinson's Disease",
    "body": "2020/08/02 - "
    }, {
    "id": 147,
    "url": "http://localhost:4000/Rebecca-Sanft/",
    "title": "Exploring Mathematical Modeling in Biology Through Case Studies and Experimental Activities",
    "body": "2020/08/02 - "
    }, {
    "id": 148,
    "url": "http://localhost:4000/Rebecca-H-Chisholm/",
    "title": "A model of population dynamics with complex household structure and mobility: implications for transmission and control of communicable diseases",
    "body": "2020/08/02 - "
    }, {
    "id": 149,
    "url": "http://localhost:4000/Rebecca-Bekker/",
    "title": "Immunological Consequences of Uniform vs Spatially Fractionated Radiotherapy",
    "body": "2020/08/02 - "
    }, {
    "id": 150,
    "url": "http://localhost:4000/Ranjini-Bhattacharya/",
    "title": "Interpreting the Evolutionary Games Played in the NSCLC Microenvironment",
    "body": "2020/08/02 - "
    }, {
    "id": 151,
    "url": "http://localhost:4000/Rahul-Bishnoi/",
    "title": "Eliminating the Need for User-Definition of Gap Penalties using Linear Programming",
    "body": "2020/08/02 - "
    }, {
    "id": 152,
    "url": "http://localhost:4000/Rahma-Jerbi/",
    "title": "modélisation mathématique",
    "body": "2020/08/02 - "
    }, {
    "id": 153,
    "url": "http://localhost:4000/Rafael-Ramon-Bravo/",
    "title": "Cancer Crusade: Crowdsourcing Adaptive Therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 154,
    "url": "http://localhost:4000/Quan-Anh-Hoang/",
    "title": "Analyzing Bacterial Motility Near a Smooth Surface",
    "body": "2020/08/02 - "
    }, {
    "id": 155,
    "url": "http://localhost:4000/Priyom-Adhyapok/",
    "title": "Modeling liver injury progression and repair",
    "body": "2020/08/02 - "
    }, {
    "id": 156,
    "url": "http://localhost:4000/Prashant-K-Srivastava/",
    "title": "The impact of information and saturated treatment with time delay in an Infectious disease model",
    "body": "2020/08/02 - "
    }, {
    "id": 157,
    "url": "http://localhost:4000/Praneeth-Reddy-Sudalagunta/",
    "title": "A pharmacodynamic model of clinical synergy in multiple myeloma",
    "body": "2020/08/02 - "
    }, {
    "id": 158,
    "url": "http://localhost:4000/Pranav-Khade/",
    "title": "Using alpha shapes to characterize protein flexibility",
    "body": "2020/08/02 - "
    }, {
    "id": 159,
    "url": "http://localhost:4000/Prachi-Bisht/",
    "title": "Interface growth driven by a single active particle",
    "body": "2020/08/02 - "
    }, {
    "id": 160,
    "url": "http://localhost:4000/Pooja-Dnyane/",
    "title": "Boolean Model for Melanogenesis",
    "body": "2020/08/02 - "
    }, {
    "id": 161,
    "url": "http://localhost:4000/Phillip-Rossbach/",
    "title": "Determination of the critical adhesion parameter for the sorting behavior of a cell system with several cell types using statistical learning methods",
    "body": "2020/08/02 - "
    }, {
    "id": 162,
    "url": "http://localhost:4000/Phebe-M.-A-Havor/",
    "title": "Dynamics of disease models with self-diffusion: Case study of Cholera in Ghana",
    "body": "2020/08/02 - "
    }, {
    "id": 163,
    "url": "http://localhost:4000/Perminov-Dmitrievich-Valeriy/",
    "title": "Agent-based models for influenza epidemic dynamics and its decision-making capability",
    "body": "2020/08/02 - "
    }, {
    "id": 164,
    "url": "http://localhost:4000/Pedro-Vilanova/",
    "title": "Synchronization in Stochastic Oscillators Subject to Common Extrinsic Noise",
    "body": "2020/08/02 - "
    }, {
    "id": 165,
    "url": "http://localhost:4000/Paul-J-Hurtado/",
    "title": "Reproduction Numbers for ODE Models of Arbitrary Finite Dimension: An Application of the Generalized Linear Chain Trick",
    "body": "2020/08/02 - "
    }, {
    "id": 166,
    "url": "http://localhost:4000/Paul-D-Alexander/",
    "title": "Treatment of Viral Co-Infections",
    "body": "2020/08/02 - "
    }, {
    "id": 167,
    "url": "http://localhost:4000/Paul-A-Roberts/",
    "title": "Using mathematics to investigate the mechanisms behind vision loss",
    "body": "2020/08/02 - "
    }, {
    "id": 168,
    "url": "http://localhost:4000/Pau-Capera-Aragones/",
    "title": "A mechanistic partial integro-differential equations model for bumblebee foraging behaviour",
    "body": "2020/08/02 - "
    }, {
    "id": 169,
    "url": "http://localhost:4000/Patrick-Gelbach/",
    "title": "Metabolomics and Mechanistic Kinetic Modeling Reveal Mechanisms Driving Intracellular Metabolism and Insulin Secretion in Pancreatic Beta Cells",
    "body": "2020/08/02 - "
    }, {
    "id": 170,
    "url": "http://localhost:4000/Pappu-Kumar/",
    "title": "Theoretical investigation of non-equilibrium bio-heat transfer during thermal therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 171,
    "url": "http://localhost:4000/Pamela-Kim-N-Salonga/",
    "title": "A mathematical model of the dynamics of lymphatic filariasis in Caraga Region, The Philippines",
    "body": "2020/08/02 - "
    }, {
    "id": 172,
    "url": "http://localhost:4000/Oke-I-Segun/",
    "title": "Mathematical model for the estrogen paradox in breast cancer treatment",
    "body": "2020/08/02 - "
    }, {
    "id": 173,
    "url": "http://localhost:4000/Noemi-Andor/",
    "title": "Invasion of homogeneous and polyploid populations in nutrient-limiting environments",
    "body": "2020/08/02 - "
    }, {
    "id": 174,
    "url": "http://localhost:4000/Nikolaos-Dimitriou/",
    "title": "Modelling geometric patterns of cancer progression",
    "body": "2020/08/02 - "
    }, {
    "id": 175,
    "url": "http://localhost:4000/Nicole-Althermeler/",
    "title": "Ancestral lines under selection for multiple sites: Pruning the Ancestral Selection Graph",
    "body": "2020/08/02 - "
    }, {
    "id": 176,
    "url": "http://localhost:4000/Nara-Yoon/",
    "title": "Modeling collaterally sensitive drug cycles: shaping heterogeneity to allow adaptive therapy",
    "body": "2020/08/02 - "
    }, {
    "id": 177,
    "url": "http://localhost:4000/Muhammad-Said/",
    "title": "Sensitivity and stability analysis of an Ebola Virus disease and GB virus C co-infection.",
    "body": "2020/08/02 - "
    }, {
    "id": 178,
    "url": "http://localhost:4000/Muhammad-Humayun-Kabir/",
    "title": "Model-aided Understanding of the Outbreak of COVID-19 in Bangladesh",
    "body": "2020/08/02 - "
    }, {
    "id": 179,
    "url": "http://localhost:4000/Mubasher-Rashid-Rather/",
    "title": "Organization of biogeochemical nitrogen pathways with switch-like adjustment in fluctuating soil redox conditions",
    "body": "2020/08/02 - "
    }, {
    "id": 180,
    "url": "http://localhost:4000/Mohsin-Saleet-Jafri/",
    "title": "Predicting Drug Resistance by Applying Machine Learning to Molecular Simulation",
    "body": "2020/08/02 - "
    }, {
    "id": 181,
    "url": "http://localhost:4000/Mohammad-U-Zahid/",
    "title": "Forecasting Individual Patient Response to Radiotherapy with a Dynamic Carrying Capacity Model in Head and Neck Cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 182,
    "url": "http://localhost:4000/Moacyr-A-H-B-Silva/",
    "title": "A death-based mathematical model of the ongoing COVID-19 pandemic",
    "body": "2020/08/02 - "
    }, {
    "id": 183,
    "url": "http://localhost:4000/Min-Song/",
    "title": "Model Predicts Distinct Mechanisms of Endothelial Cell Growth Upon the Stimulation of FGF and VEGF",
    "body": "2020/08/02 - "
    }, {
    "id": 184,
    "url": "http://localhost:4000/Miller-Orlando-Cer%C3%B3n/",
    "title": "Mathematical model with carriers and general non-linear incidence rate",
    "body": "2020/08/02 - "
    }, {
    "id": 185,
    "url": "http://localhost:4000/Milad-Ghomlaghi/",
    "title": "Akt regulates PIP3 production by PI3K to form a potent negative feedback loop",
    "body": "2020/08/02 - "
    }, {
    "id": 186,
    "url": "http://localhost:4000/Mikahl-Banwarth-Kuhn/",
    "title": "Quantifying the Biophysical Impact of Budding Cell Division on the Spatial Organization of Growing Yeast Colonies",
    "body": "2020/08/02 - "
    }, {
    "id": 187,
    "url": "http://localhost:4000/Michelle-Przedborski/",
    "title": "A combined systems biology and machine learning approach to study patient response to anti-PD1 immunotherapy",
    "body": "2020/08/02 - "
    }, {
    "id": 188,
    "url": "http://localhost:4000/Masud-M-A/",
    "title": "",
    "body": "2020/08/02 - "
    }, {
    "id": 189,
    "url": "http://localhost:4000/Marvin-Fritz/",
    "title": "On the modelling and analysis of tumor growth with phase-field equations of Cahn-Hilliard type",
    "body": "2020/08/02 - "
    }, {
    "id": 190,
    "url": "http://localhost:4000/Martin-Lopez-Garcia/",
    "title": "Exact approaches for the analysis of stochastic epidemic processes on small networks",
    "body": "2020/08/02 - "
    }, {
    "id": 191,
    "url": "http://localhost:4000/Marta-Helena-Oliveira/",
    "title": "MATHEMATICAL MODELLING OF THE INFLAMMATORY PHASE OF SKIN WOUND HEALING IN RATS",
    "body": "2020/08/02 - "
    }, {
    "id": 192,
    "url": "http://localhost:4000/Mark-Robertson-Tessi/",
    "title": "Immune predation promotes aggressive metabolic phenotypes in a context-dependent manner",
    "body": "2020/08/02 - "
    }, {
    "id": 193,
    "url": "http://localhost:4000/Marilin-Nathalya-Guerrero-Laos/",
    "title": "Approach and analysis of a mathematical model in ordinary differential equations, applied to HIV transmission, considering protection strategies.",
    "body": "2020/08/02 - "
    }, {
    "id": 194,
    "url": "http://localhost:4000/Marc-Pereyra/",
    "title": "3D Particle image velocimetry (PIV) analysis of cellular migration during embryonic development",
    "body": "2020/08/02 - "
    }, {
    "id": 195,
    "url": "http://localhost:4000/Lutz-Brusch/",
    "title": "Morpheus: A user-friendly simulation framework for multi-cellular systems biology",
    "body": "2020/08/02 - "
    }, {
    "id": 196,
    "url": "http://localhost:4000/Luke-Andrejek/",
    "title": "A Robust Mathematical Model of Adaxial-Abaxial Patterning",
    "body": "2020/08/02 - "
    }, {
    "id": 197,
    "url": "http://localhost:4000/Lucas-Boettcher/",
    "title": "Unifying continuous, discrete, and hybrid susceptible-infected-recovered processes on networks",
    "body": "2020/08/02 - "
    }, {
    "id": 198,
    "url": "http://localhost:4000/Lucas-Barberis/",
    "title": "Modeling Helps Understand the Influence of Substrate on Tumorsphere Growth",
    "body": "2020/08/02 - "
    }, {
    "id": 199,
    "url": "http://localhost:4000/Linnea-Franssen/",
    "title": "Linnea Franssen",
    "body": "2020/08/02 - layout: single-posterauthor: esmbday: “All”group: “poster”title: “A Mathematical Framework for Modelling the Metastatic Spread of Cancer”subgroup: “ONCO”presenter: “Linnea Franssen”institution: “Roche Pharmaceutical Research and Early Development (Translational Modeling and Simulation)”abstract: “Cancer is a complex disease that starts with mutations of key genes in one cell or a small group of cells at a primary site in the body. If these cancer cells continue to grow successfully and, at some later stage, invade the surrounding tissue and acquire a vascular network, they can spread to distant secondary sites in the body. This process, known as metastatic spread, is responsible for around 90— "
    }, {
    "id": 200,
    "url": "http://localhost:4000/Leili-Shahriyari/",
    "title": "A path toward personalized cancer treatments",
    "body": "2020/08/02 - "
    }, {
    "id": 201,
    "url": "http://localhost:4000/Lee-Curtin/",
    "title": "Lacunarity and fractal dimension as prognostic biomarkers in glioblastoma",
    "body": "2020/08/02 - "
    }, {
    "id": 202,
    "url": "http://localhost:4000/Lan-K-Nguyen/",
    "title": "Integrative mathematical modelling unveils hidden mechanism of resistance to PI3K  inhibition and identifies new effective combination therapies for breast cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 203,
    "url": "http://localhost:4000/Koushik-Garain/",
    "title": "Treatment control is one of the most important factor to reduce the spread of COVID-19 epidemic",
    "body": "2020/08/02 - "
    }, {
    "id": 204,
    "url": "http://localhost:4000/Khem-Raj-Ghusinga/",
    "title": "Molecular switch architecture determines response properties of signaling pathways",
    "body": "2020/08/02 - "
    }, {
    "id": 205,
    "url": "http://localhost:4000/Kento-Nakamura/",
    "title": "Optimality of the sensory system of Escherichia coli",
    "body": "2020/08/02 - "
    }, {
    "id": 206,
    "url": "http://localhost:4000/Kelly-R-Buch/",
    "title": "Mathematical Model of Basal Sprout Production in Response to Laurel Wilt",
    "body": "2020/08/02 - "
    }, {
    "id": 207,
    "url": "http://localhost:4000/Kellen-Myers/",
    "title": "Resource Limitations and Household Economics in Outbreaks",
    "body": "2020/08/02 - "
    }, {
    "id": 208,
    "url": "http://localhost:4000/Keertana-Yalamanchili/",
    "title": "Assessing the Efficacy of Various NOX Enzyme Inhibitors as Potential Treatments for Ischemic Stroke in Silico",
    "body": "2020/08/02 - "
    }, {
    "id": 209,
    "url": "http://localhost:4000/Karina-Vilches/",
    "title": "A mathematical approach for visualizing cell migration during tumor progression",
    "body": "2020/08/02 - "
    }, {
    "id": 210,
    "url": "http://localhost:4000/Junho-Lee/",
    "title": "Synergistic Effects of Bortezomib-OV Therapy and  Anti-Invasive Strategies in Glioblastoma:  A Mathematical Model",
    "body": "2020/08/02 - "
    }, {
    "id": 211,
    "url": "http://localhost:4000/Jungmin-Han/",
    "title": "Missing Data Imputation and Gene Network Inference in Single Cell Analysis",
    "body": "2020/08/02 - "
    }, {
    "id": 212,
    "url": "http://localhost:4000/Juliano-Ferrari-Gianlupi/",
    "title": "Multiscale spatiotemporal modeling of acute primary viral infection and immune response in epithelial tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 213,
    "url": "http://localhost:4000/Juliane-Schr%C3%B6ter/",
    "title": "A theoretical model for the natural course of HIV infection in infants",
    "body": "2020/08/02 - "
    }, {
    "id": 214,
    "url": "http://localhost:4000/Juan-Calvo/",
    "title": "The initial-boundary value problem for the Lifshitz-Slyozov system with inflow boundary conditions: Analysis and applications",
    "body": "2020/08/02 - "
    }, {
    "id": 215,
    "url": "http://localhost:4000/Joshua-A-Bull/",
    "title": "Description of immune cell infiltration in solid tumours using spatial statistics and topological data analysis",
    "body": "2020/08/02 - "
    }, {
    "id": 216,
    "url": "http://localhost:4000/Josephine-Naa-Ayeley-Tetteh/",
    "title": "Switching strategy for mitigation against bacterial resistance",
    "body": "2020/08/02 - "
    }, {
    "id": 217,
    "url": "http://localhost:4000/Joseph-S-Abrams/",
    "title": "A CELL-BASED MODEL OF INTERCELLULAR MECHANICS DURING CPA EQUILIBRATION IN PREANTRAL OVARIAN FOLLICLES",
    "body": "2020/08/02 - "
    }, {
    "id": 218,
    "url": "http://localhost:4000/Jordy-Jose-Cevallos-Chavez/",
    "title": "Mobility impact in the spreading of COVID-19 in Ecuador ",
    "body": "2020/08/02 - "
    }, {
    "id": 219,
    "url": "http://localhost:4000/Jonggul-Lee/",
    "title": "Machine Learning for Risk Prediction of Highly Pathogenic Avian In uenza in the Republic of Korea",
    "body": "2020/08/02 - "
    }, {
    "id": 220,
    "url": "http://localhost:4000/Jonas-Knoch/",
    "title": "A mathematical model of HIF-1 regulated cellular energy metabolism",
    "body": "2020/08/02 - "
    }, {
    "id": 221,
    "url": "http://localhost:4000/John-R-Jungck/",
    "title": "Ethnomathematics: Art, Culture and Social Justice",
    "body": "2020/08/02 - "
    }, {
    "id": 222,
    "url": "http://localhost:4000/John-Metzcar/",
    "title": "Mathematical modeling of leader-follower cell invasion of tumor-associated stroma using a novel extracellular matrix model",
    "body": "2020/08/02 - "
    }, {
    "id": 223,
    "url": "http://localhost:4000/Jim-Greene/",
    "title": "A novel COVID-19 model reveals unexpected consequences of social distancing strategies",
    "body": "2020/08/02 - "
    }, {
    "id": 224,
    "url": "http://localhost:4000/Jessica-Wellington/",
    "title": "A Mathematical Model to Investigate Iron Allocation in Plants",
    "body": "2020/08/02 - "
    }, {
    "id": 225,
    "url": "http://localhost:4000/Jessica-Conway/",
    "title": "Stochastic time-inhomogeneous HIV dynamics following treatment suspension",
    "body": "2020/08/02 - "
    }, {
    "id": 226,
    "url": "http://localhost:4000/Jayrah-Bena-E-Ri%C3%B1on/",
    "title": "A Mathematical Model and Optimal Control of Schistosomiasis in Agusan del Sur, Philippines",
    "body": "2020/08/02 - "
    }, {
    "id": 227,
    "url": "http://localhost:4000/Javier-C-Urcuyo/",
    "title": "Understanding glioblastoma-macrophage interactions through radiomics, transcriptome sequencing, and mathematical modeling",
    "body": "2020/08/02 - "
    }, {
    "id": 228,
    "url": "http://localhost:4000/Jared-Barber/",
    "title": "Mathematical and mechanical models of cell motion",
    "body": "2020/08/02 - "
    }, {
    "id": 229,
    "url": "http://localhost:4000/Janina-Hesse/",
    "title": "Diurnal variation in gene expression and sports performance: a matter of timing?",
    "body": "2020/08/02 - "
    }, {
    "id": 230,
    "url": "http://localhost:4000/Jairo-G-Silva/",
    "title": "Mathematical Models About Radioactive Iodine-Refractory Differentiated Thyroid Cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 231,
    "url": "http://localhost:4000/Jahedi-Sana/",
    "title": "When the best pandemic models are the simplest",
    "body": "2020/08/02 - "
    }, {
    "id": 232,
    "url": "http://localhost:4000/Jackelyn-M-Kembro/",
    "title": "Accelerometers as a tool to characterize reproductive behavior within social groups in long term experiments: the case of the Japanese Quill",
    "body": "2020/08/02 - "
    }, {
    "id": 233,
    "url": "http://localhost:4000/Iulia-Martina-Bulai/",
    "title": "Modeling COVID-19 considering asymptomatic cases and avoid contacts",
    "body": "2020/08/02 - "
    }, {
    "id": 234,
    "url": "http://localhost:4000/Ielyaas-Cloete/",
    "title": "Not all hormone receptors are created equal",
    "body": "2020/08/02 - "
    }, {
    "id": 235,
    "url": "http://localhost:4000/Himanshu-Aggarwal/",
    "title": "Impact of temperature change on Malaria incidence in high altitude regions of India",
    "body": "2020/08/02 - "
    }, {
    "id": 236,
    "url": "http://localhost:4000/Heber-Rocha/",
    "title": "Qualitative study of cell migration associated with hypoxia in the dynamics of tumor growth",
    "body": "2020/08/02 - "
    }, {
    "id": 237,
    "url": "http://localhost:4000/Haryana-Thomas/",
    "title": "Modeling Cellular Signaling and Mesangial Fibrosis during Diabetic Kidney",
    "body": "2020/08/02 - "
    }, {
    "id": 238,
    "url": "http://localhost:4000/Harsimran-Kaur/",
    "title": "Computational modeling of mechano-metabolic adaptation to a stiff microenvironment in Cancer cells",
    "body": "2020/08/02 - "
    }, {
    "id": 239,
    "url": "http://localhost:4000/Hana-Dobrovolny/",
    "title": "SARS-CoV-2 coinfections: Implications for the second wave",
    "body": "2020/08/02 - "
    }, {
    "id": 240,
    "url": "http://localhost:4000/Gustavo-J-Sibona/",
    "title": "SIRS dynamics on a diffusive agent system",
    "body": "2020/08/02 - "
    }, {
    "id": 241,
    "url": "http://localhost:4000/Giulio-Bonifazi/",
    "title": "Predicting excitotoxicity at the onset of Alzheimer’s disease by a model of Aβ-dependent trafficking of astrocytic glutamate transporters",
    "body": "2020/08/02 - "
    }, {
    "id": 242,
    "url": "http://localhost:4000/Giulia-Laura-Celora/",
    "title": "Analysis of the dynamics of tumour cells along a stemness axis under different oxygen conditions",
    "body": "2020/08/02 - "
    }, {
    "id": 243,
    "url": "http://localhost:4000/Gilberto-C-Gonzalez-Parra/",
    "title": "Forecasting cases of RSV using artificial neural networks and mechanistic models",
    "body": "2020/08/02 - "
    }, {
    "id": 244,
    "url": "http://localhost:4000/Gess-Iraji/",
    "title": "Modeling Clogging of Red Blood Cells in Microfluidic Devices with Simple Geometry",
    "body": "2020/08/02 - "
    }, {
    "id": 245,
    "url": "http://localhost:4000/Gemma-Massonis/",
    "title": "Structural Identifiability and Observability of Compartmental Models of the COVID-19 Pandemic",
    "body": "2020/08/02 - "
    }, {
    "id": 246,
    "url": "http://localhost:4000/Furkan-Kurtoglu/",
    "title": "Temporospatial Modeling of CRC-CAF relation using molecular and tissue level",
    "body": "2020/08/02 - "
    }, {
    "id": 247,
    "url": "http://localhost:4000/Florian-Franke/",
    "title": "Is Cell segregation just like oil and water: A phase field approach",
    "body": "2020/08/02 - "
    }, {
    "id": 248,
    "url": "http://localhost:4000/Fillipe-H-Georgiou/",
    "title": "Modelling Locust Foraging And Its Effect On Swarm Formation",
    "body": "2020/08/02 - "
    }, {
    "id": 249,
    "url": "http://localhost:4000/Felipe-Rubio/",
    "title": "Evaluating the role of memory B and T cells during secondary dengue infection",
    "body": "2020/08/02 - "
    }, {
    "id": 250,
    "url": "http://localhost:4000/Fatima-Sulayman/",
    "title": "Dynamical systems analysis of tuberculosis with the impact of transmission rate and vaccination.",
    "body": "2020/08/02 - "
    }, {
    "id": 251,
    "url": "http://localhost:4000/Euan-Smithers/",
    "title": "How do plant leaf pavement cells form puzzle piece like shapes? Using a multi-model approach to simulate chemical and visco-elastic mechanical processes and experimental methods to discover their secrets.",
    "body": "2020/08/02 - "
    }, {
    "id": 252,
    "url": "http://localhost:4000/Estadilla-S-Carlo-Delfin/",
    "title": "Optimal Strategies for Mitigating the COVID-19 Epidemic in the Philippines",
    "body": "2020/08/02 - "
    }, {
    "id": 253,
    "url": "http://localhost:4000/Erika-Tsingos/",
    "title": "A computational tool to optimise experiments for estimating cell cycle parameters",
    "body": "2020/08/02 - "
    }, {
    "id": 254,
    "url": "http://localhost:4000/Erik-Amezquita/",
    "title": "Quantifying barley morphology using Euler characteristic curves",
    "body": "2020/08/02 - "
    }, {
    "id": 255,
    "url": "http://localhost:4000/Erica-M.-Rutter/",
    "title": "Predicting Bladder Pressure and Contractions with Dense Time-Series Data",
    "body": "2020/08/02 - "
    }, {
    "id": 256,
    "url": "http://localhost:4000/Emma-Southall/",
    "title": "Identifying indicators of critical transitions in epidemiological data",
    "body": "2020/08/02 - "
    }, {
    "id": 257,
    "url": "http://localhost:4000/Emma-L-Fairbanks/",
    "title": "Re-parameterisation of a mathematical model of AHSV using data from literature",
    "body": "2020/08/02 - "
    }, {
    "id": 258,
    "url": "http://localhost:4000/Emily-Zhang/",
    "title": "Personalized Time Series Forecasting of Blood Glucose Levels",
    "body": "2020/08/02 - "
    }, {
    "id": 259,
    "url": "http://localhost:4000/Emilia-Kozlowska/",
    "title": "Mathematical modeling of palliative treatment in non-small cell lung cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 260,
    "url": "http://localhost:4000/Eman-Alwani/",
    "title": "Mathematical Analysis of Feedback Requirements for Planar Polarisation in the Fly Wing",
    "body": "2020/08/02 - "
    }, {
    "id": 261,
    "url": "http://localhost:4000/Elizabeth-A-Fedak/",
    "title": "Getting mixed messages: How p53 controls its dynamics to interpret variable upstream signals",
    "body": "2020/08/02 - "
    }, {
    "id": 262,
    "url": "http://localhost:4000/Editha-Jose/",
    "title": "Complex Balanced Equilibria of Weakly Reversible Poly-PL Kinetic Systems and Evolutionary Games",
    "body": "2020/08/02 - "
    }, {
    "id": 263,
    "url": "http://localhost:4000/Donggu-Lee/",
    "title": "Role of OCT1 in regulation of miR-451-LKB1-AMPK-OCT1-mTOR core signaling network and cell invasion in glioblastoma.",
    "body": "2020/08/02 - "
    }, {
    "id": 264,
    "url": "http://localhost:4000/Dominic-Olver/",
    "title": "Time Dependent Osmotic Damage in Sea Urchin Oocytes",
    "body": "2020/08/02 - "
    }, {
    "id": 265,
    "url": "http://localhost:4000/Domenic-PJ-Germano/",
    "title": "Towards a realistic 3D deformable model of tissues",
    "body": "2020/08/02 - "
    }, {
    "id": 266,
    "url": "http://localhost:4000/Divyoj-Singh/",
    "title": "Emergent Properties of the HNF4α-PPARγ Network May Drive Consequent Phenotypic Plasticity in NAFLD",
    "body": "2020/08/02 - "
    }, {
    "id": 267,
    "url": "http://localhost:4000/Diego-Samuel-Rodrigues/",
    "title": "Mathematical modeling of pharmacokinetic profiles of magnetic nanoparticles acquired by multichannel alternating current biosusceptrometry",
    "body": "2020/08/02 - "
    }, {
    "id": 268,
    "url": "http://localhost:4000/Dhiraj-Kumar-Das/",
    "title": "Influence of the smear-microscopy in global dynamics of tuberculosis transmission",
    "body": "2020/08/02 - "
    }, {
    "id": 269,
    "url": "http://localhost:4000/Dhananjay-Bhaskar/",
    "title": "Quantifying cellular (re)-organization in 3-D cancer models using persistent homology",
    "body": "2020/08/02 - "
    }, {
    "id": 270,
    "url": "http://localhost:4000/Derek-Park/",
    "title": "Synergizing chemotherapy with immune and evolutionary tradeoffs: Searching for Goldilocks",
    "body": "2020/08/02 - "
    }, {
    "id": 271,
    "url": "http://localhost:4000/Denis-H%C3%BCnniger/",
    "title": "Effects of tumor-originating niches on intra-tumor heterogeneity",
    "body": "2020/08/02 - "
    }, {
    "id": 272,
    "url": "http://localhost:4000/Deena-Schmidt/",
    "title": "Building age-structured network models from interaction data",
    "body": "2020/08/02 - "
    }, {
    "id": 273,
    "url": "http://localhost:4000/Debasmita-Mukherjee/",
    "title": "Atherosclerosis: A Mathematical Overview",
    "body": "2020/08/02 - "
    }, {
    "id": 274,
    "url": "http://localhost:4000/David-Gurarie/",
    "title": "Individual-based modeling of Covid-19 in local community settings",
    "body": "2020/08/02 - "
    }, {
    "id": 275,
    "url": "http://localhost:4000/David-Cheek/",
    "title": "Genetic composition of an exponentially growing cell population",
    "body": "2020/08/02 - "
    }, {
    "id": 276,
    "url": "http://localhost:4000/Darwin-Bandoy/",
    "title": "Pandemic dynamics of COVID-19 using epidemic stage, instantaneous reproductive number and pathogen genome identity (GENI) score: modeling molecular epidemiology",
    "body": "2020/08/02 - "
    }, {
    "id": 277,
    "url": "http://localhost:4000/Daniel-Tudor/",
    "title": "Probing immune cell wound recruitment signals using Bayesian inference and random walk models",
    "body": "2020/08/02 - "
    }, {
    "id": 278,
    "url": "http://localhost:4000/Daniel-Koch/",
    "title": "In signal transduction, all you need are oligomers!  Dynamic signal encoding, homeostasis, bistability and more",
    "body": "2020/08/02 - "
    }, {
    "id": 279,
    "url": "http://localhost:4000/Daniel-J-Glazar/",
    "title": "Tumor growth and inhibition model predicts progression in recurrent high-grade glioma",
    "body": "2020/08/02 - "
    }, {
    "id": 280,
    "url": "http://localhost:4000/Daniel-Abler/",
    "title": "Tumor Growth and Biomechanics – Challenges & Opportunities",
    "body": "2020/08/02 - "
    }, {
    "id": 281,
    "url": "http://localhost:4000/DaeWook-Kim/",
    "title": "Systems pharmacology model reveals the sources of the inter- and intraspecies variability in drug efficacy",
    "body": "2020/08/02 - "
    }, {
    "id": 282,
    "url": "http://localhost:4000/Cristian-C.-E.-Espitia/",
    "title": "A mathematical model of HIV/AIDS Spread in human population, the triangle transmission case.",
    "body": "2020/08/02 - "
    }, {
    "id": 283,
    "url": "http://localhost:4000/Cristeta-Jamilla/",
    "title": "Inverse Problems for Neutral Delay Differential Equations using Nature-Inspired Optimization Algorithms",
    "body": "2020/08/02 - "
    }, {
    "id": 284,
    "url": "http://localhost:4000/Cody-FitzGerald/",
    "title": "Red light and the dormancy-germination decision in Arabidopsis seeds",
    "body": "2020/08/02 - "
    }, {
    "id": 285,
    "url": "http://localhost:4000/Chiara-Villa/",
    "title": "Mechanical pattern formation in biological tissue: Relax and go with the (viscous) flow",
    "body": "2020/08/02 - "
    }, {
    "id": 286,
    "url": "http://localhost:4000/Chay-Paterson/",
    "title": "Cancer incidence as a form of convergent evolution",
    "body": "2020/08/02 - "
    }, {
    "id": 287,
    "url": "http://localhost:4000/Chandler-Gatenbee/",
    "title": "Immune escape at the onset of human colorectal cancer",
    "body": "2020/08/02 - "
    }, {
    "id": 288,
    "url": "http://localhost:4000/Caroline-Franco/",
    "title": "Modelling non-pharmaceutical interventions to mitigate COVID-19 in Sao Paulo",
    "body": "2020/08/02 - "
    }, {
    "id": 289,
    "url": "http://localhost:4000/Carlos-Enrique-Bustamante-Orellana/",
    "title": "Modeling and Preparedness: The Transmission Dynamics of COVID19 Outbreak in Provinces of Ecuador",
    "body": "2020/08/02 - "
    }, {
    "id": 290,
    "url": "http://localhost:4000/Carley-Cook/",
    "title": "Mathematical modeling of the relationship between T cell produced Wnt10b and the bone remodeling cycle",
    "body": "2020/08/02 - "
    }, {
    "id": 291,
    "url": "http://localhost:4000/Cara-Sulyok/",
    "title": "A Mathematical Framework to Augment the Q-MARSH Score in the Diagnosis of Celiac Disease",
    "body": "2020/08/02 - "
    }, {
    "id": 292,
    "url": "http://localhost:4000/Camile-FD-Kunz/",
    "title": "Chemotaxis impact on pattern formation",
    "body": "2020/08/02 - "
    }, {
    "id": 293,
    "url": "http://localhost:4000/Cameline-Nafula/",
    "title": "Modelling the spatio-temporal risk of measles outbreaks and options for their control in Kenya",
    "body": "2020/08/02 - "
    }, {
    "id": 294,
    "url": "http://localhost:4000/Calmelet-Jeanne-Colette/",
    "title": "Surfactant Driven Flows in Respiratory and Ocular Regions",
    "body": "2020/08/02 - "
    }, {
    "id": 295,
    "url": "http://localhost:4000/Bin-Zhang/",
    "title": "The Ecology of Collective Cancer Invasion: An Evolutionary Game Theory Model",
    "body": "2020/08/02 - "
    }, {
    "id": 296,
    "url": "http://localhost:4000/Bhattacharyya-Samit/",
    "title": "Evolutionary game models to explain the potential interactions among Adverse events, Vaccine scares and Individual vaccination choice",
    "body": "2020/08/02 - "
    }, {
    "id": 297,
    "url": "http://localhost:4000/Beryl-O-Musundi/",
    "title": "An immuno-epidemiological model linking within-host and between-host dynamics of cholera",
    "body": "2020/08/02 - "
    }, {
    "id": 298,
    "url": "http://localhost:4000/Bertin-Hoffman/",
    "title": "The initial engraftment of tumor cells is critical for the future growth pattern",
    "body": "2020/08/02 - "
    }, {
    "id": 299,
    "url": "http://localhost:4000/Benedikt-Sabass/",
    "title": "Molecular mechanics of type-IV pili driven migration of P. aeruginosa",
    "body": "2020/08/02 - "
    }, {
    "id": 300,
    "url": "http://localhost:4000/Baylor-Fain/",
    "title": "Modeling the impact of inoculum dose and transmission mode on viral infection with an agent-based model",
    "body": "2020/08/02 - "
    }, {
    "id": 301,
    "url": "http://localhost:4000/Baeckkyoung-Sung/",
    "title": "Mathematical modeling of a temperature-sensitive and tissue-mimicking gel matrix",
    "body": "2020/08/02 - "
    }, {
    "id": 302,
    "url": "http://localhost:4000/Augustine-Okebunor-Okolie/",
    "title": "Exact and approximate formulas for contact tracing on random trees",
    "body": "2020/08/02 - "
    }, {
    "id": 303,
    "url": "http://localhost:4000/Atchuta-Srinivas-Duddu/",
    "title": "Multistability in cellular differentiation enabled by a mutually antagonistic triad",
    "body": "2020/08/02 - "
    }, {
    "id": 304,
    "url": "http://localhost:4000/Atanaska-Dobreva/",
    "title": "Mathematical modeling of photoreceptor metabolism",
    "body": "2020/08/02 - "
    }, {
    "id": 305,
    "url": "http://localhost:4000/Artur-C%C3%A9sar-Fassoni/",
    "title": "Mathematical modeling and methodology to identify patient-specific immunological landscapes in CML treatment using TKI cessation and dose reduction data",
    "body": "2020/08/02 - "
    }, {
    "id": 306,
    "url": "http://localhost:4000/Anuraag-Bukkuri/",
    "title": "GLUT1 Production in Cancer Cells: A Tragedy of the Commons",
    "body": "2020/08/02 - "
    }, {
    "id": 307,
    "url": "http://localhost:4000/Aniruddha-Deka/",
    "title": "Optimal public health strategy during an influenza outbreak",
    "body": "2020/08/02 - "
    }, {
    "id": 308,
    "url": "http://localhost:4000/Angelo-J-Zorn/",
    "title": "An Epidemic Mobility Model with Symptomatic and Asymptomatic Individuals Allowing Variation of Contact Rates Between Individuals and Across Regions",
    "body": "2020/08/02 - "
    }, {
    "id": 309,
    "url": "http://localhost:4000/Angela-Michelle-Jarrett/",
    "title": "Modeling of the spatio-temporal evolution of tumor vasculature to improve predictions of breast cancer response to neoadjuvant chemotherapy regimens",
    "body": "2020/08/02 - "
    }, {
    "id": 310,
    "url": "http://localhost:4000/Angela-L-Moreno/",
    "title": "ART Neural Networks in the Classification of Spinal Pathologies",
    "body": "2020/08/02 - "
    }, {
    "id": 311,
    "url": "http://localhost:4000/Andreas-Buttenschoen/",
    "title": "Spatio-Temporal Heterogeneities in a Mechano-Chemical Model of Collective Cell Migration",
    "body": "2020/08/02 - "
    }, {
    "id": 312,
    "url": "http://localhost:4000/Anca-Radulescu/",
    "title": "Effects of local mutations in quadratic iterations",
    "body": "2020/08/02 - "
    }, {
    "id": 313,
    "url": "http://localhost:4000/Ananta-Samrajya-Shri-Kishore-Hari/",
    "title": "Identifying inhibitors of epithelial–mesenchymal plasticity using a network topology-based approach",
    "body": "2020/08/02 - "
    }, {
    "id": 314,
    "url": "http://localhost:4000/Amanda-Alexander/",
    "title": "Mathematical modeling of regulatory T cell mechanisms in experimental autoimmune encephalomyelitis",
    "body": "2020/08/02 - "
    }, {
    "id": 315,
    "url": "http://localhost:4000/Alexis-Erich-S-Almocera/",
    "title": "Modeling the viral infectious disease from infection to epidemic",
    "body": "2020/08/02 - "
    }, {
    "id": 316,
    "url": "http://localhost:4000/Alexey-A.-Tokarev/",
    "title": "Velocity-Amplitude relationship in the Gray-Scott autowave model in isolated conditions",
    "body": "2020/08/02 - "
    }, {
    "id": 317,
    "url": "http://localhost:4000/Alexander-P-Hoover/",
    "title": "Exploring Neuromechanical Resonance in Jellyfish Locomotion",
    "body": "2020/08/02 - "
    }, {
    "id": 318,
    "url": "http://localhost:4000/Akhil-Kumar-Srivastav/",
    "title": "Mathematical Modeling of Malaria with Saturated Treatment-A Case Study of India",
    "body": "2020/08/02 - "
    }, {
    "id": 319,
    "url": "http://localhost:4000/Adrianne-Jenner/",
    "title": "Exploring the impact of intratumoral heterogeneity on oncolytic virotherapy using agent-based modelling",
    "body": "2020/08/02 - "
    }, {
    "id": 320,
    "url": "http://localhost:4000/Adriana-Zanca/",
    "title": "Multicellular model of collective cell migration with an irregular free boundary",
    "body": "2020/08/02 - "
    }, {
    "id": 321,
    "url": "http://localhost:4000/Adnan-A-Khan/",
    "title": "Understanding the COVID-19 Outbreak in Pakistan",
    "body": "2020/08/02 - "
    }, {
    "id": 322,
    "url": "http://localhost:4000/Abdel-H.-Halloway/",
    "title": "Non-Equilibrial Dynamics in Under-Saturated Communities",
    "body": "2020/08/02 - "
    }, {
    "id": 323,
    "url": "http://localhost:4000/POPD_Wednesday_MS2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 324,
    "url": "http://localhost:4000/POPD_Wednesday_MS1/",
    "title": "Emergence and Stability of Population Structure and Biological Aggregates Across Scales",
    "body": "2020/08/01 - "
    }, {
    "id": 325,
    "url": "http://localhost:4000/POPD_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 326,
    "url": "http://localhost:4000/POPD_Tuesday_MS2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 327,
    "url": "http://localhost:4000/POPD_Tuesday_MS1/",
    "title": "Mathematical models of evolutionary rescue",
    "body": "2020/08/01 - "
    }, {
    "id": 328,
    "url": "http://localhost:4000/POPD_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 329,
    "url": "http://localhost:4000/POPD_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 330,
    "url": "http://localhost:4000/POPD_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 331,
    "url": "http://localhost:4000/POPD_Monday_MS2/",
    "title": "",
    "body": "2020/08/01 - "
    }, {
    "id": 332,
    "url": "http://localhost:4000/POPD_Monday_MS1/",
    "title": "Stochastic Population Dynamics",
    "body": "2020/08/01 - "
    }, {
    "id": 333,
    "url": "http://localhost:4000/POPD_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 334,
    "url": "http://localhost:4000/OTHE_Wednesday_MS2/",
    "title": "Flow and Transport in Bacterial Biofilms, Part II",
    "body": "2020/08/01 - "
    }, {
    "id": 335,
    "url": "http://localhost:4000/OTHE_Wednesday_MS1/",
    "title": "Flow and Transport in Complex Tissues, Part I",
    "body": "2020/08/01 - "
    }, {
    "id": 336,
    "url": "http://localhost:4000/OTHE_Wednesday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 337,
    "url": "http://localhost:4000/OTHE_Tuesday_MS1/",
    "title": "Eco-evolutionary dynamics across scales of organisation",
    "body": "2020/08/01 - "
    }, {
    "id": 338,
    "url": "http://localhost:4000/OTHE_Tuesday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 339,
    "url": "http://localhost:4000/OTHE_Thursday_CT2/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 340,
    "url": "http://localhost:4000/OTHE_Thursday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 341,
    "url": "http://localhost:4000/OTHE_Monday_CT1/",
    "title": "Other Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 342,
    "url": "http://localhost:4000/ONCO_Wednesday_MS2/",
    "title": "Applications and challenges of using quantitative imaging data for biologically-based mathematical oncology",
    "body": "2020/08/01 - "
    }, {
    "id": 343,
    "url": "http://localhost:4000/ONCO_Wednesday_MS1/",
    "title": "Differentiation and stemmness, in cell migration, cancer invasion, and development",
    "body": "2020/08/01 - "
    }, {
    "id": 344,
    "url": "http://localhost:4000/ONCO_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 345,
    "url": "http://localhost:4000/ONCO_Tuesday_MS2/",
    "title": "Data-based modeling in cancer research with focus on clinical applications",
    "body": "2020/08/01 - "
    }, {
    "id": 346,
    "url": "http://localhost:4000/ONCO_Tuesday_MS1/",
    "title": "Multiscale models of cancer heterogeneity, with applications in drug development and precision medicine",
    "body": "2020/08/01 - "
    }, {
    "id": 347,
    "url": "http://localhost:4000/ONCO_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 348,
    "url": "http://localhost:4000/ONCO_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 349,
    "url": "http://localhost:4000/ONCO_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 350,
    "url": "http://localhost:4000/ONCO_Monday_MS2/",
    "title": "Frontiers in MathOnco, Part 2",
    "body": "2020/08/01 - "
    }, {
    "id": 351,
    "url": "http://localhost:4000/ONCO_Monday_MS1/",
    "title": "Frontiers in MathOnco, Part 1",
    "body": "2020/08/01 - "
    }, {
    "id": 352,
    "url": "http://localhost:4000/ONCO_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 353,
    "url": "http://localhost:4000/NEUR_Wednesday_MS2/",
    "title": "Stochastic and Mean-Field Analysis in Neuroscience",
    "body": "2020/08/01 - "
    }, {
    "id": 354,
    "url": "http://localhost:4000/NEUR_Wednesday_MS1/",
    "title": "Dynamics and Noise in Neural Networks",
    "body": "2020/08/01 - "
    }, {
    "id": 355,
    "url": "http://localhost:4000/NEUR_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 356,
    "url": "http://localhost:4000/NEUR_Thursday_CT1/",
    "title": "Rhythms, Sleep, Reward, Vision",
    "body": "2020/08/01 - "
    }, {
    "id": 357,
    "url": "http://localhost:4000/MFBM_Wednesday_MS2/",
    "title": "Stochastic methods for epidemiology and biochemical reaction networks",
    "body": "2020/08/01 - "
    }, {
    "id": 358,
    "url": "http://localhost:4000/MFBM_Wednesday_MS1/",
    "title": "Stochastic methods for epidemiology and biochemical reaction networks",
    "body": "2020/08/01 - "
    }, {
    "id": 359,
    "url": "http://localhost:4000/MFBM_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 360,
    "url": "http://localhost:4000/MFBM_Tuesday_MS2/",
    "title": "Topological and network analyses for data",
    "body": "2020/08/01 - "
    }, {
    "id": 361,
    "url": "http://localhost:4000/MFBM_Tuesday_MS1/",
    "title": "Topological and network analyses for data",
    "body": "2020/08/01 - "
    }, {
    "id": 362,
    "url": "http://localhost:4000/MFBM_Tuesday_CT1/",
    "title": "Subgroup contributed talks",
    "body": "2020/08/01 - "
    }, {
    "id": 363,
    "url": "http://localhost:4000/MFBM_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 364,
    "url": "http://localhost:4000/MFBM_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 365,
    "url": "http://localhost:4000/MFBM_Monday_MS2/",
    "title": "Simulations and Experiments in Small Scale Bio-Locomotion",
    "body": "2020/08/01 - "
    }, {
    "id": 366,
    "url": "http://localhost:4000/MFBM_Monday_MS1/",
    "title": "Simulations and Experiments in Small Scale Bio-Locomotion",
    "body": "2020/08/01 - "
    }, {
    "id": 367,
    "url": "http://localhost:4000/MFBM_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 368,
    "url": "http://localhost:4000/MEPI_Wednesday_MS2/",
    "title": "COVID-19 Contributed Talks III",
    "body": "2020/08/01 - "
    }, {
    "id": 369,
    "url": "http://localhost:4000/MEPI_Wednesday_MS1/",
    "title": "Models of Infectious Diseases and Control",
    "body": "2020/08/01 - "
    }, {
    "id": 370,
    "url": "http://localhost:4000/MEPI_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 371,
    "url": "http://localhost:4000/MEPI_Tuesday_MS2/",
    "title": "COVID-19 Contributed Talks II",
    "body": "2020/08/01 - "
    }, {
    "id": 372,
    "url": "http://localhost:4000/MEPI_Tuesday_MS1/",
    "title": "Modeling COVID-19 to inform control efforts, Part II",
    "body": "2020/08/01 - "
    }, {
    "id": 373,
    "url": "http://localhost:4000/MEPI_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 374,
    "url": "http://localhost:4000/MEPI_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 375,
    "url": "http://localhost:4000/MEPI_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 376,
    "url": "http://localhost:4000/MEPI_Monday_MS2/",
    "title": "COVID-19 Contributed Talks I",
    "body": "2020/08/01 - "
    }, {
    "id": 377,
    "url": "http://localhost:4000/MEPI_Monday_MS1/",
    "title": "Modeling COVID-19 to inform control efforts, Part I",
    "body": "2020/08/01 - "
    }, {
    "id": 378,
    "url": "http://localhost:4000/MEPI_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 379,
    "url": "http://localhost:4000/IMMU_Wednesday_MS2/",
    "title": "Increasingly biologically accurate models of influenza A virus infection spread in vivo & in vitro",
    "body": "2020/08/01 - "
    }, {
    "id": 380,
    "url": "http://localhost:4000/IMMU_Wednesday_MS1/",
    "title": "Immunobiology and Infection Minisymposium",
    "body": "2020/08/01 - "
    }, {
    "id": 381,
    "url": "http://localhost:4000/IMMU_Tuesday_MS2/",
    "title": "Immunobiology and Infection Minisymposium",
    "body": "2020/08/01 - "
    }, {
    "id": 382,
    "url": "http://localhost:4000/IMMU_Tuesday_MS1/",
    "title": "Immunobiology and Infection Minisymposium",
    "body": "2020/08/01 - "
    }, {
    "id": 383,
    "url": "http://localhost:4000/IMMU_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 384,
    "url": "http://localhost:4000/IMMU_Thursday_CT2/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 385,
    "url": "http://localhost:4000/IMMU_Thursday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 386,
    "url": "http://localhost:4000/EDUC_Wednesday_MS2/",
    "title": "Education Minisymposium II",
    "body": "2020/08/01 - "
    }, {
    "id": 387,
    "url": "http://localhost:4000/EDUC_Wednesday_MS1/",
    "title": "Education Minisymposium II",
    "body": "2020/08/01 - "
    }, {
    "id": 388,
    "url": "http://localhost:4000/CDEV_Wednesday_MS2/",
    "title": "Understanding development through theory and experiment",
    "body": "2020/08/01 - "
    }, {
    "id": 389,
    "url": "http://localhost:4000/CDEV_Wednesday_MS1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 390,
    "url": "http://localhost:4000/CDEV_Wednesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 391,
    "url": "http://localhost:4000/CDEV_Tuesday_MS2/",
    "title": "Shapes, patterns, and forces in development biology",
    "body": "2020/08/01 - "
    }, {
    "id": 392,
    "url": "http://localhost:4000/CDEV_Tuesday_MS1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 393,
    "url": "http://localhost:4000/CDEV_Tuesday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 394,
    "url": "http://localhost:4000/CDEV_Thursday_CT2/",
    "title": "Integrating cell mechanics and cell mechanosensing",
    "body": "2020/08/01 - "
    }, {
    "id": 395,
    "url": "http://localhost:4000/CDEV_Thursday_CT1/",
    "title": "The interplay of intracellular pattern formation, geometry, polarisation and mechanics",
    "body": "2020/08/01 - "
    }, {
    "id": 396,
    "url": "http://localhost:4000/CDEV_Monday_MS2/",
    "title": "Modeling dynamics in single-cell biology",
    "body": "2020/08/01 - "
    }, {
    "id": 397,
    "url": "http://localhost:4000/CDEV_Monday_MS1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 398,
    "url": "http://localhost:4000/CDEV_Monday_CT1/",
    "title": "Subgroup Contributed Talks",
    "body": "2020/08/01 - "
    }, {
    "id": 399,
    "url": "http://localhost:4000/conference-announcement/",
    "title": "Conference Announcement",
    "body": "2020/06/08 - Never before in the history of the Society of mathematical biology has the importance of mathematical modelling been more prominent in the public eye. We will probably remember these days, in years to come, as a true opportunity to contribute to the global fight against the pandemic. Predictive mathematical models that capture the underlying mechanistic biology of specific processes are at the heart of everything we do. The COVID-19 virus pandemic has brought into sharp focus the critical importance of mathematical models in both making predictions about current trends in the disease dynamics but also how specific interventions, such as social distancing, might modify them. Understanding these interventions theoretically becomes important not only for mathematical biologists, but also for politicians and other decision makers. Most of the our society members are currently doing some form of social distancing, as is a large proportion of the world. In these confined and difficult times, anxiety and stress are running high. But it is important to remember that we have the power of mathematical modelling behind us. Their predictive ability illustrates why we must slowdown the spread of this virus by staying home. Flattening the curve has become our new mantra. Importantly mathematical modelling will also help us decide how to best end this quarantine. We are at a key point in the history of our society where our models are helping to inform policy decisions that impact millions of lives. Therefore it is fundamental that we make every effort to articulate as clearly as possible what mathematical models are, what they can be used for, and what are their limitations. The integration of mathematics and biology may sound strange at first to the public. But if you find yourself in a situation where you need to explain why certain decisions are being made, please take the time to do so. Articulate the role of predictive mathematical models, but not as something magical, even though they might seem so at times. Rather that they are based on our understanding of the physical and biological processes that underly the spread of this disease. We are a diverse group of scientists and educators who work at the interface of many disciplines not just epidemiology. Our society does have a dedicated subgroup in Mathematical Epidemiology and I encourage you join them if you are interested in this area of research or if you want to learn more. We also have 7 other subgroups covering everything from development to oncology and more - I’m extremely proud of our research diversity. The purpose of this email was to shine a little light and say thank you, to celebrate the wonderful discipline we are part of and highlight the important role mathematical models are playing right now. We are fortunate in many ways, with our ability to more readily work from home than most disciplines. We can make predictions about disease dynamics without ever actually treating a patient or being forced to make difficult decisions in medical care. We can run thousands of realisations of an experiment in a fraction of the time it would take in real life. We can make rigorously constrained predictions about the spread of an epidemic and how to slow it down. Ultimately the countermeasures needed to control and end this pandemic will rely on biomedical research on drugs and vaccines. But again, their impact can and should be studied through the lens of mathematical modelling. I would be remiss if I didn’t give a special thanks to our colleagues, friends and family members in the medical profession. Without their frontline action flattening the curve becomes meaningless. Stay safe, stay home and celebrate mathematical biology! -Sandy Anderson, SMB President "
    }];

var idx = lunr(function () {
    this.ref('id')
    this.field('title')
    this.field('body')
    

    documents.forEach(function (doc) {
        this.add(doc)
    }, this)
});


    
function lunr_search(term) {
    $('#lunrsearchresults').show( 1000 );
    $( "body" ).addClass( "modal-open" );
    
    document.getElementById('lunrsearchresults').innerHTML = '<div id="resultsmodal" class="modal fade show d-block"  tabindex="-1" role="dialog" aria-labelledby="resultsmodal"> <div class="modal-dialog shadow-lg" role="document"> <div class="modal-content"> <div class="modal-header" id="modtit"> <button type="button" class="close" id="btnx" data-dismiss="modal" aria-label="Close"> &times; </button> </div> <div class="modal-body"> <ul class="mb-0"> </ul>    </div> <div class="modal-footer"><button id="btnx" type="button" class="btn btn-secondary btn-sm" data-dismiss="modal">Close</button></div></div> </div></div>';
    if(term) {
        document.getElementById('modtit').innerHTML = "<h5 class='modal-title'>Search results for '" + term + "'</h5>" + document.getElementById('modtit').innerHTML;
        //put results on the screen.
        var results = idx.search(term);
        if(results.length>0){
            //console.log(idx.search(term));
            //if results
            for (var i = 0; i < results.length; i++) {
                // more statements
                var ref = results[i]['ref'];
                var url = documents[ref]['url'];
                var title = documents[ref]['title'];
                var body = documents[ref]['body'].substring(0,160)+'...';
                document.querySelectorAll('#lunrsearchresults ul')[0].innerHTML = document.querySelectorAll('#lunrsearchresults ul')[0].innerHTML + "<li class='lunrsearchresult'><a href='" + url + "'><span class='title'>" + title + "</span><br /><small><span class='body'>"+ title +"</span><br /><span class='url'>"+ url +"</span></small></a></li>";
            }
        } else {
            document.querySelectorAll('#lunrsearchresults ul')[0].innerHTML = "<li class='lunrsearchresult'>Sorry, no results found. Close & try a different search!</li>";
        }
    }
    return false;
}
</script>
<style>
    .lunrsearchresult .title {color: #d9230f;}
    .lunrsearchresult .url {color: silver;}
    .lunrsearchresult a {display: block; color: #777;}
    .lunrsearchresult a:hover, .lunrsearchresult a:focus {text-decoration: none;}
    .lunrsearchresult a:hover .title {text-decoration: underline;}
</style>




<form class="bd-search hidden-sm-down" onSubmit="return lunr_search(document.getElementById('lunrsearch').value);">
<input type="text" class="form-control text-small"  id="lunrsearch" name="q" value="" placeholder="Type keyword and enter..."> 
</form>
            </ul>
-->
        </div>
    </div>
    </nav>

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content">
        


<div class="container">
<div class="jumbotron jumbotron-fluid mb-3 pl-0 pt-0 pb-0 bg-white position-relative">
		<div class="h-100 tofront">
			<div class="row  justify-content-center ">
				<div class=" col-md-8  pr-0 pr-md-4 pt-4 pb-4 align-self-center">
					<p class="text-uppercase font-weight-bold">
              <span class="catlist">
              	<span class="text-all">all</span>						
              </span>
					</p>
					<h1 class="display-4 mb-4 article-headline">All Posters</h1>
					<div class="d-flex align-items-center">
                        
                        <img class="rounded-circle" src="../assets/images/logo.jpg" alt="eSMB2020" width="70"/>
                        
						<small class="ml-3"> eSMB2020 <span><a target="_blank" href="https://twitter.com/ESmb2020" class="btn btn-outline-success btn-sm btn-round ml-1">Follow</a></span>
                            <span class="text-muted d-block mt-1">2:30 - 3:30pm, Monday - Wednesday</span>
						</small>
					</div>
				</div>
                
			</div>
		</div>
	</div>
</div>





<div class="container-lg pt-4 pb-4">
	<div class="row justify-content-center">
        
        <!-- Share -->
				<div class="col-lg-2 pr-4 mb-4 col-md-12">
					<div class="sticky-top sticky-top-offset text-center">
						<div class="text-muted">
							Share this
						</div>
						<div class="share d-inline-block">
							<!-- AddToAny BEGIN -->
							<div class="a2a_kit a2a_kit_size_32 a2a_default_style">
								<a class="a2a_dd" href="https://www.addtoany.com/share"></a>
								<a class="a2a_button_facebook"></a>
								<a class="a2a_button_twitter"></a>
							</div>
							<script async src="https://static.addtoany.com/menu/page.js"></script>
							<!-- AddToAny END -->
						</div>
					</div>
				</div>
        
        
		<div class="col-md-12 col-lg-10">
			
			
			
			<div>
	<h4>Click to view posters for each subgroup</h4>
  <ul class="pagination">
	  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/all-posters.html"><span class="">All</span></a></li>
	  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/CDEV-posters.html"><span class="text-CDEV">CDEV</span></a></li>
	  <li class="page-item" ><a class="page-link text-EDUC" style="color:#000000 !important;" href="/EDUC-posters.html"><span class="text-EDUC">EDUC</span></a></li>
	  <li class="page-item" ><a class="page-link text-IMMU" style="color:#000000 !important;" href="/IMMU-posters.html"><span class="text-IMMU">IMMU</span></a></li>
	  <li class="page-item"><a class="page-link text-MEPI" style="color:#000000 !important;" href="/MEPI-posters.html"><span class="text-MEPI">MEPI</span></a></li>
	  <li class="page-item"><a class="page-link text-MFBM" style="color:#000000 !important;" href="/MFBM-posters.html"><span class="text-MFBM">MFBM</span></a></li>
	  <li class="page-item"><a class="page-link text-NEUR" style="color:#000000 !important;" href="/NEUR-posters.html"><span class="text-NEUR">NEUR</span></a></li>
	  <li class="page-item"><a class="page-link text-POPD" style="color:#000000 !important;" href="/POPD-posters.html"><span class="text-POPD">POPD</span></a></li>
	  <li class="page-item"><a class="page-link text-ONCO" style="color:#000000 !important;" href="/ONCO-posters.html"><span class="text-ONCO">ONCO</span></a></li>
	  <li class="page-item"><a class="page-link text-OTHE" style="color:#000000 !important;" href="/OTHE-posters.html"><span class="text-OTHE">OTHE</span></a></li>
	</ul>
</div>
			</br>
			</br>
			</br>
			
			<ol class="list-posters list-OTHE list-OTHE list-IMMU list-MEPI list-CDEV list-ONCO list-POPD list-EDUC list-NEUR list-MFBM">
			
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Abdel-H.-Halloway/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Abdel H. Halloway
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Purdue University</em>
						    
						    
						    
					    		
					    			<h6>"Non-Equilibrial Dynamics in Under-Saturated Communities"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The concept of the evolutionarily stable strategy (ESS) has been fundamental to the development of evolutionary game theory. It represents an equilibrial evolutionary state in which no rare invader can grow in population size. With additional work, the ESS concept has been formalized and united with other stability concepts such as convergent stabil- ity, neighborhood invasion stability, and mutual invisibility. Other work on evolutionary models, however, shows the possibility of unstable and/or non-equilibrial dynamics such as limit cycles and evolutionary suicide. Such “pathologies” remain outside of a well-defined context, especially the currently defined stability concepts of evolutionary games. Ripa et al. (2009) offer a possible reconciliation between work on non-equilibrial dynamics and the ESS concept. They noticed that the systems they analyzed show non-equilibrial dynam- ics when under-saturated and “far” from the ESS and that getting “closer” to the ESS through the addition of more species stabilized their systems. To that end, we analyzed three models of evolution, two predator-prey models and one competition model of evolu- tionary suicide, to see how the degree of saturation affects the stability of the system. In the predator-prey models, stability is linked to degree of saturation. Specifically, a fully saturated community will only show stable dynamics, and unstable dynamics occur only when the community is under-saturated. With the competition model, we demonstrate it to be permanently under-saturated, likely showing such extreme dynamics for this rea- son. Though not a general proof, our analysis of the models provide evidence of the link between community saturation and evolutionary dynamics. Our results offer a possible placement of these unstable and non-equilibrial dynamics into a wider framework. In addi- tion, the results concur with previous results showing greater evolutionary response to less biodiversity and clarifies the effect of extrinsic vs. intrinsic non-equilibrial evolutionary dynamics on a community.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Adelle-Coster/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Adelle Coster
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of New South Wales, Sydney Australia</em>
						    
						    
						    
					    		
					    			<h6>"The Distance Between: Stochastic Models of Cellular Protein Transport"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Translocation of proteins is essential for cell metabolism. Whilst mean-field models of the molecular movements within cells have identified dominant processes at the macroscopic scale, stochastic models may provide further insight into mechanisms at the molecular scale. The aim of this study was to develop a distance metric between stochastic data sets which evolve over time. This would enable the quantitative comparison of the outputs of a candidate stochastic model and the different experimental measurements of the system. A candidate stochastic model is developed for the translocation in mammalian cells of the insulin-dependent glucose transporter protein, GLUT4. The model is a closed queueing network. Various outputs of the system are compared to different experimental data sets, and synthetic data produced. Using empirical probability distributions to compare the time courses of stochastic measurements with the stochastic outputs of the model, we test different quantitative comparisons between the model output and the synthetic data, with the ultimate aim of driving parameter inference and model selection.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Adnan-A-Khan/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Adnan A Khan
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Lahore University of Management Sciences</em>
						    
						    
						    
					    		
					    			<h6>"Understanding the COVID-19 Outbreak in Pakistan"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We present a study of the transmission dynamics of the COVID-19 outbreak in Pakistan. Important transmission pathways such as the role of Asymptomatic, Quarantined and Isolated individuals are incorporated in the model. The model is then used to study the outbreak in Pakistan, considering the different lockdown and social distancing measures taken at different times by the health authorities noting that the epidemic curve in Pakistan has been quite different from those in Europe and the US, with a significantly lower disease burden and mortality rate.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Adriana-Zanca/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Adriana Zanca
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>The University of Melbourne</em>
						    
						    
						    
					    		
					    			<h6>"Multicellular model of collective cell migration with an irregular free boundary"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In many biological systems, including cancer, embryonic development and wound healing, cells migrate in a coordinated fashion. Interactions between individual neighbouring cells at a cellular scale leads to a collective movement at a tissue scale. Cell-based models allow cellular level processes to be explicitly incorporated and can allow us to investigate how tissue heterogeneities influence migration. Here we present our work using a vertex dynamics model to investigate the effect of an irregular free boundary on migration speed, cell density and orientation in a cell monolayer.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Adrianne-Jenner/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Adrianne  Jenner
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Montreal</em>
						    
						    
						    
					    		
					    			<h6>"Exploring the impact of intratumoral heterogeneity on oncolytic virotherapy using agent-based modelling"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Oncolytic viruses (OV) are an exciting immunotherapeutic modality currently being investigated for the treatment of glioblastoma multiforme (GBM), an aggressive brain cancer with a poor clinical prognosis. Unfortunately, promising pre-clinical investigations of immunotherapies have led to a number of disappointing trail results. It is clear that recapitulating the tumour microenvironment (TME) and finding useful pre-clinical models to elucidate the efficacy of OVs is crucial to improve OV treatments. CANscript is an ex vivo tumour spheroid model that has been used to improve pre-clinical evaluation as it recapitulates native, patient autologous TME. Leveraging pre-clinical GBM spheroids, we evaluated the infiltration of a herpes simplex OV in patient GBM samples, and constructed a computational representation of this system in PhysiCell, an open-source cell-based simulator, to determine OV characteristics that optimized therapeutic efficacy with respect to the impact of stromal density on OV infiltration. Additionally, we examined how intratumoural heterogeneity in the uptake rate of the OV influences efficacy. Overall, our results showed that the intracellular viral replication rate is the primary driver of OV infiltration patterns observed in the ex vivo samples. This work, therefore, has implications on the development of OVs for the treatment of GBM and in our understanding of the impact of spatial heterogeneity on new treatment approaches.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Akhil-Kumar-Srivastav/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Akhil Kumar Srivastav
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Vellore Institute of Technology</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical Modeling of Malaria with Saturated Treatment-A Case Study of India"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Malaria is a life-threatening mosquito-borne disease. It is transmitted through the bite of an infected Anopheles mosquito. People who get infected with malaria become very sick with high fevers, chills, and flu-like symptoms. Malaria may be fatal if not treated promptly. Here we propose an SIS model to study the trans- mission dynamics of malaria with saturated treatment. We assume that the mosquito population is growing logistically in the environment. Here we include a saturated type treatment function which is more suitable for the regions with limited resources. We discuss the existence and stability of different equilibria of the proposed model. We also compute the basic reproduction number R0 which plays an important role in existence and stability of equilibria of the model. We estimate the parameter corresponding to transmission of malaria using real data from different states of India by least square method. We also perform sensitivity analysis using PRCC to identify the key parameters which influence the basic reproduction number and system both, hence regulate the transmission dynamics of malaria. Numerical simulations are presented to illustrate the analytic findings.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Alexander-P-Hoover/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Alexander P Hoover
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Akron</em>
						    
						    
						    
					    		
					    			<h6>"Exploring Neuromechanical Resonance in Jellyfish Locomotion"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In order for an organism to have a robust mode of locomotion, their neuromuscular organization must be adaptable in a changing environment. In jellyfish, the activation and release of muscular tension is governed by the interaction of pacemakers with the underlying motor nerve net that communicates with the musculature. This set of equally-spaced pacemakers located at bell rim alter their firing frequency in response to environmental cues, forming a distributed mechanism to control the bell's muscular contraction. The relative simplicity of the jellyfish nervous system presents mathematicians with the opportunity to examine an intriguing multi-scale, multi-physics system with many potential applications to soft-body robotics and tissue-engineered pumps. In this talk, we explore the control of medusan neuromuscular activation in with a model jellyfish bell immersed in a viscous fluid and use numerical simulations to describe the interplay between active muscle contraction, passive body elasticity, and fluid forces. The fully-coupled fluid structure interaction problem is resolved using an adaptive and parallelized version of the immersed boundary method (IBAMR). This model is then used to explore the interplay between the speed of neuromechanical activation, fluid dynamics, and the material properties of the bell.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Alexey-A.-Tokarev/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Alexey A. Tokarev
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>People’s Friendship University of Russia (RUDN University)</em>
						    
						    
						    
					    		
					    			<h6>"Velocity-Amplitude relationship in the Gray-Scott autowave model in isolated conditions"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Various chemical and biological systems involve autocatalytic steps and positive feedbacks which in spatial conditions can give them properties of active media, in particular autowave properties. The main autowave characteristics are velocity and amplitude. This report considers the autowave velocity-amplitude relation in the general mathematical model of active reactant formation from precursor with cubic kinetics followed by a linear inhibition/death step – the Gray-Scott model – in isolated conditions. The way to derive the explicit velocity-amplitude relation is proposed. This approach may be useful for investigation of more complex active media systems in biochemistry, combustion, and disease control. The work has been supported by the «RUDN University Program 5-100» and by the Ministry of Science and Higher Education of the Russian Federation, agreement no. 075-03-2020-223/3 (FSSF-2020-0018).
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Alexis-Erich-S-Almocera/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Alexis Erich S Almocera
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Univ. of the Philippines Visayas</em>
						    
						    
						    
					    		
					    			<h6>"Modeling the viral infectious disease from infection to epidemic"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mathematical models can integrate different components of an infectious disease to reveal novel insights into the long-term effects. Conventional models focus either on pathogen infection (in-host scale) or pathogen transmission (between-host scale). However, we have yet to establish a framework that unites the two scales. We employ a conceptual modeling approach, where the standard transmission parameter becomes a function of the viral load, coupling in-host virus kinetics with an immune response to a compartmental disease model. The stability of the steady states with simulations cast light on the extent of a chronic host infection to influence the severity of an outbreak. Our results lend support to the 'multiscale' paradigm of disease modeling, which can inform long-term, personalized, and data-driven strategies in disease control and healthcare.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Amanda-Alexander/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Amanda Alexander
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Utah</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of regulatory T cell mechanisms in experimental autoimmune encephalomyelitis"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The study of T cell mediation of the adaptive immune response during multiple sclerosis (MS) can lead to development of treatments and therapies for this demyelinating disease. A prominent mouse model for MS is experimental autoimmune encephalomyelitis (EAE). EAE is mediated by populations of CD4+ T cells: regulatory T cells (Tregs), which prevent the immune system from attacking self proteins, and other effector T cells (Teffs) that are activated against myelin oligodendrocyte glycoprotein. The disease can result in either relapsing remitting or constant symptoms which can be either mild, medium, or severe for extended periods of time. These differences in severity are influenced by the numbers of precursor T cells, and by the initial dose of antigen in the system. We have developed an ODE model of Treg and Teff populations over time that incorporates immune regulatory mechanisms in order to make testable predictions about EAE disease course. This model exhibits three stable steady states (a state with all cell populations near 0, an intermediate state, and a high Teff state) for realistic parameter values, and biologically plausible initial conditions lie in the basins of attraction for all three stable steady states. Thus this model can explain biologically observed disease outcomes, and mathematical analysis can provide specific, biologically testable predictions about the cause of each outcome.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Ananta-Samrajya-Shri-Kishore-Hari/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Ananta Samrajya Shri Kishore  Hari
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>India</em>
						    
						    
						    
					    		
					    			<h6>"Identifying inhibitors of epithelial–mesenchymal plasticity using a network topology-based approach"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Metastasis is the cause of over 90% of cancer-related deaths. Cancer cells undergoing metastasis can switch dynamically between different phenotypes, enabling them to adapt to harsh challenges, such as overcoming anoikis and evading immune response. This ability, known as phenotypic plasticity, is crucial for the survival of cancer cells during metastasis, as well as acquiring therapy resistance. Various biochemical networks have been identified to contribute to phenotypic plasticity, but how plasticity emerges from the dynamics of these networks remains elusive. Here, we investigated the dynamics of various regulatory networks implicated in Epithelial–mesenchymal plasticity (EMP)—an important arm of phenotypic plasticity—through two different mathematical modelling frameworks: a discrete, parameter-independent framework (Boolean) and a continuous, parameter-agnostic modelling framework (RACIPE). Results from either framework in terms of phenotypic distributions obtained from a given EMP network are qualitatively similar and suggest that these networks are multi-stable and can give rise to phenotypic plasticity. Neither method requires specific kinetic parameters, thus our results emphasize that EMP can emerge through these networks over a wide range of parameter sets, elucidating the importance of network topology in enabling phenotypic plasticity. Furthermore, we show that the ability to exhibit phenotypic plasticity correlates positively with the number of positive feedback loops in a given network. These results pave a way toward an unorthodox network topology-based approach to identify crucial links in a given EMP network that can reduce phenotypic plasticity and possibly inhibit metastasis—by reducing the number of positive feedback loops.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Anca-Radulescu/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Anca  Radulescu
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>SUNY New Paltz</em>
						    
						    
						    
					    		
					    			<h6>"Effects of local mutations in quadratic iterations"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The study of copying mechanisms is of great importance to genetics. We study in a theoretical system how a mutation (replication error) affects the temporal evolution of the system, on both a local and global scale (from tumor formation to overall systemic unsustainability). We introduce a new mathematical framework for studying replication mechanisms, in the form of discrete iterations of complex quadratic maps. This approach builds upon a century of existing knowledge of iterated maps towards obtaining results with potential impact on applications. More specifically, our modeling framework considers a “correct” function acting on the complex plane (representing the space of genes to be copied) and a “mutation,” acting at a specific focal point, of a given size r, and moves radially toward an outer radius R. We use the Julia set of the system to quantify simultaneously the long-term behavior of the entire space under such transformations. We analyze how the position, timing and size of the mutation can alter the topology of the Julia set, hence the system’s long-term evolution, its progression into disease, but also its ability to recover or heal. In the context of genetics, such results may help shed some light on aspects such as the importance of location, size and type of mutation when evaluating a system’s prognosis, and of customizing timing of treatment to address each specific situation. Our current work is a proof of principle. Once these aspects are understood theoretically, they can be further applied to empirically driven genetic models, validated with data and used for predictions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Andreas-Buttenschoen/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Andreas Buttenschoen
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of British Columbia</em>
						    
						    
						    
					    		
					    			<h6>"Spatio-Temporal Heterogeneities in a Mechano-Chemical Model of Collective Cell Migration"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Small GTPases, such as Rac and Rho, are well known central regulators of cell morphology and motility, whose dynamics also play a role in coordinating collective cell migration. Experiments have shown GTPase dynamics to be affected by both chemical and mechanical cues, but also to be spatially and temporally heterogeneous. This heterogeneity is found both within a single cell, and between cells in a tissue. For example, sometimes the leader and follower cells display an inverted GTPase configuration. While progress on understanding GTPase dynamics in single cells has been made, a major remaining challenge is to understand the role of GTPase heterogeneity in collective cell migration. Motivated by recent one-dimensional experiments (e.g. micro-channels) we introduce a one-dimensional modelling framework allowing us to integrate cell bio-mechanics, changes in cell size, and detailed intra-cellular signalling circuits (reaction-diffusion equations). Using this framework, we build cell migration models of both loose (mesenchymal) and cohering (epithelial) tissues. We use numerical simulations, and analysis tools, such as local perturbation analysis, to provide insights into the regulatory mechanisms coordinating collective cell migration. We show how feedback from mechanical tension to GTPase activation lead to a variety of dynamics, resembling both normal and pathological behavior.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Angela-L-Moreno/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Angela L Moreno
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Federal University of Alfenas</em>
						    
						    
						    
					    		
					    			<h6>"ART Neural Networks in the Classification of Spinal Pathologies"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Spinal diseases are among the significant public health problems and have a negative impact on patients' quality of life. Of these diseases, herniated disc and spondylolisthesis are examples of spinal pathologies that cause severe pain. Currently, in various medical problems related to the diagnosis of diseases, Machine Learning techniques have been used, especially Artificial Neural Networks. From the attributes of the spine such as pelvic incidence angle, pelvic tilt, sacral angulation, pelvic radius, lumbar lordosis angle, and degree of sliding, pattern recognition techniques can be employed to classify herniated disc pathologies and spondylolisthesis. Thus, this paper presents the results obtained by using neural networks of the Adaptive Resonance Theory (ART) family for the classification of spinal pathologies, comparing the results obtained by the different ART networks with those obtained in the literature. Using a neural network in a classification problem has the advantage of robust, stable, fast models that are capable of classification even with little data about the problem. In particular, ART Fast networks are characterized by the ability, even from a few data, to classify the data and, according to the network, will expand from the insertion of new data, improving the chance right. It is also noteworthy that ART 	extit {Fast} networks perform the classification process faster than traditional ART networks, maintaining the number of hits. The methodology adopted is based on the implementation of ART networks using the Vertebral Column Data Set database, available in the UCI Machine Learning Repository, http://archive.ics.uci.edu/ml/. The results obtained by the network meant satisfactory, obtaining an accuracy of 91.26% for the binary classification problem and 90.97% for the ternary.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Angela-Lynn-Peace/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Angela Lynn Peace
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Texas Tech University</em>
						    
						    
						    
					    		
					    			<h6>"A Simple model of pathogen-mediated nutrient dynamics"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			How do nitrogen and phosphorus availability impact infectious disease, and what are the reciprocal effects of pathogens on ecosystem nutrient dynamics? These questions are fundamental to understanding the coupling between disease dynamics and nutrient cycles, yet disease-ecosystem relationships are often overlooked.  Relationships linking infectious disease with ecosystem nutrient dynamics are multidirectional and can form feedback loops, though the dynamic interdependence of these processes is little understood. We illustrate the impact of disease-ecosystem feedback loops for the dynamics of both infection outcomes and ecosystem nutrients using a simple mathematical model. The model is a nonsmooth system of ordinary differential equations combining approaches from classical ecological models (logistic and droop growth) and epidemiological models (disease transmission). Our model incorporates the effects of nutrient availability on growth rates of susceptible and infected hosts, as well as the return of nutrients to the environment following host death. Despite the simplicity of this model, our results illustrate complex dynamics in host populations, infection patterns, and ecosystem nutrients that can arise from even a simple disease-nutrient feedback.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Angela-Michelle-Jarrett/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Angela Michelle Jarrett
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>The University of Texas at Austin</em>
						    
						    
						    
					    		
					    			<h6>"Modeling of the spatio-temporal evolution of tumor vasculature to improve predictions of breast cancer response to neoadjuvant chemotherapy regimens"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			One of the great challenges for treating cancer is the inability to design optimal therapeutic regimens for individual patients. Without a reasonable mathematical framework, selecting treatment regimens for the individual patient is fundamentally limited to trial and error. We have previously established a mechanically coupled, reaction-diffusion model at the tissue scale for predicting breast tumor response to therapy. The patient-specific, 3D model is initialized with tumor cell number estimated from quantitative, diffusion-weighted magnetic resonance imaging (DW-MRI) data. Additionally, the model includes a tumor cell reduction term due to drug delivery as estimated from dynamic contrast-enhanced (DCE-) MRI data (per individual clinical patient treatment schedules). We have expanded this model to differentiate between the effects of different chemotherapies to generate personalized and, potentially, optimized regimens for individual patients. This original model’s predictions have been found to be highly correlated to actual tumor response, but one limitation is that it does not account for the spatio-temporal changes of the tumor vasculature. Therefore, we now seek to extend this work by explicitly including the dynamics of an evolving vasculature to better simulate delivery of chemotherapies and account for the effect of these drugs on the vasculature itself. Importantly, by adding a second governing equation to the mathematical model representing the vasculature, we are able to reduce the parameter space of the model by coupling proliferation to the vasculature component—instead of defining proliferation as a local parameter in space. For an initial cohort of nine breast cancer patients, we evaluate the performance of the extended model by comparing its predictive ability to that of the original model (without vasculature). We report preliminary findings that the extended model’s results have lower median errors for its predictions. Future work will focus on expanding the model to account for targeted therapies and the simulation of alternative treatment regimens. We propose that an integrated mathematical-experimental approach leveraging patient-specific imaging data can provide optimal strategies for delivering therapy for breast cancer.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Angelo-J-Zorn/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Angelo J Zorn
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Occidental College</em>
						    
						    
						    
					    		
					    			<h6>"An Epidemic Mobility Model with Symptomatic and Asymptomatic Individuals Allowing Variation of Contact Rates Between Individuals and Across Regions"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The control of contacts among individuals and across regions is of paramount importance to understand the dynamics of infectious diseases. In the current COVID-19 pandemic, many infectious individuals are asymptomatic, thereby raising the question as to whether limiting contacts with infected individuals displaying symptoms is sufficient to control the spread of the disease.


To this end, we have developed a mathematical model, called SAIRD model, that includes susceptible individuals (S), asymptomatic infectious individuals (A), infectious individuals displaying symptoms (I), individuals who recovered (R) and deceased individuals (D). The model also includes mobility of individuals across geographic regions that accounts for inter-region travel patterns.


We have considered 3 regions characterized by different inter-individual contacts. Specifically, no limitations are adopted in region 1, limitations are adopted only for symptomatic individuals in region 2, and limitations are adopted for both asymptomatic and symptomatic infectious individuals in region 3. In the absence of inter-region connection, the model predicts similar disease dynamics in regions 1 and 2, whereas region 3 experiences a notable lower number of infections and death. These results suggest that controlling inter-individual contacts in both asymptomatic and symptomatic cases is essential to contain the disease dynamics. Furthermore, the model predicts that controlling inter-individual contacts without controlling inter-region connections may nullify the gains.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Aniruddha-Deka/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Aniruddha Deka
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Shiv Nadar University</em>
						    
						    
						    
					    		
					    			<h6>"Optimal public health strategy during an influenza outbreak"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Although vaccine has proven to be the best preventive method to reduce risk of flu infection, the coverage often remains below the herd-immunity level due to individuals’ perceptions towards the vaccination and the severity of disease outbreak. This, however, brings challenges to public health for strategic decision-making in controlling flu outbreak every year. To understand the impact of behavioral issues on public health decision-making to control flu, we define vaccination decision in population as a two-strategy pairwise-contest game and integrate with the disease process model to consider vaccination during a flu outbreak. We use optimal control theory to identify the best possible strategy for public health to reduce infection at a minimum cost. Our analysis shows that the cost of public health initiatives can be minimized by putting the effort in the beginning and end of the outbreak rather than during the peak. We also consider vaccination with evolving risk perception and infection with high severity such as disease-induced death. Our model demonstrates a feed-forward mechanism in the dynamics of vaccination and exhibits an increase in vaccine uptake as the risk perception decreases with more coverage. It confers that public health effort towards disseminating disease severity or actual vaccination risk might accelerate the vaccination coverage and mitigate the infection faster.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Anna-H-Sisk/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Anna H Sisk
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Linking Immuno-Epidemiology Principles to Violence"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Societies have always struggled with the causes and effects of violence, but only recently has there been a drive to better understand violence as a disease and to consider it from a public health perspective. Through the work of many physicians and psychologists we have realized violence is less like a moral failing and more like a disease. This realization unified professionals from the medical/epidemiological fields and those in psychology in a common goal to end violence and help heal those exposed to it. Recently, interesting analogies have been made between community-level infectious disease epidemiology and how violence spreads within a community. Experts in public health and medicine have suggested that an epidemiological framework could be used to study violence.
Infectious disease studies are often approached from two different scales: outbreak/community and immune system/individual. At both the epidemiological/community and immune system/individual scales, mathematical modeling of infectious disease dynamics plays an important role. Each scale has been modeled in isolation from the other; however, there is a natural connection between the epidemiological and immune system dynamics, since a person’s immune response determines the likelihood of transmission to others. Thus, there has been a push to consider both scales in the multi-scale integrated approach of Immuno-Epidemiological (IE) modeling. We plan to apply the approaches used for IE modeling to violence by employing the epidemiological part of the model to explore violence spread on the community level and the immune system model to look at the impact that violence exposure has on an individual with respect to increasing their propensity to commit violence.
In this talk I will expand on and formalize the analogy of violence as an infectious disease and show how the well-developed principles of mathematical epidemiology and immunology is a useful framework for understanding the dynamics of violence. Next, we will look at a preliminary susceptible-exposed-infected (SEI) mathematical model for violence spread on the community level and compare this model with traditional disease modeling. Then we will explore some basic equilibrium and stability analysis of the SEI model and look at the real-world interpretations of this analysis.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Antonela-Marozzi/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Antonela  Marozzi
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>INIBIOMA-CONICET-UNCo</em>
						    
						    
						    
					    		
					    			<h6>"An alternative method for calf density and recruitment estimation using pregnancy on wild guanacos (Lama guanicoe)."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Per capita recruitment is a parameter that determines most of the variation in population growth rate in wild temperate ungulates and it is generally estimated by young:female ratio. It has been proposed that this approach should be improved since only count data is used in most cases and the probability of observing calves at heel declines with the age of the calf because it is more independent of the mother. In this study, we propose an alternative method to estimate calf density (ACD) using pregnancy rate obtained from hormonal fecal metabolites and total density estimates. Then we use ACD to calculate recruitment by young:female ratio and compare the results with recruitment estimates using traditional count data (TCD). To set the parameters of ACD we used information of a partially-migratory guanaco (Lama guanicoe) population of La Payunia Provincial Reserve (Mendoza-Argentina). We calculated ACD by the following equation:A=p×h×s×D (eq. 1), where p is the pregnancy rate (0.32), s is the probability of survival (0.61), h is female's proportion (0.60), D is total density and A is calf density. First, we ran a data simulation to calculate A, using p, h, and s as deterministic parameters and D as a random parameter with a Log-Normal distribution. With the simulated data, we calculated per capita recruitment (R) and we adjusted a density-dependent model, as is expected for large ungulates: LogR=-0,179*D*[exp⁡(-0,013*D)] (eq. 2). Second, we used real data of four population surveys to estimate density by ACD. To do this we replaced the D term of the eq. 1 with field data and then, we calculated recruitment using those results. All the other terms of eq. 1 were kept the same because they belong to the population under study. Third, we estimated recruitment by young:female ratio using count data of the same surveys and compared real data recruitment estimations by ACD and TCD. Both estimations seem to follow the same pattern of the simulated data. However, recruitment obtained using ACD (0.09; 0.06; 0.07; 0.05) were lower than those calculated by TCD (0.36; 0.10; 0.31; 0.18). We hypothesized two main reasons: 1) our estimation is assuming a constant pregnancy rate; therefore, if new pregnant females entered the area under study from nearby regions from one year to the other, that information was not considered and could have led to an underestimation of recruitment by ACD. 2) In general, only a small number of calves is counted in population surveys, which may increase data dispersion, and as a consequence an overestimation of recruitment by TCD. Our innovative approach using total density estimations and pregnancy data might be useful to estimate young densities avoiding the problems of counting calves. As recruitment is one of the most important parameters to make management decisions like population control, our approach might be an alternative to reduce count data biases and should be tested in other ungulates populations.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Anuraag-Bukkuri/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Anuraag  Bukkuri
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Minnesota</em>
						    
						    
						    
					    		
					    			<h6>"GLUT1 Production in Cancer Cells: A Tragedy of the Commons"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The tragedy of the commons, a concept originally developed by economist William Lloyd to describe overgrazing by cattle, is a phenomenon in which individual selfishness in a group setting leads to depletion of a shared resource, to the detriment of the overall population. We hypothesize that such a situation occurs in cancer cells in which cells increase production of membrane GLUT transporters for glucose in the presence of competing cells, obtaining a modest personal gain at a great group cost. To formalize this notion, we create a game-theoretic model for capturing the effects of competition on cancer cell transporter production and nutrient uptake. We show that the production of transporters per cell increases with a logistic trend as the number of competing cells in a microenvironment increase, but nutrient uptake per cell decreases in a power law fashion. By simulating GLUT1 inhibitor and glucose deprivation treatments, we demonstrate a synergistic combination of standard-of-care therapies and clustering of cancer cells, while also displaying the existence of a trade-off between competition among cancer cells and depression of the gain function. Assuming cancer cell transporter production is heritable, we then show the potential for a sucker's gambit technique to be used to counteract this trade-off, thereby allowing one to take advantage of both cellular competition and gain function depression by strategically changing environmental conditions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Artur-C%C3%A9sar-Fassoni/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Artur  César Fassoni
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidade Federal de Itajubá</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling and methodology to identify patient-specific immunological landscapes in CML treatment using TKI cessation and dose reduction data"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Chronic myeloid leukemia (CML) is an example of how mathematical models can help on understanding and describing cancer treatment. In the last years, the paradigm in CML treatment with tyrosine kinase inhibitors (TKI) changed from a life-long treatment to a scenario where patients with good response can stop treatment and remain in treatment free remission (TFR). Although it is still not clear which are the mechanisms and markers that identify those patients, recent evidence suggests that the immune response is crucial for maintaining TFR. Here, we present an ODE model for CML treatment and the role of an anti-leukemic immune response. Keeping the model as simple as possible we show that it fits well to 21 individual time courses under standard treatment. However, the optimal fits are not unique, which leads to ambiguity in the predictions about the outcome of treatment cessation. To overcome it, we show that additional data after TKI stop allows to capture the information necessary to use the model for making predictions. Applying this methodology to those 21 patients and calculating the multiple basins of attraction of stable equilibria in the patient-specific calibrated model, we identify three qualitatively different 'immunological landscapes' among which the patients are distributed. One set corresponds to those patients that require complete CML eradication to achieve TFR, meaning in practice a lifelong therapy or a likely recurrence after TKI stop. A second class corresponds to those patients where the immune system controls residual CML cells after treatment cessation if a certain threshold is achieved. A third class corresponds to patients where the immunological control of CML is achieved only if intricated balance between TKI effects and immune activation is achieved. Mathematically, this corresponds to phase portraits where one basin of attraction presents a topological defect arising from a heteroclinic bifurcation, and model simulations suggest that such optimal balance leading to TFR can be achieved with protocols of dose reduction. Finally, we show that the information necessary to classify the patient’s immunological landscape can be obtained not only from TKI stop data, but also from measuring the effects of TKI dose reduction during a six-month period. This provides a general strategy consisting of three phases: standard treatment, then standard reduced treatment and accurate observation of response, then model-based patient-specific treatments based on the previous phase. Summed up, these results illustrate the potential of mathematical modeling to the era of personalized medicine, with CML as a concrete example, but potential to more complex cancers, and also illustrates the difficulties that mathematical oncologists may encounter on this way, such as parameter unidentifiability and possibilities to circumvent it.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Atanaska-Dobreva/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Atanaska  Dobreva
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>ASU</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of photoreceptor metabolism"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Photoreceptors are the sensory cells of the eye and have the most important role in vision. They convert light to electrical signals, which are sent to the brain via the optic nerve. Vision deterioration or blindness occur if the vitality of photoreceptors is compromised. To understand how to mitigate such pathological cases, it is essential to study the metabolism of photoreceptors, as this is the factor of greatest importance for photoreceptor vitality. We develop a mathematical model of nonlinear ordinary differential equations to describe metabolic dynamics in a single photoreceptor, focusing on key metabolites, such as glucose, pyruvate and lactate. Using bifurcation techniques, we find that the model has a bistable regime, biologically corresponding to a healthy versus a pathological state. We also conduct sensitivity analysis to determine which processes have the largest impact on the photoreceptor metabolic system. The results indicate that of greatest importance are the pathways linking photoreceptor metabolism with the metabolism of the retinal pigment epithelium, a cellular layer in the retina with which photoreceptors have a reciprocal resource relation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Atchuta-Srinivas-Duddu/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Atchuta Srinivas Duddu
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Institute of Science</em>
						    
						    
						    
					    		
					    			<h6>"Multistability in cellular differentiation enabled by a mutually antagonistic triad"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Identifying the design principles of complex regulatory networks driving cellular decision-making remains essential to decode embryonic development as well as enhance cellular reprogramming. A well-studied network motif involved in cellular decision-making is a toggle switch – a set of two opposing transcription factors A and B, each of which is a master regulator of a specific cell-fate and can inhibit the activity of the other. A toggle switch can lead to two possible states – (high A, low B) and (low A, high B), and drives the ‘either-or’ choice between these two cell-fates for a common progenitor cell. However, the principles of coupled toggle switches remains unclear. Here, we investigate the dynamics of three master regulators A, B and C inhibiting each other, thus forming three coupled toggle switches to form a toggle triad. Our simulations show that this toggle triad can drive cells into three phenotypes – (high A, low B, low C) , (low A, high B, low C), and (low A, low B, high C). This network can also allow for hybrid or ‘double positive’ phenotypes – (high A, high B, low C), (low A, high B, high C) and (high A, low B, high C), especially upon including self-activation loops on A, B and C. Finally, we apply our results to understand the cellular decision-making in terms of differentiation of naïve CD4+ T cells into Th1, Th2 and Th17 states, where hybrid Th1/Th2 and hybrid Th1/Th17 cells have been reported in addition to the Th1, Th2 and Th17 ones. Our results offer novel insights into the design principles of a multistable network topology and provides a framework for synthetic biology to design tristable systems.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Augustine-Okebunor-Okolie/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Augustine Okebunor Okolie
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Technical University of Munich</em>
						    
						    
						    
					    		
					    			<h6>"Exact and approximate formulas for contact tracing on random trees"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We consider a stochastic susceptible-infected-recovered (SIR) model with contact tracing on random trees and on the configuration model. On a rooted tree, where initially all individuals are susceptible apart from the root which is infected, we are able to find exact formulas for the distribution of the infectious period. Thereto, we show how to extend the existing theory for contact tracing in homogeneously mixing populations to trees. Based on these formulas, we discuss the influence of randomness in the tree and the basic reproduction number. We find the well known results for the homogeneously mixing case as a limit of the present model (tree-shaped contact graph). Furthermore, we develop approximate mean field equations for the dynamics on trees, and – using the message passing method – also for the configuration model. The interpretation and implications of the results are discussed.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Baeckkyoung-Sung/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Baeckkyoung  Sung
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>KIST Europe/ UST</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of a temperature-sensitive and tissue-mimicking gel matrix"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Miniaturized biopolymer gel systems have been attracting interests for the application to regenerative medicine, due to their physiological compatibility/sensitivity and rapid kinetics with response to external stimuli. For explaining such responsivity in terms of gel thermodynamics and mechanics, classical mean-field Flory-Huggins-Rehner theory has long been developed with various analytical and numerical modifications. In this work, we present a novel mathematical model on the volume phase transitions of biological hybrid gels as a function of temperature. In order to mimic living soft tissues, the biological microgels are designed to comprise 3D network of extracellular matrix (ECM) protein chains such as collagen and gelatin, which are covalently cross-linked and remain swollen in aqueous media. Within the network, thermoresponsive synthetic polymer chains are doped by physical entrapment and chemical conjugations. Based on the Flory’s framework, our analytical model phenomenologically predicts well-defined volume phase behaviors of the 3D tissue mimics with response to the change in ambient thermodynamic parameters.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Baylor-Fain/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Baylor Fain
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Texas Christian Univsersity</em>
						    
						    
						    
					    		
					    			<h6>"Modeling the impact of inoculum dose and transmission mode on viral infection with an agent-based model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In a virus study, the inoculum dose is the initial amount of virus used. It is correlated to the initial amount of cells that become infected at the start of the study and thereby also correlated with the amount of virus that will be produced by infected cells at the beginning of that study. Those virus spread through a body in two known ways: cell free transmission and cell to cell transmission. While previous research has investigated viruses based on free cell transmission, few models have incorporated cell to cell transmission leading to­ unclear results and bias to certain variables. This research accounts for both modes of transmission, using an agent-based framework, and varies the initial amount of virus, to understand how inoculum dose affects the two transmission modes. Utilizing parallel processing, the model represents virus infection and spread in a two-dimensional layer of cells in order to generate total virus over time graphs for corresponding initial amount of virus. This project demonstrates how a combination of agent-based models and parallel processing can allow researchers to perform the rapid and large simulations necessary for viral dynamics research efficiently and affordably.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Benedikt-Sabass/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Benedikt  Sabass
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Forschungszentrum Jülich, LMU Munich</em>
						    
						    
						    
					    		
					    			<h6>"Molecular mechanics of type-IV pili driven migration of P. aeruginosa"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Bacteria can generate mechanical forces that are important for the colonization of surfaces, formation of biofilms, and infection of host cells.  In Gram-negative bacterial pathogens, such as Pseudomonas aeruginosa, forces result from ATP-hydrolysis-driven extension-retraction cycles of extracellular filaments called type-IV pili. How bacteria adapt their pilus-based behavior to the mechanical environment is not known. Here, we show that the early stage of surface colonization by P. aeruginosa is modulated by substrate-dependent pilus activity. Our experimental data reveals a complex response of the bacterial migration machinery to substrate properties, including adaptation of the dynamics of pili, their spatial arrangement, and their number. The combination of experimental data with mathematical modeling reveals a comprehensive picture of the interplay of active and passive molecular mechanisms during migration of P. aeruginosa on solid substrates.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
							
					    	
					    	<a href="/Bejamin-J-Jessie/">
						    	<h4><li class="subgroup-counter" style="color: #fdbe7d !important;">43</li>
							    	
							    	Benjamin J Jessie
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Texas Christian University</em>
						    
						    
						    
					    		
					    			<h6>"Respiratory Syncytia Virus"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Respiratory syncytial virus (RSV) is a common, contagious infection of the lungs and the respiratory tract. RSV is characterized by syncytia, which are multinuclear cells created by cells that have fused together. Because of experimental limitations, it is difficult to measure characteristics such as viral production rate and lifespan of the syncytia cells. We use mathematical models to study how different assumptions about the viral production and lifespan of syncytia change the resulting infection to determine whether indirect measurements can be used in place of experimental results.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
						
						
						
						 <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Bertin-Hoffman/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Bertin Hoffman
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Applied Sciences Stralsund</em>
						    
						    
						    
					    		
					    			<h6>"The initial engraftment of tumor cells is critical for the future growth pattern"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Xenograft mouse models are used to study mechanisms of tumor growth and metastasis formation as well as investigating the efficacy of different therapeutic interventions. After injection the engrafted cells form a local tumor nodule whose size can be measured repeatedly during an experiment. The so obtained experimental growth data can be described mathematically by suitable growth functions, the choice of which is not always obvious. By applying nonlinear curve fitting, growth parameters can be determined that provide information on the tumor growth.


We used self-generated synthetic data including random measurement errors to research the accuracy of parameter estimation based on caliper-measured experimental tumor data. Fit metrics were investigated to identify the most appropriate growth function for a given synthetic dataset. For curve fitting with fixed initial tumor volume, we varied the fixed initial tumor volume during curve fitting to investigate the effect on the resulting estimated parameters. To determine the number of tumor cells that survive initially after injection into mice, we performed ex vivo bioluminescence imaging of the tumor nodules on day 1, 2, 4 and 8 after injection. By this experimental approach we determined the effect of incorrect assumed initial tumor volume in experiments.


An incorrect assumed value of the initial tumor volume during the parameter estimation process leads to large deviations in the resulting growth parameters. Therefore, the actual number of cancer cells engrafting directly after subcutaneous injection is critical for future tumor growth and distinctly influences the parameters for tumor growth by curve fitting.


Hoffmann, B., Lange, T., Labitzky, V., Riecken, K., Wree, A., Schumacher, U., Wedemann, G. The initial engraftment of tumor cells is critical for the future growth pattern: a mathematical study based on simulations and animal experiments. BMC Cancer 20, 524 (2020). https://doi.org/10.1186/s12885-020-07015-9
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Beryl-O-Musundi/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Beryl O Musundi
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Technical University of Munich</em>
						    
						    
						    
					    		
					    			<h6>"An immuno-epidemiological model linking within-host and between-host dynamics of cholera"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Cholera, an acute gastrointestinal disease caused by the bacterium Vibrio cholerae, continues to be a major threat to public health with an estimated 1.3 - 4 million cases reported annually. The majority of existing cholera models focus on the between-host dynamics independent of within-host dynamics, a factor that downplays the interdependence of the two processes on the spread of the infection. In this study, an immuno-epidemiological model for cholera is formulated to analyze the effect of within- host dynamics on the population. The model is an adaptation of the immuno-epidemiological models reviewed by M. Martcheva, N. Tuncer, C. St Mary (2015). We use time-scale methods to distinguish the dynamics of the immune response and the parasite load of an individual. A thorough bifurcation analysis reveals the existence of a saddle node and Hopf bifurcation. In contrast to other immunological models, the present approach allows for clearance of the pathogen after a finite time. The between- host system is represented by a size structured model with the pathogen load considered as the linking mechanism. We derive expressions for the reproduction number and conduct stability analysis of the equilibria. Conclusions about the interdependence of the two dynamics on disease spread are drawn from the analysis.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Bhattacharyya-Samit/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Bhattacharyya  Samit
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Shiv Nadar University</em>
						    
						    
						    
					    		
					    			<h6>"Evolutionary game models to explain the potential interactions among Adverse events, Vaccine scares and Individual vaccination choice"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Rumours on adverse health outcomes either from infection or vaccination have strong impact on vaccination uptake. Understanding human behavioral responses during a vac- cine scare and its interactions with the population dynamics of disease play key role in predicting the dynamics of disease, designing intervention strategies and policymaking in public health program. Mathematical modelling is an important tool for investigating and quantifying such effects in infectious disease and control. In the first part of my talk, I will introduce evolutionary game models of vaccination dynamics in homogeneous population to describe how human attitude changes towards vaccination during a scare using empirical data of several vaccination coverage. In the second part, I will introduce vaccination game on social network to discuss how rare but severe events can impact the vaccination dynamics.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Bhawna-Malik/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Bhawna  Malik
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Shiv Nadar University</em>
						    
						    
						    
					    		
					    			<h6>"Stochastic extinction of drug-resistant strains: Modelling the role of socio-economic factors in the pattern of extinction"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Stochastic fluctuations in transmission may increase the probability of extinction of pathogens. While overuse of antibiotics leads to the emergence of new resistant strains in population by lowering its fitness cost, other socioeconomic factors may change the selection pressure and increase the probability of extinc- tion of the resistant strain. We develop a stochastic model of drug-resistance integrating socioeconomic growth in population to study the dynamics of extinction of resistant strain in the community, where it competes with the existing sensitive strain. We analytically derive the extinction threshold from the stochastic model using the multi-type branching process theory and obtain conditions for pathogen ex- tinction or persistence in population. Using numerical simulations, we compute the extinction probability, which shows a good estimate of values obtained from the branching process. Sensitivity analysis of the model also identifies parameters that have the most impact on the extinction of the strains. Although the transmission potential of respective strains plays a major role in extinction, our results illustrate that higher income, awareness, lower antibiotic use may increase the chance of extinction significantly by lowering antibiotic misuse. These analyses are beneficial to health policy makers and may quantify some parameters which are important to control the situation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Bin-Zhang/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Bin  Zhang
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Georgia State University</em>
						    
						    
						    
					    		
					    			<h6>"The Ecology of Collective Cancer Invasion: An Evolutionary Game Theory Model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Cancer is an evolutionary disease which exhibits genomic and phenotypic heterogeneity. Together with the tumor microenvironments, the cell subclones within the a tumor form a complex multi-cellular ecosystem. Tumors compromise a variety of specialized phenotypical subclones adapted to various ecological conditions, which influence the response to treatments and prognosis of the diseases. Recent experiments revealed two distinct phenotypes, leaders and followers, in non-small cell lung cancer during collective invasion. We adopt an evolutionary game theory framework to model the cancer microenvironments and the interactions between leader and follower cells. Measuring the total tumor burden and the leader fraction that drive collective invasion, we show that the pairwise interactions between leader and follower cells could alter the collective dynamics. These findings suggest potential new treatment strategies, targeting leader-follower cells interactions. Combinations treatments could reduce tumor burden as well as lower the risk for invasion.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Bj%C3%B6rn-Vessman/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Björn  Vessman
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Lausanne</em>
						    
						    
						    
					    		
					    			<h6>"A theoretical investigation of artificial community selection methods"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Our lab studies small bacterial communities that degrade polluting industrial waste fluids. While our standard species combination can degrade some 40% of the pollutants, we hypothesize that a) the community could improve over time and b) that combinations with other species may be even more efficient. We now ask whether we can assemble and breed new communities with improved degradation efficiency. Previous attempts to experimentally select for community functions, such as host phenotypes and degradation or production of certain chemicals, have shown mixed results. Fundamentally, the experimental design needs to maintain a variance in the community function and ensure that the trait that forms the basis of the function is inherited between transfers. Further, the selection method should reduce conflicts between different species and the conflict of interest between population growth and community functions, while avoiding cheater phenotypes and inadvertent selection for biomass yield. Two main experimental designs have so far been used for experimental community selection. We evaluate and compare these, and a new method that we propose, by developing theoretical models and computational simulations of the selection methods. Our first results have given us a basal understanding of how the different selection methods work on a community level. Further, the simulations show that our proposed design can explore different species combinations to select for pollutant degradation among small communities. In addition, mutations and possible mixing between communities introduce the variance between communities that allows for evolution in the long term.  We now explore how robust the results are to model assumptions. Our study can aid the design of future artificial community selection experiments, and contribute to our fundamental understanding of multi-level selection.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Bryce-Morsky/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Bryce  Morsky
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Pennsylvania</em>
						    
						    
						    
					    		
					    			<h6>"Evolution of contribution timing in public goods games"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Life-history strategies are a crucial aspect of life, which are complicated in group-living species, where pay-offs additionally depend on others’ behaviours. Previous theoretical models of public goods games have generally focused on the amounts individuals contribute to the public good. Yet a much less-studied strategic aspect of public goods games, the timing of contri- butions, can also have dramatic consequences for individual and collective performance. Here, we develop two stage game theoretical models to explore how the timing of contributions evolves. In the first stage, individuals contrib- ute to a threshold public good based on a performance schedule. The second stage begins once the threshold is met, and the individuals then compete as a function of their performance. We show how contributing rapidly is not necessarily optimal, because delayers can act as ‘cheats,’ avoiding contributing while reaping the benefits of the public good. However, delaying too long can put the delayers at a disadvantage as they may be ill-equipped to compete. These effects lead to bistability in a single group, and spatial diversity among multiple interacting groups.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Calmelet-Jeanne-Colette/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Calmelet Jeanne Colette
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>California State University Chico</em>
						    
						    
						    
					    		
					    			<h6>"Surfactant Driven Flows in Respiratory and Ocular Regions"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Jensen equations describe the dynamics of an exogenous surfactant moving on the top of an endogenous fluid layer in biological systems.  The fluid motion is caused by the decrease in surface tension at the free surface of the thin film at the region of contact with the surfactant.  The variations in space and time of the surfactant concentration and film height are analyzed under different geometries.  We focus our attention on two biological applications; first, on the respiratory system, where an exogenous surfactant is inserted on the lining of a respiratory airway wall. Second, on the ocular region, where a surfactant is instilled on the eye lining. We conclude that the presence of the surfactant increases the fluid flow of the lining and therefore, can be used as therapeutic agent to prevent potential film rupture.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Cameline-Nafula/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Cameline  Nafula
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Maseno University</em>
						    
						    
						    
					    		
					    			<h6>"Modelling the spatio-temporal risk of measles outbreaks and options for their control in Kenya"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The measles control programme in Kenya is considered to be at it end phase. There has been long-term high level coverage of measles containing vaccine (MCV) at 9m reaching around 90% in 2010-12. Supplementary immunization activities (SIAs) are undertaken periodically (last done in 2016) to reduce the build-up of susceptibles in the age range 9m to 14 years. However, sporadic outbreaks continue, and data suggests vaccine uptake of MCV dose 1 has decreased over the last 5 years (WHO & UNICEF 2017, Manakongtreecheep & Davis 2017). A second dose of MCV was introduced in 2013 at 18months of age, but coverage is only at around 35%, and there is little confidence that this can easily be improved. There is national case-based surveillance, with follow up, through IgM serology from cases of rash illness. This study seeks to understand the possible reasons for continued outbreaks and to predict the implications of various vaccine strategies by modelling the Spatio-temporal risk of measles outbreaks and options for their control in Kenya.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Camile-FD-Kunz/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Camile FD  Kunz
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Goethe University - FIAS</em>
						    
						    
						    
					    		
					    			<h6>"Chemotaxis impact on pattern formation"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			During embryo development there is a rapid growth in cell numbers that forms complex structures. Skin pattern formation is an early process during the embryogenesis and happens before the cells fully differentiate. In the present project we consider skin patterning in mouse embryos, where cell aggregates form based on a hierarchical process, involving interactions between the epidermal cell populations. The reaction-diffusion pre-pattern is driven by fibroblast growth factor (FGF20), bone morphogenic protein (BMP) and WNT. Considering mathematical models, there are two main processes involved in the pattern formation: Turing reaction-diffusion systems and chemotaxis. The Turing system models the concentration of two interacting chemicals, and the patterns arises from an instability driven by a difference between their diffusion coefficients. Some previous studies show that this behavior is essential for self-organization in the mouse hair follicle and chicken feather pre-pattern formation. Another key mechanism is chemotaxis, where the cells move in the direction of a chemical attractant, where patterns can also be observed. Experimental data indicates a hierarchical system, where cell chemotaxis is guided by a Turing system. We aim at developing mathematical models to describe the underlying biological processes leading to skin patterning, especially the interaction of chemotaxis with reaction-diffusion (Turing) systems. A mathematical model using partial differential equations is solved numerically, and some results are presented and compared to the experimental data. We study the parameter-dependence of the model and different model structures, and their impact on the pattern forming process. According to the experimental data the Turing system and the chemotaxis seems to be intrinsically related on the mouse skin patterning. Using a numerical approach for the PDE system, we develop a framework to study quantitatively how chemotaxis and Turing systems are related and their impact on the patterning process.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Cara-Sulyok/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Cara Sulyok
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Tennessee, Knoxville</em>
						    
						    
						    
					    		
					    			<h6>"A Mathematical Framework to Augment the Q-MARSH Score in the Diagnosis of Celiac Disease"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This work provides a mathematical framework to better understand the effects of immune activation on gut health. This mathematical model uses a system of ordinary differential equations to track changes in villus and crypt cell densities as well as intraepithelial lymphocytes to better understand the dynamics of small intestinal destruction. The model will be used to investigate and analyze various theories behind the progression of celiac disease by focusing on understanding the dynamics of the small intestine in situations mirroring healthy function, celiac disease, and refractory celiac disease. By doing so, we can assist in further quantifying and augmenting diagnostic measures and investigate potential therapies to mitigate the negative effects of celiac disease.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Caren-Barcel%C3%B3/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Caren  Barceló
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Projecting the time scale of initial increase in fishery yield after implementation of marine protected areas"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Marine protected areas (MPAs) are being implemented globally to achieve conservation goals and benefit fisheries. However, MPAs require adaptive management to determine whether they are meeting their stated goals. This requires projection of the timing of increased performance after MPA establishment. Analyses of the projection of abundance and biomass have identified the information required and the expected timelines for meeting conservation goals. Projection of fishery yield, is more complex because it involves uncertain larval connectivity and the way in which the fishery is managed. Here, we develop a two-patch model with age structure represented by a renewal equation model to understand and to project the initial timing of the increase in fishery yield from larvae exported outside the MPA. By convolving a species-specific recruitment index with a yield-per-recruit equation, we are able to derive a yield function and yield timescale for each species. A key result is that the projection of yield is a weighted moving average of the larval production due to the projected MPA biomass, with the weightings being the age reversed age distribution of yield. Notably, this links MPA management to fishery management. Yield projection differs from the projection of MPA biomass in the sense that it depends on an uncertain factor due to uncertain connectivity. We demonstrate this mechanism with life histories of 16 harvested species found along the Pacific coast of the United States. The lag between the time of peak biomass within the MPA and the time that the increased fishery yield reaches its maximum depends on the pattern of the contribution to yield at each age. This age distribution of yield in turn depends on the age-dependent patterns of growth, natural mortality, and harvest. For the 16 species we considered, that lag ranged from 7 to 16 years. This general model and the range of exemplary species provide broadly applicable general guidance for this important emerging aspect of fisheries management.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Carley-Cook/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Carley Cook
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Oklahoma State University</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of the relationship between T cell produced Wnt10b and the bone remodeling cycle"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Bone health is determined by many factors including the remodeling cycle. At any time, many sections of bone are going through a remodeling cycle. Depending on different signaling factors, the cycle will end with the same amount of bone as at the beginning of the remodeling cycle (healthy) or increased or decreased amounts of bone (these changes contribute to chronic bone diseases). Recently immune cells have been identified as major signaling factors for this process. However, it is unclear how and to what extent they affect bone homeostasis. One way to better understand this phenomenon is to consider different foods or medicines that activate immune cells. LGG, for example, is a probiotic that increases butyrate production in the gut. Butyrate has been shown to indirectly increase bone density through a series of interconnected processes throughout the body that involve immune cells. One key process is the stimulation by regulatory T cells of production of Wnt10b within the bone compartment. This process has been shown to increase bone density. To quantify the bone density change caused by butyrate production a multi-compartment mathematical model was developed in two parts. The first part of the model predicts how much Wnt10b is increased in the bone marrow through the immune response (the processes occurring outside of the bone compartment and not included in this talk), and the second part predicts the change in bone homeostasis caused by the increase of Wnt10b (inside the bone compartment). Here, we focus on the bone compartment. Wnt10b has been shown to alter osteoblastgenesis, osteoblast apoptosis rate, and osteoblast bone formation rate, which collectively lead to the increase of bone density. To model this change, we adapted a previously published and well-cited model of bone remodeling.  This ODE model includes the cell types typically involved in remodeling such as osteoclasts, osteoblasts, and osteocytes. The model also includes an ODE that tracks the amount of bone present at the remodeling site. We have adjusted the three terms related to an increase of Wnt10b by adding three new parameters. The parameters are estimated using data collected on mice. However, because our model is based on human physiology only normalized information will be used. The data was taken from graphs in a consistent manner by utilizing Plot Digitizer. The values of the parameters are found using MATLAB lsqcurvefit and differential equation solver ode45. This model was then validated using a separate set of mice data. The completed model connects immune system T cells to the bone remodeling cycle. This model improves the understanding of immune cell disturbances to bone homeostasis and can help identify targets for medical intervention of bone loss.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Carlos-Enrique-Bustamante-Orellana/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Carlos Enrique  Bustamante-Orellana
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Arizona State University</em>
						    
						    
						    
					    		
					    			<h6>"Modeling and Preparedness: The Transmission Dynamics of COVID19 Outbreak in Provinces of Ecuador"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			COVID-19 disease has become a pandemic just a few months after it was first detected. Ecuador has reported one of the highest rates of COVID-19 in Latin America, with more than 62,000 cases and 8,500 deaths in a country of approximately 17 million people. The dynamics of the outbreak is being observed quite different in different provinces of Ecuador with high reported prevalence in some low population density provinces. In this study, we aim to understand the variations in outbreaks between provinces and provide assistance in essential preparedness planning in order to respond effectively to ongoing COVID-19 outbreak. This study estimated the critical level of quarantine rate along with corresponding leakage in order to avoid overwhelming the local health care system. The results suggest that provinces with high population density can avoid a large disease burden provided they initiate early and stricter quarantine measures even under low isolation rate. To best of our knowledge, this study is first from the region to determine which provinces will need much preparation for current outbreak in fall and which might need more help.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Caroline-Franco/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Caroline  Franco
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Institute of Theoretical Physics - Sao Paulo State University</em>
						    
						    
						    
					    		
					    			<h6>"Modelling non-pharmaceutical interventions to mitigate COVID-19 in Sao Paulo"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The SARS-CoV-2 pandemic has had an unprecedented impact on multiple levels of society. Not only has the pandemic completely overwhelmed some health systems but it has also changed how scientific evidence is shared and increased the pace at which such evidence is published and consumed, by scientists, policymakers and the wider public. With very little experimental scientific evidence, predictive mathematical models have played an increasingly prominent role advising policymakers, even in low- and middle-income countries, such as Brazil. Through the COVID-19 Modelling (CoMo) Consortium, an international group of infectious disease modellers and public health experts collaborated to create a modelling interface that could help simulating the effect of different non-pharmaceutical interventions on mitigating the epidemic in numerous locations. Here, we describe how we adapted this modelling framework to the Brazilian context and, more specifically, to the city of Sao Paulo.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Chandler-Gatenbee/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Chandler  Gatenbee
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Immune escape at the onset of human colorectal cancer"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The evolutionary dynamics of tumor initiation remain undetermined, and the interplay between neoplastic cells and the immune system is hypothesized to be critical in transformation. Colorectal cancer (CRC) presents a unique opportunity to study the transition to malignancy as pre-cancers (adenomas) and early stage cancers are frequently detected and surgically removed. Here, we examine the role of the immune response in tumor initiation by studying tumor-immune eco-evolutionary dynamics from pre-cancer to carcinoma. The integrated approach uses a computational model, ecological analysis of digital pathology, and multi-region neoantigen prediction. Model results indicate that there are several routes to malignancy, each of which uniquely shapes the tumor ecology and sculpts intra-tumor antigenic heterogeneity (aITH). These routes include combinations of evading detection via accumulating mutations with low antigenicity, the ability to block immune attack (e.g. PD-L1), and the ability to recruit immunosuppressive cells. Modeling predicts that, in general, the most common route from benign to malignant is the construction of an immunosuppressive niche. To determine which route is dominant in CRC initiation, we used a cohort of 21 colorectal adenomas, 15 carcinomas, and 26 adenomas with a focus of carcinoma (“ca-in-ad”) cases. The immune microenvironment was characterized using the spatial distribution of 17 markers across registered whole-slide images at 40x magnification, while patterns of intra-lesion aITH were described using multi-region neoantigen prediction. Observed changes in aITH, the tumor ecology, and spatial patterns of both cell associations and gene expression are consistent with simulations where immunogenic adenomas do not progress to CRC because they are under immune control. Conversely, adenomas that progress initially avoid detection through low immunogenicity, but gradually construct an immunosuppressive niche isolated from CD8+ cytotoxic T cells, thereby evading immune elimination and allowing for an increase in neoantigen burden. Both modeling and data indicate that immune blockade (e.g. PD-L1 expression) plays a secondary role to immune suppression in tumor initiation or progression. These results suggest that re-engineering the immunosuppressive niche may prove to be a most effective immunotherapy in CRC.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Chay-Paterson/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Chay   Paterson
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>InSync Technology</em>
						    
						    
						    
					    		
					    			<h6>"Cancer incidence as a form of convergent evolution"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Cancers occur after a gradual accumulation of mutations in a tissue. Together, these mutations enable cells to grow and spread in an uncontrolled way. This process takes many years, with one problematic lineage incrementally gaining an advantage over surrounding, normal tissue. This process repeatedly involves mutations on a few key oncogenes and tumour suppressors. Starting only with the sequences of a critical set of such genes and probability theory, we show that lifetime cancer risk can be calculated with no statistical fitting. We also show that certain orders of these mutations are more likely than others, and that these orders form a structure similar to a phylogenetic tree.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Chiara-Villa/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Chiara Villa
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of St Andrews</em>
						    
						    
						    
					    		
					    			<h6>"Mechanical pattern formation in biological tissue: Relax and go with the (viscous) flow"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mechanochemical models of pattern formation in biological tissue have helped us shed light on the role different mechanical cues have in cell aggregation phenomena, by considering the mechanical interaction between cells and the extracellular matrix (ECM). The cells and ECM are modelled as a linearly viscoelastic continuum, usually assumed to be a Kelvin-Voigt material, but this may not be the best model of viscoelasticity to use for biological tissue. We here extend the theory of mechanochemical pattern formation to include a wider variety of models of linear viscoelasticity. Our results clearly indicate that models of linear viscoelasticity presenting viscous flow (linear viscous, Maxwell, 3-parameter viscous model), which are better suited to represent soft tissue, have much higher pattern formation potential than those which do not (linear elastic, Kelvin-Voigt, standard linear solid model).
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Christopher-Carlson/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Christopher  Carlson
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Pennsylvania</em>
						    
						    
						    
					    		
					    			<h6>"Partner Specificity in Mutualisms"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mutualistic species vary in their level of partner specificity; yet, the evolutionary mechanisms which underpin partner specificity and generalism are not yet fully understood. One factor which may underpin variation in specificity is the degree of antagonism/cooperation in the relationship between hosts and symbionts. It is possible that mutualist hosts cooperatively specialize, maximizing mutual symbiotic benefit with a preferred symbiont, or antagonistically specialize, maximizing resource extraction from a preferred symbiont. Specialization in a preferred symbiont reduces the benefit of association with non-preferred symbionts, while generalists receive similar benefit from all symbionts. Here, we employ evolutionary game dynamics and adaptive dynamics in order to assess the evolutionary stability of cooperative specialization, antagonistic specialization, and generalism. When hosts specialize cooperatively, our system is bistable, favoring one of the specialist hosts and its preferred symbiont. When hosts specialized antagonistically, host and symbiont frequencies cycle continuously when average specialist payoff is greater than average symbiont payoffs. Higher average generalist payoff causes generalism to be an evolutionary stable strategy. Cooperative specialization unilaterally favors greater cooperation between specialist hosts and preferred symbionts, while antagonistic specialization leads to an evolutionary arms race in which symbionts attempt to escape host exploitation. We conclude that the cooperation-antagonism continuum which exists in mutualisms may play a key role in determining the pattern of partner specificity which develops within mutualistic relationships.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Cody-FitzGerald/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Cody FitzGerald
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Utah</em>
						    
						    
						    
					    		
					    			<h6>"Red light and the dormancy-germination decision in Arabidopsis seeds"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The Arabidopsis dormancy-germination transition is known to be environmentally-cued by red light and controlled by the competing hormones abscisic acid (ABA) and gibberellin (GA) produced by the seed. Recently, new molecular details emerged concerning the propagation of red light through a complex gene regulatory network involving PhyB, PIF1, and RVE1 and two feedback loops. This network influences the formation of the PIF1-RVE1 complex. The PIF1-RVE1 complex is a transcription factor that regulates the production of ABA and GA and helps shift the balance to high concentration of ABA and low concentration of GA, which corresponds to a dormant seed state. This new gene regulatory network has not been analyzed mathematically. Our analysis shows that this gene regulatory network exhibits switch-like bistability as a function of the red light input and makes a suite of biologically-testable predictions concerning seed dormancy and germination in response to the amplitude and periodicity of an oscillatory red light input.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Connah-Griffith-Michael-Johnson/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Connah Griffith Michael Johnson
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Warwick</em>
						    
						    
						    
					    		
					    			<h6>"Developing a computational tool for multiscale simulations of chemically coupled cell populations"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Many biological systems are spatially organised, from animals and plants to microbial communities. Mathematical modelling can help us improve our understanding of, and design better-informed experiments to probe, the dynamics of such systems. The development of computational tools for modelling spatially organised biological systems has largely focused on either so-called agent-based models or on physico-chemical models based on partial differential equations (PDEs) Agent-based models can readily incorporate cell-specific properties such as speciation, cell cycle dynamics, and replication. However, these models typically allow limited spatial resolution for chemical dynamics, have a high computational cost, and can be affected from user-implementation choices, such as the chosen sequence of simulation updating rules. In contrast, PDE-based models allow a much finer grained simulation based on densities of state variables and are well suited to simulations based on physical properties, geometry, mechanical motion, and chemical reactions; however, cell-specific attributes cannot be readily incorporated in these models. Here, we combine the benefits of agent-based and PDE models, by extending an existing software library, Chaste, to allow coupling between agent-based and PDE models. Chaste is a modular, open-source PDE solver platform that is widely used by the systems biology community already. It utilises finite element solvers to simulate individual reaction-diffusion equations, coupled to a mesh-based layer that defines structures acting as chemical sources or sinks. These PDE solvers may be solved across a mutable mesh embedded with a wide range of cell-based modelling paradigms with easily customisable modular cell behaviours, aspects enhanced when compared to similar softwares. Here, we expand this CHASTE functionality to allow the simulation of complex reaction-diffusion dynamics with multiple PDE variables and multiple cell structures. The resulting system will allows us to model and simulate multicellular systems coupled through any number of shared or communicated chemicals. As such, the new system will be suitable to study the dynamics of biological systems such as bacterial biofilms. In our own work, we aim to use this extended CHASTE platform to simulate early evolution of protocellular metabolic systems, in particular, reaction systems that are separated across cell-like phase separations in an otherwise homogenous primordial soup. Spatial dynamics in such early metabolic systems have not been considered to date and it will be interesting to characterise what kind of system dynamics can emerge under different parameter regimes of metabolite diffusion, phase dynamics, and reaction kinetics.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Cristeta-Jamilla/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Cristeta Jamilla
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of the Philippines Diliman</em>
						    
						    
						    
					    		
					    			<h6>"Inverse Problems for Neutral Delay Differential Equations using Nature-Inspired Optimization Algorithms"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Neutral delay differential equations (NDDEs) are useful in modelling real life phenomenon which includes time lag. In this paper, we use nature-inspired algorithms to solve inverse problems involving NDDEs. Specifically, we show how the nature-inspired optimization algorithms can be effective in estimating parameters of NDDEs. We compare the following recent algorithms: (1) particle swarm optimization, (2) genetic algorithm with multi-parent crossover, (3) whale optimization algorithm, (4) crow search algorithm, and (5) cuckoo algorithm.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Cristian-C.-E.-Espitia/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Cristian C. E. Espitia
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Campinas</em>
						    
						    
						    
					    		
					    			<h6>"A mathematical model of HIV/AIDS Spread in human population, the triangle transmission case."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			There exist individuals that change their sexual behavior depending on the situation or at different stages in their life. A possibly common and transient example of situational sexuality is the person who self-identifies as heterosexual, but will sexually interact with a member of the same sex when lacking other opportunities. Less transient but also possibly common, a person who self-identifies as gay or lesbian (either at the time, or later) may sexually interact with a member of the opposite sex if a same-sex relationship seems unfeasible, Thompson 2008, [1]. HIV/AIDS transmission usually considers sexual contact in heterosexual and homosexual population separately, besides in sexual transmission the same format for men and women is assumed. Thus, Can the population be split in heterosexuals and homosexual and thus the group of bisexuals be ignored? Can the sexual transmission form be equal for men and women? What is the contribution of a bisexual group in the HIV transmission? and, How to consider sexual transmission in men and women according to sexual behavior? To try to answer these questions we proposed an original mathematical model considering bisexuals in the HIV transmission. Mathematical analysis undertaken and stationary points, stability analysis of disease free equilibrium and boundary equilibrium, the basic reproductive number is obtained and discussed through the next generation method; numerical simulations show that these casual contacts between bisexuals has less influence than homosexual case.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/DaeWook-Kim/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	DaeWook  Kim
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>KAIST</em>
						    
						    
						    
					    		
					    			<h6>"Systems pharmacology model reveals the sources of the inter- and intraspecies variability in drug efficacy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The majority of previous studies investigate the drug efficacy only in nocturnal species (e.g. mice) although humans are diurnal. Here, using diurnal monkeys, we examine the effect of a daily (circadian) clock-modulator drug and find the high variability in its effect between diurnal monkeys and nocturnal mice. To identify the source of the interspecies variability, we used the systems pharmacology model, which accurately simulates the intracellular action of the drug and thus its effect in the circadian clock. This revealed that the interspecies variability in the drug effect is due to the different sensitivity of nocturnal and diurnal animals to environment light, the natural clock-modulator. Furthermore, via a combination of the model simulation and experiment, we found the molecular biomarker to predict the drug effect, which explains the high interindividual variability in the drug response. Based on these findings, we developed a model-based precision medicine strategy to treat circadian disruption. Our works show how the mathematical model can be used to reveal an unrecognized biological variable in drug efficacy translation between nocturnal and diurnal animals and enable precision medicine.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Daniel-Abler/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Daniel  Abler
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Bern</em>
						    
						    
						    
					    		
					    			<h6>"Tumor Growth and Biomechanics – Challenges & Opportunities"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Physical forces are recognized to play a critical role in shaping the micro-environment of tumors. Compression of cancer and stromal cells, as well as blood and lymphatic vessels, are direct consequences of mechanical solid stress, the compressive and tensile mechanical forces exerted by the solid components of the tissue. By altering the mechanical micro-environment of tumors, elevated solid stress can affect their pathophysiology, driving tumors to more aggressive phenotypes and compromise therapeutic outcome [1]. Mechanical stress also affects healthy tissue: It causes neuronal loss in brain tissue, and is linked to neurological deficits and reduced survival in patients with glioblastoma (GBM), the most common malignant primary brain tumor in adults. Given their far-reaching micro- and macroscopic consequences, tumor-induced mechanical forces may provide mechanistic insights into inter and intra-tumor heterogeneities, differential response to treatment and other phenotypical characteristics.
In this contribution, we survey the literature of spatial tumor growth modeling from a perspective of macroscopic tissue mechanics to assess the current status of mechanically-coupled growth models and to identify opportunities for further research: We summarize the types of modeling approaches previously used for capturing tumor-induced mechanical effects and their biological or physiological consequences. Based on this review, we identify the scenarios in which accounting for tissue mechanics proved to improve calibration to and prediction of clinical data. Drawing from examples of our and others’ research on mechanically-coupled growth modeling for GBM, we discuss challenges involved in the implementation and calibration of such models. In this context, we identify areas of mechanically-coupled growth modeling where further research is needed and explore application opportunities that such models may open.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Daniel-Koch/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Daniel  Koch
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>King's College London</em>
						    
						    
						    
					    		
					    			<h6>"In signal transduction, all you need are oligomers!  Dynamic signal encoding, homeostasis, bistability and more"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Homo-oligomerisation of proteins is a ubiquitous phenomenon whose exact role remains unclear in many cases. This talk will explore general dynamical mathematical models of homo-oligomerisation. I show that homo-oligomerisation on its own allows for a remarkable variety of complex dynamic and steady state regulatory behaviour such as transient overshoots or homeostatic control of monomer concentration. Post-translational modifications could make homo-oligomerisation even more versatile: by enabling pseudo-multisite modification and kinetic pseudo-cooperativity via multi-enzyme regulation, homo-oligomerisation can lead to bistability, thereby constituting a novel motif for bistable modification reactions. If modification and demodification follow different kinetic mechanisms the modification status of homo-oligomers can furthermore exhibit sustained oscillations. Due to these potential signal processing capabilities, homo-oligomerisation could play far more versatile roles in biochemical signal transduction than previously appreciated.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Daniel-J-Glazar/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Daniel J Glazar
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Tumor growth and inhibition model predicts progression in recurrent high-grade glioma"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Recurrent high-grade glioma (HGG) remains incurable with inevitable evolution of resistance and high inter-patient heterogeneity in time to progression (TTP). Here, we evaluate if early tumor volume response dynamics can calibrate a mathematical model to predict patient-speciﬁc resistance to develop opportunities for treatment adaptation for patients with a high risk of progression. A total of 95 T1-weighted contrast-enhanced (T1post) MRIs from 14 patients treated in a phase I clinical trial with hypo-fractionated stereotactic radiation (HFSRT; 6 Gy × 5) plus pembrolizumab (100 or 200 mg, every 3 weeks) and bevacizumab (10 mg/kg, every 2 weeks; NCT02313272) were delineated to derive longitudinal tumor volumes. We developed, calibrated, and validated a mathematical model that simulates and forecasts tumor volume dynamics with rate of resistance evolution as the single patient-speciﬁc parameter. Model prediction performance is evaluated based on how early progression is predicted and the number of false-negative predictions. The model with one patient-speciﬁc parameter describing the rate of evolution of resistance to therapy ﬁts untrained data (R^2=0.70). In a leave-one-out study, for the nine patients that had T1post tumor volumes ≥1cm^3 , the model was able to predict progression on average two imaging cycles early, with a median of 9.3 (range: 3–39.3) weeks early (median progression-free survival was 27.4 weeks). Our results demonstrate that early tumor volume dynamics measured on T1post MRI has the potential to predict progression following the protocol therapy in select patients with recurrent HGG. Future work will include testing on an independent patient dataset and evaluation of the developed framework on T2/FLAIR-derived data.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Daniel-Tudor/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Daniel Tudor
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Edinburgh</em>
						    
						    
						    
					    		
					    			<h6>"Probing immune cell wound recruitment signals using Bayesian inference and random walk models"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The recruitment of immune cells to wounds is a complex spatiotemporal process with the production and diffusion of chemoattractants acting as a beacon for immune cells to respond to damage within the body. Analysing these chemoattractants can be experimentally complex, however, inference of the chemoattractant field is possible by analysing cell trajectories. These trajectories can then be used to infer the main parameters of the underlying chemoattractant.
To undertake this study, we reproduced a previously published modelling framework which utilises biased-persistent random walk to capture immune cell motion and the diffusion equation to capture the chemoattractant dynamics. By applying Bayesian inference, this framework allows us to gain an understanding of the relationship between cell migration parameters and the main chemoattractant parameters such as the diffusion coefficient and production time.
To aid transparency, we implemented an open source version of the modelling framework to allow for future research. We then applied this model to investigate the chemoattractant which is responsible for wound healing within Drosophila, this chemoattract is currently unknown and can be difficult to isolate through experiments. However, by applying the inference model it is possible to isolate the gene responsible for the expression of the chemoattractant. We compared wild type and gene deleted (mutant) datasets and found a significant difference between inferred parameters, which implies that gene deletion is consistent with no production of chemoattractant.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Darwin-Bandoy/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Darwin  Bandoy
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of California,Davis</em>
						    
						    
						    
					    		
					    			<h6>"Pandemic dynamics of COVID-19 using epidemic stage, instantaneous reproductive number and pathogen genome identity (GENI) score: modeling molecular epidemiology"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Background: Global spread of COVID-19 created an unprecedented infectious disease crisis that progressed to a pandemic with >180,000 cases in >100 countries. Reproductive number (R) is an outbreak metric estimating the transmission of a pathogen. Initial R values were published based on the early outbreak in China with limited number of cases with whole genome sequencing. Initial comparisons failed to show a direct relationship viral genomic diversity and epidemic severity was not established for SARS-Cov-2. Methods: Each country's COVID-19 outbreak status was classified according to epicurve stage (index, takeoff, exponential, decline). Instantaneous R estimates (Wallinga and Teunis method) with a short and standard serial interval examined asymptomatic spread. Whole genome sequences were used to quantify the pathogen genome identity score that were used to estimate transmission time and epicurve stage. Transmission time was estimated based on evolutionary rate of 2 mutations/month. Findings: The country-specific R revealed variable infection dynamics between and within outbreak stages. Outside China, R estimates revealed propagating epidemics poised to move into the takeoff and exponential stages. Population density and local temperatures had variable relationship to the outbreaks. GENI scores differentiated countries in index stage with cryptic transmission. Integration of incidence data with genome variation directly increases in cases with increased genome variation. Interpretation: R was dynamic for each country and during the outbreak stage. Integrating the outbreak dynamic, dynamic R, and genome variation found a direct association between cases and genome variation. Synergistically, GENI provides an evidence-based transmission metric that can be determined by sequencing the virus from each case. We calculated an instantaneous country-specific R at different stages of outbreaks and formulated a novel metric for infection dynamics using viral genome sequences to capture gaps in untraceable transmission. Integrating epidemiology with genome sequencing allows evidence-based dynamic disease outbreak tracking with predictive evidence.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/David-Cheek/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	David  Cheek
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Harvard University</em>
						    
						    
						    
					    		
					    			<h6>"Genetic composition of an exponentially growing cell population"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We study a simple model of DNA evolution in a growing population of cells. Each cell contains a nucleotide sequence which randomly mutates at cell division. Cells divide according to a branching process. Following typical parameter values in bacteria and cancer cell populations, we take the mutation rate to zero and the final number of cells to infinity. We prove that almost every site (entry of the nucleotide sequence) is mutated in only a finite number of cells, and these numbers are independent across sites. However independence breaks down for the rare sites which are mutated in a positive fraction of the population. The model is free from the popular but disputed infinite sites assumption. Violations of the infinite sites assumption are widespread while their impact on mutation frequencies is negligible at the scale of population fractions. Some results are generalised to allow for cell death, selection, and site-specific mutation rates. For illustration we estimate mutation rates in a lung adenocarcinoma.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/David-Gurarie/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	David  Gurarie
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>CWRU</em>
						    
						    
						    
					    		
					    			<h6>"Individual-based modeling of Covid-19 in local community settings"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Individual based modeling of disease transmission (IBM) offers an attractive alternative to population based approaches e.g. (continuous DE), as it allows a detailed account of biological (risk) factors, environment, and behavior. This is particularly relevant in local community settings (hospital, workplace, school, city district or county), where finite population size and host heterogeneity, in terms social interactions and disease progression, make a ‘continuous approach’ impractical. We develop such IBM methodology to simulate Covid-19 outbreaks in local settings, and explore different control-mitigation strategies. Our models feature multiple disease pathways (asymptomatic, mild and severe) typical of Covid-19, as well as heterogeneous host communities with different susceptibility levels and structured social contacts. Individual hosts undergo SEIR disease progression (Susceptible, Exposed - presymptomatic, Infected-symptomatic, Recovered) of variable stage-duration and infectivity. The crucial (S->E) transition is determined by host ‘contact-pool’ on daily basis. Unlike conventional social-contact network (‘one-to-one’ transmission), our setup features ‘many-to-many’ (multigraph) transmission. Two typical IBM examples include (i) hospital, made of interacting healthcare workers (HCW) and patients, (ii) school/college, where students + staff aggregate in classrooms, dorms and engage in other (social, sport) activities. In both cases, we used available data (a hospital in Wuhan, a college in US) to set up and calibrate our models. Different control/mitigation strategies were explored, including symptomatic and asymptomatic testing and isolation, use of PPE (hospital), social distancing, and contact tracing (college). We assessed the efficacy of each intervention, and resources required to prevent or mitigate the outbreak.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Debasmita-Mukherjee/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Debasmita Mukherjee
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Department of Statistics, Sunandan Divatia School of Science, SVKM’s NMIMS Deemed to be University, Mumbai, 400056, India</em>
						    
						    
						    
					    		
					    			<h6>"Atherosclerosis: A Mathematical Overview"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Atherosclerosis is a chronic inflammatory disease occurs due to plaque accumulation in the intima, the innermost layer of artery. Atherosclerosis is one of the prime causes behind several cardiovascular diseases over the worldwide. Here the entire biochemical process of atherosclerotic plaque formation is presented in terms of an autonomous system of ten nonlinear ordinary differential equations. Concentrations of low density lipoproteins (LDLs), high density lipoproteins (HDLs), free radicals, oxidized LDLs, chemoattractant, monocytes, macrophages, T-cells, smooth muscle cells (SMCs) and the necrotic core (or plaque cells) are assumed as the dependent variables in this nonlinear system. The present model has been found to be globally stable. Quasi steady state approximation theory is used to reduce the ten dimensional nonlinear system into a three dimensional nonlinear system. Numerical analysis of this reduced system has revealed the impact of some significant model parameters, which can be taken forward to develop some clinical strategies in controlling this disease dynamics.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Deena-Schmidt/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Deena  Schmidt
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Nevada, Reno</em>
						    
						    
						    
					    		
					    			<h6>"Building age-structured network models from interaction data"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Many methods exists for generating networks with certain pre-specified properties. However, there are network properties that arise in certain applications for which we don't have standard methods. For example, age or sub-population structure in biological applications can be a very important determinant of node connectivity, but methods for constructing networks with a given structure are still being developed. In this talk, I will discuss a method we developed for generating an age-structured human interaction network using survey data that summarizes the number of interactions between individuals within and between different age groups. We do this using the well-known POLYMOD dataset to construct an age-structured infectious disease transmission network.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Denis-H%C3%BCnniger/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Denis  Hünniger
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Applied Sciences Dresden</em>
						    
						    
						    
					    		
					    			<h6>"Effects of tumor-originating niches on intra-tumor heterogeneity"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Intra-tumor heterogeneity plays a crucial role during tumor initiation and progression. In practice, information about the genetic diversity in tumors is needed for developing individual therapies. However, there are still open questions in which manner intra-tumor heterogeneity evolves throughout tumor progression. In particular, it is unclear to which extent the architecture of the originally healthy tissue determines spatial patterns of intra-tumor heterogeneity. In this context, recent studies on the competition between tumor cells and wild-type cells lead to the concept of tumor-originating niches: Niches consist of a few cells whose competition during tumor initiation may predetermine the heterogeneity of the macroscopic, detectable tumor [1]. We study effects of tumor-originating niches on intra-tumor heterogeneity of the observed tumor and contrast them with the classical approach, where a tumor grows from a single cell. We examine in which manner mutations are spatially distributed throughout a tumor under presence and absence of tumor-originating niches. More precisely, we analyze the corresponding time scales of tumor initiation and identify spatial patterns of intra-tumor heterogeneity. To accomplish this, we use stochastic cellular automata and Markov theory for modeling, simulation and analysis. Understanding the spatial distribution of intra-tumor heterogeneity which originate from niches will contribute to more reliable prognoses in cancer therapy.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Derek-Park/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Derek  Park
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Synergizing chemotherapy with immune and evolutionary tradeoffs: Searching for Goldilocks"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			A mainstay treatment for many cancers is chemotherapy, for which the dosing strategy is primarily limited by patient toxicity. While this Maximum Tolerated Dose (MTD) approach builds upon the intuitively appealing principle that maximum therapeutic benefit is achieved by killing the largest possible number of cancer cells, there is increasing evidence that moderation may be better. The increasing use of immune therapies which seek to use the patient’s own immune system therapeutically, bring the effectiveness of MTD into question. In some cases, there may exist a ‘Goldilocks Window’ of sub-maximal chemotherapy that yields improved overall outcomes. This window reflects the complex interplay of cancer cell death, changes in immune function, emergence of chemoresistance, and the potential for metastatic dissemination. Importantly, the many changes induced by chemotherapy have tradeoffs that depend on the specific agents being used, as well as their dosing levels and scheduling. We present experimental and clinical observations across cancer types that support the idea that MTD may not always be the best approach. Our mathematical model driven results indicate which patient states would benefit most from a Goldilocks chemotherapy dosing schedule. Implementation of such a personalized treatment regime, that incorporates insights from eco-evolutionary dynamics, will require the integration of predictive mathematical models of tumor-immune responses to therapy with appropriate patient specific clinical data.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Dhananjay-Bhaskar/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Dhananjay  Bhaskar
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Center for Biomedical Engineering, Brown University</em>
						    
						    
						    
					    		
					    			<h6>"Quantifying cellular (re)-organization in 3-D cancer models using persistent homology"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Automated analysis of high-throughput, time-lapse microscopy data is essential for the development of multi-scale, patient-specific models that accurately mimic the complex behavior of cells observed in vivo. Many state-of-the-art methods for processing 3-D microscopy datasets rely on supervised machine learning methods for image segmentation, cell tracking and cell shape classification. These methods are computationally expensive, requiring difficult-to-obtain training data and parameter tuning. We propose an alternative approach, based on topological data analysis, to quantify changes in tumor architecture by analyzing point clouds obtained from cell nuclei positions. Using persistent homology, a topological barcode is extracted from each point cloud, which corresponds to the presence of topological features (clusters, acini and lumens) at multiple spatial scales. The barcode provides a unique insight into the spatial organization of data, which is often missing from typical analyses based on machine learning and statistics. By linking topological barcodes across time, the temporal persistence of topological features can be measured. The proposed methodology is able to identify nuclei associated with distinct clusters, acini and lumens in an unsupervised manner. Using this information, we ascertain the movement of cells between topological features. Furthermore, we classify qualitatively distinct organizational structures by clustering based on pairwise Wasserstein distances between topological barcodes. In this talk, I will introduce our methodology and demonstrate its potential for investigating tissue reorganization during tumorigenesis and metastasis.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Dhiraj-Kumar-Das/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Dhiraj Kumar Das
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Institute of Engineering Science and Technology, Shibpur</em>
						    
						    
						    
					    		
					    			<h6>"Influence of the smear-microscopy in global dynamics of tuberculosis transmission"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Tuberculosis, a lethal infectious disease attribute among the top 10 causes of death globally and leading cause of death from a single infectious pathogen (rank before HIV). The sputum smear-microscopy and chest X-ray are the key TB diagnosis methods available in resource-constrained health settings of many developing countries. The specificity of the diagnostic method is satisfactory whereas, sensitivity is limited and cannot detect pulmonary tuberculosis (PTB) cases below a bacterial load of 1000 organism/ml. According to the sensitivity of this diagnostic method,  PTB patients are categorized into two kinds: (a) smear-positive and (b) smear-negative. Interestingly, this categorization also scales the infectivity of PTB patients in the community. Our current study addresses this heterogeneity in the infectiousness of PTB individuals to investigate its consequences in disease dynamics.
A five-dimensional compartmental model is formulated considering the infectivity of both the smear-positive and negative PTB individuals. The expression of the basic reproduction number ($R_0$) is obtained through the next-generation matrix method. The asymptotic behaviour of the model is thoroughly discussed around the steady states of the proposed model. The global asymptotic stability of the equilibrium points is established using suitably constructed Lyapunov functions. It is observed that the disease-free equilibrium is globally asymptotically stable for $R_0<1$ and the disease-persistent equilibrium point is the same whenever $R_0>1$. The study also provides a list of normalized forward sensitivity indices of $R_0$ with respect to the involved parameters. This list showcases the influential level of the associated parameters in determining the size of the threshold quantity $R_0$. It has been found that neglecting the transmission capacity of the smear-negative individuals underestimates the value of $R_0$ whereas, ignoring the smear-negative compartment overestimates the same quantity. We also implement numerical simulations whenever necessary, using a suitable TB parameter set to visualize the obtained analytical results.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Diego-Samuel-Rodrigues/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Diego Samuel Rodrigues
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidade Estadual de Campinas</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of pharmacokinetic profiles of magnetic nanoparticles acquired by multichannel alternating current biosusceptrometry"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This study is about a novel ordinary differential equation model aimed at describing in vivo biodistribution of tracer magnetic nanoparticles (MNP). The proposed pharmacokinetic model is built based on in vivo experimental data gathered from images of alternating current biosusceptometry. Three compartments are considered: the one in which MPN is found free the blood, another for a reversible bound state of MNP with the liver sinusoid, and a third compartment for the MNP permanently absorbed by the Kupffer cells. As a result, the proposed model allowed us to suitably describe the experimental pharmacokinetic profiles, providing a promising theoretical-quantitative basis for the results of the experiments [1]. Besides, it also suggests that the multichannel biosusceptrometry system is a valuable imaging device to assess in vivo and real-time pharmacokinetics. As the latter was the main purpose of the first publication in the subject [1], further developments are needed to improve the model's parameter estimation using a Bayesian framework. The author intends to implement this last idea soon.
References: [1] Soares, G.; Próspero, A.; Calabresi, M.; Rodrigues, D. S.; Simões, L.; Quini, C.; Matos, R.; Pinto, L.; Sousa, A.; Bakuzis, A.; Mancera, P.; Miranda, J. R. Multichannel AC biosusceptometry system to map biodistribution and assess the pharmacokinetic profile of magnetic nanoparticles by imaging. IEEE Transactions on NanoBioscience, v. 18, 456–462, 2019. doi.org/10.1109/TNB.2019.2912073. Impact Factor by JCR (2018): 1.927.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Divyoj-Singh/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Divyoj  Singh
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Institute of Science</em>
						    
						    
						    
					    		
					    			<h6>"Emergent Properties of the HNF4α-PPARγ Network May Drive Consequent Phenotypic Plasticity in NAFLD"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Non-alcoholic fatty liver disease (NAFLD) is the most common form of chronic liver disease in adults and children. It is characterized by excessive accumulation of lipids in the hepatocytes of patients without any excess alcohol intake. With a global presence of 24% and limited therapeutic options, the disease burden of NAFLD is increasing. Thus, it becomes imperative to attempt to understand the dynamics of disease progression at a systems-level. Here, we decoded the emergent dynamics of underlying gene regulatory networks that were identified to drive the initiation and the progression of NAFLD. We developed a mathematical model to elucidate the dynamics of the HNF4α-PPARγ gene regulatory network. Our simulations reveal that this network can enable multiple co-existing phenotypes under certain biological conditions: an adipocyte, a hepatocyte, and a “hybrid” adipocyte-like state of the hepatocyte. These phenotypes may also switch among each other, thus enabling phenotypic plasticity and consequently leading to simultaneous deregulation of the levels of molecules that maintain a hepatic identity and/or facilitate a partial or complete acquisition of adipocytic traits. These predicted trends are supported by the analysis of clinical data, further substantiating the putative role of phenotypic plasticity in driving NAFLD. Our results unravel how the emergent dynamics of underlying regulatory networks can promote phenotypic plasticity, thereby propelling the clinically observed changes in gene expression often associated with NAFLD. This abstract is taken from an already published research paper co-authored by me (https://www.mdpi.com/2077-0383/9/3/870).
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Domenic-PJ-Germano/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Domenic PJ  Germano
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>The University of Melbourne</em>
						    
						    
						    
					    		
					    			<h6>"Towards a realistic 3D deformable model of tissues"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Colorectal Cancer is one of the most prevalent forms of cancer within western society. It is known to develop within the epithelia of the colon, localised to distinct invaginations within the intestinal wall, known as the crypts of Lieberkürn. While much is known about these crypts, the biomechanical process responsible for their structural maintenance remains unknown. One such process believed to be responsible for the crypts structural stability is believed to be a result of the surrounding stromal tissue.
Throughout this talk a 3D, multilayer, cell-centre model of tissue deformation will be presented, where cell movement is governed by the minimisation of a bending potential across the epithelium, and cell-cell adhesion. Using this model, we hope to provide a realistic description of colonic crypt epithelium. Thus far, we have found that the model is capable of describing generalised tissue deformations, and we hope to extend it to describe crypt homeostasis.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Dominic-Brass/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Dominic  Brass
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UK Centre for Ecology and Hydrology</em>
						    
						    
						    
					    		
					    			<h6>"Phenotypic plasticity as a cause and consequence of population dynamics"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Predicting how species respond to dynamically changing and novel environments is crucial for guiding conservation and mitigation strategies. Phenotypic plasticity is a mechanism of trait variation demonstrably important in determining how individuals and populations adapt to environmental change. The effects of phenotypic plasticity can be quantified in individuals by measuring environment-trait relationships, but it is often difficult to predict how phenotypic plasticity affects populations from environment-trait relationships alone. Variation in the life-history traits expressed by individuals may alter population processes, and this in turn can feedback to induce further variation in the traits expressed by individuals. This means the assumption that environment-trait relationships validated for individuals are representative of how populations respond to environmental change risks mischaracterising the effect of environmental change on populations. Predicting the effect of phenotypic plasticity on populations necessitates the development and utilisation of specialised predictive tools able to integrate empirically verified mechanisms of trait variation into a population's dynamical processes. We have derived a novel general mathematical framework linking trait variation due to phenotypic plasticity to population dynamics which we apply to the classical example of Nicholson's blowflies. This application reveals a rich set of counter-intuitive population-dynamical behaviours and highlights how seemingly sensible predictions about how environment-trait relationships generalise to population responses break down in the context of a populations dynamical processes. Our results demonstrate the importance of the interplay between phenotypic plasticity and population dynamics and the need to account for the effects of trait variation when making predictions about population responses to environmental change.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Dominic-Olver/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Dominic Olver
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Saskatchewan</em>
						    
						    
						    
					    		
					    			<h6>"Time Dependent Osmotic Damage in Sea Urchin Oocytes"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Most cryobiological protocols require loading and unloading of cryoprotective agents (CPAs) to mitigate ice damage during the freezing and thawing process. However, CPAs change the osmolality of the solution creating an osmotic gradient across the cell membrane, causing large volumetric changes. Classically, mechanical damages due to swelling or shrinking have been thought to have a constant osmotic tolerance limit (E.g. 20% reduction in survival of the population at a given hyper and hyposmolality), which are crucial in determining optimized cryoprotocols. Here we show that osmotic damage is not dependent solely on volume deviance for sea urchin (Paracentrotus lividus) oocytes, but instead osmotic damage is time-dependent. We exposed urchin oocytes (n >= 100 per treatment with 3 replicates) to two hypertonic treatments at differing osmolalities (1500, 2000, and 2500 mOsm/kg). The hypertonic solutions either had NaCl or Sucrose added to holding medium (Sea water 1000 mOsm/kg) to obtain the desired osmolality.  The exposure duration periods were for 2, 6, 15, 30, 50, 75, and 90 minutes. We tested hypotonic damage by diluting the holding medium with DI water exposing them to osmolalities of 800, 700, 600, and 500 mOsm/kg. After exposure, oocytes were returned to isosmotic holding media, in vitro fertilization was performed, and development to the 4-arm-Pluteus stage was assessed at 48 h. We fit these data to a mathematical model of population cell death that is proportional to the integration of the absolute value of the isosmotic volume minus the cell volume throughout time.  This model works well (Adjusted R2 values of 0.97, 0.96, and 0.76 for DI water, NaCl, and Sucrose respectively) to describe osmotic related damage across multiple concentrations and solution types. The next step is to include this novel model as a refined metric of mechanical or osmotic stress instead of standard osmotic tolerance limit models when determining optimal CPA equilibration protocols using cytotoxicity cost functions. This may result in more accurate models of cell damage, and better optimized protocols for loading and unloading of CPAs.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Donggu-Lee/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Donggu Lee
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Konkuk University</em>
						    
						    
						    
					    		
					    			<h6>"Role of OCT1 in regulation of miR-451-LKB1-AMPK-OCT1-mTOR core signaling network and cell invasion in glioblastoma."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Glioblastoma multiforme (GBM) is the most aggressive form of brain cancer with the short median survival time. GBM is characterized by the hallmarks of aggressive proliferation and critical cellular infiltration. miR-451 and their downstream molecules (LKB1, AMPK, OCT1, mTOR) are known to play a pivotal role in regulation of the balance of proliferation and aggressive invasion in response to metabolic stress in a tumor microenvironment (TME). Recent studies show that OCT1 and LKB1 play a significant role in regulation of the mutual inhibition between cell proliferation and migration. In this work, we develop a mathematical model of signaling pathway dynamics in GBM evolution with particular focus on the relative balance of proliferative capacity and invasion potential. In the present work we represent the miR-451/LKB1/AMPK/OCT1/mTOR pathway by a simple model and show how the effects of fluctuating glucose on tumor cells need to be reprogrammed by taking into account the recent history of glucose variations and an AMPK/miR-451 reciprocal feedback loop. The simulations show how variations in glucose significantly affect the level of signaling molecules and, in turn, lead to critical cell migration.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Easton-R.-White/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Easton R. White
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Vermont</em>
						    
						    
						    
					    		
					    			<h6>"Catastrophes and socio-ecological systems"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Amid global pandemics and climate change, it is clear that coupled models of human and environmental systems are needed. These socio-ecological models have been used to understand fisheries, disease spread, and deforestation. These models have only recently been used to understand how human actions and behavior affect coral reef fisheries, with a focus on shifts to alternative stable states. We extend this work to study the effect of catastrophes, e.g. hurricanes, on these systems. We show the conditions necessary for long-term coral reef health with fishing. We also examine the effect of the disturbance regime (timing, magnitude, type) on the overall system dynamics. These results both advance our understanding of catastrophes and socio-ecological systems as well as point to ways to build fisheries that are robust to rare events.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Editha-Jose/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Editha Jose
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of the Philippines Los Banos</em>
						    
						    
						    
					    		
					    			<h6>"Complex Balanced Equilibria of Weakly Reversible Poly-PL Kinetic Systems and Evolutionary Games"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This talk is concerned with chemical reaction networks endowed with poly-PL kinetics, that is, the positive linear combination of power law kinetic systems. We discovered that complex balanced equilibria exist for weakly reversible poly-PL kinetics with zero kinetic reactant deficiency. The result is then applied to evolutionary games with replicator dynamics such that the polynomial payoff functions lead to polynomial kinetic systems, a subset of poly-PL kinetic systems. In particular,
sufficient conditions to admit a zero kinetic reactant deficiency were derived for games with nonlinear payoff functions and poly-PL kinetics, allowing the application
of the main result.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Elizabeth-A-Fedak/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Elizabeth A Fedak
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Utah</em>
						    
						    
						    
					    		
					    			<h6>"Getting mixed messages: How p53 controls its dynamics to interpret variable upstream signals"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			p53 is one of the most widely studied proteins in molecular biology for its central role in tumorigenesis. In a healthy, replicating cell, p53 makes cell fate decisions based on signals it receives from repair pathways. Not only must p53 consolidate information from multiple sources, the signals it receives do not correspond exactly to the total amount of damage in the cell; rather, comparably lethal amounts of damage can induce dissimilar signals. For example, gamma radiation induces DNA lesions that p53-activating kinases bind to within minutes, while the DNA lesions created by UV radiation are harder for the cell to detect and only communicate with p53-activating kinases during repair.
Using a mechanistic model, we argue that this difference in response speed causes distinct dynamical profiles of p53 to arise. If p53 receives a strong signal with a short duration, as it would for a low dose of gamma radiation, the cell would be susceptible to premature apoptosis if p53 became overactive due to this signal. Instead, causing p53 to oscillate weakens its response and the signal only recovers if the damage persists after the initial round of suppression. For UV radiation, the delay between damage induction and communication to p53 creates a signal that starts low and increases over several hours. An under-regulated system may ignore a weak but long-lasting signal even if it represents extensive DNA damage. Instead, because this system can escape to a bistable region with higher levels of p53 at intermediate levels of activating signal, the cell can compensate for low kinase activation by raising the amount of available substrate. This allows active p53 to accumulate when exposed to a low but durable signal. Other models have focused on the mechanistic cause of p53 oscillations; this model provides a hypothesis as to why they exist.
Here, we focus on the surprising hypotheses that arise from reconciling p53's paradoxical behavior and discuss how this model extends our knowledge of tumor survival strategies.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Eman-Alwani/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Eman Alwani
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>The University of Sheffield</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical Analysis of Feedback Requirements for Planar Polarisation in the Fly Wing"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			During animal development, oriented cell behaviours are required to ensure appropriate growth and structure. Planar polarity, which describes polarisation within the plane of a cell sheet, is an important example of such behaviours. During this project, I built a mathematical model aiming to gain a qualitative understanding of the requirements for different feedback interactions to establish planar polarisation in the fly wing.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Emilia-Kozlowska/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Emilia  Kozlowska
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Silesian University of Technology</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of palliative treatment in non-small cell lung cancer"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The most common subtype of lung cancer is non-small cell lung cancer (NSCLC) that constitutes 80% of all lung cancer cases. NSCLC is usually diagnosed at an advanced stage because of non-specific symptoms, leading to high mortality. The standard treatment for NSCLC patients is a combination of chemotherapy and radiotherapy and, as emerging mode of treatment, immunotherapy. We collected, from a retrospective cohort of patients, 47 patients treated with platinum-doublet chemotherapy with a palliative intent or with symptoms treatment only. Thus, the patients are treated under the assumption of failed cure. From the cohort of patients, clinical data were collected, which serve as input data for computational platform. We developed a computational platform including a machine learning algorithm and a mechanistic mathematical model to find the best protocol for administration of platinum-doublet chemotherapy in palliative setting. The core of the platform is the mathematical model, in the form of a system of ordinary differential equations, describing dynamics of platinum-sensitive and platinum-resistant cancer cells and interactions reflecting competition for space and resources. The model is simulated stochastically by sampling the parameter values from a joint probability distribution. Machine learning algorithm is applied to calibrate the mathematical model and to fit it to overall survival curve. The model simulations faithfully reproduce the clinical cohort at three levels, long-term response (OS), initial response, and the relationship between number of chemotherapy cycles and time between two consecutive chemotherapy cycles. In addition, we investigated the relationship between initial and long-term response. We showed that these two variables do not correlate, which means that we cannot predict patient survival based solely on initial response. We also tested several chemotherapy schedules to find the best one for patients treated with a palliative intent. We found that optimal treatment schedule depends, among other, on the strength of competition among sensitive and resistant subclones in a tumor.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Emily-Zhang/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Emily Zhang
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>NC State</em>
						    
						    
						    
					    		
					    			<h6>"Personalized Time Series Forecasting of Blood Glucose Levels"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The development of data-driven capabilities for feedback control in the treatment of Type 1 Diabetes (T1D) requires the accurate prediction of future blood glucose (BG) levels. Specifically, the ability to predict BG levels in 30 and 60 minute time horizons could enable the time-dependent adjustment of treatment in response to the ensuing status of the patient, i.e., if hyper/hypo-glycemia occurs. By providing real-time data from continuous BG monitors, wearable sensor measurements, and self-reporting through mobile applications, the BG Level Prediction Challenge has enabled the capability to test whether models calibrated to individual-level data could ultimately be used for making individualized treatment decisions in T1D.


We trained and analyzed several direct prediction strategies, including different neural network architectures, reservoir computing, and linear regression. We found that the use of multiple linear regression models was the most accurate prediction strategy, and that reservoir computing has both the prediction power and the ability to recover the dynamics from missing intervals.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Emma-L-Fairbanks/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Emma L Fairbanks
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Nottingham</em>
						    
						    
						    
					    		
					    			<h6>"Re-parameterisation of a mathematical model of AHSV using data from literature"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Midge-borne arboviruses were once restricted to other geographical regions; however due to climate change and increased globalisation these diseases now pose a threat to the UK, with outbreaks having already occurred. African horse sickness virus (AHSV) is endemic in parts of Africa. An outbreak in Spain 1987-1990, which spread to Portugal and Morocco, demonstrated the ability of this virus to spread within Europe.
A previously published model suggested an ordinary differential equation model for AHSV in which parameters were derived from three published studies. In order to better inform the model studies documenting experimental infection of equids in vaccination trials were systematically reviewed. As we were interested in modelling emergence of AHSV in a naive population, only experimental infections of control (i.e. naive) animals were considered. Parameters derived from the systematic review were the time until the onset of viraemia, clinical signs and death after experimental infection of a naive equid. The mean latent period of horses was found to be 4.6 days, longer than previously estimated (3.7 days). The infectious periods of dying and surviving hosts were found to be 3.9 and 8.7 days, whereas previous estimations where 4.4 and 6 days, respectively. The host mortality rate was also found to be higher than previous estimations. Model simulations were compared for the previously published models parameters and an updated set of parameter values derived from the systematic review and other literature. The updated parameter values resulted in an increase in the number of host deaths and decrease in the duration of the outbreak. We also observed many more vector infections in simulations using the updated parameters. Sensitivity analysis showed that the host latent period and vector to host ratio had the greatest impact on simulation outputs.
The vector parameters in this model were also updated using literature. However, many of these were from studies on american vector species. Therefore, the stages of this work involve fitting a model developed for the vector populations to UK trap data.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Emma-Southall/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Emma Southall
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Warwick</em>
						    
						    
						    
					    		
					    			<h6>"Identifying indicators of critical transitions in epidemiological data"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			A challenging problem in infectious disease modelling is assessing when a disease has been eliminated. Control campaigns have substantial economic consequences; as such there are high demands to reduce costs and reallocate resources. However, if campaigns are stopped prematurely it can result in disease resurgence and subsequently put control efforts back by decades. Early-warning signals offer a computationally inexpensive technique to monitor the progress towards elimination, using statistical indicators calculated on time series data.
Early-warning signals are widely used in many fields to anticipate a critical threshold prior to reaching it. A system undergoes the phenomenon known as critical slowing down as it crosses through a threshold. Theory predicts that fluctuations away from the mean will recover more slowly as the system approaches a critical transition (Scheffer et al., 2009). This is key in infectious disease modelling to assess when the basic reproduction number is reduced below the threshold of one.
Recent theoretical advances have shown indicators of critical transitions in epidemiology such as measuring the variance in synthetic disease data. Our work highlights several challenges when applying this theory in practice. One potential problem is known as 'detrending' the data, which can be difficult to achieve in a single time series (Dessavre & Southall et al., 2019). Accurately detrending the signal removes the mean to obtain the fluctuations, whilst preserving any statistical properties.  We present a novel approach using a metapopulation framework to successfully detrend data using the mean of different geographical subpopulations.
A second limitation is that often only incidence-level data is available publicly. However, current theoretical analyses of statistical indicators concentrate on prevalence data, instead of new cases. We demonstrate that indicators calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data (Southall et al., 2020). Inconsistencies in time series traits between different diseases systems and a variety of disease data types could lead to misleading results when applied to collected data.
In this talk we present methods for dealing with the typical data collected and our results show promising methods for calculating early-warning signals of elimination on real-world noisy data.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Enrico-Sandro-Colizzi/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Enrico Sandro  Colizzi
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Origins Center; Leiden University</em>
						    
						    
						    
					    		
					    			<h6>"Evolution of multicellularity driven by emergent collective migration"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The evolution of multicellularity is a major evolutionary transition: individual cells give up their reproductive autonomy to form aggregates. Aggregation  evolves because it can confer a fitness advantage over unicellularity, e.g. because of protection from predators, functional specialisation or because aggregates can respond to environmental cues unavailable to single cells. These aggregate-level properties arise from cell-cell interactions, and determine the evolutionary course of the cells by imposing novel selection pressures. Thus, the evolutionary feedback between cell interactions and group-level properties is at the root of the evolution of multicellularity.
We explore the emergence of multicellular aggregates in a computational model where a population of cells searches for resources by chemotaxis in a spatially and temporally noisy gradient. Cells can evolve their adhesion to one another, and are selected on a cell's distance from the source of the gradient as a proxy for the availability of resources.
We show that undifferentiated multicellularity evolves because cell aggregates perform collective chemotaxis more efficiently than single cells. A unicellular strategy based on efficient dispersal (rather than collective movement) can also evolve when environmental changes occur frequently. We find that both strategies prevent the invasion of the other through interference competition. We conclude that collective behaviour can be an emergent driver of the evolution of adhesion - and therewith undifferentiated multicellularity.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Erica-M.-Rutter/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Erica M. Rutter
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of California, Merced</em>
						    
						    
						    
					    		
					    			<h6>"Predicting Bladder Pressure and Contractions with Dense Time-Series Data"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Bladder dysfunction due to spinal cord injury can result in incontinence and the inability to effectively void the bladder. Electrical stimulation of nerves in the bladder during a contraction can inhibit bladder contractions (eliminating incontinence) or excite bladder contractions to ensure the bladder is completely voided. However, determining when a bladder contraction will occur remains an active area of research. Our goal is to infer bladder pressure from external urethral sphincter electromyography (EUS EMG) readings from experimental data using rats. Due to the extremely dense time-series data, traditional mathematical modeling techniques are not applicable. Instead, we employ statistical methods (such as LASSO) and machine learning methods (recurrent neural networks) to make predictions of bladder pressure from external nerve data. Furthermore, to address inter-individual heterogeneity between rats, we applied a multi-task learning algorithm in which each individual rat’s prediction was a separate task – producing more generalizable results. These bladder pressures were then used to predict the onset of bladder contractions with high sensitivity and specificity.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Erik-Amezquita/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Erik Amezquita
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Michigan State University</em>
						    
						    
						    
					    		
					    			<h6>"Quantifying barley morphology using Euler characteristic curves"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Shape is data and data is shape. Biologists are accustomed to thinking about how the shape of biomolecules, cells, tissues, and organisms arise from the effects of genetics, development, and the environment. Less often do we consider that data itself has shape and structure, or that it is possible to measure the shape of data and analyze it.


Topological Data Analysis is a novel mathematical discipline that uses principles from algebraic topology to comprehensively measure shape in datasets. Using a function that relates the similarity of data points to each other, we can monitor the evolution of topological features—connected components, loops, and voids. This evolution, a topological signature, concisely summarizes large, complex datasets. Here, we focus on quantifying the morphology of barley spikes and seeds using topological descriptors based on the Euler characteristic and relate the output back to genetic information. The vision of TDA, that data is shape and shape is data, will be relevant as biology transitions into a data-driven era where meaningful interpretation of large datasets is a limiting factor.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Erika-Tsingos/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Erika Tsingos
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Centre for Organismal Studies</em>
						    
						    
						    
					    		
					    			<h6>"A computational tool to optimise experiments for estimating cell cycle parameters"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Determining how quickly cells traverse the cell cycle is of key interest in growing organ(oid)s, tissues in turnover, and tumours. Estimating cell cycle times requires direct monitoring of a large dynamic cell population [1]. Acquiring and analysing such time-resolved data is challenging, and becomes practicably impossible in complex multicellular tissues. Over the years, several experimental assays have attempted to circumvent these limitations and estimate cell cycle parameters in fixed tissue samples [2-4]. However, there is only fragmentary information on how biological variation, underlying cell heterogeneity, or technical limitations affect the accuracy of these estimates. Here, we develop a computational tool to address these issues with the aim of determining optimal experimental strategies to uncover cell cycle parameters in samples that cannot be monitored directly. We simulate a population of cells traversing a stochastic 4-phase model of the cell cycle. Based on experimental observations and previous theoretical work [1,5], we model the duration of each cell cycle phase with Erlang distributions (a special case of the Gamma distribution), which are parametrised by a shape parameter k and a rate parameter L. These two parameters are used to define the mean phase duration m=k/L and variance B=k/L^2. We implement three different assays used in the literature to estimate cell cycle parameters, then systematically test how biological and technical variability affect the accuracy of the estimate in virtual experiments. Surprisingly, the error of the estimate increases when the duration of the cell cycle is long compared to the duration of the experiment. This implies that the parameter that the assay aims to determine needs to be known beforehand. To overcome this dilemma, we suggest combining different assays to extract maximal information in as few experiments as possible.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Estadilla-S-Carlo-Delfin/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Estadilla S Carlo Delfin
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Ateneo de Manila University</em>
						    
						    
						    
					    		
					    			<h6>"Optimal Strategies for Mitigating the COVID-19 Epidemic in the Philippines"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			With one of the longest lockdowns in the world and over 50,000 cases and 1,600 deaths, the Philippines is one of the hardest hit countries of the COVID-19 pandemic in the Southeast Asian region as of July 2020. The country continues to realign its lockdown policies and increase its test-trace-and-treat capacity to control the epidemic while trying to ease the burden on the national economy. In this study, we identify the best strategy to mitigate the spread of COVID-19 for the regions with the highest cases in the Philippines by applying optimal control theory to a nonlinear dynamical model fitted to local epidemiological data. Furthermore, long-term viability of different levels of community quarantine, testing, and combinations of these two strategies to minimize the number of infectious individuals are examined through cost effectiveness analysis given the limited resources available to the country.
References: Philippine Department of Health - Epidemiology Bureau (2020), COVID-19 Tracker Philippines (accessed July 15, 2020). https://www.doh.gov.ph/covid19tracker L. S. Pontryagin, V. G. Boltyanskii, R. V. Gamkrelize, and E. F. Mishchenko,The Mathematical Theory of Optimal Processes, Wiley, 1962.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Euan-Smithers/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Euan  Smithers
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Birmingham</em>
						    
						    
						    
					    		
					    			<h6>"How do plant leaf pavement cells form puzzle piece like shapes? Using a multi-model approach to simulate chemical and visco-elastic mechanical processes and experimental methods to discover their secrets."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Pavement cells in the plant leaf epidermis form interesting and intriguing interlocking puzzle like shapes with undulations of lobes and indents. However, no one has been able to fully explain how these shapes form. There are two possible pathways of pavement cell development, one involving a combination of plant Rho like GTPases signalling proteins and cytoskeleton components, specifically microtubules which could provide a feedback loop and the second being possible mechanical effects from the tissue.
As a result, we have developed three models, one to model microtubule behaviour, the second to model the protein signalling dynamics and the third to model the mechanics of the cells, using a stochastic network, reaction diffusion equations solved via the finite element method and a visco-elastic vertex element model. I shall also outline some of the experimental procedures we have carried out to test how pavement cells develop. We can demonstrate that the signalling pathways provide a feedback loop to sustain pavement cell shape, but don’t initiate the shape, while the mechanical effects from the tissue can initiate pavement cell lobes.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Fatima-Sulayman/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Fatima Sulayman
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universiti Sains Malaysia</em>
						    
						    
						    
					    		
					    			<h6>"Dynamical systems analysis of tuberculosis with the impact of transmission rate and vaccination."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This paper examine the impact of transmission rate and vaccination on the dynamics of tuberculosis infection. Model analysis is established, the existence of hopf-bifurcation is analytically shown which deduces the oscillatory persistence of the infection in the population. The stability properties of the steady states and its bifurcation structures are investigated. We illustrate that there exist threshold values for the transmission rate and vaccination which corresponds to saddle node, transcritical and subcritical hopf bifurcation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Felipe-E-M-Campos/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Felipe E M Campos
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of São Paulo</em>
						    
						    
						    
					    		
					    			<h6>"A modelling approach to study landscape effects on abundance from patch demographic processes"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Biodiversity loss is one of the great challenges to be faced by the current and the future generations. Habitat loss and fragmentation are important sources of concern. A recent debate over the effects of habitat fragmentation on biodiversity led to contradictory results. Some argue that habitat fragmentation has profound negative effects on biodiversity, while others challenge this view and advocate that those effects are usually negligible and mostly positive when significant. One source of disagreement comes from the scale of investigations made so far. Initial studies investigated the patch-scale mechanisms by which habitat fragmentation affects biodiversity and generally supported negative effects, for example by means of edge effects, vulnerability of small populations to stochasticity, etc. Recently, however, studies seeking a landscape-scale pattern of how habitat fragmentation impacts biodiversity failed to reach consistent results. This apparent contradiction between mechanisms and observed pattern highlighted the need for mechanistic understanding on how patch-scale mechanisms lead to landscape-scale patterns.
In this study we link patch-scale mechanisms to landscape-scale abundance patterns by means of a patch model consisting of a system of ordinary differential equations. The equations describe the populations of the patches, which are interdependent and affected by four basic demographic processes: births, deaths, immigration and emigration. We promote the link between scales by deriving an insightful expression for the landscape equilibrium abundance and a necessary and a sufficient condition for landscape extinction, all of them integrating the demographic processes of all patches. As a second step, we sought to describe birth, death, immigration and emigration rates in terms of characteristics of the patches, like patch areas and inter-patch distances. This translates our landscape equilibrium abundance expression and extinction conditions in terms of landscape traits commonly used in Landscape Ecology, like habitat amount (sum of the areas of all patches) and landscape configuration metrics (functions of the traits of all patches). We use data collected from an Individual Based Model (IBM) to illustrate each of the steps of this study.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Felipe-Rubio/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Felipe Rubio
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UNICAMP</em>
						    
						    
						    
					    		
					    			<h6>"Evaluating the role of memory B and T cells during secondary dengue infection"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Dengue viruses (DENV) are transmitted by Aedes mosquito bite that causes mild dengue fever (DF) or dengue severe (DS). There are more than 50 million DF cases each year. There are four serotypes of dengue virus, DENV1 - DENV4, which has only 20%-35% divergence to each other. The cross-reactive immune response contributes to increased disease severity following heterologous infections. In general, primary infections result in either asymptomatic or mild DF disease. Secondary infections with different serotypes are either cleared or can induce dengue severe. Mechanisms responsible for the severity of secondary dengue infections are not entirely understood. One of them is the cross-reactive antibodies can enhance the disease, which is called antibody-dependent enhancement (ADE). ADE is explained as follows. When a person is first infected with one dengue strain, the host produces neutralizing antibodies specific against this strain. After the primary infection is eliminated, due to the immunological memory plasma cells produce specific antibodies for the first dengue strain, which persist in the body. If this person is secondly infected with a different dengue strain, antibodies from the primary infection bind the second virus but do not neutralize it. Besides that, macrophages recruited to clear the immune complex; they internalize this non-neutralized virus and become infected in the process of clearance. There is evidence that Fc receptors, which are proteins on the surface of some cells like macrophages and monocytes that bind to the antigen-antibody complex, might facilitate viral entry in cells and increase dengue viral replication. We propose a mathematical model composed of seven non-linear differential equations to describe the ADE phenomenon in secondary dengue infection, considering that partial cross-immunity takes place due to primary dengue infection. We consider that the immune system has not yet performed a complete response against the secondary infection. During this time, the circulating antibodies against the primary virus can facilitate heterologous secondary infection. Here we consider target cells - macrophages, a specific antibody against the primary infection, memory B cells, memory T cells, the formation of the immune complex, and dengue virus. We focus on the role of memory B and T cells during the secondary dengue infection. It is possible to determine the basic reproduction number parameter, R0, and B and T cell cloning during the secondary dengue infection. In the impossibility of cloning these cells, we find that if R0<1, there will be no possibility of ADE's appearance. However, when we introduced the possibility of memory B cell cloning, we saw that an infectious state could arise even when the basal reproduction number is less than one.  Analogously, when we analyzed only the effect of memory T cell cloning, we saw that ADE's emergence is not possible. These cells only act to decrease viral concentration.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Femke-Thon/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Femke  Thon
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Bielefeld University</em>
						    
						    
						    
					    		
					    			<h6>"Modeling intraspecific chemodiversity - theory and first results"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Explaining the causes and effects of different types of diversity is one of the key research missions in ecology and evolutionary biology. Plants produce numerous metabolites. There is a great diversity of metabolites between species, populations, and members of the same population. This chemodiversity has numerous ecological and economic implications. However, the mechanisms which maintain chemodiversity are still largely unknown. A theoretical framework is needed as a first step to bridge this gap in evolutionary knowledge. The goal of this project is therefore to develop mathematical and computational models linking genes, enzymes, metabolites, and ecological interactions to start building a theoretical framework for the evolutionary emergence and maintenance of plant chemodiversity. The screening hypothesis postulates that plants developed a set of biosynthetic pathways in which a great number of metabolites can quickly evolve. The more metabolites there are, the more likely it is that some have a defensive role against herbivores. Additionally, in existing models for the maintenance of other types of diversity, different types of negative frequency-dependent-selection (NFDS) frequently play an important role.
In this project, we develop models based on the screening hypothesis and NFDS to investigate whether these hypotheses can explain how observed chemodiversity may have evolved and may be maintained. We will work together closely with empiricists to produce models which can be used to predict possible and empirically testable evolutionary pathways for model species based on realistic assumptions about these species. In the first phase of the project, we develop an individual-based model of a plant population and implement it in C++. This model includes a submodel of the biosynthetic pathways which determine the metabolites each individual produces. In the pathway, a primary metabolite is modified by various enzymes. The coding and regulatory genes for these enzymes evolve through mutation, gene duplication, and gene loss. The resulting metabolite(s) determine the fitness effects of each individual genotype. On my poster, I will present the evolutionary thought behind the model and the results from the first phases of the implementation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Fernando-Luiz-Pio-dos-Santos/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Fernando Luiz  Pio dos Santos
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Investigation of the Aedes spread using a reaction-diffusion mathematical model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this work, we developed a reaction-diffusion mathematical model to describe the spread of dengue infection in a two-dimensional computational domain. We aimed to understand how the disease spreads from a specific location to another, considering the diffusion coefficients of both infected populations, mosquitoes, and humans. Our contribution provides an in-depth analysis of the optimal control problem and it outlines a more explicit modeling framework based on real spatial-temporal data. São Paulo Research Foundation (FAPESP), grant 2018/03116-3.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Fillipe-H-Georgiou/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Fillipe H Georgiou
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Newcastle</em>
						    
						    
						    
					    		
					    			<h6>"Modelling Locust Foraging And Its Effect On Swarm Formation"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Having plagued mankind for millennia, locust swarms continue to be a major threat to agriculture, effecting every continent except Antarctica and impacting the lives of 1 in 10 people. Locusts are short horned grasshoppers that exhibit two behaviour types depending on their local population density. These are; solitarious, where they will actively avoid other locusts, and gregarious, where they will actively seek them out. It is in this gregarious state that locusts can form massive and destructive swarms or plagues. These large scale group dynamics arise through simple individual and environment interactions.
At longer time-scales, environmental conditions such as rain events synchronize locust lifecycles and can lead to repeated outbreaks. At shorter time-scales, changes in the distribution of food can have an effect on locust gregariazation. By modifying a multi-population aggregation equation to include locust-environment dynamics we are able to investigate the effect of different food distributions on locust swarming.
Our results suggest that there is an optimal food width for locust swarm formation, and that as food becomes more densely packed gregarious locusts are able to outcompete their solitarious peers.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Florian-Franke/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Florian Franke
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>HTW Dresden</em>
						    
						    
						    
					    		
					    			<h6>"Is Cell segregation just like oil and water: A phase field approach"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Understanding the segregation of cells is crucial to answer questions about tissue formation in embryos or tumour progression. According to Steinberg's differential adhesion hypothesis [1] the separation of cells can be compared to the separation of two liquids, e.g. water and oil. Specifically, it was proposed, that similarly to the demixing of fluids, differences in the strengths of the adhesive forces in homo- and heterotypic cell contact lead to all sorting. This hypothesis has been tested on the basis of cell-based models which simulate motile cells with differential adhesive interaction on the basis of probability cellular automaton models [2]. On the other hand, the segregation of fluids like water and Oil can be well described by phase-field models as the Cahn-Hilliard-equation. Here we ask, weather the two approaches can be related to each other and how under such a relation depends on the details of the cell-based model.


We develop and test various order parameters which allow assuming the degree of segregation during time in both, cell-based probability cellular automaton models for cell sorting and phase-field models for the demixing of fluids. We identify typical benchmark scenarios where the agreement between both model classes shall be maximized and calibrate the phase-field model such that it best fit to in-silico data produced by specific cell-based models of sorting. We evaluate the good-ness of fit in these scenarios and relate out findings to the original differential adhesion hypothesis.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Furkan-Kurtoglu/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Furkan Kurtoglu
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indiana University</em>
						    
						    
						    
					    		
					    			<h6>"Temporospatial Modeling of CRC-CAF relation using molecular and tissue level"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Molecular communication between cells is a complex system when we think that the system consists of intelligent agents and microenvironment. In our project, we aimed to computationally model molecular communication between Colorectal Cancer Cells (CRCs) and Cancer-Associated Fibroblasts (CAFs) using a multi-scale agent-based modeling approach. To create this model, each cell is assumed as an agent that can change their behavior according to the microenvironment. We are using 3-D physics-based intelligent system simulator software that is called PhysiCell. While each cell is an off-lattice centered agent, the microenvironment is designed as a structured Cartesian grid. Microenvironment stores a number of external metabolites that cells can uptake from it or secrete to it with specific transfer reactions. Each cell has custom cell data that corresponds to some phenotypic attributes such as cell cycle, death, pressure. This data-structure also can be used for intracellular metabolite concentrations. Intracellular chemical reactions are modeled as ODEs that are represented as Systems Biology Markup Language (SBML). The ODE model includes a pseudo-metabolite that is called 'Energy'. This chemical is a bridge between intracellular-level and tissue-level with determining cellular cycle rate and death rate. The cross-platform 'Lib-roadrunner' is used to solve SBML models in one molecular time step. Integrating SBML to multi-scale models required to solve some design problems, such as, integration of uptake rates or defining transportable species. In addition, SBML parameters is stored as PhysiCell variable therefore each cell can have unique kinetic parameters which yields heterogeneity in the tumor. As the main goal, integrating molecular simulations with each agent might help us to understand cellular behaviors in complex systems.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Gemma-Massonis/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Gemma  Massonis
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>CSIC</em>
						    
						    
						    
					    		
					    			<h6>"Structural Identifiability and Observability of Compartmental Models of the COVID-19 Pandemic"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The recent coronavirus disease (COVID-19) outbreak has dramatically increased the public awareness and appreciation of the utility of dynamic models. At the same time, the dissemination of contradictory model predictions has highlighted their limitations.  If some parameters and/or state variables of a model cannot be determined from out-put measurements, its ability to yield correct insights – as well as the possibility of controlling the system – maybe compromised.  Epidemic dynamics are commonly analysed using compartmental models, and many variations of such models have been used for analysing and predicting the evolution of the COVID-19 pandemic.  In this paper we survey the different models proposed in the literature, assembling a list of 36 model structures and assessing their ability to provide reliable information. We address the problem using the control theoretic concepts of structural identifiability and observability. Since some parameters can vary during the course of an epidemic, we consider boththe constant and time-varying parameter assumptions.  We analyse the structural identifiability and observability ofall of the models, considering all plausible choices of outputs and time-varying parameters, which leads us to analyse 255 different model versions. We classify the models according to their structural identifiability and observability under the different assumptions and discuss the implications of the results. We also illustrate with an example several alternative ways of remedying the lack of observability of a model. Our analyses provide guidelines for choosing themost informative model for each purpose, taking into account the available knowledge and measurements.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Gess-Iraji/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Gess Iraji
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Brandeis University</em>
						    
						    
						    
					    		
					    			<h6>"Modeling Clogging of Red Blood Cells in Microfluidic Devices with Simple Geometry"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We develop a basic model to investigate clogging of red blood cells in diagnostic microfluidic devices that sort cells based on their deformability. We analyze the effects of specific clogging rate functions on the progression of clogging in a device with simple geometry. We confirm our results using stochastic simulations and numerical methods for solving ordinary differential equations.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Gilberto-C-Gonzalez-Parra/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Gilberto C Gonzalez-Parra
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>New Mexico Tech</em>
						    
						    
						    
					    		
					    			<h6>"Forecasting cases of RSV using artificial neural networks and mechanistic models"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We study and present an approach based on artificial neural networks to forecast the number of cases with the Respiratory Syncytial Virus (RSV). The number of cases of RSV in most of the countries around the world present a seasonal type behavior. We construct and develop several multilayer perceptron models that intend to forecast appropriately the number of cases of RSV. We compared our approach with a classical technique for time series, and our results are more accurate. The adjusted MLP network that we find has a fairly high accuracy of forecast.  Finally, we compare empirical and mechanistic models applied to forecasting and prediction.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Giulia-Laura-Celora/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Giulia Laura  Celora
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Oxford</em>
						    
						    
						    
					    		
					    			<h6>"Analysis of the dynamics of tumour cells along a stemness axis under different oxygen conditions"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The concept of cancer stem cells (CSCs) was first introduced to explain intra-tumour heterogeneity. According to the so called ‘CSC’ hypothesis, tumours are organised according to a rigid hierarchical structure where CSCs have the capacity to self-renew. Through asymmetric cell division, CSCs can initiate and maintain tumours that also contain differentiated cells with limited clonogenic potential. Recent studies have challenged this  framework and led to the development of the so-called ‘CSC plasticity’ hypothesis.  Here, stemness is viewed as a continuous rather than a discrete trait, and it may change in response to micro-environmental signals. In line with this conceptual model, we develop a mathematical model to describe the dynamics of a population of tumour cells structured by their stemness. Cells continuously transition between cancer stem cells (CSC) and terminally differentiated cancer cells.  Evolution along the stemness axis is driven by extrinsic (micro-environment) and intrinsic (random epimutation) ``forces'', which are represented by advective and diffusion fluxes respectively. We account for natural selection and competition by introducing a fitness landscape, i.e. phenotypic dependent net growth of the cells. We consider a well-mixed environment in which cells are exposed to a prescribed oxygen environment, and their time-evolution determined by a non-local reaction-advection-diffusion equation, where the non-locality rises from the competition between different phenotypes. We analyse two scenarios, normoxia and hypoxia, in order to capture the different niches present in vivo. In our model, oxygen levels affect not only cell fitness but also act as an extrinsic force, favouring cell maturation (under normoxia) or de-differentiation into CSC (under hypoxia). We show how the qualitative behaviour of the system dynamics and its equilibrium distribution changes as model parameters vary, with tumour extinction predicted for certain regimes. The numerical results are validated by using spectral theory which allow us to characterise the stability property of the trivial steady-state, i.e. extinction. In addition to reproducing a variety of tumour cell distributions characterised by different mean clonogenic capacity, proportion of CSCs and population size, our analysis also gives insight into the role that extrinsic and intrinsic forces play in shaping the organisation of cells in phenotypic space. Finally, we discuss how the model can be extended to incorporate treatment, specifically radiotherapy, accounting for stem-ness dependent radio-resistance.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Giulio-Bonifazi/">
						    	<h4><li class="subgroup-counter subgroup-counter-NEUR"></li>
							    	
							    	Giulio Bonifazi
							    		(<span class="text-NEUR">NEUR</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Basque Center for Applied Mathematics</em>
						    
						    
						    
					    		
					    			<h6>"Predicting excitotoxicity at the onset of Alzheimer’s disease by a model of Aβ-dependent trafficking of astrocytic glutamate transporters"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			At Alzheimer’s disease (AD) onset, extracellular accumulation of oligomeric amyloid-β (Aβ) correlates with excitotoxicity and the alteration of glutamate uptake by astrocytic transporters (GLT1). Experiments suggest that glutamate, Aβ, or a combination thereof, may dynamically regulate trafficking and expression of those transporters between perisynaptic and intracellular astrocytic compartments [1]. There is no understanding however, whether and how such mechanisms could ultimately link with the emergence of excitotoxicity which hallmarks early stages of AD. With this regard, we consider a simplified description of astrocytic transporter trafficking based on a Markov process for transporter movements between the cytoplasm and the plasma membrane, and vice versa, and we use this model to identify potential ensembles of Aβ-dependent pathways of trafficking that could account for experimental observations. Next, we consider the mean-field rate description ensuing from the Markov model in the Finite Element Method (FEM) framework of a 3D model of glutamate diffusion at synaptic terminals and their surroundings. Changing extracellular Aβ concentration, we accordingly look at the time course of extracellular glutamate, and the conditions for its accumulation in the extrasynaptic space. Since extracellular glutamate alters synaptic activity, we estimate conditions for excitotoxicity linking neural network firing activity with extracellular glutamate. Consistent with experiments, our model predicts that GLT1 surface expression decreases when extracellular Aβ increases beyond a threshold concentration. This, in turn, favors extracellular glutamate accumulation, promoting a positive feedback loop that induce further synaptic glutamate release, and thereby excitotoxicity. Because the rate of glutamate accumulation depends on the uptake capacity by astrocytic transporters, which is a function of extracellular Aβ, local gradients of Aβ may dramatically affect synaptic environment, both locally and extra-synaptically. These results provide theoretical support to the possibility of looking at Aβ-dependent astrocytic GLT1 expression as a clinical marker for early diagnosis Alzheimer’s disease.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Gustavo-J-Sibona/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Gustavo J Sibona
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>FAMAF - Universidad Nacional de Córdoba</em>
						    
						    
						    
					    		
					    			<h6>"SIRS dynamics on a diffusive agent system"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Since its introduction in 1927 by Kermack and McKendrick, SIR compartmental models have been the basis of mathematical epidemiology. In this work we consider a SIRS epidemics on a system of mobile agents, which can interact during a finite period of time that depends on their dynamics. Thus, as the probability of disease transmission depends on this contact time, the spatial dynamics will strongly influence the disease evolution. By combining individual-based simulations and mean-field arguments, we study the dependency of the equilibrium populations on motility parameters, specifically the active speed and tumbling frequency. We find that the equilibrium epidemic size exhibits two very distinct, non-trivial scaling regimes with the motility parameters, depending on whether the system is in the ballistic or diffusive regime. Our mean-field estimates lead to an effective renormalization of the transition rates that allow building a phase-diagram that separates endemic and disease free phases. We find an excellent agreement between numerical simulations and mean-field estimates.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Hana-Dobrovolny/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Hana  Dobrovolny
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Texas Christian University</em>
						    
						    
						    
					    		
					    			<h6>"SARS-CoV-2 coinfections: Implications for the second wave"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Researchers have noted that there are unexpectedly few SARS-CoV-2 coinfections with other circulating respiratory viruses. We use an in host mathematical model to examine interaction of SARS-CoV-2 with other respiratory viruses, finding that SARS-CoV-2 growth tends to be blocked by the presence of other viruses. We then formulate an epidemiological model to determine how this blocking effect might influence the impending second wave of SARS-CoV-2 infections.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Harsimran-Kaur/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Harsimran  Kaur
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Institute of Technology Bombay</em>
						    
						    
						    
					    		
					    			<h6>"Computational modeling of mechano-metabolic adaptation to a stiff microenvironment in Cancer cells"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Extracellular matrix (ECM) is a highly dynamic cohort of macromolecules present in the cellular microenvironment that regulates cellular behavior chemically and mechanically. Numerous studies have demonstrated the effect of mechanical cues on cellular differentiation and morphology. These studies also highlight the imperative role of ECM in the activation of specific signaling pathways through which ECM influences cellular behavior. Continuous remodeling of the Tumor microenvironment, which also includes ECM stiffening, has emerged as a prominent hallmark of poor disease prognosis and cancer metastasis. In addition to that, cancer cells are also known for their highly reprogrammed metabolism. The objective of this study was to establish a relationship between increased stiffness of tumor microenvironment and metabolism in cancer cells. A deterministic analytical model has been constructed to demonstrate the mechanoadaptation of metabolism in cancer cells. This mathematical model shows that increased stiffness has a positive effect on HIF1α accumulation under hypoxia. As HIF1α promotes the Warburg effect, this result highlights the potential link between Tumor mechanics and metabolism.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Haryana-Thomas/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Haryana  Thomas
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Oklahoma State University</em>
						    
						    
						    
					    		
					    			<h6>"Modeling Cellular Signaling and Mesangial Fibrosis during Diabetic Kidney"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In the U.S. alone over 250,000 people use dialysis or have received a kidney transplant due to diabetic kidney failure. Although we have come a long way in the treatment of diabetes, kidney failure due to diabetic kidney damage is still prevalent, and the need for increasing our understanding of kidney damage to enable the development of better treatment methods is ever present. Thus the goal of this research is to develop computational models to better understand the kidney damage that occurs due to diabetic kidney disease. In the kidney glomerulus lies a network of capillaries that are surrounded by interstitial tissue called the mesangium. In health, the mesangium acts as a support for the capillaries; however, during diabetic kidney disease, the mesangium expands due to excess accumulation of collagen and causes damage to the cellular environment around it. This mesangial expansion is not only a hallmark of kidneys damaged by diabetes but also many other chronic kidney diseases that lead to kidney failure. As such there has been a lot of research effort in trying to figure out the cause of the mesangial expansion. Researchers have found high glucose-induced dysfunction in the mesangial cell, a cell native to the mesangium, to be one of the main reasons for mesangial expansion. The mesangial cell dysfunction is mediated by the overstimulation of key signaling and cellular communication molecules such as TGF-B, and Ang II which play a key role in perturbing the function of downstream collagen metabolism molecules such as MMP, and TIMP leading to the accumulation of excess collagen. The complexity of the interactions necessitates the development of computational models to understand the whole, yet there are few computational models of mesangial expansion and even fewer that study the effect of the mesangial expansion on cellular communication and signaling in the glomerulus. In this work, we present a computational model of mesangial expansion to study its effect on cellular signaling. Previously, researchers have developed computational models of mesangial expansion to understand its impact on the accumulation of certain macromolecules whose accumulation has been shown to lead to glomerular damage. Our computational model builds on such a model to elucidate the impact of mesangial cell mediated mesangial expansion on cellular signaling through multiscale modeling of ECM remodeling and macromolecular transport. We are extending the previous model by incorporating a cellular environment using the Cellular Potts model, modeling mesangial expansion using fundamental biological principles of collagen fiber growth and accumulation, solving macromolecular transport equations using a solver and linking them all using python and CompuCell3D, a multiscale tissue simulation software. The model captures mesangial expansion and provides insight into cellular communication.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Heber-Rocha/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Heber  Rocha
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indiana University</em>
						    
						    
						    
					    		
					    			<h6>"Qualitative study of cell migration associated with hypoxia in the dynamics of tumor growth"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Recent results point to the importance of hypoxia in the development of cancer. In particular, hypoxia is generated by an intratumoral oxygen gradient, driving tumor cells towards a migratory phenotype, which in turn promotes invasion and metastatic risk. Recently, experimentalists developed a novel bioengineered reporter system where normoxic cells fluoresce red (DsRed) until exposure to hypoxic conditions, at which point increases in hypoxia-inducible factors (HIFs) induce a permanent genetic change to express green fluorescent protein (GFP). Observations of this system in a mouse model allow us to formulate and evaluate new hypotheses on the frequency and duration of phenotypic changes in cancer cells under the influence of hypoxic conditions. In this work, we present a qualitative study of the response of the GFP+ cells to the migratory stimulus from hypoxia, with a focus on understanding the role of phenotypic transience or permanence on cancer invasion. We use a hybrid continuum-discrete model on two scales that describes the behavior of normoxic and hypoxic cells in the dynamics of tumor growth and invasion. On the cellular scale, the cells are individually represented as discrete agents according to their physical and phenotypic attributions, while on the tissue scale, the dispersion of the oxygen pressure in the microenvironment is represented using continuum diffusion-reaction equations. As in the in vivo experiments, we model changes in red/green fluorescence based on hypoxic exposure. We calibrate changes in cell motility following hypoxic exposure to ex vivo measurements of DsRed+ and GFP+ in cells, using an Approximate Bayesian Computation (ABC) method. The initial results of the in silico model present a plausible representation of the biological experiments and suggest new research themes.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Himanshu-Aggarwal/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Himanshu  Aggarwal
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Institute of Science Education and Research (IISER) Mohali</em>
						    
						    
						    
					    		
					    			<h6>"Impact of temperature change on Malaria incidence in high altitude regions of India"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Malaria is a vector (mosquito)-borne parasitic disease, which is a major cause of death and disability in the world around the equatorial belt. The mosquitoes require a range of temperature and humidity to reproduce and persist, and the Malaria parasite (species of Plasmodium) uses both mosquito and human as their host for growth and reproduction. The cooler regions, such as the cold high altitude mountainous areas, therefore, witness fewer Malaria cases. An increase in temperature, as a possible consequence of global warming, is hypothesized to drive the spread of the malarial vectors to higher altitudes, by rendering these regions suitable for their growth, and consequent increase in Malaria incidence. But this view is hotly debated with evidence existing both against and in favour. Therefore, there is a need for more empirical evidence covering a wide variety of regions all around the world. A detailed study is performed, on a historical dataset (1975-1995) of Malaria cases in 19 contiguous districts from the three north-western states of India having an altitude higher than 1000 meters, for the analysis of the impact of temperature changes on Malaria incidence in these high altitude regions. In reality, the mean temperatures do not show a consistent increase over the years, and the vegetation, rainfall, and other environmental and demographic factors also differ among the regions and years. Using different data analytic and statistical measures, the results, though collectively do not provide a definite evidence for climate change-induced increase in Malaria in the high altitude regions under study, yet they do point towards increased malaria cases with an increase in temperature. It also points out that many other drivers may be responsible for the spread of an infectious disease like Malaria (e.g., population density, tourism, water bodies, agricultural land-use, etc.), which needs to be considered. This argues in favour of considering epidemiological data from a more interdisciplinary perspective by including demographic, environmental, social and economic driving factors in analysis and modelling.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Ielyaas-Cloete/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Ielyaas  Cloete
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Auckland</em>
						    
						    
						    
					    		
					    			<h6>"Not all hormone receptors are created equal"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Calcium in hepatocytes modulates diverse functions, including bile secretion, glucose and energy metabolism and vesicular trafficking. A major question in the study of calcium signalling in hepatocytes is how these distinct cellular processes are controlled and organised via coordinated spatial and temporal calcium signals.
Downstream cellular responses are controlled via intracellular calcium oscillations, but the underlying mechanisms which shape these oscillations have yet to be elucidated. In particular, we are interested in the effect of protein kinase C (PKC) on the purinergic family of receptors. Recent data has shown that the activation of different receptors within the family of purinergic receptors generate qualitatively different calcium responses. It is believed that PKC differentially regulates each receptor resulting in distinct calcium response patterns. Furthermore, multiple pools of PKC, with unique activation pathways, are understood to exist in the cell with discrete downstream targets.
We discuss recent progress in construction and analysis of a model of calcium oscillations that incorporates the new experimental results about likely feedback mechanisms in hepatocytes. Our model suggests that multiple, uniquely activated, PKC feedback loops acting on unique cellular substrates, present in the cell coordinate to determine the qualitative behaviour of calcium oscillations in hepatocytes.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Iulia-Martina-Bulai/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Iulia Martina Bulai
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Basilicata</em>
						    
						    
						    
					    		
					    			<h6>"Modeling COVID-19 considering asymptomatic cases and avoid contacts"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			World Health Organization (WHO) defined coronaviruses (CoV) as a large family of viruses that cause illness ranging from the common cold to more severe diseases such as Middle East Respiratory Syndrome (MERS-CoV) and Severe Acute Respiratory Syndrome (SARS CoV).The novel coronavirus (Covid-19) is a new strain that has not been previously identified in humans. Coronaviruses are zoonotic, meaning they are transmitted between animals and people. In this work is presented a mathematical model that describes the transmission of Covid-19. The model considers both symptomatic and asymptomatic cases. Several studies showed the importance of asymptomatic people in the disease transmission. This is a predictive model, we look at different scenarios, first of all assuming any prevention to avoid the diffusion of the virus is taken and secondly different scenarios where precautionary measures to avoid contact between people are taken, such as quarantine and social distancing. We consider a measures to contain the disease, already studied for a predator-prey systems with the disease in the prey population assuming that the infection rate can be decreased avoiding contacts between preys (people in our case). From the numerical results we get that avoiding contacts helps to delay the peak of the maximum number of infected people, that is important in those cases where the hospital system does not have enough seats in the intensive care unit. Furthermore we studied how the reproduction number depends on the parameters values of the model. Some of the parameters are fixed, as found in literature for Italy, while other are used as control parameters.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jack-M-Hughes/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Jack M Hughes
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Thermodynamic Inhibition in a Biofilm Reactor with Suspended Bacteria"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We formulate a biofilm reactor model with suspended bacteria that accounts for thermodynamic growth inhibition. The reactor model is a chemostat style model consisting of a single replenished growth promoting substrate, a single reaction product, suspended bacteria, and wall attached bacteria in the form of a bacterial biofilm. We present stability results for the washout equilibrium and conduct a computational study. While stability conditions are similar to a chemostat model, we find that the steady-state concentration of the replenished substrate depends on its inflow concentration. In the computational study, we find that thermodynamic inhibition limits substrate utilization/ production both inside the biofilm and inside the aqueous phase, resulting in less suspended bacteria and a thinner biofilm.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jackelyn-M-Kembro/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Jackelyn M Kembro
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>National Scientific and Technical Research Council (CONICET)</em>
						    
						    
						    
					    		
					    			<h6>"Accelerometers as a tool to characterize reproductive behavior within social groups in long term experiments: the case of the Japanese Quill"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Accelerometers are devices that convert movement into three signals belonging to each component of the acceleration vector at a high acquisition rate, up to 25 data per second. When they are fixed to an animal, each action performed by the individual leads to a particular shape in these signals that, when depicted in a computer, can be isolated and classified. Hence, accelerometer recordings can be combined with machine learning techniques in order to automatically classify signals into behavioral categories. This is particularly useful in the context of long-term social behavior studies in large or natural environments were recording from visual observation is difficult and time consuming. Herein, we placed accelerometers on the back of adult male quails (Coturnix japonica) to register their activity when they are released into a home box containing two female quails during a 1-hour period. At the same time, the experiment was video-recorded to obtain a time series of the different behaviors performed by the male and their corresponding duration by direct inspection. The accelerometric signals and behavioral time series obtained were used to train a neuronal network. Our neuronal network was able to classify reproductive behavior of males at high temporal resolution. In particular, we showed, first, that the duration of some reproductive events can be much shorter than those reported previously and transitions between different behaviors are very fast (of the order of ~100ms). Second, reproductive behavior appears to begins earlier and finish later than it is possible to observe visually using video recordings. Our results show that combining accelerometer recordings with neural network processing is a powerful method to automatically register reproductive behaviors within social groups with high accuracy. This is of particular importance given that it has the potential to replace registering from visual observation of social behavior. Moreover, the long, high resolution reproductive time series obtained by this approach can be useful for studding long-term reproductive behavioral rhythms in poultry.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jahedi-Sana/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Jahedi  Sana
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of New Brunswick</em>
						    
						    
						    
					    		
					    			<h6>"When the best pandemic models are the simplest"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			As a pandemic of coronavirus spreads across the globe, people debate policies to mitigate its severity. Many complex, highly detailed models have been developed to help policy setters make better decisions. However, the basis of these models is unlikely to be understood by non-experts. We describe the advantages of simple models for covid-19. We say a model is ' simple' if its only parameter is the rate of contact between people in the population. This contact rate can vary over time, depending on choices by policy setters. Such models can be understood by a broad audience, and thus can be helpful in explaining the policy decisions to the public. They can be used to evaluate outcomes of different policy strategies. However, simple models have a disadvantage when dealing with inhomogeneous populations. To augment the power of a simple model to evaluate complicated situations, we add what we call 'satellite' equations that do not change the original model.  For example, with the help of a satellite equation, one could know what his/her chance is of remaining uninfected through the end of epidemic. Satellite equations can model the effect of the epidemic on high-risk individuals, or death rates, or on nursing homes, and other isolated populations. To compare simple models with complex models, we introduce our 'slightly complex' Model J. We find the conclusions of simple and complex models can be quite similar. But, for each added complexity, a modeler may have to choose additional parameter values describing who will infect whom under what conditions, choices for which there is often little rationale but that can have a big impact on predictions. Our simulations suggest that the added complexity offers little predictive advantage.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jairo-G-Silva/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Jairo G Silva
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Instituto Federal de Mato Grosso</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical Models About Radioactive Iodine-Refractory Differentiated Thyroid Cancer"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Clinical and pathological evidence suggests that the progression of Differentiated Thyroid Carcinomas to a poorly differentiated stage, or even an anaplasic cancer, is a natural process in the development of malignancy. The immune regulatory molecule PD-L1, Programmed Death-Ligand 1, blocks the immune response of activated T cells by binding to the PD-1 receptor, an immune checkpoint expressed in T cells and others, to modulate their activation or inhibition. In addition to the traditional 131-I radioiodine treatment, other therapeutic options for DTC are needed when cells lose their ability to capture and concentrate iodine. Two examples of drugs used due to the evolution of DTC to a progressive state in the loss of sensitivity to RAI correspond to Lenvatinib, which is a target therapy with the function of inhibiting multiple tyrosine kinases, and thus reducing tumor cell proliferation; and Pembrolizumab, a monoclonal antibody of human immunoglobulin G4 that aims to prevent the binding of PD-1 to PD-L1, and thus restore the anti-tumor immune response of anti-tumor T cells.
We propose two mathematical models of ordinary differential equations in order to evaluate two modes of treatment for patients with DTC refractory to RAI. In the first model, considering the variables concentration of Lenvatinib, number of malignant cells and NK cells, we evaluated the effectiveness of the target therapy in the treatment. The second model includes the addition of the variable concentration of Pembrolizumab, and T cells in the group of NK immune cells, and for this reason, we simulate in this case the effectiveness of the therapeutic combination with patients. As a result, we obtained that both drugs were able to generate responses such as stable disease or partial response, however, greater control of the tumor was observed from the combination of the proposed therapies.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Janina-Hesse/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Janina  Hesse
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Charité University Hospital Berlin</em>
						    
						    
						    
					    		
					    			<h6>"Diurnal variation in gene expression and sports performance: a matter of timing?"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Virtually all living organisms show oscillations in physiology that track the daily rhythm of light and darkness. In humans, not only the sleep-wake cycle has a period of about 24 hours, also basic physiological and cellular processes such as core-body temperature, cortisol levels or cell cycle and metabolism oscillate accordingly. While circadian oscillations are well studied at many levels from genes to behaviour, the impact of the molecular profile on the behavioural output has so far been hardly studied in humans. With the aim to establish a relation between genes and behaviour, we analysed time series data sets of gene expression and sports performance from the same human subjects. Gene expression of two specific core-clock genes, measured via RT-PCR, shows circadian variation in samples of bodily fluids, which constitutes a practical source for human genetic material. Sports performance was evaluated by a set of three standardized tests probing strength and endurance. While we find overall best sports performance in the afternoon, the individual best performance times, which show larger variations, can be predicted by a machine learning approach. Besides best performance time, the variance in sports performance over the day is of interest. Here we discuss our latest findings in the field and their putative benefit for professional athletes, as well as their general implications for our well-being.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jared-Barber/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Jared Barber
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indiana University-Purdue University Indianapolis</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical and mechanical models of cell motion"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Metastasis plays a significant role in many of breast cancer deaths. The traditional route for metastasis involves several steps including successful penetration of a vessel (intravasation), passage through the circulatory system to the site of metastasis (translocation), and exiting of that vessel (extravasation). Decreasing the frequency of any of these events can help mitigate the effects of breast cancer on the approximately 3.5 million Americans affected by the disease. Experiments also suggest that mechanotransduction, a process by which mechanical forces initiate cellular processes, may play an important role in such events. Because of these observations, we have begun developing a mechanical model of breast cancer cell dynamics that is force-based and, therefore, readily informs us about force levels cells may experience during events like intravasation, translocation, and extravasation. We will share results where the model is used to simulate breast cell passage through a tapered microfluidic channel. These results show that a two-dimensional network of damped springs (viscoelastic elements) submersed in surrounding Stokes flow can be used to reproduce qualitative agreement with experiments. They further show that such a model can be used with sensitivity analysis to consider how different cell properties affect cellular dynamics. While such results are focused on translocation and physical forces (without biochemistry), additional extensions of the model are currently in progress. We will share these extensions including development of a three-dimensional version of the model as well as use of an alternative approach, the immersed boundary method, to model such cells. Both of these efforts suggest this particular modeling approach is relatively versatile and useful for considering cell migration, osteocyte dynamics, and mechanically transduced biochemical products.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Javier-C-Urcuyo/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Javier C Urcuyo
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Mayo Clinic</em>
						    
						    
						    
					    		
					    			<h6>"Understanding glioblastoma-macrophage interactions through radiomics, transcriptome sequencing, and mathematical modeling"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Glioblastoma (GBM) is the most common primary brain tumor and has a poor median overall survival of just under 15 months. To combat this heterogeneous disease, the immune system initiates an inflammatory response, where both brain-resident microglia and blood-derived macrophages work to fight the tumor. However, some immune cells are co-opted by the tumor to express immune-suppressive signals, allowing for continued tumor growth and are thereby termed ‘glioma-associated macrophages’. To better understand the spatiotemporal dynamics of the interactions between tumor cells and these two macrophage phenotypes, we proposed the Proliferation-Invasion-Macrophage (PIM) model, which is a partial differential equation model that incorporates the proliferative and invasive behavior of GBM cells, as well as populations for both ‘healthy’ and ‘glioma-associated’ macrophages. Through exploring the parameter space, we classified the various dynamics of tumor progression. To apply the model to patient data, spatially-distributed image-localized biopsies were collected from a cohort of patients and RNA sequencing was performed. Correlations between normalized RNA counts of key genetic markers (i.e. CD68, CD163, SOX2, KI67) were analyzed. Patient imaging and RNA sequencing data were then utilized to train and validate a predictive machine learning model that outputs transcriptome expression maps for the aforementioned key genetic markers. This was then used to parameterize the PIM model for each patient. In doing so, this provided us with a detailed characterization of the interactions between the GBM and macrophage populations on a patient-by-patient basis. Through gaining an understanding of the interactions between glioma cells and the macrophage phenotypes, we can work towards developing personalized immunotherapies and other immune-targeted therapeutic strategies that combat this phenomenon.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jayrah-Bena-E-Ri%C3%B1on/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Jayrah Bena E Riñon
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of the Philippines Diliman and Bicol University</em>
						    
						    
						    
					    		
					    			<h6>"A Mathematical Model and Optimal Control of Schistosomiasis in Agusan del Sur, Philippines"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Schistosomiasis, a parasitic disease caused by 	extit{Schistosoma japonicum}, is one of the neglected tropical diseases and remains endemic in the Philippines, covering 28 provinces in 12 regions. Unlike other species of 	extit{Schistosoma}, 	extit{Schistosoma japonicum} is a zoonotic parasite which infects other mammalian hosts aside from humans. With that nature of the parasite, we construct a mathematical model to study the transmission dynamics of schistosomiasis in Agusan del Sur, Philippines. Here, we consider humans and carabaos as definitive hosts, and snails as intermediate hosts. We conduct stability analysis on the proposed model, and calculate for the basic reproduction number, $R_0$. We also perform sensitivity analysis using the Latin hypercube sampling combined with partial rank correlation coefficient technique to investigate how the number of infected humans is affected by the changes in model parameters. Using the available Philippine schistosomiasis data from the Department of Health, we estimate some model parameters. Finally, we apply optimal control theory to determine optimal strategies to control and prevent the spread of schistosomiasis in the country, which may eventually lead to the elimination of the disease.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jessica-Conway/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Jessica Conway
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Penn State</em>
						    
						    
						    
					    		
					    			<h6>"Stochastic time-inhomogeneous HIV dynamics following treatment suspension"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Antiretroviral therapy (ART) effectively controls HIV infection, suppressing HIV viral loads. Typically suspension of therapy is rapidly followed by rebound of viral loads to high, pre-therapy levels. However, recent studies suggest that approximately 10% of study participants undergoing ART treatment interruption show viral rebound only months or years after interruption, while some may be controlling infection permanently. We will first define what we mean by viral rebound and describe model-supported hypotheses of HIV viral rebound and control. We will then describe our branching process model to gain broad insight into these post-treatment dynamics. Specifically we provide theory that explains both short- and long-term viral rebounds, and post-treatment control, via a branching process with time-inhomogeneous rates, validated with data from Li et al. (2016). We will discuss the associated biological interpretation and implications. Finally, treatment interruption clinical trials are used to test efficacy of drug or other interventions to delay or prevent viral rebound; we will show how our modeling can be used to guide and inform such clinical trials.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jessica-Wellington/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Jessica Wellington
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Missouri</em>
						    
						    
						    
					    		
					    			<h6>"A Mathematical Model to Investigate Iron Allocation in Plants"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This presentation discusses the development of a mathematical model to study the mobilization of nutrients in plants – specifically iron. The development of the model involved the combined use of biological principles and the theory of ordinary differential equations. The model merges both biological and mathematical principles to construct a nutrient allocation model that can accurately reproduce experimental data. At each step of the model development process, we will discuss how the principles from both fields worked together to resolve the problems we faced while building a model that produces biologically meaningful results. We demonstrate that the model can be used as a virtual laboratory to study the plant’s response to changes in iron availability in the soil. The quantitative results from the simulations give insights into how to design future lab experiments (predictive biology). The motivation for building the model is the hope that further understanding of the uptake and storage of iron in plants will allow biologists to engineer plants – through precision breeding and gene editing- that can better respond to changes in soil nutrient concentrations.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jim-Greene/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Jim  Greene
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Clarkson University</em>
						    
						    
						    
					    		
					    			<h6>"A novel COVID-19 model reveals unexpected consequences of social distancing strategies"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Early 2020 saw the onset of the COVID-19 pandemic. As of July 14, 2020, there have been over 13 million confirmed cases worldwide, which have caused over 570K fatalities; in reality, the numbers are almost certainly much higher. As a vaccine has yet to be developed, social distancing as a form of a Nonpharmaceutical Intervention (NPI) has been enacted in many countries as a means of reducing the spread of the virus. Understanding the effects of social distancing to hopefully “flatten the curve” is fundamentally important in the design of reopening policies. In this work, we introduce a framework which explicitly models socially distanced populations via separate compartments: distancing regulations are modeled by flow rates between the distanced and non-distanced populations, and the overall reduction in transmissions due to distancing is also incorporated. In this way, both the response to distancing guidelines and their stringency can be explicitly modeled, and thus the control problem can be thought of as having two inputs from a policy-design perspective. We note that many authors have studied the control problem via reduction in transmission rate, whereas flow rate control has not been sufficiently analyzed; the latter is a focus of the current work. We compute the basic reproduction number R0, which characterizes the initial outbreak of the infection, and demonstrate that at sufficiently early stages of the pandemic when there is little immunity in the population, a quick implementation of social distancing is required in order for R0<1. We also find that R0 is sensitive to the fraction of infected individuals who become symptomatic (currently highly uncertain), illustrating the importance of obtaining a confident measurement of this value before quantitative model predictions can be trusted. Similarly, as it is currently unknown how infective asymptomatic carriers are, we investigate the dependence of R0 on distancing regulations (both flow rate and transmission reduction) as a function of the asymptomatic infection rate. Dynamic simulations of time-varying distancing guidelines also provide surprising results. We discover a critical implementation delay in issuing separation mandates. That is, there is a nontrivial but tight “window of opportunity” for commencing social distancing in order to meet the capacity of healthcare resources. Different relaxation strategies are also simulated. Periodic relaxation policies suggest a schedule which may significantly inhibit peak infective load, but that this schedule is very sensitive to parameter values and the schedule’s frequency. Furthermore, we consider the impact of steadily reducing social distancing measures over time. We find that a too-sudden reopening of society may negate the progress achieved under initial distancing guidelines (which is unfortunately playing out in real-time in the U.S.), but the negative effects can be mitigated if the relaxation strategy is carefully designed.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/John-Metzcar/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	John  Metzcar
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>SICE, Indiana University</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical modeling of leader-follower cell invasion of tumor-associated stroma using a novel extracellular matrix model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Collective cell migration and invasion are challenging topics to study as diverse biological processes may drive these behaviors. Mathematical modeling informed by biological experiments can lead to new insights. Here, we focus on a particular form of collective migration: collective invasion of tumor-associated stroma via a cell-based leader-follower mechanism. For the stroma, we implement a novel, simple extracellular matrix (ECM) model using three variables to represent a unit of ECM: a fiber density, anisotropy, and orientation. Furthermore, we implement bi-directional interactions between cells, represented as discrete agents, and the ECM. Cells remodel the ECM within their vicinity based on their motion and the ECM alters cellular motility. With this representation, we attempt to recapitulate experimental results of an organoid model of invasive breast cancer through a series of models that build additively on one another to introduce new biological hypotheses as additional agent model rules. Despite the increasing complexity of individual cells and short-range interactions, we find that our results do not significantly vary from one model to the next in overall qualitative behavior. This suggests that long range ECM remodeling and asymmetric cell-cell attachment/detachment processes might be necessary to recapitulate experimental results.  By proxy, it also suggests that these phenomena may possibly be necessary to enable collective invasion of ECM in organoid systems.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/John-R-Jungck/">
						    	<h4><li class="subgroup-counter subgroup-counter-EDUC"></li>
							    	
							    	John R Jungck
							    		(<span class="text-EDUC">EDUC</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Delaware</em>
						    
						    
						    
					    		
					    			<h6>"Ethnomathematics: Art, Culture and Social Justice"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			To address the continuing need to engage students with how mathematics can contribute to issues of equity versus equality, civil rights, social justice, and the historical contributions to mathematics from cultures around the world, ethnomathematics educators have developed numerous educational materials. Activities from a recent course which applied mathematics such as fractals, tessellations, cellular automata, groups, symmetries, graph theory, and game theory will be illustrated with various student projects. I argue that by engaging students in “Brave Spaces”  (see our Numeracy article) with mathematical tools and data, students will tackle difficult complex issues with minimal professorial facilitation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jonas-Knoch/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Jonas  Knoch
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Friedrich-Alexander-Universität Erlangen-Nürnberg</em>
						    
						    
						    
					    		
					    			<h6>"A mathematical model of HIF-1 regulated cellular energy metabolism"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We formulate a mathematical model of hypoxia-inducible factor 1 (HIF-1) mediated regulation of cellular energy metabolism, describing the reprogramming of cell metabolic processes from oxidative phosphorylation to glycolysis under reduced oxygen levels as it can be observed in many diseases such as sepsis or cancer. The model considers the dynamics of fifteen biochemical species and the proton concentration at the inner mitochondrial membrane with the underlying reaction processes localized in three intracellular compartments, namely cytoplasm, mitochondria and nucleus. More than sixty parameters of the model were calibrated using both the published data and a system steady-state based identification procedure. The model is validated by generating predictions which can be compared to empirical observations.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jonggul-Lee/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Jonggul  Lee
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>National Institute for Mathematical Sciences</em>
						    
						    
						    
					    		
					    			<h6>"Machine Learning for Risk Prediction of Highly Pathogenic Avian In uenza in the Republic of Korea"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			There have been 7 outbreaks of highly pathogenic avian influenza (HPAI) in the Republic of Korea since 2003 resulting a serious economic burden on the poultry industry. Due to uncertainty of transmission from migratory birds, which is known as the main source of infection, and transmission between poultry farms linked by livestock-related vehicles, it is very difficult to predict and respond to the epidemic. In this work we aim at forecasting spatio-temporal pattern of HPAI occurrence and identifying risk factors with a machine learning technique based on Random Forest regression. Historical data on HPAI outbreaks in 250 regions from 2014 to 2017 are used as a target. Three types of features are used to train the model: epidemiological features related to information on farms infected in the past, demographic features including the number (density) of farms regarding breeding species (chicken and duck) in an area, and geographical features including the habitats of migratory birds and slaughterhouses. The model provides a highly accurate prediction of both temporal and spatial patterns of HPAI outbreaks. Furthermore, we investigate feature importance to explain which features contribute most to the local outbreak of HPAI. Results show that epidemiological features mainly contribute to prediction of the temporal pattern, while the demographic and environmental features mainly contribute to prediction of the spatial distribution.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jordy-Jose-Cevallos-Chavez/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Jordy Jose  Cevallos Chavez
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Arizona State University</em>
						    
						    
						    
					    		
					    			<h6>"Mobility impact in the spreading of COVID-19 in Ecuador "</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Ecuador has reported one of the highest per capita death rates of COVID-19 in the world,  with  more  than  5000  deaths  in  a  country  of  approxi-mately 17 million people.  Transmission of COVID-19 infection in Ecuador has been the result of contact patterns, mobility structure of the population, regional epidemiology, and efficacy of public health interventions. In this study, we link provincial-level demographic,  epidemiological,  and  transportation  information with the spread of COVID-19 outbreak to understand the role of local patterns of low and high-density provinces on the infection growth rate at the country level. The analysis is carried out using best (with no interprovincial movement) and worst (with movement patterns similar to before COVID-19 outbreak) case scenarios in Ecuador. The results suggest that human movement (instead of local epidemiology) has primarily been shaping transmission dynamics of COVID-19 in Ecuador by introducing infected individuals regularly into low-risk provinces.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jorge-Arroyo-Esquivel/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Jorge  Arroyo-Esquivel
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Department of Mathematics, University of California Davis</em>
						    
						    
						    
					    		
					    			<h6>"Spatial dynamics and spread of ecosystem engineers"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Ecosystem engineers are organisms characterized by interacting with other organisms thorough physical modifications of modifying their habitat. Examples of ecosystem engineers include Spartina alterniflora cordgrass or the zebra mussel Dreissena polymorpha. For both of these, the effect of modifying the environment can be non-local, affecting other regions farther away from the region populated by the ecosystem engineer. This shows the importance of understanding the population dynamics of ecosystem engineers in a spatial context. To do this we have developed the simplest spatially explicit model possible of ecosystem engineers, incorporating two local populations. We use this model to understand the relationship between dispersal and engineering effects, both at local and regional scales. Our main result is that the delayed Allee effect induced in the nonspatial model is extended to the spatial model, so the spread dynamics of an ecosystem engineer can be similar to the Allee case. However, there are more complex possibilities due to the effect of the environment modification.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Joseph-S-Abrams/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Joseph S Abrams
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Saskatchewan</em>
						    
						    
						    
					    		
					    			<h6>"A CELL-BASED MODEL OF INTERCELLULAR MECHANICS DURING CPA EQUILIBRATION IN PREANTRAL OVARIAN FOLLICLES"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The rational design of cryopreservation protocols is an effective method for improving the outcome of sample survival. Cryopreservation of preantral ovarian follicles is an experimental therapy for fertility preservation and is of particular value in prepubescent cancer patients. Post-cryopreservation survival of this tissue is unsatisfactorily low for clinical use. Current tissue models of cryopreservation are largely focused on mass transport and the cytotoxicity of cryopreservatives, however, the potential mechanical damage to intracellular connections in response to mass transport is not considered. Intracellular connections between the granulosa cells and the oocyte in ovarian follicles, known as transzonal projections (TZPs), previously, have been shown to sever during cryopreservation. We hypothesized that the damage to TZP’s is due to variation in mass transport responses between heterogeneous cell types. Here we present a cell-based model, informed by experimentation, to capture mass transport, toxicity, and intracellular connections during the equilibration phase of cryopreservation. Using this model we explore several methods for improving cryopreservation protocols with a focus on improving TZP survival and thus post-thaw functionality.


Source of Funding: This work was supported by funding from the Canadian National Science and Engineering Research Council (RGPIN-2017-06346), the US National Institute of Child Health and Human Development (5R01HD083930-02) and the National Institute of Health (P51OD011092). Funding from the National Institute of Health supports the Oregon National Primate Research Center (ONPRC.




Conflict of Interest: None to disclose
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Josephine-Naa-Ayeley-Tetteh/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Josephine Naa Ayeley Tetteh
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Frankfurt Institute of Advanced Studies</em>
						    
						    
						    
					    		
					    			<h6>"Switching strategy for mitigation against bacterial resistance"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The control of drug resistant infections has become difficult as there are little to no new drugs being discovered. Using control engineering approaches, we develop strategies aimed at minimizing the appearance of drug-resistant pathogens within the host. With a mathematical model based on a two-strain bacterial population, a switching strategy can be found to ensure the stability of the eradication equilibrium based on the use of a Lyapunov function. Our numerical simulations support the use of this switching strategy for mitigation against bacterial resistance.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Joshua-A-Bull/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Joshua A Bull
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Oxford</em>
						    
						    
						    
					    		
					    			<h6>"Description of immune cell infiltration in solid tumours using spatial statistics and topological data analysis"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The increasing digitization of immunohistochemistry slides provides opportunities for pathologists to automate routine tasks and improve current workflows. Techniques for identifying immune cells in biopsies or surgically resected tumours are now widespread, with several open-source or commercial image analysis platforms providing cell identification tools. While these tools can be used to calculate statistics such as the density of immune cells in a given tumour region, a property which correlates with patient prognosis, consideration of spatial statistics and topological analyses which can provide more detailed spatial descriptions of localisation is not widespread.
We introduce several spatial statistical descriptors which can be used to describe localisation of immune cells within a tumour, and apply them to macrophage distributions in human IHC images and in simulated datasets. We show that key features of the spherical contact distribution, the pair correlation function, and the J-function vary predictably as the degree of immune cell infiltration from stromal to tumour regions increases in simulated data. Using these statistics, we introduce a new method based on maximum likelihood estimation which combines the strengths of different spatial descriptors to automatically classify macrophage infiltration into tumour nests. We validate our approach by applying it to macrophage distributions from clinical datasets, obtaining infiltration indices which match the qualitative assessments of experienced pathologists. Finally, we demonstrate how topological data analysis can be applied to macrophage distributions in human IHC images and in simulated datasets to enhance these descriptions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Juan-Calvo/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Juan  Calvo
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidad de Granada</em>
						    
						    
						    
					    		
					    			<h6>"The initial-boundary value problem for the Lifshitz-Slyozov system with inflow boundary conditions: Analysis and applications"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The Lifshitz-Slyozov system describes the temporal evolution of a mixture of particles ('atoms') and aggregates, where individual atoms can attach to or detach from already existing clusters. The aggregate distribution follows a transport equation with respect to a size variable, whose transport rates are coupled to the dynamic of atoms through a mass conservation relation. Being a system traditionally designed to model phase transitions, the attachment and detachment rates proposed by Lifshitz and Slyozov are such that no boundary condition at zero size is needed. However, the scope of this model is becoming wider (e.g. descriptions of protein polymerization or tentative applications to oceanography). These situations impose attachment and detachment rates that requiere a boundary condition at zero size, which is intrepreted as the synthesis of new clusters from atoms by a nucleation process. Up to date, the mathematical results on this new setting are scarce. In this contribution we study existence and uniquenes of local-in-time solutions when nonlinear boundary conditions are used, together with continuation criteria and results on long-time behavior. We are able to deal with attachment and detachment rates that may eventually lack Lipschitz regularity, like power-law rates. This requires a careful analysis of the characteristic curves associated to the transport process.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Juliane-Schr%C3%B6ter/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Juliane Schröter
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Utrecht University, Theoretical Biology & Bioinformatics</em>
						    
						    
						    
					    		
					    			<h6>"A theoretical model for the natural course of HIV infection in infants"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			HIV infection in young children differs markedly from that in adults: (i) children have higher viral loads (VL), (ii) their set point VL is not much lower than the peak VL, and (iii) children tend to progress faster towards AIDS. We use classic simple ODE models for HIV infection to study which differences between adults and children can explain these observations. We test whether (1) increased viral replication rates, (2) increased target cell population, (3) increased production of target cell population, and/or (4) delayed/weakened immune responses in children can explain the data. Whenever possible, we feed the model with parameter estimates from untreated paediatric data available from datasets within the EPIICAL project (https://www.epiical.org/). This data allowed us to reject hypothesis (1): the viral replication rate seems to be decreased in infants compared to adults. Thus, a mathematical model might help to get a better understanding of paediatric HIV infections and provides a foundation to simulate current cure and prevention strategies.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Juliano-Ferrari-Gianlupi/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Juliano Ferrari Gianlupi
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Multiscale spatiotemporal modeling of acute primary viral infection and immune response in epithelial tissues"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Simulations of tissue-specific effects of primary acute viral infections like COVID-19 are essential for understanding differences in disease outcomes and optimizing therapeutic interventions. We present a multiscale model and simulation of an epithelial tissue infected by a virus, a simplified cellular immune response and viral and immune-induced tissue damage. The model exhibits basic patterns of infection dynamics: widespread infection, slowed infection, recurrence, containment and clearance. Inhibition of viral internalization and faster immune-cell recruitment promote containment of infection. Fast viral internalization and slower immune response lead to uncontrolled spread of infection. Because antiviral drugs can have side effects at high doses and show reduced clinical effectiveness when given later during the course of infection, we studied the effects on infection progression of both treatment potency (which combines drug effectiveness and dosage) and time-of-first treatment after infection. Simulation of a drug which reduces the replication rate of viral RNA shows that even a low potency therapy greatly decreases the total tissue damage and virus burden when given near the beginning of infection. However, even a high potency therapy rapidly loses effectiveness when given later near the time of peak viral load in the untreated case. Many combinations of dosage and treatment time lead to stochastic outcomes, with some simulation replicas showing clearance or control of the virus (treatment success), while others show rapid infection of all epithelial cells in the simulated tissue subregion (treatment failure). This switch between a regime of consistent treatment success and failure occurs as the time of treatment increases. However, stochastic variations in viral spread mean that high potency treatments at late times are occasionally effective. The model is open-source and modular, allowing rapid development and extension of its components by groups working in parallel. We're extending the model through already calibrated ODE models to have more biological meaningful behaviors. ODE models can be calibrated in a straightforward  manner, however they don't contain information about space which is meaningful. As ODE based models are not spatial they need to be spatialized in some manner for our use, we've developed a method to generate spatial models from ODE models and have the spatial model recover the ODE predicted population behaviors. While the spatialized model shows differences, we recuperate the overall ODE model behavior, and we are exploring how spatiality itself causes those differences. With this work we will bring forth more ways in which ODE models can be useful, e.g., having their overall behavior inform and predict how the COVID infection spreads through the lungs.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Jungmin-Han/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Jungmin Han
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>National Institutes of Health</em>
						    
						    
						    
					    		
					    			<h6>"Missing Data Imputation and Gene Network Inference in Single Cell Analysis"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			With advances in single-cell techniques, collecting a large quantity of data has become more accessible and efficient. In contrast, the increased complexity of data has made it more challenging to draw biologically relevant conclusions. As a result, there is an increase in demand for computational methods capable of dealing with such complexity and of providing some predictive deductions from the data. In this study, we present the use of neural networks in imputing missing data and a novel method for the inference of a gene network using least absolute deviation regression. Fowlkes et al. (Cell, 2008) published a set of gene expressions measured from 6078 Drosophila blastoderm during six different time cohorts that spanned the 50 minutes prior to the onset of gastrulation. Out of 95 genes and four proteins, only 27 of them had complete temporal information from all the cells, while the rest were measured only in a subset of cells. The missing data constituted about 37% of the whole data set. To impute the missing data, we trained and tested neural networks with one hidden layer on the complete 27 genes as predictors and the genes that were measured only in subsets of cells as targets. With the trained neural network, we imputed the missing gene expressions. To test the imputation method’s performance, we arbitrarily selected three genes from the complete 27 genes and randomly removed time points from their gene profiles. Then, the missing values were imputed using the same method. The medians of the imputed values were compared to those of the observed values and showed negligible differences. We then used a variation of least absolute deviation regression to infer a mechanistic model of the gene network that governs the discrete gene dynamics. The accuracy of the model was compared to those of mechanistic models inferred with a standard least squared regression and with non-mechanistic neural networks with multiple hidden layers. The models were used to predict the gene profiles given the initial values, and the errors were computed. Since the regression methods have different cost functions, we compared the distributions of errors in two metrics, in $L_{1}$ norm and $L_{2}^{2}$ norm. The model inferred with the least absolute deviation regression showed higher predictive power than the models using other methods. The data set was titrated to a smaller sample size to evaluate performance. In the limit of small sample size, the model inferred with our choice of regression performed better than the one inferred with least squared regression, but not as well as the trained neural networks.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Junho-Lee/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Junho  Lee
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Konkuk University</em>
						    
						    
						    
					    		
					    			<h6>"Synergistic Effects of Bortezomib-OV Therapy and  Anti-Invasive Strategies in Glioblastoma:  A Mathematical Model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Recent experimental studies have demonstrated the great potential of combination therapies, using oncolytic viruses (OVs) in conjunction with proteasome inhibitor, bortezomib (BTZ), for the treatment of glioblastoma. So, we have developed a mathematical model of combination (bortezomib+OV) therapy, including intracellular signaling network (proteasome-NKkB-Bcl2-Bax) which mediate anti-apoptosis, apoptosis, and necroptosis of tumor cells. In addition, a challenging tumor microenvironment (TME) such as gray matter and dense ECM structure in brain, has been shown to regulate tumor invasion. But the critical role of TME in such therapies has not been studied in the context of combination therapies. We show (i) how the intracellular signaling regulates tumor cell killing in the combination therapy, (ii) that the TME plays a significant role in controlling the anti-tumor efficacy in Bortezomib-OV combination therapies and generating various spatial patterns of tumor growth. The simulation results show the possibility of development of new tumor treatment options within TME and new anti-invasion strategy.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Karina-Vilches/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Karina  Vilches
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Catholic University of Maule</em>
						    
						    
						    
					    		
					    			<h6>"A mathematical approach for visualizing cell migration during tumor progression"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The present research project consists of a mathematical approach that captures and explores a wide range of mechanisms and biological variability to simulate the collective cell migration during the tumor progression. This orchestrates multiple phenomena in cancer dynamics is represented by a chemotaxis two-species system, which is supported by an extensive literature. In this respect, we promote the realization of modeling platforms that facilitate the integration of interdisciplinary perspectives in tumor progression. Furthermore, the theoretical migration scenario of tumor cells extends the previous results of the chemotaxis and chemotaxis-haptotaxis systems adding more complexity to the mathematical model for representing these cell-cell dynamics including micro-environmental influence. The seeking of regimes in which tumor invasion occurs with a low total mass of tumor cells could be an interesting initial point for an interdisciplinary discussion about theoretical results obtained, and how such analytical results could be transferred to the laboratory. This last consideration is that the density of tumor-associated macrophages in tumor site results in an important characteristic to identify the cancer state, and some researchers suggest that targeting processes required for collective migration may be effective in combating certain types of tumors.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Kathyrn-R-Fair/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Kathyrn R Fair
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Guelph</em>
						    
						    
						    
					    		
					    			<h6>"Spatial Structure in Protected Forest-Grassland Mosaics: Exploring Futures Under Climate Change"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In mosaic ecosystems, multiple land types coexist as alternative stable states exhibiting distinct spatial patterns. Forest-grassland mosaics are ecologically valuable, due to their high species richness. However, anthropogenic disturbances threaten these ecosystems. Designating protected areas is one approach to preserving natural mosaics. Such work must account for climate change, yet there are few spatially-explicit models of mosaics under climate change that can predict its effects. We construct a spatially-explicit simulation model for a natural forest-grassland mosaic, parameterized for Southern Brazil. Using this model, we investigate how the spatial structure of these systems is altered under climate change and other disturbance regimes. By including local spatial interactions and fire-mediated forest recruitment, our model reproduces important spatial features of protected real-world mosaics, including the number of forest patches and overall forest cover. Multiple concurrent changes in environmental conditions have greater impacts on tree cover and spatial structure in simulated mosaics than single changes. This sensitivity reflects the narrow range of conditions under which simulated mosaics persist and emphasizes their vulnerability. Our model predicts that, in protected mosaics, climate change impacts on the fire-mediated threshold to recruitment will likely result in substantial increases in forest cover under Representative Concentration Pathway (RCP) 8.5, with potential for mosaic loss over a broad range of initial forest cover levels. Forest cover trajectories are similar until 2150, when cover increases under RCP 8.5 outpace those under RCP 2.6. Mosaics that persist under RCP 8.5 may experience structural alterations at the patch and landscape level. Our simple model predicts several realistic aspects of spatial structure as well as plausible responses to likely regional climate shifts. Hence, further model development could provide a useful tool when building strategies for protecting these ecosystems, by informing site selection for conservation areas that will be favourable to forest-grassland mosaic under future climates.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Keertana-Yalamanchili/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Keertana  Yalamanchili
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Assessing the Efficacy of Various NOX Enzyme Inhibitors as Potential Treatments for Ischemic Stroke in Silico"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Ischemic stroke occurs when blood flow to the brain is interrupted, causing brain damage. There is evidence that ROS (reactive oxygen species) are produced by the enzyme family NADPH oxidase (NOX) following ischemic stroke, which leads to further brain injury. The ADMET profiles of each inhibitor was taken, in which four classifications, namely applicability domain, human intestinal absorption, blood brain barrier, and human oral bioavailability, were observed. Then, AutoDock Vina was used to model the docking of the inhibitors: VAS2870, GSK2795039, Apocynin, and AEBSF to NOX2, an isoform of the NOX family. The binding affinities of each of the inhibitors to NOX2 were recorded, and the value was used to calculate the Ki value of each inhibitor. It was found that VAS2870 and Apocynin were the most potent NOX2 inhibitors (p < 0.001). All the inhibitors did inhibit the NOX2 enzymes, and they all had favorable ADMET profiles. This study helps corroborate previous in vivo and in vitro studies in an in silico format, and can be used towards evidence for developing drugs to treat ischemic stroke.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Kellen-Myers/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Kellen  Myers
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Tusculum University</em>
						    
						    
						    
					    		
					    			<h6>"Resource Limitations and Household Economics in Outbreaks"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Epidemiological models have been employed with great success to explore the efficacy of alternative strategies at combating disease outbreaks. These models have often incorporated an understanding of age-based susceptibility and severity of outcome, considering how to limit the adverse outcomes or disease burden relative to an age structure. Such models frequently recommend the preferential treatment/vaccination of children or elderly, demonstrating how prevention of serious disease within these etiological subgroups can provide both protection within the subgroup itself and indirect protection to the broader population.
However, it is most frequently the case that these target populations are consumers, rather than providers, of household resources. In areas of the globe where continued health of household members relies on continued provision of resources, these models may fail to provide the most effective overall strategies for health outcomes in both target populations and overall. This is particularly important for tropical diseases impacting rural and low-income areas in which the disease may be endemic or newly emergent, particularly in the wake of natural disasters.
We propose a modified SIR model with targeted treatment in resource-limited populations. We evaluate the model over a broad parameter space. This model demonstrates how economic limitations may shift the optimal strategy. It may be advantageous to treat populations at lesser direct risk if they are responsible for providing secondary protection to higher-risk population(s) by producing household resources. Evaluation of this model over the parameter space reveals that, in some cases, targeting treatment towards consumers may result in greater numbers of consumer infections.
Our results demonstrate how household resource limitation can, in certain regions of the parameter space, drastically affect the impact of targeted treatment strategies for limiting epidemics. Depending on the economic circumstances, it is possible that focusing treatment on consumers such as children can produce a counter-intuitive outcome in which more children contract the disease.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Kelly-R-Buch/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Kelly R Buch
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Tennessee Knoxville</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical Model of Basal Sprout Production in Response to Laurel Wilt"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Laurel wilt is fatal fungal tree disease vectored by the invasive Redbay Ambrosia Beetle (RAB). Redbay trees and sassafrass trees are both susceptible to the fungal disease, and upon infection both provide suitable host material for RAB. Each of these two species responds differently to the fungal infection, with only redbay trees producing shoots that grow from the root system, called basal sprouts, and thus providing further potential host material for RAB. Using a stage structured SI model, we will explore the effect of basal sprout production on boom and bust disease cycles and the establishment of RAB. We interpret our results to provide insight on the circumstances which lead to Laurel wilt becoming endemic or local extinction of Sassafras and Redbay trees, which is vital for disease control.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Kento-Nakamura/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Kento  Nakamura
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Tokyo</em>
						    
						    
						    
					    		
					    			<h6>"Optimality of the sensory system of Escherichia coli"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Escherichia coli chemotaxis is one of the model systems from which we can obtain insights for understanding the biological sensory system. To realize chemotaxis, an E. coli cell has to detect temporal changes of ligand concentration caused by its motion based on noisy sensing of the ligand in the environment. How eﬃciently is the sensory system of E. coli designed to infer the dynamically changing environment behind the noise? While several works have analyzed the eﬀect of noise on the signaling pathway and predicted the necessary feature for the sensory system to operate against noise, these analyses relied on the linear response approximation, which may limit the predictive capacity of the model. In this work, we utilize the nonlinear ﬁltering theory to explore the requisite for information acquisition in chemotaxis. First, we derive the optimal dynamics for extracting the necessary information for chemotaxis, i.e., temporal concentration change. Then, we show how the derived dynamics can be linked to a biochemical model of the sensory system of E. coli. Furthermore, we demonstrate that the optimal dynamics obtained can reproduce a nonlinear response relation observed experimentally. These results indicate that the bacterial sensory system may be developed so as to obtain environmental information from a noisy and dynamic signal.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Khem-Raj-Ghusinga/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Khem Raj  Ghusinga
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UNC</em>
						    
						    
						    
					    		
					    			<h6>"Molecular switch architecture determines response properties of signaling pathways"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Many intracellular signaling pathways are composed of molecular switches, proteins that transition between two states—on and off. Typically, signaling is initiated when an external stimulus activates its cognate receptor that in turn causes downstream switches to transition from off to on using one of the following mechanisms: activation, in which the transition rate from the off state to the on state increases; derepression, in which the transition rate from the on state to the off state decreases; and concerted, in which activation and derepression operate simultaneously. We use mathematical modeling to compare these signaling mechanisms in terms of their dose-response curves, response times, and abilities to process upstream fluctuations. Our analysis elucidates several general principles. First, activation increases the sensitivity of the pathway, whereas derepression decreases sensitivity. Second, activation generates response times that decrease with signal strength, whereas derepression causes response times to increase with signal strength. These opposing features allow the concerted mechanism to not only show dose-response alignment, but also to decouple the response time from stimulus strength. However, these potentially beneficial properties come at the expense of increased susceptibility to upstream fluctuations. In addition to above response metrics, we also examine the effect of receptor removal on switches governed by activation and derepression. We find that if inactive (active) receptors are preferentially removed then activation (derepression) exhibits a sustained response whereas derepression (activation) adapts. In total, we show how the architecture of molecular switches govern their response properties. We also discuss the biological implications of our findings.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Koushik-Garain/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Koushik Garain
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Treatment control is one of the most important factor to reduce the spread of COVID-19 epidemic"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			On 31st December 2019, China first reported WHO about a unknown disease and on 11th February 2020, WHO announced the name COVID-19. On 11th March 2020, WHO declared COVID-19 (Novel Coronavirus) as a pandemic. It is the global health crisis in this time and is the biggest challenge we have faced since world war two. To curb the spread of COVID-19, most of the countries implemented lockdown but the pandemic is still growing significantly. Our aim is to show that lockdown is one factor only to reduce transmission of the diseases but one more important factor is treatment control. In this paper, we propose a SIRS model with a treatment function. In this model, we have determined the basic reproduction number R0, which is inversely proportional to the treatment capacity. We can control the outbreak of COVID-19 when the basic reproduction number is less than one. We will show that increasing the maximum capacity we can stop the disease. Example of China shows that treatment is playing a bigger role, China set up a special 1000 bed hospital in just 10 days. Lockdown and quarantine are not sufficient and increasing the capacity of treatment to maximum is suggested to decrease the infective population.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lan-K-Nguyen/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Lan K Nguyen
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Monash University</em>
						    
						    
						    
					    		
					    			<h6>"Integrative mathematical modelling unveils hidden mechanism of resistance to PI3K  inhibition and identifies new effective combination therapies for breast cancer"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The phosphatidylinositol 3-kinase (PI3K)-AKT-mTOR signalling pathway is a master regulator of cell growth and its activation is frequently associated with cell transformation and cancer. This is particularly common in breast cancer, where alterations in members of this pathway occur in over 50% of patients, irrespective of tumour subtype. Over the last decade, targeted drugs directed at the PI3K pathway, particularly inhibitors directed at PI3K, have been under intense clinical development. However, the emergence of acquired and/or adaptive resistance to these agents, the latter involving dynamic rewiring of signalling networks and crosstalk, has presented major challenges for the delivery of impactful treatments. This highlights the critical need to identify the molecular mechanisms through which tumour cells rewire their signalling outputs and bypass the inhibitory effect of targeted therapies, and to develop more effective combination therapies. To address these challenges, we constructed a multi-pathway mechanistic model based on differential equations that integrates the PI3K-AKT signalling axis with key cancer-relevant pathways, incorporating known feedback and cross-talk mechanisms. We calibrated this model using time-course kinetic data in response to inhibition of PI3K by a selective and clinically-relevant inhibitor BYL719 (BYL), obtained from the T47D breast cancer cell lines. Integrative simulations/experimental analyses reveal an unexpected role for the cyclin-dependent kinase inhibitor p21, which in contrary to its known growth-inhibitory function, appears to promote resistance to PI3K inhibition. Consistent with this, model simulations further predict a dynamic and adaptive reactivation of p21 following acute BYL treatment, which we validated experimentally using immunoblotting and phosphoproteomic profiling in both parental T47D cells and cells that have become resistant to BYL. Next, following a similar approach we recently published, we simulated the effect of various potential drug combinations targeting pair-wise nodes within the PI3K integrative network to identify potential co-targets that can be effectively combined with PI3K inhibition for more anti-tumour benefit. Among these, we predict dual inhibition of PI3K and the kinase PDK1 displays the most potent synergistic effect in suppressing pro-growth signalling and cancer cell growth. Model predictions were subsequently validated using immunoblotting and cell viability assays. In addition, analysis of breast cancer patient data from TCGA demonstrates concomitant overexpression of the genes encoding PIK3 and PDK1 is associated with worse patient survival, further supporting their validity as co-targets. Collectively, our integrative analyses uncovered novel resistance mechanisms against PI3K inhibition, and identified effective combination therapeutic strategies that overcome such resistance, leading to better treatment for PI3K-driven breast cancer.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lauren-Mossman-Kylie-Landa/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Lauren  Mossman, Kylie Landa
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>St.Olaf College</em>
						    
						    
						    
					    		
					    			<h6>"Phage-antibiotic synergy inhibited by temperate and chronic virus competition"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			As antibiotic resistance grows more frequent for common bacterial infections, alternative treatment strategies such as phage therapy have become more widely studied in the medical field. While many studies have explored the efficacy of antibiotics, phage therapy, or synergistic combinations of antibiotics and phage, the impact of virus competition on the efficacy of antibiotic treatment has not yet been considered. Here, we model the synergy between antibiotics and two viral types, temperate and chronic, in controlling bacterial infections. We demonstrate that while combinations of antibiotic and temperate viruses exhibit synergy, competition between temperate and chronic viruses inhibits bacterial control with antibiotics. In fact, our model reveals that antibiotic treatment counterintuitively increases the bacterial load when a large fraction of the bacteria develop antibiotic-resistance.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Laurie-Balstad/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Laurie  Balstad
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>St. Olaf College</em>
						    
						    
						    
					    		
					    			<h6>"Parasite intensity influences evolution of migratory behavior via migratory escape"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Migration can allow individuals to escape parasite infection, which can lead to a lower infection probability (prevalence) in a population and/or fewer parasites per individual (intensity). Since individuals with more parasites often have lower survival and/or fecundity, infection intensity shapes the life-history tradeoffs determining when migration is favored as a strategy to escape infection. Yet, most theory relies on susceptible-infected (SI) modeling frameworks, defining individuals as either healthy or infected, ignoring details of infection intensity. Here we develop a novel modeling approach that captures infection intensity as a spectrum, and ask under what conditions migration evolves as function of how infection intensity changes over time. We show that the relative timescales of migration and infection accumulation determine when migration is favored. We also find that population-level heterogeneity in infection intensity can lead to partial migration, where less-infected individuals migrate while more infected individuals remain resident. Our model is one of the first to consider how infection intensity can lead to migration. Our results frame migratory escape in light of infection intensity, rather than prevalence, thus demonstrating that decreased infection intensity should be considered a benefit of migration, alongside other typical drivers of migration.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lee-Altenberg/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Lee  Altenberg
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Hawaii at Manoa</em>
						    
						    
						    
					    		
					    			<h6>"Spectral Graph Theory in the Analysis of Biological Evolution"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Weighted graphs can be used to model biological evolution under natural selection and mutation: vertices correspond to genotypes, edges correspond to mutation from one genotype to another, vertex weights correspond to fitnesses, and edge weights correspond to mutation rates.  Weinberger, Stadler, Grover and others have used Fourier decompositions of the vertex weights in terms of the eigenvectors of the edge weights, in order to characterize the 'ruggedness' of these 'adaptive landscapes'.  This Fourier analysis has been used to characterize random walks on the graphs.  But it has not been used in the actual evolutionary population dynamics under selection and mutation.  Here we find that the original spectral graph results of Collatz and Sinogowitz (1957) appear as a lower bound on the asymptotic mean fitness of the population, and the spectral gap of the mutation matrix appears in an upper bound.  This reveals an intimate connection between robustness of a population to mutation and the relaxation times of population perturbations due to mutation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lee-Curtin/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Lee  Curtin
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Mayo Clinic</em>
						    
						    
						    
					    		
					    			<h6>"Lacunarity and fractal dimension as prognostic biomarkers in glioblastoma"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Glioblastoma (GBM) is the most aggressive primary brain tumor with a median survival of only 15 months with standard of care treatment. Typically, these tumors present with regions of necrosis, contrast enhancement and edema, visible on standard clinical magnetic resonance imaging (MRI). The prognostic impact of the shape of these regions has not been fully explored. Lacunarity and fractal dimension are two quantitative morphological measures that describe how shapes fill space and their complexity at varying spatial scales. Both of these measures have been shown to distinguish overall survival (OS) and progression free survival (PFS) when applied to regions of necrosis. In our cohort of patients with first-diagnosis GBM (n=400), we sought to validate these previously published results and extend this work to other tumor-induced imaging abnormalities. We calculated median lacunarity and fractal dimension values of necrosis (n=390), necrosis with contrast enhancement (n=400), and edema (n=257) on a per patient basis and searched for cutoffs that significantly distinguished survival. In our cohort, we found that lacunarity can significantly distinguish PFS when applied to necrosis and can significantly distinguish OS when applied to necrosis with contrast enhancement, or edema. We find that fractal dimension can also significantly distinguish OS when applied to edema. We believe that morphological measures such as lacunarity and fractal dimension may play an important prognosticating role in GBM presentation. This link between morphological and survival metrics could be driven by underlying biological phenomena, tumor location, or microenvironmental factors that should be further explored.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Leili-Shahriyari/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Leili  Shahriyari
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Massechusetts Amherst</em>
						    
						    
						    
					    		
					    			<h6>"A path toward personalized cancer treatments"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			A major clinical challenge for cancer therapies is to obtain an effective treatment strategy for each patient or at least identify a subset of patients who could beneﬁt from a particular treatment. Since each cancer has its own unique features, it is very important to obtain personalized cancer treatments and ﬁnd a way to tailor treatment strategies for each patient. Recently, mathematical models have been commonly used to discover, validate, and test drugs. Since these models are a complex system of nonlinear equations with many unknown parameters, estimating the values of the model's parameters is extremely difﬁcult. Existing parameter estimation methods for these models often use assembled data from various sources rather than a single curated dataset. These datasets are usually obtained through various biological experiments, in vitro and in vivo animal studies. To arrive at personalized treatments, we need to obtain values of parameters of the model for each patient separately. Since the set of variables of the model includes relative amount of each cell type and cytokines in the tumor, we developed a tumor deconvolution software, which is a combination of recently developed methods, to predict the relative amount of these variables from the gene expression profile of the tumor. The output of the tumor deconvolution software can be used to predict the values of the parameters for each patient. In other words, we propose to use patients’ gene expression data of primary tumor to estimate the values of parameters of the mathematical model for each patient separately, instead of the common approach of assuming these parameters have the same values across all patients and using animal studies to estimate them. This new approach provides us with a unique opportunity to suggest the optimal treatment strategy for each patient and predict the efﬁcacy of each treatment for each patient.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Luana-Tais-Bassani/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Luana Tais Bassani
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidade de São Paulo</em>
						    
						    
						    
					    		
					    			<h6>"Basic sanitation's role over breeding sites maintenance for Aedes aegypti development during rain absence according to a fuzzy rule-based system"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Basic sanitation refers to the set of services, infrastructures, and operational installations for the supply of drinking water, sanitary sewerage, and management of solid waste. A precarious basic sanitation service contributes to the spreading of water-diseases. The delimitation of the urban and rural areas supports the public policies of sanitation investment and vector-borne control. In continental land sized countries like Brazil, this categorization has the influence of subjective aspects from peri-urban areas. When we evaluate sanitation system efficiency, we take into account that the total population services matter, which helps to understand and avoid frontier region border diseases that can potentially spread to urban regions and cause outbreaks. Recent episodes of Aedes aegypti related diseases in Brazilian regions that used to be immune, due to the geographic location and severe winters, shows that Brazilian public health is not prepared to deal with Aedes aegypti development. We propose a fuzzy rule-based system to measure the influence of sanitation and rain lack over the Aedes aegypti population dynamics. The system entries are fuzzy sets that involve four linguistic variables, which are the percentage of the urban population, as well as the percentage of served population by water supply, frequents solid waste collection, and sanitary sewerage. We use real data disclosed by the Brazilian institutions IBGE and SNIS (Instituto Brasileiro de Geografia e Estatística - Brazilian Institute of Geography and Statistics; Sistema Nacional de Informações sobre Saneamento - Brazilian sanitation information system). The system output feeds a characteristic parameter, which aims to quantify the contribution attributed to the basic sanitation of the city on the maintenance of breeding sites of Aedes aegypti during periods without rain. The fuzzy system provides a value that represents a parameter, which we couple to the age and stage-structured population projection matrix model for Aedes aegypti. A value closer to 0 means that the sanitation panorama of the city is ideal, and the Aedes aegypti population development is limited, according to lower precipitation periods. Spite of the slow development due to the water sources' lack, it is sensitive to temperature and individuals' age. Through the model, basic sanitation plays a role over stimuli for the quiescent egg, and mortality of larva and pupa during periods with a few precipitation millimeters episodes. Through this analysis, we are forecasting population dynamics fluctuations in drier periods, which is highly influenced by basic sanitation in municipalities that offer precarious sanitation services.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lucas-Barberis/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Lucas  Barberis
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>IFEG-CONICET</em>
						    
						    
						    
					    		
					    			<h6>"Modeling Helps Understand the Influence of Substrate on Tumorsphere Growth"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Tumorspheres, cellular spheroids formed by clonal proliferation from established cell lines or tumor tissue, are experimental systems used to investigate diverse features of cancer. They may be especially useful to ascertain the effects of cancer stem cells on neoplastic development. Here we use a recently developed model that considers the interactions between cell subpopulations [L. Benitez et al, Physica A 533, 121906 (2019)] to interpret the results of experiments probing the influence of substrate hardness on tumorsphere growth [Wang et al, Oncol. Lett. 12, 1355 (2016)]. These authors cultured breast cancer stem cells on soft and hard matrix surfaces using stem cell growth factors, observing that the number of cancer stem cells increased continuously, albeit in different ways. They also cultured the cancer stem cells on hard agar in the absence of growth factors (the “control” experiment). In this case the spheroids grew faster, even if the stem cell number remained stationary. Fitting our model results to the data corresponding to the use of growth factors, we found that interspecific interactions between cells in different populations always promoted growth via a positive feedback loop. These interactions enhanced the stem cell doubling rate in what appears to be a frustrated attempt to reach the equilibrium fractions corresponding to the cancer stem cell niche. Moreover, if growth proceeded on soft agar, intraspecific interactions were always inhibitory, as we should expect from their competition for nutrients, but on hard agar the interactions between differentiated cells were strongly inhibitory while those between stem cells were collaborative. Experimental evidence also suggests that the hard substrate induces a large fraction of asymmetric stem cell divisions and the likelihood of plasticity processes, two features that appear to be absent in the case of the soft substrate. In the absence of stem cell growth factors, the barrier to differentiation is broken: although the stem cell number was conserved, overall growth was faster than in the other two cases. The interactions accelerate the effective growth rate of the differentiated cell fraction. Our interpretation of the results points to the centrality of the concept of stem cell niche and helps us to understand the relation between substrate stiffness and the dynamics of stem-cell fueled tumor growth.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lucas-Boettcher/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Lucas  Boettcher
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UCLA</em>
						    
						    
						    
					    		
					    			<h6>"Unifying continuous, discrete, and hybrid susceptible-infected-recovered processes on networks"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Waiting times between two consecutive infection and recovery events in spreading processes are often assumed to be exponentially distributed, which results in Markovian (i.e., memoryless) continuous spreading dynamics. However, this is not taking into account memory (correlation) effects and discrete interactions that have been identified as relevant in social, transportation, and disease dynamics. We introduce a framework to model continuous, discrete, and hybrid forms of (non-)Markovian susceptible-infected-recovered (SIR) stochastic processes on networks. The hybrid SIR processes that we study in this paper describe infections as discrete-time Markovian and recovery events as continuous-time non-Markovian processes, which mimic the distribution of cell cycles. Our results suggest that the effective-infection-rate description of epidemic processes fails to uniquely capture the behavior of such hybrid and also general non-Markovian disease dynamics. Providing a unifying description of general Markovian and non-Markovian disease outbreaks, we instead show that the mean transmissibility produces the same phase diagrams independent of the underlying inter-event-time distributions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lucy-Lansch-Justen/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Lucy  Lansch-Justen
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Instituto Gulbenkian de Ciência</em>
						    
						    
						    
					    		
					    			<h6>"Evolutionary rescue from mutational meltdown"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mutagenic drugs are promising candidates for the treatment of various RNA virus infections. By increasing the mutation rate of the virus they lead to rapid accumulation of deleterious mutation load, which is proposed to ultimately result in extinction as described by the theoretical concepts of mutational meltdown and lethal mutagenesis. However, the conditions and potential mechanisms of viral escape from the effects of mutagenic drugs have not been systematically explored. Here we investigate the population dynamics and genetics of a population under high mutation rates and discuss the probabilities of evolutionary rescue by means of three mechanisms: (1) “traditional” beneficial mutations increasing growth/fitness, (2) a mutation rate modifier (i.e., evolution of resistance), and (3) a modifier of the distribution of fitness effects, which either dampens or increases deleterious effects (i.e., evolution of tolerance). We investigate extinction times and we find that successful rescue mutations have to appear early to compensate the increasing mutational load. However, the observed stochasticity of rescue, especially by means of tolerance, highlights potential dangers of the use of mutagenic treatments that are almost impossible to capture in experimental trials.
Supervisor: Claudia Bank Co-author: Mark Schmitz
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Luke-Andrejek/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Luke Andrejek
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>The Ohio State University</em>
						    
						    
						    
					    		
					    			<h6>"A Robust Mathematical Model of Adaxial-Abaxial Patterning"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Biological development results from intricate and dynamic interactions between members of gene regulatory networks. This is exemplified by the production of flat leaf architecture. Leaves flatten by driving growth along the boundary between their adaxial (top) and abaxial (bottom) domains. These domains are generated by interactions between a complex network of transcription factors and small RNAs. Despite its complexity, flat leaf production is robust to genetic and environmental noise. To help us study this system, we mathematically modeled the determinants and interactions that pattern the adaxial-abaxial boundary. Our model recapitulates observations of adaxial-abaxial patterning and small RNA-target interactions. Positioning of the adaxial-abaxial boundary is highly robust to noise in the model. Furthermore, we identify degradation rates as possible factors contributing to robustness of adaxial-abaxial patterning.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Lutz-Brusch/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Lutz Brusch
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Technische Universität Dresden</em>
						    
						    
						    
					    		
					    			<h6>"Morpheus: A user-friendly simulation framework for multi-cellular systems biology"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Computational modeling and simulation become increasingly important to analyze tissue morphogenesis. A number of corresponding software tools have been developed but require scientists to encode their models in an imperative programming language. Morpheus, on the other hand, is an extensible open-source software framework that is entirely based on declarative modeling. It uses the domain-specific language MorpheusML to define multicellular models through a user-friendly GUI and has since proven applicable by a much broader community, including experimentalists and trainees. We here present how MorpheusML and the open-source framework allow for rapid model prototyping and advanced scientific work-flows. MorpheusML provides a bio-mathematical language in which symbolic identifiers in mathematical expressions describe the dynamics of and coupling between the various model components. It represents the spatial and mechanical aspects of interacting cells in terms of the cellular Potts model formalism and follows the software design rule of separation of model from implementation, enabling model sharing, versioning and archiving. A numerical simulation is then composed by parsing the MorpheusML model definition and automatic scheduling of predefined components in the simulator. Moreover, Morpheus supports simulations based on experimental data, e.g. segmented cell configurations, and offers a broad set of analysis tools to extract features right during simulation. A rich c++ API allows to extend MorpheusML and the simulator with user-tailored plugins. Finally, we apply Morpheus and image-based modeling to study the regulatory mechanisms underlying liver tissue architecture and flatworm regeneration.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Marc-Pereyra/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Marc Pereyra
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>FIAS</em>
						    
						    
						    
					    		
					    			<h6>"3D Particle image velocimetry (PIV) analysis of cellular migration during embryonic development"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Light-sheet fluorescence microscopy (LSFM) has been used to generate three dimensional datasets of the embryonic development of some organisms. This allows to study the three dimensional morphological changes and to identify embryonic structures throughout development. Visual evaluation of these datasets offers qualitative descriptions of embryonic events, but quantitative and rigorous analysis pipelines are required. This talks is about quantitatively characterizing cellular migration from such 3D datasets of the embryonic development of Tribolium Castaneum [1]. In order to analyse the volumes we implemented a 3D particle image velocimetry (PIV) analysis. This analysis is borrowed form the field of fluid mechanics, where small fluorescent or reflective particles are suspended in a fluid with turbulent flow, and the change in position of the moving particles is used as an indirect measure of direction and velocity of the flow. Overall, the availability of quantitative measurement from these datasets enables the elaboration and testing of mathematical models about embryonic events of interest.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Marilin-Nathalya-Guerrero-Laos/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Marilin Nathalya Guerrero Laos
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidad de Nariño</em>
						    
						    
						    
					    		
					    			<h6>"Approach and analysis of a mathematical model in ordinary differential equations, applied to HIV transmission, considering protection strategies."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Throughout the years, since 1981 approximately, the virus HIV marked its presence in the humanity razing with a large number of lives in which children, teenagers, and adults were involved. Although the rates of death caused by HIV have reduced notably because of antiretroviral drugs, the HIV continues spreading in the population.  Due to above, in this research HIV spreading is modelled, in special in the city of Pasto, Nariño in Colombia, where thanks to the review of data provided by the NDHI (Nariño departmental health institute)  was evident that in recent years, exactly between 2008 and 2018 the number of infected people increased. Then, this fact  lead the study of the dynamics of HIV transmission in Pasto, where is necessary a search for information of  strategies applied to control the infection, which give bases to propose a mathematical model which initially describes the dynamics of transmission of said epidemic and, finally, it will have a theoretical control incorporated based on sex education campaigns,  which pretends to maximize the number of protected people against HIV and clearly avoid the number of people with HIV increases.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Mark-Robertson-Tessi/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Mark  Robertson-Tessi
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Immune predation promotes aggressive metabolic phenotypes in a context-dependent manner"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Metabolism plays a complex but key role in the evolution of cancerous tumors. Localized hypoxia due to vascular dysfunction within the tumor microenvironment facilitates the metabolic response of the tissue (the Pasteur effect), causing acidification that leads to the evolutionary selection of acid-resistant tumor cell phenotypes. The subsequent emergence of a glycolytic phenotype in poor nutrient conditions leads to subsequent aggressive invasion. This evolutionary trajectory from normal to acid-resistant to glycolytic, is highly nonlinear and is modulated by vascular dynamics as well as the immune response. We present a multiscale hybrid-discrete-continuum cellular automata model that captures the phenotypic, vascular, microenvironmental, and spatial heterogeneity that shape acid-mediated invasion over biologically-realistic temporal scales. Specifically, we explore two major components in the interplay between tumor metabolism and immune function. First, T cells are subject to inactivation in acidic microenvironments. Second, competition for glucose inactivates the immune response through glucose starvation. These two processes are viewed as immune escape mechanisms that tumors may differentially employ in response to immune predation. A third mechanism considered is the expression of inhibitory immune checkpoint receptor (PD-L1). Model predictions indicate that fomenting a stronger immune response in a tumor leads to initial benefits with respect to additional cytotoxicity; however, this advantage is offset by the increased turnover of cells that leads to accelerated evolution and emergence of aggressive phenotypes. This creates a bimodal therapy landscape: either the immune system should be maximized for complete cure, or kept in check to avoid rapid evolution of invasive cells. The second option is akin to a natural adaptive therapy. These constraints are context-dependent and critically depend on the stability of intratumoral vascular dynamics and microenvironmental acidification.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Marta-Helena-Oliveira/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Marta Helena Oliveira
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Unesp</em>
						    
						    
						    
					    		
					    			<h6>"MATHEMATICAL MODELLING OF THE INFLAMMATORY PHASE OF SKIN WOUND HEALING IN RATS"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The skin wound healing is a complex process divided into three overlapping and interdependent phases (inflammatory, proliferative and remodelling). The inflammatory response must occur rapidly to avoid chronic inflammation and it depends on biochemical, molecular and cellular events. The effective crosstalk between leukocytes and cytokines (proinflammatory and anti-inflammatory) lead to correct  healing of the lesions. We considered a system of ordinary differential equations to model the inflammatory phase of skin wound healing process under treatments with oleoresin and hydroalcoholic cream extract from Copaifera langsdorffii. The model can exhibit two stable steady states corresponding to healthy or unhealthy skin, nevertheless this study has been concentrated in a parameter search to healthy state in order to verify the treatment efficiency comparing the results of the oleoresin against hydroalcoholic extract. Thus, we have analysed the roles among the main leukocytes (neutrophils and macrophages), present in the inflammatory phase, and the inflammatory cytokines: interleukin 6 (IL-6) and interleukin 10 (IL-10). The model solution reproduced the dynamics of the neutrophils and macrophages during inflammatory phase, however there was a lack between numeric and biological results suggesting the necessity to improve the model. One possible strategy to enhance this model is to consider the interaction between the pro-inflammatory cytokine and macrophages in the mathematical model.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Martin-Lopez-Garcia/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Martin Lopez-Garcia
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>School of Mathematics, University of Leeds</em>
						    
						    
						    
					    		
					    			<h6>"Exact approaches for the analysis of stochastic epidemic processes on small networks"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This research work is framed within the area of modelling hospital-acquired infections. I will introduce a number of existing compartmental-based approaches for modelling the spread of (typically antibiotic resistant) bacteria in hospital settings. Mathematical models with a relatively small number of compartments can be used for representing the spread of bacteria across patients and healthcare workers (HCWs), including relevant factors such as environmental contamination. However, more complex approaches (i.e., models with a large number of compartments, or network-based representations) are needed for example when introducing spatial considerations or HCW-patient contact network structures. When looking at network-based approaches, I will show some recent work on analysing exactly these epidemic dynamics on small networks. When considering an SIR epidemic process on a network, this analytic and computational approach amounts to the analysis of the exact 3^N-states continuous-time Markov chain (CTMC), and makes special focus on algorithmic aspects and the organisation of the space of states S=(S,I,R)^N. Finally, I will present some recent results on the applicability of graph-automorphism lumping techniques in these systems.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Marvin-Fritz/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Marvin  Fritz
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Technical University of Munich</em>
						    
						    
						    
					    		
					    			<h6>"On the modelling and analysis of tumor growth with phase-field equations of Cahn-Hilliard type"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this talk, we present a system of partial differential equations modelling the growth of tumor cells. We consider the effect of  phase separation into proliferative, hypoxic and necrotic phases. Further, we model the invasion due to ECM degradation and the influence of chemotaxis and haptotaxis. We establish the existence of weak solutions in appropriate function spaces via the Faedo-Galerkin method and illustrate the effects of the model on the tumor growth through numerical simulations with finite element approximations.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Masud-M-A/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Masud M A
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Pusan National University, Pusan, South Korea</em>
						    
						    
						    
					    		 
					    			<h6>"Title to be determined."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Dengue is one of the most prevalent vector-borne diseases with no medical treatment for cure. Sometimes, dengue infection develops hemorrhagic shock which is life-threatening and urges emergency medical support. At this stage, the infusion of intravenous fluid of an adequate amount is a must for the survival of the patient. However, unsystematic fluid infusion may lead to fluid overload and bring adverse outcomes. With an aim to quantify required amount of fluid infusion I extend minimal within-host dengue model to incorporate plasma dynamics as well as the intravenous fluid infusion. I experimented type I and type II functional response to model the impact of cytokines on plasma leakage, where Type II model showed better fit with published data. Optimal control theory has been used to establish the existence of a time-dependent optimal fluid infusion strategy. The forward-backward sweep method was used to solve the model numerically and deduce the optimal fluid infusion rate. The optimal strategy recommends fluid infusion initiation at a slower rate, which should be kept increasing for about 24 hours. Then the rate should be decreased gradually. The infusion requires about 4000 ml to 5000 ml within an interval of 2 to 3 days. Delay of a few hours in fluid support after initiation of leakage could be compensated. But a delay of more than one day could be life-threatening.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Mauro-Mobilia/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Mauro  Mobilia
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Population Dynamics in a Changing Environment: Random versus Periodic Switching"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Environmental changes greatly influence the evolution of populations. In this talk, we discuss the dynamics of a population of two strains, one growing slightly faster than the other, competing for resources in a time-varying binary environment modelled by a carrying capacity that switches either randomly or periodically between states of resources abundance and scarcity. The population dynamics is characterised by demographic noise (birth and death events) coupled to the fluctuating population size. By combining analytical and simulation methods, we elucidate the similarities and differences of evolving subject to stochastic and periodic switching. We show that the population size distribution is generally broader under intermediate and fast random switching than under periodic variations, with periodic changes leading to an abrupt transition from slow to fast switching regimes. The fixation probability under intermediate/fast random and periodic switching can hence vary significantly, with markedly different asymptotic behaviours. We also determine the conditions under which the fixation probability of the slow strain is maximum when the dynamics is driven by asymmetric switching. If time permits, I will outline how our methodology also allows us to analyse the complex eco-evolutionary dynamics arising when the slow strain produces public goods benefiting the entire population.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Michael-Pablo/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Michael Pablo
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of California, San Francisco | Gladstone Institutes</em>
						    
						    
						    
					    		
					    			<h6>"Early-phase decoupling between population mobility and death rates"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Reductions in human mobility have been a major strategy in controlling COVID-19 transmissions. However, analysis of publicly available data has revealed decreases in COVID-19 death rates that precede mobility changes. This suggests that, in some regions, there are mobility-independent factor(s) slowing COVID-19 deaths. Given the disproportionate impact that COVID-19 has had among nursing homes both in the US and in other countries, we hypothesized that this high-risk population might have dominated early changes in mortality rate. Simulations of a two-population SEIRD model, where one population is more vulnerable, reveal that early-phase decoupling may occur if susceptible individuals in the more vulnerable population are depleted before mobility changes can occur. More work is needed to determine whether mortality in nursing homes explains regional early-phase decoupling.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Michelle-Przedborski/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Michelle  Przedborski
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Waterloo</em>
						    
						    
						    
					    		
					    			<h6>"A combined systems biology and machine learning approach to study patient response to anti-PD1 immunotherapy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Anti-PD-1 immunotherapy has produced the highest response rate of any single-agent immunotherapy and has recently shown promise for the treatment of several aggressive cancers including melanoma, non-small-cell lung cancer, bladder, and head and neck cancers. However, there is high variability and unpredictability in the treatment outcome. While it remains an intensive area of research, this variability is thought to be driven by patient-specific biology, particularly, the interactions of the patient’s immune system with the tumor. Here I will introduce an integrative experimental and theoretical approach which was developed to study the patient-specific interactions between immune cells and tumor cells, to capture the variability in patient response to anti-PD-1 immunotherapy. This integrative approach utilizes clinical data from an ex vivo human tumor system that incorporates fragments from tumor biopsies in co-culture with patient-matched peripheral immune cells and plasma. The patient-derived cytokine expression levels and immune cell populations under control and Nivolumab treatment conditions were used to develop and calibrate a multi-scale systems biology model of the immune system which includes interactions of immune cells with the tumor cells and cytokine signaling. I will illustrate how the patient data was integrated into the model to capture the variability in patient response to treatment. Then I will show how the application of machine learning approaches to a simulated patient data set obtained from the calibrated model can be used to stratify features of response from non-response to anti-PD-1 immunotherapy. Next, I will discuss how transfer learning can be implemented using simulated clinical data with a subset of identified response features to significantly improve the response prediction accuracy on the ex vivo patient data. This approach has the potential to identify targeted experiments for patient screening as well as novel therapeutic targets that may sensitize otherwise non-responsive patients to anti-PD-1 immunotherapy. To illustrate this point further, I determine the optimal timing of triple combination therapy using IL-6 inhibition and recombinant IL-12 along with anti-PD1 immunotherapy to significantly improve patient response to treatment. Finally, I identify additional features of response beyond those encompassed in the systems biology model, which, while seemingly counter-intuitive, agree with recent clinical findings that may reshape our approach to cancer immunotherapy.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Mikahl-Banwarth-Kuhn/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Mikahl Banwarth-Kuhn
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UC Merced</em>
						    
						    
						    
					    		
					    			<h6>"Quantifying the Biophysical Impact of Budding Cell Division on the Spatial Organization of Growing Yeast Colonies"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Spatial patterns in microbial colonies are the consequence of cell-division dynamics coupled with cell-cell interactions on a physical media. Agent-based models (ABMs) are a powerful tool for understanding the emergence of large scale structure from  individual cell processes. In particular, the yeast, mph{Saccharomyces cerevisiae}, is a model eukaryote which commonly undergoes an asymmetric division process called budding. In this work, we develop and analyze an ABM to study the impact of budding cell division on yeast colony structure. We find that while large-scale properties of the colony (such as shape and size) are preserved, local spatial organization of the colony, with respect to mother-daughter relationships, subcolonies and their connectivities, are greatly impacted. This difference in spatial organization, coupled with differential growth rates from nutrient limitation, create distinct sectoring patterns in the subcolony structure, which offers novel insights into mechanisms driving experimentally observed sectored yeast colony phenotypes. Moreover, our work illustrates the need to include relevant biophysical mechanisms when using ABMs to compare to experimental studies.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Milad-Ghomlaghi/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Milad  Ghomlaghi
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Monash University</em>
						    
						    
						    
					    		
					    			<h6>"Akt regulates PIP3 production by PI3K to form a potent negative feedback loop"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The phosphoinositide 3-kinase (PI3K)-Akt pathway is a central component of signalling networks and is dysregulated in numerous pathologies. As such, its activity is under the tight control of several feedback signals, which work to control signal flow and ensure signal fidelity. A rapid overshoot in the insulin-stimulated recruitment of Akt to the plasma membrane has previously been reported, which is indicative of negative feedback operating on acute timescales. Here, using computational modelling and cell biology we show that described mTORC1/S6K-dependent feedback mechanisms do not account for this behaviour. However, our system-based analysis suggests that another negative feedback must exist within the network to explain the overshoot in the recruitment of Akt to the plasma membrane. To identify this negative feedback, six different mathematical models are constructed that represent different possible negative feedback scenarios. Interrogating these models based on their quality of fitness to the experimental data allows us to reject unlikely candidate feedback mechanisms and guide experiment towards the most likely feedback model. Integrating model simulation and biological validation using live cell imaging and biochemical assays methods, we demonstrate existence of a negative feedback from Akt to PIP3, which limits plasma membrane associated PI3K and phosphatidylinositol (3,4,5)-trisphosphate (PIP3) synthesis. This feedback is both rapid and powerful - suppression of the feedback using Akt inhibitors increased PIP3 abundance by ~5-fold within 10 min of insulin stimulation. This had profound effects on the localisation of PIP3-binding proteins such as PDK1 and GAB2, as well as the activation of MAPK signalling. As a feature of multiple cell types and growth factors, this novel Akt-dependent feedback loop plays a vital role in regulating PIP3 abundance and thus has important implications for therapies targeting Akt.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Miller-Orlando-Cer%C3%B3n/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Miller Orlando Cerón
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidad de Nariño</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical model with carriers and general non-linear incidence rate"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We analyzed a mathematical model with asymptomatic and a general incidence. We show that there are two equilibrium points and by means of conditions imposed on the functions involved in the non-linear incidence we show the global stability of the equilibrium points by Lyapunov direct method.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Min-Song/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Min  Song
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Southern California</em>
						    
						    
						    
					    		
					    			<h6>"Model Predicts Distinct Mechanisms of Endothelial Cell Growth Upon the Stimulation of FGF and VEGF"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Angiogenesis is the formation of new blood capillaries from pre-existing ones. The essential role of blood vessels in delivering nutrients makes angiogenesis important in the survival of tissues, such as wound healing process and tumor growth. Thus, targeting angiogenesis is a prominent strategy in both tissue engineering and cancer treatment. However, not all approaches to target angiogenesis lead to successful outcomes. Current therapies primarily target pro-angiogenic factors such as vascular endothelial growth factor (VEGF) and fibroblast growth factor (FGF) in isolation. However, there is a limited understanding of how these promoters combine together to stimulate angiogenesis. We aim to quantitatively characterize the crosstalk between VEGF- and FGF-mediated angiogenic signaling in endothelial cells and the effects of the interactions on a cellular level, specifically endothelial cell growth, in order to identify novel therapeutic strategies. We constructed a hybrid agent-based mathematical model that characterizes endothelial cell growth driven by FGF and VEGF-mediated signaling. The molecular interactions were implemented with our published ordinary differential equation model that focuses on FGF- and VEGF-induced mitogen-activated protein kinase (MAPK) signaling and the phosphatidylinositol 3-kinase/protein kinase B (PI3K/Akt) pathway which promote cell survival and proliferation. To link the molecular signals with the cellular responses, we assumed that the endothelial cell growth is dependent on the maximum pAkt and pERK levels upon the stimulation of FGF and VEGF within two hours, following Hill functions. We used the total number of endothelial cells as an indicator of cell growth. Cell heterogeneity within a cell population is also considered in the model. The parameters that significantly influence cell growth rate were identified using a global sensitivity analysis and estimated by fitting the model to experimental data using particle swarm optimization. The model was validated against independent experimental data. The trained and validated model predicts the optimal concentrations for mono- and co-stimulation of FGF and VEGF needed to maximize endothelial cell growth. Also, FGF and VEGF show different mechanisms in promoting the overall cell growth rate. Additionally, combinations of FGF and VEGF do not exhibit an obvious greater effect in promoting cell growth compared to FGF stimulation alone. Moreover, our model identifies the influential species and kinetic parameters that specifically modulate the cell growth, which represent potential targets for modulating angiogenesis signaling. The model provides mechanistic insight into VEGF and FGF interactions in angiogenesis and predicts the combination effects of FGF and VEGF co-stimulation. More broadly, this model can be utilized to identify targets that influence angiogenic signaling leading to cell growth and to study the effects of pro- and anti-angiogenic therapies.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Moacyr-A-H-B-Silva/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Moacyr A H B Silva
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>EMAp/FGV</em>
						    
						    
						    
					    		
					    			<h6>"A death-based mathematical model of the ongoing COVID-19 pandemic"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			ASICRD model —Susceptible, Infectious, Critically infectious, Removed and Dead was employed to model COVID-19. While the model is certainly very simple, it captures important early dynamics of diseases such as Covid-19. The model is calibrated for the deaths in the linear phase, which turns out to be a 2x2 linear system. The calibration can be done without any knowledge of the model parameters and allows for a universal fitting over the linear phase.As expected the fitted linear model has the first eigenvalue positive, while the second eigenvalue λ 2 is negative, but otherwise free. It turns out, however, that λ 2 ∈ B ⊂ (−∞,0), where B is a compact interval. For each choice of λ 2 we have a distinct value of0. For this model0(λ2) is monotonic in B. This allows for a bracketing of the possible values of 0. For all choices of the remaining free parameters, the linear dynamics is indistinguishable — though the nonlinear dynamics will be different. Another interesting point is that, once the fitting is done, the non-linear dynamics can be described by a one parameter family of sub-models. This allows for a convenient description of the epidemic evolution scenarios. (This is a joint work by Moacyr Silva, Helio Schechtman and Max O Souza)
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Mohammad-U-Zahid/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Mohammad U Zahid
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Forecasting Individual Patient Response to Radiotherapy with a Dynamic Carrying Capacity Model in Head and Neck Cancer"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Nearly 66% of all cancer patients receive radiotherapy (RT).  Currently, RT scheduling does not take into consideration tumor volume dynamics.  If response to an RT schedule can be predicted accurately, then there is a potential for treatment adjustment.  The objectives of this study are to model tumor volume dynamics in response to RT and to evaluate the patient-specific predictive power of the model for patient outcomes. Tumor volume data were collected for 2 independent cohorts of head and neck cancer patients from Moffitt Cancer Center (MCC) and M.D. Anderson Cancer Center (MDACC) that received 66-70 Gy RT in 2 Gy daily fractions.  Tumor volume measurements were derived from CT scans: 2 before RT and weekly scans during RT.  Tumor growth was described with a logistic growth model with intrinsic growth rate, λ, and tumor carrying capacity, K. The effect of RT was modeled as an instantaneous reduction in carrying capacity with fraction δ. To predict response to RT for individual patients, we combined the distribution of MCC-learned δ values and weekly measurements of volume reduction in the untrained MDACC cohort to estimate δ to predict volume reduction and patient outcomes. The model fit data from MCC with patient-specific values for λ and δ with high accuracy (R2 = 0.95).  Model analysis revealed that growth rate λ is not patient specific. A uniform λ reduces R2 to 0.92 while reducing the number of free parameters in the model (K and δ being patient specific).  This MCC-trained model was then cross-validated on the independent cohort from MDACC (R2 = 0.98), demonstrating transferability of λ.  The trained model predicts patient-specific RT responses with >70% accuracy for loco-regional control and disease-free survival without considering any patient-specific observations, and inclusion of on-treatment observations further increases prediction accuracy.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Mohsin-Saleet-Jafri/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Mohsin Saleet Jafri
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>George Mason University</em>
						    
						    
						    
					    		
					    			<h6>"Predicting Drug Resistance by Applying Machine Learning to Molecular Simulation"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Distinct missense mutations in a specific gene have been associated with different diseases as well as differing severity of a disease. Current computational methods predict potential pathogenicity of a missense variant but fail to differentiate between separate disease or severity phenotypes. We have developed a method to overcome this limitation by applying machine learning to features extracted from molecular dynamics simulations, creating a way to predict the effect of novel genetic variants in causing a disease, drug resistance, or another specific trait. As an example, we have applied this novel approach to variants in calmodulin associated with two distinct arrhythmias, as well as two different neurodegenerative diseases caused by variants in amyloid beta peptide. The new method successfully predicts the specific disease caused by a gene variant and ranks its severity with more accuracy than existing methods.  We have applied this methods to predicting cancer drug resistance for specific variants.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Mubasher-Rashid-Rather/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Mubasher Rashid Rather
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Central University of Rajasthan</em>
						    
						    
						    
					    		
					    			<h6>"Organization of biogeochemical nitrogen pathways with switch-like adjustment in fluctuating soil redox conditions"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Nitrogen is cycled throughout ecosystems by a suite of biogeochemical processes. The high complexity of the nitrogen cycle resides in an intricate interplay between reversible biochemical pathways alternatively and specifically activated in response to diverse environmental cues. Despite aggressive research, how the fundamental nitrogen biochemical processes are assembled and maintained in fluctuating soil redox conditions remains elusive. Here, we address this question using a kinetic modelling approach coupled with dynamical systems theory and microbial genomics. We show that alternative biochemical pathways play a key role in keeping nitrogen conversion and conservation properties invariant in fluctuating environments. Our results indicate that the biochemical network holds inherent adaptive capacity to stabilize ammonium and nitrate availability, and that the bistability in the formation of ammonium is linked to the transient upregulation of the amo-hao mediated nitrification pathway. The bistability is maintained by a pair of complementary subsystems acting as either source or sink type systems in response to soil redox fluctuations. It is further shown how elevated anthropogenic pressure has the potential to break down the stability of the system, altering substantially ammonium and nitrate availability in the soil, with dramatic effects on biodiversity.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Muhammad-Said/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Muhammad  Said
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Sensitivity and stability analysis of an Ebola Virus disease and GB virus C co-infection."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this work, we propose a nonlinear mathematical model to study the transmission dynamics of the Ebola Virus Disease(EVD) and the Hepatitis G virus (GBV) co-infection. The basic reproductive number is found by the next-generation matrix method. Then the infectious free and endemic equilibrium of the system is computed. The local and global stability of the system is presented as well. For local asymptotical stability, linearization, and Routh-Hurwitz criterion and show that if R_0<1, then the system is locally asymptotically stable otherwise unstable. The global asymptotical stability is found out by the Lyapunov function method. Finally, we present a numerical simulation of the proposed model.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Muhammad-Humayun-Kabir/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Muhammad Humayun  Kabir
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Jahangirnagar University</em>
						    
						    
						    
					    		
					    			<h6>"Model-aided Understanding of the Outbreak of COVID-19 in Bangladesh"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this talk, we focus on a seven compartmental model to understand the infectious dynamics of COVID-19 pandemic. We show the boundedness and non-negativity of solutions of the model. We analytically calculate the basic reproduction number of the model and perform the stability analysis at all equilibrium points to understand the epidemic and endemic cases based on the results of the basic reproduction number. Our results reveal that regional lockdown and social awareness (e.g., wearing a face mask, washing hands, social distancing) can reduce the pandemic of the current outbreak of novel coronavirus in a most densely populated country like Bangladesh.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Nara-Yoon/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Nara  Yoon
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Adelphi University</em>
						    
						    
						    
					    		
					    			<h6>"Modeling collaterally sensitive drug cycles: shaping heterogeneity to allow adaptive therapy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Despite major strides in the treatment of cancer, the development of drug resistance remains a major hurdle. One strategy which has been proposed to address this is the sequential application of drug therapies where resistance to one drug induces sensitivity to another drug, a concept called collateral sensitivity. Particularly, there is utility in a drug sequence which completes a cycle of such relationships. With such cycles, one could, in theory, generate infinitely long drug sequences which can be used in long term therapy to mitigate the evolution of resistance in a tumor. In this work, we explored the optimal therapeutic strategy using the drugs involved in such a cycle with an arbitrary length, N (>=2). We developed a mathematical model for this research, in which tumor cells are classified as one of N subpopulations represented as { R_i|i =1,2,...,N}. Each subpopulation, R_i , is resistant to Drug i and each subpopulation, R _{ i -1} (or R_N , if i =1), is sensitive to it, so that R_i increases under Drug i as it is resistant to it, and after drug-switching, decreases under Drug i+1 as it is sensitive to that drug(s). Based on the model, we found that there is an initial period of time in which the tumor is `shaped' into a specific makeup of each subpopulation, at which time all the drugs are equally effective ( R* ). After this shaping period, all the drugs are quickly switched with duration relative to their efficacy in order to maintain each subpopulation, consistent with the ideas underlying adaptive therapy. Additionally, we have developed methodologies to administer the optimal regimen under clinical or experimental situations in which no drug parameters and limited information of trackable populations data (all the subpopulations or only total population) are known. The therapy simulation based on these methodologies showed consistency with the theoretical effect of optimal therapy.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Nicole-Althermeler/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Nicole Althermeler
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Bielefeld University</em>
						    
						    
						    
					    		
					    			<h6>"Ancestral lines under selection for multiple sites: Pruning the Ancestral Selection Graph"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Understanding ancestral processes under selection and mutation is among the fundamental challenges in population genetics. An important concept here is the ancestral selection graph (ASG) extracted from the Moran Model. This is, however, difficult to handle, both analytically and computationally, if two or more sites (that is, gene loci or sequence sites) are considered. Here a computational approach to pruning the ASG in the case of M sites is presented. The basic idea explores any incoming selective arrow, putting a hold onto the previously-active line. Should the incoming line be determined to be the ancestral line, it becomes the new active line and the previously-active line is dismissed. The set of types (that is, haplotypes or alleles) can be modelled in terms of sequences s ∈ {0, 1}M or fitness classes w ∈ {0,..,M}. The pruning procedure so far takes advantage of using sequences, but may be adjusted to fitness. Some preliminary results for different mutation and selection parameters and the resulting mutation process on the ancestral line are presented. This is joint work with Ellen Baake.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Nikolaos-Dimitriou/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Nikolaos  Dimitriou
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Department of Bioengineering, McGill University</em>
						    
						    
						    
					    		
					    			<h6>"Modelling geometric patterns of cancer progression"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Throughout the years the morphological characteristics of malignant tumours have played a major role to perform cancer staging, which in turn determines the selection of therapy [1]. Studies that quantify the geometry of tumours have shown that tumours progress towards less smooth boundaries with strands of cells invading in surrounding tissues [2,3,4], thus, resulting in poor therapeutic outcomes [5,6]. Mathematical models can provide useful insights towards the understanding the morphological progression of cancer as well as improve their therapeutic outcomes. In this context, a computational framework that focuses on the modelling of complex geometric patterns is presented. The framework utilizes hybrid spatiotemporal models that describe cancer growth in terms of both tumour and cellular levels. Model validation is performed with 3D cell culture experiments of triple negative breast cancer cells (MDA-MB-231) grown in Matrigel. The model is calibrated to the experimental data with the use of combined approximate bayesian computation and monte carlo techniques (ABC-MCMC). Spatial statistical analysis methods are then utilized towards the identification of geometric patterns across tumour volumes, formed in both experiments and simulations. Results so far indicate cell organization into clusters that progressively tend to accumulate in the boundaries of the examined space. The resulted collective migration pattern suggests cell-cell cooperativity and combined with increased mobility leads to the escape from the examined space.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Noemi-Andor/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Noemi  Andor
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Invasion of homogeneous and polyploid populations in nutrient-limiting environments"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Breast cancer progresses in a multistep process from primary tumor growth and stroma invasion to metastasis. Progression is accompanied by a switch to an invasive cell phenotype. Nutrient-limiting environments promote chemotaxis with aggressive morphologies characteristic of invasion. It is unknown how co-existing cells differ in their response to nutrient limitations and how this impacts invasion of the metapopulation as a whole. We integrate mathematical modeling with microenvironmental perturbation-data to investigate invasion in nutrient-limiting environments inhabited by one or two cancer cell subpopulations. Hereby, subpopulations are defined by their energy efficiency and chemotactic ability. We estimate the invasion-distance traveled by a homogeneous population. For heterogeneous populations, our results suggest that an imbalance between nutrient efficacy and chemotactic superiority accelerates invasion. Such imbalance will spatially segregate the two populations and only one type will dominate at the invasion front. Only if these two phenotypes are balanced do the two subpopulations compete for the same space, which decelerates invasion. We investigate ploidy as a candidate biomarker of this phenotypic heterogeneity to discern circumstances when inhibiting chemotaxis amplifies internal competition and decelerates tumor progression, from circumstances that render clinical consequences of chemotactic inhibition unfavorable.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Oke-I-Segun/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Oke I Segun
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Pretoria, South Africa</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical model for the estrogen paradox in breast cancer treatment"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Background: Breast cancer is one of the major causes of mortality in women world- wide. Estrogens are known to stimulate the growth of breast cancer but are also effective in treating the disease. This is referred to as the “estrogen paradox”. Several studies have been dedicated to describe the possible mechanisms behind this paradox. Other studies highlighted the correlations between the tumor suppressor protein p53 and the estrogen receptor alpha (ERα). Aim: We investigate possible trade-offs between the tumor suppressor protein p53 and the estrogen receptor alpha (ERα) that can lead to breast cancer elimination. Methods: We propose a novel ODE-based mathematical model describing the interac- tions between both dormant and active cancer cells, estrogen hormone, a tumor suppressor protein (p53), and a treatment combination with high-dose of estrogens (HDEs) and p53. We calculate the model’s equilibrium points and determine their global stability behavior by means of a comparison theorem. Findings: We find a range for the ratio of estrogen to p53 outside with active cancer cells can be eliminated without any treatment. Inside this range, we show that active cancer cells will grow to their maximum size, and that treatment with high-dose of estrogens can achieve cancer elimination. We carry out numerical simulation to confirm our mathemat- ical finding and investigate the scenario of low, moderate, and high ratio of estrogen to p53.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pamela-Kim-N-Salonga/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Pamela Kim N Salonga
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of the Philippines Diliman</em>
						    
						    
						    
					    		
					    			<h6>"A mathematical model of the dynamics of lymphatic filariasis in Caraga Region, The Philippines"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Despite being one of the first countries to implement mass drug administration (MDA) for elimination of lymphatic filariasis (LF) in 2001 after a pilot study in 2000, the Philippines is yet to eliminate the disease as a public health problem with 6 out of the 46 endemic provinces still implementing MDA for LF as of 2018. In this work, we propose a mathematical model of the transmission dynamics of LF and its elimination using MDA in the Philippines. Using the computed basic reproduction number R0, we show that the disease-free equilibrium E0 of the model system is locally asymptotically stable when R0 < 1 and unstable when R0 > 1, whereas the endemic equilibrium E* is locally asymptotically stable when R0 > 1. Sensitivity analysis using the Latin Hypercube Sampling and Partial Rank Correlation Coefficient method suggests that the infected human population is most sensitive to the treatment parameters. Using the available LF data in Caraga Region from the Philippine Department of Health (DOH), we estimate the treatment rates r1, r2 using the least squares parameter estimation technique. Finally, we apply optimal control theory with the objective of minimizing the infected human population and the corresponding implementation cost of MDA, using the treatment coverage γ as the control parameter. Simulation results highlight the importance of maintaining a high MDA coverage per year to effectively minimize the infected population by the year 2030. This work is envisioned to be protocol-directing and policy-making. As there are still several endemic areas in the Philippines and other tropical countries in the Southeast Asia and Western Pacific regions, this study could help the DOH and other ministries of health in designing more effective implementation approaches for MDA to achieve LF elimination in the near future.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pappu-Kumar/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Pappu Kumar
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Hotilal Ramnath College, Amnour(Jai Prakash University, Chapra)</em>
						    
						    
						    
					    		
					    			<h6>"Theoretical investigation of non-equilibrium bio-heat transfer during thermal therapy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This study theoretically investigates the non-equilibrium heat transfer within living biological tissues during different thermal therapy applications. Numerical solution of the present problem has been done by Chebyshev wavelet Galerkin method. The use of Chebyshev wavelet is found to be accurate, simple and fast. Larger differences in the temperature prediction at the treatment position have been observed using different equilibrium and non-equilibrium based bioheat models. It is observed that the porosity and the convective heat transfer are the factors that contribute most to the non-equilibrium heat transfer within living biological tissues. The whole analysis is presented in dimensionless form.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Patrick-Gelbach/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Patrick  Gelbach
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Southern California Dept of Biomedical Engineering</em>
						    
						    
						    
					    		
					    			<h6>"Metabolomics and Mechanistic Kinetic Modeling Reveal Mechanisms Driving Intracellular Metabolism and Insulin Secretion in Pancreatic Beta Cells"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Pancreatic beta cells maintain blood glucose levels within a healthy range by producing insulin. Insulin production is heavily dependent on the intracellular metabolic reactions carried out by the cell, and diseases, such as Type 2 Diabetes, occur when that metabolism functions poorly. In order to better treat diabetes, we must understand glucose-stimulated insulin secretion in beta cells. Systems-focused computational modeling of metabolic processes can provide quantitative insight into the mechanisms driving insulin production in varied extracellular conditions, informing future research into novel treatments for diabetes.
We developed a kinetic, ordinary differential equation model of PBC intracellular metabolism. The model includes glycolysis, glutaminolysis, the TCA cycle, the pentose phosphate pathway, and the aldose-reductase pathway. We linked metabolism to insulin production using partial least-squares regression. We performed a global sensitivity analysis to determine the kinetic parameters that significantly influence predicted metabolite levels. We trained the model by fitting its reaction velocities (Vmax parameters) to mass spectrometry-based metabolomics measurements of metabolite levels following 5- and 30-minute stimulation of the INS-1E cell line with varied concentrations of glucose. We applied the kinetic model to simulate clinically-relevant metabolic perturbations.
The sensitivity analyses identified influential metabolic reactions. At both time points, Vmax values for the glucose transporter (GLUT2) reaction and the Glucokinase reaction were impactful on predicted metabolite levels. The results make sense, given both the primary role of beta cells (to import glucose into the cell to produce insulin) and published results showing GK to be a key regulator in overall PBC activity. At only the 30-minute time point, the Lactate Dehydrogenase, Monocarboxylate Cotransporter (MCT), and Aldose Reductase reaction velocities were found to be significantly impactful. The results suggest possible mechanisms by which extended treatment with glucose causes cells to adjust their metabolism to avoid glucotoxicity. At the 5-minute time point, the Triose-phosphate isomerase reaction velocity was influential, which may support the experimentally-seen rapid equilibration of glyceraldehyde-3-phosphate and dihydroxyacetone phosphate.  Using the fitted model, we simulated 95% knockdown and upregulation of the GLUT2, MCT, pyruvate-hydrogen shuttle, and glucose-6-phosphate dehydrogenase reactions, to understand the effect of adjusted glucose import, lactate excretion, TCA Cycle flux, and PPP flux, respectively. Though the network as a whole is robust to many changes, the model predicts that controlling flux into the TCA Cycle had substantial effects at the 5-minute time point, suggesting that targeting TCA Cycle reactions may improve insulin production.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pau-Capera-Aragones/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Pau Capera Aragones
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universität Bayreuth</em>
						    
						    
						    
					    		
					    			<h6>"A mechanistic partial integro-differential equations model for bumblebee foraging behaviour"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Bumblebees provide valuable pollination services to crops around the world. Empirical evidence has suggested that the addition of wildflower adjacent to cultivated crops could increase its pollination services. However, a quantification of the location, quantity and type of the wildflowers needed to optimize the pollination services is unknown and a call for modellers has been made.Here we develop a partial integro-differential equation model to predict the spatial distribution of foraging bumblebees in dynamic heterogeneous landscapes. The foraging population is divided into two subpopulations engaged in intensive search mode (modelled by diffusion) and extensive search mode (modelled by advection) respectively. Our model considers the effects of resource-dependent transition rates between movement modes, resource depletion, central-place foraging behaviour and the effects of memory in the spatial distribution of foraging bees.
We use our model to quantify the benefits that planting wildflowers adjacent to a crop can have on its pollination services and show that small plantations in specific locations can lead to an increase of crop's pollination services.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Paul-A-Roberts/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Paul A Roberts
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Sussex</em>
						    
						    
						    
					    		
					    			<h6>"Using mathematics to investigate the mechanisms behind vision loss"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			(poster is withdrawn -- please see the NEUR Thursday 9:30am session for Paul Roberts' talk.)
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Paul-D-Alexander/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Paul D Alexander
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Treatment of Viral Co-Infections"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Previous reports show that it is not uncommon for patients to have two viruses at the same time. At the current time, we do not know how to treat co-infections. In order to test the effects of having these concurrent infections, we simulate the two infections using a mathematical model. We use our model to simulate influenza A virus (IAV) coinfected with respiratory syncytial virus (RSV) and parainfluenza virus (PIV) coinfected with human rhinovirus (hRV). Using the model, we can estimate the co-duration of the viruses, the individual duration, and the peak virus amount for both viruses, both with and without drug treatment of the infections to figure out the best treatment strategies for co-infections. We find that sometimes treating one infection can lead to the lengthening of the other infection.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Paul-J-Hurtado/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Paul J Hurtado
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Nevada, Reno</em>
						    
						    
						    
					    		
					    			<h6>"Reproduction Numbers for ODE Models of Arbitrary Finite Dimension: An Application of the Generalized Linear Chain Trick"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The Generalized Linear Chain Trick (GLCT) is a conceptually and practically useful approach for deriving mean field ODE models, since it describes how the structure of mean-field ODE models (and quantities like the basic reproduction number) reflect the assumptions of an often unspecified underlying continuous-time, stochastic state-transition model. In this talk, I will first describe how to generalize an existing ODE model -- such as the SEIR model or Rosenzweig-MacArthur consumer-resource model -- using the GLCT to incorporate non-exponential dwell times (e.g., latent periods in SEIR models, or predator maturation times in consumer-resource models) that are Erlang distributed or, more generally, are phase-type distributed. The phase-type family of distributions are the absorption time distributions for continuous time Markov chains, and include exponential, Erlang, generalized Erlang, and Coxian distributions. Second, I will show how the structure of the resulting ODE model, which is of arbitrary finite dimension, can be exploited to obtain a general expression for the (basic) reproduction number. These results illustrate the utility of the GLCT, not just for model derivation, but also for model analysis and interpretation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pawe%C5%82-Klimasara/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Paweł  Klimasara
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Information Technology and Management in Rzeszów, University of Silesia in Katowice</em>
						    
						    
						    
					    		
					    			<h6>"A Savanna Dynamics Model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Savannas are mixed woodland-grassland ecosystems that cover fifth of Earth’s land surface. They are characterised by a continuous grass layer and a discontinuous layer of woody plants. In understanding complex savanna dynamics of main interest is the question how do trees and grasses co-exist without one dominating the other? Amongst possible explanations there has been recognized that disturbances like fires or browsing and grazing may play fundamental role. There is quite rich literature on models of tree-grass coexistence in savannas with such non-equilibrium approach. From the mathematical point of view models containing many different factors often lack formal stochasticity and their analysis is usually based on numerical simulations. Moreover, another important determinant of savanna structure - water availability - is of less unpredictable nature following seasonality. Most of the plants growth happens during wet seasons while large amounts of dry grasses additionally support fires during dry seasons. These effects usually are not directly included in existing models of savanna dynamics. We introduced two minimalistic models of tree-grass coexistence driven by fire disturbances. We provided careful mathematical analysis of appropriate piecewise deterministic Markov process and showed the existence of a unique stationary distribution of tree and grass biomasses using the tools of stochastic semigroup theory. Continuing this approach we work on analytic results for more realistic setting including the seasonality via its effects on plants growth rate as well as a probability of occurrence and intensity of fires. Moreover, we want to include in the final model also grazing and browsing effects (similarly to the one provided in (6) but along with seasonality).
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pedro-Vilanova/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Pedro Vilanova
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>NJIT</em>
						    
						    
						    
					    		
					    			<h6>"Synchronization in Stochastic Oscillators Subject to Common Extrinsic Noise"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this work we study the level of synchronization in stochastic biochemical reaction networks that support stable mean-field limit cycles and are subject to common external switching noise. Synchronization in stochastic limit cycle oscillators due to common noise is usually demonstrated by applying Ito's Lemma to the logarithm of the phase difference. However, this argument cannot be straightforwardly extended to our case because of its discrete state space. Assuming the intrinsic and extrinsic noise operate at different time-scales, we prove that the average level of synchronization is of order of the rate of the intrinsic noise (inversely proportional to the system volume) times the square of the switching rate of the external noise. Moreover, we show in numerical experiments the approximate asymptotic value of the synchronization level by applying this result to classical oscillators found in the literature. Joint work with James MacLaurin.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Perminov-Dmitrievich-Valeriy/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Perminov Dmitrievich Valeriy
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>New bandage's materials, Ltd.</em>
						    
						    
						    
					    		
					    			<h6>"Agent-based models for influenza epidemic dynamics and its decision-making capability"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Agents-based models (ABM) become more and more popular in applied mathematics. During last 15 years a large number of ABM have been created and used in different scientific area (ecology, economy, epidemiology, human behavior to name a few), but in this paper, only ABM for influenza epidemic/pandemic dynamics in cities are considered in detail. Based on a critical review of currently accepted ABM of such special type new ABM has been proposed. Unlike the old ABM, it can be used for analysis of efficiency and cost of all interventions (how for ones had been carried out before and during epidemic or pandemic under consideration and ones that could be implemented but had not been carried out for some reasons). Moreover, under some conditions, new ABM gives us an opportunity to analyze efficiency and cost of different interventions for future oncoming epidemics (first of all pandemics) and to select its optimal combination.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Phebe-M.-A-Havor/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Phebe M. A Havor
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Kwame Nkrumah University of Science and Technology</em>
						    
						    
						    
					    		
					    			<h6>"Dynamics of disease models with self-diffusion: Case study of Cholera in Ghana"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Modeling with reaction-diffusion systems involves constituents locally transformed into each other by chemical reactions and transported in space by diffusion. With this in mind, the attention to mathematical and disease epidemiology has increased, as disease epidemics have become a predominant worldwide health issue. The case of Vibrio Cholerae (blue-death) is no different especially in a country like Ghana. Factors that affect the transmission of such a disease includes mainly both human and environmental factors. Proposing a Reaction-Diffusion SIR-B mathematical model for Cholera with proliferate stability analysis on the epidemic and endemic equilibrium, that incorporates an environmental reservoir is formulated to capture the movement of human hosts and host organisms in a heterogeneous environment. Findings here are supported by the results of numerical simulations and based on these results, an evolutionary process that involves organism distribution and their interaction of spatially distributed population with local diffusion is presented. Results show that the model dynamics exhibit a diffusion-controlled formation of patterns which attribute to diffusion having a great influence on the spread of the disease.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Phillip-Rossbach/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Phillip Rossbach
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Unknown</em>
						    
						    
						    
					    		
					    			<h6>"Determination of the critical adhesion parameter for the sorting behavior of a cell system with several cell types using statistical learning methods"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The process of cell sorting plays an essential role in development and maintenance of tissues. To understand the basic mechanisms of this process, mathematical modeling can assist cell biological research by providing means to analyze the consequences of different hypotheses on the underlying mechanisms. Three basic theoretical descriptions of cell segregation already exist: the Differential Surface Contraction Hypothesis of Harris (1976), the Differential Interface Tension Hypothesis of Brodland and Chen (2000) and the Differential Adhesion Hypothesis (DAH) of Steinberg. In DAH it is assumed that cell sorting is determined by quantitative differences in cell type speciffc intercellular adhesion strengths. An implementation of the DAH is the cell based Differential Migration Model (DMM) by Voss-Bohme and Deutsch. This DMM is based on modulated migration properties of cells with respect to their intercellular adhesion strengths and allows to study analytically the factors that determine pattern formation during cell sorting. In particular, a critical adhesion parameter for systems with two cell types can be derived analytically which predicts the sorting pattern of the two cell types as a function of the intercellular adhesion strengths. Here, we investigate numerically the existence of a critical parameter which determines the sorting behavior for more complex systems with more than two cell types. We rely on in-silico time-series data that is produced by a probabilistic cellular automaton which emulates the DMM and classify the segregation behavior using statistical learning methods such as Support Vector Machines and Logistic Regression Models. The well-understood case of two cell-types is used as benchmark-problem to evaluate our tools. The order parameter and statistical learning tools developed in this context provide a methodic approach applicable to the analysis of spatio-temporal in-vitro data, as well.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/PIJUSH-PANDAY/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Pijush Panday
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Statistical Institute</em>
						    
						    
						    
					    		
					    			<h6>"Dynamical behaviour of a stage-structured predator-prey model by incorporating cost and benefits of group defense"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In predator-prey theory, the predator can affect on prey population by the killing of the prey and by causing predation fear on the prey population. The prey population also adjusts some behavioral approaches to reduce their predation risk which may influence their long term survival. In the present study, we formulate a predator-prey model dividing the prey population into two stages: juvenile and adult. We assume that when adult preys are sensitive to predation, they adapt group defense as an anti-predator strategy to lower their predation risk. To include group defense in the adult prey population, we consider Holling type IV functional responses for adult prey and predator interaction. But group defense has a negative effect by decreasing their reproduction potential. A parameter predator-taxis sensitivity introduces to interlink benefits of group defense and its costs. Increasing predator-taxis sensitivity also increases the group defense level of adult preys and benefits them by lowering predation risk. But also causes a detrimental effect by decreasing their reproduction rate simultaneously. We study some mathematical properties such as positivity, boundedness, local stability of equilibrium points, and bifurcation behaviors of the model. Our result suggests that the maturation rate can destabilize the system by producing oscillatory coexistence. For higher maturation rate the predator population suddenly extinct from the system, where oscillatory coexistence may disappear and the system becomes stable around the predator-free state. We also observe that predator-taxis sensitivity enhances the destabilizing nature of the system. However, for increasing the level of fear, the destabilization vanishes and the system shows stable behavior. It is also observed that predator- taxis sensitivity can be beneficial for adult prey as their density may increase with increasing the values of predator-taxis sensitivity. We also notice that above a threshold value of predator- taxis sensitivity the system shows bistable behavior. Our fear-induced stage-structured model exhibits interesting and rich dynamical behaviors.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pooja-Dnyane/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Pooja Dnyane
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>CSIR-National Chemical Laboratory</em>
						    
						    
						    
					    		
					    			<h6>"Boolean Model for Melanogenesis"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Melanogenesis is a highly regulated process through which the pigment melanin is produced in the skin cells. Irregularities in the molecular events that govern the process of skin pigmentation can cause disorders like vitiligo. In order to understand the biology of disease progression, it is important to have an in depth understanding of intracellular events. Mathematical models provide an integrated view of intracellular signaling. There are very few models to date that incorporate intracellular processes relevant to melanogenesis and only one to our knowledge that simulates the dynamics of response to varying levels of input. Here, we report the formulation of the largest Boolean model (265 nodes) for melanogenesis to date. The model was built on the basis of a detailed interaction network graph published by Raghunath et al. Through additional manual curation of the reported interactions, we converted the graph into a set of Boolean rules, following the procedure of the first Boolean model (61 nodes) for melanogenesis published by Lee et al. Simulations show that the predicted response to varying UV levels for most of the nodes is similar to the predictions of the existing model. The greater complexity allows investigation of the sensitivity of melanin to additional nodes. We carried out perturbation analysis of the network through node deletion and constitutive activation to identify the sensitivity of outcomes, and compared the nodes identified as sensitive to previous reports
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Prachi-Bisht/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Prachi Bisht
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Interface growth driven by a single active particle"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We study pattern formation, fluctuations, and scaling induced by a growth-promoting active walker on an otherwise static interface. Active particles on an interface define a simple model for energy-consuming proteins embedded in the plasma membrane, responsible for membrane deformation and cell movement. In our model, the active particle overturns local valleys of the interface into hills, simulating growth, while itself sliding and seeking new valleys. In one dimension, this “overturn-slide-search” dynamics of the active particle causes it to move superdiffusively in the transverse direction while pulling the immobile interface upward. Using Monte Carlo simulations, we find an emerging tentlike mean profile developing with time, despite large fluctuations. The roughness of the interface follows scaling with the growth, dynamic, and roughness exponents, derived using simple arguments as beta = 2/3, z = 3/2, and alpha = 1/2, respectively, implying a breakdown of the usual scaling law beta = alpha/z, due to very local growth of the interface. The transverse displacement of the puller on the interface scales as~t^{2/3} and the probability distribution of its displacement is bimodal, with an unusual linear cusp at the origin. Both the mean interface pattern and probability distribution display scaling. A puller on a static two-dimensional interface also displays aspects of scaling in the mean profile and probability distribution. We also show that a pusher on a fluctuating interface moves subdiffusively leading to a separation of timescale in pusher motion and interface response.


PHYSICAL REVIEW E 100, 052120 (2019)
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Pranav-Khade/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Pranav Khade
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Iowa State University</em>
						    
						    
						    
					    		
					    			<h6>"Using alpha shapes to characterize protein flexibility"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			There are only limited methods available to study the global motions of the protein such as their hinge motions and shear motions. These motions take place over a broad range of time scales, from microseconds to seconds; however, molecular dynamics methods can only model easily the motions occurring on the time scale from picoseconds to microseconds, and in addition, such simulations require that replicas be run. Thus, extracting the meaningful slow motions is difficult. Hence, there is a need to model the global motions of the protein. The important motions depend on a multiscale phenomenon known as protein packing. In this study, we have explored alpha shapes (a subset of Delaunay tessellations) for the protein backbone coordinates as a model of protein packing. We demonstrate that the method can predict the protein hinges which are responsible for the global motions of the proteins.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Praneeth-Reddy-Sudalagunta/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Praneeth Reddy  Sudalagunta
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>H. Lee Moffitt Cancer Center & Research Institute</em>
						    
						    
						    
					    		
					    			<h6>"A pharmacodynamic model of clinical synergy in multiple myeloma"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Most anti-cancer therapies involve combinations of three or more agents, with the rationale that combining drugs with different mechanisms of action could maximize efficacy by targeting several subpopulations of a heterogeneous tumor. Synergy cannot be investigated in a clinical trial setting as the same patient cannot simultaneously receive single agents and their combination to quantify synergistic effect, while pre-clinical studies successfully quantify this effect, they do so in homogeneous cell lines. For this reason, clinical synergy remains difficult to investigate and translate into clinical utility. We focus our study on Multiple Myeloma (MM), an incurable hematological malignancy, due to its widespread use of combination therapy and the importance of timely therapeutic decisions in extending patient survival. We developed a mathematical framework that employs a second-order drug response model to fit patient-specific ex vivo responses of 203 MM patients to inform a novel pharmacodynamic model that accounts for two-way combination effects for 130 two-drug combinations. We have demonstrated that this model is sufficiently parameterized by single-agent and fixed-ratio combination responses, by validating model estimates with ex vivo combination responses for different concentration ratios, using a checkerboard assay. This novel model reconciles ex vivo observations from both Loewe and BLISS synergy models, by accounting for the dimension of time, as opposed to focusing on arbitrary time-points or drug effect. Clinical outcomes of patients were simulated by coupling patient-specific drug combination models with pharmacokinetic data from phase I clinical trials. Combination screening showed 1 in 5 combinations (21.43% by LD50, 18.42% by AUC) were synergistic ex vivo with statistical significance (P<0.05), but clinical synergy was predicted for only 1 in 10 combinations (8.69%), which was attributed to the role of pharmacokinetics and dosing schedules. Pre-clinical model predictions were used to accurately classify patients’ responses with statistically significant (P<0.05) accuracy as per International Myeloma Working Group response stratification criteria. This high-throughput combination screening framework identified drug combinations that are putatively clinically synergistic, and thus could potentially be used to screen for combinations that are likely candidates for a phase-III clinical trial. This could greatly benefit patients enrolling in these trials by improving the response on the experimental arm. The combinations shown to be most synergistic could be investigated to identify molecular pathways that govern their synergistic interaction.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Prashant-K-Srivastava/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Prashant K Srivastava
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>IIT Patna</em>
						    
						    
						    
					    		
					    			<h6>"The impact of information and saturated treatment with time delay in an Infectious disease model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Here we propose a mathematical model with a saturated treatment rate in the presence of information. We consider that the information about the disease affects the transmission rate of infection and hence the transmission rate is corrected. We also assume that people are losing their immunity against disease and the model is of SIRS type. We analyse the stability of the model system and our analysis shows that the model possesses the existence of backward bifurcation and multiple endemic steady states. Various situations of multiple endemic equilibrium points are explored numerically. Further, we extend the model to include the time lags in information and we found that in presence of time delay, the endemic steady state destabilizes and oscillations are observed. Thus, we conclude that if information dissemination is delayed beyond a threshold time then the infection oscillates in population and it may lead to difficulty in controlling the disease. Also, nonlinear incidence rate and saturated treatment may cause the existence of multiple endemic equilibrium and hence leads to complex dynamics.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Priyom-Adhyapok/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Priyom  Adhyapok
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indiana University</em>
						    
						    
						    
					    		
					    			<h6>"Modeling liver injury progression and repair"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Drug induced liver injury (DILI) can result in a build-up of oxidative stress in hepatocytes,causing them to become stressed and die. Experiments using APAP as a model of DILI show an initial pattern of centrilobular damage which gets amplified by stressed cells communicating through gap junctions and the activation of the immune system in response to this injury. While hepatocyte proliferation takes place to combat liver mass loss, higher doses could still be lethal to the tissue. This talk will try to address the question of what tips the balance, that the same set of cell behaviors that are needed for tissue survival can also lead to widespread tissue death in some other situations? To answer this, I will discuss results of a computational model based on the competing biological processes of hepatocyte proliferation, necrosis and injury propagation. This model sheds light on the evolution of tissue damage or recovery and predicts the potential for divergent fates given different rates of the parameters related to these three processes.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Quan-Anh-Hoang/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Quan Anh Hoang
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Vietnam National University</em>
						    
						    
						    
					    		
					    			<h6>"Analyzing Bacterial Motility Near a Smooth Surface"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Motile bacteria play a pivotal role for life on Earth and studying them has many real-world applications. In particular, studying how motile bacteria interact with a smooth surface provides fundamental understanding about their transition from living as free-swimmers in the fluid to being a part of a surface aggregated community. Such knowledge can be useful in the resolution of medical problems like infections in the lungs of cystic fibrosis patients.


In this work, we report the reconstructed three-dimensional motion of a motile bacterium from its two-dimensional images generated by total internal reflection fluorescence microscopy. First, the Trackpy package keeps track of the bacterium's position on a plane parallel to the surface at each time step. Then, our in-house Ellipsoid Fitting Algorithm analyzes the intensity profile of the bacterium to reconstruct its three-dimensional position and orientation relative to the surface. From these parameters, we further extract the velocity and the localized turning radii of the bacterium's trajectory in space.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rafael-Ramon-Bravo/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Rafael Ramon Bravo
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Cancer Crusade: Crowdsourcing Adaptive Therapy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Cancer Crusade is a citizen science game in which players design treatment strategies to influence a virtual tumor growth model. The treatment regimens that the players construct are collected and sent to an online database for analysis, which may give insights into clinically effective adaptive strategies. In parallel, a machine learning-based approach was applied to the game, and the optimized therapy generated is compared to those generated by the human players. For the core game engine in Cancer Crusade, we started with a previously published hybrid cellular automata model of tumor metabolism and growth. The original model was used to study how tumor cells evolve acid-mediated invasion and the impact of treatment on these metabolic processes. We expanded the model by adding more drugs and cell phenotypes, including drug-resistant cells. The game was released on mobile platforms and has been generating data from plays for several years. The treatment strategies were analyzed using dendrogram clustering to find key decision differences and profile their relative performance. We also performed network analysis to observe treatment transitions. These analyses indicated several effective strategies, which tended to oscillate between chemotherapy and a pro-angiogenic drug, or chemotherapy and a hypoxia-activated prodrug, or combinations of all three of these drugs. These results parallel a machine learning (Q-learning) approach to the problem, which yielded a preferred strategy based on chemotherapy combined with a pro-angiogenic drug. The existence of several human-discovered alternative strategies suggests that in general human players may offer greater variety of successful strategies via citizen science games than Q-learning.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rahma-Jerbi/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Rahma  Jerbi
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Tunisie</em>
						    
						    
						    
					    		
					    			<h6>"modélisation mathématique"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Je suis Rahma JERBI, doctorante en mathématiquées appliquées à l'université de Sfax (Tunisie) pour la préparation du doctorat sous le thème ' Etudes théoriques et numériques de quelques problèmes inverses', ma thèse deonc est basée sur la modélisation mathématique de quelques problèmes inverses. Pour céla je suis intéressée par votre conférence pour améliorer mes compétences en mathématiques appliquées.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rahul-Bishnoi/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Rahul  Bishnoi
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>IIT Guwahati</em>
						    
						    
						    
					    		
					    			<h6>"Eliminating the Need for User-Definition of Gap Penalties using Linear Programming"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In the widely renowned `Needleman-Wunsch’ algorithm for sequence alignment, an affine gap penalty system is used for penalizing the algorithm for the usage of gaps. There are two types of gaps used: opening gap penalty and extending gap penalty, where the former refers to the introduction of a gap and the latter to the extension of pre-existing gaps. The choice of determining the values of these gap penalties has to be predefined by the user. As a result, there may arise an inefficient sequence alignment solely due to the poor choice of opening and extending gap penalties. In this poster, we aim to address this concern by constructing a mathematical black-box model which is governed by a set of linear equations in the form of a modified scoring system for judging the quality of the alignment. We use three dynamic programming states (Identity, Similarity, and DP score) to determine the optimal alignment. While iterating through various values of the scoring parameters, the highest cumulative score determines an optimal alignment and the respective gap penalty. For distantly-related and average-related sequences, our algorithm shows a significant increase in terms of identity and similarity for all the cases as compared to EMBOSS-Needle. We eliminate the need for the user to choose the gap penalties based on mere intuition and judgement and provide an automated method of choosing the optimal values specifically for each set of sequences.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Ranjini-Bhattacharya/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Ranjini Bhattacharya
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Interpreting the Evolutionary Games Played in the NSCLC Microenvironment"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Lung cancer is the second most common type of cancer and non- small cell lung cancer (NSCLC) accounts for 84% of lung cancer diagnoses. Like other cancers, NSCLC is driven by somatic selection of fitter cells that can cheat the host. While there are treatments available to these patients, tumor heterogeneity enables selection of resistant cells. The tumor microenvironment can promote drug resistance and relapse by aiding tumor growth, angiogenesis, metastasis etc. Evolutionary Game Theory (EGT) can be used as a framework to map out the dynamics of different cellular strategies in a given tumor context to study cancer evolution. In our work we employ EGT to study the evolution of resistance to EML4- ALK positive NSCLC, with a focus on three cellular strategies- producers of hepatocyte growth factor (HGF), resistant cells, and sensitive cells. Tyrosine kinase inhibitors have been developed to block the oncogenic tyrosine kinase activity of ALK and inhibit cancer progression. progression. Resistance to TKIs has been attributed to expression of HGF which activates the alternative MET pathway, enabling cancer progression. We set up in vitro game assays to study the pairwise interactions between the three phenotypic strategies in control and drug (Alectinib) exposed environments. We find that HGF producers are the fittest and that they extend a protective effect on sensitive cells thus, enhancing their fitness. Curiously, this protection depends on the frequency of producers and undergoes saturation after a frequency of 0.4. Resistant cells do not show any significant interactions with the other two types. We then try to extrapolate the predictions from the pairwise games to predict outcomes of the three-player game involving all three phenotypes. Our study can give novel insights into possible therapeutic interventions targeting NSCLC and provides a framework for studying evolution of other cancers.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rebecca-Bekker/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Rebecca  Bekker
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Immunological Consequences of Uniform vs Spatially Fractionated Radiotherapy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Radiotherapy (RT) is the single most frequently used cancer treatment, with approximately 60% of patients undergoing either monotherapy or combination with other therapeutics. The immunogenicity of the tumor micro-environment (TME) affects how well a tumor responds to treatment.  TMEs can be hot or cold, with tumors in immunologically cold micro-environments generally showing less of a radiation response than those in immunologically hot micro-environments. Radiation can be immunosuppressive or immuno-stimulatory, but these contradictory effects are poorly understood.  It is thought that the immune response initiated by radiotherapy is curtailed by subsequent application of RT. Spatially fractionated radiotherapy shields areas of the tumor, thereby potentially protecting the immune environment. An ABM model was developed to evaluate the effects that different radiation doses, scheduling and SFRT architectures have on the immunological consequences of RT. We identify which types of TME respond better to spatially uniform or spatially fractionated radiotherapy, in the pursuit of advancing radiotherapy personalization.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rebecca-Sanft/">
						    	<h4><li class="subgroup-counter subgroup-counter-EDUC"></li>
							    	
							    	Rebecca  Sanft, Anne Walter
							    		(<span class="text-EDUC">EDUC</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UNC Asheville</em>
						    
						    
						    
					    		
					    			<h6>"Exploring Mathematical Modeling in Biology Through Case Studies and Experimental Activities"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Exploring Mathematical Modeling in Biology Through Case Studies and Experimental Activities, written collaboratively by a mathematician and biologist, provides supporting materials for a course taken simultaneously by students majoring in mathematics or computer science and those in the life sciences. The text is designed to actively engage students in the process of modeling through a collection of case studies and wet labs connecting mathematical models to real data. The supporting mathematical coding and biological background helps readers practice and build confidence in asking questions, formulating mathematical models, articulating model assumptions, estimating parameters, analyzing models, and interpreting the results. These skills can be applied in the case studies across multiple levels of organization and areas of biological inquiry. The labs reveal the practical issues of collecting data suitable for model formulation and validation. Moreover, collecting data in the context of a modeling course helps clarify experimental questions and design, and the model analysis often raises new research questions to explore. Through the case studies and labs, the reader will see the utility of models for understanding complex systems, making predictions, and identifying further questions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rebecca-H-Chisholm/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Rebecca H Chisholm
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>La Trobe University</em>
						    
						    
						    
					    		
					    			<h6>"A model of population dynamics with complex household structure and mobility: implications for transmission and control of communicable diseases"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Households are known to be high-risk locations for the transmission of communicable diseases.   Numerous modelling studies have demonstrated the important role of households in sustaining both communicable diseases outbreaks and endemic transmission, and as the focus for control efforts.  However, these studies typically assume that households are associated with a single dwelling and have static membership. This assumption  does  not appropriately  reflect  households in some populations, such as those in remote Australian Indigenous communities, which can be distributed across more than one physical dwelling, leading to the occupancy of individual dwellings changing rapidly over time. In this study, we developed an individual-based model of an infectious disease outbreak in communities with demographic and household structure reflective of a remote Australian Indigenous community.   We used the model to compare the dynamics of unmitigated outbreaks, and outbreaks constrained by a household-focused prophylaxis intervention, in communities exhibiting fluid versus stable dwelling occupancy.   Our findings suggest that fluid dwelling occupancy can lead to larger and faster outbreaks, interfere with the effectiveness of household-focused interventions, and may contribute to the considerable burden of communicable diseases in communities exhibiting this type of structure.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Reginaldo-Jos%C3%A9-da-Silva/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Reginaldo José da Silva
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Federal University of Alfenas, Alfenas, Brazil</em>
						    
						    
						    
					    		
					    			<h6>"Acoustic measures for voices in the classification of Parkinson's Disease"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The ART Family Networks are neural networks based on the Adaptive Resonance Theory that has as a characteristic the resonance between the input data and the cluster center as the data was classified. In particular, self-expanding ART Neural Networks characterized by the ability, even with little data, to perform the data classification and to expand according to the inclusion of the data. This work uses the Fuzzy ART Self-expanding neural network for the diagnosis of Parkinson's Disease. For this, it uses the Parkinson Speech Dataset with Multiple Types of Sound Recordings database, available in the UCI Machine Learning Repository repository. This database is composed of data from tested individuals, based on the characteristics extracted from 26 different voice samples per individual. The problem is to classify patients as healthy or with Parkinson's disease. The correlation coefficient used to select which voice resources were most relevant. After implementation using the resources that showed the strongest positive correlation, an accuracy of 98.56% obtained with a Mattews Correlation Coefficient of 0.9716 using the 10-fold cross-validation method.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Renato-Antunes-Costa-de-Andrade/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Renato  Antunes Costa de Andrade
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Glasgow</em>
						    
						    
						    
					    		
					    			<h6>"Persistence of intraguild predation in a 1D finite domain using variational approximations"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Fragmentation of the natural landscapes due to human activity has an undeniable undeniable eundeniable effect over the wildlife. The most known example being deforestation and the subsequent extinction of forest inhabiting species. In this context, a natural question to ask is: what is the minimum habitat size that allows for (co)existence of the species living in a given environment? The work described here then sought to address this inquiry for the particular case of 3 ecologically interacting species: a prey, its predator and a common resource. A system known as an intraguild predation module. We proposed and studied a model consisting of 3 coupled reaction-di_x000B_usion partial differential equations. In addition to numerical simulations, we used a method of approximation based on variational principles capable of providing analytical estimates for critical habitat sizes for the coexistence of the species involved in the proposed model. life. The most known example being deforestation and the subsequent extinction of forest inhabiting species. In this context, a natural question to ask is: what is the minimum habitat size that allows for (co)existence of the species living in a given environment? The work described here then sought to address this inquiry for the particular case of 3 ecologically interacting species: a prey, its predator and a common resource. A system known as an intraguild predation module. We proposed and studied a model consisting of 3 coupled reaction-diffusion partial differential equations. In addition to numerical simulations, we used a method of approximation based on variational principles capable of providing analytical estimates for critical habitat sizes for the coexistence of the species involved in the proposed model.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Renee-Dale/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Renee  Dale
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Using a trait-based dynamic mathematical framework to investigate the relationship between phenotypic dynamical parameters and the plant genome"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Phenotypic data is used to measure a variety of traits that can be traced to genes. In this study I use a mathematical approach to generate new traits that describe dynamic processes. By abstracting the process of the above ground Setaria plant tissue growth, a top-down trait based dynamical model was constructed. This model describes events that occur within the growth and development of above-ground tissue.
The mathematical framework considers above-ground plant tissue as either ‘resource generating’ or ‘non-resource generating’/’structural’. Care was taken in the mathematical representation to model the underlying growth and developmental processes, as well as the actual measurables described in the data. The data consist of biomass and height estimates by the image processing software PlantCV, as well as total water usage over 250 Setaria lines across 1100 plants in drought and well-watered conditions. To estimate the parameters of the dynamic processes, the model was fitted to the data.
The model parameters were constrained when possible based on plant physiological understanding of Setaria growth and development as well as plant biomass measurements. The heritability of the parameters were calculated for both wet and dry conditions. We found that heritability of these parameters differs between wet and dry conditions, as well as certain processes that describe events critical to the dynamics of Setaria growth as described by the model.
The parameters in our model are describing growth and developmental decisions of the plant. This method provides a novel way to identify plant phenotypic trait for identifying new genes that control dynamic processes. This novel framework will be used in the future to understand if phenotypic variability may be emergent from the interaction between environmental space searching strategies, biomass allocation strategies, and genotype.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Renier-G-Mendoza/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Renier G Mendoza
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Explicit solution of an age-structured model using a generalized Lambert W function"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Structured population models, which account for the state of individuals given features such as age, gender, and size, are widely used in the fields of ecology and biology. In this paper, we consider an age-structured population model describing the population of adults and juveniles. The model consists of a system of ordinary and neutral delay differential equations. We present an explicit solution to the model using a generalization of the Lambert W function called the r-Lambert W function. The r-Lambert W function, denoted Wr(a), is a function satisfying Wr(a)e^(W_r(a))+rWr(a)-a=0, where a is a nonzero complex number and r is a real number. Numerical simulations with varying parameters and initial conditions are done to illustrate the obtained solution.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rey-Audie-Escosio/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Rey Audie Escosio
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of the Philippines Diliman</em>
						    
						    
						    
					    		
					    			<h6>"Parameter Estimation of the Fitzhugh-Nagumo Model via a Perturbed Accelerated Gradient Descent Algorithm with an n-Dimensional Golden Section Search Method"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The Fitzhugh-Nagumo equations is a system of first-order nonlinear ordinary differential equations based on the researches of Fitzhugh and Nagumo et al. This reduced model captures the simplistic essence of neuronal spiking dependent on two variables, the membrane potential and the recovery variable.


From a collection of data points, the three parameters can be determined using the process of parameter estimation. This method minimizes the mean-squared difference between the data points and the solutions of the system. Researches by Ramsay et al. have shown the complexity and computational cost of this problem. The minimization may not converge properly using classic algorithms such as gradient descent and conjugate gradient. Hence, we propose a deterministic method that employs an accelerated gradient descent for iterating on the function surface and activates the local search to escape saddle points and local minima.


We apply the n-dimensional golden section search as the deterministic local search. It is a novel generalized technique for convex optimization by subsequently enclosing this optimum until convergence. Furthermore, partitioned n-spherical coordinate system is used which creates an adjusted smaller search
spaces as an equidistant ball centered on the iterate.


For the parameter estimation, the data points used is generated by applying noise to the deterministic solution of the system. The proposed algorithm, in comparison with other gradient-based methods such as the conjugate gradient and the steepest descent, highly performed in terms of its convergence, accuracy, and precision to the true value of the parameters amidst increasing noise level.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rhudaina-Z.-Mohammad/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Rhudaina Z. Mohammad
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of the Philippines Diliman</em>
						    
						    
						    
					    		
					    			<h6>"Cellular patterning in sensory systems: An interface evolution problem"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This work in collaboration with Karel Svadlenka (Kyoto University), Hideru Togashi (Kobe University), and Hideki Murakawa (Ryukoku University) focuses on modeling cellular rearrangements in tissue morphogenesis, with emphasis on observed cellular pattern formations in sensory epithelia. Adopting the viewpoint of free energy minimization principle, we focus on the energy associated with cell-cell junction, an interface between cells. We take cellular rearrangement as an $L^2$-gradient flow of a weighted interfacial energy constrained by each cell's preferred volume, where the weights are related to physical parameters of the cells, for example, cell-cell adhesion and cell contractility. Unlike existing models such as vertex dynamics model and cellular Potts model, which are also based on free energy minimization, we propose a level set-based approach which allows for cell-cell junctions with nonzero curvatures, realizes the correct cell contact angles, has minimal possible number of parameters, and naturally handles topological changes, e.g., cell intercalation, without relying on ad hoc algorithms that inevitably involve unnatural parameters. This model successfully reproduces the development of cellular patterns in embryonic auditory and olfactory epithelial tissues.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rifaldy-Fajar/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Rifaldy Fajar
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Yogyakarta State University</em>
						    
						    
						    
					    		
					    			<h6>"Analysis of Mathematical Model on the Development of Tumor Cells after Drug Therapy"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mathematical modeling way to explain the reality to the mathematic equations. One of the phenomena that can be modeling is the development of tumor cells after drug therapy. Tumor cells defend mutations with a process of cell reproduction and cells will be a move to all of the body. Cells occupy one of the other organs. Splitting about this case used drug therapy or chemotherapy. This research has the purpose of identifying and analysis mathematical models on the development of tumor cells after drug therapy. Identification of mathematical modeling includes the fixed point, the stability around the fixed point, and computer simulations. System of equations in this research using system differential equations non-linear of the first order and it is using four variables. They are immune cells I(t), tumor cells T(t), normal cells N(t), and drug therapy u(t). This system of equations obtained two fixed points is a fixed point of disease-free tumor and influence tumor. Stability around the fixed point will be stable when the fixed points of tumor cells T(t) = 0 and T(t) not equal to 0, with the fixed point tumor cells T(t) = 0,6900203854 cells/㎣, immune cells I(t) = 0,3671110057 cells/㎣, normal cells N(t) = 0,1835555029 cells/㎣, and drug therapy u(t) = 1 pg/ml. From the numerical simulation results can be the comparison between the graph model populations of tumor cells before and after administration drug therapy. Before the population of tumor cell given drug therapy will be increased and decline after being given drug therapy, whereas immune cells and a normal cell is increasing. This suggests drug therapy can impede the growth of tumor cells and increase the population of immune cells and normal cells.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Robyn-Shuttleworth/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Robyn  Shuttleworth
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Saskatchewan</em>
						    
						    
						    
					    		
					    			<h6>"Application of a cell-dense triphasic model to cryoprotectant equilibration"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The loading of cryoprotectants (CPAs) into tissue remains challenging due to the risk of both mechanical strain on the tissue and the risk of toxicity damage from the cryoprotectants flooding into the tissue. Many models have been developed to describe the loading of CPAs into either individual cells, or continuously into a thin slab of tissue, however there is little evidence of the two models being combined. To that end, we propose a model that builds upon the triphasic model for articular cartilage introduced in Abazari et. al. 2009, using a system of partial differential equations to describe the mass transport of each component, namely, water, CPA, salt, and the solid matrix. Within this system we incorporate the well-known two-parameter model (Kleinhans, 1998) to describe the cell membrane transport of both water and CPA within individual cells. Combining these two systems allows us to investigate the stress placed on the tissue by considering the interactions at both an extracellular and intracellular fluid level. In addition, this general model allows us to specify properties of a tissue, ranging from their structure and composition, i.e., their percentage of tissue solids and cells, to their hydraulic conductivity and CPA permeability rates. Using all of this information, and by defining a bath solution containing a CPA concentration, our model is able to predict the amount of stress that will be placed on the tissue during CPA loading. We will use our results to create optimised loading protocols to reduce the overall strain on the tissue during CPA loading.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rodrigo-Garcia/">
						    	<h4><li class="subgroup-counter subgroup-counter-NEUR"></li>
							    	
							    	Rodrigo Garcia
							    		(<span class="text-NEUR">NEUR</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universdad de la Republica</em>
						    
						    
						    
					    		
					    			<h6>"Small-worldness favours network inference in synthetic neural networks"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			A main goal in the analysis of a complex system is to infer its underlying network structure from time-series observations of its behaviour. The inference process is often done by using bi-variate similarity measures, such as the cross-correlation (CC) or mutual information (MI), however, the main factors favouring or hindering its success are still puzzling. Here, we use synthetic neuron models in order to reveal the main topological properties that frustrate or facilitate inferring the underlying network from CC measurements. Specifically, we use pulse-coupled Izhikevich neurons connected as in the Caenorhabditis elegans neural networks as well as in networks with similar randomness and small-worldness. We analyse the effectiveness and robustness of the inference process under different observations and collective dynamics, contrasting the results obtained from using membrane potentials and inter-spike interval time-series. We find that overall, small-worldness favours network inference and degree heterogeneity hinders it. In particular, success rates in C. elegans networks – that combine small-world properties with degree heterogeneity – are closer to success rates in Erdös-Rényi network models rather than those in Watts-Strogatz network models. These results are relevant to understand better the relationship between topological properties and function in different neural networks. Reference: García, R.A., Martí, A.C., Cabeza, C. et al. Small-worldness favours network inference in synthetic neural networks. Sci Rep 10, 2296 (2020).
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Rom%C3%A1n-Zapi%C3%A9n-Campos/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Román  Zapién-Campos
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Max Planck Institute for Evolutionary Biology</em>
						    
						    
						    
					    		
					    			<h6>"When timescales meet: Microbiome dynamics is influenced by hosts’ life-history."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Microbial life is highly abundant in the biosphere. Macroscopic lifeforms are no exception. We harbor a large number of microorganisms in different parts of our bodies, often referred to as the microbiome. What separates us from abiotic habitats is that we undergo life-cycles.
We have developed stochastic models to understand the consequences of host life-history on the ecology of the microbiome. Particularly, we focus on the effect of the host lifespan and initial microbiome. Our results point at the limits imposed by life-history, but also at the diverse dynamics, even in contexts free of selection at the level of microbes and hosts. Multiple reported experimental observations in organisms like nematodes, fruit flies, and zebrafish can be unified around colonization. These include the emergence and coexistence of alternative microbiome states, the persistence of microbe-free hosts, and the inconsistent occurrence of microbial types.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Roxana-L%C3%B3pez-Cruz/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Roxana  López Cruz
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Universidad Nacional Mayor de San Marcos</em>
						    
						    
						    
					    		
					    			<h6>"A Coupled Mathematical Model Introducing Biological Control of Dengue"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			This work aims to study the dynamical behavior of a dengue epidemics model by introducing biological control. The first model represents the SEIR-SEI model of dengue epidemics and the second model corresponds to coupling a model of biological control. Controlling mosquitoes is considered by an infestation of bacteria that inhibit the transmission of dengue in humans. We determine an analytic expression of replacement ratios that depends on biological control. The results obtained show that the global stability of the disease-free equilibrium is determined by the value of a certain threshold parameter called the basic reproductive number R0 and of the replacement ratios of the biological model RU, RW. The sensitivity analysis shows that the vector to human transmission rate promotes more changes to the biological control system than other parameters. The simulation analysis of the last model showed the efficiency of the biologic control of dengue transmission.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Russell-Milne/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Russell  Milne
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Effects of Variation in Fishing Rate and Nutrient Loading on Coral Reef Health with Implications for Marine Protected Area Design"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Coral reefs rank among the highest-biodiversity habitats in the world, hosting many fish and invertebrate species found nowhere else. Additionally, reef ecosystems generate billions of dollars in revenue annually for coastal communities via fishing and tourism. However, from the Caribbean to Australia, coral reefs are in decline worldwide. Causes of this include anthropogenic stressors such as overfishing and excess input of nitrogen and other nutrients. To evaluate the threats posed to reefs by these processes, I simulate reef dynamics using a mechanistic, spatially explicit model fit using field data. I find three major regimes: one where coral dominates with periodic algal blooms, one where coral and algae coexist, and one where coral is driven to extinction by algae, in order from lowest to highest fishing rates. For moderate fishing rates, both a healthy coral population and a profitable local fishing industry can exist. Also, establishing a marine protected area (MPA) with no fishing in 20 percent of the simulated area is enough to maintain the coral-dominant equilibrium in the rest of the system, even when fishing rates outside the MPA are very high. Decreasing nutrient input into the system can also shift it towards the coral-dominant equilibrium. The rates of nutrient loading at which regime shifts are predicted to occur vary nonlinearly with fishing rate.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Ryan-Murphy/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Ryan  Murphy
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Queensland University of Technology</em>
						    
						    
						    
					    		
					    			<h6>"Mechanical cell competition in heterogeneous epithelial tissues"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mechanical cell competition is important during tissue development, cancer invasion, and tissue ageing. Heterogeneity plays a key role in practical applications since cancer cells can have different cell stiffness and different proliferation rates than normal cells. To study this phenomenon, we propose a one-dimensional mechanical model of heterogeneous epithelial tissue dynamics that includes cell-length-dependent proliferation and death mechanisms. Proliferation and death are incorporated into the discrete model stochastically and arise as source/sink terms in the corresponding continuum model that we derive. Using the new discrete model and continuum description, we explore several applications including the evolution of homogeneous tissues experiencing proliferation and death, and competition in a heterogeneous setting with a cancerous tissue competing for space with an adjacent normal tissue. This framework allows us to postulate new mechanisms that explain the ability of cancer cells to outcompete healthy cells through mechanical differences rather than by having some intrinsic proliferative advantage. We advise when the continuum model is beneficial and demonstrate why naively adding source/sink terms to a continuum model without considering the underlying discrete model may lead to incorrect results.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Ryan-Schenck/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Ryan  Schenck
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"The Tick-Tock of the Molecular Clock: Random methylation state changes inform homeostasis in the intestinal crypt"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The small intestinal and colon crypts are hierarchical, dynamic systems. Small numbers of stem cells give rise to daughter cells which proliferate within the transient amplifying zone, giving rise to a differentiated cell population. Stem cell numbers are constant, but survival is stochastic because divisions may result in renewal, expansion, or extinction. This hierarchy is largely maintained even in the face of disease and early dysplasias, where the microenvironment strives to re-establish homeostasis. Surprisingly, little is known about the stem cell numbers within these crypts. We use a distribution of CpG sites taken from four individual crypts across 60 patients to integrate with a statistical fitting approach and an agent based mechanistic model of the homeostatic crypt. Using this integrative approach we are better able to understand the dynamics of the stem cell pool. Most notably, using this method we can determine the number of stem cells and estimation of the error rates associated with DNA methyltransferase during cell division. The model splits the crypt into two compartments, the base and body of the crypt, and incorporates base pair resolution genomes and CpG sites. We calibrate our exclamatory bowel model with normal human small intestinal and colon crypt data using Approximate Bayesian Computation. This approach provides numerous insights into the dynamics of aging and underlying diseases within human crypts. The model can also be used to investigate why small intestinal crypts rarely develop cancers while colon cancer is frequently seen.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Sara-Sommariva/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Sara  Sommariva
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Geneva</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical model of loss and gain of function mutations in a chemical reaction network for colorectal cancer cells."</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			All cellular functions are regulated by a complex network of chemical reactions that translates extracellular signals into cellular responses. Most cancer diseases are induced by alterations of this signalling network due to loss or gain of function mutations that respectively reduce or enhance the activity of specific proteins. Here we present a computational tool for simulating chemical reaction networks and their alteration due to loss and gain of function mutations. By applying mass action kinetics, we first describe the concentration dynamics of the species involved in the reaction network through a system of ordinary differential equations (ODEs), whose stationary stable state describes the species concentrations in the physiological cell. We then show that loss of function mutations can be implemented in the model via modification of the initial conditions of the system while gain of function mutations can be implemented by eliminating specific reactions. Eventually our model is extended to account for the concatenation of multiple mutations. As example we consider the chemical reaction network devised by Tortolina and colleagues for  the G1-S transition point in colorectal cancer cells. We validate our approach by simulating the most frequent mutations in this type of cancer and comparing the results predicted by our model with those in the literature.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Sarah-MacQueen/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Sarah MacQueen
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Modelling the effects of site constancy in bumble bees"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Foraging site constancy, or repeated return to the same location forage, is an important aspect of bumble bee behaviour, and should therefore be an important consideration when using modelling to predict the pollination services provided by bumble bees. However, it is unknown exactly how bumble bees select their foraging site, and most modelling studies do not account for this uncertainty. We used an individual based model to explore how predictions of pollination services and bee fitness change under different foraging site selection methods. Pollination services are measured as the percent of fields and number of flowers visited, and bee fitness is measured as the amount of different resource types collected and using behavioural budgets. We tested two different site-reconnaissance or searching methods (random and realistic exploration behaviour) and four different site- selection methods (random and optimizing based on distance from the nest, local wildflower density, or net rate of energy return), as well as comparing results on landscapes with different total amounts of resource and proportional amounts of crop. We found that site- selection methods have a greater impact on crop pollination services and bee fitness than do site-reconnaissance or landscape characteristics, indicating that the site-selection method is an important consideration when modelling bumble bee pollination services. In general, site-selection based on optimizing for the net rate of energy return leads to both the highest crop pollination services and the longest foraging trips. The percent of crop fields visited, amount of time spent foraging, number of foraging sites located in crops, and the number of flowers visited may be used to make hypotheses about how real bees select their foraging sites.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Sayan-Biswas/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Sayan Biswas
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Institute for Stem Cell Science & Regenerative Medicine</em>
						    
						    
						    
					    		
					    			<h6>"Predicting mitochondrial functional state : An imaging based SVM tool"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mitochondria are dynamic organelles, shown to provide signatures of onset of diseases or cellular stresses. In this work we attempt to probe whether specific patterns exist for stress in mitochondria which is assessed though quantitative imaging followed by machine learning. Feature extraction is performed to assess mitochondrial morphology, intensity and intra-mitochondrial structural heterogeneity features from confocal micrographs of mitochondria stained with micro-viscosity sensing and potentiometric dyes. These features examined from the acquired con-focal images of cells with mitochondria in apriori known perturbed (or stress) or unperturbed functional state. Perturbation dependent signatures in these assessed features were studied by employing supervised learning - Support Vector Machine. The classification between perturbed and unperturbed states was performed with an accuracy of nearly 93%. Furthermore, accuracy improvement methods were engineered by optimising the feature space through which an optimal classification accuracy of 98% was achieved and accounted the importance of intra mitochondrial heterogeneity. Overall, the derived features showed the presence of computationally identifiable unique functional state dependent patterns strengthening the way for assembling predictive models for assaying mitochondrial functional states. This could assist in developing less resource-intensive method to classify and study stress conditions. This tool provides a promising application of mathematics in biology that could be applied to characterize stress conditions, drug screening to aid the identification of functional stress state.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Sebastian-Ruhle/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Sebastian Ruhle
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>HTW Dresden</em>
						    
						    
						    
					    		
					    			<h6>"Analysis of cell contact inhibition during growth of epithelial tissue"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The question of dominating mechanisms in the development of healthy epithelial tissue is subject to contemporary research, especially for tumour progression. While experiments suggest, that biomechanical cell-cell-interactions are crucial for the development of the tissue, it is usually oversimplified or neglected in theoretical approaches. For instance, the impact of cell migration, competition or contact inhibition on development of a cell colony is barely quantified. Puliafito et al. performed experiments on MDCK-cells (Madin-Darby-Canine-Kidney-cells) [1] and proposed, that the behaviour of the colony during the growth phase can be solely explained by contact inhibition To test this hypothesis, we develop a cell-based model and compare the numerical results with the experimental data. We introduce a novel cell-based model based on a probabilistic cellular automaton  as our basic model and which is capable of emulating single cell behaviour like persistent cell migration, growth or proliferation and cell-cell-interactions like adhesion. The model parameters are calibrated by evaluating experimental single cell tracking measurements. Subsequently, we compare the temporal development of emergent quantities in the model like colony area, density, shape, cell size distribution, effective cell proliferation rate and the characteristic length scale of collective cell motion with the experiment.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Seokjoo-Chae/">
						    	<h4><li class="subgroup-counter subgroup-counter-NEUR"></li>
							    	
							    	Seokjoo Chae
							    		(<span class="text-NEUR">NEUR</span>)
						    	</h4>
					    	</a>
						     
						    	<em>KAIST</em>
						    
						    
						    
					    		
					    			<h6>"The data-based inference method reveals the network structure of the SCN"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The suprachiasmatic nucleus (SCN) is the central circadian pacemaker in mammals. Even though SCN is composed of thousands of heterogeneous self-oscillating cells, the SCN can synchronize its component oscillators through the SCN neuronal network. To understand the SCN network structure, previous studies used the time series data to infer the network structure. However, because the SCN is synchronized, previous methods falsely inferred the network as if all the SCN cells were coupled with each other. To circumvent this, we develop a novel data-based method, which can successfully infer the SCN network from the time series data. In particular, our method accurately infers the SCN network with single-cell resolution bioluminescence data from 2,000 mice SCN cells.  Furthermore, our method can infer the directionality of the coupling between SCN cells.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Shadi-Esmaeili-Wellman/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Shadi Esmaeili-Wellman
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of California Davis</em>
						    
						    
						    
					    		
					    			<h6>"Density Dependent Resource Budget Model for Alternate Bearing"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Alternate bearing is the variability of the fruit and nut production with a strongly biennial pattern and is observed in many types of plants. This phenomenon is observed in collective synchrony among trees that are coupled directly and indirectly in orchards and natural forests and is known as masting. This is a yearly phenomenon so discrete time models, coupled nonlinear difference equations, are the appropriate modeling framework, with alternation between local in space dynamics and exchange between locations. The well-known resource budget model, while proposing a mechanism for alternate bearing behavior, can only model the synchronization observed in systems where the trees are coupled through indirect coupling (pollen coupling). We developed a density-dependent resource budget model based on the balance between photosynthesis and reproduction process. We analyzed the model through examination of the bifurcation structure for the uncoupled (local in space) model and numerically for spatially coupled versions. By addressing some of the shortcomings of the well-known resource budget model, our new approach can model the alternate bearing behavior and the synchronization phenomenon observed in trees coupled through direct coupling mechanism (root grafting).
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Shaon-C-Chakrabarti/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Shaon C Chakrabarti
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>National Centre for Biological Sciences</em>
						    
						    
						    
					    		
					    			<h6>"Why cousins are more similar than mother daughters: implications for cell cycle control"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The origin of lineage correlations among single cells and heterogeneity in their intermitotic and apoptosis times (IMT and AT) may reflect underlying principles of cell cycle control. We developed lineage-tracking experiments and computational algorithms to uncover correlations and heterogeneity in the IMT and AT of a colon cancer cell line before and during cisplatin treatment. These correlations could not be explained using simple protein production/degradation models. Sister cell fates were similar regardless of whether they divided before or after cisplatin administration and did not arise from proximity-related factors, suggesting fate determination early in a cell’s lifetime. Based on these findings, we developed a theoretical model explaining how the observed correlation structure can arise from oscillatory mechanisms underlying cell fate control. Our model recapitulated the data only with very specific oscillation periods that fit measured circadian rhythms, thereby suggesting an important role of the circadian clock in controlling cell cycle progression.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Shawn-Ryan/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Shawn Ryan
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Cleveland State University</em>
						    
						    
						    
					    		
					    			<h6>"Role of Hydrodynamics in Chemotaxis of Bacterial Populations"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			How bacteria sense local chemical gradients and decide to move has been a fascinating area of recent study. Chemotaxis of bacterial populations has been traditionally modeled using either individual-based models describing the motion of a single bacterium as a velocity jump process, or macroscopic PDE models that describe the evolution of the bacterial density. In these models, the hydrodynamic interaction between the bacteria is usually ignored. However, hydrodynamic interaction has been shown to induce collective bacterial motion and self-organization resulting in larger mesoscale structures. In this talk, the role of hydrodynamic interactions in bacterial chemotaxis is investigated by extending a hybrid computational model that incorporates hydrodynamic interactions and adding components from a classical velocity jump model. It is shown that by including hydrodynamic interactions, a suspension with a low initial volume fraction can exhibit locally high concentrations in bacterial aggregates. Also, it is shown that hydrodynamic interactions enhance the merging of the small aggregates into larger ones and lead to qualitatively different aggregate behavior than possible with pure chemotaxis models. Namely, differences in the shape, number, and dynamics of these emergent clusters.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Shelby-M-Scott/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Shelby M Scott
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"COVID and Crime: Analysis of Crime Dynamics Amidst Social Distancing Protocols"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In response to the spread of the global pandemic in early 2020, many cities implemented states of emergency and stay at home orders to reduce virus spread. Changes in social dynamics due to restrictions has had an impact on cities across the United States. One change of interest is how crime dynamics shifted in response to quarantine. In this paper, we compare the crimes that occurred before the implementation of stay at home orders and the two weeks after these orders were put in place across three cities. Using t-tests, we find that in Chicago, Illinois, Baltimore, Maryland, and Baton Rouge, Louisiana, total crimes showed significant declines in the two weeks following stay at home orders. Chicago showed the most stark differences between these two time periods, but in all three cities the crime types contributing to these declines were related to property crime rather than interpersonal interactions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Shohel-Ahmed/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Shohel  Ahmed
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Alberta</em>
						    
						    
						    
					    		
					    			<h6>"Global Asymptotic Stability for a Diffusive Opportunistic Diseases Model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this study, we consider a spatial infectious disease model under Opportunistic epidemics which allows for the continuous contribution of extracellular compartments. We show that the proposed model has a unique steady state that is asymptotically stable. Using an appropriately constructed Lyapunov functional, we establish its global asymptotic stability. Numerical results obtained through MATLAB simulations are presented to confirm the theoretical findings of this study.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Shota-Shibasaki/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Shota Shibasaki
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Lausanne</em>
						    
						    
						    
					    		
					    			<h6>"Fluctuating environments affect the strength of species interactions and diversity in microbial communities similarly"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Microorganisms live in environments that often fluctuate between mild and harsh conditions. Although such fluctuations are bound to cause local extinctions and thereby affect species diversity in microbial communities, it is still unknown (i) how species diversity changes over the rate of environmental fluctuations and (ii) how this relates to changes in species interactions. Here, we use a mathematical model to describe the dynamics of resources, toxins and species abundances in a chemostat where resource supplies switch between scarce and abundant. Over the majority of the explored parameter space, species compete with one another, but the strength of competition between species pairs changed over the switching rate in a pattern that depended on their sensitivity to toxins. When their toxin sensitivities were low, an effect of competition was highest at a low switching rate. At other toxin sensitivity values, competition was instead highest at intermediate or high switching rates. In communities of up to ten species, the strength of competition in species pairs was a good predictor for how community beta diversity changed over the environmental switching rate: diversity was lowest when competition was highest. This shows that an analysis of pairwise species interactions can be used to estimate how beta diversity changes over environmental switching rates. Our results also indicate that predicting how environmental switching affects communities is very difficult a priori, as it depends on the properties of its members, such as their tolerance to environmental toxicity. This may explain the contradicting results of some earlier studies on the intermediate disturbance hypothesis.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Silas-Poloni/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Silas  Poloni
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Institute for Theoretical Physics- São Paulo State University</em>
						    
						    
						    
					    		
					    			<h6>"Intraguild Predation in Periodic Habitats"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Fragmentation of natural landscapes is an ongoing process, mainly led by human activities, such as urban growth, roadway construction and farming. This phenomena may lead to many changes in the dynamics of populations that live in such landscapes, posing new challenges to our understanding of population persistence and diversity therein.
One of the first approaches on how we may treat such problems mathematically was given by Shigesada  in 1986, where a single population invading an infinite unidimensional habitat, composed of two types of patches layed on the real line alternately, is considered.  Along that, theoretical aspects of how populations behave on the borders between different types of patches were developed, unraveling new classes of realistic boundary conditions for spatial ecology models, giving us new results and insights in this field.
In this work we consider an Intraguild Predation (IGP) model, a community module composed of two consumers of a shared resource, with a predation relation between such consumers, usually referred as IG-Prey and IG-Predator. First we deal with invasions of both IG-Prey and IG-Predator in an homogeneous landscape, with either the resource established alone or together with the other consumer. Then, using Cobbold and Yurk's homogenization technique, we formulate and investigate the problem in a periodic habitat, composed of two types of patches where IGP relations are present, but allowed to have different parameters, such as less resource consumption, enhanced mortality or reduced resource productivity in one of the patches.
Our results show that coexistence  between IG-Prey and IG-Predator is possible within a range of resource productivity in homogeneous landscapes, being such range determined via analysis of the minimal speeds of invasion. In heterogeneous landscapes, with IGP being viable on both patches, we find that the necessary conditions for coexistence may be relaxed given certain movement behavior of both consumers and resource alike, whilst some configurations restrict such condition. We also explore how the ranges of coexistence in terms of resource productivity change with the sizes of the two habitat types considered, finding that such regions are also diminished or enlarged, depending on the movement behavior of the IGP populations.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Simon-Mitchell/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Simon  Mitchell
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Sussex</em>
						    
						    
						    
					    		
					    			<h6>"A Systems Biology Approach Predicts Distinct Roles for NFkB Subunits cRel and RelA in DLBCL"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Heterogeneity in therapeutic response presents a challenge to the successful treatment of Diffuse Large B-Cell Lymphoma (DLBCL). Despite the recognition that DLBCL cases have diverse genetic and transcriptional characteristics, standard first-line therapy has remained unchanged for more than a decade. Canonical Nuclear Factor KappaB (NFκB) is a dimeric transcription factor usually consisting of either cRel or RelA bound to p50. While aberrant NFκB activation is frequently observed in DLBCL, subunit composition in individual DLBCL cases is not routinely characterized but has the potential to improve stratification and identify novel molecular targets for treatment. Computational simulations of NFκB control over B-cell proliferation and apoptosis accurately predict experimental results with accuracy at both single-cell and cell-population scale. However, the key regulatory networks controlling B-cell differentiation were not factored into these predictive models. Simulations based on known regulatory interactions were insufficient to recapitulate healthy B-cell differentiation. Using a systems biology approach we found that although cRel drives B-cell proliferation, it also blocks terminal differentiation to antibody-secreting plasma cells; dynamic downregulation of cRel by Blimp1 was a pre-requisite for differentiation. Inclusion of this interaction into multiscale computational models enabled simulations to accurately predict B-cell population dynamics in wild-type (WT) and cRel knockout cells. Simulations of aberrantly increased NFκB activity, recapitulated the increased proliferation and cell survival seen in both ABC- and GC-DLBCL. In order to interrogate the subunit-specific roles of cRel and RelA in DLBCL we performed simulations in which each subunit was individually upregulated. Both of these models predicted hyperproliferation and apoptosis avoidance, but only upregulation of cRel resulted in an inability to exit the germinal centre as seen in GC-DLBCL. In contrast, RelA- specific upregulation resulted in population expansion without a block on differentiation, with cells predicted to take on a more differentiated state consistent with cell-of-origin classification of ABC- DLBCL. This subunit-specific control over DLBCL sub-type aligns with experimental observations of the less differentiated state of GC- compared to ABC-DLBCL. Other commonly occurring mutations in DLBCL affecting BCL2, IRF4, and MYC were simulated to recapitulate dysregulated apoptosis, differentiation and cell-cycle respectively, along with “double-hit” mutations. These mutation-specific simulations of DLBCL represent in silico laboratories where biomarkers can be identified to stratify and target lymphoma.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Simona-Catozzi/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Simona  Catozzi
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Systems Biology Ireland</em>
						    
						    
						    
					    		
					    			<h6>"Predicted ‘wiring landscape’ of Ras network in 29 human tissues"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Ras is an important hub protein at the head of numerous signaling pathways and plays a starring role in various types of cancers, notably in pancreas, colon and lung adenomas. The usual suspects are three oncogenic isoforms - i.e. HRAS, KRAS and NRAS - that are highly mutated and drive tumorigenesis. Our study is based on the paradigm of network medicine that sees disease as a perturbation of a network of interconnected proteins finely orchestrating cell's physiology and phenotype through the onset of downstream signal transduction. As such, we built a mechanistic model of the interactions of the three Ras oncoproteins with their direct interactors (known as 'effectors'), with protein abundances and binding affinities being the system's parameters, in order to study elementary pathological and physiological conditions of Ras network. Using high-quality proteomic data from 29 (healthy) human tissues, we quantified the amount of individual Ras-effector complexes, and characterized the (stationary, reference) Ras “wiring landscape” specific to each tissue. We simulated mutant- and stimulus-induced network re-configurations, miming respectively cancerous and physiological state, and compared them to the reference network. Moreover, we investigated the contribution of the input parameters (binding affinities and effector concentrations) in determining the complex formations underlying the specific wiring landscape, by 3D data interpolation onto (tissue-specific) surfaces. This revealed that high affinity - more than high concentration, - is critical for complex formation. As a consequence, we analyzed local and global binding affinity fluctuations and assessed their impact on the system's robustness. Further research will aim at the calibration of the binding affinity parameters, based on the Ras-effector complexes and the activation of the associated downstream pathway.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/So-Nakashima/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	So  Nakashima
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"Lineage EM Algorithm for Inferring Latent States from Cellular Lineage Trees"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			A population of genetically identical cells is phenotypically heterogeneous. The heterogeneity is partially inherited over generations and can work as a bet-hedging strategy of the survival of the population under fluctuating environments. A typical instance of the bet-hedging strategy is the bacterial persistence. To understand such strategies, we need to identify the phenotypes of each cell and its inheritance. For this purpose, recent advancements in single-cell analysis and microfluidic devices offer us useful lineage data, though such data accommodate but do not explicitly show the phenotypic information of each cell. Several studies have attempted to overcome the difficulty by inferring the phenotypes from lineage data via latent-variable estimation. However, we must correct the bias caused by the growth of the population, which we call the survivorship bias, in the estimation. In this work, we characterize the survivorship bias and establish a correction method of the bias. Then, we propose an expectation-maximization (EM) type latent variable estimation, which we call Lineage EM algorithm (LEM). LEM is bias-free and applicable to various kinds of lineage data to characterize the phenotype of the cells. Finally, we apply LEM to a synthetic and a real lineage tree of E. coli and validate the performance.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Soumen-Bera/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Soumen Bera
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Central University of Rajasthan, India</em>
						    
						    
						    
					    		
					    			<h6>"Biphasic Adaptive Activity of Plant Nitrate Transporter NRT1.1"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Defective nitrate signaling in plants causes disorder in nitrogen metabolism, and it negatively affects nitrate transport systems, which toggle between high- and low-affinity modes in variable soil nitrate conditions. Recent discovery of a plasma membrane nitrate transceptor protein NRT1.1—a transporter cum sensor—provides a clue on this toggling mechanism. However, the general mechanistic description still remains poorly understood. Here, we illustrate adaptive responses and regulation of NRT1.1-mediated nitrate signaling in a wide range of extracellular nitrate concentrations. The results show that the homodimeric structure of NRT1.1 and its dimeric switch play an important role in eliciting specific cytosolic calcium waves sensed by the calcineurin-B-like calcium sensor CBL9, which activates the kinase CIPK23, in low nitrate concentration that is, however, impeded in high nitrate concentration. Nitrate binding at the high-affinity unit initiates NRT1.1 dimer decoupling and priming of the Thr101 site for phosphorylation by CIPK23. This phosphorylation stabilizes the NRT1.1 monomeric state, acting as a high-affinity nitrate transceptor. However, nitrate binding in both monomers, retaining the unmodified NRT1.1 state through dimerization, attenuates CIPK23 activity and thereby maintains the low-affinity mode of nitrate signaling and transport. This phosphorylation-led modulation of NRT1.1 activity shows bistable behavior controlled by an incoherent feedforward loop, which integrates nitrate-induced positive and negative regulatory effects on CIPK23. These results, therefore, advance our molecular understanding of adaptation in fluctuating nutrient availability and are a way forward for improving plant nitrogen use efficiency.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Stefano-Pasetto/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Stefano  Pasetto
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Moffitt Cancer Center</em>
						    
						    
						    
					    		
					    			<h6>"Bayesian framework for tumor board decision making"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Traditionally, the specific treatment for a cancer patient is decided by a multidisciplinary tumor board, which integrates prior clinical experience, published data, and patient-specific factors to develop a consensus on an optimal therapeutic strategy. However, tumor boards often encounter patients who incompletely match extant data and for whom several treatment options must be evaluated based on imprecise criteria. We propose optimizing treatment outcomes will require a flexible but rigorous mathematical tool that can define the probability of success of given therapies. Here, we propose a Bayesian approach to tumor forecasting using a multi-model framework that can predict response to different targeted therapies within individual patients.  By exploiting the integrative power of the Bayesian decision theory, we demonstrate multiple therapeutic options can be simultaneously examined so that the resulting clinical course can be forecasted. From this, we detail a general decisional methodology built upon a robust and well-established mathematical framework that can support the clinical decision process for individual patients within a clinical tumor board.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Stephen-Y-Zhang/">
						    	<h4><li class="subgroup-counter subgroup-counter-CDEV"></li>
							    	
							    	Stephen Y  Zhang
							    		(<span class="text-CDEV">CDEV</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Univ. British Columbia</em>
						    
						    
						    
					    		
					    			<h6>"Inference of stochastic cellular dynamics from time-series data using optimal transport"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Cellular and developmental biology presents a wealth of processes that are inherently stochastic in nature, ranging from development to wound healing and carcinogenesis. Modern technologies such as single-cell transcriptomics and epigenomics have enabled interrogation of biological phenomena with unprecedented precision and throughput. These technologies necessarily destroy the cells being measured. Thus, any instance of a biological process can only be measured once to produce a static snapshot, and the underlying behaviour of cells over time is lost. The development of tools for reconstructing temporal dynamics from such snapshots is therefore a major challenge that is crucial to painting an accurate biological picture.
We propose a method for inferring governing dynamics from a series of temporal snapshots (such as single-cell transcriptomic profiles) sampled from populations of cells that evolve following some biological stochastic process. Our approach is based on optimal transport, a contemporary mathematical theory at the intersection of analysis, probability and geometry that provides a natural means of comparing probability distributions. Equipped with a corresponding convex optimisation framework, we provide an initial demonstration of accurate recovery of dynamics from simulated data. We also discuss how we tackle the biologically important challenge of dealing with high dimensionality and cellular growth, with a view to application to experimental datasets.
This is a joint work with Young-Heon Kim, Hugo Lavenant and Geoffrey Schiebinger.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Subbalakshmi-A-R/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Subbalakshmi  A R
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indian Institute of Science</em>
						    
						    
						    
					    		
					    			<h6>"NFATc acts as a non-canonical phenotypic stability factor for a hybrid epithelial/mesenchymal phenotype"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			More than 90% of cancer-related deaths can be attributed to metastasis. Cells adapt to their changing environmental conditions and avoid therapy and immune response during metastasis by employing the phenotypic plasticity. Reversible transitions between epithelial and mesenchymal phenotypes - Epithelial-Mesenchymal Transition (EMT) and its reverse Mesenchymal-Epithelial Transition (MET) - form a key axis of phenotypic plasticity during metastasis and therapy resistance. Recent studies have shown that the cells undergoing EMT/MET can attain one or more hybrid epithelial/mesenchymal (E/M) phenotypes, the process of which is termed as partial EMT/MET. Cells in hybrid E/M phenotype(s) can be more aggressive than those in either epithelial or mesenchymal state. Thus, it is crucial to identify the factors and regulatory networks enabling such hybrid E/M phenotypes. Here, employing an integrated computational-experimental approach, we show that the transcription factor NFATc can inhibit the process of complete EMT, thus stabilizing the hybrid E/M phenotype. It increases the range of parameters enabling the existence of a hybrid E/M phenotype, thus behaving as a phenotypic stability factor (PSF). However, unlike previously identified PSFs, it does not increase the mean residence time of the cells in hybrid E/M phenotypes, as shown by stochastic simulations; rather it enables the co-existence of epithelial, mesenchymal and hybrid E/M phenotypes and transitions among them. Clinical data suggests the effect of NFATc on patient survival in a tissue-specific or context-dependent manner. Together, our results indicate that NFATc behaves as a non-canonical phenotypic stability factor for a hybrid E/M phenotype.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Sungyoung-Shin/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Sungyoung  Shin
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Monash University</em>
						    
						    
						    
					    		
					    			<h6>"A mathematical model of GβL (de)ubiquitination switch uncovers biphasic response within the PI3K/mTOR signalling network"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The PI3K/AKT/mTOR signalling pathway is a critical pathway in mammalian cells that regulates a broad array of cellular processes, including proliferation, survival and metabolism. G-protein beta-subunit-like (GβL or mLST8) has been long known as one of the shared subunits of both the mTORC1 and mTORC2 complexes. Recently, it was reported that the dynamic (de)ubiquitination of GβL generates a molecular switch mechanism that governs the binding of GβL towards Raptor and Sin1, core subunits of mTORC1 and 2, respectively; thereby actively coordinating the formation and abundances of these complexes. This new switch mechanism adds an extra layer of complexity to an already complex signalling network featuring abundant interlinked feedback regulation. However, how the GβL-mediated switch interplays with other regulatory mechanisms to control the dynamics and steady state behaviors of the PI3K/AKT/mTOR network is poorly understood. Here we integrate computational network modelling and biological experiments in a systems-based framework to characterize the network-level properties of PI3K/mTOR signalling conferred by the GβL-regulated switch, and interrogate the impact on network behavior when this switch is disrupted. To this end, we construct a novel mechanistic mathematical model of the PI3K/mTOR network that explicitly considers the GβL switch. The model is quantitatively calibrated and kinetic parameters are estimated using time-course data obtained from Mouse Embryonic Fibroblasts (MEF) cells. In contrast to previous studies indicating GβL is required for mTORC1 formation but not activity, our integrative in-silico/experimental analyses demonstrate that GβL is essential for formation as well as activation of both mTOR complexes. Importantly, mode simulations predict a previously unknown biphasic dependence of mTORC1 activity on Sin1, an integral component of mTORC2, revealing an intriguing non-linear functional linkage between the complexes. An increase of Sin1 from a low level initially promotes mTORC1 activity (first phase), but further increase of mSIN1 beyond a critical threshold (second phase) instead downregulates mTORC1 activity. We subsequently validate this prediction experimentally in MEF cells. In summary, this study presents a novel mathematical model of the PI3K/mTOR pathway that enables quantitative analysis of the role of GβL in regulating network behaviours. Modelling and experimental validation confirms a biphasic dependency between mTORC1 and Sin1, which may help explain context-specific biological observations in cells with low and high levels of Sin1.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Suzanne-Sindi/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Suzanne  Sindi
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UC Merced</em>
						    
						    
						    
					    		
					    			<h6>"Multiscale Modeling of Protein Aggregation"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Protein aggregation poses major health challenges for humans, but instead confer beneficial phenotypes for yeast. However, experiments on yeast systems necessarily involve multiple scales cell/colony. In this talk, I describe efforts towards linking these scales with mathematical models.
More specifically, propagon counting assays reflect the intracellular amplification process of prion aggregation coupled with cell-division. We model these experiments with generation and aggregate structured population models. Through an inverse problem formulation we attempt to infer the intracellular replication rates of these aggregates.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Tatiana-Yakushkina/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Tatiana  Yakushkina
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>NRU Higher School of Economics</em>
						    
						    
						    
					    		
					    			<h6>"Modified Replicator Systems with Ecological DIversification"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In this study, we construct new variations of replicator systems, which include experimentally observed properties of living systems. First, we analyze a quasispecies system with niche diversification and mutator effect.  We examine the case with different fitness landscapes in the first and second habitats and migration flow between them.  It is shown that such systems have rich phase structure, governed by the transition and mutation rates. Second,  we focus on such modifications of classical models of microbiological evolution that include explicit specification on nutrition type. For the cases with one and multiple common nutrients, the evolutionary dynamics of the population is discussed.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Thi-Minh-Thao-Le/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Thi Minh Thao  Le
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Université de Tours</em>
						    
						    
						    
					    		
					    			<h6>"Quasi-neutral dynamics in a co-infection system with N strains"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Understanding the interplay of different traits in a co-infection system with multiple strains has many applications in ecology and epidemiology. Because of high dimensionality and complex feedbacks between traits, manifested in infection and co-infection, the study of such systems remains a challenge.  In the case where the strains are similar (quasi-neutrality assumption), we can model trait variation as perturbations in parameters, which simplifies analysis. Applying perturbation to many parameters at the same time is mathematically not easy. In this study, we advance in this direction. We consider and study such a quasi-neutral model of susceptible-infected-susceptible (SIS) type with N strains and variation in transmission, clearance, and co-colonization traits. The slow-fast dynamics and the Tikhonov's theorem are essential approaches that we use to analyze the system, under the perspective of the replicator equation, where the variables are frequencies of N strains. Coefficients of this replicator system, that inherently are pairwise invasion fitnesses of strains, characterize not only pairwise outcomes but also determine collective behavior. We illustrate the model framework by investigating particularly dynamics with two strains (N=2), and explicitly analyzing different fitness dimensions and their interplay for maintenance and stabilization of diversity.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Thomas-N-Vilches/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Thomas N Vilches
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UNICAMP-Brazil</em>
						    
						    
						    
					    		
					    			<h6>"Assessing the effects of the diagnostic methods on the schistosomiasis dynamics"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Schistosomiasis is a neglected tropical disease that affects around 200 million people worldwide. It is a macroparasite infection, caused by trematode from genus Schistosoma, whose intermediate host is a snail from genus Biomphalaria that is caracteristic of places with fresh water. Moreover, it is estimated that, every year, 100 thousand individuals die due to schistosomiasis-related causes. In Brazil, schistosomiasis is endemic in 13 states and affects around 25 million people that live in risk areas, with 6 million infected individuals (estimated).
Its biological cycle is very complex, having five different life stages: egg, miracidium, sporocyst, cercaria and schistosomula, which makes its control even more difficult. The recomended diagnostic method, named Kato-Katz, seeks for eggs in the individual's feces and does not have a high sensitivity. In 2007, a group of researchers developed a new method, named Helmintex, that uses paramagnetic markers to find the eggs in feces and showed to be three times more sensitive than Kato-Katz.
We sought to investigate the effects on the schistosomiasis dynamics of applying a mass diagnostic strategy, the idea is that infected individuals, who are diagnosed, are treated. In order to do that, we build an ordinary differential equations model that considers: a human population divided in susceptible individuals and three classes of infected people that represent different levels of worm burden; a snail population, susceptible and infected, and a miracidium reservoir, that is important in order to take in to account the reinfection effects of the highest level of worm burden. The cercaria dynamics is implicitly considered through the human infection parameters.
We performed the equilibrium and local stability analysis for different scenarios in order to compare the results. (i) First, considering that a more sensitive diagnostic method, the Helmitex, is applyed, setting to zero the human population in the two highest levels of worm burden; (ii) considering that a less sensitive method, the Kato-Katz, is applyed, setting to zero only the highest level of worm burden; and the last case, (iii) the complete model that represents the scenario in which there is no treatment/diagnostic strategy.
Our results, besides the conditions for existence and local stability of the endemic equilibrium points, suggest that the low sensitivity of the classic method can explain the why it is so difficult to control the infections and why, usually, after stop the treatment on a population, or precaution strategies, the infection prevalences returns to a high level.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Tiffany-Leung/">
						    	<h4><li class="subgroup-counter subgroup-counter-MEPI"></li>
							    	
							    	Tiffany  Leung
							    		(<span class="text-MEPI">MEPI</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Fred Hutch</em>
						    
						    
						    
					    		
					    			<h6>"Evaluating the effectiveness of social distancing interventions: delaying the epidemic or flattening the curve?"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			In March 2020, the World Health Organization declared coronavirus disease a pandemic. We used a mathematical model to investigate the effectiveness of social distancing interventions in a mid-sized city. Interventions reduced contacts of adults >60 years of age, adults 20–59 years of age, and children <19 years of age for 6 weeks. Our results suggest interventions started earlier in the epidemic delay the epidemic curve and interventions started later flatten the epidemic curve. We noted that, while social distancing interventions were in place, most new cases, hospitalizations, and deaths were averted, even with modest reductions in contact among adults. However, when interventions ended, the epidemic rebounded. Our models suggest that social distancing can provide crucial time to increase healthcare capacity but must occur in conjunction with testing and contact tracing of all suspected cases to mitigate virus transmission.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vahini-Reddy-Nareddy/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Vahini Reddy  Nareddy
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Massachusetts Amherst</em>
						    
						    
						    
					    		
					    			<h6>"Representing spatial ecological oscillators by dynamical Ising model with memory"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Spatial synchronization in many biological systems are known to develop from short-range interactions of local oscillators. Locally-coupled ecological oscillators with noise and two-cycle behavior undergo a phase transition from incoherence to synchrony. These phase transitions exist in the Ising universality class, ensuring that the stationary properties of the ecological systems can be replicated by the simple Ising model. The universal properties shared by all the models in the universality class match that of the Ising model. Here we are interested in studying the dynamical properties shared between the ecological oscillators and the Ising model as synchronization is a dynamic phenomenon. We show that we need to go beyond the simple Ising model with nearest neighbor coupling and add a memory term to explain the tendency of local oscillators to maintain their phase of oscillations. We infer the Ising parameters using maximum likelihood methods by representing the ecological oscillators with the dynamical Ising model with memory. This correspondence to the dynamical Ising model is useful as it reveals that the spatial properties arise independent of local dynamics and the Ising parameters play a clear role in both understanding and predicting the dynamics of the ecological system. We study the location of phase transition in Ising parameter space and the ability of the dynamical Ising model to predict the future dynamics. We find that the simple dynamical Ising model is reasonable good at representing the ecological oscillators. This agreement between the dynamics of spatially-coupled ecological oscillators and the dynamical Ising model suggests the potential for simplification of many complex biological systems.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vandana-Revathi-Venkateswaran/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Vandana  Revathi Venkateswaran
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em></em>
						    
						    
						    
					    		
					    			<h6>"The effect of parental investment on immunocompetence and sexual immune dimorphism"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Sexes of a species show different characteristics beyond the differences in their sex- ual organs; this is known as sexual dimorphism and applies to immunocompetence as well. Immunocompetence is the ability of an individual to mount an immune response when exposed to pathogens. Females are shown to have increased longevity that comes with higher immunocompetence as compared to males and this may also lead to an increased probability of autoimmune disease in females. However, for some species such as pipefishes and seahorses belonging to the Syngnathid family, studies show that the males have a higher immunocompetence. Experimental evidences suggest that this could be due to the fact that these males undergo pregnancy i.e. the males have brood pouches where the eggs are fertilized; the fathers provide oxygen and nutrition to their offspring until they give birth to the juveniles. Therefore, an increase in immunocompetence may also be related to the amount of parental investment. In this study, using state dependent life-history theory, we show that for most species systems it is optimal to invest more in immunocompetence when the time spent in parental investment is longer. Our findings also show that an increase in parental investment brings about an earlier immunosenescence i.e. the gradual deterioration of the immune system that occurs with aging. We observe that an increase in investment towards immunocompetence is more pronounced in short-lived species with long brooding periods whereas species with a longer lifespan allocate more reserves towards offspring production. Our model also accounts for intraspecies scenarios: if a sex spends a longer fraction of its reproductive season in pregnancy or brooding (as compared to the other sex), then we find that this sex would invest more towards immunocompetence.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vedang-Narain/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Vedang Narain
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indiana University</em>
						    
						    
						    
					    		
					    			<h6>"A boids-based model of collective cell migration in wound healing"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			The re-epithelialization of a wound is a critical phase in the healing process, the disruption of which can lead to hypertrophic scarring. An accurate simulation of collective eukaryotic cell migration is critical for the development of computational models of wound healing. Using an agent-based model, we explored the feasibility of leveraging modified boids mechanics and adaptive proliferation rules to replicate contact-based locomotion and inhibition. Cells in our centre-based in silico simulation migrated from the edges of the cutaneous wound and successfully restored 'skin' integrity. When compared to simulations incorporating random walk movements, our boids-based model appeared to generate an improved qualitative approximation of in vivo observations. These simple rules may be useful for replicating various instances of collective cell migration.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vehpi-Yildirim/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Vehpi Yildirim
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Erzurum Technical University</em>
						    
						    
						    
					    		
					    			<h6>"Mathematical Modeling Postprandial Lipoprotein Metabolism and Investigating the Effects of the Bariatric Surgery"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Obesity has become one of the most serious public health issues over the past decades. Dyslipidemia, which is characterized by elevated plasma triglyceride-rich lipoprotein levels and disrupted plasma cholesterol profiles, is a major health risk associated with obesity. Bariatric surgery is one of the most effective methods for treating obesity. In addition to a significant weight loss, surgery induces remarkable improvements in plasma lipid profiles and insulin sensitivity indices. Even though the improving effects of bariatric surgery on the plasma lipid profiles and lipoprotein metabolism are well recognized, due to the complex nature of metabolism, the underlying mechanism is not fully understood. Lipoproteins are complex biochemical assemblies of lipids and apoproteins that transport water-insoluble triglycerides and cholesterol from the liver and intestines to the peripheral tissues. Studies show that lipoprotein metabolism is regulated by insulin and different lipoprotein species compete for the same clearance pathways in the circulation. The complexity induced by these regulatory mechanisms, interactions and feedbacks make computational models very effective for investigating lipoprotein metabolism.


In this study, we introduce a physiologically based multicompartmental model of hepatic and intestinal lipoprotein metabolisms. The model is designed to utilize stable isotopic enrichment and biochemical concentration data that has been collected during a mixed meal test. Hence, unlike several other models in the literature, the current model enables estimating metabolic parameters under non-steady-state conditions. An insulin module has been incorporated into the model to explore insulin-mediated regulations by utilizing available insulin data. The gastrointestinal module is designed to simulate the anatomical changes induced by gastric bypass surgery. This way, the model can comparatively analyze pre and post-surgery data to better understand the improvements induced at each metabolic pathway following the surgery. Finally, we test our model with pre- and post-surgery clinical data that has been collected from patients that went through Roux-en-y gastric bypass surgery. Our results indicate that, after the surgery, postprandial plasma lipoprotein clearance is significantly increased. Another key finding is that insulin-mediated stimulation of lipoprotein clearance is ameliorated. Furthermore, measured insulin responsiveness indices are significantly correlated with model estimates. Work done with
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Viktor-Zenkov/">
						    	<h4><li class="subgroup-counter subgroup-counter-IMMU"></li>
							    	
							    	Viktor Zenkov
							    		(<span class="text-IMMU">IMMU</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Tennessee, Knoxville</em>
						    
						    
						    
					    		
					    			<h6>"A minority of liver-resident CD8 T cells searching for Plasmodium-infected hepatocytes demonstrate difficult-to-detect attraction"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Malaria is a disease caused by parasites from genus Plasmodium that causes over 200 million infections and kills over 400,000 people every year. A critical step of malaria infection is when mosquito-injected sporozoites travel to the liver and form liver stages. Several malaria vaccine candidates induce high levels of Plasmodium-specific CD8 T cells which are able to eliminate all liver stages, thus providing sterilizing immunity against the disease. However, how CD8 T cells locate the site of infection is not well understood. We generated and analyzed data from intravital microscopy experiments in mice in which movement of T cells relative to the liver stage was recorded in several different settings. To detect attraction of T cells towards the infection site, we developed a novel metric based on the Von Mises-Fisher (VMF) distribution, which is more powerful than previously used metrics. We found that the majority (85-95%) of Plasmodium-specific CD8 T cells and T cells of irrelevant specificity did not display attraction towards the parasite when the parasite was not found by T cells, which was consistent with the random search for infection. In contrast, when some T cells located the parasite and formed a cluster, a minority of other T cells did display strong attraction towards the infection. Interestingly, the speed of T cell movement (and small turning angles) correlated with the bias of T cell movement towards the infection site (while many other parameters do not), suggesting that a deeper understanding of what determines the speed of T cell movement in the liver may help with improving T cell vaccine efficacy. Stochastic simulations suggested that a small movement bias towards the parasite dramatically reduces the number of CD8 T cells needed for the complete elimination of all malaria liver stages, and yet, to detect such attraction by individual cells requires extremely long imaging experiments which may not be currently feasible. Our developed methodology can be allied to detect weak attraction of moving agents in other conditions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vitor-de-Oliveira-Sudbrack/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Vitor de Oliveira Sudbrack
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>IFT-UNESP</em>
						    
						    
						    
					    		
					    			<h6>"Population dynamics in highly fragmented landscapes"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Human  action  fragments  the  natural  habitat  of  several  species  all  around the world.  Understanding the effects of fragmentation to ecosystems is key to elaborate the best policies to avoid species extinctions.  Therefore,  it is important  to  study  how  the  populations  and  ecosystems  respond  to  these kinds of changes in landscapes.  In this work, we use numerical methods to simulate reaction-diffusion equations in artificial landscapes generated with different structural distributions while keeping the total amount of habitat constant.  This guarantees we are observing phenomena caused by fragmentation per se. We  discuss  the  net  effects  of  fragmentation  into  the  steady total population.  In order to do that correctly, we established the correlation between fragmentation metrics with fixed amount of habitat, to ensure that  conclusions  are  not  biased  by  interdependencies  of  metrics.   We  have also analytically calculated the critical size to allow population growth for bidimensional landscapes within our model, with given symmetries.  These results prove that habitat area is not the only factor when it comes to population settling, and hence patch shape matters.  Recent explorations on our model revel the presence of different movement scales, intra-patch and inter-patches.   Future  prospects  of  this  project  are  studying  how  fragmentation affects  features  of  population  spatial  distribution  and  investigations  about regimes of fragmentation that allow non-interactive (or weakly interacting) subpopulations to form.  We will also explore consequences of fragmentation to communities.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vivian-Dornelas/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	Vivian  Dornelas
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>IFT-UNESP</em>
						    
						    
						    
					    		
					    			<h6>"Impact of the landscape heterogeneity on the spatial organization of a single-species population"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			It is common to observe in nature the emergence of collective behavior in biological populations, such as pattern formation. In this work, we are interested in characterizing the distribution of a single-species population (such as some bacteria or vegetation), based on mathematical models that describe the spatio-temporal evolution of the density, governed by elementary processes, such as dispersion, growth, and nonlocal competition by resources. Using a generalization of the FKPP equation, we study the role that a heterogeneous environment has in the spatial organization of a population. We investigate the structures that emerge near the border from one environment to the other. We found that, depending on the shape of nonlocal interaction and other model parameters, three diﬀerent proﬁles can emerge from the interface: sustained oscillations (or spatial patterns, without amplitude decay); attenuated oscillations (with amplitude decreasing from the interface); exponential decay (without oscillations) to a flat proﬁle. We related the wavelength and the rate of decay of oscillations with the parameters of the interaction (characteristic length and form of decay with distance). We discussed how the heterogeneities of the environment allow access to information about the biological phenomena of the system, hidden in the homogeneous case, such as those that mediate competitive interactions.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Vojtech-Kumpost/">
						    	<h4><li class="subgroup-counter subgroup-counter-OTHE"></li>
							    	
							    	Vojtech  Kumpost
							    		(<span class="text-OTHE">OTHE</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Karlsruhe Institute of Technology</em>
						    
						    
						    
					    		
					    			<h6>"Elucidating Mechanisms Underlying Experimentally Induced Changes in the Zebrafish Circadian Clock"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Zebrafish embryonic cell lines are a unique experimental model of the circadian regulation in vertebrates. They can be directly paced by external light stimuli and the oscillatory dynamics of gene expression can be monitored by luciferase reporter assays. These assays are designed to convert transcriptional activation to bioluminescence with an excellent time resolution and obvious changes in the measured waveforms can be observed as a result of light stimuli, gene knockouts or drug treatment. Here, we aim to utilize mathematical modeling to clarify how those changes in the recorded data correspond to the specific cellular mechanisms that generate them. We start with a simplified ODE Kim-Forger model proposed for the mammalian clock. This model consists of three state variables connected in a single negative feedback loop, which represents the core mechanism of the circadian clock mechanism. The model is further adjusted to correspond to the specifics of the zebrafish circadian regulatory system. Next, we validate the proposed model on a previously published data set with a variety of light-pacing regimes. In our further work, we will use stochastic modeling to account for molecular noise on the single-cell level that affects the observed dynamics of the recorded cell population. Finally, we plan to quantify the effect of a variety of drug treatments by fitting the model parameters to the drug-treated cell cultures and exploring the parameter space that can explain such variations. Our work represents a novel approach to studying bioluminescence recordings of zebrafish embryonic cell lines and aims to better quantify the observed differences due to drug treatments.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Will-Leone/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Will Leone
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UTK</em>
						    
						    
						    
					    		
					    			<h6>"Understanding factors contributing to bacterial burden in granulomas of Mycobacterium tuberculosis-infected monkeys"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Mycobacterium tuberculosis (Mtb) is the bacterium that causes tuberculosis (TB) and kills more people per year than any other infectious agent. Factors that influence the spread of Mtb within individuals are an ongoing topic of investigation. We analyzed novel data from 25 Mtb-infected rhesus macaques on the number of colony forming units (CFUs) in individual granulomas in lungs of the animal. In these experiments, macaques were infected with different initial doses (varying between 1-40 CFUs); animals underwent different Mtb-controlling treatments, and measurement of lung granuloma CFUs were done at different time points after the infection. We found that a higher initial dose resulted in a larger average number of CFUs per granuloma when comparing macaques given an initial dose of 8 or 40 CFUs. The caveat in this comparison, however, is that time since infection and controlling treatments were varied between the doses. Interestingly, we found that variability in CFU/granuloma (estimated using coefficient of variation, CV) is higher among all animals as compared to CV estimated for individual animals. This suggests that infection dynamics in granulomas of a given animal proceeds more similarly than infection between two randomly chosen granulomas in two different animals. This result challenges the commonly stated hypothesis that dynamics of Mtb in individual granulomas in one animal are independent. Our analysis also suggested that the CFU/granuloma in macaques is dependent on a combination of the initial dose of Mtb, treatment, and time since infection. These results lead further research into assessing the relative contributions the dose and time since infection have on TB infection in the lungs of the macaque and is the focus of ongoing research.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/William-(Bill)-Sherwin/">
						    	<h4><li class="subgroup-counter subgroup-counter-POPD"></li>
							    	
							    	William (Bill)  Sherwin
							    		(<span class="text-POPD">POPD</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UNSW Sydney</em>
						    
						    
						    
					    		
					    			<h6>"Monitoring and Forecasting Diversity: Entropy Unifies Molecules and Ecosystems"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			At all scales from molecules to ecosystems, we measure biodiversity to indicate outcomes of natural changes or threatening processes, so that we can compare these with forecasts under various management schemes.   Every biodiversity level has four basic processes – dispersal, adaptation, random change, and generation of novel ecosystems, species, or genetic variants.  How can we exploit this similarity?  Entropy is an obvious choice, being a general forecasting and measurement tool throughout science.  It is also a simple transform of the biodiversity-measure ‘profile’:  Richness; Gini-Simpson; and Shannon.  Conservation managers mostly use Richness and Shannon for biodiversity measurement, and have some forecasts for MaxEnt (Shannon) and Simpson - so there is a mismatch between what is forecast and measured.  In contrast, measures and forecasts in molecular ecology are now well developed for the entire profile of biodiversity-measures, within and between areas (Trends Ecol.Evol. 32:948).  Shannon approaches outperform others in some important tasks, such as tracing rangeshift or invasion, and genetic estimates of dispersal for input to metapopulation models.  Thus the stage is set to unify our monitoring and forecasting of these four processes that are common across all biodiversity levels, using a complete diversity profile that encompasses Richness, Shannon and Simpson.  This will integrate well with the many entropic methods in studies of the physical environment.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/William-D-Martinson/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	William D Martinson
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Oxford</em>
						    
						    
						    
					    		
					    			<h6>"Comparative analysis of continuum angiogenesis models"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			While discrete approaches are increasingly used to model biological phenomena, it remains unclear in such frameworks how complex population-level behaviours emerge from the rules used to describe interactions between individuals. Insight may be gained by deriving coarse-grained continuum models, which describe the mean-field dynamics of a discrete model. Differential equations derived from such discrete-to-continuum approaches, however, often contain nonlinearities that depend on microscopic rules in the discrete model, and there has been little work done to analytically compare these coarse-grained equations with those constructed from simpler phenomenological frameworks. We address this problem in the context of angiogenesis (the creation of new blood vessels from existing vasculature). We compare asymptotic solutions of a classical, phenomenological 'snail-trail' partial differential equation (PDE) model for angiogenesis with those of a more complicated, fully nonlinear PDE system derived via a systematic coarse-graining procedure. For distinguished parameter regimes corresponding to chemotaxis-dominated cell movement and low branching rates, both continuum models reduce at leading order to an identical system of PDEs. Numerical and analytical results confirm that solutions to the two continuum models are in good agreement if these conditions hold, which allows us to determine when we can use the simpler model to capture the results of a more complicated coarse-grained system that describes the same biological process.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Xinzhe-Zuo/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Xinzhe Zuo
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>UCLA</em>
						    
						    
						    
					    		
					    			<h6>"Backtracking in RNA polymerization"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Backtracking of RNA polymerase (RNAP) is an important pausing mechanism during DNA transcription that is part of the error correction process that enhances transcription fidelity. We model the backtracking mechanism of RNA polymerase which usually happens when the polymerase tries to incorporate a mismatched nucleoside triphosphate. Previous models have made assumptions for easier calculations. One of the key assumptions made is that there is no trailing polymerase behind the backtracking polymerase or the trailing polymerase remains stationary when the leading polymerase backtracks. We derive analytic solutions for a stochastic model that allows for locally interacting RNAPs to explicitly show how a trailing RNAP influences the probability that an error is corrected or incorporated by the leading backtracking RNAP. We also provide a related method for computing the statistics of the times to error correction or incorporation given an initial local RNAP configuration. Our model and the associated results provide the components needed in more complete multi-RNAP descriptions. For example, all RNAPs along a transcript may be considered using exclusion processes such as the TASEP model that has been used to describe mRNA. In the many-body picture, one would be able to address multiple, simultaneously stalled RNAPs on how their interactions affect their probabilities of correction or incorporating an error. A competition between transcription fidelity and RNA production rate would be expected to arise
and will be the subject of future investigation.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Yafei-Wang/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Yafei  Wang
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>Indiana University Bloomington</em>
						    
						    
						    
					    		
					    			<h6>"Multicellular simulation in cancer treatment with nanoparticles"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Cancer is a complex systems problem that involves tumor cells and their microenvironment. Recently, research shows that engineered nanomedicine is playing an important role in cancer treatment. The traditional experimental methods involve intensive cost and time investments, as well as many operational challenges. With the increasing power of high throughput computing, it has become feasible to explore a vast variety of therapeutic designs in multicellular systems with computational modelling. In this talk, we propose an agent-based model (with PhysiCell) to investigate the therapeutic designs of cancer treatment with nanoparticles (NPs), where NPs uptaken or internalization, drug release and drug effects on tumor cells are explored. Our simulation studies show that drug-loaded nanoparticles have some allow promising new options for cancer therapy, and the point to the power of using large-scale model exploration to tune and improve therapy. In particular, we introduce a novel tracking of nanoparticle populations in each individual cells, allowing better modeling of drug release by internalized nanoparticles and long-term therapeutic implications. In this talk, I will cover the following parts:(1) how cells uptake NPs; (2) how NPs release drug inside of cells; (3) how daughter cells inherit NPs and drug from their parents; (4) how released drug treats tumor cells.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Yahe-Yu/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Yahe Yu
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>NC State</em>
						    
						    
						    
					    		
					    			<h6>"Finding Optimal STI Strategies for HIV Using Fitted Q Algorithm with XGBoost Regression"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			We are trying to use the fitted Q algorithm with XGBoost regression to compute the optimal structured treatment interruption strategies for HIV infected patients. By using the combination of these two methods, we showed that this approach is computationally more efficient, at the same time the optimal results were obtained using fewer training data. And the speed of convergence of the optimal strategy is much fast than the Extra-tress regression method used in the fitted Q algorithm.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Yeeren-Low/">
						    	<h4><li class="subgroup-counter subgroup-counter-MFBM"></li>
							    	
							    	Yeeren Low
							    		(<span class="text-MFBM">MFBM</span>)
						    	</h4>
					    	</a>
						     
						    	<em>McGill University</em>
						    
						    
						    
					    		
					    			<h6>"Data-driven modeling of T cell morphodynamics during migration"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			T cells can spontaneously migrate rapidly through the extracellular matrix using an 'amoeboid' mechanism, which is believed to aid their search for antigens. While modeling and experiments have in part addressed motility parameters and the impact of matrix characteristics, the morphodynamics during locomotion remains not well understood. We consider a data-driven approach using low–spatial resolution time-lapse fluorescence microscopy videos of activated T cells migrating in collagen matrix or under agarose gel. We analyze the resulting cell shapes by using an autoencoder to extract a low-dimensional 'shape space'. We find time-irreversible motion in shape space, as expected from Purcell's theorem for a swimmer at low Reynolds number. We also find evidence of distinct signatures of turning behavior as a result of cell–matrix interactions. Our statistical analysis allows us to generate artificial trajectories of cells and their shapes using a coarse-grained morphology. This approach confers the possibility of inferring predictive dynamical laws which would inform biophysical models of cell morphology during migration and interaction with the environment.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Zeinab-Tajik-Mansoury/">
						    	<h4><li class="subgroup-counter subgroup-counter-NEUR"></li>
							    	
							    	Zeinab Tajik Mansoury
							    		(<span class="text-NEUR">NEUR</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of Tehran</em>
						    
						    
						    
					    		
					    			<h6>"Opioid Addiction Affects Neuronal Synchronization in the Hippocampus: A Computational Model"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			Drug addiction can affect the limbic system. Many computational models for drug addiction have been proposed, but most are for the reward system and behavioral models (Redish et al.,2004 and Gutkin et al.,2006). There are a few cellular mathematical models for drug addiction in the hippocampus that are very detailed (Borjkhani et al.,2018). We study a functional model for the synapses in the hippocampus to investigate the effect of opioid (Morphine) addiction on neuronal synchronization. We consider Pankratova et al.,2019 computational model. The model considers an astrocyte in synapses of two postsynaptic neurons in the hippocampus and studies the synchronization of the two neurons. In the model, we consider that Morphine addiction affects this tripartite synapses. The model consists of six differential equations for each neuron. One of the equations expresses the dynamics of the mean-field amount of released neurotransmitters based on presynaptic Poisson spike train and the astrocytic released glutamates in the synaptic cleft. Also, there is one differential equation for the mean-field amount of excitatory postsynaptic currents(EPSCs) based on presynaptic Poisson spike train and the D serine released from astrocyte. The four other equations indicate the Hodgkin-Huxley model (one of them for membrane voltage and the three other for gating variables), which its synaptic current input has been created by integrating EPSCs. Two differential equations describe the function of the astrocyte existed in synapses of the two neurons. One equation indicates the dynamics of the mean-field amount of astrocytic released glutamates based on the mean-field amount of the neurotransmitters of both neurons. The other equation expresses the dynamics of the mean-field amount of astrocytic released D serine based on the mean-field amount of the neurotransmitters of both neurons. Morphine addiction can affect three positions in synapses: presynaptic, astrocytic, and postsynaptic cells. It can cause an increase in neurotransmitters' concentration due to the disinhibitory mechanism of opioid receptors on presynaptic inhibitory neurons. Also, it can decrease the activity of astrocytic transporters causing fewer neurotransmitters reuptake by astrocyte. Morphine activates the opioid receptors on the postsynaptic neuron that increase NMDA's currents. In the model, we consider that Morphine influences the frequency of the Poisson spike train, the steady-state amount of neurotransmitters, and the gain of astrocytic glutamates. The amplitude of the EPCSs is also affected due to the Morphine addiction. The coefﬁcient of synchronization is the ratio of synchronous spikes to the total spike number of both neurons. The results indicate that morphine addiction increases the synchrony of neurons. Thus, it can represent memory formation. In future works, we will study the withdrawal state in this model to analyze the neuronal synchronization and how the cues can result in relapse and how to control it.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
				
					
					
						
						
					
					    
					    	
					    	<a href="/Zviiteyi-Chazuka/">
						    	<h4><li class="subgroup-counter subgroup-counter-ONCO"></li>
							    	
							    	Zviiteyi Chazuka
							    		(<span class="text-ONCO">ONCO</span>)
						    	</h4>
					    	</a>
						     
						    	<em>University of South Africa</em>
						    
						    
						    
					    		
					    			<h6>"A mathematical model for in host dynamics of an immune evading virus"</h6>
					    		
					    	
						    
						    
					    	
					    	<article class="article-post text-justify" style="font-size:12px;">
						    	
						    		
						    			High risk human papillomaviruses HPV types 16,18,31,45 are one of the major causative agents of cervical cancer in women globally and it is estimated that about 80% of women are infected by HPV mainly due to sexual activities within their life time. Out of these infections and re-infections some develop into persistent infections that lead to cancer lesions while some can be cleared provided they are detected by the immune system. Immune response within the body plays a pivotal role in clearing most infections that constantly affect us. Interestingly viruses such as HPV are seemingly very 'clever' in concealing their presence within cells as they devise many ways of avoiding detection by the immune system and therefore manage to create an anti-inflammatory micro environment. This leads us to interesting mathematical modelling research of a little 'clever immune evading virus'. We create a mathematical model for the dynamics of HPV in the presence of immune response and rigorous mathematical calculations show that there exist three equilibrium points whose stability both local and global is shown. An investigation into the probable possibility of a bifurcation is done using the centre manifold theory. Results show that a forward bifurcation exists and hence the endemic equilibrium is locally asymptotically stable provided that the reproduction number (R0) is less than unity and unstable otherwise. Numerical simulations prove and support the theoretical work presented. We also establish that HPV can be eliminated from the body when R0<1 and persistence occurs either when there is immune response evasion R0>1, Rk<1 where (Rk) is the immune response reproduction number or when there is immune response R0 >1, Rk>1.  It is envisaged that the results of the study will be used further on to analyse the epidemiological link within the complex dynamics of HIV/HPV in the presence of stochastic perturbations, which is the core of the PhD work.
						    		
						    	
						    	 
									
								</article>
								</br></br>
					     <!-- end presenter -->
				     <!-- end addition -->
					    
					
				
			
			
			</ol>
								
							      
			<!-- Author Box -->
      				
			<div class="row mt-5">
				<div class="col-md-2 align-self-center">
                       
                      <img class="rounded-circle" src="../assets/images/logo.jpg" alt="eSMB2020" width="90"/>
                       
				</div>
				<div class="col-md-10">		
                      <h5 class="font-weight-bold">Hosted by eSMB2020 <span><a target="_blank" href="https://twitter.com/ESmb2020" class="btn btn-outline-success btn-sm btn-round ml-2">Follow</a></span></h5>
					Virtual conference of the Society for Mathematical Biology, 2020.					
				</div>
			</div>				
      
      
      <!-- Comments -->
      
          <!--  Don't edit anything here. Set your disqus id in _config.yml -->

<div id="comments" class="mt-5">
    <div id="disqus_thread">
    </div>
    <script type="text/javascript">
        var disqus_shortname = 'smb2020'; 
        var disqus_developer = 0;
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = window.location.protocol + '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>
    Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a>
    </noscript>
</div>
      
            
		</div>
        
        
	</div>
</div>


<!-- Aletbar Prev/Next -->
<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
          
          
        </div>
    </div>
</div>

    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    

    <!-- Footer <div class="row justify-content-between"> -->
    <footer class="bg-white border-top p-3 text-muted small">
      <div class="container">
        <div class="row">
          <div class="col-12">
	          	
	          	<span class="navbar-brand mr-2 mb-0"><strong>SMB 2020 Sponsors</strong></span></br>
	          	<img class="w-100" src="/assets/images/sponsors.png" alt="SMB Sponsors">
                
          </div>
        </div>
        
        </br>
        
        <div class="row">
          <div class="col-12"> 
						<span class="navbar-brand mr-2 mb-0"><strong>Organizing Committee</strong></span> 
          </div>
        </div>
        
        </br>
        <div class="row">
	        	        
          <div class="col-4"> 
	           
						<ol class="list-featured">
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Heiko Enderling (Co-Chair)
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">H. Lee Moffitt Cancer Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Amber Smith (Co-Chair)
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Tennessee Health Science Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Sandy Anderson
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">H. Lee Moffitt Cancer Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
				  	
				  								        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Ruth Baker
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Oxford, UK<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        
			        
						  </ol>
          </div>
            
          <div class="col-4">
            <ol class="list-featured-second">	
	            <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Denise Kirschner
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Michigan, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Torbjörn Lundh
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">Chalmers and University of Gothenburg, Sweden<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>					        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  John Nardini
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">NC State University, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Marissa Renardy
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Michigan, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			      </ol>
          </div>		
          
          <div class="col-4">
	          <ol class="list-featured-second">	
	          
	          	<li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Santiago Schnell
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">University of Michigan, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Elissa Schwartz
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">Washington State University, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
			                  Jeffrey West
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">H. Lee Moffitt Cancer Center, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
			        
			        <li class="mb-4">
			          <span>
			              <h6 class="font-weight-bold" style="margin-bottom: -1px;">
	                  		Ping Ye 
			              </h6>
			              <span class="d-block text-muted">
			                  <span class="catlist text-capitalize text-muted smoothscroll">South Dakota State University, USA<span class="sep"></span></span>
			              </span>
			          </span>
			        </li>
	          </ol>
	          
	          <span>Copyright SMB © <script>document.write(new Date().getFullYear())</script>.</span>
          </div>
                
        </div> <! -- end row justify-content-between --> 
      </div>
    </footer>

    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
