---
layout: group
author: esmb
day: "Monday"
group: "CT1"
subgroup: "CDEV"
title:  "Subgroup Contributed Talks"
author1: "Nathan Weinstein"
author2: "Aden Forrow"
author3: "Claus Kadelka"
inst1: "UNAM"
inst2: "University of Oxford"
inst3: "Iowa State"
title1: "A computational model of the differentiation of endothelial cells into macrophages"
title2: "Learning developmental trajectories from CRISPR lineage tracing and single-cell gene expression"
title3: "Unraveling the Design Principles of Gene Regulatory Networks: a Meta-Analysis"
abstract1: "Endothelial cells (ECs), macrophages (MPs), and pericytes are the main cellular components of microvascular networks. The principal process that allows microvascular network remodeling and adaptation is angiogenesis. Angiogenesis begins when hypoxia or lack of nutrients triggers the release of a vascular endothelial growth factor (VEGF), which is typically VEGFA. The VEGF forms a gradient, and when the concentration of VEGF reaches a threshold around a segment of the blood vessel, it causes the segment to destabilize and to lose its mural cells. During the subsequent stage of sprouting angiogenesis, some of the ECs become tip cells that extend filipodia and migrate following the VEGF gradient. The ECs adjacent to the tip cells become stalk cells that elongate, proliferate, and trail the tip cell. Once the tip cell reaches another tip cell or blood vessel segment, an anastomosis forms establishing a new connection in the microvascular network. During anastomosis formation macrophages act as chaperones for the tip cell. Additionally, macrophages phagocytize the ECs that undergo apoptosis during microvascular pruning. Definitive hematopoiesis occurs first during early embryogenesis when a group of endothelial cells undergo the endothelial-to-hematopoietic transition (EHT) and become hematopoietic stem cells (HSC). Some of the HSCs become granulocyte monocyte precursor cells (GMPs). Certain GMPs differentiate into monocytes, and macrophages differentiate from monocytes.  In adult humans, ECs retain the capability to undergo EHT and we aim to study the potential of ECs to differentiate into macrophages and to elucidate the extracellular microenvironmental conditions that promote EC-to-macrophage differentiation. Our approach is to first assemble a molecular regulatory network based on the information available in the literature about the main molecules involved in the regulation of the process and the interactions between them. Then we will transform the model into a dynamic system in the form of a Boolean Network. Then, we simulate and analyze the dynamic behavior of the model. Afterward, we will validate the model by comparing the observed effect of the available relevant mutations with their simulated effect." 
abstract2: "Recent research has shown that the mathematical theory of optimal transport can effectively reconstruct developmental trajectories from time courses of single cell gene expression; however, this approach requires expensive experiments with fine time resolution. In this work, we present a novel framework that leverages new types of lineage-tracing information measured simultaneously with gene expression. These experimental techniques use heritable CRISPR-induced genetic barcodes to trace the history of cell divisions over the course of development. Crucially, these barcodes can be measured in the same cells as gene expression. Our method, designed for lineage-tracing time courses, learns from both kinds of information together using mathematical tools from graphical models, structural equation models, and optimal transport. We find that lineage data improves optimal transport’s effectiveness in disentangling complex state transitions with lower temporal resolution, thereby reducing experimental cost. Joint work with Geoffrey Schiebinger (University of British Columbia)"
abstract3: "Gene regulatory networks (GRNs), frequently modeled using Boolean networks, describe how a collection of genes governs the processes within a cell. Boolean networks are intuitive, simple to describe, and yield qualitative results even when data is limited. This talk will outline a research program aimed at harnessing the collective knowledge contained in hundreds of diverse Boolean GRN models in order to understand the impact of network structure and topology on network dynamics and stability, and in particular the role of canalization in gene regulation. The biological term canalization reflects a cell’s ability to maintain a stable phenotype despite ongoing environmental perturbations. Accordingly, Boolean canalizing functions are functions where the output is already determined if a certain, canalizing variable takes on its canalizing input, regardless of all other inputs. Using text- and data-mining techniques and the PubMed search engine, we generated an expandable database of published, expert-curated Boolean GRN models, and extracted more than 5, 000 rules governing these networks. A meta-analysis of all these rules confirmed a strong overrepresentation of certain types of canalizing functions. We also studied the abundance of small structures within the networks, called network motifs, and found significant differences between published GRNs but also when comparing to random networks. Furthermore, we analyzed the dynamical robustness of GRNs and found most of them to operate at a “critical threshold” between order and chaos. These findings highlight how our continuously-expanding database provides a versatile tool to identify the overarching design principles underlying gene regulation."
---

