---
layout: group
author: esmb
day: "Monday"
group: "MS1"
subgroup: "CDEV"
title:  "Subgroup Contributed Talks"
author1: "Andrew Krause"
author2: "Daniel Lobo"
author3: "Jason Ko"
author4: "Timothy Ostler"
author5: "Maria Abou Chakra"
inst1: "University of Oxford"
inst2: "University of Maryland, Baltimore County"
inst3: "University of Maryland, Baltimore County"
inst4: "Cardiff University"
inst5: "University of Toronto"
title1: "Matching Theory to Real Biology: Recent Progress and Open Questions in Turing's Theory of Morphogenesis"
title2: "A Turing system explains regeneration patterning and fission behavior in planaria"
title3: "Regulated Cell Adhesion Dynamics in a Continuous Model: Sorting, Intercalation, and Involution"
title4: "Choosing the best embryo in In-Vitro Fertilization"
title5: "Control of tissue development by cell-cycle dependent transcriptional filtering"
abstract1: "Turing's reaction-diffusion theory of morphogenesis has been enormously well-studied from a variety of perspectives. While incredibly successful in motivating enormous theoretical and experimental work, there are many open questions in elucidating specific aspects of Turing-type morphogenesis in real developmental settings. I will present recent work on developing new tools and perspectives on matching Turing's idealized theory with the complexity of real biological development. This includes recent extensions to the classical theory of linear instability analysis to account for heterogeneity in space and time, curvature, growth, as well as open boundary conditions combining Turing's theory with other ideas in understanding spatial pattern formation, such as positional information. While this extends and confirms intuitive insights gained from experiments and simulations over the past few decades, it raises an enormous number of questions regarding how far we can push such extensions. I will briefly mention some of these, hopefully stimulating broader perspectives on how to develop simple yet physically interpretable theories of pattern formation." 
abstract2: "Planarian worms have the extraordinary ability to regenerate any body part after an amputation. This ability allows them to reproduce asexually by fission, cutting themselves to produce two separated pieces each repatterning and regenerating a complete animal. The induction of this process is known to be dependent on the size of the worm as well as on environmental factors such as population density, temperature, and light intensity. Models based on Turing systems can explain the self-regulation of many biological mechanisms, from skin patterns to digit formation. Here, we combine experimental evidence with a modeling approach to show how a cross-inhibited Turing system can explain at once both the signaling mechanism of regeneration and fission in planaria. The model explains in a growing domain the precise signals that control the regeneration of the different body parts after amputations as well as when and where planaria fission, and its dependence on the worm length. We provide molecular implementations of the proposed model, which also explains the effects of environmental factors in the signaling of fission. In summary, the proposed controlled cross-inhibited Turing system represents a completely self-regulated model of the whole-body regeneration and fission signaling in planaria."
abstract3: "Cell-cell adhesion can dictate tissue growth and multicellular pattern formation and it is crucial for the cellular dynamics during embryogenesis and cancer progression. While it is known that these adhesive forces are generated by cell adhesion molecules (CAMs), the regulation of CAMs is not well understood due to complex nonlinear interactions that span multiple levels of biological organization–from genetic regulation to whole-organism shape formation. We present a novel continuous model using partial differential equations that can explain the dynamic relationships between genetic regulation, CAM expression, and differential adhesion. This approach can demonstrate the mechanisms responsible for cell-sorting behaviors, cell intercalation in proliferating populations, and the involution of zebrafish germ layer cells during gastrulation. The model can predict the physical parameters controlling the amplitude and wavelength of a cellular intercalation interface as shown in vitro. We demonstrate the crucial role of N-cadherin regulation for the involution and migration of cells beyond the gradient of the morphogen Nodal during zebrafish gastrulation. Integrating the emergent spatial tissue behaviors with the regulation of genes responsible for essential cellular properties such as adhesion will pave the way toward understanding the genetic regulation of large-scale complex patterns and shapes formation in developmental, regenerative, and cancer biology."
abstract4: "We aim to characterise the shape and size of thawing embryos after they have been cryopreserved during In-Vitro Fertilization (IVF). Through image segmentation techniques and data analysis, we seek to determine appropriate metrics that can predict pregnancy. We also model the increasing temperature of thawing embryos, determining the conditions that will prevent damage. This is work undertaken through an academia-industry grant at Cardiff University in collaboration with the London Women's Clinic. Joint work with: Katerina Kaouri, Thomas Woolley, Karl Swann (Cardiff University), Andrew Thompson, Giles Palmer (London Women’s Clinic), with funding from the KESS2 Scholarship. Knowledge Economy Skills Scholarships (KESS 2) is a pan-Wales higher level skills initiative led by Bangor University on behalf of the HE sector in Wales. It is part funded by the Welsh Government’s European Social Fund (ESF) convergence programme for West Wales and the Valleys.sector in Wales. It is part funded by the Welsh Government’s European Social Fund (ESF) convergence programme for West Wales and the Valleys."
abstract5: "A fundamental question in biology is how a single eukaryotic cell produces the complexity required to develop into an organism. Cell cycle duration changes dramatically during development, starting out fast to generate cells quickly and slowing down over time as the organism matures. The cell cycle may also act as a transcriptional filter to control the expression of long genes which can’t be completely transcribed in short cycles. Using mathematical simulations,  we discovered an inherent trade-off where fast cycling cells serve to increase cell number while slower cycling cells contribute to cell diversity by introducing genes in a controlled manner. Simulations show that cell-cycle duration can fine tune cell number, cell diversity and cell proportions in a tissue. Our predictions are supported by comparison to single-cell RNA-seq data captured over embryonic development. Our results support the idea that cell-cycle dynamics may be important for controlling gene expression and cell fate."
---

