---
layout: group
author: esmb
day: "Tuesday"
group: "CT1"
subgroup: "CDEV"
title:  "Subgroup Contributed Talks"
author1: "Jonathan Harrison"
author2: "Marcin Zagorski"
author3: "Tim Liebisch"
inst1: "University of Warwick"
inst2: "Jagiellonian University"
inst3: "Frankfurt Inst. of Advanced Study"
title1: "Hierarchical Bayesian modelling of chromosome segregation allows characterisation of a distinct dynamic signature of errors in cell division"
title2: "How is information decoded in developmental systems?"
title3: "Cell Fate Clusters in ICM Organoids Arise from Cell Fate Heredity & Division – a Modelling Approach"
abstract1: "Cells divide via a self-organising process known as mitosis where a crucial step is the high fidelity separation of duplicated chromosomes to daughter cells. Errors in segregating chromosomes during cell division are a hallmark of cancer and are associated with developmental syndromes. How cell division achieves high fidelity remains an outstanding question, in particular how errors are detected and corrected. Through automated tracking of chromosomes at fine spatio-temporal resolution over long timescales, we can produce detailed quantification of the behaviour of human cells during mitosis. We propose a force-based stochastic differential equation model, dependent on hidden states governed by a Markov process, to describe the oscillations and segregation of chromosomes in mitosis. By fitting this dynamic model to experimental data in a Bayesian framework, we can infer the timing of the metaphase-anaphase transition (chromosome separation) for each duplicated chromosome pair. By extending this to a hierarchical Bayesian framework, we are able to capture rare reversal events during anaphase in the model. Model comparison provides evidence that the hierarchical model with reversals is preferred over a model without reversals. Application of this computational modelling pipeline to experimental data allows characterisation of a distinct signature of model parameters related to lagging chromosomes, and subsequent correction of these errors by the cell."
abstract2: "The development of multicellular organisms is a dynamic process in which cells divide, rearrange, and interpret molecular signals to adopt specific cell fates. Despite the intrinsic stochasticity of cellular events, the cells identify their position within the tissue with striking precision of one cell diameter in fruit fly or three cell diameters in vertebrate spinal cord. How do cells acquire this positional information? How is this information encoded and how do cells decode it to achieve the observed level of cell fate reproducibility? These are fundamental questions in biology that are still poorly understood. In this talk, I will combine both information theory methods and mechanistic models to address these questions in the context of spinal cord development. I will consider the two opposing morphogen signals that are integrated to specify the arrayed pattern of neural progenitor domains that later on give raise to different type of neurons. Based on the maximum likelihood estimation rule I will define decoding map that provides predictions for shifts in the target gene domains in mutants. The predictions will be validated using experimental data obtained from naïve chick neural plate explants and from embryos with altered ventral morphogen signaling. I will present a simple model of a gene regulatory network that integrates the two morphogen signals and is sufficient to recapitulate the observed shifts in the target domains. I will investigate to what extent the level of noise in the input signals affects precision of the resulting gene expression pattern. Interestingly, the observed precision of gene expression pattern is close to the theoretical limit of precision of decoding of noisy signals."
abstract3: "Background: During the mammalian preimplantation phase, cells undergo two subsequent cell fate decisions. During the first cell fate decision, cells become either part of an outer trophectoderm or part of the inner cell mass. Subsequently, the inner cell mass (ICM) segregates into the epiblast and the primitive endoderm, giving rise to the embryo and the placenta respectively. Recently, ICM organoids have been published as an in vitro model system towards preimplantational development. ICM organoids mimic the second cell fate decision taking place in the in vivo mouse embryos. In a previous study, the spatial pattern of the different cell lineage types was investigated. The study revealed that cells of the same fate tend to cluster stronger than expected for the currently hypothesised purely random cell fate distribution. Three major processes are hypothesised to contribute to the final cell fate arrangements at the mid and late blastocysts or 24 h old and 48 h old ICM organoids, respectively: 1) intra- and intercellular chemical signalling; 2) a cell sorting process; 3) cell proliferation.  Methods & Results: In order to quantify the influence of cell proliferation on the emergence of the observed cell lineage type clustering behaviour, an agent-based model was developed. The model accounts for mechanical cell-cell interactions, cell growth and cell division and was applied to compare several current assumptions of how ICM neighbourhood structures are generated. The model supports the hypothesis that initial cell fate acquisition is a stochastically driven process, taking place in the early development of inner cell mass organoids. The model further shows that the observed neighbourhood structures can emerge due to cell fate heredity during cell division and allows the inference of a time point for the cell fate decision.  Discussion: Simulations based on the model show that cell divisions involving cell fate heredity seem sufficient to lead to the local clustering observed in 24 h old ICM organoids, and that the initial cell differentiation process takes place only during a small time window, during or prior to ICM organoid composition. Our results leave little room for extracellular signaling believed to be important in cell fate decision, therefore we are discussing an alternative role of chemical signaling in this process."
---