---
layout: group
author: esmb
day: "Tuesday"
group: "CT1"
title: "Other Contributed Talks"
subgroup: "OTHE"
author1: "Anne Talkington"
author2: "Tatiana Marquez-Lago"
author3: "Camile Kunz"
inst1: "Chapel Hill"
inst2: "UAB"
inst3: "Goethe U."
title1: "A PBPK model for clearance of PEGylated nanomedicines"
title2: "Mathematical methods for microbiome research"
title3: "Chemotaxis impact on pattern formationChemotaxis impact on pattern formation"
abstract1: "Physiologically based pharmacokinetic (PBPK) models are a means of conducting virtual experiments on a large scale as an alternative to extensive trials that would be prohibitively time-consuming, unethical, or otherwise costly.  PBPK can be used to compute and test optimal dosing strategies, among other features, of proposed treatments using known or learned kinetics of the system mimicking complex human physiology.  In turn, PBPK modeling can enable more efficient design and optimization of in vivo experiments, and consequently accelerate pre-clinical screening and development.  I will discuss the application of PBPK modeling to an important problem in the medical community – the accelerated clearance of PEGylated drugs in the presence of anti-PEG antibodies (APA).  While this phenomenon renders an entire class of drugs (i.e., PEGylated drugs) ineffective in many patients, the medical community is largely unaware of how drastically this can alter prognosis or how to mitigate this effect.  I will describe a multi-compartment PBPK model to accurately capture and ultimately predict clearance behavior, with the goal of validating results against drug biodistribution data obtained via PET/CT technology.  Specifically, I will focus on the initial transient dynamics as nanomedicines are cleared from the circulation in patients with high APA titers.  I will then discuss further applications for this model in the context of targeted therapeutics." 
abstract2: "The complex community of the human microbiota and its specific role in health maintenance and disease has become an intense topic of study –and debate- over the last years. We do know that microorganisms colonizing human bodies exceed the total number of human cells, and that the number of microbial genes inside our bodies is roughly 100 times higher than the number of genes contained in the human genome, impacting human biology in various ways. For instance, the human immune system is in great part composed of and trained by resident microorganisms, and different microbiome compositions associate with the onset and progression of a large variety of human diseases, including diseases typically considered as non-communicable. Functional, causal links remain largely unexplored in many cases, however, in great part due to sampling limitations, data volume and integration complexity. Due to this gap and the importance of studying microbiome interactions, we have developed methods and tools toward multi-scale analysis and predictive (mechanistic) modeling, as well as integration of multi ‘omics’ data. On the one hand, feature selection and machine learning allows identification of patterns and relationships in large data collections, such as those in human microbiota studies. On the other, mathematical modeling and simulations provide a comprehensive framework to identify connections and key (onset) mechanisms in disease models. Both approaches are essential for analysis and forward engineering personalized therapeutics. In this talk, I will discuss available tools and mathematical methods in this area of research, and what is still needed toward integrative models of host-microbiota dynamics."
abstract3: "During embryo development there is a rapid growth in cell numbers that forms complex structures. Skin pattern formation is an early process during the embryogenesis and happens before the cells fully differentiate. In the present project we consider skin patterning in mouse embryos, where cell aggregates form based on a hierarchical process, involving interactions between the epidermal cell populations. The reaction-diffusion pre-pattern is driven by fibroblast growth factor (FGF20), bone morphogenic protein (BMP) and WNT. Considering mathematical models, there are two main processes involved in the pattern formation: Turing reaction-diffusion systems and chemotaxis. The Turing system models the concentration of two interacting chemicals, and the patterns arises from an instability driven by a difference between their diffusion coefficients. Some previous studies show that this behavior is essential for self-organization in the mouse hair follicle and chicken feather pre-pattern formation. Another key mechanism is chemotaxis, where the cells move in the direction of a chemical attractant, where patterns can also be observed. Experimental data indicates a hierarchical system, where cell chemotaxis is guided by a Turing system. We aim at developing mathematical models to describe the underlying biological processes leading to skin patterning, especially the interaction of chemotaxis with reaction-diffusion (Turing) systems. A mathematical model using partial differential equations is solved numerically, and some results are presented and compared to the experimental data. We study the parameter-dependence of the model and different model structures, and their impact on the pattern forming process. According to the experimental data the Turing system and the chemotaxis seems to be intrinsically related on the mouse skin patterning. Using a numerical approach for the PDE system, we develop a framework to study quantitatively how chemotaxis and Turing systems are related and their impact on the patterning process.During embryo development there is a rapid growth in cell numbers that forms complex structures. Skin pattern formation is an early process during the embryogenesis and happens before the cells fully differentiate. In the present project we consider skin patterning in mouse embryos, where cell aggregates form based on a hierarchical process, involving interactions between the epidermal cell populations. The reaction-diffusion pre-pattern is driven by fibroblast growth factor (FGF20), bone morphogenic protein (BMP) and WNT. Considering mathematical models, there are two main processes involved in the pattern formation: Turing reaction-diffusion systems and chemotaxis. The Turing system models the concentration of two interacting chemicals, and the patterns arises from an instability driven by a difference between their diffusion coefficients. Some previous studies show that this behavior is essential for self-organization in the mouse hair follicle and chicken feather pre-pattern formation. Another key mechanism is chemotaxis, where the cells move in the direction of a chemical attractant, where patterns can also be observed. Experimental data indicates a hierarchical system, where cell chemotaxis is guided by a Turing system. We aim at developing mathematical models to describe the underlying biological processes leading to skin patterning, especially the interaction of chemotaxis with reaction-diffusion (Turing) systems. A mathematical model using partial differential equations is solved numerically, and some results are presented and compared to the experimental data. We study the parameter-dependence of the model and different model structures, and their impact on the pattern forming process. According to the experimental data the Turing system and the chemotaxis seems to be intrinsically related on the mouse skin patterning. Using a numerical approach for the PDE system, we develop a framework to study quantitatively how chemotaxis and Turing systems are related and their impact on the patterning process."
---